{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 119,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'list'>\n"
     ]
    }
   ],
   "source": [
    "with open (\"protein.info.v11.0.txt\", \"r\", encoding=\"utf8\") as myfile:\n",
    "    proteins=myfile.readlines()\n",
    "    myfile.close()\n",
    "print(type(proteins))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "24584629\n"
     ]
    }
   ],
   "source": [
    "print(len(proteins))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 121,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "protein_external_id\tpreferred_name\tprotein_size\tannotation\n",
      "\n",
      "287.DR97_1\tDR97_1\t749\tannotation not available\n",
      "\n",
      "['287.DR97_1', 'DR97_1', '749', 'annotation not available\\n']\n"
     ]
    }
   ],
   "source": [
    "print(proteins[0])\n",
    "print(proteins[1])\n",
    "\n",
    "\n",
    "print(proteins[1].split(\"\t\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "24584629\n",
      "24584629\n",
      "24584629\n"
     ]
    }
   ],
   "source": [
    "protein_names = [protein.split(\"\t\")[1] for protein in proteins]\n",
    "protein_annotations = [protein.split(\"\t\")[3] for protein in proteins]\n",
    "\n",
    "protein_english = [protein.split(\";\")[0] for protein in protein_annotations]\n",
    "\n",
    "print(len(proteins))\n",
    "print(len(protein_names))\n",
    "print(len(protein_annotations))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like protein 1-like\n",
      "\n",
      "Dystrophin-like protein \n",
      "\n",
      "Dystrophin, putative \n",
      "\n",
      "Dystrophin-like protein 1; Together with dys-1 and hlh-1, participates in a common muscular function\n",
      "\n",
      "Dystrophin-1; Plays a role in cholinergic transmission and as a functional partner of dystrobrevin (dyb-1), necessary for muscle maintenance. Required for neuronal positioning\n",
      "\n",
      "Dystrophin, isoforms A/C/F/G/H-like\n",
      "\n",
      "Dystrophin-like isoform X1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin \n",
      "\n",
      "Dystrophin-1-like Protein\n",
      "\n",
      "Dystrophin, isoform D-like Protein\n",
      "\n",
      "Dystrophin, isoforms A/C/F/G/H-like Protein\n",
      "\n",
      "AAEL006651-PA; Dystrophin\n",
      "\n",
      "AGAP010261-PA; Dystrophin Dp427c isoform\n",
      "\n",
      "Dystrophin major muscle isoform\n",
      "\n",
      "Dystrophin, isoforms A/C/F/G/H-like isoform X1\n",
      "\n",
      "Dystrophin, isoform E-like\n",
      "\n",
      "Dystrophin, isoform D-like\n",
      "\n",
      "Dystrophin, isoforms A/C/F/G/H; Required for the maintenance of appropriate synaptic retrograde communication and the stabilization of muscle cell architecture or physiology. Both det and Dg are required for maintenance of early dpp signaling in the presumptive crossvein. Isoform A is not required to maintain muscle integrity, but plays a role in neuromuscular homeostasis by regulating neurotransmitter release. May play a role in anchoring the cytoskeleton to the plasma membrane\n",
      "\n",
      "Uncharacterized protein; Dystrophin, isoform D-like\n",
      "\n",
      "Uncharacterized protein; Dystrophin, isoforms A/C/F/G/H-like\n",
      "\n",
      "Dystrophin, isoform D-like\n",
      "\n",
      "Uncharacterized protein; Dystrophin, isoform E-like\n",
      "\n",
      "Dystrophin-like; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 3 mRNAs, 49 ESTs, 43 Proteins, and 93% coverage of the annotated genomic feature by RNAseq alignments\n",
      "\n",
      "Dystrophin-like protein 1\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin; May play a role in anchoring the cytoskeleton to the plasma membrane\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin; May play a role in anchoring the cytoskeleton to the plasma membrane\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin-related protein 2-like\n",
      "\n",
      "Dystrophin; The sequence of the model RefSeq protein was modified relative to its source genomic sequence to represent the inferred CDS: added 150 bases not found in genome assembly\n",
      "\n",
      "Dystrophin-related protein 2; The sequence of the model RefSeq protein was modified relative to its source genomic sequence to represent the inferred CDS: added 361 bases not found in genome assembly\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 16 Proteins, and 100% coverage of the annotated genomic feature by RNAseq alignments, including 5 samples with support for all annotated introns\n",
      "\n",
      "Dystrophin isoform X1; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 1 EST, 4 Proteins, and 100% coverage of the annotated genomic feature by RNAseq alignments, including 5 samples with support for all annotated introns\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin; May play a role in anchoring the cytoskeleton to the plasma membrane\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin; May play a role in anchoring the cytoskeleton to the plasma membrane\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin; May play a role in anchoring the cytoskeleton to the plasma membrane\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin isoform X1\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin related protein 2; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 23 Proteins, and 100% coverage of the annotated genomic feature by RNAseq alignments, including 3 samples with support for all annotated introns\n",
      "\n",
      "Dystrophin; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 3 Proteins, and 100% coverage of the annotated genomic feature by RNAseq alignments, including 15 samples with support for all annotated introns\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin-related protein 2; Utrophin-like\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin; May play a role in anchoring the cytoskeleton to the plasma membrane\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin isoform X1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin-like isoform X1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin isoform X1\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 6 Proteins, and 99% coverage of the annotated genomic feature by RNAseq alignments, including 1 sample with support for all annotated introns\n",
      "\n",
      "Dystrophin-related protein 2; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 14 Proteins, and 100% coverage of the annotated genomic feature by RNAseq alignments, including 3 samples with support for all annotated introns\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin-related protein 2 isoform 1\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 4 mRNAs, 179 ESTs, 8 Proteins, and 99% coverage of the annotated genomic feature by RNAseq alignments\n",
      "\n",
      "Syntrophin, alpha 1 (Dystrophin-associated protein A1, 59kDa, acidic component); Uncharacterized protein \n",
      "\n",
      "Dystroglycan 1 (Dystrophin-associated glycoprotein 1); Uncharacterized protein \n",
      "\n",
      "Syntrophin, beta 1 (Dystrophin-associated protein A1, 59kDa, basic component 1); Uncharacterized protein \n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin; Anchors the extracellular matrix to the cytoskeleton via F-actin. Ligand for dystroglycan. Component of the dystrophin- associated glycoprotein complex which accumulates at the neuromuscular junction (NMJ) and at a variety of synapses in the peripheral and central nervous systems and has a structural function in stabilizing the sarcolemma. Also implicated in signaling events and synaptic transmission; X-linked mental retardation\n",
      "\n",
      "Dystrophin-related protein 2; Required for normal myelination and for normal organization of the cytoplasm and the formation of Cajal bands in myelinating Schwann cells. Required for normal PRX location at appositions between the abaxonal surface of the myelin sheath and the Schwann cell plasma membrane. Possibly involved in membrane- cytoskeleton interactions of the central nervous system; Zinc fingers ZZ-type\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin; Anchors the extracellular matrix to the cytoskeleton via F-actin. Ligand for dystroglycan. Component of the dystrophin- associated glycoprotein complex which accumulates at the neuromuscular junction (NMJ) and at a variety of synapses in the peripheral and central nervous systems and has a structural function in stabilizing the sarcolemma. Also implicated in signaling events and synaptic transmission\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2 isoform 1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2-like\n",
      "\n",
      "Dystrophin-like isoform 1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2 isoform 1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like isoform X1\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin-related protein 2 isoform 1\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin; Anchors the extracellular matrix to the cytoskeleton via F-actin. Ligand for dystroglycan. Component of the dystrophin- associated glycoprotein complex which accumulates at the neuromuscular junction (NMJ) and at a variety of synapses in the peripheral and central nervous systems and has a structural function in stabilizing the sarcolemma. Also implicated in signaling events and synaptic transmission\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin-related protein 2 isoform X2\n",
      "\n",
      "Dystrophin; The sequence of the model RefSeq protein was modified relative to its source genomic sequence to represent the inferred CDS: added 273 bases not found in genome assembly\n",
      "\n",
      "Dystrophin; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 2 Proteins, and 98% coverage of the annotated genomic feature by RNAseq alignments, including 7 samples with support for all annotated introns\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin isoform X1\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin isoform X3\n",
      "\n",
      "Dystrophin; Anchors the extracellular matrix to the cytoskeleton via F-actin. Ligand for dystroglycan. Component of the dystrophin- associated glycoprotein complex which accumulates at the neuromuscular junction (NMJ) and at a variety of synapses in the peripheral and central nervous systems and has a structural function in stabilizing the sarcolemma. Also implicated in signaling events and synaptic transmission\n",
      "\n",
      "Dystrophin-related protein 2; Required for normal myelination and for normal organization of the cytoplasm and the formation of Cajal bands in myelinating Schwann cells. Required for normal PRX location at appositions between the abaxonal surface of the myelin sheath and the Schwann cell plasma membrane. Possibly involved in membrane-cytoskeleton interactions of the central nervous system\n",
      "\n",
      "Dystrophin; Anchors the extracellular matrix to the cytoskeleton via F-actin. Ligand for dystroglycan. Component of the dystrophin- associated glycoprotein complex which accumulates at the neuromuscular junction (NMJ) and at a variety of synapses in the peripheral and central nervous systems and has a structural function in stabilizing the sarcolemma. Also implicated in signaling events and synaptic transmission\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin-related protein 2; Required for normal myelination and for normal organization of the cytoplasm and the formation of Cajal bands in myelinating Schwann cells. Required for normal PRX location at appositions between the abaxonal surface of the myelin sheath and the Schwann cell plasma membrane. Possibly involved in membrane- cytoskeleton interactions of the central nervous system\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin-like isoform X1\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin, isoforms A/C/F/G/H-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin, isoforms A/C/F/G/H-like\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Dystrophin related protein 2\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin isoform X1\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin; May play a role in anchoring the cytoskeleton to the plasma membrane\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin isoform X1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin; The sequence of the model RefSeq protein was modified relative to its source genomic sequence to represent the inferred CDS: inserted 3 bases in 2 codons; substituted 2 bases at 2 genomic stop codons\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin isoform X1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin; May play a role in anchoring the cytoskeleton to the plasma membrane\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin-related protein 2-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin isoform X1\n",
      "\n",
      "Dystrophin isoform X5; May play a role in anchoring the cytoskeleton to the plasma membrane\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin-like isoform X1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin isoform X1\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 1 mRNA, 30 ESTs, 2 Proteins\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin isoform X1\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Uncharacterized protein; Dystrophin\n",
      "\n",
      "Dystrophin; May play a role in anchoring the cytoskeleton to the plasma membrane\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin, isoform E-like isoform X1\n",
      "\n",
      "Dystrophin, isoform E isoform X1\n",
      "\n",
      "Dystrophin-like isoform X1\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin-like isoform X1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Sentrin-specific protease 2; Dystrophin-like\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin, putative\n",
      "\n",
      "Dystrophin, putative\n",
      "\n",
      "Dystrophin, putative\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin related protein 2\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin related protein 2; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 18 Proteins, and 100% coverage of the annotated genomic feature by RNAseq alignments, including 3 samples with support for all annotated introns\n",
      "\n",
      "Dystrophin; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 8 Proteins, and 100% coverage of the annotated genomic feature by RNAseq alignments, including 2 samples with support for all annotated introns\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin isoform X1\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin-like isoform X1\n",
      "\n",
      "Uncharacterized protein; SMART: SANT SANT  SWI3, ADA2, N-CoR and TFIIIB'' DNA-binding domains; SMART: ZnF_ZZ Zinc-binding domain, present in Dystrophin, CREB-binding protein\n",
      "\n",
      "Putative calcium ion binding protein; SMART: ZnF_ZZ Zinc-binding domain, present in Dystrophin, CREB-binding protein; SMART: EFh EF-hand, calcium binding motif\n",
      "\n",
      "Present in dystrophin, CREB-binding protein; SMART: ZnF_ZZ Zinc-binding domain, present in Dystrophin, CREB-binding protein\n",
      "\n",
      "Dystrophin; May play a role in anchoring the cytoskeleton to the plasma membrane\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 1 Protein, and 5% coverage of the annotated genomic feature by RNAseq alignments\n",
      "\n",
      "Dystrophin-related protein 2-like isoform X1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "LOW QUALITY PROTEIN: dystrophin; Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2\n",
      "\n",
      "Dystrophin, isoform D-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2; Centromere protein I\n",
      "\n",
      "Dystrophin related protein 2; The sequence of the model RefSeq protein was modified relative to its source genomic sequence to represent the inferred complete CDS: deleted 1 base in 1 codon\n",
      "\n",
      "Dystrophin-related protein 2-like isoform X1\n",
      "\n",
      "Dystrophin-like isoform X1\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 13 Proteins, and 99% coverage of the annotated genomic feature by RNAseq alignments\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n",
      "Dystrophin; Derived by automated computational analysis using gene prediction method: Gnomon. Supporting evidence includes similarity to: 1 mRNA, 10 Proteins, and 99% coverage of the annotated genomic feature by RNAseq alignments\n",
      "\n",
      "Dystrophin-like\n",
      "\n",
      "Dystrophin-related protein 2 isoform X1\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for p in protein_annotations:\n",
    "    if \"Dystrophin\" in p:\n",
    "        print(p)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1619037\n"
     ]
    }
   ],
   "source": [
    "names = set(protein_english)\n",
    "print(len(names))\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1619037\n"
     ]
    }
   ],
   "source": [
    "from collections import Counter\n",
    "\n",
    "\n",
    "counted = Counter(protein_english)\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "print(len(counted))\n",
    "\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('annotation not available\\n', 11773370)\n",
      "('Uncharacterized protein\\n', 1503917)\n",
      "('Uncharacterized protein', 1481887)\n",
      "('Predicted protein\\n', 109898)\n",
      "('Derived by automated computational analysis using gene prediction method: Protein Homology\\n', 108291)\n",
      "('Hypothetical protein\\n', 75755)\n",
      "('Membrane protein', 26088)\n",
      "('Putative uncharacterized protein\\n', 20687)\n",
      "('Expressed protein\\n', 18121)\n",
      "('Derived by automated computational analysis using gene prediction method: GeneMarkS+\\n', 15874)\n",
      "('Transcriptional regulator', 15381)\n",
      "('Histidine kinase', 14283)\n",
      "('Upstream in-frame stop codon\\n', 13611)\n",
      "('Conserved hypothetical protein\\n', 12067)\n",
      "('Peptidyl-prolyl cis-trans isomerase', 9653)\n",
      "('ABC transporter, ATP-binding protein', 9579)\n",
      "('PFAM: ABC transporter related', 9498)\n",
      "('KEGG: hypothetical protein\\n', 9167)\n",
      "('Putative membrane protein', 9159)\n",
      "('Predicted protein', 8988)\n",
      "('ABC transporter permease', 8846)\n",
      "('Pseudouridine synthase', 8562)\n",
      "('Transcriptional regulator, LysR family', 8540)\n",
      "('Conserved gene of unknown function.[Source:PGD', 8479)\n",
      "('PFAM: binding-protein-dependent transport systems inner membrane component', 8120)\n",
      "('ABC transporter ATP-binding protein', 7185)\n",
      "('Transposase', 6938)\n",
      "('Uncharacterized protein, isoform A', 6794)\n",
      "('Oxidoreductase', 6722)\n",
      "('PFAM: major facilitator superfamily MFS_1', 6363)\n",
      "('LysR family transcriptional regulator', 6258)\n",
      "('Glycosyltransferase', 6200)\n",
      "('MFS transporter', 6161)\n",
      "('PFAM: short-chain dehydrogenase/reductase SDR', 5980)\n",
      "('Transcriptional regulator, AraC family', 5832)\n",
      "('PFAM: GCN5-related N-acetyltransferase', 5687)\n",
      "('Multifunctional fusion protein', 5657)\n",
      "('Transporter', 5639)\n",
      "('Probable membrane transporter protein', 5590)\n",
      "('AraC family transcriptional regulator', 5417)\n",
      "('COG0583 Transcriptional regulator\\n', 5310)\n",
      "('The sequence of the model RefSeq protein was modified relative to its source genomic sequence to represent the inferred complete CDS: substituted 1 base at 1 genomic stop codon\\n', 5303)\n",
      "('ABC transporter substrate-binding protein', 5135)\n",
      "('Olfactory receptor\\n', 5051)\n",
      "('DNA-directed RNA polymerase subunit beta', 5036)\n",
      "('Peptide deformylase', 4986)\n",
      "('Acyl carrier protein', 4935)\n",
      "('ATP-dependent Clp protease proteolytic subunit', 4794)\n",
      "('Aminotransferase', 4476)\n",
      "('Serine hydroxymethyltransferase', 4447)\n",
      "('Hypothetical protein, conserved.[Source:GeneDB', 4429)\n",
      "('Putative transcriptional regulator', 4408)\n",
      "('TetR family transcriptional regulator', 4334)\n",
      "('DNA helicase', 4273)\n",
      "('The sequence of the model RefSeq protein was modified relative to its source genomic sequence to represent the inferred CDS: substituted 1 base at 1 genomic stop codon\\n', 4238)\n",
      "('Transmembrane protein, putative\\n', 4206)\n",
      "('Transcriptional regulator, TetR family', 4137)\n",
      "('ABC transporter', 4107)\n",
      "('Carbonic anhydrase', 4101)\n",
      "('COG1309 Transcriptional regulator\\n', 4073)\n",
      "('Methyltransferase', 4065)\n",
      "('ABC transporter, permease protein', 4030)\n",
      "('ATP-dependent zinc metalloprotease FtsH', 4016)\n",
      "('ATPase', 3956)\n",
      "('Antitoxin', 3927)\n",
      "('Thioredoxin', 3914)\n",
      "('PFAM: regulatory protein TetR', 3831)\n",
      "('Two component transcriptional regulator, winged helix family', 3809)\n",
      "('RNA polymerase sigma factor', 3797)\n",
      "('Alkyl hydroperoxide reductase AhpD', 3768)\n",
      "('S-adenosylmethionine synthase', 3755)\n",
      "('D-alanine--D-alanine ligase', 3749)\n",
      "('Diguanylate cyclase', 3748)\n",
      "('CTP synthase', 3735)\n",
      "('Transport permease protein', 3735)\n",
      "('Short-chain dehydrogenase/reductase SDR', 3732)\n",
      "('COG0477 Permeases of the major facilitator superfamily\\n', 3732)\n",
      "('Superoxide dismutase', 3713)\n",
      "('Ferredoxin', 3700)\n",
      "('Adenylate kinase', 3666)\n",
      "('Elongation factor Tu', 3651)\n",
      "('Peroxidase', 3649)\n",
      "('60 kDa chaperonin', 3645)\n",
      "('UvrABC system protein A', 3637)\n",
      "('Tyrosine--tRNA ligase', 3634)\n",
      "('Olfactory receptor \\n', 3629)\n",
      "('Alanine racemase', 3623)\n",
      "('Adenylosuccinate synthetase', 3609)\n",
      "('DNA ligase', 3588)\n",
      "('UDP-N-acetylglucosamine 1-carboxyvinyltransferase', 3580)\n",
      "('Acetyltransferase', 3556)\n",
      "('Elongation factor G', 3546)\n",
      "('Peptidylprolyl isomerase', 3541)\n",
      "('ATP synthase subunit beta', 3532)\n",
      "('Alanine--tRNA ligase', 3527)\n",
      "('10 kDa chaperonin', 3495)\n",
      "('Enolase', 3486)\n",
      "('Chaperone protein DnaK', 3485)\n",
      "('Glutamate--tRNA ligase', 3458)\n",
      "('Amidohydrolase', 3452)\n",
      "('4-hydroxy-tetrahydrodipicolinate synthase', 3449)\n",
      "('Carboxylic ester hydrolase', 3446)\n"
     ]
    }
   ],
   "source": [
    "most_common_counted = counted.most_common()\n",
    "\n",
    "\n",
    "for m in range(len(most_common_counted)):\n",
    "    print(most_common_counted[m])\n",
    "    if m>100:\n",
    "        break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['annotation', 'not', 'available\\n']\n",
      "['Uncharacterized', 'protein\\n']\n",
      "['Uncharacterized', 'protein']\n",
      "['Predicted', 'protein\\n']\n",
      "['Derived', 'by', 'automated', 'computational', 'analysis', 'using', 'gene', 'prediction', 'method:', 'Protein', 'Homology\\n']\n",
      "['Hypothetical', 'protein\\n']\n",
      "['Membrane', 'protein']\n",
      "['Putative', 'uncharacterized', 'protein\\n']\n",
      "['Expressed', 'protein\\n']\n",
      "['Derived', 'by', 'automated', 'computational', 'analysis', 'using', 'gene', 'prediction', 'method:', 'GeneMarkS+\\n']\n",
      "['Transcriptional', 'regulator']\n",
      "['Histidine', 'kinase']\n"
     ]
    }
   ],
   "source": [
    "for m in range(len(most_common_counted)):\n",
    "    print(most_common_counted[m][0].split(\" \"))\n",
    "    if m>10:\n",
    "        break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "96356\n"
     ]
    }
   ],
   "source": [
    "most_common_counted = [c[0].lower() for c in most_common_counted if len(c[0].split(\" \")) ==1 ]\n",
    "print(len(most_common_counted))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 129,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "96355\n"
     ]
    }
   ],
   "source": [
    "most_common_counted = [m.rstrip() for m in most_common_counted if m != \"\"]\n",
    "print(len(most_common_counted))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 130,
   "metadata": {},
   "outputs": [],
   "source": [
    "if \"\" in most_common_counted:\n",
    "    print(True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "transposase\n",
      "oxidoreductase\n",
      "glycosyltransferase\n",
      "transporter\n",
      "aminotransferase\n",
      "methyltransferase\n",
      "atpase\n",
      "antitoxin\n",
      "thioredoxin\n",
      "ferredoxin\n",
      "peroxidase\n",
      "acetyltransferase\n",
      "enolase\n",
      "amidohydrolase\n",
      "hexosyltransferase\n",
      "flagellin\n",
      "hydrolase\n",
      "undecaprenyl-diphosphatase\n",
      "amidophosphoribosyltransferase\n",
      "beta-lactamase\n",
      "phospho-n-acetylmuramoyl-pentapeptide-transferase\n",
      "dihydroorotase\n",
      "lipoprotein\n",
      "transketolase\n",
      "inositol-1-monophosphatase\n",
      "ferrochelatase\n",
      "ribokinase\n",
      "peptidase\n",
      "palmitoyltransferase\n",
      "glutaredoxin\n",
      "transaldolase\n",
      "glyoxalase\n",
      "ferritin\n",
      "integrase\n",
      "acyltransferase\n",
      "aminomethyltransferase\n",
      "octanoyltransferase\n",
      "peroxiredoxin\n",
      "adenosylhomocysteinase\n",
      "catalase\n",
      "permease\n",
      "metallophosphoesterase\n",
      "phosphoesterase\n",
      "sulfotransferase\n",
      "aspartokinase\n",
      "porin\n",
      "sulfurtransferase\n",
      "beta-galactosidase\n",
      "flavodoxin\n",
      "dehydrogenase\n",
      "alpha-1,2-mannosidase\n",
      "carboxypeptidase\n",
      "bacterioferritin\n",
      "esterase\n",
      "patatin\n",
      "amidase\n",
      "cupin\n",
      "beta-glucosidase\n",
      "catalase-peroxidase\n",
      "pectinesterase\n",
      "thioesterase\n",
      "aminopeptidase\n",
      "phosphopentomutase\n",
      "ribonuclease\n",
      "arylsulfatase\n",
      "phosphotransferase\n",
      "phosphomannomutase\n",
      "gamma-glutamylcyclotransferase\n",
      "claudin\n",
      "oligoribonuclease\n",
      "similarity:fasta\n",
      "kinase\n",
      "galactokinase\n",
      "alpha-galactosidase\n",
      "laccase\n",
      "alpha-amylase\n",
      "lipase\n",
      "kynureninase\n",
      "sulfatase\n",
      "agmatinase\n",
      "monooxygenase\n",
      "beta-hexosaminidase\n",
      "protease\n",
      "lysozyme\n",
      "gluconokinase\n",
      "beta-n-acetylhexosaminidase\n",
      "helicase\n",
      "derlin\n",
      "phosphodiesterase\n",
      "glucokinase\n",
      "endoglucanase\n",
      "nitroreductase\n",
      "dioxygenase\n",
      "s-acyltransferase\n",
      "endonuclease\n",
      "1,3-beta-glucanosyltransferase\n",
      "rubredoxin\n",
      "dipeptidase\n",
      "beta-xylanase\n",
      "6-phosphogluconolactonase\n",
      "phosphohydrolase\n",
      "alpha/beta-hydrolase\n"
     ]
    }
   ],
   "source": [
    "for m in range(len(most_common_counted)):\n",
    "    print(most_common_counted[m])\n",
    "    if m>100:\n",
    "        break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "True\n",
      "1976\n",
      "dystrophin\n"
     ]
    }
   ],
   "source": [
    "if \"Dystrophin\".lower() in most_common_counted:\n",
    "    print(True)\n",
    "    print(most_common_counted.index(\"Dystrophin\".lower()))\n",
    "    print(most_common_counted[1976])\n",
    "    \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 133,
   "metadata": {},
   "outputs": [],
   "source": [
    "text_file = open(\"protein_list.txt\", \"w\",encoding='utf-8')\n",
    "\n",
    "for p in most_common_counted:\n",
    "    text_file.write(p+\"\\n\")\n",
    "text_file.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "metadata": {},
   "outputs": [],
   "source": [
    "import winsound\n",
    "import time\n",
    "\n",
    "def done():\n",
    "    frequency = 1000  # Set Frequency To 2500 Hertz\n",
    "    duration = 500  # Set Duration To 1000 ms == 1 second\n",
    "    \n",
    "    \n",
    "    for i in range(5):\n",
    "        winsound.Beep(frequency, duration)\n",
    "        time.sleep(0.1)\n",
    "    \n",
    "\n",
    "done()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
