{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pymed import PubMed\n",
    "\n",
    "my_email = \"norockderipa@yahoo.com\"\n",
    "\n",
    "# Create a PubMed object that GraphQL can use to query\n",
    "# Note that the parameters are not required but kindly requested by PubMed Central\n",
    "# https://www.ncbi.nlm.nih.gov/pmc/tools/developers/\n",
    "pubmed = PubMed(tool=\"Protein Interaction Text Miner\", email=my_email)\n",
    "\n",
    "\n",
    "class Publication:\n",
    "    def __init__(self, \n",
    "                 pubmed_id, \n",
    "                 title,\n",
    "                 publication_date,\n",
    "                 abstract,\n",
    "                 keywords = \"\"):\n",
    "        \n",
    "        self.pubmed_id = pubmed_id\n",
    "        self.url = \"https://www.ncbi.nlm.nih.gov/pubmed/\"+pubmed_id.split(\"\\n\")[0]\n",
    "        self.title = title        \n",
    "        self.publication_date = publication_date\n",
    "        self.abstract = abstract\n",
    "        if abstract is not None and abstract is not \"\":\n",
    "            self.words = len(abstract.split(\" \"))  \n",
    "        else:\n",
    "            self.words = 0\n",
    "        self.keywords = keywords"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "def queryPubmed(query,results):   \n",
    "\n",
    "\n",
    "    # Execute the query against the API\n",
    "    results = pubmed.query(query, max_results=results)\n",
    "\n",
    "\n",
    "\n",
    "    # Loop over the retrieved articles\n",
    "    publications = []\n",
    "    for article in results:\n",
    "        if hasattr(article, 'keywords'):\n",
    "            publications +=[Publication(article.pubmed_id,\n",
    "                                        article.title,\n",
    "                                        article.publication_date,\n",
    "                                        article.abstract,\n",
    "                                        article.keywords)]\n",
    "        else:\n",
    "            publications +=[Publication(article.pubmed_id,\n",
    "                                article.title,\n",
    "                                article.publication_date,\n",
    "                                article.abstract)]\n",
    "    \n",
    "    publications = [publication for publication in publications if publication.words is not 0]\n",
    "    \n",
    "    publications = [publication for publication in publications if query in publication.abstract]\n",
    "    \n",
    "    print(\"Extracted\",len(publications),\"articles\")\n",
    "    \n",
    "    return publications"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "207468 proteins imported\n",
      "236736 common words imported\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[nltk_data] Downloading package words to\n",
      "[nltk_data]     C:\\Users\\Frost\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package words is already up-to-date!\n"
     ]
    }
   ],
   "source": [
    "with open (\"protein_list.txt\", \"r\", encoding=\"utf8\") as myfile:\n",
    "    lines=myfile.readlines()\n",
    "    myfile.close()\n",
    "\n",
    "proteins = [l.rstrip() for l in lines]    \n",
    "print(len(proteins),\"proteins imported\")\n",
    "\n",
    "import nltk\n",
    "nltk.download('words')\n",
    "from nltk.corpus import words\n",
    "word_list = words.words()\n",
    "# prints 236736\n",
    "\n",
    "\n",
    "common = [w for w in word_list]\n",
    "print(len(common),\"common words imported\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import random\n",
    "import joblib\n",
    "import pickle\n",
    "from keras.preprocessing.text import Tokenizer\n",
    "from keras.preprocessing.sequence import pad_sequences\n",
    "\n",
    "def wordSplitter(string, length):\n",
    "    \n",
    "    words = string.split(\" \")\n",
    "    \n",
    "    lists = list((words[0+i:length+i] for i in range(0, len(words), length)))        \n",
    "        \n",
    "    sentences = [\" \".join(l) for l in lists]        \n",
    "        \n",
    "    return sentences\n",
    "\n",
    "\n",
    "\n",
    "PPIDetector = joblib.load('PPINetwork.pkl')\n",
    "\n",
    "with open('tokenizer.pickle', 'rb') as handle:\n",
    "    tokenizer = pickle.load(handle) \n",
    "\n",
    "\n",
    "def PPI_Detection_Network(text):    \n",
    "    \n",
    "    tokenized = tokenizer.texts_to_sequences([text])\n",
    "    \n",
    "    padded_tokenized = pad_sequences(tokenized, maxlen=50)\n",
    "    \n",
    "    prediction = round(PPIDetector.predict([[padded_tokenized[0]]])[0][0]  + 0.1)\n",
    "    \n",
    "    return prediction\n",
    "\n",
    "\n",
    "\n",
    "def PPI_Estimator(abstract):\n",
    "    \n",
    "    results = []\n",
    "    \n",
    "    for piece in wordSplitter(abstract, 50):\n",
    "        results += [PPI_Detection_Network(piece)]\n",
    "    \n",
    "    return bool(round(sum(results)/len(results)))\n",
    "\n",
    "def getPossiblePPP(publications):\n",
    "    \n",
    "    possibleInteractions = []\n",
    "    \n",
    "    for p in publications:\n",
    "        if PPI_Estimator(p.abstract):\n",
    "            possibleInteractions += [p]\n",
    "        \n",
    "    return possibleInteractions\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "import nltk\n",
    "\n",
    "def processString(sentence):\n",
    "    text=sentence.lower()\n",
    "    tokens = nltk.word_tokenize(text)\n",
    "    #punctuation\n",
    "    tokens=nltk.tokenize.RegexpTokenizer(r'\\w+').tokenize(text)\n",
    "    #stopwords\n",
    "    stopwords=set(nltk.corpus.stopwords.words('english'))\n",
    "    tokens=list(filter(lambda word: not word in stopwords,tokens))\n",
    "    \n",
    "    \n",
    "    return set(tokens)\n",
    "\n",
    "\n",
    "\n",
    "import joblib\n",
    "import spacy\n",
    "import scispacy\n",
    "nlp = spacy.load(\"en_core_sci_sm\")\n",
    "\n",
    "def word2vec(x):\n",
    "    return nlp(x).vector\n",
    "\n",
    "\n",
    "proteinRecognitionNetwork = joblib.load('proteinRecognitionNetwork.pkl')\n",
    "\n",
    "def getProteinPrediction(word):\n",
    "    return round(proteinRecognitionNetwork.predict(np.array([word2vec(word)]))[0][0])\n",
    "\n",
    "\n",
    "def getCandidateProteins(abstract):\n",
    "    candidate_words = processString(abstract)\n",
    "    candidates = []\n",
    "    for w in candidate_words:\n",
    "        #if w in proteins:\n",
    "            #candidates +=[w]\n",
    "        if getProteinPrediction(w):\n",
    "            candidates +=[w]\n",
    "    return candidates\n",
    "\n",
    "\n",
    "\n",
    "def findRelations(protein,abstract):\n",
    "    \n",
    "    protein = protein.lower()\n",
    "    \n",
    "    abstract = abstract.lower()\n",
    "    \n",
    "    #print(abstract)\n",
    "    #print()\n",
    "    \n",
    "    abstract = abstract.replace(protein,\"PROTEINA\")\n",
    "    \n",
    "    #print(abstract)\n",
    "    #print()\n",
    "    \n",
    "    candidates = getCandidateProteins(abstract)\n",
    "    \n",
    "    print(candidates)\n",
    "    \n",
    "    if protein in candidates:\n",
    "        candidates.remove(protein)\n",
    "        \n",
    "    finalListOfProteins = []\n",
    "        \n",
    "    for candidate in candidates:\n",
    "        tempAbstract = abstract\n",
    "        tempAbstract = tempAbstract.replace(candidate,\"PROTEINB\")\n",
    "\n",
    "        #print(candidate)\n",
    "        #print(tempAbstract)\n",
    "        #print()\n",
    "        \n",
    "        if PPI_Estimator(tempAbstract):\n",
    "            finalListOfProteins +=[candidate]\n",
    "    \n",
    "    #print(finalListOfProteins)\n",
    "    \n",
    "    return finalListOfProteins\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Extracted 6 articles\n",
      "<class 'list'>\n"
     ]
    }
   ],
   "source": [
    "publications = queryPubmed(query = \"Dystrophin\",results = 50)\n",
    "\n",
    "print(type(publications))\n",
    "\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "def checkArticlesForPPI(protein):\n",
    "\n",
    "    publications = queryPubmed(query = protein,results = 500)\n",
    "\n",
    "    interactions = []\n",
    "    \n",
    "    for p in publications:\n",
    "              \n",
    "        interactions += findRelations(\"Dystrophin\",p.abstract)\n",
    "        \n",
    "    return interactions\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Extracted 58 articles\n",
      "['years', 'ef', 'provides', 'cell', 'functions', 'c', 'zz', 'two', 'dystroglycan', 'membrane', 'Î²', 'ww', 'protein', 'beta']\n",
      "['diseases', 'mutations', 'proteinaopathies']\n",
      "['cell', 'role', 'membrane', 'term', 'neurons', 'long', 'synapses', 'symptoms', 'mice', '427', '2j', 'kda', 'ca1', 'mdx', 'dmd']\n",
      "['another', 'mutations', '356', 'x', 'lab', 'patients', '65', 'exons', 'proteinaopathies', 'six', '2', 'xp21', '15', 'allows', 'polymerase', 'mlpa', 'applications', '27', 'male', 'years', 'lead', '10', 'spot']\n",
      "['60', 'processes', 'lamp2', 'adeno', 'transcription', 'factor', 'protein', 'tfeb', 'extent', '1Î±', 'mice', 'old', 'eb', 'p62', 'via', 'gastrocnemii', 'lc3ii', '6', '3', 'appears', 'mdx', 'dmd']\n",
      "['moctn3', 'moctn1', 'disrupts', 'octn2', 'long', 'qrt', 'peroxisomal', 'mitochondrial', 'mice', 'transporter', 'octn1', 'octn3', 'acyl', 'cn', 'male', 'mrna', '10scsn', 'moctn2', 'membrane', '10snj', 'c57bl', 'l', 'mdx', 'protein', 'dmd', 'j']\n",
      "['junctions', 'glycoprotein', 'dystroglycans', 'utrophin', 'symptoms', 'actin', 'deficiencies', 'homolog', 'role', 'sarcospan', 'protein', 'nmjs']\n",
      "['receptor', 'ds', 'cdn1163', 'lab', '11966966', 'antioxidant', 'extent', 'mice', 'molecule', '95', '70', '21', 'presence', 'sensitizes', 'small', 'nac', 'variables', '10', 'mdx', 'ranges', 'dmd']\n",
      "['vibrations', 'two', 'patients', 'lines', 'dysferlin', 'utrophin', 'mice', 'types', 'problems', 'structure', 'diseases', 'number', 'dystroglycan', 'Î²', 'analyses', 'strains']\n",
      "['5', 'junctions', 'utrophin', 'article', 'utr', 'translation', 'x', 'two', 'adults', 'c2c12', 'dmd', 'variants']\n",
      "['mutations', 'x', 'mate', 'high', 'protein', 'mice', 'allows', 'male', '57', 'mdx', 'dmd', 'doses']\n",
      "['cell', 'mutations', 'patients', 'high', 'neuropathophysiology', 'protein', 'symptoms', 'functions', 'become', 'dna', 'correlations', 'frameshift', 'expressed', 'many', 'localises', 'suggests', '71', 'dp71', 'dmd', 'translation']\n",
      "['fgf', 'receptor', 'co', 'hypothesize', 'factor', 'protein', 'osteoclasts', 'klb', 'utrophin', 'mice', 'bones', 'children', 'non', 'expressed', '21', 'mrna', 'fgfrs', '2019', 'macrophages', 'p', 'via', 'fgf21', 'small', 'Î²', 'dmd', 'klotho']\n",
      "['mri', 'mutations', 'patients', '22', '4', 'proteinaopathies', 'glycoprotein', '11', '2', '14', 'allows', 'lgmd2i', '9', '21', '2i']\n",
      "['cell', 'prevents', 'another', 'two', 'membrane', 'sans', 'protein', '11', 'orientations', 'r19', 'c', 'r11', '2', '15', 'constraints', 'r12', 'abd2', 'measurements', 'actin', 'r15', 'binding', 'dystroglycan', 'r4', 'spectrin', 'dmd']\n",
      "['mab', 'assessed', 'dmd']\n",
      "['environments', 'cell', 'membrane', 'full', 'protein', 'glycoprotein', 'Î³', 'sds', 'hydrophobic', 'structure', 'sarcoglycan', 'number', 'glycoproteins']\n",
      "['co', 'mutations', '20', 'pmo', 'implications', 'mice', '2', 'peptide', 'kg', 'ao', 'gf', 'mdx', 'dmd']\n",
      "['improvements', 'mutations', 'x', 'patients', 'dec', 'mdx', 'dmd', 'mb']\n",
      "['001', 'patients', 'adults', '24', 'twist', 'grmd', 'echocardiography', '7', '2', 'old', '50', '9', 'dogs', 'p', 'lv', 'dmd', '23']\n",
      "['5', 'mlpa', '51', '50', '55', '49', 'two', 'patients', '45', 'str', '3', 'dmd', 'tandem', 'exons']\n",
      "['cs', 'side', 'dmd']\n",
      "['30', 'x', 'patients', 'protein', 'proteinaopathies', 'ef', 'crt', 'symptoms', 'icd', 'cm', 'devices', 'ht', 'parallel', 'hf', 'evolves', 'far', 'pro', 'dmd', 'xl']\n",
      "['genome', 'cell', 'mutations', 'x', '20', 'role', 'protein', 'functions', 'mrna', 'number', 'suggests', 'cas9', 'dmd']\n",
      "['hiv', 'cell', 'mutations', 'patients', 'transcription', 'protein', 'mice', 'r16', 'gfp', '17', 'expressed', 'binding', 'many', 'activator', 'tat', '16', 'diseases', 'trans', 'synthase', 'among', 'spectrin', 'dmd']\n",
      "['51', '30', 'mutations', 'baseline', '25', 'two', 'patients', 'term', '24', '6mwt', 'versus', '12', 'long', 'combined', 'ten', '202', 'aims', '256', 'kg', '50', '418', 'markers', '240', '201', '341', '28', '10', '6', '330', 'infusions', 'dmd']\n",
      "['60', 'years', '5', 'symptoms', 'x', 'patients', '2', '3', '67', '1', 'dmd']\n",
      "['60', 'two', 'micturition', 'protein', 'become', 'dp140', '210', '202', 'expressed', 'problems', 'markers', '2019', 'rat', 'dp71', 'dmd']\n",
      "['mice', 'hypertrophy', 'yap', 'underlies', 'protein']\n",
      "['cell', 'processes', 'patients', 'term', 'long', 'mice', '2', 'side', 'cdc', 'mdx', 'dmd']\n",
      "['ig', 'patients', 'protein', 'glycoprotein', 'utrophin', 'functions', 'c', 'titin', 'actin', 'binding', 'many', 'stabilizes', 'element', 'via', 'dystroglycan', 'Î²', 'spectrin', 'dmd']\n",
      "['7497_7506delggtgggtgac', 'mutations', 'variations', '9421_9422delaa', '22', 'database', '4', '8', '3487c', 'c', '14', 'frameshift', 'children', 'ngs', 'mlpa', '1208delg', '8910_8913deltctc', '6290g', '1', 'dmd']\n",
      "['glutamate', 'cdc42', 'role', 'dystrobrevin', 'gtpase', 'glycoprotein', 'junctions', 'release', 'rho', 'conserved']\n",
      "['atp', 'cell', 'receptor', 'mutations', 'patients', 'high', 'ecto', 'symptoms', 'combined', 'p2x7', 'non', 'release', 'diseases', 'sarcoglycan', 'lead', 'atpase', 'appears', 'secondary', 'dmd']\n",
      "['mbp', 'full', 'transcription', 'protein', 'long', 'functions', 'mice', 'ko', '2', '100', 'mitochondrial', 'binding', '26', 'kbp']\n",
      "['instars', 'adults', 'role', 'protein', 'pupae', 'long', 'aim', 'viruses', 'actin', 'tropomyosin', 'via', 'iv', 'herbs', 'plantlets', 'roots', 'conserved']\n",
      "['provides', 'neurons', 'receptor', 'functions', 'two', 'components', 'dp186']\n",
      "['mutations', 'x', 'two', 'patients', 'protein', 'xcr', 'ion', 'alleles', 'rnaseq', 'qrt', 'male', 'mrna', 'analyses', 'dmd']\n",
      "['ratios', 'markers', 'patients', '22', '24', 'protein', 'dmd', 'dynamic', 'children']\n",
      "['60', 'cell', 'receptor', 'd20', '20', 'neurons', 'dp140', 'mice', 'compartments', 'd10', 'dps', 'comprises', 'via', 'dystroglycan', '10', 'Î²', 'dp71', 'suggests', 'sub', 'dp427']\n",
      "['six', '2032_2033delca', 'nucleotide', 'c', 'x', 'dna', '2', 'probable', 'case', 'q678dfsx41', 'discusses', 'dmd', 'follicle', 'p']\n",
      "['5', 'cell', 'dp71ab', 'mutations', 'patients', '74', 'role', 'transcription', 'full', '24', 'protein', 'exons', 'six', '8', 'c', 'ten', '15', '17', 'g1', 'polymerase', 'kda', '78', '18', 'u', 'dp71bc', 'mrna', '68', '16', '76', '77', '216', 'dp71b', '73', '79', 'dp71g', 'amino', 'dp71', '71', 'dmd', '251', 'variants']\n",
      "['adeno', 'long', 'mice', 'components', 'old', '18', 'diseases', '6', 'mdx', 'dmd']\n",
      "['cell', '25', 'nm', 'membrane', '24', 'protein', 'pn', '427', 'kda', 'expressed', '800', 'cytoplasmic', 'costameres', 'spectrin']\n",
      "['5', 'mutations', 'boys', '938', 'span', 'iq', '24', 'iii', 'versus', 'executive', 'contributions', 'set', '17', 'children', 'ses', '50', 'assessed', '43', '928', 'analyses', 'assessments', '2018']\n",
      "['mutations', 'role', 'protein', 'ion', 'proteinaopathies']\n",
      "['role', 'syntrophin', '24', 'protein', 'implications', 'termini', 'long', 'functions', 'aim', '17', 'binding', 'Î±', 'amino', 'suggests', 'synthase', 'appears', 'spectrin', 'dmd']\n",
      "['co', 'another', 'mutations', 'boys', 'x', 'patients', 'database', '4', 'exons', 'proteinaopathies', '36', '2', '15', 'non', 'others', '40', '32', 'mlpa', '38', 'wes', 'diseases', 'small', '94', 'hotspots', 'individuals', 'dmd', 'variants']\n",
      "['cell', 'advancements', 'mutations', 'two', 'patients', 'g', 'protein', 'proteinaopathies', 'efforts', 'carte', 'la', 'diseases', 'via', 'among', 'dmd']\n",
      "['patient', 'protein', 'shifts', 'mice', 'mir', 'types', 'release', 'expressed', 'piwi', '483', 'fragment', 'sera', 'diseases', 'small', 'mdx', 'dmd', 'biology']\n",
      "['cell', 'mutations', 'role', 'ripk1', 'mice', 'c2c12', 'ripk3', 'mdx', 'dmd']\n",
      "['hz', '3d', 'myobundle', '3mn', '10', 'membrane', '2', '3', 'myotubes', '1', 'protein', '19', 'mm']\n",
      "['functions', 'mice', 'assessed', 'two', '2', '1', 'mdx']\n",
      "['cell', 'myotube', 'myotubes', 'patient', 'protein', 'laminin', 'iih6', 'heparin', '14', 'non', 'types', 'Î±', 'parallel', 'dystroglycan', 'dmd']\n",
      "['years', 'interventions', 'become', 'patients', 'number', 'discusses']\n",
      "['patients', 'term', 'ie', 'long', 'utrophin', 'symptoms', 'mice', 'ta', 'scn4a', 'tnni1', 'calcineurin', 'measurements', '1', 'cacna1s', 'myh2', 'cyclosporin', 'mdx']\n",
      "['34', 'patients', 'protein', '4', 'durations', 'proteinaopathies', 'long', 'aim', '36', '2', '14', '9', 'male', 'years', 'oxygen', '10', 'suggests', 'synthase', 'secondary']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['co', 'high', '4998_5000', 'c', 'cag', 'p', '73', 'small', '1667', 'sub']\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "checkArticlesForPPI(\"Dystrophin\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
