{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<H1>Dataset Loading and Interaction Extraction</h1>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "class Sentence:\n",
    "    def __init__(self):\n",
    "        self.text = \"\"\n",
    "        self.proteins = []\n",
    "        self.interactions = []\n",
    "        self.reconstructedText = \"\"\n",
    "        self.words = []\n",
    "        self.valid = False\n",
    "\n",
    "    def show(self):\n",
    "        print(self.text)\n",
    "        print(self.proteins)\n",
    "        if len(self.interactions):\n",
    "            for i in self.interactions:\n",
    "                print(i)\n",
    "        print()\n",
    "        \n",
    "        \n",
    "    def processText(self):\n",
    "        #self.text = self.text.replace(\"_/_\",\"_ _\")\n",
    "        #self.text = self.text.replace(\"_:_\",\"_ _\")\n",
    "        #self.text = self.text.replace(\"_-_\",\"_ _\")\n",
    "        self.text = re.sub(\"(\\_[^_]_)\", \"_ _\", self.text)#regexp to remove things like _:_\n",
    "        self.text = self.text.replace(\"(_\",\"_\")\n",
    "        self.text = self.text.replace(\"_)\",\"_\")\n",
    "    \n",
    "    def checkValidity(self):\n",
    "        count = 0\n",
    "        for w in range(len(self.words)):\n",
    "            if \"__\" in self.words[w]:\n",
    "                count = count +1\n",
    "        \n",
    "        if count == len(self.proteins):\n",
    "            self.valid = True\n",
    "            #print(self.valid)\n",
    "        \n",
    "    \n",
    "    def reconstructText(self):\n",
    "        self.words = self.text.split(\" \")\n",
    "        self.checkValidity()\n",
    "        #print(self.words)\n",
    "        if self.valid:\n",
    "            p=0\n",
    "            for w in range(len(self.words)):\n",
    "                if \"__\" in self.words[w]:\n",
    "                    self.words[w] = self.proteins[p]\n",
    "                    #self.words[w] = \"PROTEIN\"+str(p)\n",
    "                    p=p+1\n",
    "            #print(self.words)\n",
    "            #print()\n",
    "        \n",
    "    def reassembleSentence(self):\n",
    "        print(\" \".join(self.words))\n",
    "        \n",
    "    def fullProcess(self):\n",
    "        self.processText()\n",
    "        self.reconstructText()\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2245\n",
      "529\n"
     ]
    }
   ],
   "source": [
    "import pickle\n",
    "\n",
    "with open('datasetTrain.pkl', 'rb') as input1:\n",
    "    trainSetRaw = pickle.load(input1)\n",
    "    \n",
    "with open('datasetTest.pkl', 'rb') as input2:    \n",
    "    testSetRaw = pickle.load(input2)\n",
    "    \n",
    "try:\n",
    "    print(len(trainSetRaw))\n",
    "    print(len(testSetRaw))\n",
    "except NameError:\n",
    "    print(\"nope\")   "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "def findMaxWords(inputList):\n",
    "    maxwords = -1 \n",
    "    for i in inputList:\n",
    "        if len(i.words)>maxwords:\n",
    "            maxwords = len(i.words)\n",
    "    print(maxwords)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "93\n",
      "74\n"
     ]
    }
   ],
   "source": [
    "findMaxWords(trainSetRaw)\n",
    "findMaxWords(testSetRaw)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "We start out with\n",
      "2245 training examples\n",
      "529 test examples\n",
      "\n",
      "mean 24.980894015861573\n",
      "std 10.856406734698178\n",
      "80 words: 2770\n",
      "70 words: 2766\n",
      "60 words: 2751\n",
      "50 words: 2697\n",
      "48 words: 2674\n",
      "40 words: 2544\n",
      "30 words: 2086\n",
      "\n",
      "Selecting for under 48 words we are left with\n",
      "2166 training examples\n",
      "508 test examples\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAX0AAAD7CAYAAACG50QgAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4xLjIsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy8li6FKAAAO5ElEQVR4nO3df6hfd33H8edriVbbUkzX2xKTbIkQ1FRwdZeu6hBZhXZWTP8py6BbkI7AyGYVh0v8R/ZHoAMRHaxCqLoMxRJqoUGds0RlG2ztUitoGkuD6ZJrY3N1+GP+UU1974/v6fZtctP0fr/pvTd9Px9Qzjnv7+d8P5/7Iff1PT3ne85NVSFJ6uE3lnsAkqSlY+hLUiOGviQ1YuhLUiOGviQ1YuhLUiPnDf0kn0lyKsl3x2pXJnkwyRPDcs3Ya7uTHE3yeJKbxuq/m+Q7w2t/lyQX/seRJL2QF3Ok/w/AzWfUdgEHq2ozcHDYJskWYBtw7bDP3UlWDft8CtgBbB7+O/M9JUkvsdXna1BV/5Jk4xnlrcA7h/V9wDeBvx7q91bVM8CxJEeB65M8CVxRVf8OkOQfgVuBfzpf/1dddVVt3Hhm95KkF/LII4/8qKpmzqyfN/TP4ZqqOglQVSeTXD3U1wH/MdZubqj9alg/s35eGzdu5NChQxMOU5J6SvJfC9Uv9IXchc7T1wvUF36TZEeSQ0kOzc/PX7DBSVJ3k4b+00nWAgzLU0N9Dtgw1m498NRQX79AfUFVtbeqZqtqdmbmrP87kSRNaNLQPwBsH9a3Aw+M1bcluSTJJkYXbB8eTgX9PMkNw7d2/nRsH0nSEjnvOf0kX2B00faqJHPAR4G7gP1J7gCOA7cBVNXhJPuBx4DTwM6qenZ4qz9n9E2gVzO6gHvei7iSpAsrK/3RyrOzs+WFXElanCSPVNXsmXXvyJWkRgx9SWrE0JekRgx9SWpk0jty9QI27vrysvX95F23LFvfklY+j/QlqRFDX5IaMfQlqRFDX5IaMfQlqRFDX5IaMfQlqRFDX5IaMfQlqRFDX5IaMfQlqRGfvfMys1zP/fGZP9LFwSN9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRgx9SWpkqtBP8sEkh5N8N8kXkrwqyZVJHkzyxLBcM9Z+d5KjSR5PctP0w5ckLcbEoZ9kHfB+YLaq3gSsArYBu4CDVbUZODhsk2TL8Pq1wM3A3UlWTTd8SdJiTHt6ZzXw6iSrgUuBp4CtwL7h9X3ArcP6VuDeqnqmqo4BR4Hrp+xfkrQIE4d+Vf0A+BhwHDgJ/LSqvgZcU1UnhzYngauHXdYBJ8beYm6oSZKWyDSnd9YwOnrfBLwWuCzJ7S+0ywK1Osd770hyKMmh+fn5SYcoSTrDNKd33gUcq6r5qvoVcD/wNuDpJGsBhuWpof0csGFs//WMTgedpar2VtVsVc3OzMxMMURJ0rhpQv84cEOSS5MEuBE4AhwAtg9ttgMPDOsHgG1JLkmyCdgMPDxF/5KkRVo96Y5V9VCS+4BvAaeBR4G9wOXA/iR3MPpguG1ofzjJfuCxof3Oqnp2yvFLkhZh4tAHqKqPAh89o/wMo6P+hdrvAfZM06ckaXLekStJjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjUwV+klek+S+JN9LciTJW5NcmeTBJE8MyzVj7XcnOZrk8SQ3TT98SdJiTHuk/0ngq1X1BuDNwBFgF3CwqjYDB4dtkmwBtgHXAjcDdydZNWX/kqRFmDj0k1wBvAP4NEBV/bKqfgJsBfYNzfYBtw7rW4F7q+qZqjoGHAWun7R/SdLiTXOk/zpgHvhskkeT3JPkMuCaqjoJMCyvHtqvA06M7T831M6SZEeSQ0kOzc/PTzFESdK4aUJ/NfAW4FNVdR3wC4ZTOeeQBWq1UMOq2ltVs1U1OzMzM8UQJUnjpgn9OWCuqh4atu9j9CHwdJK1AMPy1Fj7DWP7rweemqJ/SdIiTRz6VfVD4ESS1w+lG4HHgAPA9qG2HXhgWD8AbEtySZJNwGbg4Un7lyQt3uop9/9L4PNJXgl8H3gfow+S/UnuAI4DtwFU1eEk+xl9MJwGdlbVs1P2L0lahKlCv6q+Dcwu8NKN52i/B9gzTZ+SpMl5R64kNWLoS1Ijhr4kNWLoS1Ijhr4kNWLoS1Ijhr4kNWLoS1Ijhr4kNWLoS1Ijhr4kNWLoS1Ijhr4kNWLoS1Ijhr4kNWLoS1Ijhr4kNWLoS1Ijhr4kNWLoS1Ijhr4kNWLoS1Ijhr4kNWLoS1Ijhr4kNWLoS1Ijhr4kNWLoS1Ijhr4kNWLoS1Ijhr4kNbJ6uQegl4eNu768bH0/edcty9a3dLHxSF+SGjH0JamRqUM/yaokjyb50rB9ZZIHkzwxLNeMtd2d5GiSx5PcNG3fkqTFuRBH+ncCR8a2dwEHq2ozcHDYJskWYBtwLXAzcHeSVRegf0nSizRV6CdZD9wC3DNW3grsG9b3AbeO1e+tqmeq6hhwFLh+mv4lSYsz7ZH+J4APA78eq11TVScBhuXVQ30dcGKs3dxQO0uSHUkOJTk0Pz8/5RAlSc+ZOPSTvAc4VVWPvNhdFqjVQg2ram9VzVbV7MzMzKRDlCSdYZrv6b8deG+SdwOvAq5I8jng6SRrq+pkkrXAqaH9HLBhbP/1wFNT9C9JWqSJj/SrandVra+qjYwu0H69qm4HDgDbh2bbgQeG9QPAtiSXJNkEbAYennjkkqRFeynuyL0L2J/kDuA4cBtAVR1Osh94DDgN7KyqZ1+C/iVJ53BBQr+qvgl8c1j/MXDjOdrtAfZciD4lSYvnHbmS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNGPqS1IihL0mNrF7uAbyUNu768nIPQZJWFI/0JamRiUM/yYYk30hyJMnhJHcO9SuTPJjkiWG5Zmyf3UmOJnk8yU0X4geQJL140xzpnwY+VFVvBG4AdibZAuwCDlbVZuDgsM3w2jbgWuBm4O4kq6YZvCRpcSYO/ao6WVXfGtZ/DhwB1gFbgX1Ds33ArcP6VuDeqnqmqo4BR4HrJ+1fkrR4F+ScfpKNwHXAQ8A1VXUSRh8MwNVDs3XAibHd5oaaJGmJTB36SS4Hvgh8oKp+9kJNF6jVOd5zR5JDSQ7Nz89PO0RJ0mCq0E/yCkaB//mqun8oP51k7fD6WuDUUJ8DNoztvh54aqH3raq9VTVbVbMzMzPTDFGSNGaab+8E+DRwpKo+PvbSAWD7sL4deGCsvi3JJUk2AZuBhyftX5K0eNPcnPV24E+A7yT59lD7CHAXsD/JHcBx4DaAqjqcZD/wGKNv/uysqmen6F+StEgTh35V/RsLn6cHuPEc++wB9kzapyRpOt6RK0mNGPqS1IihL0mNvKyfsqkelutpqk/edcuy9CtNwyN9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRgx9SWrE0JekRvzD6NKElusPsoN/lF2T80hfkhox9CWpEUNfkhox9CWpEUNfkhox9CWpEUNfkhox9CWpEW/Oki5Cy3VjmDeFXfyW/Eg/yc1JHk9yNMmupe5fkjpb0tBPsgr4e+APgS3AHyfZspRjkKTOlvr0zvXA0ar6PkCSe4GtwGNLPA5JE/C00sVvqUN/HXBibHsO+L0lHoMkvWgvtw+6pQ79LFCrsxolO4Adw+b/JPkx8KOXcmAXoatwTsY5H8/3spqP/O0FeZuLak4uwM/82wsVlzr054ANY9vrgafObFRVe4G9z20nOVRVsy/98C4ezsnzOR/P53yczTkZWepv7/wnsDnJpiSvBLYBB5Z4DJLU1pIe6VfV6SR/AfwzsAr4TFUdXsoxSFJnS35zVlV9BfjKInfbe/4m7Tgnz+d8PJ/zcTbnBEjVWddRJUkvUz57R5IaWfGh3/2xDUk2JPlGkiNJDie5c6hfmeTBJE8MyzXLPdallGRVkkeTfGnY7j4fr0lyX5LvDf9W3tp5TpJ8cPh9+W6SLyR5Vef5GLeiQ9/HNgBwGvhQVb0RuAHYOczBLuBgVW0GDg7bndwJHBnb7j4fnwS+WlVvAN7MaG5azkmSdcD7gdmqehOjL41so+l8nGlFhz5jj22oql8Czz22oY2qOllV3xrWf87ol3kdo3nYNzTbB9y6PCNceknWA7cA94yVO8/HFcA7gE8DVNUvq+onNJ4TRl9SeXWS1cCljO4H6jwf/2elh/5Cj21Yt0xjWXZJNgLXAQ8B11TVSRh9MABXL9/IltwngA8Dvx6rdZ6P1wHzwGeHU173JLmMpnNSVT8APgYcB04CP62qr9F0Ps600kP/RT22oYMklwNfBD5QVT9b7vEslyTvAU5V1SPLPZYVZDXwFuBTVXUd8AuanroAGM7VbwU2Aa8FLkty+/KOauVY6aH/oh7b8HKX5BWMAv/zVXX/UH46ydrh9bXAqeUa3xJ7O/DeJE8yOt33B0k+R9/5gNHvyVxVPTRs38foQ6DrnLwLOFZV81X1K+B+4G30nY/nWemh3/6xDUnC6Fztkar6+NhLB4Dtw/p24IGlHttyqKrdVbW+qjYy+vfw9aq6nabzAVBVPwROJHn9ULqR0ePKu87JceCGJJcOvz83MroW1nU+nmfF35yV5N2MzuE+99iGPcs8pCWV5PeBfwW+w/+fw/4Io/P6+4HfYvSP/Laq+u9lGeQySfJO4K+q6j1JfpPG85Hkdxhd2H4l8H3gfYwO6lrOSZK/Af6I0bffHgX+DLicpvMxbsWHviTpwlnpp3ckSReQoS9JjRj6ktSIoS9JjRj6ktSIoS9JjRj6ktSIoS9JjfwvxL/jXnvt4d8AAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 432x288 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "print(\"We start out with\")\n",
    "\n",
    "print(len(trainSetRaw),\"training examples\")\n",
    "print(len(testSetRaw),\"test examples\")\n",
    "\n",
    "print()\n",
    "\n",
    "allSet = [len(x.words) for x in trainSetRaw+testSetRaw]\n",
    "\n",
    "from matplotlib import pyplot as plt \n",
    "import numpy as np  \n",
    "\n",
    "print(\"mean\",np.mean(allSet))\n",
    "\n",
    "print(\"std\",np.std(allSet))\n",
    "\n",
    "plt.hist(allSet)\n",
    "\n",
    "\n",
    "\n",
    "selectedSet = [x for x in trainSetRaw+testSetRaw if len(x.words)<=80 ]\n",
    "\n",
    "print(\"80 words:\",len(selectedSet))\n",
    "\n",
    "\n",
    "selectedSet = [x for x in trainSetRaw+testSetRaw if len(x.words)<=70 ]\n",
    "\n",
    "print(\"70 words:\",len(selectedSet))\n",
    "\n",
    "selectedSet = [x for x in trainSetRaw+testSetRaw if len(x.words)<=60 ]\n",
    "\n",
    "print(\"60 words:\",len(selectedSet))\n",
    "\n",
    "\n",
    "selectedSet = [x for x in trainSetRaw+testSetRaw if len(x.words)<=50 ]\n",
    "\n",
    "print(\"50 words:\",len(selectedSet))\n",
    "\n",
    "selectedSet = [x for x in trainSetRaw+testSetRaw if len(x.words)<=48 ]\n",
    "\n",
    "print(\"48 words:\",len(selectedSet))\n",
    "\n",
    "\n",
    "selectedSet = [x for x in trainSetRaw+testSetRaw if len(x.words)<=40 ]\n",
    "\n",
    "print(\"40 words:\",len(selectedSet))\n",
    "\n",
    "selectedSet = [x for x in trainSetRaw+testSetRaw if len(x.words)<=30 ]\n",
    "\n",
    "print(\"30 words:\",len(selectedSet))\n",
    "\n",
    "\n",
    "\n",
    "trainSetRaw = [x for x in trainSetRaw if len(x.words)<=48]\n",
    "testSetRaw = [x for x in testSetRaw if len(x.words)<=48]\n",
    "\n",
    "\n",
    "print()\n",
    "print(\"Selecting for under 48 words we are left with\")\n",
    "print(len(trainSetRaw),\"training examples\")\n",
    "print(len(testSetRaw),\"test examples\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10238\n",
      "10238\n"
     ]
    }
   ],
   "source": [
    "trainSet = []\n",
    "trainLabel = []\n",
    "\n",
    "for example in trainSetRaw:\n",
    "    for interaction in example.interactions:\n",
    "        #print(example.words)\n",
    "        #print(interaction)        \n",
    "        sentence = interaction[0] + \" \" + interaction[1] + \" \" + \" \".join(example.words)\n",
    "        #print(sentence)\n",
    "        #print(type(sentence))\n",
    "        trainSet = trainSet + [sentence]\n",
    "        if interaction[2] == \"True\":\n",
    "            trainLabel = trainLabel + [1]\n",
    "            #print(\"HIT\")\n",
    "        else:\n",
    "            trainLabel = trainLabel + [0]\n",
    "            #print(\"MISS\")\n",
    "        #print()\n",
    "        \n",
    "print(len(trainSet))\n",
    "print(len(trainLabel))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2274\n",
      "2274\n"
     ]
    }
   ],
   "source": [
    "testSet = []\n",
    "testLabel = []\n",
    "\n",
    "for example in testSetRaw:\n",
    "    for interaction in example.interactions:\n",
    "        #print(example.words)\n",
    "        #print(interaction)        \n",
    "        sentence = interaction[0] + \" \" + interaction[1] + \" \" + \" \".join(example.words)\n",
    "        #print(sentence)\n",
    "        #print(type(sentence))\n",
    "        testSet = testSet + [sentence]\n",
    "        if interaction[2] == \"True\":\n",
    "            testLabel = testLabel + [1]\n",
    "            #print(\"HIT\")\n",
    "        else:\n",
    "            testLabel = testLabel + [0]\n",
    "            #print(\"MISS\")\n",
    "        #print()\n",
    "        \n",
    "print(len(testSet))\n",
    "print(len(testLabel))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PROTEIN0 PROTEIN1 Cytokines measurements during PROTEIN0 treatment showed a trend to decreasing levels of PROTEIN1 at 4, 12, and 24 weeks.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Levels of PROTEIN0 were slightly increased following PROTEIN1 treatment (P = 0.09).\n",
      "\n",
      "PROTEIN0 PROTEIN1 This study describes the distributions of PROTEIN0 as well as mRNAs for PROTEIN1 PROTEIN2 collagen types II PROTEIN3 and III PROTEIN4 in a growing \"cartilage cap\" of osteochondroma.\n",
      "\n",
      "PROTEIN0 PROTEIN2 This study describes the distributions of PROTEIN0 as well as mRNAs for PROTEIN1 PROTEIN2 collagen types II PROTEIN3 and III PROTEIN4 in a growing \"cartilage cap\" of osteochondroma.\n",
      "\n",
      "PROTEIN0 PROTEIN3 This study describes the distributions of PROTEIN0 as well as mRNAs for PROTEIN1 PROTEIN2 collagen types II PROTEIN3 and III PROTEIN4 in a growing \"cartilage cap\" of osteochondroma.\n",
      "\n",
      "PROTEIN0 PROTEIN4 This study describes the distributions of PROTEIN0 as well as mRNAs for PROTEIN1 PROTEIN2 collagen types II PROTEIN3 and III PROTEIN4 in a growing \"cartilage cap\" of osteochondroma.\n",
      "\n",
      "PROTEIN1 PROTEIN2 This study describes the distributions of PROTEIN0 as well as mRNAs for PROTEIN1 PROTEIN2 collagen types II PROTEIN3 and III PROTEIN4 in a growing \"cartilage cap\" of osteochondroma.\n",
      "\n",
      "PROTEIN1 PROTEIN3 This study describes the distributions of PROTEIN0 as well as mRNAs for PROTEIN1 PROTEIN2 collagen types II PROTEIN3 and III PROTEIN4 in a growing \"cartilage cap\" of osteochondroma.\n",
      "\n",
      "PROTEIN1 PROTEIN4 This study describes the distributions of PROTEIN0 as well as mRNAs for PROTEIN1 PROTEIN2 collagen types II PROTEIN3 and III PROTEIN4 in a growing \"cartilage cap\" of osteochondroma.\n",
      "\n",
      "PROTEIN2 PROTEIN3 This study describes the distributions of PROTEIN0 as well as mRNAs for PROTEIN1 PROTEIN2 collagen types II PROTEIN3 and III PROTEIN4 in a growing \"cartilage cap\" of osteochondroma.\n",
      "\n",
      "PROTEIN2 PROTEIN4 This study describes the distributions of PROTEIN0 as well as mRNAs for PROTEIN1 PROTEIN2 collagen types II PROTEIN3 and III PROTEIN4 in a growing \"cartilage cap\" of osteochondroma.\n",
      "\n",
      "PROTEIN3 PROTEIN4 This study describes the distributions of PROTEIN0 as well as mRNAs for PROTEIN1 PROTEIN2 collagen types II PROTEIN3 and III PROTEIN4 in a growing \"cartilage cap\" of osteochondroma.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 was detected in mesenchymal cells in the outer fibrous layer and chondrocytes in the inner cartilaginous matrix, positive for PROTEIN1 and PROTEIN2 respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 was detected in mesenchymal cells in the outer fibrous layer and chondrocytes in the inner cartilaginous matrix, positive for PROTEIN1 and PROTEIN2 respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 was detected in mesenchymal cells in the outer fibrous layer and chondrocytes in the inner cartilaginous matrix, positive for PROTEIN1 and PROTEIN2 respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This is the first study to provide observational evidence of the involvement of PROTEIN0 signaling in the pathogenesis of cartilage cap of osteochondroma. and suggests the role of PROTEIN1 in the growth of cartilage cap in osteochondroma.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The molar ratio of serum PROTEIN0 PROTEIN1 to PROTEIN2 PROTEIN3 is not useful to assess vitamin A status during infection in hospitalised children.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The molar ratio of serum PROTEIN0 PROTEIN1 to PROTEIN2 PROTEIN3 is not useful to assess vitamin A status during infection in hospitalised children.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The molar ratio of serum PROTEIN0 PROTEIN1 to PROTEIN2 PROTEIN3 is not useful to assess vitamin A status during infection in hospitalised children.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The molar ratio of serum PROTEIN0 PROTEIN1 to PROTEIN2 PROTEIN3 is not useful to assess vitamin A status during infection in hospitalised children.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The molar ratio of serum PROTEIN0 PROTEIN1 to PROTEIN2 PROTEIN3 is not useful to assess vitamin A status during infection in hospitalised children.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The molar ratio of serum PROTEIN0 PROTEIN1 to PROTEIN2 PROTEIN3 is not useful to assess vitamin A status during infection in hospitalised children.\n",
      "\n",
      "PROTEIN0 PROTEIN1 OBJECTIVE: To assess the usefulness of the molar ratio of serum PROTEIN0 PROTEIN1 to PROTEIN2 PROTEIN3 to determine vitamin A (VA) status during infection.\n",
      "\n",
      "PROTEIN0 PROTEIN2 OBJECTIVE: To assess the usefulness of the molar ratio of serum PROTEIN0 PROTEIN1 to PROTEIN2 PROTEIN3 to determine vitamin A (VA) status during infection.\n",
      "\n",
      "PROTEIN0 PROTEIN3 OBJECTIVE: To assess the usefulness of the molar ratio of serum PROTEIN0 PROTEIN1 to PROTEIN2 PROTEIN3 to determine vitamin A (VA) status during infection.\n",
      "\n",
      "PROTEIN1 PROTEIN2 OBJECTIVE: To assess the usefulness of the molar ratio of serum PROTEIN0 PROTEIN1 to PROTEIN2 PROTEIN3 to determine vitamin A (VA) status during infection.\n",
      "\n",
      "PROTEIN1 PROTEIN3 OBJECTIVE: To assess the usefulness of the molar ratio of serum PROTEIN0 PROTEIN1 to PROTEIN2 PROTEIN3 to determine vitamin A (VA) status during infection.\n",
      "\n",
      "PROTEIN2 PROTEIN3 OBJECTIVE: To assess the usefulness of the molar ratio of serum PROTEIN0 PROTEIN1 to PROTEIN2 PROTEIN3 to determine vitamin A (VA) status during infection.\n",
      "\n",
      "PROTEIN0 PROTEIN1 DESIGN: We took advantage of previously collected data during a randomised double-blind, placebo-controlled clinical trial to conduct a secondary analysis of the PROTEIN0 PROTEIN1 ratio and its relationship to infection and VA status.\n",
      "\n",
      "PROTEIN0 PROTEIN1 MAIN OUTCOME MEASURES: PROTEIN0 PROTEIN1 molar ratio after 7 days hospitalisation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Genetic profile of 22 pancreatic carcinoma cell lines. Analysis of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Genetic profile of 22 pancreatic carcinoma cell lines. Analysis of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Genetic profile of 22 pancreatic carcinoma cell lines. Analysis of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Genetic profile of 22 pancreatic carcinoma cell lines. Analysis of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Genetic profile of 22 pancreatic carcinoma cell lines. Analysis of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Genetic profile of 22 pancreatic carcinoma cell lines. Analysis of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Genetic profile of 22 pancreatic carcinoma cell lines. Analysis of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Genetic profile of 22 pancreatic carcinoma cell lines. Analysis of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Genetic profile of 22 pancreatic carcinoma cell lines. Analysis of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Genetic profile of 22 pancreatic carcinoma cell lines. Analysis of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 genes are among those most frequently altered in pancreatic ductal carcinoma.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 genes are among those most frequently altered in pancreatic ductal carcinoma.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 genes are among those most frequently altered in pancreatic ductal carcinoma.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 genes are among those most frequently altered in pancreatic ductal carcinoma.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 genes are among those most frequently altered in pancreatic ductal carcinoma.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 genes are among those most frequently altered in pancreatic ductal carcinoma.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These cell lines showed mutations in PROTEIN0 and PROTEIN1 at frequencies of 91% and 95%, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The most typical mutational profile involved PROTEIN0 PROTEIN1 and PROTEIN2 concurrently aberrated in 20 cases (91%).\n",
      "\n",
      "PROTEIN0 PROTEIN2 The most typical mutational profile involved PROTEIN0 PROTEIN1 and PROTEIN2 concurrently aberrated in 20 cases (91%).\n",
      "\n",
      "PROTEIN1 PROTEIN2 The most typical mutational profile involved PROTEIN0 PROTEIN1 and PROTEIN2 concurrently aberrated in 20 cases (91%).\n",
      "\n",
      "PROTEIN0 PROTEIN1 We investigated the mechanism of calcium-stimulated PROTEIN0 release from gastrinoma cells in vitro with a particular focus on the PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 We investigated the mechanism of calcium-stimulated PROTEIN0 release from gastrinoma cells in vitro with a particular focus on the PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 We investigated the mechanism of calcium-stimulated PROTEIN0 release from gastrinoma cells in vitro with a particular focus on the PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our results demonstrated that PROTEIN0 was expressed in human gastrinoma cells and could be involved in the mechanism of calcium-evoked PROTEIN1 release.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 involvement in the osteogenic properties of the HeLa cell line.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We postulate that the mechanism of bone induction is based on the secretion of bone morphogenetic proteins PROTEIN0 and PROTEIN1 by the grafted HeLa cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 MAIN OUTCOME MEASURES: Measurement of soluble PROTEIN0 PROTEIN1 and PROTEIN2 in serum samples and fluid of recent blisters.\n",
      "\n",
      "PROTEIN0 PROTEIN2 MAIN OUTCOME MEASURES: Measurement of soluble PROTEIN0 PROTEIN1 and PROTEIN2 in serum samples and fluid of recent blisters.\n",
      "\n",
      "PROTEIN1 PROTEIN2 MAIN OUTCOME MEASURES: Measurement of soluble PROTEIN0 PROTEIN1 and PROTEIN2 in serum samples and fluid of recent blisters.\n",
      "\n",
      "PROTEIN0 PROTEIN1 RESULTS: In the blister fluid of patients with TEN, we found significantly higher levels of PROTEIN0 than in patients with burns, whereas PROTEIN1 levels were higher in the blister fluid of burned patients.\n",
      "\n",
      "PROTEIN0 PROTEIN1 No significant differences were found in serum samples of patients with TEN and burns, in either PROTEIN0 or PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In TEN we also found significantly higher levels of PROTEIN0 in the blister fluid compared with serum samples, pointing to a predominantly local production contrasting with the low concentration of PROTEIN1 in the blister fluid of burned patients.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Concentrations of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in cerebrospinal fluid (CSF) were measured in patients with second-stage Trypanosoma brucei gambiense sleeping sickness.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Concentrations of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in cerebrospinal fluid (CSF) were measured in patients with second-stage Trypanosoma brucei gambiense sleeping sickness.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Concentrations of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in cerebrospinal fluid (CSF) were measured in patients with second-stage Trypanosoma brucei gambiense sleeping sickness.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Concentrations of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in cerebrospinal fluid (CSF) were measured in patients with second-stage Trypanosoma brucei gambiense sleeping sickness.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Concentrations of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in cerebrospinal fluid (CSF) were measured in patients with second-stage Trypanosoma brucei gambiense sleeping sickness.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Concentrations of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in cerebrospinal fluid (CSF) were measured in patients with second-stage Trypanosoma brucei gambiense sleeping sickness.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Correlations between PROTEIN0 and PROTEIN1 in CSF as markers for astrogliosis and neurodegeneration, and clinical and biological data were investigated.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Abnormal levels of PROTEIN0 and PROTEIN1 were significantly associated with increasing CSF cell number and protein concentration, and with the absence of lymph nodes or the absence of trypanosomes in lymph node aspirate.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We investigated whether the chronic inflammation and pain experienced by patients may be caused by an imbalance of proinflammatory versus anti-inflammatory cytokines within the prostate, namely PROTEIN0 PROTEIN1 and PROTEIN2 versus PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 We investigated whether the chronic inflammation and pain experienced by patients may be caused by an imbalance of proinflammatory versus anti-inflammatory cytokines within the prostate, namely PROTEIN0 PROTEIN1 and PROTEIN2 versus PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 We investigated whether the chronic inflammation and pain experienced by patients may be caused by an imbalance of proinflammatory versus anti-inflammatory cytokines within the prostate, namely PROTEIN0 PROTEIN1 and PROTEIN2 versus PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 We investigated whether the chronic inflammation and pain experienced by patients may be caused by an imbalance of proinflammatory versus anti-inflammatory cytokines within the prostate, namely PROTEIN0 PROTEIN1 and PROTEIN2 versus PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 We investigated whether the chronic inflammation and pain experienced by patients may be caused by an imbalance of proinflammatory versus anti-inflammatory cytokines within the prostate, namely PROTEIN0 PROTEIN1 and PROTEIN2 versus PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 We investigated whether the chronic inflammation and pain experienced by patients may be caused by an imbalance of proinflammatory versus anti-inflammatory cytokines within the prostate, namely PROTEIN0 PROTEIN1 and PROTEIN2 versus PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 MATERIALS AND METHODS: We measured levels of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in the seminal plasma of 31 patients with chronic prostatitis-chronic pelvic pain syndrome and 14 controls using enzyme-linked immunosorbent assay.\n",
      "\n",
      "PROTEIN0 PROTEIN2 MATERIALS AND METHODS: We measured levels of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in the seminal plasma of 31 patients with chronic prostatitis-chronic pelvic pain syndrome and 14 controls using enzyme-linked immunosorbent assay.\n",
      "\n",
      "PROTEIN0 PROTEIN3 MATERIALS AND METHODS: We measured levels of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in the seminal plasma of 31 patients with chronic prostatitis-chronic pelvic pain syndrome and 14 controls using enzyme-linked immunosorbent assay.\n",
      "\n",
      "PROTEIN1 PROTEIN2 MATERIALS AND METHODS: We measured levels of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in the seminal plasma of 31 patients with chronic prostatitis-chronic pelvic pain syndrome and 14 controls using enzyme-linked immunosorbent assay.\n",
      "\n",
      "PROTEIN1 PROTEIN3 MATERIALS AND METHODS: We measured levels of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in the seminal plasma of 31 patients with chronic prostatitis-chronic pelvic pain syndrome and 14 controls using enzyme-linked immunosorbent assay.\n",
      "\n",
      "PROTEIN2 PROTEIN3 MATERIALS AND METHODS: We measured levels of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in the seminal plasma of 31 patients with chronic prostatitis-chronic pelvic pain syndrome and 14 controls using enzyme-linked immunosorbent assay.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 and PROTEIN2 levels were statistically greater in patients.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 levels were statistically greater in patients.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 levels were statistically greater in patients.\n",
      "\n",
      "PROTEIN0 PROTEIN1 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN0 PROTEIN2 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN0 PROTEIN3 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN0 PROTEIN4 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN0 PROTEIN5 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN1 PROTEIN2 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN1 PROTEIN3 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN1 PROTEIN4 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN1 PROTEIN5 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN2 PROTEIN3 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN2 PROTEIN4 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN2 PROTEIN5 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN3 PROTEIN4 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN3 PROTEIN5 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN4 PROTEIN5 METHODS: The genes for dopamine synthetic enzymes, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 an enzyme critical for tetrahydrobiopterin synthesis) were contransduced into 293 cells with separate AAV vectors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Expressions of PROTEIN0 PROTEIN1 and PROTEIN2 were detected by Western blot analysis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Expressions of PROTEIN0 PROTEIN1 and PROTEIN2 were detected by Western blot analysis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Expressions of PROTEIN0 PROTEIN1 and PROTEIN2 were detected by Western blot analysis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 RESULTS: PROTEIN0 PROTEIN1 and PROTEIN2 were effectively coexpressed in transduced cells with three separate AAV vectors, PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 CONCLUSION: The triple transduction of PROTEIN0 PROTEIN1 and PROTEIN2 genes with separate AAV vectors is effective, which might be important to gene therapy for Parkinson's disease.\n",
      "\n",
      "PROTEIN0 PROTEIN2 CONCLUSION: The triple transduction of PROTEIN0 PROTEIN1 and PROTEIN2 genes with separate AAV vectors is effective, which might be important to gene therapy for Parkinson's disease.\n",
      "\n",
      "PROTEIN1 PROTEIN2 CONCLUSION: The triple transduction of PROTEIN0 PROTEIN1 and PROTEIN2 genes with separate AAV vectors is effective, which might be important to gene therapy for Parkinson's disease.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This overview summarizes the current knowledge on PROTEIN0 PROTEIN1 in the progression of atherosclerosis and the influence of MMPs in aneurysm formation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, a possible role for PROTEIN0 the product of PROTEIN1 in atherosclerotic lesion development is put into perspective.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN0 PROTEIN2 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN0 PROTEIN3 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN0 PROTEIN4 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN0 PROTEIN5 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN0 PROTEIN6 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN0 PROTEIN7 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN0 PROTEIN8 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN0 PROTEIN9 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN0 PROTEIN0 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN1 PROTEIN2 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN1 PROTEIN3 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN1 PROTEIN4 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN1 PROTEIN5 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN1 PROTEIN6 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN1 PROTEIN7 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN1 PROTEIN8 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN1 PROTEIN9 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN1 PROTEIN0 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN2 PROTEIN3 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN2 PROTEIN4 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN2 PROTEIN5 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN2 PROTEIN6 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN2 PROTEIN7 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN2 PROTEIN8 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN2 PROTEIN9 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN2 PROTEIN0 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN3 PROTEIN4 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN3 PROTEIN5 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN3 PROTEIN6 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN3 PROTEIN7 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN3 PROTEIN8 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN3 PROTEIN9 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN3 PROTEIN0 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN4 PROTEIN5 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN4 PROTEIN6 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN4 PROTEIN7 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN4 PROTEIN8 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN4 PROTEIN9 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN4 PROTEIN0 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN5 PROTEIN6 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN5 PROTEIN7 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN5 PROTEIN8 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN5 PROTEIN9 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN5 PROTEIN0 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN6 PROTEIN7 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN6 PROTEIN8 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN6 PROTEIN9 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN6 PROTEIN0 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN7 PROTEIN8 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN7 PROTEIN9 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN7 PROTEIN0 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN8 PROTEIN9 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN8 PROTEIN0 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN9 PROTEIN0 Cytokines PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 (Immunotech, France), PROTEIN5 and PROTEIN6 PROTEIN7 sP- and PROTEIN8 PROTEIN9 and PROTEIN0 (R&D, UK) have been determined using ELISA kits (photometer, Biomek-1000, Beckman, USA).\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and sP- and PROTEIN1 levels were not related to function of lymphocytes (by MDM data) as seen in both tests; it is probable that another cell-cell communication mechanism had been switched on.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We established a strong correlation between chromatin activity of lymphocytes and the serum concentration of PROTEIN0 PROTEIN1 and PROTEIN2 which are the active participants in the pro- and anti-inflammatory program in atherogenesis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We established a strong correlation between chromatin activity of lymphocytes and the serum concentration of PROTEIN0 PROTEIN1 and PROTEIN2 which are the active participants in the pro- and anti-inflammatory program in atherogenesis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We established a strong correlation between chromatin activity of lymphocytes and the serum concentration of PROTEIN0 PROTEIN1 and PROTEIN2 which are the active participants in the pro- and anti-inflammatory program in atherogenesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 Accordingly, a number of population studies have explored the association of ischaemic heart disease with gene polymorphisms of the inflammatory molecules PROTEIN0 and beta, PROTEIN1 and 2, PROTEIN2 and its receptor antagonist PROTEIN3 PROTEIN4 (the receptor for lipopolysaccharide), P and E selectins, and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Present data concerning the PROTEIN0 PROTEIN1 system of extrarenal origin.\n",
      "\n",
      "PROTEIN0 PROTEIN1 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 After a brief historical incursion regarding PROTEIN0 of renal origin, we present the main extrarenal PROTEIN1 enzymes, starting with PROTEIN2 PROTEIN3 D and PROTEIN4 and ending with the conversion enzyme and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 While the circulating PROTEIN0 of renal origin is acutely activated to contribute to the accomplishment of short term homeostatic reactions, tissue PROTEIN1 in paracrine or autocrine ways, exerts local-regional adaptive actions of long duration.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the present study, we examined the presence of glucocorticoid receptors (GRs) and PROTEIN0 PROTEIN1 in the nasal polyps of nonallergic patients as compared with normal controls.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, in normal nasal tissues, PROTEIN0 labeling was only found in the vascular wall, and the expression was weaker--a finding demonstrating that PROTEIN1 is upregulated in nasal polyps.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Therefore, 1) the presence of GRs and PROTEIN0 in nasal polyps from nonallergic patients, as well as upregulation of PROTEIN1 expression, suggests that inflammation may play an important role in the pathophysiology of nasal polyps; and 2) glucocorticoid could be a potential treatment method for suppressing inflammatory processes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Assaying of PROTEIN0 complement factors, and PROTEIN1 in the diagnosis of malignant serous effusions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN0 PROTEIN6 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN1 PROTEIN6 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN2 PROTEIN6 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN3 PROTEIN6 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN4 PROTEIN6 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN5 PROTEIN6 The objective of this study was to measure the concentrations of PROTEIN0 PROTEIN1 in pleural and peritoneal effusions of different causes and to verify whether PROTEIN2 PROTEIN3 PROTEIN4 and complement factors PROTEIN5 and PROTEIN6 can be used in the differential diagnosis of serous effusion.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 and complement factors PROTEIN2 and PROTEIN3 concentrations were measured simultaneously in blood and serous effusion using commercially available methods.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 and complement factors PROTEIN2 and PROTEIN3 concentrations were measured simultaneously in blood and serous effusion using commercially available methods.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 and complement factors PROTEIN2 and PROTEIN3 concentrations were measured simultaneously in blood and serous effusion using commercially available methods.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 and complement factors PROTEIN2 and PROTEIN3 concentrations were measured simultaneously in blood and serous effusion using commercially available methods.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 and complement factors PROTEIN2 and PROTEIN3 concentrations were measured simultaneously in blood and serous effusion using commercially available methods.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 and complement factors PROTEIN2 and PROTEIN3 concentrations were measured simultaneously in blood and serous effusion using commercially available methods.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Their order of selection was PROTEIN0 effusion, PROTEIN1 serum, PROTEIN2 and PROTEIN3 effusion.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Their order of selection was PROTEIN0 effusion, PROTEIN1 serum, PROTEIN2 and PROTEIN3 effusion.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Their order of selection was PROTEIN0 effusion, PROTEIN1 serum, PROTEIN2 and PROTEIN3 effusion.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Their order of selection was PROTEIN0 effusion, PROTEIN1 serum, PROTEIN2 and PROTEIN3 effusion.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Their order of selection was PROTEIN0 effusion, PROTEIN1 serum, PROTEIN2 and PROTEIN3 effusion.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Their order of selection was PROTEIN0 effusion, PROTEIN1 serum, PROTEIN2 and PROTEIN3 effusion.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Determination of PROTEIN0 complement factors PROTEIN1 and PROTEIN2 and PROTEIN3 may be a significant parameter in the differential diagnosis of serous effusions, particularly in those patients with malignant disease.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Determination of PROTEIN0 complement factors PROTEIN1 and PROTEIN2 and PROTEIN3 may be a significant parameter in the differential diagnosis of serous effusions, particularly in those patients with malignant disease.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Determination of PROTEIN0 complement factors PROTEIN1 and PROTEIN2 and PROTEIN3 may be a significant parameter in the differential diagnosis of serous effusions, particularly in those patients with malignant disease.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Determination of PROTEIN0 complement factors PROTEIN1 and PROTEIN2 and PROTEIN3 may be a significant parameter in the differential diagnosis of serous effusions, particularly in those patients with malignant disease.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Determination of PROTEIN0 complement factors PROTEIN1 and PROTEIN2 and PROTEIN3 may be a significant parameter in the differential diagnosis of serous effusions, particularly in those patients with malignant disease.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Determination of PROTEIN0 complement factors PROTEIN1 and PROTEIN2 and PROTEIN3 may be a significant parameter in the differential diagnosis of serous effusions, particularly in those patients with malignant disease.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Differential expression of PROTEIN0 and PROTEIN1 by NK cells in cancer patients and correlation of impaired PROTEIN2 expression withprogression of cancer.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Differential expression of PROTEIN0 and PROTEIN1 by NK cells in cancer patients and correlation of impaired PROTEIN2 expression withprogression of cancer.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Differential expression of PROTEIN0 and PROTEIN1 by NK cells in cancer patients and correlation of impaired PROTEIN2 expression withprogression of cancer.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The aim of the present study was to elucidate whether intracellular expression of PROTEIN0 and PROTEIN1 by NK cells might be associated with progression of cancer.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Flow cytometric analysis demonstrated high levels of PROTEIN0 and PROTEIN1 expression by PROTEIN2 PROTEIN3 cells in healthy controls.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Flow cytometric analysis demonstrated high levels of PROTEIN0 and PROTEIN1 expression by PROTEIN2 PROTEIN3 cells in healthy controls.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Flow cytometric analysis demonstrated high levels of PROTEIN0 and PROTEIN1 expression by PROTEIN2 PROTEIN3 cells in healthy controls.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Flow cytometric analysis demonstrated high levels of PROTEIN0 and PROTEIN1 expression by PROTEIN2 PROTEIN3 cells in healthy controls.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Flow cytometric analysis demonstrated high levels of PROTEIN0 and PROTEIN1 expression by PROTEIN2 PROTEIN3 cells in healthy controls.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Flow cytometric analysis demonstrated high levels of PROTEIN0 and PROTEIN1 expression by PROTEIN2 PROTEIN3 cells in healthy controls.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, cancer patients exhibited significantly decreased levels of PROTEIN0 expression ( P<0.005), despite having equally high levels of PROTEIN1 expression in comparison with healthy controls.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The tumor-free patients expressed PROTEIN0 at levels similar to healthy controls, while the progressive tumor-bearing patients expressed remarkably lower levels of PROTEIN1 compared to healthy controls ( P<0.0001).\n",
      "\n",
      "PROTEIN0 PROTEIN1 In conclusion, impaired expression of PROTEIN0 by NK cells correlates with progression of cancer, and determination of PROTEIN1 expression might prove informative for assessing the immunological condition of cancer patients.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 a member of the PROTEIN1 superfamily, has various effects on diverse biological systems, including cell growth inhibition in many cell types.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Based on the experiences of all the authors, we emphasized cell cycle inhibitors such as PROTEIN0 and PROTEIN1 and regulators of apoptosis such as PROTEIN2 and members of the PROTEIN3 family.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Based on the experiences of all the authors, we emphasized cell cycle inhibitors such as PROTEIN0 and PROTEIN1 and regulators of apoptosis such as PROTEIN2 and members of the PROTEIN3 family.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Based on the experiences of all the authors, we emphasized cell cycle inhibitors such as PROTEIN0 and PROTEIN1 and regulators of apoptosis such as PROTEIN2 and members of the PROTEIN3 family.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Based on the experiences of all the authors, we emphasized cell cycle inhibitors such as PROTEIN0 and PROTEIN1 and regulators of apoptosis such as PROTEIN2 and members of the PROTEIN3 family.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Based on the experiences of all the authors, we emphasized cell cycle inhibitors such as PROTEIN0 and PROTEIN1 and regulators of apoptosis such as PROTEIN2 and members of the PROTEIN3 family.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Based on the experiences of all the authors, we emphasized cell cycle inhibitors such as PROTEIN0 and PROTEIN1 and regulators of apoptosis such as PROTEIN2 and members of the PROTEIN3 family.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have probed an epitope sequence (His18-Pro19-Lys20-Phe21) in PROTEIN0 PROTEIN1 by site-directed mutagenesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These Ala mutants triggered neutrophil degranulation and induced calcium responses mediated by PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Single Asp or Ser substitutions, H18D, F21D, F21S, and double substitutions, H18A/F21D, H18A/F21S, and H18D/F21D, reduced up to 431-fold the binding affinity to PROTEIN0 PROTEIN1 and the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Single Asp or Ser substitutions, H18D, F21D, F21S, and double substitutions, H18A/F21D, H18A/F21S, and H18D/F21D, reduced up to 431-fold the binding affinity to PROTEIN0 PROTEIN1 and the PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Single Asp or Ser substitutions, H18D, F21D, F21S, and double substitutions, H18A/F21D, H18A/F21S, and H18D/F21D, reduced up to 431-fold the binding affinity to PROTEIN0 PROTEIN1 and the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 This study demonstrates that PROTEIN0 recognizes and activates PROTEIN1 PROTEIN2 and the PROTEIN3 by distinct mechanisms.\n",
      "\n",
      "PROTEIN0 PROTEIN2 This study demonstrates that PROTEIN0 recognizes and activates PROTEIN1 PROTEIN2 and the PROTEIN3 by distinct mechanisms.\n",
      "\n",
      "PROTEIN0 PROTEIN3 This study demonstrates that PROTEIN0 recognizes and activates PROTEIN1 PROTEIN2 and the PROTEIN3 by distinct mechanisms.\n",
      "\n",
      "PROTEIN1 PROTEIN2 This study demonstrates that PROTEIN0 recognizes and activates PROTEIN1 PROTEIN2 and the PROTEIN3 by distinct mechanisms.\n",
      "\n",
      "PROTEIN1 PROTEIN3 This study demonstrates that PROTEIN0 recognizes and activates PROTEIN1 PROTEIN2 and the PROTEIN3 by distinct mechanisms.\n",
      "\n",
      "PROTEIN2 PROTEIN3 This study demonstrates that PROTEIN0 recognizes and activates PROTEIN1 PROTEIN2 and the PROTEIN3 by distinct mechanisms.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Isolation of human PROTEIN0 and its binding specificity with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 We screened proteins for interaction with PROTEIN0 and cloned the full-length cDNA of human PROTEIN1 which encoded 1225 amino acids.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Yeast two-hybrid assay, GST binding assay and immunoprecipitation demonstrated that PROTEIN0 interacted with a hydrophilic loop region in the endoproteolytic C-terminal fragment of PROTEIN1 but not with that of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Yeast two-hybrid assay, GST binding assay and immunoprecipitation demonstrated that PROTEIN0 interacted with a hydrophilic loop region in the endoproteolytic C-terminal fragment of PROTEIN1 but not with that of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Yeast two-hybrid assay, GST binding assay and immunoprecipitation demonstrated that PROTEIN0 interacted with a hydrophilic loop region in the endoproteolytic C-terminal fragment of PROTEIN1 but not with that of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 and PROTEIN1 partly differ in function.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We also found another PROTEIN0 PROTEIN1 interacted with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 We also found another PROTEIN0 PROTEIN1 interacted with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 We also found another PROTEIN0 PROTEIN1 interacted with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 We used alanine substitution of PROTEIN0 residues that contact PROTEIN1 in the crystal structure to investigate the function of these residues in both PROTEIN2 and PROTEIN3 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We used alanine substitution of PROTEIN0 residues that contact PROTEIN1 in the crystal structure to investigate the function of these residues in both PROTEIN2 and PROTEIN3 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We used alanine substitution of PROTEIN0 residues that contact PROTEIN1 in the crystal structure to investigate the function of these residues in both PROTEIN2 and PROTEIN3 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We used alanine substitution of PROTEIN0 residues that contact PROTEIN1 in the crystal structure to investigate the function of these residues in both PROTEIN2 and PROTEIN3 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We used alanine substitution of PROTEIN0 residues that contact PROTEIN1 in the crystal structure to investigate the function of these residues in both PROTEIN2 and PROTEIN3 binding.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We used alanine substitution of PROTEIN0 residues that contact PROTEIN1 in the crystal structure to investigate the function of these residues in both PROTEIN2 and PROTEIN3 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our results indicate that Phe93 and Phe205 are important for both PROTEIN0 and PROTEIN1 binding, Met150 is not important for PROTEIN2 binding but is critical for PROTEIN3 binding, and Thr151 is not important for binding either ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Our results indicate that Phe93 and Phe205 are important for both PROTEIN0 and PROTEIN1 binding, Met150 is not important for PROTEIN2 binding but is critical for PROTEIN3 binding, and Thr151 is not important for binding either ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Our results indicate that Phe93 and Phe205 are important for both PROTEIN0 and PROTEIN1 binding, Met150 is not important for PROTEIN2 binding but is critical for PROTEIN3 binding, and Thr151 is not important for binding either ligand.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Our results indicate that Phe93 and Phe205 are important for both PROTEIN0 and PROTEIN1 binding, Met150 is not important for PROTEIN2 binding but is critical for PROTEIN3 binding, and Thr151 is not important for binding either ligand.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Our results indicate that Phe93 and Phe205 are important for both PROTEIN0 and PROTEIN1 binding, Met150 is not important for PROTEIN2 binding but is critical for PROTEIN3 binding, and Thr151 is not important for binding either ligand.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Our results indicate that Phe93 and Phe205 are important for both PROTEIN0 and PROTEIN1 binding, Met150 is not important for PROTEIN2 binding but is critical for PROTEIN3 binding, and Thr151 is not important for binding either ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, Phe93 and Phe205 are important binding determinants for both PROTEIN0 and PROTEIN1 even though these ligands share no sequence or structural homology, suggesting that these residues may represent a minimum epitope on the PROTEIN2 for productive ligand binding.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, Phe93 and Phe205 are important binding determinants for both PROTEIN0 and PROTEIN1 even though these ligands share no sequence or structural homology, suggesting that these residues may represent a minimum epitope on the PROTEIN2 for productive ligand binding.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, Phe93 and Phe205 are important binding determinants for both PROTEIN0 and PROTEIN1 even though these ligands share no sequence or structural homology, suggesting that these residues may represent a minimum epitope on the PROTEIN2 for productive ligand binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Native PROTEIN0 and PROTEIN1 formed a heterodimer in solution of physiological ionic strength with a free-energy change DeltaG degrees of -8.3 kcal/mol and a dissociation constant Kd of 0.6 microM (at 20 degrees C) that was ionic strength- and temperature-dependent.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Native PROTEIN0 also formed a heterodimer with PROTEIN1 low concentrations of polyionic ligands such as protamine and suramin inhibited the interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Suramin induced high-affinity trimerization of PROTEIN0 (Kd = 0.10 microM at 20 degrees C) and dimerization of PROTEIN1 (Kd = 0.86 microM at 20 degrees C).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Analysis of sedimentation equilibria and also of the fluorescence enhancement of suramin when bound to protein provided evidence for two suramin-binding sites on each PROTEIN0 and three on each PROTEIN1 in the oligomers.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Most familial early-onset Alzheimer's disease cases are caused by mutations in the PROTEIN0 PROTEIN1 gene.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Although numerous studies revealed predominant localization of PROTEIN0 to endoplasmic reticulum and Golgi, there are conflicting reports on the localization of PROTEIN1 to the cell surface.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cell surface PROTEIN0 formed complexes in vivo with actin-binding protein PROTEIN1 PROTEIN2 which is known to form bridges between cell surface receptors and cytoskeleton and mediate cell adhesion and cell motility.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Cell surface PROTEIN0 formed complexes in vivo with actin-binding protein PROTEIN1 PROTEIN2 which is known to form bridges between cell surface receptors and cytoskeleton and mediate cell adhesion and cell motility.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Cell surface PROTEIN0 formed complexes in vivo with actin-binding protein PROTEIN1 PROTEIN2 which is known to form bridges between cell surface receptors and cytoskeleton and mediate cell adhesion and cell motility.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Molecular determinants of the interaction between the C-terminal domain of Alzheimer's PROTEIN0 and PROTEIN1 alpha-helices.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In a previous work, we predicted and demonstrated that the 29-42-residue fragment of PROTEIN0 PROTEIN1 has in vitro capacities close to those of the tilted fragment of viral fusion proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We further demonstrated that PROTEIN0 and E3 but not PROTEIN1 can decrease the fusogenic activity of PROTEIN2 via a direct interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We further demonstrated that PROTEIN0 and E3 but not PROTEIN1 can decrease the fusogenic activity of PROTEIN2 via a direct interaction.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We further demonstrated that PROTEIN0 and E3 but not PROTEIN1 can decrease the fusogenic activity of PROTEIN2 via a direct interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Therefore, we suggested that this fragment is implicated in the neurotoxicity of PROTEIN0 and in the protective effects of PROTEIN1 in Alzheimer's disease.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this study, we simulated interactions of all amphipathic helices of PROTEIN0 and A-I with PROTEIN1 and simian immunodeficiency virus (SIV) fusogenic fragments by molecular modeling.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binds multiple CC-chemokines: PROTEIN1 acts as a natural antagonist.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 was first characterized as a receptor for PROTEIN1 PROTEIN2 and PROTEIN3 and was rapidly shown to be the main coreceptor for M-tropic human immunodeficiency virus (HIV)-1 strains and simian immunodeficiency virus (SIV).\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 was first characterized as a receptor for PROTEIN1 PROTEIN2 and PROTEIN3 and was rapidly shown to be the main coreceptor for M-tropic human immunodeficiency virus (HIV)-1 strains and simian immunodeficiency virus (SIV).\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 was first characterized as a receptor for PROTEIN1 PROTEIN2 and PROTEIN3 and was rapidly shown to be the main coreceptor for M-tropic human immunodeficiency virus (HIV)-1 strains and simian immunodeficiency virus (SIV).\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 was first characterized as a receptor for PROTEIN1 PROTEIN2 and PROTEIN3 and was rapidly shown to be the main coreceptor for M-tropic human immunodeficiency virus (HIV)-1 strains and simian immunodeficiency virus (SIV).\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 was first characterized as a receptor for PROTEIN1 PROTEIN2 and PROTEIN3 and was rapidly shown to be the main coreceptor for M-tropic human immunodeficiency virus (HIV)-1 strains and simian immunodeficiency virus (SIV).\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 was first characterized as a receptor for PROTEIN1 PROTEIN2 and PROTEIN3 and was rapidly shown to be the main coreceptor for M-tropic human immunodeficiency virus (HIV)-1 strains and simian immunodeficiency virus (SIV).\n",
      "\n",
      "PROTEIN0 PROTEIN1 All CC-chemokines currently available were tested for their ability to compete with PROTEIN0 binding on a stable cell line expressing recombinant PROTEIN1 and/or to induce a functional response in these cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN5 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN6 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN7 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN8 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN5 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN6 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN7 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN8 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN5 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN6 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN7 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN8 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN3 PROTEIN5 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN3 PROTEIN6 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN3 PROTEIN7 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN3 PROTEIN8 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN4 PROTEIN5 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN4 PROTEIN6 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN4 PROTEIN7 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN4 PROTEIN8 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN5 PROTEIN6 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN5 PROTEIN7 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN5 PROTEIN8 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN6 PROTEIN7 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN6 PROTEIN8 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN7 PROTEIN8 We found that in addition to PROTEIN0 PROTEIN1 and PROTEIN2 five other CC-chemokines could compete for PROTEIN3 binding: PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 binding was characterized by IC(50) values of 0.22, 2.14, 5.89, 29.9, and 21.7 nmol/L, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 Among these ligands, PROTEIN0 had the remarkable property of binding PROTEIN1 with high affinity without eliciting a functional response, PROTEIN2 could also inhibit the activation of PROTEIN3 by PROTEIN4 and may therefore be considered as a natural antagonist for PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Chemokines that could compete with high affinity for PROTEIN0 binding could also compete for monomeric PROTEIN1 binding, although with variable potencies; maximal PROTEIN2 binding inhibition was 80% for PROTEIN3 but only 30% for PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Chemokines that could compete with high affinity for PROTEIN0 binding could also compete for monomeric PROTEIN1 binding, although with variable potencies; maximal PROTEIN2 binding inhibition was 80% for PROTEIN3 but only 30% for PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Chemokines that could compete with high affinity for PROTEIN0 binding could also compete for monomeric PROTEIN1 binding, although with variable potencies; maximal PROTEIN2 binding inhibition was 80% for PROTEIN3 but only 30% for PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Chemokines that could compete with high affinity for PROTEIN0 binding could also compete for monomeric PROTEIN1 binding, although with variable potencies; maximal PROTEIN2 binding inhibition was 80% for PROTEIN3 but only 30% for PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Chemokines that could compete with high affinity for PROTEIN0 binding could also compete for monomeric PROTEIN1 binding, although with variable potencies; maximal PROTEIN2 binding inhibition was 80% for PROTEIN3 but only 30% for PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Chemokines that could compete with high affinity for PROTEIN0 binding could also compete for monomeric PROTEIN1 binding, although with variable potencies; maximal PROTEIN2 binding inhibition was 80% for PROTEIN3 but only 30% for PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Chemokines that could compete with high affinity for PROTEIN0 binding could also compete for monomeric PROTEIN1 binding, although with variable potencies; maximal PROTEIN2 binding inhibition was 80% for PROTEIN3 but only 30% for PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Chemokines that could compete with high affinity for PROTEIN0 binding could also compete for monomeric PROTEIN1 binding, although with variable potencies; maximal PROTEIN2 binding inhibition was 80% for PROTEIN3 but only 30% for PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Chemokines that could compete with high affinity for PROTEIN0 binding could also compete for monomeric PROTEIN1 binding, although with variable potencies; maximal PROTEIN2 binding inhibition was 80% for PROTEIN3 but only 30% for PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Chemokines that could compete with high affinity for PROTEIN0 binding could also compete for monomeric PROTEIN1 binding, although with variable potencies; maximal PROTEIN2 binding inhibition was 80% for PROTEIN3 but only 30% for PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 could compete efficiently for PROTEIN1 binding but was, however, found to be a weak inhibitor of HIV infection, probably as a consequence of its inability to downregulate the receptor.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 suppresses the function of PROTEIN1 homodimers via interaction with PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 suppresses the function of PROTEIN1 homodimers via interaction with PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 suppresses the function of PROTEIN1 homodimers via interaction with PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 suppresses the function of PROTEIN1 homodimers via interaction with PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 suppresses the function of PROTEIN1 homodimers via interaction with PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 suppresses the function of PROTEIN1 homodimers via interaction with PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 is the causative gene for an autosomal dominant familial Alzheimer's disease (AD) mapped to chromosome 14.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we show that PROTEIN0 PROTEIN1 a negative regulator of PROTEIN2 is a candidate to mediate the function of PROTEIN3 in the cell.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we show that PROTEIN0 PROTEIN1 a negative regulator of PROTEIN2 is a candidate to mediate the function of PROTEIN3 in the cell.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here we show that PROTEIN0 PROTEIN1 a negative regulator of PROTEIN2 is a candidate to mediate the function of PROTEIN3 in the cell.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we show that PROTEIN0 PROTEIN1 a negative regulator of PROTEIN2 is a candidate to mediate the function of PROTEIN3 in the cell.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here we show that PROTEIN0 PROTEIN1 a negative regulator of PROTEIN2 is a candidate to mediate the function of PROTEIN3 in the cell.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here we show that PROTEIN0 PROTEIN1 a negative regulator of PROTEIN2 is a candidate to mediate the function of PROTEIN3 in the cell.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We screened for proteins that bind to PROTEIN0 from a human embryonic brain cDNA library using the two-hybrid method and isolated one clone encoding the PROTEIN1 PROTEIN2 gene.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We screened for proteins that bind to PROTEIN0 from a human embryonic brain cDNA library using the two-hybrid method and isolated one clone encoding the PROTEIN1 PROTEIN2 gene.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We screened for proteins that bind to PROTEIN0 from a human embryonic brain cDNA library using the two-hybrid method and isolated one clone encoding the PROTEIN1 PROTEIN2 gene.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The binding of PROTEIN0 PROTEIN1 to full-length PROTEIN2 was confirmed in vitro by pull-down assay, and in vivo by immunoprecipitation assays with human samples, including AD brains.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The binding of PROTEIN0 PROTEIN1 to full-length PROTEIN2 was confirmed in vitro by pull-down assay, and in vivo by immunoprecipitation assays with human samples, including AD brains.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The binding of PROTEIN0 PROTEIN1 to full-length PROTEIN2 was confirmed in vitro by pull-down assay, and in vivo by immunoprecipitation assays with human samples, including AD brains.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoelectronmicroscopic analysis showed that PROTEIN0 PROTEIN1 and PROTEIN2 are colocalized at the endoplasmic reticulum, and the nuclear matrix in human brain neurons.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunoelectronmicroscopic analysis showed that PROTEIN0 PROTEIN1 and PROTEIN2 are colocalized at the endoplasmic reticulum, and the nuclear matrix in human brain neurons.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunoelectronmicroscopic analysis showed that PROTEIN0 PROTEIN1 and PROTEIN2 are colocalized at the endoplasmic reticulum, and the nuclear matrix in human brain neurons.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN2 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN3 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN4 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN5 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN6 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN2 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN3 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN4 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN5 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN6 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN3 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN4 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN5 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN6 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN4 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN5 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN6 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN5 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN6 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN5 PROTEIN6 Chloramphenicol acetyltransferase assays in F9 cells showed that PROTEIN0 suppresses transactivation by PROTEIN1 PROTEIN2 but not by PROTEIN3 PROTEIN4 heterodimers, consistent with the reported function of PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN1 By monitoring fluorescent recombinant protein and by gel mobility shift assays, PROTEIN0 was shown to accelerate the translocation of PROTEIN1 from the cytoplasm to the nucleus and to thereby suppress the binding of PROTEIN2 homodimer to 12-O-tetradecanoylphorbol-13- acetate (TPA)-responsive element (TRE).\n",
      "\n",
      "PROTEIN0 PROTEIN2 By monitoring fluorescent recombinant protein and by gel mobility shift assays, PROTEIN0 was shown to accelerate the translocation of PROTEIN1 from the cytoplasm to the nucleus and to thereby suppress the binding of PROTEIN2 homodimer to 12-O-tetradecanoylphorbol-13- acetate (TPA)-responsive element (TRE).\n",
      "\n",
      "PROTEIN1 PROTEIN2 By monitoring fluorescent recombinant protein and by gel mobility shift assays, PROTEIN0 was shown to accelerate the translocation of PROTEIN1 from the cytoplasm to the nucleus and to thereby suppress the binding of PROTEIN2 homodimer to 12-O-tetradecanoylphorbol-13- acetate (TPA)-responsive element (TRE).\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 suppressed PROTEIN1 apoptosis by retinoic acid in F9 embryonic carcinoma cells, whereas this suppression of apoptosis is attenuated by mutation in PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 suppressed PROTEIN1 apoptosis by retinoic acid in F9 embryonic carcinoma cells, whereas this suppression of apoptosis is attenuated by mutation in PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 suppressed PROTEIN1 apoptosis by retinoic acid in F9 embryonic carcinoma cells, whereas this suppression of apoptosis is attenuated by mutation in PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Collectively, the novel function of PROTEIN0 via PROTEIN1 PROTEIN2 influences PROTEIN3 transcription and apoptosis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Collectively, the novel function of PROTEIN0 via PROTEIN1 PROTEIN2 influences PROTEIN3 transcription and apoptosis.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Collectively, the novel function of PROTEIN0 via PROTEIN1 PROTEIN2 influences PROTEIN3 transcription and apoptosis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Collectively, the novel function of PROTEIN0 via PROTEIN1 PROTEIN2 influences PROTEIN3 transcription and apoptosis.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Collectively, the novel function of PROTEIN0 via PROTEIN1 PROTEIN2 influences PROTEIN3 transcription and apoptosis.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Collectively, the novel function of PROTEIN0 via PROTEIN1 PROTEIN2 influences PROTEIN3 transcription and apoptosis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have crystallized a complex between human PROTEIN0 and a two-domain extracellular fragment of human PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The 2.8 A crystal structure of a complex between the extracellular portions of PROTEIN0 and PROTEIN1 shows two PROTEIN2 molecules which grasp each side of a twofold symmetric PROTEIN3 dimer.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The 2.8 A crystal structure of a complex between the extracellular portions of PROTEIN0 and PROTEIN1 shows two PROTEIN2 molecules which grasp each side of a twofold symmetric PROTEIN3 dimer.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The 2.8 A crystal structure of a complex between the extracellular portions of PROTEIN0 and PROTEIN1 shows two PROTEIN2 molecules which grasp each side of a twofold symmetric PROTEIN3 dimer.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The 2.8 A crystal structure of a complex between the extracellular portions of PROTEIN0 and PROTEIN1 shows two PROTEIN2 molecules which grasp each side of a twofold symmetric PROTEIN3 dimer.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The 2.8 A crystal structure of a complex between the extracellular portions of PROTEIN0 and PROTEIN1 shows two PROTEIN2 molecules which grasp each side of a twofold symmetric PROTEIN3 dimer.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The 2.8 A crystal structure of a complex between the extracellular portions of PROTEIN0 and PROTEIN1 shows two PROTEIN2 molecules which grasp each side of a twofold symmetric PROTEIN3 dimer.\n",
      "\n",
      "PROTEIN0 PROTEIN1 On a cell membrane containing both proteins, PROTEIN0 would 'lie down' parallel to the membrane, such that the PROTEIN1 helices that delineate the counterpart of the MHC peptide-binding groove make extensive contacts with helices in the PROTEIN2 dimerization domain.\n",
      "\n",
      "PROTEIN0 PROTEIN2 On a cell membrane containing both proteins, PROTEIN0 would 'lie down' parallel to the membrane, such that the PROTEIN1 helices that delineate the counterpart of the MHC peptide-binding groove make extensive contacts with helices in the PROTEIN2 dimerization domain.\n",
      "\n",
      "PROTEIN1 PROTEIN2 On a cell membrane containing both proteins, PROTEIN0 would 'lie down' parallel to the membrane, such that the PROTEIN1 helices that delineate the counterpart of the MHC peptide-binding groove make extensive contacts with helices in the PROTEIN2 dimerization domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The structures of PROTEIN0 alone and complexed with PROTEIN1 differ in their domain arrangement and dimer interfaces, providing a mechanism for communicating binding events between PROTEIN2 chains.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The structures of PROTEIN0 alone and complexed with PROTEIN1 differ in their domain arrangement and dimer interfaces, providing a mechanism for communicating binding events between PROTEIN2 chains.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The structures of PROTEIN0 alone and complexed with PROTEIN1 differ in their domain arrangement and dimer interfaces, providing a mechanism for communicating binding events between PROTEIN2 chains.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 complex suggests a binding site for transferrin on PROTEIN2 and sheds light upon the function of PROTEIN3 in regulating iron homeostasis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 PROTEIN1 complex suggests a binding site for transferrin on PROTEIN2 and sheds light upon the function of PROTEIN3 in regulating iron homeostasis.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 PROTEIN1 complex suggests a binding site for transferrin on PROTEIN2 and sheds light upon the function of PROTEIN3 in regulating iron homeostasis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 PROTEIN1 complex suggests a binding site for transferrin on PROTEIN2 and sheds light upon the function of PROTEIN3 in regulating iron homeostasis.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 PROTEIN1 complex suggests a binding site for transferrin on PROTEIN2 and sheds light upon the function of PROTEIN3 in regulating iron homeostasis.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 PROTEIN1 complex suggests a binding site for transferrin on PROTEIN2 and sheds light upon the function of PROTEIN3 in regulating iron homeostasis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The BMP receptors belong to the family of serine/threonine kinase receptors, whose activation has been investigated intensively for the PROTEIN0 PROTEIN1 receptor subfamily.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using several complementary approaches, we investigated the formation of homomeric and heteromeric complexes between the two known BMP type I receptors PROTEIN0 and PROTEIN1 and the PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using several complementary approaches, we investigated the formation of homomeric and heteromeric complexes between the two known BMP type I receptors PROTEIN0 and PROTEIN1 and the PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Using several complementary approaches, we investigated the formation of homomeric and heteromeric complexes between the two known BMP type I receptors PROTEIN0 and PROTEIN1 and the PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using several complementary approaches, we investigated the formation of homomeric and heteromeric complexes between the two known BMP type I receptors PROTEIN0 and PROTEIN1 and the PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Using several complementary approaches, we investigated the formation of homomeric and heteromeric complexes between the two known BMP type I receptors PROTEIN0 and PROTEIN1 and the PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Using several complementary approaches, we investigated the formation of homomeric and heteromeric complexes between the two known BMP type I receptors PROTEIN0 and PROTEIN1 and the PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN7 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN8 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN9 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN0 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN1 PROTEIN7 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN1 PROTEIN8 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN1 PROTEIN9 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN1 PROTEIN0 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN1 PROTEIN1 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN2 PROTEIN7 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN2 PROTEIN8 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN2 PROTEIN9 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN2 PROTEIN0 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN2 PROTEIN1 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN3 PROTEIN7 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN3 PROTEIN8 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN3 PROTEIN9 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN3 PROTEIN0 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN3 PROTEIN1 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN4 PROTEIN7 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN4 PROTEIN8 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN4 PROTEIN9 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN4 PROTEIN0 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN4 PROTEIN1 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN5 PROTEIN7 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN5 PROTEIN8 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN5 PROTEIN9 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN5 PROTEIN0 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN5 PROTEIN1 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN6 PROTEIN7 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN6 PROTEIN8 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN6 PROTEIN9 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN6 PROTEIN0 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN6 PROTEIN1 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN7 PROTEIN8 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN7 PROTEIN9 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN7 PROTEIN0 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN7 PROTEIN1 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN8 PROTEIN9 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN8 PROTEIN0 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN8 PROTEIN1 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN9 PROTEIN0 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN9 PROTEIN1 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using antibody-mediated immunofluorescence copatching of epitope-tagged receptors, we provide evidence in live cells for preexisting heteromeric PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 and homomeric PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and also PROTEIN0 PROTEIN1 oligomers in the absence of ligand.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binding significantly increased hetero- and homo-oligomerization (except for the PROTEIN1 homo-oligomer, which binds ligand poorly in the absence of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 binding significantly increased hetero- and homo-oligomerization (except for the PROTEIN1 homo-oligomer, which binds ligand poorly in the absence of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 binding significantly increased hetero- and homo-oligomerization (except for the PROTEIN1 homo-oligomer, which binds ligand poorly in the absence of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Although the individual amino acids in PROTEIN0 involved in binding to PROTEIN1 PROTEIN2 and those in PROTEIN3 involved in binding to PROTEIN4 have been mapped in this domain, the Trk amino acids that provide specificity remained unclear.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Although the individual amino acids in PROTEIN0 involved in binding to PROTEIN1 PROTEIN2 and those in PROTEIN3 involved in binding to PROTEIN4 have been mapped in this domain, the Trk amino acids that provide specificity remained unclear.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Although the individual amino acids in PROTEIN0 involved in binding to PROTEIN1 PROTEIN2 and those in PROTEIN3 involved in binding to PROTEIN4 have been mapped in this domain, the Trk amino acids that provide specificity remained unclear.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Although the individual amino acids in PROTEIN0 involved in binding to PROTEIN1 PROTEIN2 and those in PROTEIN3 involved in binding to PROTEIN4 have been mapped in this domain, the Trk amino acids that provide specificity remained unclear.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Although the individual amino acids in PROTEIN0 involved in binding to PROTEIN1 PROTEIN2 and those in PROTEIN3 involved in binding to PROTEIN4 have been mapped in this domain, the Trk amino acids that provide specificity remained unclear.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Although the individual amino acids in PROTEIN0 involved in binding to PROTEIN1 PROTEIN2 and those in PROTEIN3 involved in binding to PROTEIN4 have been mapped in this domain, the Trk amino acids that provide specificity remained unclear.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Although the individual amino acids in PROTEIN0 involved in binding to PROTEIN1 PROTEIN2 and those in PROTEIN3 involved in binding to PROTEIN4 have been mapped in this domain, the Trk amino acids that provide specificity remained unclear.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Although the individual amino acids in PROTEIN0 involved in binding to PROTEIN1 PROTEIN2 and those in PROTEIN3 involved in binding to PROTEIN4 have been mapped in this domain, the Trk amino acids that provide specificity remained unclear.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Although the individual amino acids in PROTEIN0 involved in binding to PROTEIN1 PROTEIN2 and those in PROTEIN3 involved in binding to PROTEIN4 have been mapped in this domain, the Trk amino acids that provide specificity remained unclear.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Although the individual amino acids in PROTEIN0 involved in binding to PROTEIN1 PROTEIN2 and those in PROTEIN3 involved in binding to PROTEIN4 have been mapped in this domain, the Trk amino acids that provide specificity remained unclear.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this study, a minimum set of residues in the human PROTEIN0 second immunoglobulin-like domain, which does not bind PROTEIN1 PROTEIN2 were substituted with those from human PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this study, a minimum set of residues in the human PROTEIN0 second immunoglobulin-like domain, which does not bind PROTEIN1 PROTEIN2 were substituted with those from human PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 In this study, a minimum set of residues in the human PROTEIN0 second immunoglobulin-like domain, which does not bind PROTEIN1 PROTEIN2 were substituted with those from human PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this study, a minimum set of residues in the human PROTEIN0 second immunoglobulin-like domain, which does not bind PROTEIN1 PROTEIN2 were substituted with those from human PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 In this study, a minimum set of residues in the human PROTEIN0 second immunoglobulin-like domain, which does not bind PROTEIN1 PROTEIN2 were substituted with those from human PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 In this study, a minimum set of residues in the human PROTEIN0 second immunoglobulin-like domain, which does not bind PROTEIN1 PROTEIN2 were substituted with those from human PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 The resulting Trk variant recruited binding of PROTEIN0 equivalent to PROTEIN1 maintained PROTEIN2 binding equivalent to PROTEIN3 and also bound PROTEIN4 although with lower affinity compared with PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Characterization of the PROTEIN0 binding domain on the human PROTEIN1 molecule using PROTEIN2 antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Characterization of the PROTEIN0 binding domain on the human PROTEIN1 molecule using PROTEIN2 antibodies.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Characterization of the PROTEIN0 binding domain on the human PROTEIN1 molecule using PROTEIN2 antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using a soluble PROTEIN0 PROTEIN1 chimeric protein, we assessed the ability of PROTEIN2 MAb to inhibit the binding of PROTEIN3 to PROTEIN4 expressed by the CD30+ Karpas 299 cell line.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using a soluble PROTEIN0 PROTEIN1 chimeric protein, we assessed the ability of PROTEIN2 MAb to inhibit the binding of PROTEIN3 to PROTEIN4 expressed by the CD30+ Karpas 299 cell line.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Using a soluble PROTEIN0 PROTEIN1 chimeric protein, we assessed the ability of PROTEIN2 MAb to inhibit the binding of PROTEIN3 to PROTEIN4 expressed by the CD30+ Karpas 299 cell line.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Using a soluble PROTEIN0 PROTEIN1 chimeric protein, we assessed the ability of PROTEIN2 MAb to inhibit the binding of PROTEIN3 to PROTEIN4 expressed by the CD30+ Karpas 299 cell line.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using a soluble PROTEIN0 PROTEIN1 chimeric protein, we assessed the ability of PROTEIN2 MAb to inhibit the binding of PROTEIN3 to PROTEIN4 expressed by the CD30+ Karpas 299 cell line.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Using a soluble PROTEIN0 PROTEIN1 chimeric protein, we assessed the ability of PROTEIN2 MAb to inhibit the binding of PROTEIN3 to PROTEIN4 expressed by the CD30+ Karpas 299 cell line.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Using a soluble PROTEIN0 PROTEIN1 chimeric protein, we assessed the ability of PROTEIN2 MAb to inhibit the binding of PROTEIN3 to PROTEIN4 expressed by the CD30+ Karpas 299 cell line.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Using a soluble PROTEIN0 PROTEIN1 chimeric protein, we assessed the ability of PROTEIN2 MAb to inhibit the binding of PROTEIN3 to PROTEIN4 expressed by the CD30+ Karpas 299 cell line.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Using a soluble PROTEIN0 PROTEIN1 chimeric protein, we assessed the ability of PROTEIN2 MAb to inhibit the binding of PROTEIN3 to PROTEIN4 expressed by the CD30+ Karpas 299 cell line.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Using a soluble PROTEIN0 PROTEIN1 chimeric protein, we assessed the ability of PROTEIN2 MAb to inhibit the binding of PROTEIN3 to PROTEIN4 expressed by the CD30+ Karpas 299 cell line.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binding by PROTEIN1 is blocked by MAb that recognize epitopes belonging to cluster Group A (like Ber-H2, Ber-H8, and HRS-4) as well as cluster Group C (like HeFi-1 and M44).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cluster Group B antibodies, including M67 and Ki-1, do not affect PROTEIN0 binding to PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The pattern of PROTEIN0 binding inhibition shows only limited correspondence to the functional capacity of some PROTEIN1 MAb to trigger PROTEIN2 signaling.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The pattern of PROTEIN0 binding inhibition shows only limited correspondence to the functional capacity of some PROTEIN1 MAb to trigger PROTEIN2 signaling.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The pattern of PROTEIN0 binding inhibition shows only limited correspondence to the functional capacity of some PROTEIN1 MAb to trigger PROTEIN2 signaling.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Finally, we demonstrate that the PROTEIN0 MAb M81 also completely inhibits PROTEIN1 PROTEIN2 interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Finally, we demonstrate that the PROTEIN0 MAb M81 also completely inhibits PROTEIN1 PROTEIN2 interaction.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Finally, we demonstrate that the PROTEIN0 MAb M81 also completely inhibits PROTEIN1 PROTEIN2 interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This information is useful for applying these MAbs in functional studies to further investigate the PROTEIN0 PROTEIN1 system and for designing assays for soluble PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 This information is useful for applying these MAbs in functional studies to further investigate the PROTEIN0 PROTEIN1 system and for designing assays for soluble PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 This information is useful for applying these MAbs in functional studies to further investigate the PROTEIN0 PROTEIN1 system and for designing assays for soluble PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 antagonists emerge from alterations in the low-affinity binding epitope for receptor PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 induces bone formation and regeneration in adult vertebrates and regulates important developmental processes in all animals.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a homodimeric cysteine knot protein that, as a member of the PROTEIN1 PROTEIN2 superfamily, signals by oligomerizing type I and type II receptor serine-kinases in the cell membrane.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is a homodimeric cysteine knot protein that, as a member of the PROTEIN1 PROTEIN2 superfamily, signals by oligomerizing type I and type II receptor serine-kinases in the cell membrane.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is a homodimeric cysteine knot protein that, as a member of the PROTEIN1 PROTEIN2 superfamily, signals by oligomerizing type I and type II receptor serine-kinases in the cell membrane.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The binding epitopes of PROTEIN0 for PROTEIN1 (type I) and PROTEIN2 or PROTEIN3 (type II) were characterized using PROTEIN4 mutant proteins for analysis of interactions with receptor ectodomains.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The binding epitopes of PROTEIN0 for PROTEIN1 (type I) and PROTEIN2 or PROTEIN3 (type II) were characterized using PROTEIN4 mutant proteins for analysis of interactions with receptor ectodomains.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The binding epitopes of PROTEIN0 for PROTEIN1 (type I) and PROTEIN2 or PROTEIN3 (type II) were characterized using PROTEIN4 mutant proteins for analysis of interactions with receptor ectodomains.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The binding epitopes of PROTEIN0 for PROTEIN1 (type I) and PROTEIN2 or PROTEIN3 (type II) were characterized using PROTEIN4 mutant proteins for analysis of interactions with receptor ectodomains.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The binding epitopes of PROTEIN0 for PROTEIN1 (type I) and PROTEIN2 or PROTEIN3 (type II) were characterized using PROTEIN4 mutant proteins for analysis of interactions with receptor ectodomains.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The binding epitopes of PROTEIN0 for PROTEIN1 (type I) and PROTEIN2 or PROTEIN3 (type II) were characterized using PROTEIN4 mutant proteins for analysis of interactions with receptor ectodomains.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The binding epitopes of PROTEIN0 for PROTEIN1 (type I) and PROTEIN2 or PROTEIN3 (type II) were characterized using PROTEIN4 mutant proteins for analysis of interactions with receptor ectodomains.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The binding epitopes of PROTEIN0 for PROTEIN1 (type I) and PROTEIN2 or PROTEIN3 (type II) were characterized using PROTEIN4 mutant proteins for analysis of interactions with receptor ectodomains.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The binding epitopes of PROTEIN0 for PROTEIN1 (type I) and PROTEIN2 or PROTEIN3 (type II) were characterized using PROTEIN4 mutant proteins for analysis of interactions with receptor ectodomains.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The binding epitopes of PROTEIN0 for PROTEIN1 (type I) and PROTEIN2 or PROTEIN3 (type II) were characterized using PROTEIN4 mutant proteins for analysis of interactions with receptor ectodomains.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A large epitope 1 for high-affinity PROTEIN0 binding was detected spanning the interface of the PROTEIN1 dimer.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Crystal structure of the PROTEIN0 PROTEIN1 ectodomain complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Bone morphogenetic proteins (BMPs) belong to the large PROTEIN0 PROTEIN1 superfamily of multifunctional cytokines.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The receptor chains bind to the 'wrist' epitopes of the PROTEIN0 dimer and contact both PROTEIN1 monomers.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 induces chemotaxis and adhesion by interacting with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 induces chemotaxis and adhesion by interacting with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 induces chemotaxis and adhesion by interacting with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 PROTEIN1 is an unusually large CC chemokine, which is also known as PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 We report chemotaxis and competitive binding studies that show PROTEIN0 binds to and activates PROTEIN1 and PROTEIN2 transfected HEK-293 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We report chemotaxis and competitive binding studies that show PROTEIN0 binds to and activates PROTEIN1 and PROTEIN2 transfected HEK-293 cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We report chemotaxis and competitive binding studies that show PROTEIN0 binds to and activates PROTEIN1 and PROTEIN2 transfected HEK-293 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 induced maximal migration of PROTEIN1 and PROTEIN2 transfected cells at 89.3 nmol/L and cell adhesion at 5.6 nmol/L.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 induced maximal migration of PROTEIN1 and PROTEIN2 transfected cells at 89.3 nmol/L and cell adhesion at 5.6 nmol/L.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 induced maximal migration of PROTEIN1 and PROTEIN2 transfected cells at 89.3 nmol/L and cell adhesion at 5.6 nmol/L.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The molar concentration of PROTEIN0 required to induce maximum cell migration is 20- to 200-fold greater than that required for PROTEIN1 or PROTEIN2 respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The molar concentration of PROTEIN0 required to induce maximum cell migration is 20- to 200-fold greater than that required for PROTEIN1 or PROTEIN2 respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The molar concentration of PROTEIN0 required to induce maximum cell migration is 20- to 200-fold greater than that required for PROTEIN1 or PROTEIN2 respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A neutralizing polyclonal antibody to PROTEIN0 was developed to demonstrate that the unusually high concentration of PROTEIN1 required to induce chemotaxis was a property of PROTEIN2 and not as a result of an irrelevant protein contamination.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A neutralizing polyclonal antibody to PROTEIN0 was developed to demonstrate that the unusually high concentration of PROTEIN1 required to induce chemotaxis was a property of PROTEIN2 and not as a result of an irrelevant protein contamination.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A neutralizing polyclonal antibody to PROTEIN0 was developed to demonstrate that the unusually high concentration of PROTEIN1 required to induce chemotaxis was a property of PROTEIN2 and not as a result of an irrelevant protein contamination.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 contains this primary site as well as a region that restricts interaction with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The sequences that confer on PROTEIN0 its specific binding to PROTEIN1 have not been identified.\n",
      "\n",
      "PROTEIN0 PROTEIN1 By utilizing domain swapping and site-directed mutagenesis we have found that the loop connecting the beta4-beta5 strands of PROTEIN0 contributes to high affinity receptor binding and is critical for PROTEIN1 recognition.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Replacement of this loop with the homologous loop from PROTEIN0 dramatically reduced both the affinity of PROTEIN1 for PROTEIN2 and its biological potency but did not result in the ability to bind PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Replacement of this loop with the homologous loop from PROTEIN0 dramatically reduced both the affinity of PROTEIN1 for PROTEIN2 and its biological potency but did not result in the ability to bind PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Replacement of this loop with the homologous loop from PROTEIN0 dramatically reduced both the affinity of PROTEIN1 for PROTEIN2 and its biological potency but did not result in the ability to bind PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Replacement of this loop with the homologous loop from PROTEIN0 dramatically reduced both the affinity of PROTEIN1 for PROTEIN2 and its biological potency but did not result in the ability to bind PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Replacement of this loop with the homologous loop from PROTEIN0 dramatically reduced both the affinity of PROTEIN1 for PROTEIN2 and its biological potency but did not result in the ability to bind PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Replacement of this loop with the homologous loop from PROTEIN0 dramatically reduced both the affinity of PROTEIN1 for PROTEIN2 and its biological potency but did not result in the ability to bind PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The reciprocal loop replacement mutant PROTEIN0 retained PROTEIN1 like affinity for PROTEIN2 and for PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The reciprocal loop replacement mutant PROTEIN0 retained PROTEIN1 like affinity for PROTEIN2 and for PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The reciprocal loop replacement mutant PROTEIN0 retained PROTEIN1 like affinity for PROTEIN2 and for PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The reciprocal loop replacement mutant PROTEIN0 retained PROTEIN1 like affinity for PROTEIN2 and for PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The reciprocal loop replacement mutant PROTEIN0 retained PROTEIN1 like affinity for PROTEIN2 and for PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The reciprocal loop replacement mutant PROTEIN0 retained PROTEIN1 like affinity for PROTEIN2 and for PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Type I interferons bind to two cell surface receptors, PROTEIN0 and PROTEIN1 as the first step in the activation of several signal transduction pathways that elicit an anti-viral state and an anti-proliferative response.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, we quantitatively mapped the complete binding region of PROTEIN0 on PROTEIN1 by 35 individual mutations to alanine and isosteric residues.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interestingly, the PROTEIN0 binding site overlaps the largest continuous hydrophobic patch on PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Relating the anti-viral and anti-proliferative activity of the various interferon mutants with their affinity toward PROTEIN0 results in linear function over the whole range of affinities investigated, suggesting that PROTEIN1 binding is the rate-determining step in cellular activation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we report the crystal structure of the PROTEIN0 ectodomain in a dimeric form that is induced by simultaneous binding to PROTEIN1 and a heparin decasaccharide.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The structure of the PROTEIN0 PROTEIN1 ternary complex provides a structural basis for the essential role of heparan sulphate in FGF signalling.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN2 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN3 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN4 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN5 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN6 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN7 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN8 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN2 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN3 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN4 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN5 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN6 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN7 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN8 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN3 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN4 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN5 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN6 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN7 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN8 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN4 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN5 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN6 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN7 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN8 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN5 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN6 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN7 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN8 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN5 PROTEIN6 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN5 PROTEIN7 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN5 PROTEIN8 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN6 PROTEIN7 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN6 PROTEIN8 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN7 PROTEIN8 We show that the S252W mutation allows the mesenchymal splice form of PROTEIN0 PROTEIN1 to bind and to be activated by the mesenchymally expressed ligands PROTEIN2 or PROTEIN3 and the epithelial splice form of PROTEIN4 PROTEIN5 to be activated by PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN1 Kaposi's sarcoma-associated herpesvirus encodes a functional homolog of human PROTEIN0 PROTEIN1 that activates human PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Kaposi's sarcoma-associated herpesvirus encodes a functional homolog of human PROTEIN0 PROTEIN1 that activates human PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Kaposi's sarcoma-associated herpesvirus encodes a functional homolog of human PROTEIN0 PROTEIN1 that activates human PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the 2.4 angstrom crystal structure of the extracellular signaling assembly between viral PROTEIN0 and human PROTEIN1 two complexes are cross-linked into a tetramer through direct interactions between the immunoglobulin domain of PROTEIN2 and site III of viral PROTEIN3 which is necessary for receptor activation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the 2.4 angstrom crystal structure of the extracellular signaling assembly between viral PROTEIN0 and human PROTEIN1 two complexes are cross-linked into a tetramer through direct interactions between the immunoglobulin domain of PROTEIN2 and site III of viral PROTEIN3 which is necessary for receptor activation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In the 2.4 angstrom crystal structure of the extracellular signaling assembly between viral PROTEIN0 and human PROTEIN1 two complexes are cross-linked into a tetramer through direct interactions between the immunoglobulin domain of PROTEIN2 and site III of viral PROTEIN3 which is necessary for receptor activation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the 2.4 angstrom crystal structure of the extracellular signaling assembly between viral PROTEIN0 and human PROTEIN1 two complexes are cross-linked into a tetramer through direct interactions between the immunoglobulin domain of PROTEIN2 and site III of viral PROTEIN3 which is necessary for receptor activation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In the 2.4 angstrom crystal structure of the extracellular signaling assembly between viral PROTEIN0 and human PROTEIN1 two complexes are cross-linked into a tetramer through direct interactions between the immunoglobulin domain of PROTEIN2 and site III of viral PROTEIN3 which is necessary for receptor activation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In the 2.4 angstrom crystal structure of the extracellular signaling assembly between viral PROTEIN0 and human PROTEIN1 two complexes are cross-linked into a tetramer through direct interactions between the immunoglobulin domain of PROTEIN2 and site III of viral PROTEIN3 which is necessary for receptor activation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Unlike human PROTEIN0 (which uses many hydrophilic residues), the viral cytokine largely uses hydrophobic amino acids to contact PROTEIN1 which enhances the complementarity of the viral PROTEIN2 PROTEIN3 binding interfaces.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Unlike human PROTEIN0 (which uses many hydrophilic residues), the viral cytokine largely uses hydrophobic amino acids to contact PROTEIN1 which enhances the complementarity of the viral PROTEIN2 PROTEIN3 binding interfaces.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Unlike human PROTEIN0 (which uses many hydrophilic residues), the viral cytokine largely uses hydrophobic amino acids to contact PROTEIN1 which enhances the complementarity of the viral PROTEIN2 PROTEIN3 binding interfaces.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Unlike human PROTEIN0 (which uses many hydrophilic residues), the viral cytokine largely uses hydrophobic amino acids to contact PROTEIN1 which enhances the complementarity of the viral PROTEIN2 PROTEIN3 binding interfaces.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Unlike human PROTEIN0 (which uses many hydrophilic residues), the viral cytokine largely uses hydrophobic amino acids to contact PROTEIN1 which enhances the complementarity of the viral PROTEIN2 PROTEIN3 binding interfaces.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Unlike human PROTEIN0 (which uses many hydrophilic residues), the viral cytokine largely uses hydrophobic amino acids to contact PROTEIN1 which enhances the complementarity of the viral PROTEIN2 PROTEIN3 binding interfaces.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The cross-reactivity of PROTEIN0 is apparently due to a chemical plasticity evident in the amphipathic PROTEIN1 cytokine-binding sites.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 a novel growth factor that binds to PROTEIN2 and beta receptor.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 a novel growth factor that binds to PROTEIN2 and beta receptor.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 a novel growth factor that binds to PROTEIN2 and beta receptor.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a multidomain protein with the N-terminal region homologous to the extracellular CUB domain of PROTEIN1 and the C-terminal region consists of a growth factor domain (GFD) with homology to vascular PROTEIN2 (25%) and PROTEIN3 (23%).\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is a multidomain protein with the N-terminal region homologous to the extracellular CUB domain of PROTEIN1 and the C-terminal region consists of a growth factor domain (GFD) with homology to vascular PROTEIN2 (25%) and PROTEIN3 (23%).\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 is a multidomain protein with the N-terminal region homologous to the extracellular CUB domain of PROTEIN1 and the C-terminal region consists of a growth factor domain (GFD) with homology to vascular PROTEIN2 (25%) and PROTEIN3 (23%).\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is a multidomain protein with the N-terminal region homologous to the extracellular CUB domain of PROTEIN1 and the C-terminal region consists of a growth factor domain (GFD) with homology to vascular PROTEIN2 (25%) and PROTEIN3 (23%).\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 is a multidomain protein with the N-terminal region homologous to the extracellular CUB domain of PROTEIN1 and the C-terminal region consists of a growth factor domain (GFD) with homology to vascular PROTEIN2 (25%) and PROTEIN3 (23%).\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 is a multidomain protein with the N-terminal region homologous to the extracellular CUB domain of PROTEIN1 and the C-terminal region consists of a growth factor domain (GFD) with homology to vascular PROTEIN2 (25%) and PROTEIN3 (23%).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Competition binding and immunoprecipitation studies on cells bearing both PROTEIN0 and beta receptors reveal a high affinity binding of recombinant GFD PROTEIN1 to PROTEIN2 homodimers and PROTEIN3 heterodimers.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Competition binding and immunoprecipitation studies on cells bearing both PROTEIN0 and beta receptors reveal a high affinity binding of recombinant GFD PROTEIN1 to PROTEIN2 homodimers and PROTEIN3 heterodimers.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Competition binding and immunoprecipitation studies on cells bearing both PROTEIN0 and beta receptors reveal a high affinity binding of recombinant GFD PROTEIN1 to PROTEIN2 homodimers and PROTEIN3 heterodimers.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Competition binding and immunoprecipitation studies on cells bearing both PROTEIN0 and beta receptors reveal a high affinity binding of recombinant GFD PROTEIN1 to PROTEIN2 homodimers and PROTEIN3 heterodimers.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Competition binding and immunoprecipitation studies on cells bearing both PROTEIN0 and beta receptors reveal a high affinity binding of recombinant GFD PROTEIN1 to PROTEIN2 homodimers and PROTEIN3 heterodimers.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Competition binding and immunoprecipitation studies on cells bearing both PROTEIN0 and beta receptors reveal a high affinity binding of recombinant GFD PROTEIN1 to PROTEIN2 homodimers and PROTEIN3 heterodimers.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 exhibits greater mitogenic potency than PROTEIN1 and comparable or greater mitogenic activity than PROTEIN2 and PROTEIN3 on several mesenchymal cell types.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 exhibits greater mitogenic potency than PROTEIN1 and comparable or greater mitogenic activity than PROTEIN2 and PROTEIN3 on several mesenchymal cell types.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 exhibits greater mitogenic potency than PROTEIN1 and comparable or greater mitogenic activity than PROTEIN2 and PROTEIN3 on several mesenchymal cell types.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 exhibits greater mitogenic potency than PROTEIN1 and comparable or greater mitogenic activity than PROTEIN2 and PROTEIN3 on several mesenchymal cell types.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 exhibits greater mitogenic potency than PROTEIN1 and comparable or greater mitogenic activity than PROTEIN2 and PROTEIN3 on several mesenchymal cell types.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 exhibits greater mitogenic potency than PROTEIN1 and comparable or greater mitogenic activity than PROTEIN2 and PROTEIN3 on several mesenchymal cell types.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Analysis of PROTEIN0 in vivo in a diabetic mouse model of delayed wound healing showed that PROTEIN1 significantly enhanced repair of a full-thickness skin excision.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Together, these studies describe a third member of the PDGF family PROTEIN0 as a potent mitogen for cells of mesenchymal origin in in vitro and in vivo systems with a binding pattern similar to PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cloning of PROTEIN0 a ligand for PROTEIN1 on human T cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A cDNA clone, designated PROTEIN0 PROTEIN1 was isolated from a collection of T cell activation genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is highly similar to an identified murine PROTEIN1 both at the cDNA (82.8% identity) and the protein (77.4% identity) levels, and related to tumor necrosis PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is highly similar to an identified murine PROTEIN1 both at the cDNA (82.8% identity) and the protein (77.4% identity) levels, and related to tumor necrosis PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is highly similar to an identified murine PROTEIN1 both at the cDNA (82.8% identity) and the protein (77.4% identity) levels, and related to tumor necrosis PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Expressed in a murine myeloma, PROTEIN0 was identified as a ligand for PROTEIN1 by binding to a soluble PROTEIN2 construct.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Expressed in a murine myeloma, PROTEIN0 was identified as a ligand for PROTEIN1 by binding to a soluble PROTEIN2 construct.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Expressed in a murine myeloma, PROTEIN0 was identified as a ligand for PROTEIN1 by binding to a soluble PROTEIN2 construct.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Reconstitution of the multiprotein complex of PROTEIN0 PROTEIN1 and PROTEIN2 in a cell-free system.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Reconstitution of the multiprotein complex of PROTEIN0 PROTEIN1 and PROTEIN2 in a cell-free system.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Reconstitution of the multiprotein complex of PROTEIN0 PROTEIN1 and PROTEIN2 in a cell-free system.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A rabbit reticulocyte lysate system that has been used to reconstitute functional complexes between steroid receptors and the 90-kDa heat shock protein PROTEIN0 has been used here to form complexes between the PROTEIN1 tyrosine kinase and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 A rabbit reticulocyte lysate system that has been used to reconstitute functional complexes between steroid receptors and the 90-kDa heat shock protein PROTEIN0 has been used here to form complexes between the PROTEIN1 tyrosine kinase and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 A rabbit reticulocyte lysate system that has been used to reconstitute functional complexes between steroid receptors and the 90-kDa heat shock protein PROTEIN0 has been used here to form complexes between the PROTEIN1 tyrosine kinase and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Reticulocyte lysate forms complexes between PROTEIN0 and a temperature-sensitive mutant of Rous sarcoma virus PROTEIN1 which is normally present in cytosol virtually entirely in the multiprotein complex form.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The native and reconstituted PROTEIN0 PROTEIN1 complexes have similar thermal stability and, like steroid receptor heterocomplexes, they are stabilized by molybdate.\n",
      "\n",
      "PROTEIN0 PROTEIN1 As previously shown with reticulocyte lysate-reconstituted steroid receptor heteroprotein complexes, the reconstituted PROTEIN0 multiprotein complex contains PROTEIN1 which is a major candidate for providing the protein unfoldase activity required for PROTEIN2 association.\n",
      "\n",
      "PROTEIN0 PROTEIN2 As previously shown with reticulocyte lysate-reconstituted steroid receptor heteroprotein complexes, the reconstituted PROTEIN0 multiprotein complex contains PROTEIN1 which is a major candidate for providing the protein unfoldase activity required for PROTEIN2 association.\n",
      "\n",
      "PROTEIN1 PROTEIN2 As previously shown with reticulocyte lysate-reconstituted steroid receptor heteroprotein complexes, the reconstituted PROTEIN0 multiprotein complex contains PROTEIN1 which is a major candidate for providing the protein unfoldase activity required for PROTEIN2 association.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A cDNA clone encoding a novel, widely expressed protein (called PROTEIN0 or PROTEIN1 containing one src homology 2 (SH2) domain and two SH3 domains was isolated.\n",
      "\n",
      "PROTEIN0 PROTEIN1 While injection of PROTEIN0 or PROTEIN1 proteins alone into quiescent rat fibroblasts did not have mitogenic effect, microinjection of PROTEIN2 together with PROTEIN3 protein stimulated DNA synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 While injection of PROTEIN0 or PROTEIN1 proteins alone into quiescent rat fibroblasts did not have mitogenic effect, microinjection of PROTEIN2 together with PROTEIN3 protein stimulated DNA synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN3 While injection of PROTEIN0 or PROTEIN1 proteins alone into quiescent rat fibroblasts did not have mitogenic effect, microinjection of PROTEIN2 together with PROTEIN3 protein stimulated DNA synthesis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 While injection of PROTEIN0 or PROTEIN1 proteins alone into quiescent rat fibroblasts did not have mitogenic effect, microinjection of PROTEIN2 together with PROTEIN3 protein stimulated DNA synthesis.\n",
      "\n",
      "PROTEIN1 PROTEIN3 While injection of PROTEIN0 or PROTEIN1 proteins alone into quiescent rat fibroblasts did not have mitogenic effect, microinjection of PROTEIN2 together with PROTEIN3 protein stimulated DNA synthesis.\n",
      "\n",
      "PROTEIN2 PROTEIN3 While injection of PROTEIN0 or PROTEIN1 proteins alone into quiescent rat fibroblasts did not have mitogenic effect, microinjection of PROTEIN2 together with PROTEIN3 protein stimulated DNA synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 PROTEIN1 plays a crucial role in a highly conserved mechanism for growth factor control of ras signaling.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Analysis of the oligomerization of PROTEIN0 and PROTEIN1 products in vivo using a two-hybrid assay system.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a muscle-specific HLH protein that binds DNA in vitro as a heterodimer with several widely expressed HLH proteins, such as the PROTEIN1 gene products PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is a muscle-specific HLH protein that binds DNA in vitro as a heterodimer with several widely expressed HLH proteins, such as the PROTEIN1 gene products PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 is a muscle-specific HLH protein that binds DNA in vitro as a heterodimer with several widely expressed HLH proteins, such as the PROTEIN1 gene products PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is a muscle-specific HLH protein that binds DNA in vitro as a heterodimer with several widely expressed HLH proteins, such as the PROTEIN1 gene products PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 is a muscle-specific HLH protein that binds DNA in vitro as a heterodimer with several widely expressed HLH proteins, such as the PROTEIN1 gene products PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 is a muscle-specific HLH protein that binds DNA in vitro as a heterodimer with several widely expressed HLH proteins, such as the PROTEIN1 gene products PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The results show that PROTEIN0 heterodimerizes with PROTEIN1 and PROTEIN2 in vivo, but it does not homodimerize to a measurable extent.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The results show that PROTEIN0 heterodimerizes with PROTEIN1 and PROTEIN2 in vivo, but it does not homodimerize to a measurable extent.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The results show that PROTEIN0 heterodimerizes with PROTEIN1 and PROTEIN2 in vivo, but it does not homodimerize to a measurable extent.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Peptide growth factors, as well as the immediate early gene products PROTEIN0 PROTEIN1 and PROTEIN2 inhibit the activity of myogenin through a mechanism independent of its association with PROTEIN3 products.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Peptide growth factors, as well as the immediate early gene products PROTEIN0 PROTEIN1 and PROTEIN2 inhibit the activity of myogenin through a mechanism independent of its association with PROTEIN3 products.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Peptide growth factors, as well as the immediate early gene products PROTEIN0 PROTEIN1 and PROTEIN2 inhibit the activity of myogenin through a mechanism independent of its association with PROTEIN3 products.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Peptide growth factors, as well as the immediate early gene products PROTEIN0 PROTEIN1 and PROTEIN2 inhibit the activity of myogenin through a mechanism independent of its association with PROTEIN3 products.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Peptide growth factors, as well as the immediate early gene products PROTEIN0 PROTEIN1 and PROTEIN2 inhibit the activity of myogenin through a mechanism independent of its association with PROTEIN3 products.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Peptide growth factors, as well as the immediate early gene products PROTEIN0 PROTEIN1 and PROTEIN2 inhibit the activity of myogenin through a mechanism independent of its association with PROTEIN3 products.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and the protein kinase that activates it PROTEIN1 were constitutively activated in NIH 3T3 cells infected with viruses containing either of two oncogenic forms (p35EC12, p3722W) of the PROTEIN2 protein kinase.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 and the protein kinase that activates it PROTEIN1 were constitutively activated in NIH 3T3 cells infected with viruses containing either of two oncogenic forms (p35EC12, p3722W) of the PROTEIN2 protein kinase.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 and the protein kinase that activates it PROTEIN1 were constitutively activated in NIH 3T3 cells infected with viruses containing either of two oncogenic forms (p35EC12, p3722W) of the PROTEIN2 protein kinase.\n",
      "\n",
      "PROTEIN0 PROTEIN1 An expression plasmid encoding the human 75-kDa PROTEIN0 PROTEIN1 was constructed and used to generate a stable human cell line PROTEIN2 overexpressing PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 An expression plasmid encoding the human 75-kDa PROTEIN0 PROTEIN1 was constructed and used to generate a stable human cell line PROTEIN2 overexpressing PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 An expression plasmid encoding the human 75-kDa PROTEIN0 PROTEIN1 was constructed and used to generate a stable human cell line PROTEIN2 overexpressing PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 An expression plasmid encoding the human 75-kDa PROTEIN0 PROTEIN1 was constructed and used to generate a stable human cell line PROTEIN2 overexpressing PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 An expression plasmid encoding the human 75-kDa PROTEIN0 PROTEIN1 was constructed and used to generate a stable human cell line PROTEIN2 overexpressing PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 An expression plasmid encoding the human 75-kDa PROTEIN0 PROTEIN1 was constructed and used to generate a stable human cell line PROTEIN2 overexpressing PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Ligand binding analysis revealed high affinity binding (Kd = 0.2 nM) with approximately 94,000 +/- 7,500 sites/cell for PROTEIN0 and approximately 5-fold lower affinity for PROTEIN1 (Kd = 1.1 nM) with 264,000 +/- 2,000 sites/cell.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Cross-linking of PROTEIN0 and PROTEIN1 to PROTEIN2 cells yielded predominant complexes with apparent molecular weights of 211,000 for PROTEIN3 and 205,000 and 244,000 for PROTEIN4 suggesting these complexes contain two or three PROTEIN5 molecules.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoprecipitation of PROTEIN0 from 32P-labeled PROTEIN1 cells demonstrated that the receptor is phosphorylated.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 treatment of PROTEIN1 cells did not significantly affect the degree or pattern of phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Shedding of the extracellular domain of PROTEIN0 was induced by 4 beta-phorbol 12-myristate 13-acetate but not by PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Shedding of the extracellular domain of PROTEIN0 was induced by 4 beta-phorbol 12-myristate 13-acetate but not by PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Shedding of the extracellular domain of PROTEIN0 was induced by 4 beta-phorbol 12-myristate 13-acetate but not by PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Coassociation of PROTEIN0 PROTEIN1 with PROTEIN2 on the surface of human T lymphocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Coassociation of PROTEIN0 PROTEIN1 with PROTEIN2 on the surface of human T lymphocytes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Coassociation of PROTEIN0 PROTEIN1 with PROTEIN2 on the surface of human T lymphocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the present report, we demonstrated that modulation of PROTEIN0 from T cell surface induced by PROTEIN1 (1F7) led to enhanced phosphorylation of PROTEIN2 zeta tyrosine residues and increased PROTEIN3 associated p56lck tyrosine kinase activity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the present report, we demonstrated that modulation of PROTEIN0 from T cell surface induced by PROTEIN1 (1F7) led to enhanced phosphorylation of PROTEIN2 zeta tyrosine residues and increased PROTEIN3 associated p56lck tyrosine kinase activity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In the present report, we demonstrated that modulation of PROTEIN0 from T cell surface induced by PROTEIN1 (1F7) led to enhanced phosphorylation of PROTEIN2 zeta tyrosine residues and increased PROTEIN3 associated p56lck tyrosine kinase activity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the present report, we demonstrated that modulation of PROTEIN0 from T cell surface induced by PROTEIN1 (1F7) led to enhanced phosphorylation of PROTEIN2 zeta tyrosine residues and increased PROTEIN3 associated p56lck tyrosine kinase activity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In the present report, we demonstrated that modulation of PROTEIN0 from T cell surface induced by PROTEIN1 (1F7) led to enhanced phosphorylation of PROTEIN2 zeta tyrosine residues and increased PROTEIN3 associated p56lck tyrosine kinase activity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In the present report, we demonstrated that modulation of PROTEIN0 from T cell surface induced by PROTEIN1 (1F7) led to enhanced phosphorylation of PROTEIN2 zeta tyrosine residues and increased PROTEIN3 associated p56lck tyrosine kinase activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We further showed that PROTEIN0 was comodulated on the T cell surface with PROTEIN1 a known membrane-linked protein tyrosine phosphatase and that PROTEIN2 was capable of precipitating PROTEIN3 from T cell lysates.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We further showed that PROTEIN0 was comodulated on the T cell surface with PROTEIN1 a known membrane-linked protein tyrosine phosphatase and that PROTEIN2 was capable of precipitating PROTEIN3 from T cell lysates.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We further showed that PROTEIN0 was comodulated on the T cell surface with PROTEIN1 a known membrane-linked protein tyrosine phosphatase and that PROTEIN2 was capable of precipitating PROTEIN3 from T cell lysates.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We further showed that PROTEIN0 was comodulated on the T cell surface with PROTEIN1 a known membrane-linked protein tyrosine phosphatase and that PROTEIN2 was capable of precipitating PROTEIN3 from T cell lysates.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We further showed that PROTEIN0 was comodulated on the T cell surface with PROTEIN1 a known membrane-linked protein tyrosine phosphatase and that PROTEIN2 was capable of precipitating PROTEIN3 from T cell lysates.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We further showed that PROTEIN0 was comodulated on the T cell surface with PROTEIN1 a known membrane-linked protein tyrosine phosphatase and that PROTEIN2 was capable of precipitating PROTEIN3 from T cell lysates.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These findings strongly suggest that PROTEIN0 may be closely associated with the PROTEIN1 protein tyrosine phosphatase on T cell surface and further support the notion that the interaction of PROTEIN2 with PROTEIN3 results in enhanced tyrosine kinase activity, zeta chain phosphorylation, and T cell activation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These findings strongly suggest that PROTEIN0 may be closely associated with the PROTEIN1 protein tyrosine phosphatase on T cell surface and further support the notion that the interaction of PROTEIN2 with PROTEIN3 results in enhanced tyrosine kinase activity, zeta chain phosphorylation, and T cell activation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 These findings strongly suggest that PROTEIN0 may be closely associated with the PROTEIN1 protein tyrosine phosphatase on T cell surface and further support the notion that the interaction of PROTEIN2 with PROTEIN3 results in enhanced tyrosine kinase activity, zeta chain phosphorylation, and T cell activation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These findings strongly suggest that PROTEIN0 may be closely associated with the PROTEIN1 protein tyrosine phosphatase on T cell surface and further support the notion that the interaction of PROTEIN2 with PROTEIN3 results in enhanced tyrosine kinase activity, zeta chain phosphorylation, and T cell activation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 These findings strongly suggest that PROTEIN0 may be closely associated with the PROTEIN1 protein tyrosine phosphatase on T cell surface and further support the notion that the interaction of PROTEIN2 with PROTEIN3 results in enhanced tyrosine kinase activity, zeta chain phosphorylation, and T cell activation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 These findings strongly suggest that PROTEIN0 may be closely associated with the PROTEIN1 protein tyrosine phosphatase on T cell surface and further support the notion that the interaction of PROTEIN2 with PROTEIN3 results in enhanced tyrosine kinase activity, zeta chain phosphorylation, and T cell activation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have isolated and characterized a cDNA clone, PROTEIN0 encoding a human fibroblast growth factor (FGF) receptor. PROTEIN1 contains an open reading frame which encodes an 820 amino acid polypeptide with three immunoglobulin-like domains in the extracellular part and an intracellular split tyrosine kinase domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Transient expression in COS-1 cells and immunoprecipitation using an antiserum raised against a C-terminal peptide, gave rise to two components, representing mature (130 kDa) and precursor (115 kDa) forms of the PROTEIN0 encoded polypeptide, which was denoted PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 Crosslinking of iodinated PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 to transiently expressing COS-1 cells revealed a major band of 95 kDa, which was competed for by both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 From Scatchard analyses, the Kd:s for binding of PROTEIN0 and PROTEIN1 to PROTEIN2 were estimated to 25 pM and 41 pM, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN2 From Scatchard analyses, the Kd:s for binding of PROTEIN0 and PROTEIN1 to PROTEIN2 were estimated to 25 pM and 41 pM, respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN2 From Scatchard analyses, the Kd:s for binding of PROTEIN0 and PROTEIN1 to PROTEIN2 were estimated to 25 pM and 41 pM, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, PROTEIN0 encodes a human FGF receptor with high affinity for both PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, PROTEIN0 encodes a human FGF receptor with high affinity for both PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, PROTEIN0 encodes a human FGF receptor with high affinity for both PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cross-family dimerization of transcription factors PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 alters DNA binding specificity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Cross-family dimerization of transcription factors PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 alters DNA binding specificity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Cross-family dimerization of transcription factors PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 alters DNA binding specificity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Cross-family dimerization of transcription factors PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 alters DNA binding specificity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Cross-family dimerization of transcription factors PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 alters DNA binding specificity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Cross-family dimerization of transcription factors PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 alters DNA binding specificity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 families of transcription factors function in coupling extracellular signals to alterations in expression of specific target genes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 families of transcription factors function in coupling extracellular signals to alterations in expression of specific target genes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 families of transcription factors function in coupling extracellular signals to alterations in expression of specific target genes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 families of transcription factors function in coupling extracellular signals to alterations in expression of specific target genes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 families of transcription factors function in coupling extracellular signals to alterations in expression of specific target genes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 families of transcription factors function in coupling extracellular signals to alterations in expression of specific target genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Although PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 were previously thought to interact preferentially with different DNA regulatory elements (the AP-1/TRE and ATF/CRE sites, respectively), we find that members of these two families form selective cross-family heterodimers.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Although PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 were previously thought to interact preferentially with different DNA regulatory elements (the AP-1/TRE and ATF/CRE sites, respectively), we find that members of these two families form selective cross-family heterodimers.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Although PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 were previously thought to interact preferentially with different DNA regulatory elements (the AP-1/TRE and ATF/CRE sites, respectively), we find that members of these two families form selective cross-family heterodimers.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Although PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 were previously thought to interact preferentially with different DNA regulatory elements (the AP-1/TRE and ATF/CRE sites, respectively), we find that members of these two families form selective cross-family heterodimers.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Although PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 were previously thought to interact preferentially with different DNA regulatory elements (the AP-1/TRE and ATF/CRE sites, respectively), we find that members of these two families form selective cross-family heterodimers.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Although PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 were previously thought to interact preferentially with different DNA regulatory elements (the AP-1/TRE and ATF/CRE sites, respectively), we find that members of these two families form selective cross-family heterodimers.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These findings indicate that the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 families of transcription factors are not as distinct as was previously thought.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These findings indicate that the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 families of transcription factors are not as distinct as was previously thought.\n",
      "\n",
      "PROTEIN0 PROTEIN3 These findings indicate that the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 families of transcription factors are not as distinct as was previously thought.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These findings indicate that the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 families of transcription factors are not as distinct as was previously thought.\n",
      "\n",
      "PROTEIN1 PROTEIN3 These findings indicate that the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 families of transcription factors are not as distinct as was previously thought.\n",
      "\n",
      "PROTEIN2 PROTEIN3 These findings indicate that the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 families of transcription factors are not as distinct as was previously thought.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Two previously unidentified human PROTEIN0 genes have been isolated, PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Two previously unidentified human PROTEIN0 genes have been isolated, PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Two previously unidentified human PROTEIN0 genes have been isolated, PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Microinjection of PROTEIN0 antibodies into HeLa cells causes their arrest in mitosis. PROTEIN1 and PROTEIN2 display endogenous tyrosine phosphatase activity that is stimulated several-fold, in the absence of cdc2, by stoichiometric addition of either PROTEIN3 or B2 but not A or D1.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Microinjection of PROTEIN0 antibodies into HeLa cells causes their arrest in mitosis. PROTEIN1 and PROTEIN2 display endogenous tyrosine phosphatase activity that is stimulated several-fold, in the absence of cdc2, by stoichiometric addition of either PROTEIN3 or B2 but not A or D1.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Microinjection of PROTEIN0 antibodies into HeLa cells causes their arrest in mitosis. PROTEIN1 and PROTEIN2 display endogenous tyrosine phosphatase activity that is stimulated several-fold, in the absence of cdc2, by stoichiometric addition of either PROTEIN3 or B2 but not A or D1.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Microinjection of PROTEIN0 antibodies into HeLa cells causes their arrest in mitosis. PROTEIN1 and PROTEIN2 display endogenous tyrosine phosphatase activity that is stimulated several-fold, in the absence of cdc2, by stoichiometric addition of either PROTEIN3 or B2 but not A or D1.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Microinjection of PROTEIN0 antibodies into HeLa cells causes their arrest in mitosis. PROTEIN1 and PROTEIN2 display endogenous tyrosine phosphatase activity that is stimulated several-fold, in the absence of cdc2, by stoichiometric addition of either PROTEIN3 or B2 but not A or D1.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Microinjection of PROTEIN0 antibodies into HeLa cells causes their arrest in mitosis. PROTEIN1 and PROTEIN2 display endogenous tyrosine phosphatase activity that is stimulated several-fold, in the absence of cdc2, by stoichiometric addition of either PROTEIN3 or B2 but not A or D1.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Association between PROTEIN0 and PROTEIN1 PROTEIN2 was detected in the HeLa cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Association between PROTEIN0 and PROTEIN1 PROTEIN2 was detected in the HeLa cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Association between PROTEIN0 and PROTEIN1 PROTEIN2 was detected in the HeLa cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These findings indicate that B-type cyclins are multifunctional proteins that not only act as M phase regulatory subunits of the PROTEIN0 protein kinase, but also activate the PROTEIN1 tyrosine phosphatase, of which PROTEIN2 is the physiological substrate.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These findings indicate that B-type cyclins are multifunctional proteins that not only act as M phase regulatory subunits of the PROTEIN0 protein kinase, but also activate the PROTEIN1 tyrosine phosphatase, of which PROTEIN2 is the physiological substrate.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These findings indicate that B-type cyclins are multifunctional proteins that not only act as M phase regulatory subunits of the PROTEIN0 protein kinase, but also activate the PROTEIN1 tyrosine phosphatase, of which PROTEIN2 is the physiological substrate.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The motif may represent an activating domain that has to be provided to PROTEIN0 by intermolecular interaction with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Activation of PROTEIN0 decreases phosphorylation of PROTEIN1 at sites that negatively regulate its DNA-binding activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In resting human epithelial and fibroblastic cells, PROTEIN0 is phosphorylated on serine and threonine at five sites, three of which are phosphorylated in vitro by PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In resting human epithelial and fibroblastic cells, PROTEIN0 is phosphorylated on serine and threonine at five sites, three of which are phosphorylated in vitro by PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In resting human epithelial and fibroblastic cells, PROTEIN0 is phosphorylated on serine and threonine at five sites, three of which are phosphorylated in vitro by PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Activation of PROTEIN0 results in rapid, site-specific dephosphorylation of PROTEIN1 at one or more of these three sites and is coincident with increased PROTEIN2 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Activation of PROTEIN0 results in rapid, site-specific dephosphorylation of PROTEIN1 at one or more of these three sites and is coincident with increased PROTEIN2 activity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Activation of PROTEIN0 results in rapid, site-specific dephosphorylation of PROTEIN1 at one or more of these three sites and is coincident with increased PROTEIN2 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Phosphorylation of recombinant human PROTEIN0 proteins in vitro by PROTEIN1 decreases their DNA-binding activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We propose that PROTEIN0 is present in resting cells in a latent, phosphorylated form that can be activated by site-specific dephosphorylation in response to PROTEIN1 activation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interaction of the regulatory subunit PROTEIN0 of PROTEIN1 with PROTEIN2 proteins occurs through an amphipathic helix binding motif.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Interaction of the regulatory subunit PROTEIN0 of PROTEIN1 with PROTEIN2 proteins occurs through an amphipathic helix binding motif.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Interaction of the regulatory subunit PROTEIN0 of PROTEIN1 with PROTEIN2 proteins occurs through an amphipathic helix binding motif.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 is localized to specific subcellular environments through the binding of the regulatory subunit PROTEIN1 dimer to PROTEIN2 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 is localized to specific subcellular environments through the binding of the regulatory subunit PROTEIN1 dimer to PROTEIN2 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 is localized to specific subcellular environments through the binding of the regulatory subunit PROTEIN1 dimer to PROTEIN2 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Computer-aided analysis of secondary structure, performed on four PROTEIN0 protein sequences (the PROTEIN1 PROTEIN2 and two thyroid proteins PROTEIN3 and PROTEIN4 has identified common regions of approximately 14 residues which display high probabilities of forming amphipathic helices.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Computer-aided analysis of secondary structure, performed on four PROTEIN0 protein sequences (the PROTEIN1 PROTEIN2 and two thyroid proteins PROTEIN3 and PROTEIN4 has identified common regions of approximately 14 residues which display high probabilities of forming amphipathic helices.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Computer-aided analysis of secondary structure, performed on four PROTEIN0 protein sequences (the PROTEIN1 PROTEIN2 and two thyroid proteins PROTEIN3 and PROTEIN4 has identified common regions of approximately 14 residues which display high probabilities of forming amphipathic helices.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Computer-aided analysis of secondary structure, performed on four PROTEIN0 protein sequences (the PROTEIN1 PROTEIN2 and two thyroid proteins PROTEIN3 and PROTEIN4 has identified common regions of approximately 14 residues which display high probabilities of forming amphipathic helices.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Computer-aided analysis of secondary structure, performed on four PROTEIN0 protein sequences (the PROTEIN1 PROTEIN2 and two thyroid proteins PROTEIN3 and PROTEIN4 has identified common regions of approximately 14 residues which display high probabilities of forming amphipathic helices.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Computer-aided analysis of secondary structure, performed on four PROTEIN0 protein sequences (the PROTEIN1 PROTEIN2 and two thyroid proteins PROTEIN3 and PROTEIN4 has identified common regions of approximately 14 residues which display high probabilities of forming amphipathic helices.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Computer-aided analysis of secondary structure, performed on four PROTEIN0 protein sequences (the PROTEIN1 PROTEIN2 and two thyroid proteins PROTEIN3 and PROTEIN4 has identified common regions of approximately 14 residues which display high probabilities of forming amphipathic helices.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Computer-aided analysis of secondary structure, performed on four PROTEIN0 protein sequences (the PROTEIN1 PROTEIN2 and two thyroid proteins PROTEIN3 and PROTEIN4 has identified common regions of approximately 14 residues which display high probabilities of forming amphipathic helices.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Computer-aided analysis of secondary structure, performed on four PROTEIN0 protein sequences (the PROTEIN1 PROTEIN2 and two thyroid proteins PROTEIN3 and PROTEIN4 has identified common regions of approximately 14 residues which display high probabilities of forming amphipathic helices.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Computer-aided analysis of secondary structure, performed on four PROTEIN0 protein sequences (the PROTEIN1 PROTEIN2 and two thyroid proteins PROTEIN3 and PROTEIN4 has identified common regions of approximately 14 residues which display high probabilities of forming amphipathic helices.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The proto-oncogene PROTEIN0 is a component of the PROTEIN1 transcription factor family involved in the mediation of nuclear events elicited by extracellular stimuli.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We present evidence that PROTEIN0 PROTEIN1 and PROTEIN2 specifically phosphorylate these sites and that their phosphorylation positively regulates the transacting activity of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 We present evidence that PROTEIN0 PROTEIN1 and PROTEIN2 specifically phosphorylate these sites and that their phosphorylation positively regulates the transacting activity of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 We present evidence that PROTEIN0 PROTEIN1 and PROTEIN2 specifically phosphorylate these sites and that their phosphorylation positively regulates the transacting activity of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 We present evidence that PROTEIN0 PROTEIN1 and PROTEIN2 specifically phosphorylate these sites and that their phosphorylation positively regulates the transacting activity of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 We present evidence that PROTEIN0 PROTEIN1 and PROTEIN2 specifically phosphorylate these sites and that their phosphorylation positively regulates the transacting activity of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 We present evidence that PROTEIN0 PROTEIN1 and PROTEIN2 specifically phosphorylate these sites and that their phosphorylation positively regulates the transacting activity of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 enzymes PROTEIN1 and PROTEIN2 are regulated by tyrosine as well as serine/threonine phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 enzymes PROTEIN1 and PROTEIN2 are regulated by tyrosine as well as serine/threonine phosphorylation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 enzymes PROTEIN1 and PROTEIN2 are regulated by tyrosine as well as serine/threonine phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 activation of PROTEIN1 may underlie the common stimulation of this transcription factor by mitogens, growth factors and oncogenes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The extracellular cysteine-rich domain of the TNF-R is homologous to the nerve growth factor receptor and the PROTEIN0 PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Unlabeled PROTEIN0 and PROTEIN1 were equally effective at displacing the binding of labeled PROTEIN2 to TNF-R expressing cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Unlabeled PROTEIN0 and PROTEIN1 were equally effective at displacing the binding of labeled PROTEIN2 to TNF-R expressing cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Unlabeled PROTEIN0 and PROTEIN1 were equally effective at displacing the binding of labeled PROTEIN2 to TNF-R expressing cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Two cDNA clones encoding a receptor for human PROTEIN0 PROTEIN1 were isolated by expression screening of a library made from human placental mRNA.\n",
      "\n",
      "PROTEIN0 PROTEIN1 It does not contain a tyrosine kinase domain nor show homology with members of the immunoglobulin super gene family, but does show some significant sequence homologies with receptors for several other haemopoietic growth factors, including those for PROTEIN0 PROTEIN1 and PROTEIN2 (beta-chain) and also to the prolactin receptor.\n",
      "\n",
      "PROTEIN0 PROTEIN2 It does not contain a tyrosine kinase domain nor show homology with members of the immunoglobulin super gene family, but does show some significant sequence homologies with receptors for several other haemopoietic growth factors, including those for PROTEIN0 PROTEIN1 and PROTEIN2 (beta-chain) and also to the prolactin receptor.\n",
      "\n",
      "PROTEIN1 PROTEIN2 It does not contain a tyrosine kinase domain nor show homology with members of the immunoglobulin super gene family, but does show some significant sequence homologies with receptors for several other haemopoietic growth factors, including those for PROTEIN0 PROTEIN1 and PROTEIN2 (beta-chain) and also to the prolactin receptor.\n",
      "\n",
      "PROTEIN0 PROTEIN1 When transfected into COS cells the cloned cDNA directed the expression of a PROTEIN0 showing a single class of affinity (KD = 2(-8) nM) and specificity for human PROTEIN1 but not PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 When transfected into COS cells the cloned cDNA directed the expression of a PROTEIN0 showing a single class of affinity (KD = 2(-8) nM) and specificity for human PROTEIN1 but not PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 When transfected into COS cells the cloned cDNA directed the expression of a PROTEIN0 showing a single class of affinity (KD = 2(-8) nM) and specificity for human PROTEIN1 but not PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The T-cell antigen PROTEIN0 acts as a receptor and substrate for the protein-tyrosine kinase PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a T-cell-specific antigen which binds to the B-cell antigen PROTEIN1 and acts as a coreceptor in the stimulation of T-cell growth.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this study, we demonstrate that PROTEIN0 is coupled to the protein-tyrosine kinase PROTEIN1 and can act as a substrate for PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this study, we demonstrate that PROTEIN0 is coupled to the protein-tyrosine kinase PROTEIN1 and can act as a substrate for PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this study, we demonstrate that PROTEIN0 is coupled to the protein-tyrosine kinase PROTEIN1 and can act as a substrate for PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Coexpression of PROTEIN0 with PROTEIN1 in the baculovirus expression system resulted in the phosphorylation of PROTEIN2 on tyrosine residues.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Coexpression of PROTEIN0 with PROTEIN1 in the baculovirus expression system resulted in the phosphorylation of PROTEIN2 on tyrosine residues.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Coexpression of PROTEIN0 with PROTEIN1 in the baculovirus expression system resulted in the phosphorylation of PROTEIN2 on tyrosine residues.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Further, PROTEIN0 and PROTEIN1 coprecipitated each other in a variety of detergents, including Nonidet P-40 and Triton X-100.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 also precipitated the kinase from various T cells irrespective of the expression of PROTEIN1 PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 also precipitated the kinase from various T cells irrespective of the expression of PROTEIN1 PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 also precipitated the kinase from various T cells irrespective of the expression of PROTEIN1 PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 also precipitated the kinase from various T cells irrespective of the expression of PROTEIN1 PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 also precipitated the kinase from various T cells irrespective of the expression of PROTEIN1 PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 also precipitated the kinase from various T cells irrespective of the expression of PROTEIN1 PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 No binding between PROTEIN0 and PROTEIN1 was detected in T cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The binding of PROTEIN0 to PROTEIN1 induced a 10- to 15-fold increase in PROTEIN2 catalytic activity, as measured by in vitro kinase analysis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The binding of PROTEIN0 to PROTEIN1 induced a 10- to 15-fold increase in PROTEIN2 catalytic activity, as measured by in vitro kinase analysis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The binding of PROTEIN0 to PROTEIN1 induced a 10- to 15-fold increase in PROTEIN2 catalytic activity, as measured by in vitro kinase analysis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In vivo labelling with 32P(i) also showed a four- to fivefold increase in Y-394 occupancy in PROTEIN0 when associated with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The use of glutathione S-transferase-Lck fusion proteins in precipitation analysis showed that the SH2 domain of PROTEIN0 could recognize PROTEIN1 as expressed in the baculovirus expression system.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 interaction PROTEIN1 PROTEIN2 represents a novel variant of a PROTEIN3 in which receptor can also serve as substrate. ( ABSTRACT TRUNCATED AT 250 WORDS )\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 interaction PROTEIN1 PROTEIN2 represents a novel variant of a PROTEIN3 in which receptor can also serve as substrate. ( ABSTRACT TRUNCATED AT 250 WORDS )\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 interaction PROTEIN1 PROTEIN2 represents a novel variant of a PROTEIN3 in which receptor can also serve as substrate. ( ABSTRACT TRUNCATED AT 250 WORDS )\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 interaction PROTEIN1 PROTEIN2 represents a novel variant of a PROTEIN3 in which receptor can also serve as substrate. ( ABSTRACT TRUNCATED AT 250 WORDS )\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 interaction PROTEIN1 PROTEIN2 represents a novel variant of a PROTEIN3 in which receptor can also serve as substrate. ( ABSTRACT TRUNCATED AT 250 WORDS )\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 interaction PROTEIN1 PROTEIN2 represents a novel variant of a PROTEIN3 in which receptor can also serve as substrate. ( ABSTRACT TRUNCATED AT 250 WORDS )\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 a desmosomal component lacking a transmembrane domain, has been implicated in this interaction, although most studies have been conducted with cells that contain few or no desmosomes, and efforts to demonstrate direct interactions between desmoplakin and intermediate filaments have not been successful.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We show that the carboxy terminal \"tail\" of PROTEIN0 associates directly with the amino terminal \"head\" of type II epidermal keratins, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 We show that the carboxy terminal \"tail\" of PROTEIN0 associates directly with the amino terminal \"head\" of type II epidermal keratins, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 We show that the carboxy terminal \"tail\" of PROTEIN0 associates directly with the amino terminal \"head\" of type II epidermal keratins, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 We show that the carboxy terminal \"tail\" of PROTEIN0 associates directly with the amino terminal \"head\" of type II epidermal keratins, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 We show that the carboxy terminal \"tail\" of PROTEIN0 associates directly with the amino terminal \"head\" of type II epidermal keratins, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 We show that the carboxy terminal \"tail\" of PROTEIN0 associates directly with the amino terminal \"head\" of type II epidermal keratins, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 We show that the carboxy terminal \"tail\" of PROTEIN0 associates directly with the amino terminal \"head\" of type II epidermal keratins, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 We show that the carboxy terminal \"tail\" of PROTEIN0 associates directly with the amino terminal \"head\" of type II epidermal keratins, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 We show that the carboxy terminal \"tail\" of PROTEIN0 associates directly with the amino terminal \"head\" of type II epidermal keratins, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 We show that the carboxy terminal \"tail\" of PROTEIN0 associates directly with the amino terminal \"head\" of type II epidermal keratins, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have engineered and purified recombinant PROTEIN0 head and PROTEIN1 tail, and we demonstrate direct interaction in vitro by solution-binding assays and by ligand blot assays.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have identified an 18-amino acid residue stretch in the PROTEIN0 head that is conserved only among type II epidermal keratins and that appears to play some role in PROTEIN1 tail binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 protein binds to PROTEIN1 a zinc-finger protein that shares an epitope with the adenovirus PROTEIN2 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 protein binds to PROTEIN1 a zinc-finger protein that shares an epitope with the adenovirus PROTEIN2 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 protein binds to PROTEIN1 a zinc-finger protein that shares an epitope with the adenovirus PROTEIN2 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 is a target of viral oncoproteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 It contains an PROTEIN0 motif that shares an antigenic epitope with the C terminus of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 A domain is conserved between PROTEIN0 and the PROTEIN1 PROTEIN2 differentiation factor.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A domain is conserved between PROTEIN0 and the PROTEIN1 PROTEIN2 differentiation factor.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A domain is conserved between PROTEIN0 and the PROTEIN1 PROTEIN2 differentiation factor.\n",
      "\n",
      "PROTEIN0 PROTEIN1 While the function of PROTEIN0 is not yet clear, its structure and pattern of expression suggest a role for PROTEIN1 in transcriptional regulation during neuronal differentiation and pathogenesis of retinoblastoma.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 a novel death domain-containing protein, interacts with the death domain of PROTEIN1 and initiates apoptosis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN3 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN4 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN5 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN6 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN7 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN8 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN9 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN3 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN4 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN5 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN6 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN7 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN8 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN9 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN3 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN4 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN5 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN6 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN7 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN8 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN9 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN4 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN5 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN6 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN7 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN8 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN9 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN5 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN6 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN7 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN8 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN9 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN5 PROTEIN6 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN5 PROTEIN7 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN5 PROTEIN8 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN5 PROTEIN9 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN6 PROTEIN7 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN6 PROTEIN8 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN6 PROTEIN9 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN7 PROTEIN8 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN7 PROTEIN9 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN8 PROTEIN9 Using the cytoplasmic domain of PROTEIN0 in the yeast two-hybrid system, we have identified a novel interacting protein, PROTEIN1 which binds PROTEIN2 and PROTEIN3 PROTEIN4 a mutant of PROTEIN5 possessing enhanced killing activity, but not the functionally inactive mutants PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 contains a death domain homologous to the death domains of PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 contains a death domain homologous to the death domains of PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 contains a death domain homologous to the death domains of PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 A point mutation in PROTEIN0 analogous to the lpr mutation of PROTEIN1 abolishes its ability to bind PROTEIN2 suggesting a death domain to death domain interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A point mutation in PROTEIN0 analogous to the lpr mutation of PROTEIN1 abolishes its ability to bind PROTEIN2 suggesting a death domain to death domain interaction.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A point mutation in PROTEIN0 analogous to the lpr mutation of PROTEIN1 abolishes its ability to bind PROTEIN2 suggesting a death domain to death domain interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These findings suggest that PROTEIN0 may play an important role in the proximal signal transduction of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 a novel protein containing a death domain that interacts with PROTEIN1 PROTEIN2 PROTEIN3 in yeast and causes cell death.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 a novel protein containing a death domain that interacts with PROTEIN1 PROTEIN2 PROTEIN3 in yeast and causes cell death.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 a novel protein containing a death domain that interacts with PROTEIN1 PROTEIN2 PROTEIN3 in yeast and causes cell death.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 a novel protein containing a death domain that interacts with PROTEIN1 PROTEIN2 PROTEIN3 in yeast and causes cell death.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 a novel protein containing a death domain that interacts with PROTEIN1 PROTEIN2 PROTEIN3 in yeast and causes cell death.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 a novel protein containing a death domain that interacts with PROTEIN1 PROTEIN2 PROTEIN3 in yeast and causes cell death.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Ligation of the extracellular domain of the cell surface receptor PROTEIN0 PROTEIN1 PROTEIN2 elicits a characteristic programmed death response in susceptible cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Ligation of the extracellular domain of the cell surface receptor PROTEIN0 PROTEIN1 PROTEIN2 elicits a characteristic programmed death response in susceptible cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Ligation of the extracellular domain of the cell surface receptor PROTEIN0 PROTEIN1 PROTEIN2 elicits a characteristic programmed death response in susceptible cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using a genetic selection based on protein-protein interaction in yeast, we have identified two gene products that associate with the intracellular domain of PROTEIN0 PROTEIN1 itself, and a novel 74 kDa protein we have named PROTEIN2 for PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using a genetic selection based on protein-protein interaction in yeast, we have identified two gene products that associate with the intracellular domain of PROTEIN0 PROTEIN1 itself, and a novel 74 kDa protein we have named PROTEIN2 for PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Using a genetic selection based on protein-protein interaction in yeast, we have identified two gene products that associate with the intracellular domain of PROTEIN0 PROTEIN1 itself, and a novel 74 kDa protein we have named PROTEIN2 for PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using a genetic selection based on protein-protein interaction in yeast, we have identified two gene products that associate with the intracellular domain of PROTEIN0 PROTEIN1 itself, and a novel 74 kDa protein we have named PROTEIN2 for PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Using a genetic selection based on protein-protein interaction in yeast, we have identified two gene products that associate with the intracellular domain of PROTEIN0 PROTEIN1 itself, and a novel 74 kDa protein we have named PROTEIN2 for PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Using a genetic selection based on protein-protein interaction in yeast, we have identified two gene products that associate with the intracellular domain of PROTEIN0 PROTEIN1 itself, and a novel 74 kDa protein we have named PROTEIN2 for PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 also interacts weakly with the PROTEIN1 tumor necrosis factor receptor PROTEIN2 intracellular domain, but not with a mutant version of PROTEIN3 corresponding to the murine lprcg mutation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 also interacts weakly with the PROTEIN1 tumor necrosis factor receptor PROTEIN2 intracellular domain, but not with a mutant version of PROTEIN3 corresponding to the murine lprcg mutation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 also interacts weakly with the PROTEIN1 tumor necrosis factor receptor PROTEIN2 intracellular domain, but not with a mutant version of PROTEIN3 corresponding to the murine lprcg mutation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 also interacts weakly with the PROTEIN1 tumor necrosis factor receptor PROTEIN2 intracellular domain, but not with a mutant version of PROTEIN3 corresponding to the murine lprcg mutation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 also interacts weakly with the PROTEIN1 tumor necrosis factor receptor PROTEIN2 intracellular domain, but not with a mutant version of PROTEIN3 corresponding to the murine lprcg mutation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 also interacts weakly with the PROTEIN1 tumor necrosis factor receptor PROTEIN2 intracellular domain, but not with a mutant version of PROTEIN3 corresponding to the murine lprcg mutation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 contains an N-terminal region with homology to protein kinases and a C-terminal region containing a cytoplasmic motif (death domain) present in the PROTEIN1 and PROTEIN2 intracellular domains.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 contains an N-terminal region with homology to protein kinases and a C-terminal region containing a cytoplasmic motif (death domain) present in the PROTEIN1 and PROTEIN2 intracellular domains.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 contains an N-terminal region with homology to protein kinases and a C-terminal region containing a cytoplasmic motif (death domain) present in the PROTEIN1 and PROTEIN2 intracellular domains.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Identification of an 11-kDa PROTEIN0 domain within the 289-kDa PROTEIN1 and characterization of a critical serine residue.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using an in vitro transcription/translation assay method coupled with proteolysis studies, we have identified an 11-kDa PROTEIN0 domain within PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 By rational mutagenesis, receptor-specific functional analysis, and visualization of complex formation in solution, we identified individual amino acid side chains involved specifically in the interaction of PROTEIN0 PROTEIN1 with PROTEIN2 and not with the beta-components, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 By rational mutagenesis, receptor-specific functional analysis, and visualization of complex formation in solution, we identified individual amino acid side chains involved specifically in the interaction of PROTEIN0 PROTEIN1 with PROTEIN2 and not with the beta-components, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 By rational mutagenesis, receptor-specific functional analysis, and visualization of complex formation in solution, we identified individual amino acid side chains involved specifically in the interaction of PROTEIN0 PROTEIN1 with PROTEIN2 and not with the beta-components, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 By rational mutagenesis, receptor-specific functional analysis, and visualization of complex formation in solution, we identified individual amino acid side chains involved specifically in the interaction of PROTEIN0 PROTEIN1 with PROTEIN2 and not with the beta-components, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 By rational mutagenesis, receptor-specific functional analysis, and visualization of complex formation in solution, we identified individual amino acid side chains involved specifically in the interaction of PROTEIN0 PROTEIN1 with PROTEIN2 and not with the beta-components, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 By rational mutagenesis, receptor-specific functional analysis, and visualization of complex formation in solution, we identified individual amino acid side chains involved specifically in the interaction of PROTEIN0 PROTEIN1 with PROTEIN2 and not with the beta-components, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 By rational mutagenesis, receptor-specific functional analysis, and visualization of complex formation in solution, we identified individual amino acid side chains involved specifically in the interaction of PROTEIN0 PROTEIN1 with PROTEIN2 and not with the beta-components, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 By rational mutagenesis, receptor-specific functional analysis, and visualization of complex formation in solution, we identified individual amino acid side chains involved specifically in the interaction of PROTEIN0 PROTEIN1 with PROTEIN2 and not with the beta-components, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 By rational mutagenesis, receptor-specific functional analysis, and visualization of complex formation in solution, we identified individual amino acid side chains involved specifically in the interaction of PROTEIN0 PROTEIN1 with PROTEIN2 and not with the beta-components, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 By rational mutagenesis, receptor-specific functional analysis, and visualization of complex formation in solution, we identified individual amino acid side chains involved specifically in the interaction of PROTEIN0 PROTEIN1 with PROTEIN2 and not with the beta-components, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 Superposition of the PROTEIN0 and growth hormone structures showed that the location of these epitopes on PROTEIN1 is analogous to the location of the first and second receptor epitopes on the surface of growth hormone.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 physically associates with PROTEIN1 through Src homology 3 domains.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the second of a series of experiments designed to identify PROTEIN0 homology 3 (SH3)-binding molecules, we report the cloning of PROTEIN1 PROTEIN2 from an HL60 cDNA expression library.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the second of a series of experiments designed to identify PROTEIN0 homology 3 (SH3)-binding molecules, we report the cloning of PROTEIN1 PROTEIN2 from an HL60 cDNA expression library.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the second of a series of experiments designed to identify PROTEIN0 homology 3 (SH3)-binding molecules, we report the cloning of PROTEIN1 PROTEIN2 from an HL60 cDNA expression library.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Studies in vivo and in vitro demonstrated a highly specific interaction between the SH3 domains of PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In vitro translation studies identified the 66-kDa protein as the protein product of PROTEIN0 and subcellular fractionation experiments showed that PROTEIN1 is mainly present in the cytosol fraction, although significant amounts are also present in membrane and nuclear fractions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The main PROTEIN0 region implicated in the association with PROTEIN1 was found to be the carboxy-terminal SH3 domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interaction of PROTEIN0 subunit of PROTEIN1 with PROTEIN2 and PROTEIN3 receptors analysed by using the two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Interaction of PROTEIN0 subunit of PROTEIN1 with PROTEIN2 and PROTEIN3 receptors analysed by using the two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Interaction of PROTEIN0 subunit of PROTEIN1 with PROTEIN2 and PROTEIN3 receptors analysed by using the two-hybrid system.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Interaction of PROTEIN0 subunit of PROTEIN1 with PROTEIN2 and PROTEIN3 receptors analysed by using the two-hybrid system.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Interaction of PROTEIN0 subunit of PROTEIN1 with PROTEIN2 and PROTEIN3 receptors analysed by using the two-hybrid system.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Interaction of PROTEIN0 subunit of PROTEIN1 with PROTEIN2 and PROTEIN3 receptors analysed by using the two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The experiments were performed with the cytoplasmic beta domain (wild type or mutated) of PROTEIN0 and PROTEIN1 and PROTEIN2 or its subdomains (N + C-SH2, N-SH2, C-SH2, SH3 + N-SH2).\n",
      "\n",
      "PROTEIN0 PROTEIN2 The experiments were performed with the cytoplasmic beta domain (wild type or mutated) of PROTEIN0 and PROTEIN1 and PROTEIN2 or its subdomains (N + C-SH2, N-SH2, C-SH2, SH3 + N-SH2).\n",
      "\n",
      "PROTEIN1 PROTEIN2 The experiments were performed with the cytoplasmic beta domain (wild type or mutated) of PROTEIN0 and PROTEIN1 and PROTEIN2 or its subdomains (N + C-SH2, N-SH2, C-SH2, SH3 + N-SH2).\n",
      "\n",
      "PROTEIN0 PROTEIN1 The internal binding site thus revealed was not altered by further mutating Y960/F for PROTEIN0 or Y950/F for PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Activation of lacZ upon interaction of PROTEIN0 with PROTEIN1 PROTEIN2 was 4-fold less as compared to PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Activation of lacZ upon interaction of PROTEIN0 with PROTEIN1 PROTEIN2 was 4-fold less as compared to PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Activation of lacZ upon interaction of PROTEIN0 with PROTEIN1 PROTEIN2 was 4-fold less as compared to PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Activation of lacZ upon interaction of PROTEIN0 with PROTEIN1 PROTEIN2 was 4-fold less as compared to PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Activation of lacZ upon interaction of PROTEIN0 with PROTEIN1 PROTEIN2 was 4-fold less as compared to PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Activation of lacZ upon interaction of PROTEIN0 with PROTEIN1 PROTEIN2 was 4-fold less as compared to PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, the C-terminal domain in PROTEIN0 appears to exert a negative control on binding of PROTEIN1 thereby providing a possible regulatory mechanism for direct activation of the PROTEIN2 pathway.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, the C-terminal domain in PROTEIN0 appears to exert a negative control on binding of PROTEIN1 thereby providing a possible regulatory mechanism for direct activation of the PROTEIN2 pathway.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, the C-terminal domain in PROTEIN0 appears to exert a negative control on binding of PROTEIN1 thereby providing a possible regulatory mechanism for direct activation of the PROTEIN2 pathway.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The mitogenic effects of the PROTEIN0 are mediated through cell surface PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoprecipitation experiments employing a polyclonal PROTEIN0 antibody provided further evidence that the 60-kDa protein isolated in this study was PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Further, polyclonal antibodies to human PROTEIN0 significantly inhibited the mitogenic activity of PROTEIN1 on human fibroblasts.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The present study has shown that cell surface PROTEIN0 binds to the PROTEIN1 mediating its mitogenic activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Multiple proteins physically interact with PROTEIN0 Transcriptional synergy with PROTEIN1 PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Multiple proteins physically interact with PROTEIN0 Transcriptional synergy with PROTEIN1 PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Multiple proteins physically interact with PROTEIN0 Transcriptional synergy with PROTEIN1 PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Multiple proteins physically interact with PROTEIN0 Transcriptional synergy with PROTEIN1 PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Multiple proteins physically interact with PROTEIN0 Transcriptional synergy with PROTEIN1 PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Multiple proteins physically interact with PROTEIN0 Transcriptional synergy with PROTEIN1 PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using radiolabeled PROTEIN0 protein as a probe, we screened a B cell cDNA expression library and isolated a number of clones encoding PROTEIN1 interacting proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Three of these clones encode DNA binding proteins PROTEIN0 PROTEIN1 and PROTEIN2 one clone encodes a chaperone protein, and another clone encodes a multifunctional phosphatase.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Three of these clones encode DNA binding proteins PROTEIN0 PROTEIN1 and PROTEIN2 one clone encodes a chaperone protein, and another clone encodes a multifunctional phosphatase.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Three of these clones encode DNA binding proteins PROTEIN0 PROTEIN1 and PROTEIN2 one clone encodes a chaperone protein, and another clone encodes a multifunctional phosphatase.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have characterized the physical and functional interactions between PROTEIN0 and PROTEIN1 a leucine zipper transcription factor implicated in inflammatory responses.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We found that deletion of the carboxyl-terminal 28 amino acids of PROTEIN0 disrupted PROTEIN1 PROTEIN2 physical interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We found that deletion of the carboxyl-terminal 28 amino acids of PROTEIN0 disrupted PROTEIN1 PROTEIN2 physical interaction.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We found that deletion of the carboxyl-terminal 28 amino acids of PROTEIN0 disrupted PROTEIN1 PROTEIN2 physical interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In transient expression assays, we found that PROTEIN0 and PROTEIN1 can functionally cooperate to synergistically activate transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Electrophoretic mobility shift assays showed that PROTEIN0 and PROTEIN1 can simultaneously bind to adjacent DNA binding sites, but apparently do not influence the kinetics or affinity of each other's DNA binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Molecular cloning and expression of the 32-kDa subunit of human PROTEIN0 reveals interactions with PROTEIN1 and PROTEIN2 that mediate transcriptional activation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Molecular cloning and expression of the 32-kDa subunit of human PROTEIN0 reveals interactions with PROTEIN1 and PROTEIN2 that mediate transcriptional activation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Molecular cloning and expression of the 32-kDa subunit of human PROTEIN0 reveals interactions with PROTEIN1 and PROTEIN2 that mediate transcriptional activation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Transcription factor PROTEIN0 consists of PROTEIN1 PROTEIN2 and at least eight PROTEIN3 factors (TAFs).\n",
      "\n",
      "PROTEIN0 PROTEIN2 Transcription factor PROTEIN0 consists of PROTEIN1 PROTEIN2 and at least eight PROTEIN3 factors (TAFs).\n",
      "\n",
      "PROTEIN0 PROTEIN3 Transcription factor PROTEIN0 consists of PROTEIN1 PROTEIN2 and at least eight PROTEIN3 factors (TAFs).\n",
      "\n",
      "PROTEIN1 PROTEIN2 Transcription factor PROTEIN0 consists of PROTEIN1 PROTEIN2 and at least eight PROTEIN3 factors (TAFs).\n",
      "\n",
      "PROTEIN1 PROTEIN3 Transcription factor PROTEIN0 consists of PROTEIN1 PROTEIN2 and at least eight PROTEIN3 factors (TAFs).\n",
      "\n",
      "PROTEIN2 PROTEIN3 Transcription factor PROTEIN0 consists of PROTEIN1 PROTEIN2 and at least eight PROTEIN3 factors (TAFs).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we report the cloning, expression, and biochemical characterization of the 32-kDa subunit of human (h) PROTEIN0 termed PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 We find that PROTEIN0 is the human homologue of Drosophila PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In vitro protein-protein interaction assays reveal that as observed with Drosophila PROTEIN0 PROTEIN1 interacts with the C-terminal 39-amino acid activation domain of the acidic transactivator PROTEIN2 PROTEIN3 as well as with the general transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 In vitro protein-protein interaction assays reveal that as observed with Drosophila PROTEIN0 PROTEIN1 interacts with the C-terminal 39-amino acid activation domain of the acidic transactivator PROTEIN2 PROTEIN3 as well as with the general transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 In vitro protein-protein interaction assays reveal that as observed with Drosophila PROTEIN0 PROTEIN1 interacts with the C-terminal 39-amino acid activation domain of the acidic transactivator PROTEIN2 PROTEIN3 as well as with the general transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 In vitro protein-protein interaction assays reveal that as observed with Drosophila PROTEIN0 PROTEIN1 interacts with the C-terminal 39-amino acid activation domain of the acidic transactivator PROTEIN2 PROTEIN3 as well as with the general transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 In vitro protein-protein interaction assays reveal that as observed with Drosophila PROTEIN0 PROTEIN1 interacts with the C-terminal 39-amino acid activation domain of the acidic transactivator PROTEIN2 PROTEIN3 as well as with the general transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 In vitro protein-protein interaction assays reveal that as observed with Drosophila PROTEIN0 PROTEIN1 interacts with the C-terminal 39-amino acid activation domain of the acidic transactivator PROTEIN2 PROTEIN3 as well as with the general transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 In vitro protein-protein interaction assays reveal that as observed with Drosophila PROTEIN0 PROTEIN1 interacts with the C-terminal 39-amino acid activation domain of the acidic transactivator PROTEIN2 PROTEIN3 as well as with the general transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 In vitro protein-protein interaction assays reveal that as observed with Drosophila PROTEIN0 PROTEIN1 interacts with the C-terminal 39-amino acid activation domain of the acidic transactivator PROTEIN2 PROTEIN3 as well as with the general transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 In vitro protein-protein interaction assays reveal that as observed with Drosophila PROTEIN0 PROTEIN1 interacts with the C-terminal 39-amino acid activation domain of the acidic transactivator PROTEIN2 PROTEIN3 as well as with the general transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 In vitro protein-protein interaction assays reveal that as observed with Drosophila PROTEIN0 PROTEIN1 interacts with the C-terminal 39-amino acid activation domain of the acidic transactivator PROTEIN2 PROTEIN3 as well as with the general transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover, a partial recombinant PROTEIN0 complex containing PROTEIN1 was capable of mediating in vitro transcriptional activation by the PROTEIN2 activation domain.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Moreover, a partial recombinant PROTEIN0 complex containing PROTEIN1 was capable of mediating in vitro transcriptional activation by the PROTEIN2 activation domain.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Moreover, a partial recombinant PROTEIN0 complex containing PROTEIN1 was capable of mediating in vitro transcriptional activation by the PROTEIN2 activation domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A potential interaction of PROTEIN0 and PROTEIN1 PROTEIN2 receptors revealed by affinity crosslinking and immunoprecipitation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A potential interaction of PROTEIN0 and PROTEIN1 PROTEIN2 receptors revealed by affinity crosslinking and immunoprecipitation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A potential interaction of PROTEIN0 and PROTEIN1 PROTEIN2 receptors revealed by affinity crosslinking and immunoprecipitation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we have utilized affinity crosslinking agents with 125I-NGF to detect an association of PROTEIN0 and PROTEIN1 receptors in embryonic spinal cord and brain tissues enriched in the two receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our results indicate that the PROTEIN0 and PROTEIN1 receptors can potentially interact, and that such an association may be responsible for the generation of high affinity PROTEIN2 binding sites.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Our results indicate that the PROTEIN0 and PROTEIN1 receptors can potentially interact, and that such an association may be responsible for the generation of high affinity PROTEIN2 binding sites.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Our results indicate that the PROTEIN0 and PROTEIN1 receptors can potentially interact, and that such an association may be responsible for the generation of high affinity PROTEIN2 binding sites.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Eosinophils undergo chemotaxis, degranulate, and exhibit [C2+]i changes in response to the human CC chemokines macrophage inflammatory protein PROTEIN0 regulated on activation, normal T expressed and secreted PROTEIN1 and PROTEIN2 PROTEIN3 but the receptors involved have not been defined.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Eosinophils undergo chemotaxis, degranulate, and exhibit [C2+]i changes in response to the human CC chemokines macrophage inflammatory protein PROTEIN0 regulated on activation, normal T expressed and secreted PROTEIN1 and PROTEIN2 PROTEIN3 but the receptors involved have not been defined.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Eosinophils undergo chemotaxis, degranulate, and exhibit [C2+]i changes in response to the human CC chemokines macrophage inflammatory protein PROTEIN0 regulated on activation, normal T expressed and secreted PROTEIN1 and PROTEIN2 PROTEIN3 but the receptors involved have not been defined.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Eosinophils undergo chemotaxis, degranulate, and exhibit [C2+]i changes in response to the human CC chemokines macrophage inflammatory protein PROTEIN0 regulated on activation, normal T expressed and secreted PROTEIN1 and PROTEIN2 PROTEIN3 but the receptors involved have not been defined.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Eosinophils undergo chemotaxis, degranulate, and exhibit [C2+]i changes in response to the human CC chemokines macrophage inflammatory protein PROTEIN0 regulated on activation, normal T expressed and secreted PROTEIN1 and PROTEIN2 PROTEIN3 but the receptors involved have not been defined.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Eosinophils undergo chemotaxis, degranulate, and exhibit [C2+]i changes in response to the human CC chemokines macrophage inflammatory protein PROTEIN0 regulated on activation, normal T expressed and secreted PROTEIN1 and PROTEIN2 PROTEIN3 but the receptors involved have not been defined.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have isolated a human cDNA encoding the first eosinophil-selective chemokine receptor, designated PROTEIN0 PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a seven-transmembrane domain G protein-coupled receptor most closely related to the previously reported monocyte- and neutrophil-selective receptor PROTEIN1 (also known as the PROTEIN2 PROTEIN3 receptor).\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is a seven-transmembrane domain G protein-coupled receptor most closely related to the previously reported monocyte- and neutrophil-selective receptor PROTEIN1 (also known as the PROTEIN2 PROTEIN3 receptor).\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 is a seven-transmembrane domain G protein-coupled receptor most closely related to the previously reported monocyte- and neutrophil-selective receptor PROTEIN1 (also known as the PROTEIN2 PROTEIN3 receptor).\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is a seven-transmembrane domain G protein-coupled receptor most closely related to the previously reported monocyte- and neutrophil-selective receptor PROTEIN1 (also known as the PROTEIN2 PROTEIN3 receptor).\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 is a seven-transmembrane domain G protein-coupled receptor most closely related to the previously reported monocyte- and neutrophil-selective receptor PROTEIN1 (also known as the PROTEIN2 PROTEIN3 receptor).\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 is a seven-transmembrane domain G protein-coupled receptor most closely related to the previously reported monocyte- and neutrophil-selective receptor PROTEIN1 (also known as the PROTEIN2 PROTEIN3 receptor).\n",
      "\n",
      "PROTEIN0 PROTEIN1 When [Ca2+]i changes were monitored in stably transfected human embryonic kidney 293 cells, PROTEIN0 PROTEIN1 and PROTEIN2 were both potent agonists for PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 When [Ca2+]i changes were monitored in stably transfected human embryonic kidney 293 cells, PROTEIN0 PROTEIN1 and PROTEIN2 were both potent agonists for PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 When [Ca2+]i changes were monitored in stably transfected human embryonic kidney 293 cells, PROTEIN0 PROTEIN1 and PROTEIN2 were both potent agonists for PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 When [Ca2+]i changes were monitored in stably transfected human embryonic kidney 293 cells, PROTEIN0 PROTEIN1 and PROTEIN2 were both potent agonists for PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 When [Ca2+]i changes were monitored in stably transfected human embryonic kidney 293 cells, PROTEIN0 PROTEIN1 and PROTEIN2 were both potent agonists for PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 When [Ca2+]i changes were monitored in stably transfected human embryonic kidney 293 cells, PROTEIN0 PROTEIN1 and PROTEIN2 were both potent agonists for PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 When [Ca2+]i changes were monitored in stably transfected human embryonic kidney 293 cells, PROTEIN0 PROTEIN1 and PROTEIN2 were both potent agonists for PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 When [Ca2+]i changes were monitored in stably transfected human embryonic kidney 293 cells, PROTEIN0 PROTEIN1 and PROTEIN2 were both potent agonists for PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 When [Ca2+]i changes were monitored in stably transfected human embryonic kidney 293 cells, PROTEIN0 PROTEIN1 and PROTEIN2 were both potent agonists for PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 When [Ca2+]i changes were monitored in stably transfected human embryonic kidney 293 cells, PROTEIN0 PROTEIN1 and PROTEIN2 were both potent agonists for PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 However, PROTEIN0 was also an agonist for PROTEIN1 but not PROTEIN2 PROTEIN3 was an agonist for PROTEIN4 but not PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mechanism of CDK activation revealed by the structure of a PROTEIN0 PROTEIN1 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The crystal structure of the human PROTEIN0 PROTEIN1 PROTEIN2 complex has been determined at 2.3 A resolution.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The crystal structure of the human PROTEIN0 PROTEIN1 PROTEIN2 complex has been determined at 2.3 A resolution.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The crystal structure of the human PROTEIN0 PROTEIN1 PROTEIN2 complex has been determined at 2.3 A resolution.\n",
      "\n",
      "PROTEIN0 PROTEIN1 It binds homophilically, i.e., PROTEIN0 on the surface of one cell binds to PROTEIN1 on an apposing cell, and the binding site lies within the immunoglobulin domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this article, we demonstrate that PROTEIN0 associates with a complex containing PROTEIN1 and PROTEIN2 in mink lung (MvLu) cells, and in rat heart, lung, and brain tissues.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this article, we demonstrate that PROTEIN0 associates with a complex containing PROTEIN1 and PROTEIN2 in mink lung (MvLu) cells, and in rat heart, lung, and brain tissues.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this article, we demonstrate that PROTEIN0 associates with a complex containing PROTEIN1 and PROTEIN2 in mink lung (MvLu) cells, and in rat heart, lung, and brain tissues.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In vitro binding studies demonstrated that the intracellular segment of PROTEIN0 binds directly to the intracellular domain of PROTEIN1 but not to alpha- or PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In vitro binding studies demonstrated that the intracellular segment of PROTEIN0 binds directly to the intracellular domain of PROTEIN1 but not to alpha- or PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In vitro binding studies demonstrated that the intracellular segment of PROTEIN0 binds directly to the intracellular domain of PROTEIN1 but not to alpha- or PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 After pervanadate treatment of MvLu cells, which inhibits cellular tyrosine phosphatase activity including PROTEIN0 the cadherins associated with PROTEIN1 are now found in a tyrosine-phosphorylated form, indicating that the cadherins may be an endogenous substrate for PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 After pervanadate treatment of MvLu cells, which inhibits cellular tyrosine phosphatase activity including PROTEIN0 the cadherins associated with PROTEIN1 are now found in a tyrosine-phosphorylated form, indicating that the cadherins may be an endogenous substrate for PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 After pervanadate treatment of MvLu cells, which inhibits cellular tyrosine phosphatase activity including PROTEIN0 the cadherins associated with PROTEIN1 are now found in a tyrosine-phosphorylated form, indicating that the cadherins may be an endogenous substrate for PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we report the cDNA cloning and characterization of a human type II receptor for BMPs PROTEIN0 which is distantly related to PROTEIN1 a PROTEIN2 from Caenorhabditis elegans.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we report the cDNA cloning and characterization of a human type II receptor for BMPs PROTEIN0 which is distantly related to PROTEIN1 a PROTEIN2 from Caenorhabditis elegans.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we report the cDNA cloning and characterization of a human type II receptor for BMPs PROTEIN0 which is distantly related to PROTEIN1 a PROTEIN2 from Caenorhabditis elegans.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In transfected COS-1 cells, PROTEIN0 PROTEIN1 and less efficiently PROTEIN2 bound to PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 In transfected COS-1 cells, PROTEIN0 PROTEIN1 and less efficiently PROTEIN2 bound to PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 In transfected COS-1 cells, PROTEIN0 PROTEIN1 and less efficiently PROTEIN2 bound to PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 In transfected COS-1 cells, PROTEIN0 PROTEIN1 and less efficiently PROTEIN2 bound to PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 In transfected COS-1 cells, PROTEIN0 PROTEIN1 and less efficiently PROTEIN2 bound to PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 In transfected COS-1 cells, PROTEIN0 PROTEIN1 and less efficiently PROTEIN2 bound to PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Binding of PROTEIN0 PROTEIN1 to PROTEIN2 was also observed in nontransfected cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Binding of PROTEIN0 PROTEIN1 to PROTEIN2 was also observed in nontransfected cell lines.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Binding of PROTEIN0 PROTEIN1 to PROTEIN2 was also observed in nontransfected cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover, a transcriptional activation signal was transduced by PROTEIN0 in the presence of type I receptors after stimulation by PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Moreover, a transcriptional activation signal was transduced by PROTEIN0 in the presence of type I receptors after stimulation by PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Moreover, a transcriptional activation signal was transduced by PROTEIN0 in the presence of type I receptors after stimulation by PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 which is known to bind 14-3-3 proteins, has recently been shown to associate with PROTEIN1 and to stimulate its phosphatase activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 14-3-3 protein, however, has no effect on the PROTEIN0 activity of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Instead, 14-3-3 may facilitate the association of PROTEIN0 with PROTEIN1 in vivo, participating in the linkage between mitogenic signaling and the cell cycle machinery.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The cloned PROTEIN0 gene encodes a member of a protein family that includes the yeast PROTEIN1 protein, a component of the transcriptional regulator PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The cloned PROTEIN0 gene encodes a member of a protein family that includes the yeast PROTEIN1 protein, a component of the transcriptional regulator PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The cloned PROTEIN0 gene encodes a member of a protein family that includes the yeast PROTEIN1 protein, a component of the transcriptional regulator PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The cloned PROTEIN0 gene encodes a member of a protein family that includes the yeast PROTEIN1 protein, a component of the transcriptional regulator PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The cloned PROTEIN0 gene encodes a member of a protein family that includes the yeast PROTEIN1 protein, a component of the transcriptional regulator PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The cloned PROTEIN0 gene encodes a member of a protein family that includes the yeast PROTEIN1 protein, a component of the transcriptional regulator PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 protein interacts with PROTEIN1 protein and with PROTEIN2 protein, a subunit of the human RNA polymerase II transcription factor IIH.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 protein interacts with PROTEIN1 protein and with PROTEIN2 protein, a subunit of the human RNA polymerase II transcription factor IIH.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 protein interacts with PROTEIN1 protein and with PROTEIN2 protein, a subunit of the human RNA polymerase II transcription factor IIH.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These observations suggest that the products of the PROTEIN0 and PROTEIN1 genes are involved in transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Biochemical studies established that some coactivators are associated with the PROTEIN0 PROTEIN1 to form the PROTEIN2 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Biochemical studies established that some coactivators are associated with the PROTEIN0 PROTEIN1 to form the PROTEIN2 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Biochemical studies established that some coactivators are associated with the PROTEIN0 PROTEIN1 to form the PROTEIN2 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 gene is identical to a gene, PROTEIN1 (ref 7,8), implicated in cell-cycle progression.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Recombinant PROTEIN0 binds directly to PROTEIN1 both in vitro and in yeast, and participates in the formation of the PROTEIN2 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Recombinant PROTEIN0 binds directly to PROTEIN1 both in vitro and in yeast, and participates in the formation of the PROTEIN2 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Recombinant PROTEIN0 binds directly to PROTEIN1 both in vitro and in yeast, and participates in the formation of the PROTEIN2 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This largest TAF may therefore play a central role in PROTEIN0 assembly by interacting with both PROTEIN1 and other TAFs, as well as serving to link the control of transcription to the cell cycle.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The bZIP domains of PROTEIN0 and PROTEIN1 mediate a physical association with the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The bZIP domains of PROTEIN0 and PROTEIN1 mediate a physical association with the PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The bZIP domains of PROTEIN0 and PROTEIN1 mediate a physical association with the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 oncoproteins form a complex that regulates transcription from promoters containing AP-1 binding sites.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The ability of PROTEIN0 and PROTEIN1 proteins to interact directly with the PROTEIN2 PROTEIN3 the general transcription factor required for initiating the assembly of transcription complexes, was investigated.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The ability of PROTEIN0 and PROTEIN1 proteins to interact directly with the PROTEIN2 PROTEIN3 the general transcription factor required for initiating the assembly of transcription complexes, was investigated.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The ability of PROTEIN0 and PROTEIN1 proteins to interact directly with the PROTEIN2 PROTEIN3 the general transcription factor required for initiating the assembly of transcription complexes, was investigated.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The ability of PROTEIN0 and PROTEIN1 proteins to interact directly with the PROTEIN2 PROTEIN3 the general transcription factor required for initiating the assembly of transcription complexes, was investigated.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The ability of PROTEIN0 and PROTEIN1 proteins to interact directly with the PROTEIN2 PROTEIN3 the general transcription factor required for initiating the assembly of transcription complexes, was investigated.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The ability of PROTEIN0 and PROTEIN1 proteins to interact directly with the PROTEIN2 PROTEIN3 the general transcription factor required for initiating the assembly of transcription complexes, was investigated.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 Using co-immunoprecipitation and protein-protein association assays, we show that both PROTEIN0 and PROTEIN1 bind specifically and stably to PROTEIN2 analysis demonstrates that both the basic region and leucine zipper domains of PROTEIN3 and PROTEIN4 are necessary and sufficient for stable association with PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 A 51-residue region from the conserved C-terminal region of PROTEIN0 previously shown to be the binding site for the viral activator protein PROTEIN1 interacts with PROTEIN2 and PROTEIN3 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A 51-residue region from the conserved C-terminal region of PROTEIN0 previously shown to be the binding site for the viral activator protein PROTEIN1 interacts with PROTEIN2 and PROTEIN3 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN3 A 51-residue region from the conserved C-terminal region of PROTEIN0 previously shown to be the binding site for the viral activator protein PROTEIN1 interacts with PROTEIN2 and PROTEIN3 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A 51-residue region from the conserved C-terminal region of PROTEIN0 previously shown to be the binding site for the viral activator protein PROTEIN1 interacts with PROTEIN2 and PROTEIN3 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN3 A 51-residue region from the conserved C-terminal region of PROTEIN0 previously shown to be the binding site for the viral activator protein PROTEIN1 interacts with PROTEIN2 and PROTEIN3 proteins.\n",
      "\n",
      "PROTEIN2 PROTEIN3 A 51-residue region from the conserved C-terminal region of PROTEIN0 previously shown to be the binding site for the viral activator protein PROTEIN1 interacts with PROTEIN2 and PROTEIN3 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We propose that PROTEIN0 and PROTEIN1 proteins function as transcriptional activators, in part by recruiting PROTEIN2 to form complexes to initiate RNA synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We propose that PROTEIN0 and PROTEIN1 proteins function as transcriptional activators, in part by recruiting PROTEIN2 to form complexes to initiate RNA synthesis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We propose that PROTEIN0 and PROTEIN1 proteins function as transcriptional activators, in part by recruiting PROTEIN2 to form complexes to initiate RNA synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 the protein product of the PROTEIN1 tumor-suppressor gene, regulates the activity of specific transcription factors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we report that stimulation of PROTEIN0 transcription by PROTEIN1 is partially abrogated at the nonpermissive temperature in ts13 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The stimulation of PROTEIN0 transcription by PROTEIN1 in ts13 cells at the nonpermissive temperature could be restored by the introduction of wild-type human PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The stimulation of PROTEIN0 transcription by PROTEIN1 in ts13 cells at the nonpermissive temperature could be restored by the introduction of wild-type human PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The stimulation of PROTEIN0 transcription by PROTEIN1 in ts13 cells at the nonpermissive temperature could be restored by the introduction of wild-type human PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, we demonstrate that PROTEIN0 binds directly to PROTEIN1 in vitro and in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 can confer transcriptional regulation and possibly cell cycle control and tumor suppression through an interaction with PROTEIN1 in particular with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results suggest that PROTEIN0 can confer transcriptional regulation and possibly cell cycle control and tumor suppression through an interaction with PROTEIN1 in particular with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results suggest that PROTEIN0 can confer transcriptional regulation and possibly cell cycle control and tumor suppression through an interaction with PROTEIN1 in particular with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cloning and characterization of PROTEIN0 PROTEIN1 and PROTEIN2 three subunits of the human transcription factor PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Cloning and characterization of PROTEIN0 PROTEIN1 and PROTEIN2 three subunits of the human transcription factor PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Cloning and characterization of PROTEIN0 PROTEIN1 and PROTEIN2 three subunits of the human transcription factor PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Cloning and characterization of PROTEIN0 PROTEIN1 and PROTEIN2 three subunits of the human transcription factor PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Cloning and characterization of PROTEIN0 PROTEIN1 and PROTEIN2 three subunits of the human transcription factor PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Cloning and characterization of PROTEIN0 PROTEIN1 and PROTEIN2 three subunits of the human transcription factor PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 We show that PROTEIN0 interacts with PROTEIN1 both in vitro and intracellularly; in contrast to its Drosophila homologue, PROTEIN2 also interacts directly with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 We show that PROTEIN0 interacts with PROTEIN1 both in vitro and intracellularly; in contrast to its Drosophila homologue, PROTEIN2 also interacts directly with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 We show that PROTEIN0 interacts with PROTEIN1 both in vitro and intracellularly; in contrast to its Drosophila homologue, PROTEIN2 also interacts directly with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 We show that PROTEIN0 interacts with PROTEIN1 both in vitro and intracellularly; in contrast to its Drosophila homologue, PROTEIN2 also interacts directly with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 We show that PROTEIN0 interacts with PROTEIN1 both in vitro and intracellularly; in contrast to its Drosophila homologue, PROTEIN2 also interacts directly with PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 We show that PROTEIN0 interacts with PROTEIN1 both in vitro and intracellularly; in contrast to its Drosophila homologue, PROTEIN2 also interacts directly with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN2 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN3 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN4 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN5 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN6 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN2 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN3 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN4 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN5 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN6 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN3 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN4 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN5 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN6 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN4 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN5 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN6 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN5 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN6 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN5 PROTEIN6 Deletion analysis indicates that PROTEIN0 and PROTEIN1 bind to distinct domains of PROTEIN2 PROTEIN3 also interacts with PROTEIN4 but it interacts more strongly with PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN1 The binding of PROTEIN0 and PROTEIN1 requires distinct domains of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The binding of PROTEIN0 and PROTEIN1 requires distinct domains of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The binding of PROTEIN0 and PROTEIN1 requires distinct domains of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 As observed with the homologous Drosophila proteins, PROTEIN0 interacts directly with PROTEIN1 however, additional interactions between PROTEIN2 and PROTEIN3 or PROTEIN4 were detected.\n",
      "\n",
      "PROTEIN0 PROTEIN2 As observed with the homologous Drosophila proteins, PROTEIN0 interacts directly with PROTEIN1 however, additional interactions between PROTEIN2 and PROTEIN3 or PROTEIN4 were detected.\n",
      "\n",
      "PROTEIN0 PROTEIN3 As observed with the homologous Drosophila proteins, PROTEIN0 interacts directly with PROTEIN1 however, additional interactions between PROTEIN2 and PROTEIN3 or PROTEIN4 were detected.\n",
      "\n",
      "PROTEIN0 PROTEIN4 As observed with the homologous Drosophila proteins, PROTEIN0 interacts directly with PROTEIN1 however, additional interactions between PROTEIN2 and PROTEIN3 or PROTEIN4 were detected.\n",
      "\n",
      "PROTEIN1 PROTEIN2 As observed with the homologous Drosophila proteins, PROTEIN0 interacts directly with PROTEIN1 however, additional interactions between PROTEIN2 and PROTEIN3 or PROTEIN4 were detected.\n",
      "\n",
      "PROTEIN1 PROTEIN3 As observed with the homologous Drosophila proteins, PROTEIN0 interacts directly with PROTEIN1 however, additional interactions between PROTEIN2 and PROTEIN3 or PROTEIN4 were detected.\n",
      "\n",
      "PROTEIN1 PROTEIN4 As observed with the homologous Drosophila proteins, PROTEIN0 interacts directly with PROTEIN1 however, additional interactions between PROTEIN2 and PROTEIN3 or PROTEIN4 were detected.\n",
      "\n",
      "PROTEIN2 PROTEIN3 As observed with the homologous Drosophila proteins, PROTEIN0 interacts directly with PROTEIN1 however, additional interactions between PROTEIN2 and PROTEIN3 or PROTEIN4 were detected.\n",
      "\n",
      "PROTEIN2 PROTEIN4 As observed with the homologous Drosophila proteins, PROTEIN0 interacts directly with PROTEIN1 however, additional interactions between PROTEIN2 and PROTEIN3 or PROTEIN4 were detected.\n",
      "\n",
      "PROTEIN3 PROTEIN4 As observed with the homologous Drosophila proteins, PROTEIN0 interacts directly with PROTEIN1 however, additional interactions between PROTEIN2 and PROTEIN3 or PROTEIN4 were detected.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results reveal differences not only in subunit composition, but also in the organization of PROTEIN0 and PROTEIN1 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This transcriptional activity is inhibited by association with the PROTEIN0 PROTEIN1 or its relatives PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 This transcriptional activity is inhibited by association with the PROTEIN0 PROTEIN1 or its relatives PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 This transcriptional activity is inhibited by association with the PROTEIN0 PROTEIN1 or its relatives PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 This transcriptional activity is inhibited by association with the PROTEIN0 PROTEIN1 or its relatives PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 This transcriptional activity is inhibited by association with the PROTEIN0 PROTEIN1 or its relatives PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 This transcriptional activity is inhibited by association with the PROTEIN0 PROTEIN1 or its relatives PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The first cDNA from the PROTEIN0 family to be cloned was designated PROTEIN1 and multiple PROTEIN2 family members have now been identified.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The first cDNA from the PROTEIN0 family to be cloned was designated PROTEIN1 and multiple PROTEIN2 family members have now been identified.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The first cDNA from the PROTEIN0 family to be cloned was designated PROTEIN1 and multiple PROTEIN2 family members have now been identified.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we demonstrate that DP is also a family of polypeptides with at least two members PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Both PROTEIN0 and PROTEIN1 bind to all PROTEIN2 family members in vivo, and each complex is capable of activating transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Both PROTEIN0 and PROTEIN1 bind to all PROTEIN2 family members in vivo, and each complex is capable of activating transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Both PROTEIN0 and PROTEIN1 bind to all PROTEIN2 family members in vivo, and each complex is capable of activating transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN0 PROTEIN3 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN0 PROTEIN4 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN0 PROTEIN5 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN1 PROTEIN3 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN1 PROTEIN4 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN1 PROTEIN5 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN2 PROTEIN3 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN2 PROTEIN4 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN2 PROTEIN5 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN3 PROTEIN4 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN3 PROTEIN5 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN4 PROTEIN5 However, the various PROTEIN0 complexes display strong differences in the ability to bind to either PROTEIN1 or PROTEIN2 in vivo, and the specificity of PROTEIN3 or PROTEIN4 binding is mediated by the PROTEIN5 subunit.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Two distinct and independent sites on PROTEIN0 trigger PROTEIN1 dimer formation and signalling.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The helical cytokine PROTEIN0 and its specific binding subunit PROTEIN1 form a 1:1 complex which, by promoting homodimerization of the signalling subunit PROTEIN2 on the surface of target cells, triggers intracellular responses.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The helical cytokine PROTEIN0 and its specific binding subunit PROTEIN1 form a 1:1 complex which, by promoting homodimerization of the signalling subunit PROTEIN2 on the surface of target cells, triggers intracellular responses.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The helical cytokine PROTEIN0 and its specific binding subunit PROTEIN1 form a 1:1 complex which, by promoting homodimerization of the signalling subunit PROTEIN2 on the surface of target cells, triggers intracellular responses.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We expressed differently tagged forms of PROTEIN0 and used them in solution-phase binding assays to show that the soluble extracellular domains of PROTEIN1 undergo dimerization in the absence of membranes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We demonstrate here that both the site 2 and site 3 PROTEIN0 variants complexed with PROTEIN1 bind a single PROTEIN2 molecule but are unable to dimerize it, whereas the combined site 2/3 variants lose the ability to interact with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 We demonstrate here that both the site 2 and site 3 PROTEIN0 variants complexed with PROTEIN1 bind a single PROTEIN2 molecule but are unable to dimerize it, whereas the combined site 2/3 variants lose the ability to interact with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 We demonstrate here that both the site 2 and site 3 PROTEIN0 variants complexed with PROTEIN1 bind a single PROTEIN2 molecule but are unable to dimerize it, whereas the combined site 2/3 variants lose the ability to interact with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 We demonstrate here that both the site 2 and site 3 PROTEIN0 variants complexed with PROTEIN1 bind a single PROTEIN2 molecule but are unable to dimerize it, whereas the combined site 2/3 variants lose the ability to interact with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 We demonstrate here that both the site 2 and site 3 PROTEIN0 variants complexed with PROTEIN1 bind a single PROTEIN2 molecule but are unable to dimerize it, whereas the combined site 2/3 variants lose the ability to interact with PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 We demonstrate here that both the site 2 and site 3 PROTEIN0 variants complexed with PROTEIN1 bind a single PROTEIN2 molecule but are unable to dimerize it, whereas the combined site 2/3 variants lose the ability to interact with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The binding properties of these variants in vitro, and the result of using a neutralizing monoclonal antibody directed against site 3, lead to the conclusion that PROTEIN0 dimer is formed through direct binding at two independent and differently oriented sites on PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoprecipitation experiments further reveal that the fully assembled receptor complex is composed of two PROTEIN0 two PROTEIN1 and two PROTEIN2 molecules.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunoprecipitation experiments further reveal that the fully assembled receptor complex is composed of two PROTEIN0 two PROTEIN1 and two PROTEIN2 molecules.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunoprecipitation experiments further reveal that the fully assembled receptor complex is composed of two PROTEIN0 two PROTEIN1 and two PROTEIN2 molecules.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 transcriptional activation mediated by coactivators PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 transcriptional activation mediated by coactivators PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 transcriptional activation mediated by coactivators PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The multisubunit PROTEIN0 PROTEIN1 is thought to be a primary target for site-specific activators of transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, a direct interaction between the activation domain of PROTEIN0 and two subunits of the PROTEIN1 complex, PROTEIN2 and PROTEIN3 is reported.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here, a direct interaction between the activation domain of PROTEIN0 and two subunits of the PROTEIN1 complex, PROTEIN2 and PROTEIN3 is reported.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here, a direct interaction between the activation domain of PROTEIN0 and two subunits of the PROTEIN1 complex, PROTEIN2 and PROTEIN3 is reported.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here, a direct interaction between the activation domain of PROTEIN0 and two subunits of the PROTEIN1 complex, PROTEIN2 and PROTEIN3 is reported.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here, a direct interaction between the activation domain of PROTEIN0 and two subunits of the PROTEIN1 complex, PROTEIN2 and PROTEIN3 is reported.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here, a direct interaction between the activation domain of PROTEIN0 and two subunits of the PROTEIN1 complex, PROTEIN2 and PROTEIN3 is reported.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A double point mutation in the activation domain of PROTEIN0 impaired the ability of this domain to activate transcription and, simultaneously, its ability to interact with both PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 A double point mutation in the activation domain of PROTEIN0 impaired the ability of this domain to activate transcription and, simultaneously, its ability to interact with both PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 A double point mutation in the activation domain of PROTEIN0 impaired the ability of this domain to activate transcription and, simultaneously, its ability to interact with both PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN7 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN8 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN7 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN8 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN2 PROTEIN7 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN2 PROTEIN8 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN3 PROTEIN7 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN3 PROTEIN8 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN4 PROTEIN7 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN4 PROTEIN8 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN5 PROTEIN7 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN5 PROTEIN8 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN6 PROTEIN7 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN6 PROTEIN8 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN7 PROTEIN8 Furthermore, a partial PROTEIN0 complex containing Drosophila PROTEIN1 PROTEIN2 human PROTEIN3 PROTEIN4 and PROTEIN5 supported activation by a Gal4-p53 fusion protein in vitro, whereas PROTEIN6 or a subcomplex lacking PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Together, these results suggest that PROTEIN0 and PROTEIN1 are important targets for transmitting activation signals between PROTEIN2 and the initiation complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Together, these results suggest that PROTEIN0 and PROTEIN1 are important targets for transmitting activation signals between PROTEIN2 and the initiation complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Together, these results suggest that PROTEIN0 and PROTEIN1 are important targets for transmitting activation signals between PROTEIN2 and the initiation complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Crystal structure of the heterodimeric bZIP transcription factor PROTEIN0 PROTEIN1 bound to DNA.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 and PROTEIN1 families of eukaryotic transcription factors heterodimerize to form complexes capable of binding 5'-TGAGTCA-3' DNA elements.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have determined the X-ray crystal structure of a heterodimer of the bZIP regions of PROTEIN0 and PROTEIN1 bound to DNA.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Comparison of the two crystallographically distinct protein-DNA complexes show that the coiled-coil is flexibly joined to the basic regions and that the PROTEIN0 PROTEIN1 heterodimer does not recognize the asymmetric 5'-TGAGTCA-3' recognition element in a unique orientation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a multisubunit protein complex comprised of the PROTEIN1 PROTEIN2 and multiple TBP-associated factors (TAFs).\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is a multisubunit protein complex comprised of the PROTEIN1 PROTEIN2 and multiple TBP-associated factors (TAFs).\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is a multisubunit protein complex comprised of the PROTEIN1 PROTEIN2 and multiple TBP-associated factors (TAFs).\n",
      "\n",
      "PROTEIN0 PROTEIN1 The cloning of a complementary DNA encoding a human PROTEIN0 TAF, PROTEIN1 that has no known homolog in Drosophila PROTEIN2 is now described.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The cloning of a complementary DNA encoding a human PROTEIN0 TAF, PROTEIN1 that has no known homolog in Drosophila PROTEIN2 is now described.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The cloning of a complementary DNA encoding a human PROTEIN0 TAF, PROTEIN1 that has no known homolog in Drosophila PROTEIN2 is now described.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN0 PROTEIN6 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PROTEIN0 PROTEIN7 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN0 PROTEIN8 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN1 PROTEIN6 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN1 PROTEIN7 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN1 PROTEIN8 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN2 PROTEIN6 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN2 PROTEIN7 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN2 PROTEIN8 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN3 PROTEIN6 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN3 PROTEIN7 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN3 PROTEIN8 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN4 PROTEIN6 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN4 PROTEIN7 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN4 PROTEIN8 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN5 PROTEIN6 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN5 PROTEIN7 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN5 PROTEIN8 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN6 PROTEIN7 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN6 PROTEIN8 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN7 PROTEIN8 PROTEIN0 is shown to interact with the largest subunit PROTEIN1 of human PROTEIN2 through its central region and with multiple activators--including PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 adenoviral PROTEIN7 and human immunodeficiency virus-type 1 PROTEIN8 proteins--through a distinct amino-terminal domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 region of PROTEIN1 was localized to its DNA-binding domain, which is distinct from the glutamine-rich activation domains previously shown to interact with Drosophila PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 region of PROTEIN1 was localized to its DNA-binding domain, which is distinct from the glutamine-rich activation domains previously shown to interact with Drosophila PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 region of PROTEIN1 was localized to its DNA-binding domain, which is distinct from the glutamine-rich activation domains previously shown to interact with Drosophila PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Human spleen tyrosine kinase PROTEIN0 associates with the Src-family kinase PROTEIN1 PROTEIN2 and a PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Human spleen tyrosine kinase PROTEIN0 associates with the Src-family kinase PROTEIN1 PROTEIN2 and a PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Human spleen tyrosine kinase PROTEIN0 associates with the Src-family kinase PROTEIN1 PROTEIN2 and a PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Human spleen tyrosine kinase PROTEIN0 associates with the Src-family kinase PROTEIN1 PROTEIN2 and a PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Human spleen tyrosine kinase PROTEIN0 associates with the Src-family kinase PROTEIN1 PROTEIN2 and a PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Human spleen tyrosine kinase PROTEIN0 associates with the Src-family kinase PROTEIN1 PROTEIN2 and a PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The 72-kDa spleen tyrosine kinase PROTEIN0 and Src-family kinase PROTEIN1 PROTEIN2 PROTEIN3 contribute to signaling via the B-cell antigen receptor complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The 72-kDa spleen tyrosine kinase PROTEIN0 and Src-family kinase PROTEIN1 PROTEIN2 PROTEIN3 contribute to signaling via the B-cell antigen receptor complex.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The 72-kDa spleen tyrosine kinase PROTEIN0 and Src-family kinase PROTEIN1 PROTEIN2 PROTEIN3 contribute to signaling via the B-cell antigen receptor complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The 72-kDa spleen tyrosine kinase PROTEIN0 and Src-family kinase PROTEIN1 PROTEIN2 PROTEIN3 contribute to signaling via the B-cell antigen receptor complex.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The 72-kDa spleen tyrosine kinase PROTEIN0 and Src-family kinase PROTEIN1 PROTEIN2 PROTEIN3 contribute to signaling via the B-cell antigen receptor complex.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The 72-kDa spleen tyrosine kinase PROTEIN0 and Src-family kinase PROTEIN1 PROTEIN2 PROTEIN3 contribute to signaling via the B-cell antigen receptor complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we show that PROTEIN0 and PROTEIN1 from human B lymphocytes can interact directly.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 coimmunoprecipitated from mature and activated B-cell lines, and gel-purified PROTEIN2 and PROTEIN3 reassociated in vitro, demonstrating their direct interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 and PROTEIN1 coimmunoprecipitated from mature and activated B-cell lines, and gel-purified PROTEIN2 and PROTEIN3 reassociated in vitro, demonstrating their direct interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 and PROTEIN1 coimmunoprecipitated from mature and activated B-cell lines, and gel-purified PROTEIN2 and PROTEIN3 reassociated in vitro, demonstrating their direct interaction.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 and PROTEIN1 coimmunoprecipitated from mature and activated B-cell lines, and gel-purified PROTEIN2 and PROTEIN3 reassociated in vitro, demonstrating their direct interaction.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 and PROTEIN1 coimmunoprecipitated from mature and activated B-cell lines, and gel-purified PROTEIN2 and PROTEIN3 reassociated in vitro, demonstrating their direct interaction.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 and PROTEIN1 coimmunoprecipitated from mature and activated B-cell lines, and gel-purified PROTEIN2 and PROTEIN3 reassociated in vitro, demonstrating their direct interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN2 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN3 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN4 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN5 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN2 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN3 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN4 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN5 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN2 PROTEIN3 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN2 PROTEIN4 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN2 PROTEIN5 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN3 PROTEIN4 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN3 PROTEIN5 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN4 PROTEIN5 This PROTEIN0 PROTEIN1 interaction may be dependent on the stage of B-cell differentiation, since PROTEIN2 PROTEIN3 associations were not detected in pre-B and myeloma cell lines and PROTEIN4 from an immature B-cell line did not reassociate with PROTEIN5 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Crosslinking of cell surface IgM led to rapid activation of both tyrosine and serine/threonine protein kinase activities that resulted in phosphorylation in vitro of proteins coprecipitating with PROTEIN0 particular, a serine/threonine phosphorylated protein 120 kDa in size PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Several phosphoproteins, including one of 72 kDa and one of 120 kDa, coprecipitated with PROTEIN0 PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Sequential immunoprecipitation identified the 72-kDa protein associated with PROTEIN0 as PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The 120-kDa serine/threonine phosphorylated protein that coprecipitated with PROTEIN0 resembled the PROTEIN1 PROTEIN2 by several criteria.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The 120-kDa serine/threonine phosphorylated protein that coprecipitated with PROTEIN0 resembled the PROTEIN1 PROTEIN2 by several criteria.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The 120-kDa serine/threonine phosphorylated protein that coprecipitated with PROTEIN0 resembled the PROTEIN1 PROTEIN2 by several criteria.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, PROTEIN0 may serve as a link between PROTEIN1 and PROTEIN2 coupling the B-cell antigen receptor to the phosphatidylinositol pathway.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, PROTEIN0 may serve as a link between PROTEIN1 and PROTEIN2 coupling the B-cell antigen receptor to the phosphatidylinositol pathway.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, PROTEIN0 may serve as a link between PROTEIN1 and PROTEIN2 coupling the B-cell antigen receptor to the phosphatidylinositol pathway.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 protein shares no significant homology with PROTEIN1 or other PROTEIN2 family proteins, which can form homo- and heterodimers.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 protein shares no significant homology with PROTEIN1 or other PROTEIN2 family proteins, which can form homo- and heterodimers.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 protein shares no significant homology with PROTEIN1 or other PROTEIN2 family proteins, which can form homo- and heterodimers.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The findings indicate that bag-1 represents a new type of anti-cell death gene and suggest that some routes of apoptosis induction previously ascribed to PROTEIN0 pathways may instead reflect a need for the combination of PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The findings indicate that bag-1 represents a new type of anti-cell death gene and suggest that some routes of apoptosis induction previously ascribed to PROTEIN0 pathways may instead reflect a need for the combination of PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The findings indicate that bag-1 represents a new type of anti-cell death gene and suggest that some routes of apoptosis induction previously ascribed to PROTEIN0 pathways may instead reflect a need for the combination of PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binds to an alternatively spliced exon of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 We expressed the cDNA of human PROTEIN0 and the COOH terminus of human PROTEIN1 in reticulocyte lysates using an in vitro transcription/translation system.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We demonstrate that the 53 amino acids encoded on exon 74 of PROTEIN0 an alternatively spliced exon, are necessary and sufficient for interaction with translated PROTEIN1 in our assay.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have found a COOH-terminal 37-kD fragment of PROTEIN0 sufficient to interact with translated PROTEIN1 and its homologues, suggesting that the PROTEIN2 binding site on PROTEIN3 occurs on a region that is conserved among the three syntrophin homologues.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have found a COOH-terminal 37-kD fragment of PROTEIN0 sufficient to interact with translated PROTEIN1 and its homologues, suggesting that the PROTEIN2 binding site on PROTEIN3 occurs on a region that is conserved among the three syntrophin homologues.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We have found a COOH-terminal 37-kD fragment of PROTEIN0 sufficient to interact with translated PROTEIN1 and its homologues, suggesting that the PROTEIN2 binding site on PROTEIN3 occurs on a region that is conserved among the three syntrophin homologues.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have found a COOH-terminal 37-kD fragment of PROTEIN0 sufficient to interact with translated PROTEIN1 and its homologues, suggesting that the PROTEIN2 binding site on PROTEIN3 occurs on a region that is conserved among the three syntrophin homologues.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We have found a COOH-terminal 37-kD fragment of PROTEIN0 sufficient to interact with translated PROTEIN1 and its homologues, suggesting that the PROTEIN2 binding site on PROTEIN3 occurs on a region that is conserved among the three syntrophin homologues.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We have found a COOH-terminal 37-kD fragment of PROTEIN0 sufficient to interact with translated PROTEIN1 and its homologues, suggesting that the PROTEIN2 binding site on PROTEIN3 occurs on a region that is conserved among the three syntrophin homologues.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interaction with PROTEIN0 subunit of PROTEIN1 is required for transcriptional activation by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Interaction with PROTEIN0 subunit of PROTEIN1 is required for transcriptional activation by PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Interaction with PROTEIN0 subunit of PROTEIN1 is required for transcriptional activation by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 A few general transcription factors, in particular PROTEIN0 and PROTEIN1 have been found to bind transcriptional activators.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we show that the general transcription factor PROTEIN0 is also a target for a transcriptional activator, namely PROTEIN1 PROTEIN2 which binds to the PROTEIN3 promoter.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we show that the general transcription factor PROTEIN0 is also a target for a transcriptional activator, namely PROTEIN1 PROTEIN2 which binds to the PROTEIN3 promoter.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here we show that the general transcription factor PROTEIN0 is also a target for a transcriptional activator, namely PROTEIN1 PROTEIN2 which binds to the PROTEIN3 promoter.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we show that the general transcription factor PROTEIN0 is also a target for a transcriptional activator, namely PROTEIN1 PROTEIN2 which binds to the PROTEIN3 promoter.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here we show that the general transcription factor PROTEIN0 is also a target for a transcriptional activator, namely PROTEIN1 PROTEIN2 which binds to the PROTEIN3 promoter.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here we show that the general transcription factor PROTEIN0 is also a target for a transcriptional activator, namely PROTEIN1 PROTEIN2 which binds to the PROTEIN3 promoter.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using a yeast interaction assay, we find that PROTEIN0 binds the PROTEIN1 subunit of PROTEIN2 and that PROTEIN3 transcriptional activation domain is the region involved in this binding.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using a yeast interaction assay, we find that PROTEIN0 binds the PROTEIN1 subunit of PROTEIN2 and that PROTEIN3 transcriptional activation domain is the region involved in this binding.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Using a yeast interaction assay, we find that PROTEIN0 binds the PROTEIN1 subunit of PROTEIN2 and that PROTEIN3 transcriptional activation domain is the region involved in this binding.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using a yeast interaction assay, we find that PROTEIN0 binds the PROTEIN1 subunit of PROTEIN2 and that PROTEIN3 transcriptional activation domain is the region involved in this binding.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Using a yeast interaction assay, we find that PROTEIN0 binds the PROTEIN1 subunit of PROTEIN2 and that PROTEIN3 transcriptional activation domain is the region involved in this binding.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Using a yeast interaction assay, we find that PROTEIN0 binds the PROTEIN1 subunit of PROTEIN2 and that PROTEIN3 transcriptional activation domain is the region involved in this binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Further, PROTEIN0 central charged cluster domain is required for binding to PROTEIN1 activation domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Deletion of this domain impairs PROTEIN0 ability to support PROTEIN1 transcription in vitro but has little effect on the protein's basal transcription activity or its ability to support PROTEIN2 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Deletion of this domain impairs PROTEIN0 ability to support PROTEIN1 transcription in vitro but has little effect on the protein's basal transcription activity or its ability to support PROTEIN2 transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Deletion of this domain impairs PROTEIN0 ability to support PROTEIN1 transcription in vitro but has little effect on the protein's basal transcription activity or its ability to support PROTEIN2 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The correlation of PROTEIN0 PROTEIN1 binding with transcriptional activation suggests that PROTEIN2 is a critical target for PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The correlation of PROTEIN0 PROTEIN1 binding with transcriptional activation suggests that PROTEIN2 is a critical target for PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The correlation of PROTEIN0 PROTEIN1 binding with transcriptional activation suggests that PROTEIN2 is a critical target for PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The correlation of PROTEIN0 PROTEIN1 binding with transcriptional activation suggests that PROTEIN2 is a critical target for PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The correlation of PROTEIN0 PROTEIN1 binding with transcriptional activation suggests that PROTEIN2 is a critical target for PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The correlation of PROTEIN0 PROTEIN1 binding with transcriptional activation suggests that PROTEIN2 is a critical target for PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Activity of RNA polymerase I transcription factor PROTEIN0 blocked by PROTEIN1 gene product.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The protein encoded by the PROTEIN0 PROTEIN1 functions as a tumour suppressor and negative growth regulator.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This last observation was confirmed by affinity chromatography and immunoprecipitation, which demonstrated an interaction between PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results indicate that there is an additional mechanism by which PROTEIN0 suppresses cell growth, namely that PROTEIN1 directly represses transcription of the rRNA genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 homodimer and PROTEIN1 heterodimer are mutual antagonists which share a common beta subunit.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Activin type I receptors, PROTEIN0 and PROTEIN1 were first cloned from well differentiated human hepatoma cells (HepG2).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we show that PROTEIN0 binds to the PROTEIN1 PROTEIN2 which induces activin binding to the type I receptor kinase PROTEIN3 to form PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we show that PROTEIN0 binds to the PROTEIN1 PROTEIN2 which induces activin binding to the type I receptor kinase PROTEIN3 to form PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here we show that PROTEIN0 binds to the PROTEIN1 PROTEIN2 which induces activin binding to the type I receptor kinase PROTEIN3 to form PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Here we show that PROTEIN0 binds to the PROTEIN1 PROTEIN2 which induces activin binding to the type I receptor kinase PROTEIN3 to form PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we show that PROTEIN0 binds to the PROTEIN1 PROTEIN2 which induces activin binding to the type I receptor kinase PROTEIN3 to form PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here we show that PROTEIN0 binds to the PROTEIN1 PROTEIN2 which induces activin binding to the type I receptor kinase PROTEIN3 to form PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Here we show that PROTEIN0 binds to the PROTEIN1 PROTEIN2 which induces activin binding to the type I receptor kinase PROTEIN3 to form PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here we show that PROTEIN0 binds to the PROTEIN1 PROTEIN2 which induces activin binding to the type I receptor kinase PROTEIN3 to form PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Here we show that PROTEIN0 binds to the PROTEIN1 PROTEIN2 which induces activin binding to the type I receptor kinase PROTEIN3 to form PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Here we show that PROTEIN0 binds to the PROTEIN1 PROTEIN2 which induces activin binding to the type I receptor kinase PROTEIN3 to form PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 complexes in which an PROTEIN1 chain occupies each receptor subunit.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 also binds to PROTEIN1 through its beta subunit, competes with the binding of activin to PROTEIN2 but fails to form the PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 also binds to PROTEIN1 through its beta subunit, competes with the binding of activin to PROTEIN2 but fails to form the PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 also binds to PROTEIN1 through its beta subunit, competes with the binding of activin to PROTEIN2 but fails to form the PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 also binds to PROTEIN1 through its beta subunit, competes with the binding of activin to PROTEIN2 but fails to form the PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 also binds to PROTEIN1 through its beta subunit, competes with the binding of activin to PROTEIN2 but fails to form the PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 also binds to PROTEIN1 through its beta subunit, competes with the binding of activin to PROTEIN2 but fails to form the PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 inhibitor of cyclin-dependent kinases controls DNA replication by interaction with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 tumour-suppressor protein controls the expression of a gene encoding the PROTEIN1 cyclin-dependent protein kinase (CDK) regulator.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Levels of PROTEIN0 protein are increased in senescent cells and PROTEIN1 overexpression blocks the growth of tumour cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In normal human cells, but not in many tumour cells, PROTEIN0 exists in a quaternary complex with a cyclin, a CDK, and the PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In normal human cells, but not in many tumour cells, PROTEIN0 exists in a quaternary complex with a cyclin, a CDK, and the PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In normal human cells, but not in many tumour cells, PROTEIN0 exists in a quaternary complex with a cyclin, a CDK, and the PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 controls CDK activity, thereby affecting cell-cycle control, whereas PROTEIN1 functions in both DNA replication and repair.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we use simian virus 40 DNA replication in vitro to show than PROTEIN0 directly inhibits PROTEIN1 DNA replication in the absence of a cyclin/CDK.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, PROTEIN0 blocks the ability of PROTEIN1 to activate PROTEIN2 the principal replicative DNA polymerase.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Furthermore, PROTEIN0 blocks the ability of PROTEIN1 to activate PROTEIN2 the principal replicative DNA polymerase.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Furthermore, PROTEIN0 blocks the ability of PROTEIN1 to activate PROTEIN2 the principal replicative DNA polymerase.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This regulation results from a direct interaction between PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, during PROTEIN0 suppression of cell proliferation, PROTEIN1 and PROTEIN2 may be important for coordinating cell-cycle progression, DNA replication and repair of damaged DNA.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, during PROTEIN0 suppression of cell proliferation, PROTEIN1 and PROTEIN2 may be important for coordinating cell-cycle progression, DNA replication and repair of damaged DNA.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, during PROTEIN0 suppression of cell proliferation, PROTEIN1 and PROTEIN2 may be important for coordinating cell-cycle progression, DNA replication and repair of damaged DNA.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Nuclear protein PROTEIN0 is a coactivator for the transcription factor PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is activated through phosphorylation by PROTEIN1 PROTEIN2 but precisely how phosphorylation stimulates PROTEIN3 function is unknown.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is activated through phosphorylation by PROTEIN1 PROTEIN2 but precisely how phosphorylation stimulates PROTEIN3 function is unknown.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 is activated through phosphorylation by PROTEIN1 PROTEIN2 but precisely how phosphorylation stimulates PROTEIN3 function is unknown.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is activated through phosphorylation by PROTEIN1 PROTEIN2 but precisely how phosphorylation stimulates PROTEIN3 function is unknown.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 is activated through phosphorylation by PROTEIN1 PROTEIN2 but precisely how phosphorylation stimulates PROTEIN3 function is unknown.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 is activated through phosphorylation by PROTEIN1 PROTEIN2 but precisely how phosphorylation stimulates PROTEIN3 function is unknown.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have previously identified a nuclear protein of M(r)265K, PROTEIN0 that binds specifically to the PROTEIN1 form of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have previously identified a nuclear protein of M(r)265K, PROTEIN0 that binds specifically to the PROTEIN1 form of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have previously identified a nuclear protein of M(r)265K, PROTEIN0 that binds specifically to the PROTEIN1 form of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have used fluorescence anisotropy measurements to define the equilibrium binding parameters of the PROTEIN0 PROTEIN1 interaction and report here that PROTEIN2 can activate transcription through a region in its carboxy terminus.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have used fluorescence anisotropy measurements to define the equilibrium binding parameters of the PROTEIN0 PROTEIN1 interaction and report here that PROTEIN2 can activate transcription through a region in its carboxy terminus.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have used fluorescence anisotropy measurements to define the equilibrium binding parameters of the PROTEIN0 PROTEIN1 interaction and report here that PROTEIN2 can activate transcription through a region in its carboxy terminus.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The activation domain of PROTEIN0 interacts with the basal transcription factor PROTEIN1 through a domain that is conserved in the yeast coactivator PROTEIN2 (ref. 8).\n",
      "\n",
      "PROTEIN0 PROTEIN2 The activation domain of PROTEIN0 interacts with the basal transcription factor PROTEIN1 through a domain that is conserved in the yeast coactivator PROTEIN2 (ref. 8).\n",
      "\n",
      "PROTEIN1 PROTEIN2 The activation domain of PROTEIN0 interacts with the basal transcription factor PROTEIN1 through a domain that is conserved in the yeast coactivator PROTEIN2 (ref. 8).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Consistent with its role as a coactivator, PROTEIN0 augments the activity of phosphorylated PROTEIN1 to activate transcription of cAMP-responsive genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Specific binding of the transglutaminase, PROTEIN0 to PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, the function of PROTEIN0 and the identity of proteins that interact with PROTEIN1 remain unknown.\n",
      "\n",
      "PROTEIN0 PROTEIN1 To identify specific PROTEIN0 interactions, a recombinant fusion protein affinity reagent was constructed and used to identify proteins associating with PROTEIN1 from human platelet lysates and erythroleukemia cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoblotting studies were used to confirm the binding of PROTEIN0 from fresh platelet lysates to the PROTEIN1 fusion protein.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 also was shown to coprecipitate with PROTEIN1 in immunoprecipitation studies and to colocalize with PROTEIN2 in immunofluorescence studies of intact glass-activated platelets.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 also was shown to coprecipitate with PROTEIN1 in immunoprecipitation studies and to colocalize with PROTEIN2 in immunofluorescence studies of intact glass-activated platelets.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 also was shown to coprecipitate with PROTEIN1 in immunoprecipitation studies and to colocalize with PROTEIN2 in immunofluorescence studies of intact glass-activated platelets.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The data thus demonstrate specific binding of platelet PROTEIN0 to PROTEIN1 and suggest that PROTEIN2 may participate in the cellular localization and/or enzymatic regulation of platelet PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The data thus demonstrate specific binding of platelet PROTEIN0 to PROTEIN1 and suggest that PROTEIN2 may participate in the cellular localization and/or enzymatic regulation of platelet PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The data thus demonstrate specific binding of platelet PROTEIN0 to PROTEIN1 and suggest that PROTEIN2 may participate in the cellular localization and/or enzymatic regulation of platelet PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The data thus demonstrate specific binding of platelet PROTEIN0 to PROTEIN1 and suggest that PROTEIN2 may participate in the cellular localization and/or enzymatic regulation of platelet PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The data thus demonstrate specific binding of platelet PROTEIN0 to PROTEIN1 and suggest that PROTEIN2 may participate in the cellular localization and/or enzymatic regulation of platelet PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The data thus demonstrate specific binding of platelet PROTEIN0 to PROTEIN1 and suggest that PROTEIN2 may participate in the cellular localization and/or enzymatic regulation of platelet PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Rather, the hydrophobic amino acid residues Leu-22 and Trp-23 of human PROTEIN0 are both required for trans-activation activity, binding to the adenovirus PROTEIN1 55-kD protein and the human PROTEIN2 PROTEIN3 protein in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Rather, the hydrophobic amino acid residues Leu-22 and Trp-23 of human PROTEIN0 are both required for trans-activation activity, binding to the adenovirus PROTEIN1 55-kD protein and the human PROTEIN2 PROTEIN3 protein in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Rather, the hydrophobic amino acid residues Leu-22 and Trp-23 of human PROTEIN0 are both required for trans-activation activity, binding to the adenovirus PROTEIN1 55-kD protein and the human PROTEIN2 PROTEIN3 protein in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Rather, the hydrophobic amino acid residues Leu-22 and Trp-23 of human PROTEIN0 are both required for trans-activation activity, binding to the adenovirus PROTEIN1 55-kD protein and the human PROTEIN2 PROTEIN3 protein in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Rather, the hydrophobic amino acid residues Leu-22 and Trp-23 of human PROTEIN0 are both required for trans-activation activity, binding to the adenovirus PROTEIN1 55-kD protein and the human PROTEIN2 PROTEIN3 protein in vitro.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Rather, the hydrophobic amino acid residues Leu-22 and Trp-23 of human PROTEIN0 are both required for trans-activation activity, binding to the adenovirus PROTEIN1 55-kD protein and the human PROTEIN2 PROTEIN3 protein in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, hydrophobic residues Leu-14 and Phe-19 are crucial for the interactions between PROTEIN0 and human PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In addition, hydrophobic residues Leu-14 and Phe-19 are crucial for the interactions between PROTEIN0 and human PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In addition, hydrophobic residues Leu-14 and Phe-19 are crucial for the interactions between PROTEIN0 and human PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 These mutations have no impact on the ability of the PROTEIN0 protein to bind to a PROTEIN1 DNA element.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These very same hydrophobic residues contact the PROTEIN0 and Ad5 PROTEIN1 55-kD oncogene products.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In previous studies (Scott, J. D., Stofko, R. E., McDonald, J. R., Comer, J. D., Vitalis, E. A., and Mangili J. (1990) J. Biol. Chem. 265, 21561-21566) we have shown that dimerization of PROTEIN0 was required for interaction with the cytoskeletal component PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Measurement of PROTEIN0 I3A,I5A interaction with the human PROTEIN1 PROTEIN2 by two independent methods suggests that mutation of isoleucines 3 and 5 decreases affinity by at least 6-fold.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Measurement of PROTEIN0 I3A,I5A interaction with the human PROTEIN1 PROTEIN2 by two independent methods suggests that mutation of isoleucines 3 and 5 decreases affinity by at least 6-fold.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Measurement of PROTEIN0 I3A,I5A interaction with the human PROTEIN1 PROTEIN2 by two independent methods suggests that mutation of isoleucines 3 and 5 decreases affinity by at least 6-fold.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Characterization of an interaction between PROTEIN0 and the PROTEIN1 by using the two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 is a major substrate of the PROTEIN2 and has been implicated in PROTEIN3 signaling.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 is a major substrate of the PROTEIN2 and has been implicated in PROTEIN3 signaling.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 is a major substrate of the PROTEIN2 and has been implicated in PROTEIN3 signaling.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 is a major substrate of the PROTEIN2 and has been implicated in PROTEIN3 signaling.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 is a major substrate of the PROTEIN2 and has been implicated in PROTEIN3 signaling.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 is a major substrate of the PROTEIN2 and has been implicated in PROTEIN3 signaling.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We demonstrate that PROTEIN0 forms a specific complex with the cytoplasmic domain of the PROTEIN1 when both are expressed as hybrid proteins in yeast cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Lastly, in agreement with our findings for yeast cells, we show that the PROTEIN0 is unable to phosphorylate an PROTEIN1 protein containing a deletion of amino acids 45 to 516 when expressed in COS cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The protein PROTEIN0 a key mediator of mitogenesis and differentiation, associates with PROTEIN1 (refs 1-3).\n",
      "\n",
      "PROTEIN0 PROTEIN1 These proteins, PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 are members of the 14-3-3 family of proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These proteins, PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 are members of the 14-3-3 family of proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN3 These proteins, PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 are members of the 14-3-3 family of proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These proteins, PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 are members of the 14-3-3 family of proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN3 These proteins, PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 are members of the 14-3-3 family of proteins.\n",
      "\n",
      "PROTEIN2 PROTEIN3 These proteins, PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 are members of the 14-3-3 family of proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Expression of 14-3-3 proteins in Xenopus oocytes enhanced PROTEIN0 activity and promoted PROTEIN1 oocyte maturation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Interactions of the PROTEIN0 protein with itself and other members of the PROTEIN1 family, including PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 were explored with a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 also interacted with PROTEIN1 and PROTEIN2 and with the dominant inhibitors PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 also interacted with PROTEIN1 and PROTEIN2 and with the dominant inhibitors PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 also interacted with PROTEIN1 and PROTEIN2 and with the dominant inhibitors PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 also interacted with PROTEIN1 and PROTEIN2 and with the dominant inhibitors PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 also interacted with PROTEIN1 and PROTEIN2 and with the dominant inhibitors PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 also interacted with PROTEIN1 and PROTEIN2 and with the dominant inhibitors PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 also interacted with PROTEIN1 and PROTEIN2 and with the dominant inhibitors PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 also interacted with PROTEIN1 and PROTEIN2 and with the dominant inhibitors PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 also interacted with PROTEIN1 and PROTEIN2 and with the dominant inhibitors PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 also interacted with PROTEIN1 and PROTEIN2 and with the dominant inhibitors PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 displayed the same pattern of combinatorial interactions with PROTEIN1 family proteins as PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 displayed the same pattern of combinatorial interactions with PROTEIN1 family proteins as PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 displayed the same pattern of combinatorial interactions with PROTEIN1 family proteins as PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast to PROTEIN0 fusion proteins, expression of a PROTEIN1 protein was lethal to yeast.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This cytotoxicity could be abrogated by B42 fusion proteins containing PROTEIN0 PROTEIN1 or PROTEIN2 but not those containing PROTEIN3 (an alternatively spliced form of PROTEIN4 that lacks a well-conserved 63-amino acid region).\n",
      "\n",
      "PROTEIN0 PROTEIN2 This cytotoxicity could be abrogated by B42 fusion proteins containing PROTEIN0 PROTEIN1 or PROTEIN2 but not those containing PROTEIN3 (an alternatively spliced form of PROTEIN4 that lacks a well-conserved 63-amino acid region).\n",
      "\n",
      "PROTEIN0 PROTEIN3 This cytotoxicity could be abrogated by B42 fusion proteins containing PROTEIN0 PROTEIN1 or PROTEIN2 but not those containing PROTEIN3 (an alternatively spliced form of PROTEIN4 that lacks a well-conserved 63-amino acid region).\n",
      "\n",
      "PROTEIN0 PROTEIN4 This cytotoxicity could be abrogated by B42 fusion proteins containing PROTEIN0 PROTEIN1 or PROTEIN2 but not those containing PROTEIN3 (an alternatively spliced form of PROTEIN4 that lacks a well-conserved 63-amino acid region).\n",
      "\n",
      "PROTEIN1 PROTEIN2 This cytotoxicity could be abrogated by B42 fusion proteins containing PROTEIN0 PROTEIN1 or PROTEIN2 but not those containing PROTEIN3 (an alternatively spliced form of PROTEIN4 that lacks a well-conserved 63-amino acid region).\n",
      "\n",
      "PROTEIN1 PROTEIN3 This cytotoxicity could be abrogated by B42 fusion proteins containing PROTEIN0 PROTEIN1 or PROTEIN2 but not those containing PROTEIN3 (an alternatively spliced form of PROTEIN4 that lacks a well-conserved 63-amino acid region).\n",
      "\n",
      "PROTEIN1 PROTEIN4 This cytotoxicity could be abrogated by B42 fusion proteins containing PROTEIN0 PROTEIN1 or PROTEIN2 but not those containing PROTEIN3 (an alternatively spliced form of PROTEIN4 that lacks a well-conserved 63-amino acid region).\n",
      "\n",
      "PROTEIN2 PROTEIN3 This cytotoxicity could be abrogated by B42 fusion proteins containing PROTEIN0 PROTEIN1 or PROTEIN2 but not those containing PROTEIN3 (an alternatively spliced form of PROTEIN4 that lacks a well-conserved 63-amino acid region).\n",
      "\n",
      "PROTEIN2 PROTEIN4 This cytotoxicity could be abrogated by B42 fusion proteins containing PROTEIN0 PROTEIN1 or PROTEIN2 but not those containing PROTEIN3 (an alternatively spliced form of PROTEIN4 that lacks a well-conserved 63-amino acid region).\n",
      "\n",
      "PROTEIN3 PROTEIN4 This cytotoxicity could be abrogated by B42 fusion proteins containing PROTEIN0 PROTEIN1 or PROTEIN2 but not those containing PROTEIN3 (an alternatively spliced form of PROTEIN4 that lacks a well-conserved 63-amino acid region).\n",
      "\n",
      "PROTEIN0 PROTEIN1 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN0 PROTEIN6 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN1 PROTEIN6 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN2 PROTEIN6 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN3 PROTEIN6 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN4 PROTEIN6 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN5 PROTEIN6 The findings suggest a model whereby PROTEIN0 and PROTEIN1 differentially regulate PROTEIN2 function, and indicate that requirements for PROTEIN3 PROTEIN4 heterodimerization may be different from those for PROTEIN5 PROTEIN6 homodimerization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The rates of commitment to renaturation of PROTEIN0 and PROTEIN1 in the presence of the groE chaperonins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using immunoprecipitation techniques, we have determined the rates at which PROTEIN0 and PROTEIN1 PROTEIN2 are released from PROTEIN3 in a form committed to refold to active enzyme.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using immunoprecipitation techniques, we have determined the rates at which PROTEIN0 and PROTEIN1 PROTEIN2 are released from PROTEIN3 in a form committed to refold to active enzyme.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Using immunoprecipitation techniques, we have determined the rates at which PROTEIN0 and PROTEIN1 PROTEIN2 are released from PROTEIN3 in a form committed to refold to active enzyme.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using immunoprecipitation techniques, we have determined the rates at which PROTEIN0 and PROTEIN1 PROTEIN2 are released from PROTEIN3 in a form committed to refold to active enzyme.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Using immunoprecipitation techniques, we have determined the rates at which PROTEIN0 and PROTEIN1 PROTEIN2 are released from PROTEIN3 in a form committed to refold to active enzyme.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Using immunoprecipitation techniques, we have determined the rates at which PROTEIN0 and PROTEIN1 PROTEIN2 are released from PROTEIN3 in a form committed to refold to active enzyme.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 were chosen as substrates because they exhibit different solution requirements for the chaperonin system and they form stable \"folding arrested\" complexes with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 and PROTEIN1 were chosen as substrates because they exhibit different solution requirements for the chaperonin system and they form stable \"folding arrested\" complexes with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 and PROTEIN1 were chosen as substrates because they exhibit different solution requirements for the chaperonin system and they form stable \"folding arrested\" complexes with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 At 25 degrees C, the rate profiles indicate the release and commitment to folding of PROTEIN0 to its native state occurs far earlier (t1/2 < 1 min) than for PROTEIN1 (t1/2 = 5 min).\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, the renaturation rate for PROTEIN0 with the groE chaperonins mirrored the committed renaturation rates following PROTEIN1 depletion.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This suggests that PROTEIN0 must interact with PROTEIN1 throughout most of its folding reaction before it acquires a folding competent (groE independent) state.\n",
      "\n",
      "PROTEIN0 PROTEIN1 If current models of chaperonin mechanism are correct, PROTEIN0 undergoes more rebinding and release cycles than does PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interaction of PROTEIN0 and gamma c chains with PROTEIN1 and PROTEIN2 implications for XSCID and XCID.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Interaction of PROTEIN0 and gamma c chains with PROTEIN1 and PROTEIN2 implications for XSCID and XCID.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Interaction of PROTEIN0 and gamma c chains with PROTEIN1 and PROTEIN2 implications for XSCID and XCID.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 (whose receptors are known to contain gamma c), and PROTEIN3 (whose receptor is shown here to contain gamma c) induced the tyrosine phosphorylation and activation of the Janus family tyrosine kinases PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN0 PROTEIN6 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN1 PROTEIN6 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN2 PROTEIN6 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN3 PROTEIN6 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN4 PROTEIN6 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN5 PROTEIN6 PROTEIN0 and PROTEIN1 associated with PROTEIN2 and gamma c, respectively; PROTEIN3 induced PROTEIN4 PROTEIN5 and increased PROTEIN6 c associations.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, gamma c mutations in at least some XSCID and XCID patients prevent normal PROTEIN0 activation, suggesting that mutations of PROTEIN1 may result in an XSCID-like phenotype.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The X-ray structure of a PROTEIN0 PROTEIN1 receptor complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The human pituitary hormones, PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 regulate a large variety of physiological processes, among which are growth and differentiation of muscle, bone and cartilage cells, and lactation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The human pituitary hormones, PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 regulate a large variety of physiological processes, among which are growth and differentiation of muscle, bone and cartilage cells, and lactation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The human pituitary hormones, PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 regulate a large variety of physiological processes, among which are growth and differentiation of muscle, bone and cartilage cells, and lactation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The human pituitary hormones, PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 regulate a large variety of physiological processes, among which are growth and differentiation of muscle, bone and cartilage cells, and lactation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The human pituitary hormones, PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 regulate a large variety of physiological processes, among which are growth and differentiation of muscle, bone and cartilage cells, and lactation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The human pituitary hormones, PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 regulate a large variety of physiological processes, among which are growth and differentiation of muscle, bone and cartilage cells, and lactation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 and PROTEIN1 receptors PROTEIN2 and PROTEIN3 respectively) are single-pass transmembrane receptors from class 1 of the haematopoietic receptor superfamily.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 and PROTEIN1 receptors PROTEIN2 and PROTEIN3 respectively) are single-pass transmembrane receptors from class 1 of the haematopoietic receptor superfamily.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 and PROTEIN1 receptors PROTEIN2 and PROTEIN3 respectively) are single-pass transmembrane receptors from class 1 of the haematopoietic receptor superfamily.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 and PROTEIN1 receptors PROTEIN2 and PROTEIN3 respectively) are single-pass transmembrane receptors from class 1 of the haematopoietic receptor superfamily.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 and PROTEIN1 receptors PROTEIN2 and PROTEIN3 respectively) are single-pass transmembrane receptors from class 1 of the haematopoietic receptor superfamily.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 and PROTEIN1 receptors PROTEIN2 and PROTEIN3 respectively) are single-pass transmembrane receptors from class 1 of the haematopoietic receptor superfamily.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We present here the crystal structure of the 1:1 complex of PROTEIN0 bound to the extracellular domain of the PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Comparisons with the PROTEIN0 PROTEIN1 complex reveal how PROTEIN2 can bind to the two distinctly different receptor binding surfaces.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Comparisons with the PROTEIN0 PROTEIN1 complex reveal how PROTEIN2 can bind to the two distinctly different receptor binding surfaces.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Comparisons with the PROTEIN0 PROTEIN1 complex reveal how PROTEIN2 can bind to the two distinctly different receptor binding surfaces.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoprecipitation of unactivated avian progesterone receptor results in the copurification of PROTEIN0 PROTEIN1 and three additional proteins, PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunoprecipitation of unactivated avian progesterone receptor results in the copurification of PROTEIN0 PROTEIN1 and three additional proteins, PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunoprecipitation of unactivated avian progesterone receptor results in the copurification of PROTEIN0 PROTEIN1 and three additional proteins, PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Immunoprecipitation of unactivated avian progesterone receptor results in the copurification of PROTEIN0 PROTEIN1 and three additional proteins, PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunoprecipitation of unactivated avian progesterone receptor results in the copurification of PROTEIN0 PROTEIN1 and three additional proteins, PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunoprecipitation of unactivated avian progesterone receptor results in the copurification of PROTEIN0 PROTEIN1 and three additional proteins, PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Immunoprecipitation of unactivated avian progesterone receptor results in the copurification of PROTEIN0 PROTEIN1 and three additional proteins, PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunoprecipitation of unactivated avian progesterone receptor results in the copurification of PROTEIN0 PROTEIN1 and three additional proteins, PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Immunoprecipitation of unactivated avian progesterone receptor results in the copurification of PROTEIN0 PROTEIN1 and three additional proteins, PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Immunoprecipitation of unactivated avian progesterone receptor results in the copurification of PROTEIN0 PROTEIN1 and three additional proteins, PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is also present in immunoaffinity-purified PROTEIN1 complexes along with PROTEIN2 and another protein, PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is also present in immunoaffinity-purified PROTEIN1 complexes along with PROTEIN2 and another protein, PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 is also present in immunoaffinity-purified PROTEIN1 complexes along with PROTEIN2 and another protein, PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is also present in immunoaffinity-purified PROTEIN1 complexes along with PROTEIN2 and another protein, PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 is also present in immunoaffinity-purified PROTEIN1 complexes along with PROTEIN2 and another protein, PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 is also present in immunoaffinity-purified PROTEIN1 complexes along with PROTEIN2 and another protein, PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Antibodies to PROTEIN0 detect similar levels of PROTEIN1 in all tissues tested and cross-react with a protein of the same size in mice, rabbits, guinea pigs, humans, and Saccharomyces cerevisiae, indicating that PROTEIN2 is a conserved protein of broad tissue distribution.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Antibodies to PROTEIN0 detect similar levels of PROTEIN1 in all tissues tested and cross-react with a protein of the same size in mice, rabbits, guinea pigs, humans, and Saccharomyces cerevisiae, indicating that PROTEIN2 is a conserved protein of broad tissue distribution.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Antibodies to PROTEIN0 detect similar levels of PROTEIN1 in all tissues tested and cross-react with a protein of the same size in mice, rabbits, guinea pigs, humans, and Saccharomyces cerevisiae, indicating that PROTEIN2 is a conserved protein of broad tissue distribution.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Bacterially overexpressed human PROTEIN0 was used to raise several monoclonal antibodies to PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 These antibodies specifically immunoprecipitate PROTEIN0 in complex with PROTEIN1 in all tissues tested and can be used to immunoaffinity isolate progesterone receptor complexes from chicken oviduct cytosol.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cloning of the complete coding region for human PROTEIN0 using the two hybrid system and expression of PROTEIN1 in E. coli.\n",
      "\n",
      "PROTEIN0 PROTEIN1 cDNA clones encoding a human PROTEIN0 PROTEIN1 from a Burkitt lymphoma cell line, BJAB, express a 90 to 110 kDa polypeptide in COS7 cells that binds PROTEIN2 specifically.\n",
      "\n",
      "PROTEIN0 PROTEIN2 cDNA clones encoding a human PROTEIN0 PROTEIN1 from a Burkitt lymphoma cell line, BJAB, express a 90 to 110 kDa polypeptide in COS7 cells that binds PROTEIN2 specifically.\n",
      "\n",
      "PROTEIN1 PROTEIN2 cDNA clones encoding a human PROTEIN0 PROTEIN1 from a Burkitt lymphoma cell line, BJAB, express a 90 to 110 kDa polypeptide in COS7 cells that binds PROTEIN2 specifically.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN2 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN3 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN4 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN5 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN6 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN2 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN3 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN4 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN5 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN6 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN3 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN4 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN5 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN6 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN4 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN5 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN6 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN5 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN6 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN5 PROTEIN6 The predicted amino acid sequence of PROTEIN0 is 60% identical and 73% similar to mouse PROTEIN1 PROTEIN2 PROTEIN3 expressed in an PROTEIN4 mouse pro-B cell line (Ba/F3) binds PROTEIN5 with high affinity (200 to 250 pM), and the transfected cells exhibit a proliferative response to PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mouse PROTEIN0 does not bind to PROTEIN1 and PROTEIN2 Ba/F3 cells do not respond to the mouse cytokine, observations consistent with the known species specificity of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Mouse PROTEIN0 does not bind to PROTEIN1 and PROTEIN2 Ba/F3 cells do not respond to the mouse cytokine, observations consistent with the known species specificity of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Mouse PROTEIN0 does not bind to PROTEIN1 and PROTEIN2 Ba/F3 cells do not respond to the mouse cytokine, observations consistent with the known species specificity of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Mouse PROTEIN0 does not bind to PROTEIN1 and PROTEIN2 Ba/F3 cells do not respond to the mouse cytokine, observations consistent with the known species specificity of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Mouse PROTEIN0 does not bind to PROTEIN1 and PROTEIN2 Ba/F3 cells do not respond to the mouse cytokine, observations consistent with the known species specificity of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Mouse PROTEIN0 does not bind to PROTEIN1 and PROTEIN2 Ba/F3 cells do not respond to the mouse cytokine, observations consistent with the known species specificity of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 In a number of human T cell clones, expression of PROTEIN0 mRNA is down-regulated after activation of the cells with PROTEIN1 Ab and phorbol ester.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Like PROTEIN0 PROTEIN1 is structurally related to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Like PROTEIN0 PROTEIN1 is structurally related to PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Like PROTEIN0 PROTEIN1 is structurally related to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Because PROTEIN0 inhibits macrophage activation by PROTEIN1 this relationship suggests possible shared receptor or signal transduction pathway components.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Studies with monoclonal antibodies specific for the PROTEIN0 receptor gamma chain showed that the gamma chain participates in the functional high-affinity receptor complexes for PROTEIN1 that are involved in the differentiation of T and B cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Tumor necrosis factor (TNF) and PROTEIN0 PROTEIN1 are members of a family of secreted and cell surface cytokines that participate in the regulation of immune and inflammatory responses.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The cell surface form of PROTEIN0 is assembled during biosynthesis as a heteromeric complex with PROTEIN1 PROTEIN2 a type II transmembrane protein that is another member of the TNF ligand family.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The cell surface form of PROTEIN0 is assembled during biosynthesis as a heteromeric complex with PROTEIN1 PROTEIN2 a type II transmembrane protein that is another member of the TNF ligand family.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The cell surface form of PROTEIN0 is assembled during biosynthesis as a heteromeric complex with PROTEIN1 PROTEIN2 a type II transmembrane protein that is another member of the TNF ligand family.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Secreted PROTEIN0 is a homotrimer that binds to distinct TNF receptors of 60 and 80 kilodaltons; however, these receptors do not recognize the major cell surface PROTEIN1 PROTEIN2 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Secreted PROTEIN0 is a homotrimer that binds to distinct TNF receptors of 60 and 80 kilodaltons; however, these receptors do not recognize the major cell surface PROTEIN1 PROTEIN2 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Secreted PROTEIN0 is a homotrimer that binds to distinct TNF receptors of 60 and 80 kilodaltons; however, these receptors do not recognize the major cell surface PROTEIN1 PROTEIN2 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A receptor specific for human PROTEIN0 was identified, which suggests that cell surface LT may have functions that are distinct from those of secreted PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The cytokine, designated PROTEIN0 PROTEIN1 is produced by a wide variety of cells and tissues and shares many biological properties with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The cytokine, designated PROTEIN0 PROTEIN1 is produced by a wide variety of cells and tissues and shares many biological properties with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The cytokine, designated PROTEIN0 PROTEIN1 is produced by a wide variety of cells and tissues and shares many biological properties with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The POU domains of the PROTEIN0 and PROTEIN1 transcription factors mediate specific interaction with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The POU domains of the PROTEIN0 and PROTEIN1 transcription factors mediate specific interaction with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The POU domains of the PROTEIN0 and PROTEIN1 transcription factors mediate specific interaction with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 We had previously shown that the ubiquitous PROTEIN0 and the lymphoid-specific PROTEIN1 transcription factors stimulate transcription at the level of stable preinitiation complex formation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have therefore investigated whether the octamer binding proteins might physically interact with PROTEIN0 the PROTEIN1 component of the PROTEIN2 factor.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have therefore investigated whether the octamer binding proteins might physically interact with PROTEIN0 the PROTEIN1 component of the PROTEIN2 factor.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have therefore investigated whether the octamer binding proteins might physically interact with PROTEIN0 the PROTEIN1 component of the PROTEIN2 factor.\n",
      "\n",
      "PROTEIN0 PROTEIN1 By using several different experimental systems we show that PROTEIN0 efficiently associates with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 By using several different experimental systems we show that PROTEIN0 efficiently associates with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 By using several different experimental systems we show that PROTEIN0 efficiently associates with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 We show that the POU homeodomain of PROTEIN0 and the evolutionarily conserved C-terminal core domain of PROTEIN1 are both required and sufficient for the interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 POU domain, which is highly homologous to the PROTEIN1 POU domain, likewise mediates interaction with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 POU domain, which is highly homologous to the PROTEIN1 POU domain, likewise mediates interaction with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 POU domain, which is highly homologous to the PROTEIN1 POU domain, likewise mediates interaction with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The interaction can also be observed in vivo, as PROTEIN0 can be co-precipitated with PROTEIN1 from co-transfected Cos1 cells and PROTEIN2 co-immunoprecipitates with the endogenous PROTEIN3 from HeLa cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The interaction can also be observed in vivo, as PROTEIN0 can be co-precipitated with PROTEIN1 from co-transfected Cos1 cells and PROTEIN2 co-immunoprecipitates with the endogenous PROTEIN3 from HeLa cells.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The interaction can also be observed in vivo, as PROTEIN0 can be co-precipitated with PROTEIN1 from co-transfected Cos1 cells and PROTEIN2 co-immunoprecipitates with the endogenous PROTEIN3 from HeLa cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The interaction can also be observed in vivo, as PROTEIN0 can be co-precipitated with PROTEIN1 from co-transfected Cos1 cells and PROTEIN2 co-immunoprecipitates with the endogenous PROTEIN3 from HeLa cells.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The interaction can also be observed in vivo, as PROTEIN0 can be co-precipitated with PROTEIN1 from co-transfected Cos1 cells and PROTEIN2 co-immunoprecipitates with the endogenous PROTEIN3 from HeLa cells.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The interaction can also be observed in vivo, as PROTEIN0 can be co-precipitated with PROTEIN1 from co-transfected Cos1 cells and PROTEIN2 co-immunoprecipitates with the endogenous PROTEIN3 from HeLa cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Co-transfection of human PROTEIN0 and PROTEIN1 expression vectors into B cells resulted in a synergistic activation of an octamer motif containing promoter.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interaction of mammalian splicing factor PROTEIN0 with PROTEIN1 and relation of its 60-kD subunit to yeast PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Interaction of mammalian splicing factor PROTEIN0 with PROTEIN1 and relation of its 60-kD subunit to yeast PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Interaction of mammalian splicing factor PROTEIN0 with PROTEIN1 and relation of its 60-kD subunit to yeast PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the assembly of a prespliceosome, PROTEIN0 (snRNP) functions in pre-messenger RNA (mRNA) splicing together with splicing factors (SFs) 3a, PROTEIN1 and several other proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here it is shown that the PROTEIN0 subunits correspond to three of the 17S PROTEIN1 polypeptides.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN0 PROTEIN6 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN1 PROTEIN6 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN2 PROTEIN6 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN3 PROTEIN6 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN4 PROTEIN6 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN5 PROTEIN6 PROTEIN0 interacts with PROTEIN1 in the presence of PROTEIN2 to generate a structure similar to 17S PROTEIN3 which suggests a function for PROTEIN4 and PROTEIN5 in the incorporation of PROTEIN6 into the spliceosome.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, the 60-kilodalton subunit of PROTEIN0 is related to the yeast splicing protein PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 an Ala-->Gly substitution in helix 1 of PROTEIN1 repeat 17 that severely disrupts the structure and self-association of the erythrocyte spectrin heterodimer.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The self-association site appears to be formed by interactions between helices 1 and 2 of PROTEIN0 repeat 17 of one dimer with helix 3 of PROTEIN1 repeat 1 of the other dimer to form two combined alpha-beta triple-helical segments.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Following linkage studies which were most consistent with a PROTEIN0 gene mutation, a nucleotide change was identified in codon 2018, resulting in an Ala-->Gly substitution in the first helical domain of PROTEIN1 repeat 17.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 Cdk-interacting protein PROTEIN1 is a potent inhibitor of G1 cyclin-dependent kinases.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The cyclin-dependent kinase PROTEIN0 associates with PROTEIN1 D, and E and has been implicated in the control of the G1 to S phase transition in mammals.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 encodes a novel 21 kd protein that is found in PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 immunoprecipitates.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 encodes a novel 21 kd protein that is found in PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 immunoprecipitates.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 encodes a novel 21 kd protein that is found in PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 immunoprecipitates.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 encodes a novel 21 kd protein that is found in PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 immunoprecipitates.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 encodes a novel 21 kd protein that is found in PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 immunoprecipitates.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 encodes a novel 21 kd protein that is found in PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 immunoprecipitates.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 encodes a novel 21 kd protein that is found in PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 immunoprecipitates.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 encodes a novel 21 kd protein that is found in PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 immunoprecipitates.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 encodes a novel 21 kd protein that is found in PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 immunoprecipitates.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 encodes a novel 21 kd protein that is found in PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 immunoprecipitates.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN6 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN7 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN8 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN9 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN6 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN7 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN8 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN9 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN6 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN7 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN8 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN9 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN3 PROTEIN6 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN3 PROTEIN7 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN3 PROTEIN8 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN3 PROTEIN9 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN4 PROTEIN6 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN4 PROTEIN7 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN4 PROTEIN8 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN4 PROTEIN9 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN5 PROTEIN6 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN5 PROTEIN7 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN5 PROTEIN8 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN5 PROTEIN9 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN6 PROTEIN7 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN6 PROTEIN8 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN6 PROTEIN9 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN7 PROTEIN8 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN7 PROTEIN9 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN8 PROTEIN9 PROTEIN0 is a potent, tight-binding inhibitor of Cdks and can inhibit the phosphorylation of PROTEIN1 by PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cotransfection experiments indicate that PROTEIN0 and PROTEIN1 function in a mutually antagonistic manner to control cell cycle progression.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Evidence for physical interaction between the zinc-finger transcription factors PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The recently cloned PROTEIN0 is such an PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The ability of PROTEIN0 to initiate transcription is augmented by the presence of a TATA motif or binding sites for transcription factor PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 To study the mechanism underlying the apparent functional cooperation between PROTEIN0 and PROTEIN1 we explored the possibility of protein-protein interactions between these two transcription factors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We found that PROTEIN0 and PROTEIN1 can form a physical complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, we identified domains within PROTEIN0 and PROTEIN1 that mediate their interactions with each other.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The physical interaction between PROTEIN0 and PROTEIN1 may thus form the basis for the functional interplay observed previously.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Involvement of PROTEIN0 signal transducer PROTEIN1 in PROTEIN2 signal transduction.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Involvement of PROTEIN0 signal transducer PROTEIN1 in PROTEIN2 signal transduction.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Involvement of PROTEIN0 signal transducer PROTEIN1 in PROTEIN2 signal transduction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a novel cytokine with a variety of biofunctions which overlap with those of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have previously identified PROTEIN0 specific binding protein which is distinct from that of PROTEIN1 in a number of cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The similarities in biofunctions and differences in ligand binding proteins between PROTEIN0 and PROTEIN1 prompted us to investigate whether PROTEIN2 shares common signal transduction mechanisms with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The similarities in biofunctions and differences in ligand binding proteins between PROTEIN0 and PROTEIN1 prompted us to investigate whether PROTEIN2 shares common signal transduction mechanisms with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The similarities in biofunctions and differences in ligand binding proteins between PROTEIN0 and PROTEIN1 prompted us to investigate whether PROTEIN2 shares common signal transduction mechanisms with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The similarities in biofunctions and differences in ligand binding proteins between PROTEIN0 and PROTEIN1 prompted us to investigate whether PROTEIN2 shares common signal transduction mechanisms with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The similarities in biofunctions and differences in ligand binding proteins between PROTEIN0 and PROTEIN1 prompted us to investigate whether PROTEIN2 shares common signal transduction mechanisms with PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The similarities in biofunctions and differences in ligand binding proteins between PROTEIN0 and PROTEIN1 prompted us to investigate whether PROTEIN2 shares common signal transduction mechanisms with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have examined early signals triggered by PROTEIN0 or PROTEIN1 in a multifactor-dependent human erythroleukemic cell line, TF-1.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The results showed that PROTEIN0 and PROTEIN1 can both stimulate cell proliferation, induce similar pattern of protein tyrosine phosphorylation, and activate the same proto-oncogene PROTEIN2 expression in TF-1 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The results showed that PROTEIN0 and PROTEIN1 can both stimulate cell proliferation, induce similar pattern of protein tyrosine phosphorylation, and activate the same proto-oncogene PROTEIN2 expression in TF-1 cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The results showed that PROTEIN0 and PROTEIN1 can both stimulate cell proliferation, induce similar pattern of protein tyrosine phosphorylation, and activate the same proto-oncogene PROTEIN2 expression in TF-1 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These findings imply that PROTEIN0 and PROTEIN1 share similar early signaling events with the possibility of using the same signal transducer, PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 These findings imply that PROTEIN0 and PROTEIN1 share similar early signaling events with the possibility of using the same signal transducer, PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 These findings imply that PROTEIN0 and PROTEIN1 share similar early signaling events with the possibility of using the same signal transducer, PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 We next tested whether PROTEIN0 induced signaling can be inhibited by PROTEIN1 antibodies which blocked PROTEIN2 functions.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We next tested whether PROTEIN0 induced signaling can be inhibited by PROTEIN1 antibodies which blocked PROTEIN2 functions.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We next tested whether PROTEIN0 induced signaling can be inhibited by PROTEIN1 antibodies which blocked PROTEIN2 functions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 It was observed that PROTEIN0 antibodies abolished cell proliferation, protein tyrosine phosphorylation, and PROTEIN1 gene expression elicited by PROTEIN2 or PROTEIN3 in TF-1 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 It was observed that PROTEIN0 antibodies abolished cell proliferation, protein tyrosine phosphorylation, and PROTEIN1 gene expression elicited by PROTEIN2 or PROTEIN3 in TF-1 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN3 It was observed that PROTEIN0 antibodies abolished cell proliferation, protein tyrosine phosphorylation, and PROTEIN1 gene expression elicited by PROTEIN2 or PROTEIN3 in TF-1 cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 It was observed that PROTEIN0 antibodies abolished cell proliferation, protein tyrosine phosphorylation, and PROTEIN1 gene expression elicited by PROTEIN2 or PROTEIN3 in TF-1 cells.\n",
      "\n",
      "PROTEIN1 PROTEIN3 It was observed that PROTEIN0 antibodies abolished cell proliferation, protein tyrosine phosphorylation, and PROTEIN1 gene expression elicited by PROTEIN2 or PROTEIN3 in TF-1 cells.\n",
      "\n",
      "PROTEIN2 PROTEIN3 It was observed that PROTEIN0 antibodies abolished cell proliferation, protein tyrosine phosphorylation, and PROTEIN1 gene expression elicited by PROTEIN2 or PROTEIN3 in TF-1 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Finally, PROTEIN0 antibody inhibited the ability of PROTEIN1 but not PROTEIN2 to transduce early signals in TF-1 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Finally, PROTEIN0 antibody inhibited the ability of PROTEIN1 but not PROTEIN2 to transduce early signals in TF-1 cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Finally, PROTEIN0 antibody inhibited the ability of PROTEIN1 but not PROTEIN2 to transduce early signals in TF-1 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results demonstrate that PROTEIN0 and PROTEIN1 utilize different ligand binding proteins, but share common signal transducer, PROTEIN2 in TF-1 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results demonstrate that PROTEIN0 and PROTEIN1 utilize different ligand binding proteins, but share common signal transducer, PROTEIN2 in TF-1 cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results demonstrate that PROTEIN0 and PROTEIN1 utilize different ligand binding proteins, but share common signal transducer, PROTEIN2 in TF-1 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 associates with the PROTEIN1 catalytic subunit.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 interacts with many cellular proteins in complexes potentially important for its growth-suppressing function.\n",
      "\n",
      "PROTEIN0 PROTEIN1 One clone encodes a novel PROTEIN0 catalytic subunit PROTEIN1 which differs from the originally defined PROTEIN2 by an amino-terminal 11-amino-acid insert.\n",
      "\n",
      "PROTEIN0 PROTEIN2 One clone encodes a novel PROTEIN0 catalytic subunit PROTEIN1 which differs from the originally defined PROTEIN2 by an amino-terminal 11-amino-acid insert.\n",
      "\n",
      "PROTEIN1 PROTEIN2 One clone encodes a novel PROTEIN0 catalytic subunit PROTEIN1 which differs from the originally defined PROTEIN2 by an amino-terminal 11-amino-acid insert.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In vitro-binding assays demonstrated that PROTEIN0 isoforms preferentially bind the hypophosphorylated form of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover, similar PROTEIN0 sequences are required for binding PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Moreover, similar PROTEIN0 sequences are required for binding PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Moreover, similar PROTEIN0 sequences are required for binding PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The novel fold of the TNF receptor structure is likely to be representative of the PROTEIN0 PROTEIN1 receptor family as a whole.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have previously demonstrated such changes by affinity cross-linking studies, and have shown that unstimulated human T cells express a single PROTEIN0 of 90 kDa, whereas activated T cells express an additional 76-kDa PROTEIN1 binding protein.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this study the origin and function of the PROTEIN0 and PROTEIN1 molecules have been investigated.\n",
      "\n",
      "PROTEIN0 PROTEIN1 To determine the role of each of these receptors in PROTEIN0 driven proliferation, PROTEIN1 expression and proliferative capacity were monitored during mitogenic stimulation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These analyses showed that the ability of PBMC to proliferate to PROTEIN0 correlated with expression of the PROTEIN1 PROTEIN2 and not with expression of the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 These analyses showed that the ability of PBMC to proliferate to PROTEIN0 correlated with expression of the PROTEIN1 PROTEIN2 and not with expression of the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 These analyses showed that the ability of PBMC to proliferate to PROTEIN0 correlated with expression of the PROTEIN1 PROTEIN2 and not with expression of the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 These analyses showed that the ability of PBMC to proliferate to PROTEIN0 correlated with expression of the PROTEIN1 PROTEIN2 and not with expression of the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 These analyses showed that the ability of PBMC to proliferate to PROTEIN0 correlated with expression of the PROTEIN1 PROTEIN2 and not with expression of the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 These analyses showed that the ability of PBMC to proliferate to PROTEIN0 correlated with expression of the PROTEIN1 PROTEIN2 and not with expression of the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 These analyses showed that the ability of PBMC to proliferate to PROTEIN0 correlated with expression of the PROTEIN1 PROTEIN2 and not with expression of the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 These analyses showed that the ability of PBMC to proliferate to PROTEIN0 correlated with expression of the PROTEIN1 PROTEIN2 and not with expression of the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 These analyses showed that the ability of PBMC to proliferate to PROTEIN0 correlated with expression of the PROTEIN1 PROTEIN2 and not with expression of the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 These analyses showed that the ability of PBMC to proliferate to PROTEIN0 correlated with expression of the PROTEIN1 PROTEIN2 and not with expression of the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN0 PROTEIN6 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN1 PROTEIN6 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN2 PROTEIN6 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN3 PROTEIN6 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN4 PROTEIN6 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN5 PROTEIN6 PROTEIN0 proliferation is mediated via high affinity PROTEIN1 and accordingly, Scatchard analysis revealed that, like the PROTEIN2 PROTEIN3 the PROTEIN4 PROTEIN5 bound PROTEIN6 with dual (high; Kd 38 pM and low; Kd 360 pM) affinity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Deglycosylation studies showed that the PROTEIN0 and PROTEIN1 PROTEIN2 are not simply differently glycosylated isoforms of a single receptor.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Deglycosylation studies showed that the PROTEIN0 and PROTEIN1 PROTEIN2 are not simply differently glycosylated isoforms of a single receptor.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Deglycosylation studies showed that the PROTEIN0 and PROTEIN1 PROTEIN2 are not simply differently glycosylated isoforms of a single receptor.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In agreement, mAb to the previously cloned PROTEIN0 were found to stain unstimulated T cells that express only the PROTEIN1 PROTEIN2 but not T-cell clones that express predominantly the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 In agreement, mAb to the previously cloned PROTEIN0 were found to stain unstimulated T cells that express only the PROTEIN1 PROTEIN2 but not T-cell clones that express predominantly the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 In agreement, mAb to the previously cloned PROTEIN0 were found to stain unstimulated T cells that express only the PROTEIN1 PROTEIN2 but not T-cell clones that express predominantly the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 In agreement, mAb to the previously cloned PROTEIN0 were found to stain unstimulated T cells that express only the PROTEIN1 PROTEIN2 but not T-cell clones that express predominantly the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 In agreement, mAb to the previously cloned PROTEIN0 were found to stain unstimulated T cells that express only the PROTEIN1 PROTEIN2 but not T-cell clones that express predominantly the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 In agreement, mAb to the previously cloned PROTEIN0 were found to stain unstimulated T cells that express only the PROTEIN1 PROTEIN2 but not T-cell clones that express predominantly the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 In agreement, mAb to the previously cloned PROTEIN0 were found to stain unstimulated T cells that express only the PROTEIN1 PROTEIN2 but not T-cell clones that express predominantly the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 In agreement, mAb to the previously cloned PROTEIN0 were found to stain unstimulated T cells that express only the PROTEIN1 PROTEIN2 but not T-cell clones that express predominantly the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 In agreement, mAb to the previously cloned PROTEIN0 were found to stain unstimulated T cells that express only the PROTEIN1 PROTEIN2 but not T-cell clones that express predominantly the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 In agreement, mAb to the previously cloned PROTEIN0 were found to stain unstimulated T cells that express only the PROTEIN1 PROTEIN2 but not T-cell clones that express predominantly the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 These antibodies also immunoprecipitated the cloned PROTEIN0 as a 90-kDa species from both 125-I-surface-labeled resting and activated T cells, but were unable to precipitate the 76-kDa PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, PCR analysis of PROTEIN0 cells could not detect splicing of the extracellular domain of the cloned PROTEIN1 thereby excluding the possibility that the PROTEIN2 PROTEIN3 is a previously undescribed splice variant of the cloned PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 In addition, PCR analysis of PROTEIN0 cells could not detect splicing of the extracellular domain of the cloned PROTEIN1 thereby excluding the possibility that the PROTEIN2 PROTEIN3 is a previously undescribed splice variant of the cloned PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 In addition, PCR analysis of PROTEIN0 cells could not detect splicing of the extracellular domain of the cloned PROTEIN1 thereby excluding the possibility that the PROTEIN2 PROTEIN3 is a previously undescribed splice variant of the cloned PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 In addition, PCR analysis of PROTEIN0 cells could not detect splicing of the extracellular domain of the cloned PROTEIN1 thereby excluding the possibility that the PROTEIN2 PROTEIN3 is a previously undescribed splice variant of the cloned PROTEIN4\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PROTEIN1 PROTEIN2 In addition, PCR analysis of PROTEIN0 cells could not detect splicing of the extracellular domain of the cloned PROTEIN1 thereby excluding the possibility that the PROTEIN2 PROTEIN3 is a previously undescribed splice variant of the cloned PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 In addition, PCR analysis of PROTEIN0 cells could not detect splicing of the extracellular domain of the cloned PROTEIN1 thereby excluding the possibility that the PROTEIN2 PROTEIN3 is a previously undescribed splice variant of the cloned PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 In addition, PCR analysis of PROTEIN0 cells could not detect splicing of the extracellular domain of the cloned PROTEIN1 thereby excluding the possibility that the PROTEIN2 PROTEIN3 is a previously undescribed splice variant of the cloned PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 In addition, PCR analysis of PROTEIN0 cells could not detect splicing of the extracellular domain of the cloned PROTEIN1 thereby excluding the possibility that the PROTEIN2 PROTEIN3 is a previously undescribed splice variant of the cloned PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 In addition, PCR analysis of PROTEIN0 cells could not detect splicing of the extracellular domain of the cloned PROTEIN1 thereby excluding the possibility that the PROTEIN2 PROTEIN3 is a previously undescribed splice variant of the cloned PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 In addition, PCR analysis of PROTEIN0 cells could not detect splicing of the extracellular domain of the cloned PROTEIN1 thereby excluding the possibility that the PROTEIN2 PROTEIN3 is a previously undescribed splice variant of the cloned PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN0 PROTEIN3 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN0 PROTEIN4 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN0 PROTEIN5 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN1 PROTEIN3 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN1 PROTEIN4 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN1 PROTEIN5 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN2 PROTEIN3 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN2 PROTEIN4 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN2 PROTEIN5 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN3 PROTEIN4 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN3 PROTEIN5 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN4 PROTEIN5 These data demonstrate that the PROTEIN0 PROTEIN1 is the T-cell homologue of the cloned PROTEIN2 and imply that the PROTEIN3 and PROTEIN4 PROTEIN5 have different extracellular domains.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Taken together these data suggest that the 76-kDa receptor is a novel high affinity PROTEIN0 that may be necessary for PROTEIN1 driven proliferation in human T cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Characterization of the PROTEIN0 binding proteins PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Characterization of the PROTEIN0 binding proteins PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Characterization of the PROTEIN0 binding proteins PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 gene product, PROTEIN1 regulates cell proliferation by binding to and inhibiting the activity of key growth promoting proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have now isolated cDNA clones that contain the full-length coding sequence of two other proteins, PROTEIN0 and PROTEIN1 cloned originally by their interaction with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have now isolated cDNA clones that contain the full-length coding sequence of two other proteins, PROTEIN0 and PROTEIN1 cloned originally by their interaction with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have now isolated cDNA clones that contain the full-length coding sequence of two other proteins, PROTEIN0 and PROTEIN1 cloned originally by their interaction with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The products of the PROTEIN0 and PROTEIN1 genes are ubiquitously expressed, large (200 kDa for PROTEIN2 and 195 kDa for PROTEIN3 nuclear phosphoproteins with structural motifs that suggest a role in transcriptional regulation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The products of the PROTEIN0 and PROTEIN1 genes are ubiquitously expressed, large (200 kDa for PROTEIN2 and 195 kDa for PROTEIN3 nuclear phosphoproteins with structural motifs that suggest a role in transcriptional regulation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The products of the PROTEIN0 and PROTEIN1 genes are ubiquitously expressed, large (200 kDa for PROTEIN2 and 195 kDa for PROTEIN3 nuclear phosphoproteins with structural motifs that suggest a role in transcriptional regulation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The products of the PROTEIN0 and PROTEIN1 genes are ubiquitously expressed, large (200 kDa for PROTEIN2 and 195 kDa for PROTEIN3 nuclear phosphoproteins with structural motifs that suggest a role in transcriptional regulation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The products of the PROTEIN0 and PROTEIN1 genes are ubiquitously expressed, large (200 kDa for PROTEIN2 and 195 kDa for PROTEIN3 nuclear phosphoproteins with structural motifs that suggest a role in transcriptional regulation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The products of the PROTEIN0 and PROTEIN1 genes are ubiquitously expressed, large (200 kDa for PROTEIN2 and 195 kDa for PROTEIN3 nuclear phosphoproteins with structural motifs that suggest a role in transcriptional regulation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition we have been able to identify complexes of PROTEIN0 and PROTEIN1 in vivo that are dissociated in the presence of purified human papillomavirus PROTEIN2 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In addition we have been able to identify complexes of PROTEIN0 and PROTEIN1 in vivo that are dissociated in the presence of purified human papillomavirus PROTEIN2 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In addition we have been able to identify complexes of PROTEIN0 and PROTEIN1 in vivo that are dissociated in the presence of purified human papillomavirus PROTEIN2 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Two potentially oncogenic cyclins, PROTEIN0 and PROTEIN1 share common properties of subunit configuration, tyrosine phosphorylation and physical association with the PROTEIN2 protein\n",
      "\n",
      "PROTEIN0 PROTEIN2 Two potentially oncogenic cyclins, PROTEIN0 and PROTEIN1 share common properties of subunit configuration, tyrosine phosphorylation and physical association with the PROTEIN2 protein\n",
      "\n",
      "PROTEIN1 PROTEIN2 Two potentially oncogenic cyclins, PROTEIN0 and PROTEIN1 share common properties of subunit configuration, tyrosine phosphorylation and physical association with the PROTEIN2 protein\n",
      "\n",
      "PROTEIN0 PROTEIN1 .Originally identified as a 'mitotic cyclin', PROTEIN0 exhibits properties of growth factor sensitivity, susceptibility to viral subversion and association with a tumor-suppressor protein, properties which are indicative of an PROTEIN1 PROTEIN2 as well as a candidate proto-oncogene.\n",
      "\n",
      "PROTEIN0 PROTEIN2 .Originally identified as a 'mitotic cyclin', PROTEIN0 exhibits properties of growth factor sensitivity, susceptibility to viral subversion and association with a tumor-suppressor protein, properties which are indicative of an PROTEIN1 PROTEIN2 as well as a candidate proto-oncogene.\n",
      "\n",
      "PROTEIN1 PROTEIN2 .Originally identified as a 'mitotic cyclin', PROTEIN0 exhibits properties of growth factor sensitivity, susceptibility to viral subversion and association with a tumor-suppressor protein, properties which are indicative of an PROTEIN1 PROTEIN2 as well as a candidate proto-oncogene.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Other recent studies have identified human PROTEIN0 PROTEIN1 as a putative G1 cyclin and candidate proto-oncogene.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover, PROTEIN0 was found to be phosphorylated on tyrosine residues in vivo and, like PROTEIN1 was readily phosphorylated by PROTEIN2 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Moreover, PROTEIN0 was found to be phosphorylated on tyrosine residues in vivo and, like PROTEIN1 was readily phosphorylated by PROTEIN2 in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Moreover, PROTEIN0 was found to be phosphorylated on tyrosine residues in vivo and, like PROTEIN1 was readily phosphorylated by PROTEIN2 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In synchronized human osteosarcoma cells, PROTEIN0 is induced in early G1 and becomes associated with PROTEIN1 a Cdk-binding subunit.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoprecipitation experiments with human osteosarcoma cells and Ewing's sarcoma cells demonstrated that PROTEIN0 is associated with both PROTEIN1 and PROTEIN2 and that PROTEIN3 immune complexes exhibit appreciable histone H1 kinase activity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunoprecipitation experiments with human osteosarcoma cells and Ewing's sarcoma cells demonstrated that PROTEIN0 is associated with both PROTEIN1 and PROTEIN2 and that PROTEIN3 immune complexes exhibit appreciable histone H1 kinase activity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunoprecipitation experiments with human osteosarcoma cells and Ewing's sarcoma cells demonstrated that PROTEIN0 is associated with both PROTEIN1 and PROTEIN2 and that PROTEIN3 immune complexes exhibit appreciable histone H1 kinase activity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunoprecipitation experiments with human osteosarcoma cells and Ewing's sarcoma cells demonstrated that PROTEIN0 is associated with both PROTEIN1 and PROTEIN2 and that PROTEIN3 immune complexes exhibit appreciable histone H1 kinase activity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunoprecipitation experiments with human osteosarcoma cells and Ewing's sarcoma cells demonstrated that PROTEIN0 is associated with both PROTEIN1 and PROTEIN2 and that PROTEIN3 immune complexes exhibit appreciable histone H1 kinase activity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunoprecipitation experiments with human osteosarcoma cells and Ewing's sarcoma cells demonstrated that PROTEIN0 is associated with both PROTEIN1 and PROTEIN2 and that PROTEIN3 immune complexes exhibit appreciable histone H1 kinase activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Regulation of the PROTEIN0 transcription factor PROTEIN1 by binding to the PROTEIN2 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Regulation of the PROTEIN0 transcription factor PROTEIN1 by binding to the PROTEIN2 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Regulation of the PROTEIN0 transcription factor PROTEIN1 by binding to the PROTEIN2 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 gene product PROTEIN1 is a nuclear phosphoprotein that regulates cell cycle progression.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a lymphoid-specific PROTEIN1 transcription factor that regulates inducible gene expression during T cell activation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this report, it is demonstrated that PROTEIN0 contains a sequence motif that is highly related to the PROTEIN1 binding sites of several viral oncoproteins and binds to the pocket region of PROTEIN2 both in vitro and in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this report, it is demonstrated that PROTEIN0 contains a sequence motif that is highly related to the PROTEIN1 binding sites of several viral oncoproteins and binds to the pocket region of PROTEIN2 both in vitro and in vivo.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this report, it is demonstrated that PROTEIN0 contains a sequence motif that is highly related to the PROTEIN1 binding sites of several viral oncoproteins and binds to the pocket region of PROTEIN2 both in vitro and in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binds exclusively to the underphosphorylated form of PROTEIN1 and fails to bind to PROTEIN2 mutants derived from patients with retinoblastoma.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 binds exclusively to the underphosphorylated form of PROTEIN1 and fails to bind to PROTEIN2 mutants derived from patients with retinoblastoma.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 binds exclusively to the underphosphorylated form of PROTEIN1 and fails to bind to PROTEIN2 mutants derived from patients with retinoblastoma.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Co-immunoprecipitation experiments demonstrated an association between PROTEIN0 and PROTEIN1 in resting normal human T cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 After T cell activation, the phosphorylation of PROTEIN0 results in the release of PROTEIN1 which is correlated temporally with the activation of PROTEIN2 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN2 After T cell activation, the phosphorylation of PROTEIN0 results in the release of PROTEIN1 which is correlated temporally with the activation of PROTEIN2 transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN2 After T cell activation, the phosphorylation of PROTEIN0 results in the release of PROTEIN1 which is correlated temporally with the activation of PROTEIN2 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Overexpression of a phosphorylation-defective form of PROTEIN0 inhibited PROTEIN1 transcription during T cell activation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results demonstrate that PROTEIN0 interacts specifically with a lineage-restricted PROTEIN1 transcription factor.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 specifically recognizes the PSEs of both RNA polymerase II- and RNA polymerase III-transcribed small nuclear RNA (snRNA) genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We now report the isolation and expression of cDNAs encoding PROTEIN0 gamma and PROTEIN1 delta, as well as functional studies with cognate antibodies that recognize the native PROTEIN2 complex in HeLa extracts.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We now report the isolation and expression of cDNAs encoding PROTEIN0 gamma and PROTEIN1 delta, as well as functional studies with cognate antibodies that recognize the native PROTEIN2 complex in HeLa extracts.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We now report the isolation and expression of cDNAs encoding PROTEIN0 gamma and PROTEIN1 delta, as well as functional studies with cognate antibodies that recognize the native PROTEIN2 complex in HeLa extracts.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN0 PROTEIN7 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN1 PROTEIN7 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN2 PROTEIN7 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN3 PROTEIN7 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN4 PROTEIN7 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN5 PROTEIN7 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN6 PROTEIN7 Immunoprecipitation assays also show a weak substoichiometric association of the PROTEIN0 PROTEIN1 with PROTEIN2 consistent with the previous report of a PROTEIN3 complex PROTEIN4 containing substoichiometric levels of PROTEIN5 and a component PROTEIN6 identical in sequence to the PROTEIN7 gamma reported here.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 interactions define two distinct PROTEIN4 signal transduction pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 interactions define two distinct PROTEIN4 signal transduction pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 interactions define two distinct PROTEIN4 signal transduction pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 interactions define two distinct PROTEIN4 signal transduction pathways.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 interactions define two distinct PROTEIN4 signal transduction pathways.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 interactions define two distinct PROTEIN4 signal transduction pathways.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 interactions define two distinct PROTEIN4 signal transduction pathways.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 interactions define two distinct PROTEIN4 signal transduction pathways.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 interactions define two distinct PROTEIN4 signal transduction pathways.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 interactions define two distinct PROTEIN4 signal transduction pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Tumor necrosis factor (TNF) can induce apoptosis and activate PROTEIN0 through signaling cascades emanating from PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Tumor necrosis factor (TNF) can induce apoptosis and activate PROTEIN0 through signaling cascades emanating from PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Tumor necrosis factor (TNF) can induce apoptosis and activate PROTEIN0 through signaling cascades emanating from PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a PROTEIN1 signal transducer that is involved in activating both pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we show that PROTEIN0 directly interacts with PROTEIN1 and PROTEIN2 signal transducers that activate PROTEIN3 and induce apoptosis, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we show that PROTEIN0 directly interacts with PROTEIN1 and PROTEIN2 signal transducers that activate PROTEIN3 and induce apoptosis, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here we show that PROTEIN0 directly interacts with PROTEIN1 and PROTEIN2 signal transducers that activate PROTEIN3 and induce apoptosis, respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we show that PROTEIN0 directly interacts with PROTEIN1 and PROTEIN2 signal transducers that activate PROTEIN3 and induce apoptosis, respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here we show that PROTEIN0 directly interacts with PROTEIN1 and PROTEIN2 signal transducers that activate PROTEIN3 and induce apoptosis, respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here we show that PROTEIN0 directly interacts with PROTEIN1 and PROTEIN2 signal transducers that activate PROTEIN3 and induce apoptosis, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A PROTEIN0 mutant lacking its N-terminal RING finger domain is a dominant-negative inhibitor of TNF-mediated PROTEIN1 activation, but does not affect TNF-induced apoptosis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Conversely, a PROTEIN0 mutant lacking its N-terminal 79 amino acids is a dominant-negative inhibitor of TNF-induced apoptosis, but does not inhibit PROTEIN1 activation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, these two PROTEIN0 PROTEIN1 signaling cascades appear to bifurcate at PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, these two PROTEIN0 PROTEIN1 signaling cascades appear to bifurcate at PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, these two PROTEIN0 PROTEIN1 signaling cascades appear to bifurcate at PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Characterization of PROTEIN0 binding to human PROTEIN1 on T cells and its biologic role in immune response.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 a T cell activation Ag, also known as PROTEIN1 directly associated with PROTEIN2 PROTEIN3 on the surface of T cells and T cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 a T cell activation Ag, also known as PROTEIN1 directly associated with PROTEIN2 PROTEIN3 on the surface of T cells and T cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 a T cell activation Ag, also known as PROTEIN1 directly associated with PROTEIN2 PROTEIN3 on the surface of T cells and T cell lines.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 a T cell activation Ag, also known as PROTEIN1 directly associated with PROTEIN2 PROTEIN3 on the surface of T cells and T cell lines.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 a T cell activation Ag, also known as PROTEIN1 directly associated with PROTEIN2 PROTEIN3 on the surface of T cells and T cell lines.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 a T cell activation Ag, also known as PROTEIN1 directly associated with PROTEIN2 PROTEIN3 on the surface of T cells and T cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the present study, we examined both the binding of PROTEIN0 and PROTEIN1 and the functional consequences of this interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN0 PROTEIN5 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN1 PROTEIN5 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN2 PROTEIN5 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN3 PROTEIN5 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN4 PROTEIN5 We found that PROTEIN0 was associated with PROTEIN1 on T cell lines lacking either PROTEIN2 or PROTEIN3 enzymatic activity, indicating that the association between PROTEIN4 and PROTEIN5 did not require enzymatic activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover, using immunoelectron microscopy, we demonstrated that PROTEIN0 and PROTEIN1 co-localized on the cell surface, but not inside cells, suggesting that PROTEIN2 did not transport PROTEIN3 to the surface.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Moreover, using immunoelectron microscopy, we demonstrated that PROTEIN0 and PROTEIN1 co-localized on the cell surface, but not inside cells, suggesting that PROTEIN2 did not transport PROTEIN3 to the surface.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Moreover, using immunoelectron microscopy, we demonstrated that PROTEIN0 and PROTEIN1 co-localized on the cell surface, but not inside cells, suggesting that PROTEIN2 did not transport PROTEIN3 to the surface.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Moreover, using immunoelectron microscopy, we demonstrated that PROTEIN0 and PROTEIN1 co-localized on the cell surface, but not inside cells, suggesting that PROTEIN2 did not transport PROTEIN3 to the surface.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Moreover, using immunoelectron microscopy, we demonstrated that PROTEIN0 and PROTEIN1 co-localized on the cell surface, but not inside cells, suggesting that PROTEIN2 did not transport PROTEIN3 to the surface.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Moreover, using immunoelectron microscopy, we demonstrated that PROTEIN0 and PROTEIN1 co-localized on the cell surface, but not inside cells, suggesting that PROTEIN2 did not transport PROTEIN3 to the surface.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In keeping with this observation, we showed that human PROTEIN0 murine pre-B cell lines lacking human PROTEIN1 acquired PROTEIN2 from an extracellular source.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In keeping with this observation, we showed that human PROTEIN0 murine pre-B cell lines lacking human PROTEIN1 acquired PROTEIN2 from an extracellular source.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In keeping with this observation, we showed that human PROTEIN0 murine pre-B cell lines lacking human PROTEIN1 acquired PROTEIN2 from an extracellular source.\n",
      "\n",
      "PROTEIN0 PROTEIN1 More importantly, adenosine in the absence of cell surface PROTEIN0 inhibited T cell proliferation and PROTEIN1 production induced by various stimuli.\n",
      "\n",
      "PROTEIN0 PROTEIN1 On the other hand, cells expressing PROTEIN0 and PROTEIN1 on the surface were much more resistant to the inhibitory effect of adenosine.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These data suggest that PROTEIN0 on the cell surface is involved in an important immunoregulatory mechanism by which released PROTEIN1 binds to cell surface PROTEIN2 and this complex is capable of reducing the local concentration of adenosine.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These data suggest that PROTEIN0 on the cell surface is involved in an important immunoregulatory mechanism by which released PROTEIN1 binds to cell surface PROTEIN2 and this complex is capable of reducing the local concentration of adenosine.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These data suggest that PROTEIN0 on the cell surface is involved in an important immunoregulatory mechanism by which released PROTEIN1 binds to cell surface PROTEIN2 and this complex is capable of reducing the local concentration of adenosine.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have cloned and characterized human PROTEIN0 and a second isoform of human PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Somatic cell hybrids and fluorescent in situ hybridization were both used to determine their chromosomal locations: PROTEIN0 to chromosome 16q22-23 and PROTEIN1 to chromosome 20q11.2.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 Similarly, in mammalian cells the cyclic adenosine 3',5'-monophosphate-dependent protein kinase PROTEIN0 and PROTEIN1 PROTEIN2 PROTEIN3 are complexed by an PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Deletion analysis and binding studies demonstrate that a third enzyme, PROTEIN0 PROTEIN1 binds PROTEIN2 at a site distinct from those bound by PROTEIN3 or PROTEIN4\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PROTEIN0 PROTEIN2 Deletion analysis and binding studies demonstrate that a third enzyme, PROTEIN0 PROTEIN1 binds PROTEIN2 at a site distinct from those bound by PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Deletion analysis and binding studies demonstrate that a third enzyme, PROTEIN0 PROTEIN1 binds PROTEIN2 at a site distinct from those bound by PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Deletion analysis and binding studies demonstrate that a third enzyme, PROTEIN0 PROTEIN1 binds PROTEIN2 at a site distinct from those bound by PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Deletion analysis and binding studies demonstrate that a third enzyme, PROTEIN0 PROTEIN1 binds PROTEIN2 at a site distinct from those bound by PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Deletion analysis and binding studies demonstrate that a third enzyme, PROTEIN0 PROTEIN1 binds PROTEIN2 at a site distinct from those bound by PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Deletion analysis and binding studies demonstrate that a third enzyme, PROTEIN0 PROTEIN1 binds PROTEIN2 at a site distinct from those bound by PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Deletion analysis and binding studies demonstrate that a third enzyme, PROTEIN0 PROTEIN1 binds PROTEIN2 at a site distinct from those bound by PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Deletion analysis and binding studies demonstrate that a third enzyme, PROTEIN0 PROTEIN1 binds PROTEIN2 at a site distinct from those bound by PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Deletion analysis and binding studies demonstrate that a third enzyme, PROTEIN0 PROTEIN1 binds PROTEIN2 at a site distinct from those bound by PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 The subcellular distributions of PROTEIN0 and PROTEIN1 were similar in neurons.\n",
      "\n",
      "PROTEIN0 PROTEIN1 TNF-dependent recruitment of the protein kinase PROTEIN0 to the PROTEIN1 signaling complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The death domain of PROTEIN0 PROTEIN1 triggers distinct signaling pathways leading to apoptosis and PROTEIN2 activation through its interaction with the death domain protein PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The death domain of PROTEIN0 PROTEIN1 triggers distinct signaling pathways leading to apoptosis and PROTEIN2 activation through its interaction with the death domain protein PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The death domain of PROTEIN0 PROTEIN1 triggers distinct signaling pathways leading to apoptosis and PROTEIN2 activation through its interaction with the death domain protein PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The death domain of PROTEIN0 PROTEIN1 triggers distinct signaling pathways leading to apoptosis and PROTEIN2 activation through its interaction with the death domain protein PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The death domain of PROTEIN0 PROTEIN1 triggers distinct signaling pathways leading to apoptosis and PROTEIN2 activation through its interaction with the death domain protein PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The death domain of PROTEIN0 PROTEIN1 triggers distinct signaling pathways leading to apoptosis and PROTEIN2 activation through its interaction with the death domain protein PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, we show that PROTEIN0 interacts strongly with PROTEIN1 another death domain protein that was shown previously to associate with PROTEIN2 antigen.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here, we show that PROTEIN0 interacts strongly with PROTEIN1 another death domain protein that was shown previously to associate with PROTEIN2 antigen.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here, we show that PROTEIN0 interacts strongly with PROTEIN1 another death domain protein that was shown previously to associate with PROTEIN2 antigen.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We also show that PROTEIN0 is a serine-threonine kinase that is recruited by PROTEIN1 to PROTEIN2 in a TNF-dependent process.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We also show that PROTEIN0 is a serine-threonine kinase that is recruited by PROTEIN1 to PROTEIN2 in a TNF-dependent process.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We also show that PROTEIN0 is a serine-threonine kinase that is recruited by PROTEIN1 to PROTEIN2 in a TNF-dependent process.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Overexpression of the intact PROTEIN0 protein induces both PROTEIN1 activation and apoptosis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, expression of the death domain of PROTEIN0 Induces apoptosis, but potently inhibits PROTEIN1 activation by TNF.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that distinct domains of PROTEIN0 participate in the TNF signaling cascades leading to apoptosis and PROTEIN1 activation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The carboxy-terminal region of PROTEIN0 has previously been shown to interact directly with PROTEIN1 a cytoplasmic component of the PROTEIN2 complex, by in vitro biochemical studies such as overlay assay or immunoprecipitation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The carboxy-terminal region of PROTEIN0 has previously been shown to interact directly with PROTEIN1 a cytoplasmic component of the PROTEIN2 complex, by in vitro biochemical studies such as overlay assay or immunoprecipitation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The carboxy-terminal region of PROTEIN0 has previously been shown to interact directly with PROTEIN1 a cytoplasmic component of the PROTEIN2 complex, by in vitro biochemical studies such as overlay assay or immunoprecipitation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Identification of vascular endothelial growth factor determinants for binding PROTEIN0 and PROTEIN1 receptors. Generation of receptor-selective PROTEIN2 variants by site-directed mutagenesis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Identification of vascular endothelial growth factor determinants for binding PROTEIN0 and PROTEIN1 receptors. Generation of receptor-selective PROTEIN2 variants by site-directed mutagenesis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Identification of vascular endothelial growth factor determinants for binding PROTEIN0 and PROTEIN1 receptors. Generation of receptor-selective PROTEIN2 variants by site-directed mutagenesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 expression in various cell types is induced by hypoxia and other stimuli.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Alanine-scanning mutagenesis was used to identify a positively charged surface in PROTEIN0 that mediates binding to PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Alanine-scanning mutagenesis was used to identify a positively charged surface in PROTEIN0 that mediates binding to PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Alanine-scanning mutagenesis was used to identify a positively charged surface in PROTEIN0 that mediates binding to PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Arg82, Lys84 and His86, located in a hairpin loop, were found to be critical for binding PROTEIN0 PROTEIN1 while negatively charged residues, Asp63, Glu64, and Glu67, were associated with PROTEIN2 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Arg82, Lys84 and His86, located in a hairpin loop, were found to be critical for binding PROTEIN0 PROTEIN1 while negatively charged residues, Asp63, Glu64, and Glu67, were associated with PROTEIN2 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Arg82, Lys84 and His86, located in a hairpin loop, were found to be critical for binding PROTEIN0 PROTEIN1 while negatively charged residues, Asp63, Glu64, and Glu67, were associated with PROTEIN2 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A PROTEIN0 model based on PROTEIN1 indicated these positively and negatively charged regions are distal in the monomer but are spatially close in the dimer.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mutations within the PROTEIN0 site had minimal effect on PROTEIN1 binding, and mutants deficient in PROTEIN2 binding did not affect PROTEIN3 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Mutations within the PROTEIN0 site had minimal effect on PROTEIN1 binding, and mutants deficient in PROTEIN2 binding did not affect PROTEIN3 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Mutations within the PROTEIN0 site had minimal effect on PROTEIN1 binding, and mutants deficient in PROTEIN2 binding did not affect PROTEIN3 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Mutations within the PROTEIN0 site had minimal effect on PROTEIN1 binding, and mutants deficient in PROTEIN2 binding did not affect PROTEIN3 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Mutations within the PROTEIN0 site had minimal effect on PROTEIN1 binding, and mutants deficient in PROTEIN2 binding did not affect PROTEIN3 binding.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Mutations within the PROTEIN0 site had minimal effect on PROTEIN1 binding, and mutants deficient in PROTEIN2 binding did not affect PROTEIN3 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Endothelial cell mitogenesis was abolished in mutants lacking PROTEIN0 affinity; however, PROTEIN1 deficient mutants induced normal proliferation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, this mutational analysis implicates PROTEIN0 but not PROTEIN1 in PROTEIN2 induction of endothelial cell proliferation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Furthermore, this mutational analysis implicates PROTEIN0 but not PROTEIN1 in PROTEIN2 induction of endothelial cell proliferation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Furthermore, this mutational analysis implicates PROTEIN0 but not PROTEIN1 in PROTEIN2 induction of endothelial cell proliferation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Use of the two hybrid system to detect the association of the protein-tyrosine-phosphatase, PROTEIN0 with another SH2-containing protein, PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using PROTEIN0 as bait and supplying an exogenous tyrosine kinase gene to the yeast cells, we have found that PROTEIN1 interacts with another signaling protein, PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using PROTEIN0 as bait and supplying an exogenous tyrosine kinase gene to the yeast cells, we have found that PROTEIN1 interacts with another signaling protein, PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using PROTEIN0 as bait and supplying an exogenous tyrosine kinase gene to the yeast cells, we have found that PROTEIN1 interacts with another signaling protein, PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have localized the region of interaction to tyrosine 580 in the carboxyl end of PROTEIN0 and to the SH2 domain in the carboxy-terminus of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 We demonstrate that PROTEIN0 binds to PROTEIN1 in vitro under conditions where the latter is tyrosine-phosphorylated.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We found that the human PROTEIN0 PROTEIN1 physically interacted with PROTEIN2 via its DNA binding domain.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We found that the human PROTEIN0 PROTEIN1 physically interacted with PROTEIN2 via its DNA binding domain.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We found that the human PROTEIN0 PROTEIN1 physically interacted with PROTEIN2 via its DNA binding domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 As a result of this physical interaction, binding of PROTEIN0 to its hormone response elements either as homodimer or as a heterodimer with the PROTEIN1 was inhibited by PROTEIN2 in a concentration-dependent manner.\n",
      "\n",
      "PROTEIN0 PROTEIN2 As a result of this physical interaction, binding of PROTEIN0 to its hormone response elements either as homodimer or as a heterodimer with the PROTEIN1 was inhibited by PROTEIN2 in a concentration-dependent manner.\n",
      "\n",
      "PROTEIN1 PROTEIN2 As a result of this physical interaction, binding of PROTEIN0 to its hormone response elements either as homodimer or as a heterodimer with the PROTEIN1 was inhibited by PROTEIN2 in a concentration-dependent manner.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In transfected cells, wild-type PROTEIN0 repressed the hormone-dependent transcriptional activation of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, mutant PROTEIN0 either had no effect or activated the transcriptional activity of PROTEIN1 depending on the type of hormone response elements.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Enhanced degradation of PROTEIN0 receptors by a sorting PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Enhanced degradation of PROTEIN0 receptors by a sorting PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Enhanced degradation of PROTEIN0 receptors by a sorting PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 A protein, PROTEIN0 PROTEIN1 was identified in a human cell line that bound to a region of the PROTEIN2 PROTEIN3 containing the lysosomal targeting code.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A protein, PROTEIN0 PROTEIN1 was identified in a human cell line that bound to a region of the PROTEIN2 PROTEIN3 containing the lysosomal targeting code.\n",
      "\n",
      "PROTEIN0 PROTEIN3 A protein, PROTEIN0 PROTEIN1 was identified in a human cell line that bound to a region of the PROTEIN2 PROTEIN3 containing the lysosomal targeting code.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A protein, PROTEIN0 PROTEIN1 was identified in a human cell line that bound to a region of the PROTEIN2 PROTEIN3 containing the lysosomal targeting code.\n",
      "\n",
      "PROTEIN1 PROTEIN3 A protein, PROTEIN0 PROTEIN1 was identified in a human cell line that bound to a region of the PROTEIN2 PROTEIN3 containing the lysosomal targeting code.\n",
      "\n",
      "PROTEIN2 PROTEIN3 A protein, PROTEIN0 PROTEIN1 was identified in a human cell line that bound to a region of the PROTEIN2 PROTEIN3 containing the lysosomal targeting code.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 contains a region of homology to a yeast vacuolar sorting protein, and overexpression of PROTEIN1 decreased the amount of PROTEIN2 on the cell surface as a result of enhanced rates of constitutive and ligand-induced degradation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 contains a region of homology to a yeast vacuolar sorting protein, and overexpression of PROTEIN1 decreased the amount of PROTEIN2 on the cell surface as a result of enhanced rates of constitutive and ligand-induced degradation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 contains a region of homology to a yeast vacuolar sorting protein, and overexpression of PROTEIN1 decreased the amount of PROTEIN2 on the cell surface as a result of enhanced rates of constitutive and ligand-induced degradation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, PROTEIN0 is likely to play a role in sorting PROTEIN1 to lysosomes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The splicing factor U2AF (U2 snRNP auxiliary factor) is a heterodimer with subunits of 65 and 35 kD PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binds specifically to 3' splice sites, but previous studies failed to demonstrate a function for PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Nuclear extracts deficient in PROTEIN0 were inactive; however, both constitutive and enhancer-dependent splicing could be restored by the addition of purified recombinant PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In vitro protein-RNA interaction studies with pre-mRNAs containing either a constitutive or regulated splicing enhancer revealed that PROTEIN0 directly mediates interactions between PROTEIN1 and proteins bound to the enhancers.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, PROTEIN0 functions as a bridge between PROTEIN1 and the enhancer complex to recruit PROTEIN2 to the adjacent intron.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, PROTEIN0 functions as a bridge between PROTEIN1 and the enhancer complex to recruit PROTEIN2 to the adjacent intron.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, PROTEIN0 functions as a bridge between PROTEIN1 and the enhancer complex to recruit PROTEIN2 to the adjacent intron.\n",
      "\n",
      "PROTEIN0 PROTEIN1 At least two of the subunits of PROTEIN0 PROTEIN1 and PROTEIN2 proteins) are implicated in the disease xeroderma pigmentosum (XP).\n",
      "\n",
      "PROTEIN0 PROTEIN2 At least two of the subunits of PROTEIN0 PROTEIN1 and PROTEIN2 proteins) are implicated in the disease xeroderma pigmentosum (XP).\n",
      "\n",
      "PROTEIN1 PROTEIN2 At least two of the subunits of PROTEIN0 PROTEIN1 and PROTEIN2 proteins) are implicated in the disease xeroderma pigmentosum (XP).\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN0 PROTEIN5 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN1 PROTEIN5 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN2 PROTEIN5 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN3 PROTEIN5 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN4 PROTEIN5 We have exploited the availability of the cloned PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 genes (all of which encode polypeptide subunits of PROTEIN5 to examine interactions between in vitro-translated polypeptides by co-immunoprecipitation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Additionally we have examined interactions between PROTEIN0 components, the human NER protein PROTEIN1 and the PROTEIN2 protein which is implicated in Cockayne syndrome (CS).\n",
      "\n",
      "PROTEIN0 PROTEIN2 Additionally we have examined interactions between PROTEIN0 components, the human NER protein PROTEIN1 and the PROTEIN2 protein which is implicated in Cockayne syndrome (CS).\n",
      "\n",
      "PROTEIN1 PROTEIN2 Additionally we have examined interactions between PROTEIN0 components, the human NER protein PROTEIN1 and the PROTEIN2 protein which is implicated in Cockayne syndrome (CS).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our analyses demonstrate that the PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins interact with each other.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Our analyses demonstrate that the PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins interact with each other.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Our analyses demonstrate that the PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins interact with each other.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Our analyses demonstrate that the PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins interact with each other.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Our analyses demonstrate that the PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins interact with each other.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Our analyses demonstrate that the PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins interact with each other.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 protein interacts with multiple subunits of PROTEIN1 and with PROTEIN2 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 protein interacts with multiple subunits of PROTEIN1 and with PROTEIN2 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 protein interacts with multiple subunits of PROTEIN1 and with PROTEIN2 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Residues in human PROTEIN0 PROTEIN1 crucial for binding to both the human PROTEIN2 (R) and PROTEIN3 were identified by analysis of alanine scanning mutants of PROTEIN4 in assays for both receptor binding and bioactivity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Residues in human PROTEIN0 PROTEIN1 crucial for binding to both the human PROTEIN2 (R) and PROTEIN3 were identified by analysis of alanine scanning mutants of PROTEIN4 in assays for both receptor binding and bioactivity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Residues in human PROTEIN0 PROTEIN1 crucial for binding to both the human PROTEIN2 (R) and PROTEIN3 were identified by analysis of alanine scanning mutants of PROTEIN4 in assays for both receptor binding and bioactivity.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Residues in human PROTEIN0 PROTEIN1 crucial for binding to both the human PROTEIN2 (R) and PROTEIN3 were identified by analysis of alanine scanning mutants of PROTEIN4 in assays for both receptor binding and bioactivity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Residues in human PROTEIN0 PROTEIN1 crucial for binding to both the human PROTEIN2 (R) and PROTEIN3 were identified by analysis of alanine scanning mutants of PROTEIN4 in assays for both receptor binding and bioactivity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Residues in human PROTEIN0 PROTEIN1 crucial for binding to both the human PROTEIN2 (R) and PROTEIN3 were identified by analysis of alanine scanning mutants of PROTEIN4 in assays for both receptor binding and bioactivity.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Residues in human PROTEIN0 PROTEIN1 crucial for binding to both the human PROTEIN2 (R) and PROTEIN3 were identified by analysis of alanine scanning mutants of PROTEIN4 in assays for both receptor binding and bioactivity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Residues in human PROTEIN0 PROTEIN1 crucial for binding to both the human PROTEIN2 (R) and PROTEIN3 were identified by analysis of alanine scanning mutants of PROTEIN4 in assays for both receptor binding and bioactivity.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Residues in human PROTEIN0 PROTEIN1 crucial for binding to both the human PROTEIN2 (R) and PROTEIN3 were identified by analysis of alanine scanning mutants of PROTEIN4 in assays for both receptor binding and bioactivity.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Residues in human PROTEIN0 PROTEIN1 crucial for binding to both the human PROTEIN2 (R) and PROTEIN3 were identified by analysis of alanine scanning mutants of PROTEIN4 in assays for both receptor binding and bioactivity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The region of PROTEIN0 most important for binding to the PROTEIN1 is composed of residues from the amino terminus of the D-helix, carboxyl terminus of the B-helix, and C-D loop.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The two residues of PROTEIN0 that contribute the majority of free energy for PROTEIN1 binding, Phe-156 and Lys-159 are surrounded by other residues which have only a moderate impact.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A second region of PROTEIN0 that includes residues from the carboxyl terminus of the D-helix and A-B loop also had a weak influence on PROTEIN1 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Residues in PROTEIN0 from both the A- and C-helices are involved in binding the PROTEIN1 co-receptor.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Abolition of the PROTEIN0 binding site in PROTEIN1 created antagonists of PROTEIN2 action.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Abolition of the PROTEIN0 binding site in PROTEIN1 created antagonists of PROTEIN2 action.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Abolition of the PROTEIN0 binding site in PROTEIN1 created antagonists of PROTEIN2 action.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 glucose transporter interacts with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 glucose transporter interacts with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 glucose transporter interacts with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 A truncated version of the integral membrane glycoprotein PROTEIN0 PROTEIN1 was synthesized in a rabbit reticulocyte translation system in the presence of canine pancreatic microsomes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Following immunoprecipitation with an anticalnexin antiserum, a cross-linker-independent association was observed between PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, the anti-calnexin antiserum immunoprecipitated a UV-dependent cross-linking product consisting of PROTEIN0 and a protein of approximately 60 kDa designated PROTEIN1 PROTEIN2 of 60 kDa).\n",
      "\n",
      "PROTEIN0 PROTEIN2 In addition, the anti-calnexin antiserum immunoprecipitated a UV-dependent cross-linking product consisting of PROTEIN0 and a protein of approximately 60 kDa designated PROTEIN1 PROTEIN2 of 60 kDa).\n",
      "\n",
      "PROTEIN1 PROTEIN2 In addition, the anti-calnexin antiserum immunoprecipitated a UV-dependent cross-linking product consisting of PROTEIN0 and a protein of approximately 60 kDa designated PROTEIN1 PROTEIN2 of 60 kDa).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Both the PROTEIN0 PROTEIN1 and the PROTEIN2 PROTEIN3 interactions were dependent on the presence of a correctly modified oligosaccharide group on PROTEIN4 a characteristic of many PROTEIN5 interactions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A PROTEIN0 mutant that was not glycosylated (AGGT155) did not associate with PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 A PROTEIN0 mutant that was not glycosylated (AGGT155) did not associate with PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 A PROTEIN0 mutant that was not glycosylated (AGGT155) did not associate with PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 the soluble homologue of PROTEIN1 was also shown to interact with PROTEIN2 only when the protein bore a correctly modified oligosaccharide group.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 the soluble homologue of PROTEIN1 was also shown to interact with PROTEIN2 only when the protein bore a correctly modified oligosaccharide group.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 the soluble homologue of PROTEIN1 was also shown to interact with PROTEIN2 only when the protein bore a correctly modified oligosaccharide group.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, our data show that both PROTEIN0 and PROTEIN1 interact with PROTEIN2 in a glycosylation-dependent manner.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, our data show that both PROTEIN0 and PROTEIN1 interact with PROTEIN2 in a glycosylation-dependent manner.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, our data show that both PROTEIN0 and PROTEIN1 interact with PROTEIN2 in a glycosylation-dependent manner.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have used site-specific mutagenesis within the extracellular domain of the PROTEIN0 to search for amino acid residues involved in PROTEIN1 PROTEIN2 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have used site-specific mutagenesis within the extracellular domain of the PROTEIN0 to search for amino acid residues involved in PROTEIN1 PROTEIN2 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have used site-specific mutagenesis within the extracellular domain of the PROTEIN0 to search for amino acid residues involved in PROTEIN1 PROTEIN2 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mutant proteins were expressed in bacteria as soluble PROTEIN0 binding proteins PROTEIN1 and characterized for PROTEIN2 binding activity in a number of different assays.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Mutant proteins were expressed in bacteria as soluble PROTEIN0 binding proteins PROTEIN1 and characterized for PROTEIN2 binding activity in a number of different assays.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Mutant proteins were expressed in bacteria as soluble PROTEIN0 binding proteins PROTEIN1 and characterized for PROTEIN2 binding activity in a number of different assays.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Substitution of phenylalanine at position 93 (Phe93) with alanine (F93A mutation) resulted in a drastic reduction in PROTEIN0 binding in the PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, the F93A mutation in full-length PROTEIN0 expressed in COS cells abolished detectable PROTEIN1 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These data indicate that Phe93 is a critical PROTEIN0 binding determinant of the PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 is a cytokine secreted by activated T lymphocytes that shares many, but not all, biological activities with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 is a cytokine secreted by activated T lymphocytes that shares many, but not all, biological activities with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 is a cytokine secreted by activated T lymphocytes that shares many, but not all, biological activities with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have cloned a cDNA encoding an PROTEIN0 binding protein PROTEIN1 from the Caki-1 human renal carcinoma cell line.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 shows homology with the PROTEIN1 receptor, and to a lesser extent, with the prolactin receptor.\n",
      "\n",
      "PROTEIN0 PROTEIN1 COS-7 cells transfected with the PROTEIN0 cDNA bind PROTEIN1 with high affinity but do not bind PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 COS-7 cells transfected with the PROTEIN0 cDNA bind PROTEIN1 with high affinity but do not bind PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 COS-7 cells transfected with the PROTEIN0 cDNA bind PROTEIN1 with high affinity but do not bind PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 COS-7 cells co-transfected with the cloned PROTEIN0 cDNA and PROTEIN1 cDNA resulted in the reconstitution of a small number of receptors that recognized both PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 COS-7 cells co-transfected with the cloned PROTEIN0 cDNA and PROTEIN1 cDNA resulted in the reconstitution of a small number of receptors that recognized both PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 COS-7 cells co-transfected with the cloned PROTEIN0 cDNA and PROTEIN1 cDNA resulted in the reconstitution of a small number of receptors that recognized both PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 COS-7 cells co-transfected with the cloned PROTEIN0 cDNA and PROTEIN1 cDNA resulted in the reconstitution of a small number of receptors that recognized both PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 COS-7 cells co-transfected with the cloned PROTEIN0 cDNA and PROTEIN1 cDNA resulted in the reconstitution of a small number of receptors that recognized both PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 COS-7 cells co-transfected with the cloned PROTEIN0 cDNA and PROTEIN1 cDNA resulted in the reconstitution of a small number of receptors that recognized both PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Crystal structure of the PROTEIN0 cyclin-dependent-kinase inhibitor bound to the PROTEIN1 PROTEIN2 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Crystal structure of the PROTEIN0 cyclin-dependent-kinase inhibitor bound to the PROTEIN1 PROTEIN2 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Crystal structure of the PROTEIN0 cyclin-dependent-kinase inhibitor bound to the PROTEIN1 PROTEIN2 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The crystal structure of the human PROTEIN0 inhibitory domain bound to the phosphorylated PROTEIN1 PROTEIN2 PROTEIN3 complex has been determined at 2.3 angstrom.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The crystal structure of the human PROTEIN0 inhibitory domain bound to the phosphorylated PROTEIN1 PROTEIN2 PROTEIN3 complex has been determined at 2.3 angstrom.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The crystal structure of the human PROTEIN0 inhibitory domain bound to the phosphorylated PROTEIN1 PROTEIN2 PROTEIN3 complex has been determined at 2.3 angstrom.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The crystal structure of the human PROTEIN0 inhibitory domain bound to the phosphorylated PROTEIN1 PROTEIN2 PROTEIN3 complex has been determined at 2.3 angstrom.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The crystal structure of the human PROTEIN0 inhibitory domain bound to the phosphorylated PROTEIN1 PROTEIN2 PROTEIN3 complex has been determined at 2.3 angstrom.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The crystal structure of the human PROTEIN0 inhibitory domain bound to the phosphorylated PROTEIN1 PROTEIN2 PROTEIN3 complex has been determined at 2.3 angstrom.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binds the complex as an extended structure interacting with both PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 binds the complex as an extended structure interacting with both PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 binds the complex as an extended structure interacting with both PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The tyrosine kinases PROTEIN0 PROTEIN1 and PROTEIN2 (or PROTEIN3 constitute a family of endothelial cell-specific receptors with seven immunoglobulin-like domains and a split kinase domain.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The tyrosine kinases PROTEIN0 PROTEIN1 and PROTEIN2 (or PROTEIN3 constitute a family of endothelial cell-specific receptors with seven immunoglobulin-like domains and a split kinase domain.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The tyrosine kinases PROTEIN0 PROTEIN1 and PROTEIN2 (or PROTEIN3 constitute a family of endothelial cell-specific receptors with seven immunoglobulin-like domains and a split kinase domain.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The tyrosine kinases PROTEIN0 PROTEIN1 and PROTEIN2 (or PROTEIN3 constitute a family of endothelial cell-specific receptors with seven immunoglobulin-like domains and a split kinase domain.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The tyrosine kinases PROTEIN0 PROTEIN1 and PROTEIN2 (or PROTEIN3 constitute a family of endothelial cell-specific receptors with seven immunoglobulin-like domains and a split kinase domain.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The tyrosine kinases PROTEIN0 PROTEIN1 and PROTEIN2 (or PROTEIN3 constitute a family of endothelial cell-specific receptors with seven immunoglobulin-like domains and a split kinase domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 have been shown to play key roles in vascular development; these two receptors bind and are activated by PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 and PROTEIN1 have been shown to play key roles in vascular development; these two receptors bind and are activated by PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 and PROTEIN1 have been shown to play key roles in vascular development; these two receptors bind and are activated by PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 and PROTEIN1 have been shown to play key roles in vascular development; these two receptors bind and are activated by PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 and PROTEIN1 have been shown to play key roles in vascular development; these two receptors bind and are activated by PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 and PROTEIN1 have been shown to play key roles in vascular development; these two receptors bind and are activated by PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 fails to bind appreciably to the extracellular domain of PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 fails to bind appreciably to the extracellular domain of PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 fails to bind appreciably to the extracellular domain of PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The kinetics of amyloid fibril formation by PROTEIN0 PROTEIN1 are typical of a nucleation-dependent polymerization mechanism.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Fusion proteins were created by linking the PROTEIN0 fragment to a PROTEIN1 DNA-binding domain (bait) and also to a PROTEIN2 transactivation domain (prey).\n",
      "\n",
      "PROTEIN0 PROTEIN2 Fusion proteins were created by linking the PROTEIN0 fragment to a PROTEIN1 DNA-binding domain (bait) and also to a PROTEIN2 transactivation domain (prey).\n",
      "\n",
      "PROTEIN1 PROTEIN2 Fusion proteins were created by linking the PROTEIN0 fragment to a PROTEIN1 DNA-binding domain (bait) and also to a PROTEIN2 transactivation domain (prey).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Protein-protein interactions were measured by expression of these fusion proteins in Saccharomyces cerevisiae harboring PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 genes under the control of LexA-dependent operators.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Protein-protein interactions were measured by expression of these fusion proteins in Saccharomyces cerevisiae harboring PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 genes under the control of LexA-dependent operators.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Protein-protein interactions were measured by expression of these fusion proteins in Saccharomyces cerevisiae harboring PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 genes under the control of LexA-dependent operators.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Protein-protein interactions were measured by expression of these fusion proteins in Saccharomyces cerevisiae harboring PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 genes under the control of LexA-dependent operators.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Protein-protein interactions were measured by expression of these fusion proteins in Saccharomyces cerevisiae harboring PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 genes under the control of LexA-dependent operators.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Protein-protein interactions were measured by expression of these fusion proteins in Saccharomyces cerevisiae harboring PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 genes under the control of LexA-dependent operators.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN0 PROTEIN6 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN0 PROTEIN7 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN1 PROTEIN6 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN1 PROTEIN7 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN2 PROTEIN6 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN2 PROTEIN7 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN3 PROTEIN6 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN3 PROTEIN7 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN4 PROTEIN6 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN4 PROTEIN7 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN5 PROTEIN6 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN5 PROTEIN7 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN6 PROTEIN7 PROTEIN0 protein fused to the Drosophila protein PROTEIN1 PROTEIN2 PROTEIN3 failed to interact with the PROTEIN4 fragment fused to PROTEIN5 indicating that the observed PROTEIN6 PROTEIN7 interaction was specific.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Although PROTEIN0 has a higher binding affinity for immobilized PROTEIN1 as determined by biosensor analysis employing surface plasmon resonance detection, PROTEIN2 is more potent than PROTEIN3 in a STAT3 phosphorylation assay.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Although PROTEIN0 has a higher binding affinity for immobilized PROTEIN1 as determined by biosensor analysis employing surface plasmon resonance detection, PROTEIN2 is more potent than PROTEIN3 in a STAT3 phosphorylation assay.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Although PROTEIN0 has a higher binding affinity for immobilized PROTEIN1 as determined by biosensor analysis employing surface plasmon resonance detection, PROTEIN2 is more potent than PROTEIN3 in a STAT3 phosphorylation assay.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Although PROTEIN0 has a higher binding affinity for immobilized PROTEIN1 as determined by biosensor analysis employing surface plasmon resonance detection, PROTEIN2 is more potent than PROTEIN3 in a STAT3 phosphorylation assay.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Although PROTEIN0 has a higher binding affinity for immobilized PROTEIN1 as determined by biosensor analysis employing surface plasmon resonance detection, PROTEIN2 is more potent than PROTEIN3 in a STAT3 phosphorylation assay.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Although PROTEIN0 has a higher binding affinity for immobilized PROTEIN1 as determined by biosensor analysis employing surface plasmon resonance detection, PROTEIN2 is more potent than PROTEIN3 in a STAT3 phosphorylation assay.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The increased binding affinity of PROTEIN0 appears to be due to its ability to cross-link two PROTEIN1 molecules on the biosensor surface.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Studies of the PROTEIN0 ternary complex formation demonstrated that the reduced biological potency of PROTEIN1 resulted from a decreased ability of the PROTEIN2 PROTEIN3 complex to couple with the soluble portion of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Studies of the PROTEIN0 ternary complex formation demonstrated that the reduced biological potency of PROTEIN1 resulted from a decreased ability of the PROTEIN2 PROTEIN3 complex to couple with the soluble portion of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Studies of the PROTEIN0 ternary complex formation demonstrated that the reduced biological potency of PROTEIN1 resulted from a decreased ability of the PROTEIN2 PROTEIN3 complex to couple with the soluble portion of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Studies of the PROTEIN0 ternary complex formation demonstrated that the reduced biological potency of PROTEIN1 resulted from a decreased ability of the PROTEIN2 PROTEIN3 complex to couple with the soluble portion of PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Studies of the PROTEIN0 ternary complex formation demonstrated that the reduced biological potency of PROTEIN1 resulted from a decreased ability of the PROTEIN2 PROTEIN3 complex to couple with the soluble portion of PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Studies of the PROTEIN0 ternary complex formation demonstrated that the reduced biological potency of PROTEIN1 resulted from a decreased ability of the PROTEIN2 PROTEIN3 complex to couple with the soluble portion of PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Studies of the PROTEIN0 ternary complex formation demonstrated that the reduced biological potency of PROTEIN1 resulted from a decreased ability of the PROTEIN2 PROTEIN3 complex to couple with the soluble portion of PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Studies of the PROTEIN0 ternary complex formation demonstrated that the reduced biological potency of PROTEIN1 resulted from a decreased ability of the PROTEIN2 PROTEIN3 complex to couple with the soluble portion of PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Studies of the PROTEIN0 ternary complex formation demonstrated that the reduced biological potency of PROTEIN1 resulted from a decreased ability of the PROTEIN2 PROTEIN3 complex to couple with the soluble portion of PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Studies of the PROTEIN0 ternary complex formation demonstrated that the reduced biological potency of PROTEIN1 resulted from a decreased ability of the PROTEIN2 PROTEIN3 complex to couple with the soluble portion of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 These data imply that PROTEIN0 dimerization of PROTEIN1 is not the driving force in promoting formation of the hexameric PROTEIN2 PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These data imply that PROTEIN0 dimerization of PROTEIN1 is not the driving force in promoting formation of the hexameric PROTEIN2 PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN3 These data imply that PROTEIN0 dimerization of PROTEIN1 is not the driving force in promoting formation of the hexameric PROTEIN2 PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN4 These data imply that PROTEIN0 dimerization of PROTEIN1 is not the driving force in promoting formation of the hexameric PROTEIN2 PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These data imply that PROTEIN0 dimerization of PROTEIN1 is not the driving force in promoting formation of the hexameric PROTEIN2 PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN3 These data imply that PROTEIN0 dimerization of PROTEIN1 is not the driving force in promoting formation of the hexameric PROTEIN2 PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN4 These data imply that PROTEIN0 dimerization of PROTEIN1 is not the driving force in promoting formation of the hexameric PROTEIN2 PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN3 These data imply that PROTEIN0 dimerization of PROTEIN1 is not the driving force in promoting formation of the hexameric PROTEIN2 PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN4 These data imply that PROTEIN0 dimerization of PROTEIN1 is not the driving force in promoting formation of the hexameric PROTEIN2 PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN3 PROTEIN4 These data imply that PROTEIN0 dimerization of PROTEIN1 is not the driving force in promoting formation of the hexameric PROTEIN2 PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A model is presented whereby the trimeric complex of PROTEIN0 PROTEIN1 and PROTEIN2 forms before the functional hexamer.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A model is presented whereby the trimeric complex of PROTEIN0 PROTEIN1 and PROTEIN2 forms before the functional hexamer.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A model is presented whereby the trimeric complex of PROTEIN0 PROTEIN1 and PROTEIN2 forms before the functional hexamer.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Due to its increased affinity for the PROTEIN0 but its decreased ability to couple with PROTEIN1 we suggest that a stable PROTEIN2 dimer may be an efficient PROTEIN3 antagonist.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Due to its increased affinity for the PROTEIN0 but its decreased ability to couple with PROTEIN1 we suggest that a stable PROTEIN2 dimer may be an efficient PROTEIN3 antagonist.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Due to its increased affinity for the PROTEIN0 but its decreased ability to couple with PROTEIN1 we suggest that a stable PROTEIN2 dimer may be an efficient PROTEIN3 antagonist.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Due to its increased affinity for the PROTEIN0 but its decreased ability to couple with PROTEIN1 we suggest that a stable PROTEIN2 dimer may be an efficient PROTEIN3 antagonist.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Due to its increased affinity for the PROTEIN0 but its decreased ability to couple with PROTEIN1 we suggest that a stable PROTEIN2 dimer may be an efficient PROTEIN3 antagonist.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Due to its increased affinity for the PROTEIN0 but its decreased ability to couple with PROTEIN1 we suggest that a stable PROTEIN2 dimer may be an efficient PROTEIN3 antagonist.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Physical interaction of mammalian PROTEIN0 with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 was originally identified as a Start gene in budding yeast and has been shown to be required for association of PROTEIN1 with cyclins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a PROTEIN1 that controls progression through G1 of the mammalian cell cycle.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have detected a specific association of PROTEIN0 with the molecular chaperon PROTEIN1 and a 44-kDa protein that we identify as mammalian PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have detected a specific association of PROTEIN0 with the molecular chaperon PROTEIN1 and a 44-kDa protein that we identify as mammalian PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have detected a specific association of PROTEIN0 with the molecular chaperon PROTEIN1 and a 44-kDa protein that we identify as mammalian PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 A physical interaction between PROTEIN0 and PROTEIN1 suggests that PROTEIN2 may regulate the mammalian cell cycle through a direct effect on PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 A physical interaction between PROTEIN0 and PROTEIN1 suggests that PROTEIN2 may regulate the mammalian cell cycle through a direct effect on PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 A physical interaction between PROTEIN0 and PROTEIN1 suggests that PROTEIN2 may regulate the mammalian cell cycle through a direct effect on PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 A physical interaction between PROTEIN0 and PROTEIN1 suggests that PROTEIN2 may regulate the mammalian cell cycle through a direct effect on PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 A physical interaction between PROTEIN0 and PROTEIN1 suggests that PROTEIN2 may regulate the mammalian cell cycle through a direct effect on PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 A physical interaction between PROTEIN0 and PROTEIN1 suggests that PROTEIN2 may regulate the mammalian cell cycle through a direct effect on PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Association of PROTEIN0 with both PROTEIN1 and PROTEIN2 may also imply a mechanistic link between the functions of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Association of PROTEIN0 with both PROTEIN1 and PROTEIN2 may also imply a mechanistic link between the functions of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Association of PROTEIN0 with both PROTEIN1 and PROTEIN2 may also imply a mechanistic link between the functions of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Association of PROTEIN0 with both PROTEIN1 and PROTEIN2 may also imply a mechanistic link between the functions of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Association of PROTEIN0 with both PROTEIN1 and PROTEIN2 may also imply a mechanistic link between the functions of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Association of PROTEIN0 with both PROTEIN1 and PROTEIN2 may also imply a mechanistic link between the functions of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Association of PROTEIN0 with both PROTEIN1 and PROTEIN2 may also imply a mechanistic link between the functions of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Association of PROTEIN0 with both PROTEIN1 and PROTEIN2 may also imply a mechanistic link between the functions of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Association of PROTEIN0 with both PROTEIN1 and PROTEIN2 may also imply a mechanistic link between the functions of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Association of PROTEIN0 with both PROTEIN1 and PROTEIN2 may also imply a mechanistic link between the functions of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interactions of PROTEIN0 a mediator of PROTEIN1 assembly, with heat shock proteins PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Interactions of PROTEIN0 a mediator of PROTEIN1 assembly, with heat shock proteins PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Interactions of PROTEIN0 a mediator of PROTEIN1 assembly, with heat shock proteins PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Interactions of PROTEIN0 a mediator of PROTEIN1 assembly, with heat shock proteins PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Interactions of PROTEIN0 a mediator of PROTEIN1 assembly, with heat shock proteins PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Interactions of PROTEIN0 a mediator of PROTEIN1 assembly, with heat shock proteins PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Previous studies on the assembly of PROTEIN0 PROTEIN1 complexes in vitro have suggested that PROTEIN2 assembly is a dynamic, ordered process involving at least eight nonreceptor proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Previous studies on the assembly of PROTEIN0 PROTEIN1 complexes in vitro have suggested that PROTEIN2 assembly is a dynamic, ordered process involving at least eight nonreceptor proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Previous studies on the assembly of PROTEIN0 PROTEIN1 complexes in vitro have suggested that PROTEIN2 assembly is a dynamic, ordered process involving at least eight nonreceptor proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 One of these proteins, PROTEIN0 appears transiently during assembly and is not a component of functionally mature PROTEIN1 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN3 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN4 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN5 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN6 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN7 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN8 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN3 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN4 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN5 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN6 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN7 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN8 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN3 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN4 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN5 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN6 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN7 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN8 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN4 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN5 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN6 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN7 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN8 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN5 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN6 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN7 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN8 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN5 PROTEIN6 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN5 PROTEIN7 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN5 PROTEIN8 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN6 PROTEIN7 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN6 PROTEIN8 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN7 PROTEIN8 In the present study we observe that a monoclonal antibody specific for PROTEIN0 can, on the one hand, inhibit formation of mature PROTEIN1 complexes containing PROTEIN2 PROTEIN3 PROTEIN4 and immunophilins and, on the other, enhance recovery of early PROTEIN5 complexes containing PROTEIN6 and PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN1 This observation supports a model in which PROTEIN0 functions at an intermediate stage of PROTEIN1 assembly to facilitate formation of subsequent PROTEIN2 complexes lacking PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 This observation supports a model in which PROTEIN0 functions at an intermediate stage of PROTEIN1 assembly to facilitate formation of subsequent PROTEIN2 complexes lacking PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 This observation supports a model in which PROTEIN0 functions at an intermediate stage of PROTEIN1 assembly to facilitate formation of subsequent PROTEIN2 complexes lacking PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 This observation supports a model in which PROTEIN0 functions at an intermediate stage of PROTEIN1 assembly to facilitate formation of subsequent PROTEIN2 complexes lacking PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 This observation supports a model in which PROTEIN0 functions at an intermediate stage of PROTEIN1 assembly to facilitate formation of subsequent PROTEIN2 complexes lacking PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 This observation supports a model in which PROTEIN0 functions at an intermediate stage of PROTEIN1 assembly to facilitate formation of subsequent PROTEIN2 complexes lacking PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Since PROTEIN0 is typically found in a complex with PROTEIN1 and PROTEIN2 we have further characterized its interactions with these proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Since PROTEIN0 is typically found in a complex with PROTEIN1 and PROTEIN2 we have further characterized its interactions with these proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Since PROTEIN0 is typically found in a complex with PROTEIN1 and PROTEIN2 we have further characterized its interactions with these proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 can bind either PROTEIN1 or PROTEIN2 independently and in an ATP-independent manner.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 can bind either PROTEIN1 or PROTEIN2 independently and in an ATP-independent manner.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 can bind either PROTEIN1 or PROTEIN2 independently and in an ATP-independent manner.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Since PROTEIN0 and PROTEIN1 do not readily associate on their own, it appears that PROTEIN2 is the central organizing component of an PROTEIN3 PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mutational analysis of PROTEIN0 indicates that the N terminus is required for PROTEIN1 binding, and a central region containing tetratricopeptide repeat motifs is necessary for binding PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Mutational analysis of PROTEIN0 indicates that the N terminus is required for PROTEIN1 binding, and a central region containing tetratricopeptide repeat motifs is necessary for binding PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Mutational analysis of PROTEIN0 indicates that the N terminus is required for PROTEIN1 binding, and a central region containing tetratricopeptide repeat motifs is necessary for binding PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Mutational analysis of PROTEIN0 indicates that the N terminus is required for PROTEIN1 binding, and a central region containing tetratricopeptide repeat motifs is necessary for binding PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Mutational analysis of PROTEIN0 indicates that the N terminus is required for PROTEIN1 binding, and a central region containing tetratricopeptide repeat motifs is necessary for binding PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Mutational analysis of PROTEIN0 indicates that the N terminus is required for PROTEIN1 binding, and a central region containing tetratricopeptide repeat motifs is necessary for binding PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 PROTEIN2 multichaperone complex is highly dynamic and does not appear to be affected by the PROTEIN3 drug geldanamycin.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 PROTEIN1 PROTEIN2 multichaperone complex is highly dynamic and does not appear to be affected by the PROTEIN3 drug geldanamycin.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 PROTEIN1 PROTEIN2 multichaperone complex is highly dynamic and does not appear to be affected by the PROTEIN3 drug geldanamycin.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 PROTEIN1 PROTEIN2 multichaperone complex is highly dynamic and does not appear to be affected by the PROTEIN3 drug geldanamycin.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 PROTEIN1 PROTEIN2 multichaperone complex is highly dynamic and does not appear to be affected by the PROTEIN3 drug geldanamycin.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 PROTEIN1 PROTEIN2 multichaperone complex is highly dynamic and does not appear to be affected by the PROTEIN3 drug geldanamycin.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN0 PROTEIN2 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN0 PROTEIN3 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN0 PROTEIN4 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN0 PROTEIN5 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN1 PROTEIN2 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN1 PROTEIN3 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN1 PROTEIN4 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN1 PROTEIN5 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN2 PROTEIN3 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN2 PROTEIN4 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN2 PROTEIN5 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN3 PROTEIN4 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN3 PROTEIN5 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN4 PROTEIN5 The interactions of PROTEIN0 and PROTEIN1 in intermediate PROTEIN2 complexes are shown to be distinct from their separate interactions in early PROTEIN3 complexes PROTEIN4 or in mature PROTEIN5 complexes (hsp90).\n",
      "\n",
      "PROTEIN0 PROTEIN1 From these results, it appears that PROTEIN0 is a key mediator in the chaperoned assembly and functional maturation of PROTEIN1 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have selected such mutations in the transcription factor PROTEIN0 that affect its ability to heterodimerize with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The growth-promoting effects of PROTEIN0 on normal and neoplastic gastrointestinal tissues have been shown to be mediated by the PROTEIN1 PROTEIN2 which belongs to the family of G protein-coupled receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The growth-promoting effects of PROTEIN0 on normal and neoplastic gastrointestinal tissues have been shown to be mediated by the PROTEIN1 PROTEIN2 which belongs to the family of G protein-coupled receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The growth-promoting effects of PROTEIN0 on normal and neoplastic gastrointestinal tissues have been shown to be mediated by the PROTEIN1 PROTEIN2 which belongs to the family of G protein-coupled receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 phosphorylation of PROTEIN1 was rapid and transient, occurring within 30 s of treatment and diminishing thereafter.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Following PROTEIN0 stimulation, we observed a time- and dose-dependent association of PROTEIN1 with the PROTEIN2 regulatory subunit of PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Following PROTEIN0 stimulation, we observed a time- and dose-dependent association of PROTEIN1 with the PROTEIN2 regulatory subunit of PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Following PROTEIN0 stimulation, we observed a time- and dose-dependent association of PROTEIN1 with the PROTEIN2 regulatory subunit of PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Following PROTEIN0 stimulation, we observed a time- and dose-dependent association of PROTEIN1 with the PROTEIN2 regulatory subunit of PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Following PROTEIN0 stimulation, we observed a time- and dose-dependent association of PROTEIN1 with the PROTEIN2 regulatory subunit of PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Following PROTEIN0 stimulation, we observed a time- and dose-dependent association of PROTEIN1 with the PROTEIN2 regulatory subunit of PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Following PROTEIN0 stimulation, we observed a time- and dose-dependent association of PROTEIN1 with the PROTEIN2 regulatory subunit of PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Following PROTEIN0 stimulation, we observed a time- and dose-dependent association of PROTEIN1 with the PROTEIN2 regulatory subunit of PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Following PROTEIN0 stimulation, we observed a time- and dose-dependent association of PROTEIN1 with the PROTEIN2 regulatory subunit of PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Following PROTEIN0 stimulation, we observed a time- and dose-dependent association of PROTEIN1 with the PROTEIN2 regulatory subunit of PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN2 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN3 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN4 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN5 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN6 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN2 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN3 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN4 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN5 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN6 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN3 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN4 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN5 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN6 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN4 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN5 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN6 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN5 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN6 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN5 PROTEIN6 In addition, activation of PROTEIN0 was detected in PROTEIN1 immunoprecipitates from PROTEIN2 cells, suggesting that tyrosine phosphorylation of PROTEIN3 which leads to the rapid recruitment of PROTEIN4 might be one mechanism used by PROTEIN5 to activate PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have previously reported that tyrosine phosphorylation of PROTEIN0 and its association with the PROTEIN1 PROTEIN2 complex may contribute to the activation of the PROTEIN3 pathway by PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have previously reported that tyrosine phosphorylation of PROTEIN0 and its association with the PROTEIN1 PROTEIN2 complex may contribute to the activation of the PROTEIN3 pathway by PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 We have previously reported that tyrosine phosphorylation of PROTEIN0 and its association with the PROTEIN1 PROTEIN2 complex may contribute to the activation of the PROTEIN3 pathway by PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 We have previously reported that tyrosine phosphorylation of PROTEIN0 and its association with the PROTEIN1 PROTEIN2 complex may contribute to the activation of the PROTEIN3 pathway by PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have previously reported that tyrosine phosphorylation of PROTEIN0 and its association with the PROTEIN1 PROTEIN2 complex may contribute to the activation of the PROTEIN3 pathway by PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 We have previously reported that tyrosine phosphorylation of PROTEIN0 and its association with the PROTEIN1 PROTEIN2 complex may contribute to the activation of the PROTEIN3 pathway by PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 We have previously reported that tyrosine phosphorylation of PROTEIN0 and its association with the PROTEIN1 PROTEIN2 complex may contribute to the activation of the PROTEIN3 pathway by PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 We have previously reported that tyrosine phosphorylation of PROTEIN0 and its association with the PROTEIN1 PROTEIN2 complex may contribute to the activation of the PROTEIN3 pathway by PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 We have previously reported that tyrosine phosphorylation of PROTEIN0 and its association with the PROTEIN1 PROTEIN2 complex may contribute to the activation of the PROTEIN3 pathway by PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 We have previously reported that tyrosine phosphorylation of PROTEIN0 and its association with the PROTEIN1 PROTEIN2 complex may contribute to the activation of the PROTEIN3 pathway by PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 We report here that PROTEIN0 also interacts with tyrosine-phosphorylated PROTEIN1 in response to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 We report here that PROTEIN0 also interacts with tyrosine-phosphorylated PROTEIN1 in response to PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 We report here that PROTEIN0 also interacts with tyrosine-phosphorylated PROTEIN1 in response to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 a PROTEIN4 PROTEIN5 substrate important for the growth and plasticity of neuronal connections, has been implicated in vitro in several signal transduction pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the yeast-based cloning system, the only strong interaction that was detected between PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the yeast-based cloning system, the only strong interaction that was detected between PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 In the yeast-based cloning system, the only strong interaction that was detected between PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the yeast-based cloning system, the only strong interaction that was detected between PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 In the yeast-based cloning system, the only strong interaction that was detected between PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 In the yeast-based cloning system, the only strong interaction that was detected between PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 phosphorylates PROTEIN1 on serine 41.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Surprisingly, the N-terminal third of PROTEIN0 alone bound PROTEIN1 more strongly than did intact PROTEIN2 suggesting that the protein's C-terminus may play a role in modulating the interaction with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Surprisingly, the N-terminal third of PROTEIN0 alone bound PROTEIN1 more strongly than did intact PROTEIN2 suggesting that the protein's C-terminus may play a role in modulating the interaction with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Surprisingly, the N-terminal third of PROTEIN0 alone bound PROTEIN1 more strongly than did intact PROTEIN2 suggesting that the protein's C-terminus may play a role in modulating the interaction with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Surprisingly, the N-terminal third of PROTEIN0 alone bound PROTEIN1 more strongly than did intact PROTEIN2 suggesting that the protein's C-terminus may play a role in modulating the interaction with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Surprisingly, the N-terminal third of PROTEIN0 alone bound PROTEIN1 more strongly than did intact PROTEIN2 suggesting that the protein's C-terminus may play a role in modulating the interaction with PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Surprisingly, the N-terminal third of PROTEIN0 alone bound PROTEIN1 more strongly than did intact PROTEIN2 suggesting that the protein's C-terminus may play a role in modulating the interaction with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results, along with other recent findings, suggest a novel role for the interaction between PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Structure of the PROTEIN0 oncoprotein bound to the PROTEIN1 tumor suppressor transactivation domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 oncoprotein is a cellular inhibitor of the PROTEIN1 tumor suppressor in that it can bind the transactivation domain of PROTEIN2 and downregulate its ability to activate transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 oncoprotein is a cellular inhibitor of the PROTEIN1 tumor suppressor in that it can bind the transactivation domain of PROTEIN2 and downregulate its ability to activate transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 oncoprotein is a cellular inhibitor of the PROTEIN1 tumor suppressor in that it can bind the transactivation domain of PROTEIN2 and downregulate its ability to activate transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In certain cancers, PROTEIN0 amplification is a common event and contributes to the inactivation of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The crystal structure of the 109-residue amino-terminal domain of PROTEIN0 bound to a 15-residue transactivation domain peptide of PROTEIN1 revealed that PROTEIN2 has a deep hydrophobic cleft on which the PROTEIN3 peptide binds as an amphipathic alpha helix.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The crystal structure of the 109-residue amino-terminal domain of PROTEIN0 bound to a 15-residue transactivation domain peptide of PROTEIN1 revealed that PROTEIN2 has a deep hydrophobic cleft on which the PROTEIN3 peptide binds as an amphipathic alpha helix.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The crystal structure of the 109-residue amino-terminal domain of PROTEIN0 bound to a 15-residue transactivation domain peptide of PROTEIN1 revealed that PROTEIN2 has a deep hydrophobic cleft on which the PROTEIN3 peptide binds as an amphipathic alpha helix.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The crystal structure of the 109-residue amino-terminal domain of PROTEIN0 bound to a 15-residue transactivation domain peptide of PROTEIN1 revealed that PROTEIN2 has a deep hydrophobic cleft on which the PROTEIN3 peptide binds as an amphipathic alpha helix.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The crystal structure of the 109-residue amino-terminal domain of PROTEIN0 bound to a 15-residue transactivation domain peptide of PROTEIN1 revealed that PROTEIN2 has a deep hydrophobic cleft on which the PROTEIN3 peptide binds as an amphipathic alpha helix.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The crystal structure of the 109-residue amino-terminal domain of PROTEIN0 bound to a 15-residue transactivation domain peptide of PROTEIN1 revealed that PROTEIN2 has a deep hydrophobic cleft on which the PROTEIN3 peptide binds as an amphipathic alpha helix.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The interface relies on the steric complementarity between the PROTEIN0 cleft and the hydrophobic face of the PROTEIN1 alpha helix and, in particular, on a triad of PROTEIN2 amino acids-Phe19, Trp23, and Leu26-which insert deep into the PROTEIN3 cleft.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The interface relies on the steric complementarity between the PROTEIN0 cleft and the hydrophobic face of the PROTEIN1 alpha helix and, in particular, on a triad of PROTEIN2 amino acids-Phe19, Trp23, and Leu26-which insert deep into the PROTEIN3 cleft.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The interface relies on the steric complementarity between the PROTEIN0 cleft and the hydrophobic face of the PROTEIN1 alpha helix and, in particular, on a triad of PROTEIN2 amino acids-Phe19, Trp23, and Leu26-which insert deep into the PROTEIN3 cleft.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The interface relies on the steric complementarity between the PROTEIN0 cleft and the hydrophobic face of the PROTEIN1 alpha helix and, in particular, on a triad of PROTEIN2 amino acids-Phe19, Trp23, and Leu26-which insert deep into the PROTEIN3 cleft.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The interface relies on the steric complementarity between the PROTEIN0 cleft and the hydrophobic face of the PROTEIN1 alpha helix and, in particular, on a triad of PROTEIN2 amino acids-Phe19, Trp23, and Leu26-which insert deep into the PROTEIN3 cleft.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The interface relies on the steric complementarity between the PROTEIN0 cleft and the hydrophobic face of the PROTEIN1 alpha helix and, in particular, on a triad of PROTEIN2 amino acids-Phe19, Trp23, and Leu26-which insert deep into the PROTEIN3 cleft.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These same PROTEIN0 residues are also involved in transactivation, supporting the hypothesis that PROTEIN1 inactivates PROTEIN2 by concealing its transactivation domain.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These same PROTEIN0 residues are also involved in transactivation, supporting the hypothesis that PROTEIN1 inactivates PROTEIN2 by concealing its transactivation domain.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These same PROTEIN0 residues are also involved in transactivation, supporting the hypothesis that PROTEIN1 inactivates PROTEIN2 by concealing its transactivation domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The cytokine-activated tyrosine kinase PROTEIN0 activates PROTEIN1 in a PROTEIN2 manner.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The cytokine-activated tyrosine kinase PROTEIN0 activates PROTEIN1 in a PROTEIN2 manner.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The cytokine-activated tyrosine kinase PROTEIN0 activates PROTEIN1 in a PROTEIN2 manner.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 a member of the Janus kinase superfamily was found to interact functionally with PROTEIN1 a central component of the PROTEIN2 PROTEIN3 signal transduction pathway.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 a member of the Janus kinase superfamily was found to interact functionally with PROTEIN1 a central component of the PROTEIN2 PROTEIN3 signal transduction pathway.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 a member of the Janus kinase superfamily was found to interact functionally with PROTEIN1 a central component of the PROTEIN2 PROTEIN3 signal transduction pathway.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 a member of the Janus kinase superfamily was found to interact functionally with PROTEIN1 a central component of the PROTEIN2 PROTEIN3 signal transduction pathway.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 a member of the Janus kinase superfamily was found to interact functionally with PROTEIN1 a central component of the PROTEIN2 PROTEIN3 signal transduction pathway.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 a member of the Janus kinase superfamily was found to interact functionally with PROTEIN1 a central component of the PROTEIN2 PROTEIN3 signal transduction pathway.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the baculovirus coexpression system, PROTEIN0 was activated by PROTEIN1 in the presence of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the baculovirus coexpression system, PROTEIN0 was activated by PROTEIN1 in the presence of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the baculovirus coexpression system, PROTEIN0 was activated by PROTEIN1 in the presence of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Under these conditions, a ternary complex of PROTEIN0 PROTEIN1 and PROTEIN2 was observed.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Under these conditions, a ternary complex of PROTEIN0 PROTEIN1 and PROTEIN2 was observed.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Under these conditions, a ternary complex of PROTEIN0 PROTEIN1 and PROTEIN2 was observed.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, in the absence of PROTEIN0 coexpression of PROTEIN1 and PROTEIN2 resulted in an overall decrease in the PROTEIN3 kinase activity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In contrast, in the absence of PROTEIN0 coexpression of PROTEIN1 and PROTEIN2 resulted in an overall decrease in the PROTEIN3 kinase activity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In contrast, in the absence of PROTEIN0 coexpression of PROTEIN1 and PROTEIN2 resulted in an overall decrease in the PROTEIN3 kinase activity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In contrast, in the absence of PROTEIN0 coexpression of PROTEIN1 and PROTEIN2 resulted in an overall decrease in the PROTEIN3 kinase activity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In contrast, in the absence of PROTEIN0 coexpression of PROTEIN1 and PROTEIN2 resulted in an overall decrease in the PROTEIN3 kinase activity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In contrast, in the absence of PROTEIN0 coexpression of PROTEIN1 and PROTEIN2 resulted in an overall decrease in the PROTEIN3 kinase activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, PROTEIN0 phosphorylated PROTEIN1 at sites different from those phosphorylated by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In addition, PROTEIN0 phosphorylated PROTEIN1 at sites different from those phosphorylated by PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In addition, PROTEIN0 phosphorylated PROTEIN1 at sites different from those phosphorylated by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 In mammalian cells treated with either erythropoietin or interferon-gamma, a small fraction of PROTEIN0 coimmunoprecipitated with PROTEIN1 in lysates of cells in which PROTEIN2 was activated as judged by its state of tyrosine phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In mammalian cells treated with either erythropoietin or interferon-gamma, a small fraction of PROTEIN0 coimmunoprecipitated with PROTEIN1 in lysates of cells in which PROTEIN2 was activated as judged by its state of tyrosine phosphorylation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In mammalian cells treated with either erythropoietin or interferon-gamma, a small fraction of PROTEIN0 coimmunoprecipitated with PROTEIN1 in lysates of cells in which PROTEIN2 was activated as judged by its state of tyrosine phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Taken together, these data suggest that PROTEIN0 and PROTEIN1 cooperate to activate PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Taken together, these data suggest that PROTEIN0 and PROTEIN1 cooperate to activate PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Taken together, these data suggest that PROTEIN0 and PROTEIN1 cooperate to activate PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 interaction of primary HIV-1 PROTEIN1 glycoproteins with the chemokine receptor PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 interaction of primary HIV-1 PROTEIN1 glycoproteins with the chemokine receptor PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 interaction of primary HIV-1 PROTEIN1 glycoproteins with the chemokine receptor PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 For efficient entry into target cells, primary macrophage-tropic and laboratory-adapted human immunodeficiency viruses type 1 (HIV-1) require particular chemokine receptors, PROTEIN0 and PROTEIN1 respectively, as well as the primary receptor PROTEIN2 (refs 1-6).\n",
      "\n",
      "PROTEIN0 PROTEIN2 For efficient entry into target cells, primary macrophage-tropic and laboratory-adapted human immunodeficiency viruses type 1 (HIV-1) require particular chemokine receptors, PROTEIN0 and PROTEIN1 respectively, as well as the primary receptor PROTEIN2 (refs 1-6).\n",
      "\n",
      "PROTEIN1 PROTEIN2 For efficient entry into target cells, primary macrophage-tropic and laboratory-adapted human immunodeficiency viruses type 1 (HIV-1) require particular chemokine receptors, PROTEIN0 and PROTEIN1 respectively, as well as the primary receptor PROTEIN2 (refs 1-6).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN0 PROTEIN4 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN0 PROTEIN5 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN1 PROTEIN4 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN1 PROTEIN5 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN2 PROTEIN4 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN2 PROTEIN5 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN3 PROTEIN4 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN3 PROTEIN5 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN4 PROTEIN5 Here we show that a complex of PROTEIN0 the exterior envelope glycoprotein, of macrophage-tropic primary HIV-1 and soluble PROTEIN1 interacts specifically with PROTEIN2 and inhibits the binding of the natural PROTEIN3 ligands, PROTEIN4 and PROTEIN5 (refs 7, 8).\n",
      "\n",
      "PROTEIN0 PROTEIN1 The apparent affinity of the interaction between PROTEIN0 and PROTEIN1 was dramatically lower in the absence of soluble PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The apparent affinity of the interaction between PROTEIN0 and PROTEIN1 was dramatically lower in the absence of soluble PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The apparent affinity of the interaction between PROTEIN0 and PROTEIN1 was dramatically lower in the absence of soluble PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Additionally, in the absence of PROTEIN0 an interaction between a two-domain PROTEIN1 fragment and PROTEIN2 was observed.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Additionally, in the absence of PROTEIN0 an interaction between a two-domain PROTEIN1 fragment and PROTEIN2 was observed.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Additionally, in the absence of PROTEIN0 an interaction between a two-domain PROTEIN1 fragment and PROTEIN2 was observed.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A PROTEIN0 fragment retaining the PROTEIN1 site and overlapping epitopes was able to interact with PROTEIN2 only if the V3 loop, which can specify HIV-1 tropism and chemokine receptor choice, was also present on the molecule.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A PROTEIN0 fragment retaining the PROTEIN1 site and overlapping epitopes was able to interact with PROTEIN2 only if the V3 loop, which can specify HIV-1 tropism and chemokine receptor choice, was also present on the molecule.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A PROTEIN0 fragment retaining the PROTEIN1 site and overlapping epitopes was able to interact with PROTEIN2 only if the V3 loop, which can specify HIV-1 tropism and chemokine receptor choice, was also present on the molecule.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Neutralizing antibodies directed against either PROTEIN0 or V3 epitopes on PROTEIN1 blocked the interaction of PROTEIN2 PROTEIN3 complexes with PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Neutralizing antibodies directed against either PROTEIN0 or V3 epitopes on PROTEIN1 blocked the interaction of PROTEIN2 PROTEIN3 complexes with PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Neutralizing antibodies directed against either PROTEIN0 or V3 epitopes on PROTEIN1 blocked the interaction of PROTEIN2 PROTEIN3 complexes with PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Neutralizing antibodies directed against either PROTEIN0 or V3 epitopes on PROTEIN1 blocked the interaction of PROTEIN2 PROTEIN3 complexes with PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Neutralizing antibodies directed against either PROTEIN0 or V3 epitopes on PROTEIN1 blocked the interaction of PROTEIN2 PROTEIN3 complexes with PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Neutralizing antibodies directed against either PROTEIN0 or V3 epitopes on PROTEIN1 blocked the interaction of PROTEIN2 PROTEIN3 complexes with PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Neutralizing antibodies directed against either PROTEIN0 or V3 epitopes on PROTEIN1 blocked the interaction of PROTEIN2 PROTEIN3 complexes with PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Neutralizing antibodies directed against either PROTEIN0 or V3 epitopes on PROTEIN1 blocked the interaction of PROTEIN2 PROTEIN3 complexes with PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Neutralizing antibodies directed against either PROTEIN0 or V3 epitopes on PROTEIN1 blocked the interaction of PROTEIN2 PROTEIN3 complexes with PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Neutralizing antibodies directed against either PROTEIN0 or V3 epitopes on PROTEIN1 blocked the interaction of PROTEIN2 PROTEIN3 complexes with PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that HIV-1 attachment to PROTEIN0 creates a high-affinity binding site for PROTEIN1 leading to membrane fusion and virus entry.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In all cells tested, PROTEIN0 activated PROTEIN1 and PROTEIN2 and induced the formation of three distinct DNA binding complexes that contained different combinations of these two transcription factors.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In all cells tested, PROTEIN0 activated PROTEIN1 and PROTEIN2 and induced the formation of three distinct DNA binding complexes that contained different combinations of these two transcription factors.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In all cells tested, PROTEIN0 activated PROTEIN1 and PROTEIN2 and induced the formation of three distinct DNA binding complexes that contained different combinations of these two transcription factors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Twelve amino acid peptides encompassing either of these two tyrosine residues in phosphorylated form coprecipitated PROTEIN0 but not PROTEIN1 and blocked PROTEIN2 PROTEIN3 phosphorylation in a cell-free system.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Twelve amino acid peptides encompassing either of these two tyrosine residues in phosphorylated form coprecipitated PROTEIN0 but not PROTEIN1 and blocked PROTEIN2 PROTEIN3 phosphorylation in a cell-free system.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Twelve amino acid peptides encompassing either of these two tyrosine residues in phosphorylated form coprecipitated PROTEIN0 but not PROTEIN1 and blocked PROTEIN2 PROTEIN3 phosphorylation in a cell-free system.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Twelve amino acid peptides encompassing either of these two tyrosine residues in phosphorylated form coprecipitated PROTEIN0 but not PROTEIN1 and blocked PROTEIN2 PROTEIN3 phosphorylation in a cell-free system.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Twelve amino acid peptides encompassing either of these two tyrosine residues in phosphorylated form coprecipitated PROTEIN0 but not PROTEIN1 and blocked PROTEIN2 PROTEIN3 phosphorylation in a cell-free system.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Twelve amino acid peptides encompassing either of these two tyrosine residues in phosphorylated form coprecipitated PROTEIN0 but not PROTEIN1 and blocked PROTEIN2 PROTEIN3 phosphorylation in a cell-free system.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, tyrosine-phosphorylated peptides containing Tyr374 or Tyr396 did not interact with PROTEIN0 or block PROTEIN1 activation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interaction of PROTEIN0 with PROTEIN1 binding and inhibition of amyloid formation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Aggregated PROTEIN0 PROTEIN1 is a key component of the amyloid depositions found in the brains of patients with Alzheimer's disease.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The analysis of complexes of PROTEIN0 with CSF proteins in a KBr gradient revealed an association of PROTEIN1 only with free proteins and not with lipoprotein particles.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 a second major CSF protein, formed SDS-stable complexes with PROTEIN2 and significantly decreased the rate of PROTEIN3 fibril formation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 a second major CSF protein, formed SDS-stable complexes with PROTEIN2 and significantly decreased the rate of PROTEIN3 fibril formation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 a second major CSF protein, formed SDS-stable complexes with PROTEIN2 and significantly decreased the rate of PROTEIN3 fibril formation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 a second major CSF protein, formed SDS-stable complexes with PROTEIN2 and significantly decreased the rate of PROTEIN3 fibril formation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 a second major CSF protein, formed SDS-stable complexes with PROTEIN2 and significantly decreased the rate of PROTEIN3 fibril formation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 a second major CSF protein, formed SDS-stable complexes with PROTEIN2 and significantly decreased the rate of PROTEIN3 fibril formation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In physiological buffers and CSF, PROTEIN0 exclusively decreased the level of PROTEIN1 pentamers.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Endogenous PROTEIN0 PROTEIN1 complexes were detected in human CSF by immunoprecipitation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using site-directed mutagenesis and computer-assisted modelling, we identified amino acid residues on the surface of the PROTEIN0 monomer that interact with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 We propose that PROTEIN0 binding proteins play a key role in the modulation of PROTEIN1 aggregation and normally prevent amyloid formation in biological fluids and in the brain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Discrete protein interactions with the PROTEIN0 PROTEIN1 complex in PROTEIN2 and PROTEIN3 myeloid cell proliferation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Discrete protein interactions with the PROTEIN0 PROTEIN1 complex in PROTEIN2 and PROTEIN3 myeloid cell proliferation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Discrete protein interactions with the PROTEIN0 PROTEIN1 complex in PROTEIN2 and PROTEIN3 myeloid cell proliferation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Discrete protein interactions with the PROTEIN0 PROTEIN1 complex in PROTEIN2 and PROTEIN3 myeloid cell proliferation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Discrete protein interactions with the PROTEIN0 PROTEIN1 complex in PROTEIN2 and PROTEIN3 myeloid cell proliferation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Discrete protein interactions with the PROTEIN0 PROTEIN1 complex in PROTEIN2 and PROTEIN3 myeloid cell proliferation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Hemopoietic cell proliferation is mediated by non-tyrosine and tyrosine kinases that signal via uncommon and common sets of downstream effector molecules including the PROTEIN0 PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the present study we evaluated tyrosine phosphorylation of PROTEIN0 and the interaction of the PROTEIN1 PROTEIN2 complex with signaling proteins upon activation of non-tyrosine PROTEIN3 and tyrosine kinase PROTEIN4 receptors leading to myeloid cell proliferation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the present study we evaluated tyrosine phosphorylation of PROTEIN0 and the interaction of the PROTEIN1 PROTEIN2 complex with signaling proteins upon activation of non-tyrosine PROTEIN3 and tyrosine kinase PROTEIN4 receptors leading to myeloid cell proliferation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In the present study we evaluated tyrosine phosphorylation of PROTEIN0 and the interaction of the PROTEIN1 PROTEIN2 complex with signaling proteins upon activation of non-tyrosine PROTEIN3 and tyrosine kinase PROTEIN4 receptors leading to myeloid cell proliferation.\n",
      "\n",
      "PROTEIN0 PROTEIN4 In the present study we evaluated tyrosine phosphorylation of PROTEIN0 and the interaction of the PROTEIN1 PROTEIN2 complex with signaling proteins upon activation of non-tyrosine PROTEIN3 and tyrosine kinase PROTEIN4 receptors leading to myeloid cell proliferation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the present study we evaluated tyrosine phosphorylation of PROTEIN0 and the interaction of the PROTEIN1 PROTEIN2 complex with signaling proteins upon activation of non-tyrosine PROTEIN3 and tyrosine kinase PROTEIN4 receptors leading to myeloid cell proliferation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In the present study we evaluated tyrosine phosphorylation of PROTEIN0 and the interaction of the PROTEIN1 PROTEIN2 complex with signaling proteins upon activation of non-tyrosine PROTEIN3 and tyrosine kinase PROTEIN4 receptors leading to myeloid cell proliferation.\n",
      "\n",
      "PROTEIN1 PROTEIN4 In the present study we evaluated tyrosine phosphorylation of PROTEIN0 and the interaction of the PROTEIN1 PROTEIN2 complex with signaling proteins upon activation of non-tyrosine PROTEIN3 and tyrosine kinase PROTEIN4 receptors leading to myeloid cell proliferation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In the present study we evaluated tyrosine phosphorylation of PROTEIN0 and the interaction of the PROTEIN1 PROTEIN2 complex with signaling proteins upon activation of non-tyrosine PROTEIN3 and tyrosine kinase PROTEIN4 receptors leading to myeloid cell proliferation.\n",
      "\n",
      "PROTEIN2 PROTEIN4 In the present study we evaluated tyrosine phosphorylation of PROTEIN0 and the interaction of the PROTEIN1 PROTEIN2 complex with signaling proteins upon activation of non-tyrosine PROTEIN3 and tyrosine kinase PROTEIN4 receptors leading to myeloid cell proliferation.\n",
      "\n",
      "PROTEIN3 PROTEIN4 In the present study we evaluated tyrosine phosphorylation of PROTEIN0 and the interaction of the PROTEIN1 PROTEIN2 complex with signaling proteins upon activation of non-tyrosine PROTEIN3 and tyrosine kinase PROTEIN4 receptors leading to myeloid cell proliferation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 By using the growth factor dependent M-07e cell line, we found that both PROTEIN0 and PROTEIN1 ligands, namely: PROTEIN2 and PROTEIN3 induce PROTEIN4 tyrosine phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 By using the growth factor dependent M-07e cell line, we found that both PROTEIN0 and PROTEIN1 ligands, namely: PROTEIN2 and PROTEIN3 induce PROTEIN4 tyrosine phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 By using the growth factor dependent M-07e cell line, we found that both PROTEIN0 and PROTEIN1 ligands, namely: PROTEIN2 and PROTEIN3 induce PROTEIN4 tyrosine phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN4 By using the growth factor dependent M-07e cell line, we found that both PROTEIN0 and PROTEIN1 ligands, namely: PROTEIN2 and PROTEIN3 induce PROTEIN4 tyrosine phosphorylation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 By using the growth factor dependent M-07e cell line, we found that both PROTEIN0 and PROTEIN1 ligands, namely: PROTEIN2 and PROTEIN3 induce PROTEIN4 tyrosine phosphorylation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 By using the growth factor dependent M-07e cell line, we found that both PROTEIN0 and PROTEIN1 ligands, namely: PROTEIN2 and PROTEIN3 induce PROTEIN4 tyrosine phosphorylation.\n",
      "\n",
      "PROTEIN1 PROTEIN4 By using the growth factor dependent M-07e cell line, we found that both PROTEIN0 and PROTEIN1 ligands, namely: PROTEIN2 and PROTEIN3 induce PROTEIN4 tyrosine phosphorylation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 By using the growth factor dependent M-07e cell line, we found that both PROTEIN0 and PROTEIN1 ligands, namely: PROTEIN2 and PROTEIN3 induce PROTEIN4 tyrosine phosphorylation.\n",
      "\n",
      "PROTEIN2 PROTEIN4 By using the growth factor dependent M-07e cell line, we found that both PROTEIN0 and PROTEIN1 ligands, namely: PROTEIN2 and PROTEIN3 induce PROTEIN4 tyrosine phosphorylation.\n",
      "\n",
      "PROTEIN3 PROTEIN4 By using the growth factor dependent M-07e cell line, we found that both PROTEIN0 and PROTEIN1 ligands, namely: PROTEIN2 and PROTEIN3 induce PROTEIN4 tyrosine phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In these cells the adaptor protein PROTEIN0 constitutively binds a substantial fraction of PROTEIN1 through the N-terminal SH3 domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In vitro experiments showed that the stable PROTEIN0 PROTEIN1 complex interacts, through the PROTEIN2 SH2 domain, with the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 In vitro experiments showed that the stable PROTEIN0 PROTEIN1 complex interacts, through the PROTEIN2 SH2 domain, with the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 In vitro experiments showed that the stable PROTEIN0 PROTEIN1 complex interacts, through the PROTEIN2 SH2 domain, with the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 In vitro experiments showed that the stable PROTEIN0 PROTEIN1 complex interacts, through the PROTEIN2 SH2 domain, with the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 In vitro experiments showed that the stable PROTEIN0 PROTEIN1 complex interacts, through the PROTEIN2 SH2 domain, with the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 In vitro experiments showed that the stable PROTEIN0 PROTEIN1 complex interacts, through the PROTEIN2 SH2 domain, with the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 In vitro experiments showed that the stable PROTEIN0 PROTEIN1 complex interacts, through the PROTEIN2 SH2 domain, with the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 In vitro experiments showed that the stable PROTEIN0 PROTEIN1 complex interacts, through the PROTEIN2 SH2 domain, with the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 In vitro experiments showed that the stable PROTEIN0 PROTEIN1 complex interacts, through the PROTEIN2 SH2 domain, with the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 In vitro experiments showed that the stable PROTEIN0 PROTEIN1 complex interacts, through the PROTEIN2 SH2 domain, with the PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 By contrast stimulation with PROTEIN0 leads to the formation of a PROTEIN1 complex containing PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 By contrast stimulation with PROTEIN0 leads to the formation of a PROTEIN1 complex containing PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 By contrast stimulation with PROTEIN0 leads to the formation of a PROTEIN1 complex containing PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 In vitro and in vivo experiments support the hypothesis that PROTEIN0 mediates the association of PROTEIN1 with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In vitro and in vivo experiments support the hypothesis that PROTEIN0 mediates the association of PROTEIN1 with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In vitro and in vivo experiments support the hypothesis that PROTEIN0 mediates the association of PROTEIN1 with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover we found that, upon PROTEIN0 stimulation, the PROTEIN1 PROTEIN2 complex recruits PROTEIN3 probably via PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Moreover we found that, upon PROTEIN0 stimulation, the PROTEIN1 PROTEIN2 complex recruits PROTEIN3 probably via PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Moreover we found that, upon PROTEIN0 stimulation, the PROTEIN1 PROTEIN2 complex recruits PROTEIN3 probably via PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Moreover we found that, upon PROTEIN0 stimulation, the PROTEIN1 PROTEIN2 complex recruits PROTEIN3 probably via PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Moreover we found that, upon PROTEIN0 stimulation, the PROTEIN1 PROTEIN2 complex recruits PROTEIN3 probably via PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Moreover we found that, upon PROTEIN0 stimulation, the PROTEIN1 PROTEIN2 complex recruits PROTEIN3 probably via PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Moreover we found that, upon PROTEIN0 stimulation, the PROTEIN1 PROTEIN2 complex recruits PROTEIN3 probably via PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Moreover we found that, upon PROTEIN0 stimulation, the PROTEIN1 PROTEIN2 complex recruits PROTEIN3 probably via PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Moreover we found that, upon PROTEIN0 stimulation, the PROTEIN1 PROTEIN2 complex recruits PROTEIN3 probably via PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Moreover we found that, upon PROTEIN0 stimulation, the PROTEIN1 PROTEIN2 complex recruits PROTEIN3 probably via PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN4 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN5 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN6 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN4 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN5 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN6 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN4 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN5 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN6 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN3 PROTEIN4 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN3 PROTEIN5 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN3 PROTEIN6 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN4 PROTEIN5 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN4 PROTEIN6 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN5 PROTEIN6 By contrast the PROTEIN0 exchanger factor PROTEIN1 was not detected in PROTEIN2 immunoprecipitates suggesting that PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6 are present in different complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Taken together our results demonstrate that PROTEIN0 plays an important role in coupling both tyrosine and non-tyrosine kinase receptors to downstream effector molecules and that different signaling molecules interact with PROTEIN1 PROTEIN2 complex when non-tyrosine or tyrosine kinase receptors are activated.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Taken together our results demonstrate that PROTEIN0 plays an important role in coupling both tyrosine and non-tyrosine kinase receptors to downstream effector molecules and that different signaling molecules interact with PROTEIN1 PROTEIN2 complex when non-tyrosine or tyrosine kinase receptors are activated.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Taken together our results demonstrate that PROTEIN0 plays an important role in coupling both tyrosine and non-tyrosine kinase receptors to downstream effector molecules and that different signaling molecules interact with PROTEIN1 PROTEIN2 complex when non-tyrosine or tyrosine kinase receptors are activated.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Lipopolysaccharide-activated PROTEIN0 (ICE-/-) Kupffer cells synthesized the PROTEIN1 precursor but failed to process it into the active form.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 were diminished in the sera of PROTEIN2 mice exposed to Propionibacterium acnes and lipopolysaccharide.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 and PROTEIN1 were diminished in the sera of PROTEIN2 mice exposed to Propionibacterium acnes and lipopolysaccharide.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 and PROTEIN1 were diminished in the sera of PROTEIN2 mice exposed to Propionibacterium acnes and lipopolysaccharide.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Physical and functional interactions between the transcriptional inhibitors PROTEIN0 and PROTEIN1 Evidence toward a novel mechanism regulating muscle-specific gene expression.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have used an interaction cloning strategy to identify an inhibitory isoform of the PROTEIN0 transcription factor, PROTEIN1 that interacts with the transcriptional inhibitor PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have used an interaction cloning strategy to identify an inhibitory isoform of the PROTEIN0 transcription factor, PROTEIN1 that interacts with the transcriptional inhibitor PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 We have used an interaction cloning strategy to identify an inhibitory isoform of the PROTEIN0 transcription factor, PROTEIN1 that interacts with the transcriptional inhibitor PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have used an interaction cloning strategy to identify an inhibitory isoform of the PROTEIN0 transcription factor, PROTEIN1 that interacts with the transcriptional inhibitor PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 We have used an interaction cloning strategy to identify an inhibitory isoform of the PROTEIN0 transcription factor, PROTEIN1 that interacts with the transcriptional inhibitor PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 We have used an interaction cloning strategy to identify an inhibitory isoform of the PROTEIN0 transcription factor, PROTEIN1 that interacts with the transcriptional inhibitor PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 As expected, overexpression of either PROTEIN0 PROTEIN1 or PROTEIN2 effectively inhibited the activation of the muscle-specific PROTEIN3 promoter by the myogenic transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 As expected, overexpression of either PROTEIN0 PROTEIN1 or PROTEIN2 effectively inhibited the activation of the muscle-specific PROTEIN3 promoter by the myogenic transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 As expected, overexpression of either PROTEIN0 PROTEIN1 or PROTEIN2 effectively inhibited the activation of the muscle-specific PROTEIN3 promoter by the myogenic transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 As expected, overexpression of either PROTEIN0 PROTEIN1 or PROTEIN2 effectively inhibited the activation of the muscle-specific PROTEIN3 promoter by the myogenic transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 As expected, overexpression of either PROTEIN0 PROTEIN1 or PROTEIN2 effectively inhibited the activation of the muscle-specific PROTEIN3 promoter by the myogenic transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 As expected, overexpression of either PROTEIN0 PROTEIN1 or PROTEIN2 effectively inhibited the activation of the muscle-specific PROTEIN3 promoter by the myogenic transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 As expected, overexpression of either PROTEIN0 PROTEIN1 or PROTEIN2 effectively inhibited the activation of the muscle-specific PROTEIN3 promoter by the myogenic transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 As expected, overexpression of either PROTEIN0 PROTEIN1 or PROTEIN2 effectively inhibited the activation of the muscle-specific PROTEIN3 promoter by the myogenic transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 As expected, overexpression of either PROTEIN0 PROTEIN1 or PROTEIN2 effectively inhibited the activation of the muscle-specific PROTEIN3 promoter by the myogenic transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 As expected, overexpression of either PROTEIN0 PROTEIN1 or PROTEIN2 effectively inhibited the activation of the muscle-specific PROTEIN3 promoter by the myogenic transcription factor PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover, while PROTEIN0 inhibited the PROTEIN1 promoter, it acted as a weak transactivator on an artificial promoter consisting of three tandem copies of the consensus myogenic factor DNA binding site.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Further investigation indicated that the PROTEIN0 PROTEIN1 heterodimer bound to its specific DNA binding site in vitro, and the DNA binding was effectively blocked by PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Further investigation indicated that the PROTEIN0 PROTEIN1 heterodimer bound to its specific DNA binding site in vitro, and the DNA binding was effectively blocked by PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Further investigation indicated that the PROTEIN0 PROTEIN1 heterodimer bound to its specific DNA binding site in vitro, and the DNA binding was effectively blocked by PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Further investigation indicated that the PROTEIN0 PROTEIN1 heterodimer bound to its specific DNA binding site in vitro, and the DNA binding was effectively blocked by PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Further investigation indicated that the PROTEIN0 PROTEIN1 heterodimer bound to its specific DNA binding site in vitro, and the DNA binding was effectively blocked by PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Further investigation indicated that the PROTEIN0 PROTEIN1 heterodimer bound to its specific DNA binding site in vitro, and the DNA binding was effectively blocked by PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Additional analysis revealed the presence of transcripts for both the activating PROTEIN0 and inhibitory PROTEIN1 isoforms in differentiating C2C12 cultures, suggesting that both isoforms might participate in regulating the differentiation process.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Competitive binding of PROTEIN0 and PROTEIN1 to the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Competitive binding of PROTEIN0 and PROTEIN1 to the PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Competitive binding of PROTEIN0 and PROTEIN1 to the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Intriguingly, PROTEIN0 PROTEIN1 binding is directly antagonized by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Intriguingly, PROTEIN0 PROTEIN1 binding is directly antagonized by PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Intriguingly, PROTEIN0 PROTEIN1 binding is directly antagonized by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 interacts with protein phosphatases PROTEIN1 and PROTEIN2 and disrupts a G2/M cell-cycle checkpoint.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 interacts with protein phosphatases PROTEIN1 and PROTEIN2 and disrupts a G2/M cell-cycle checkpoint.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 interacts with protein phosphatases PROTEIN1 and PROTEIN2 and disrupts a G2/M cell-cycle checkpoint.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus PROTEIN0 is a cellular oncogene that targets PROTEIN1 and PROTEIN2 both of which are targets for oncogenic viruses and chemical tumour promoters.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus PROTEIN0 is a cellular oncogene that targets PROTEIN1 and PROTEIN2 both of which are targets for oncogenic viruses and chemical tumour promoters.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus PROTEIN0 is a cellular oncogene that targets PROTEIN1 and PROTEIN2 both of which are targets for oncogenic viruses and chemical tumour promoters.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Following this hypothesis, we altered in PROTEIN0 and PROTEIN1 residues predicted to reside in the contact region and to interact with each other.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We studied the capacity of these mutants to form an PROTEIN0 PROTEIN1 complex and their ability to transduce the signal.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Site-specific mutants were expressed in Escherichia coli as soluble PROTEIN0 and analyzed for PROTEIN1 binding in several different assay formats.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, selected mutant proteins were expressed as full-length PROTEIN0 on the surface of COS cells and analyzed for PROTEIN1 binding in receptor binding assays.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using these methods, we have identified residues that appear to be involved in PROTEIN0 binding as well as other residues, most of which are conserved in receptors of the cytokine receptor family, that appear to be necessary for the proper folding and/or stability of the PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Stimulation of PROTEIN0 PROTEIN1 results in a variety of functional effects, including regulation of epithelial cell growth and differentiation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Activation of PROTEIN0 upon ligand binding resulted in activation of PROTEIN1 pathway.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 utilizes two different pools of adapter protein PROTEIN1 to link to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 utilizes two different pools of adapter protein PROTEIN1 to link to PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 utilizes two different pools of adapter protein PROTEIN1 to link to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Activated PROTEIN0 predominantly interacts with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 in complex with a previously identified 90-kDa protein and designated protein PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Tyrosine phophorylation of the 66-kDa protein was dependent on ligand activation of PROTEIN0 suggesting that the 66-kDa protein may play an important role in PROTEIN1 signaling.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition to this unique pathway, PROTEIN0 also links to PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 complex via the adapter protein PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, activated PROTEIN0 was not able to induce dissociation of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 complex following PROTEIN1 phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In summary, PROTEIN0 signaling pathway utilizes two PROTEIN1 complexes; PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In summary, PROTEIN0 signaling pathway utilizes two PROTEIN1 complexes; PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In summary, PROTEIN0 signaling pathway utilizes two PROTEIN1 complexes; PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 and PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 and PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 these two complexes may alternatively link PROTEIN1 to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 these two complexes may alternatively link PROTEIN1 to PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 these two complexes may alternatively link PROTEIN1 to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Finally, activated PROTEIN0 was also found to result in phosphorylation of PROTEIN1 but reduced phosphorylation of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Finally, activated PROTEIN0 was also found to result in phosphorylation of PROTEIN1 but reduced phosphorylation of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Finally, activated PROTEIN0 was also found to result in phosphorylation of PROTEIN1 but reduced phosphorylation of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 is an important mediator of inflammatory disease.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 family currently consists of two agonists, PROTEIN1 and PROTEIN2 and one antagonist, PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 family currently consists of two agonists, PROTEIN1 and PROTEIN2 and one antagonist, PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 family currently consists of two agonists, PROTEIN1 and PROTEIN2 and one antagonist, PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 family currently consists of two agonists, PROTEIN1 and PROTEIN2 and one antagonist, PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 family currently consists of two agonists, PROTEIN1 and PROTEIN2 and one antagonist, PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 family currently consists of two agonists, PROTEIN1 and PROTEIN2 and one antagonist, PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The binding of PROTEIN0 or PROTEIN1 to PROTEIN2 is an early step in PROTEIN3 signal transduction and blocking this interaction may therefore be a useful target for the development of new drugs.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The binding of PROTEIN0 or PROTEIN1 to PROTEIN2 is an early step in PROTEIN3 signal transduction and blocking this interaction may therefore be a useful target for the development of new drugs.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The binding of PROTEIN0 or PROTEIN1 to PROTEIN2 is an early step in PROTEIN3 signal transduction and blocking this interaction may therefore be a useful target for the development of new drugs.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The binding of PROTEIN0 or PROTEIN1 to PROTEIN2 is an early step in PROTEIN3 signal transduction and blocking this interaction may therefore be a useful target for the development of new drugs.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The binding of PROTEIN0 or PROTEIN1 to PROTEIN2 is an early step in PROTEIN3 signal transduction and blocking this interaction may therefore be a useful target for the development of new drugs.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The binding of PROTEIN0 or PROTEIN1 to PROTEIN2 is an early step in PROTEIN3 signal transduction and blocking this interaction may therefore be a useful target for the development of new drugs.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we report the three-dimensional structure of PROTEIN0 bound to the extracellular domain of PROTEIN1 PROTEIN2 at 2.5 A resolution.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we report the three-dimensional structure of PROTEIN0 bound to the extracellular domain of PROTEIN1 PROTEIN2 at 2.5 A resolution.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we report the three-dimensional structure of PROTEIN0 bound to the extracellular domain of PROTEIN1 PROTEIN2 at 2.5 A resolution.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binds to PROTEIN1 with a 1:1 stoichiometry.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The crystal structure shows that PROTEIN0 consists of three immunoglobulin-like domains which wrap around PROTEIN1 in a manner distinct from the structures of previously described cytokine-receptor complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The catalytic domain of activated PROTEIN0 PROTEIN1 is absolutely required for interaction with its specific inhibitor, PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The catalytic domain of activated PROTEIN0 PROTEIN1 is absolutely required for interaction with its specific inhibitor, PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The catalytic domain of activated PROTEIN0 PROTEIN1 is absolutely required for interaction with its specific inhibitor, PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The catalytic domain of activated PROTEIN0 PROTEIN1 is absolutely required for interaction with its specific inhibitor, PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The catalytic domain of activated PROTEIN0 PROTEIN1 is absolutely required for interaction with its specific inhibitor, PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The catalytic domain of activated PROTEIN0 PROTEIN1 is absolutely required for interaction with its specific inhibitor, PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, we describe the production of PROTEIN0 PROTEIN1 and wild-type and mutant human PROTEIN2 PROTEIN3 proteins in SF9 cells by the baculovirus expression system.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here, we describe the production of PROTEIN0 PROTEIN1 and wild-type and mutant human PROTEIN2 PROTEIN3 proteins in SF9 cells by the baculovirus expression system.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here, we describe the production of PROTEIN0 PROTEIN1 and wild-type and mutant human PROTEIN2 PROTEIN3 proteins in SF9 cells by the baculovirus expression system.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here, we describe the production of PROTEIN0 PROTEIN1 and wild-type and mutant human PROTEIN2 PROTEIN3 proteins in SF9 cells by the baculovirus expression system.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here, we describe the production of PROTEIN0 PROTEIN1 and wild-type and mutant human PROTEIN2 PROTEIN3 proteins in SF9 cells by the baculovirus expression system.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here, we describe the production of PROTEIN0 PROTEIN1 and wild-type and mutant human PROTEIN2 PROTEIN3 proteins in SF9 cells by the baculovirus expression system.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Wild-type PROTEIN0 as well as the PROTEIN1 mutant lacking the C-terminal hemopexin-like domain PROTEIN2 exhibit enzymatic activity upon cleavage of the prodomain by treatment with PROTEIN3 or 4-aminophenylmercuric acetate.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Wild-type PROTEIN0 as well as the PROTEIN1 mutant lacking the C-terminal hemopexin-like domain PROTEIN2 exhibit enzymatic activity upon cleavage of the prodomain by treatment with PROTEIN3 or 4-aminophenylmercuric acetate.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Wild-type PROTEIN0 as well as the PROTEIN1 mutant lacking the C-terminal hemopexin-like domain PROTEIN2 exhibit enzymatic activity upon cleavage of the prodomain by treatment with PROTEIN3 or 4-aminophenylmercuric acetate.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Wild-type PROTEIN0 as well as the PROTEIN1 mutant lacking the C-terminal hemopexin-like domain PROTEIN2 exhibit enzymatic activity upon cleavage of the prodomain by treatment with PROTEIN3 or 4-aminophenylmercuric acetate.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Wild-type PROTEIN0 as well as the PROTEIN1 mutant lacking the C-terminal hemopexin-like domain PROTEIN2 exhibit enzymatic activity upon cleavage of the prodomain by treatment with PROTEIN3 or 4-aminophenylmercuric acetate.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Wild-type PROTEIN0 as well as the PROTEIN1 mutant lacking the C-terminal hemopexin-like domain PROTEIN2 exhibit enzymatic activity upon cleavage of the prodomain by treatment with PROTEIN3 or 4-aminophenylmercuric acetate.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Deletion of the complete active-site PROTEIN0 within the catalytic domain, or mutation of a single His residue of the Zn2+ binding domain (His199), generates stable forms of PROTEIN1 proteins which are unable to digest PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Deletion of the complete active-site PROTEIN0 within the catalytic domain, or mutation of a single His residue of the Zn2+ binding domain (His199), generates stable forms of PROTEIN1 proteins which are unable to digest PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Deletion of the complete active-site PROTEIN0 within the catalytic domain, or mutation of a single His residue of the Zn2+ binding domain (His199), generates stable forms of PROTEIN1 proteins which are unable to digest PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Deletion of the complete active-site PROTEIN0 within the catalytic domain, or mutation of a single His residue of the Zn2+ binding domain (His199), generates stable forms of PROTEIN1 proteins which are unable to digest PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Deletion of the complete active-site PROTEIN0 within the catalytic domain, or mutation of a single His residue of the Zn2+ binding domain (His199), generates stable forms of PROTEIN1 proteins which are unable to digest PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Deletion of the complete active-site PROTEIN0 within the catalytic domain, or mutation of a single His residue of the Zn2+ binding domain (His199), generates stable forms of PROTEIN1 proteins which are unable to digest PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition to co-immunoprecipitation analysis, we have established a rapid and sensitive ELISA assay using immobilized PROTEIN0 to determine the structural requirements of PROTEIN1 to form complexes with its inhibitor.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Only the activated and not the latent forms of wild-type and C-terminal mutant PROTEIN0 proteins are able to form complexes with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Neither mutation of His199, nor deletion mutants PROTEIN0 and PROTEIN1 interact with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Neither mutation of His199, nor deletion mutants PROTEIN0 and PROTEIN1 interact with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Neither mutation of His199, nor deletion mutants PROTEIN0 and PROTEIN1 interact with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 This demonstrates that the C-terminal hemopexin domain of PROTEIN0 in contrast to the corresponding regions of PROTEIN1 and PROTEIN2 does not interact with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 This demonstrates that the C-terminal hemopexin domain of PROTEIN0 in contrast to the corresponding regions of PROTEIN1 and PROTEIN2 does not interact with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 This demonstrates that the C-terminal hemopexin domain of PROTEIN0 in contrast to the corresponding regions of PROTEIN1 and PROTEIN2 does not interact with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 This demonstrates that the C-terminal hemopexin domain of PROTEIN0 in contrast to the corresponding regions of PROTEIN1 and PROTEIN2 does not interact with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 This demonstrates that the C-terminal hemopexin domain of PROTEIN0 in contrast to the corresponding regions of PROTEIN1 and PROTEIN2 does not interact with PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 This demonstrates that the C-terminal hemopexin domain of PROTEIN0 in contrast to the corresponding regions of PROTEIN1 and PROTEIN2 does not interact with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 In summary, we have shown that the integrity of the catalytic domain of PROTEIN0 and its ability to bind Zn2+ is absolutely required for complex formation with PROTEIN1 which further underlines the importance of this region for proper regulation of enzymatic activity of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In summary, we have shown that the integrity of the catalytic domain of PROTEIN0 and its ability to bind Zn2+ is absolutely required for complex formation with PROTEIN1 which further underlines the importance of this region for proper regulation of enzymatic activity of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In summary, we have shown that the integrity of the catalytic domain of PROTEIN0 and its ability to bind Zn2+ is absolutely required for complex formation with PROTEIN1 which further underlines the importance of this region for proper regulation of enzymatic activity of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thymocyte activation induces the association of PROTEIN0 and PROTEIN1 with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thymocyte activation induces the association of PROTEIN0 and PROTEIN1 with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thymocyte activation induces the association of PROTEIN0 and PROTEIN1 with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Antibody-mediated cross-linking studies or studies on PROTEIN0 knockout mice implicate PROTEIN1 as a co-stimulatory or negative regulatory molecule.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Y429 and Y441 occur in an imperfect PROTEIN0 PROTEIN1 sequence.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We investigated whether PROTEIN0 which binds to tyrosine-phosphorylated PROTEIN1 interacts with PROTEIN2 following T cell activation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We investigated whether PROTEIN0 which binds to tyrosine-phosphorylated PROTEIN1 interacts with PROTEIN2 following T cell activation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We investigated whether PROTEIN0 which binds to tyrosine-phosphorylated PROTEIN1 interacts with PROTEIN2 following T cell activation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 activity and the regulatory PROTEIN1 subunit of PROTEIN2 associated with PROTEIN3 in pervanadate-stimulated, but not in unstimulated thymocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 activity and the regulatory PROTEIN1 subunit of PROTEIN2 associated with PROTEIN3 in pervanadate-stimulated, but not in unstimulated thymocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 activity and the regulatory PROTEIN1 subunit of PROTEIN2 associated with PROTEIN3 in pervanadate-stimulated, but not in unstimulated thymocytes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 activity and the regulatory PROTEIN1 subunit of PROTEIN2 associated with PROTEIN3 in pervanadate-stimulated, but not in unstimulated thymocytes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 activity and the regulatory PROTEIN1 subunit of PROTEIN2 associated with PROTEIN3 in pervanadate-stimulated, but not in unstimulated thymocytes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 activity and the regulatory PROTEIN1 subunit of PROTEIN2 associated with PROTEIN3 in pervanadate-stimulated, but not in unstimulated thymocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cellular PROTEIN0 as well as the recombinant Src homology 2 (SH2) domains of PROTEIN1 bound a tyrosine-phosphorylated peptide encompassing Y463 with approximately threefold greater affinity than a doubly tyrosine-phosphorylated Y429-Y441 peptide.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A PROTEIN0 PROTEIN1 associated with PROTEIN2 and specifically with the Y429-Y441 phosphopeptide in stimulated thymocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A PROTEIN0 PROTEIN1 associated with PROTEIN2 and specifically with the Y429-Y441 phosphopeptide in stimulated thymocytes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A PROTEIN0 PROTEIN1 associated with PROTEIN2 and specifically with the Y429-Y441 phosphopeptide in stimulated thymocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We conclude that stimulation of thymocytes with pervanadate induces the recruitment of PROTEIN0 and PROTEIN1 to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 We conclude that stimulation of thymocytes with pervanadate induces the recruitment of PROTEIN0 and PROTEIN1 to PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 We conclude that stimulation of thymocytes with pervanadate induces the recruitment of PROTEIN0 and PROTEIN1 to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 As PROTEIN0 forms complexes with beta 1 integrins and the integrins are known to regulate keratinocyte behaviour, we investigated PROTEIN1 expression and function in human epidermal keratinocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 There was extensive co-localisation of PROTEIN0 and beta 1 integrins on microvilli and at cell-cell borders of basal keratinocytes; however, in contrast to the integrins, PROTEIN1 was not found in focal adhesions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 was detected in beta 1 integrin immunoprecipitates and also in immunoprecipitates of PROTEIN1 and PROTEIN2 but not of cadherins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 was detected in beta 1 integrin immunoprecipitates and also in immunoprecipitates of PROTEIN1 and PROTEIN2 but not of cadherins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 was detected in beta 1 integrin immunoprecipitates and also in immunoprecipitates of PROTEIN1 and PROTEIN2 but not of cadherins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 was associated with alpha 3 beta 1 but not alpha 5 beta 1; small amounts of PROTEIN1 also co-immunoprecipitated with antibodies to alpha 2 beta 1 and alpha 6 beta 4.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Antibodies to PROTEIN0 did not affect the proportion of keratinocytes that adhered to PROTEIN1 PROTEIN2 and PROTEIN3 but did inhibit motility of keratinocytes on tissue culture plastic.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Antibodies to PROTEIN0 did not affect the proportion of keratinocytes that adhered to PROTEIN1 PROTEIN2 and PROTEIN3 but did inhibit motility of keratinocytes on tissue culture plastic.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Antibodies to PROTEIN0 did not affect the proportion of keratinocytes that adhered to PROTEIN1 PROTEIN2 and PROTEIN3 but did inhibit motility of keratinocytes on tissue culture plastic.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Antibodies to PROTEIN0 did not affect the proportion of keratinocytes that adhered to PROTEIN1 PROTEIN2 and PROTEIN3 but did inhibit motility of keratinocytes on tissue culture plastic.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Antibodies to PROTEIN0 did not affect the proportion of keratinocytes that adhered to PROTEIN1 PROTEIN2 and PROTEIN3 but did inhibit motility of keratinocytes on tissue culture plastic.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Antibodies to PROTEIN0 did not affect the proportion of keratinocytes that adhered to PROTEIN1 PROTEIN2 and PROTEIN3 but did inhibit motility of keratinocytes on tissue culture plastic.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Like antibodies to the PROTEIN0 subunit, PROTEIN1 inhibited suspension-induced terminal differentiation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 prevents PROTEIN1 PROTEIN2 containing PROTEIN3 complexes formation in terminally differentiated myocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 prevents PROTEIN1 PROTEIN2 containing PROTEIN3 complexes formation in terminally differentiated myocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 prevents PROTEIN1 PROTEIN2 containing PROTEIN3 complexes formation in terminally differentiated myocytes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 prevents PROTEIN1 PROTEIN2 containing PROTEIN3 complexes formation in terminally differentiated myocytes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 prevents PROTEIN1 PROTEIN2 containing PROTEIN3 complexes formation in terminally differentiated myocytes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 prevents PROTEIN1 PROTEIN2 containing PROTEIN3 complexes formation in terminally differentiated myocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN2 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN3 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN4 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN5 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN6 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN2 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN3 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN4 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN5 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN6 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN3 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN4 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN5 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN6 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN4 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN5 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN6 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN5 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN6 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN5 PROTEIN6 Withdrawal from the cell cycle of differentiating myocytes is regulated by the PROTEIN0 PROTEIN1 protein PROTEIN2 and the pocket proteins PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN1 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN0 PROTEIN2 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN0 PROTEIN3 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN0 PROTEIN4 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN0 PROTEIN5 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN0 PROTEIN6 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN0 PROTEIN7 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN0 PROTEIN8 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN1 PROTEIN2 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN1 PROTEIN3 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN1 PROTEIN4 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN1 PROTEIN5 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN1 PROTEIN6 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN1 PROTEIN7 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN1 PROTEIN8 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN2 PROTEIN3 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN2 PROTEIN4 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN2 PROTEIN5 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN2 PROTEIN6 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN2 PROTEIN7 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN2 PROTEIN8 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN3 PROTEIN4 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN3 PROTEIN5 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN3 PROTEIN6 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN3 PROTEIN7 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN3 PROTEIN8 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN4 PROTEIN5 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN4 PROTEIN6 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN4 PROTEIN7 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN4 PROTEIN8 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN5 PROTEIN6 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN5 PROTEIN7 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN5 PROTEIN8 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN6 PROTEIN7 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN6 PROTEIN8 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN7 PROTEIN8 An PROTEIN0 complex containing mainly PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complex) appears when DNA synthesis arrests, replacing the PROTEIN5 PROTEIN6 containing PROTEIN7 complex of proliferating myoblasts PROTEIN8 complex).\n",
      "\n",
      "PROTEIN0 PROTEIN1 In differentiating C2C12 cells, PROTEIN0 complexes switch and DNA synthesis in response to serum are prevented when PROTEIN1 DNA binding activity and the cdks inhibitor PROTEIN2 downstream effector PROTEIN3 are induced.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In differentiating C2C12 cells, PROTEIN0 complexes switch and DNA synthesis in response to serum are prevented when PROTEIN1 DNA binding activity and the cdks inhibitor PROTEIN2 downstream effector PROTEIN3 are induced.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In differentiating C2C12 cells, PROTEIN0 complexes switch and DNA synthesis in response to serum are prevented when PROTEIN1 DNA binding activity and the cdks inhibitor PROTEIN2 downstream effector PROTEIN3 are induced.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In differentiating C2C12 cells, PROTEIN0 complexes switch and DNA synthesis in response to serum are prevented when PROTEIN1 DNA binding activity and the cdks inhibitor PROTEIN2 downstream effector PROTEIN3 are induced.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In differentiating C2C12 cells, PROTEIN0 complexes switch and DNA synthesis in response to serum are prevented when PROTEIN1 DNA binding activity and the cdks inhibitor PROTEIN2 downstream effector PROTEIN3 are induced.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In differentiating C2C12 cells, PROTEIN0 complexes switch and DNA synthesis in response to serum are prevented when PROTEIN1 DNA binding activity and the cdks inhibitor PROTEIN2 downstream effector PROTEIN3 are induced.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, during myogenic differentiation, formation of PROTEIN0 and PROTEIN1 PROTEIN2 containing complexes is an early event, but not enough on its own to prevent the reactivation of DNA synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, during myogenic differentiation, formation of PROTEIN0 and PROTEIN1 PROTEIN2 containing complexes is an early event, but not enough on its own to prevent the reactivation of DNA synthesis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, during myogenic differentiation, formation of PROTEIN0 and PROTEIN1 PROTEIN2 containing complexes is an early event, but not enough on its own to prevent the reactivation of DNA synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our data indicate that, in differentiating myocytes, one mechanism through which PROTEIN0 induces permanent cell cycle arrest involves PROTEIN1 upregulation and suppression of the proliferation-associated cdks-containing PROTEIN2 complexes formation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Our data indicate that, in differentiating myocytes, one mechanism through which PROTEIN0 induces permanent cell cycle arrest involves PROTEIN1 upregulation and suppression of the proliferation-associated cdks-containing PROTEIN2 complexes formation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Our data indicate that, in differentiating myocytes, one mechanism through which PROTEIN0 induces permanent cell cycle arrest involves PROTEIN1 upregulation and suppression of the proliferation-associated cdks-containing PROTEIN2 complexes formation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We now show that full-length PROTEIN0 interacts with activated PROTEIN1 in mammalian cells and defines a minimum region of 434 aa required for efficient PROTEIN2 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We now show that full-length PROTEIN0 interacts with activated PROTEIN1 in mammalian cells and defines a minimum region of 434 aa required for efficient PROTEIN2 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We now show that full-length PROTEIN0 interacts with activated PROTEIN1 in mammalian cells and defines a minimum region of 434 aa required for efficient PROTEIN2 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 interacts with the \"effector domain\" of PROTEIN1 and employs some PROTEIN2 determinants that are common to, and others that are distinct from, those required for the binding of PROTEIN3 a known PROTEIN4 effector.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 interacts with the \"effector domain\" of PROTEIN1 and employs some PROTEIN2 determinants that are common to, and others that are distinct from, those required for the binding of PROTEIN3 a known PROTEIN4 effector.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 interacts with the \"effector domain\" of PROTEIN1 and employs some PROTEIN2 determinants that are common to, and others that are distinct from, those required for the binding of PROTEIN3 a known PROTEIN4 effector.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 interacts with the \"effector domain\" of PROTEIN1 and employs some PROTEIN2 determinants that are common to, and others that are distinct from, those required for the binding of PROTEIN3 a known PROTEIN4 effector.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 interacts with the \"effector domain\" of PROTEIN1 and employs some PROTEIN2 determinants that are common to, and others that are distinct from, those required for the binding of PROTEIN3 a known PROTEIN4 effector.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 interacts with the \"effector domain\" of PROTEIN1 and employs some PROTEIN2 determinants that are common to, and others that are distinct from, those required for the binding of PROTEIN3 a known PROTEIN4 effector.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 interacts with the \"effector domain\" of PROTEIN1 and employs some PROTEIN2 determinants that are common to, and others that are distinct from, those required for the binding of PROTEIN3 a known PROTEIN4 effector.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 interacts with the \"effector domain\" of PROTEIN1 and employs some PROTEIN2 determinants that are common to, and others that are distinct from, those required for the binding of PROTEIN3 a known PROTEIN4 effector.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 interacts with the \"effector domain\" of PROTEIN1 and employs some PROTEIN2 determinants that are common to, and others that are distinct from, those required for the binding of PROTEIN3 a known PROTEIN4 effector.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 interacts with the \"effector domain\" of PROTEIN1 and employs some PROTEIN2 determinants that are common to, and others that are distinct from, those required for the binding of PROTEIN3 a known PROTEIN4 effector.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The same domain of PROTEIN0 that binds PROTEIN1 also interacts with 14-3-3 proteins, extending the similarity between PROTEIN2 and other PROTEIN3 effectors.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The same domain of PROTEIN0 that binds PROTEIN1 also interacts with 14-3-3 proteins, extending the similarity between PROTEIN2 and other PROTEIN3 effectors.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The same domain of PROTEIN0 that binds PROTEIN1 also interacts with 14-3-3 proteins, extending the similarity between PROTEIN2 and other PROTEIN3 effectors.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The same domain of PROTEIN0 that binds PROTEIN1 also interacts with 14-3-3 proteins, extending the similarity between PROTEIN2 and other PROTEIN3 effectors.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The same domain of PROTEIN0 that binds PROTEIN1 also interacts with 14-3-3 proteins, extending the similarity between PROTEIN2 and other PROTEIN3 effectors.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The same domain of PROTEIN0 that binds PROTEIN1 also interacts with 14-3-3 proteins, extending the similarity between PROTEIN2 and other PROTEIN3 effectors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 When expressed in mammalian cells, the PROTEIN0 binding domain of PROTEIN1 can act as a dominant negative signal transduction blocker.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This PROTEIN0 sequence shows preferential binding to the PROTEIN1 domain in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover, the amino-terminal domain of PROTEIN0 directly associates with, and is tyrosine phosphorylated by, PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Through their interaction with the ErbB family of receptors PROTEIN0 PROTEIN1 and PROTEIN2 neuregulins help to regulate cell growth and differentiation in many tissues.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Through their interaction with the ErbB family of receptors PROTEIN0 PROTEIN1 and PROTEIN2 neuregulins help to regulate cell growth and differentiation in many tissues.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Through their interaction with the ErbB family of receptors PROTEIN0 PROTEIN1 and PROTEIN2 neuregulins help to regulate cell growth and differentiation in many tissues.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The encoded product of the PROTEIN0 gene has a motif structure similar to that of neuregulins and an alternative splicing site in the PROTEIN1 PROTEIN2 domain gives rise to two isoforms (alpha and beta).\n",
      "\n",
      "PROTEIN0 PROTEIN2 The encoded product of the PROTEIN0 gene has a motif structure similar to that of neuregulins and an alternative splicing site in the PROTEIN1 PROTEIN2 domain gives rise to two isoforms (alpha and beta).\n",
      "\n",
      "PROTEIN1 PROTEIN2 The encoded product of the PROTEIN0 gene has a motif structure similar to that of neuregulins and an alternative splicing site in the PROTEIN1 PROTEIN2 domain gives rise to two isoforms (alpha and beta).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Recombinant PROTEIN0 induces the tyrosine-phosphorylation of PROTEIN1 PROTEIN2 and PROTEIN3 in cell lines expressing all of these ErbB-family receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Recombinant PROTEIN0 induces the tyrosine-phosphorylation of PROTEIN1 PROTEIN2 and PROTEIN3 in cell lines expressing all of these ErbB-family receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Recombinant PROTEIN0 induces the tyrosine-phosphorylation of PROTEIN1 PROTEIN2 and PROTEIN3 in cell lines expressing all of these ErbB-family receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Recombinant PROTEIN0 induces the tyrosine-phosphorylation of PROTEIN1 PROTEIN2 and PROTEIN3 in cell lines expressing all of these ErbB-family receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Recombinant PROTEIN0 induces the tyrosine-phosphorylation of PROTEIN1 PROTEIN2 and PROTEIN3 in cell lines expressing all of these ErbB-family receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Recombinant PROTEIN0 induces the tyrosine-phosphorylation of PROTEIN1 PROTEIN2 and PROTEIN3 in cell lines expressing all of these ErbB-family receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN3 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN4 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN5 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN3 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN4 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN5 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN3 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN4 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN5 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN3 PROTEIN4 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN3 PROTEIN5 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN4 PROTEIN5 However, in cell lines with defined combinations of ErbBs, PROTEIN0 only activates those with PROTEIN1 and/or PROTEIN2 suggesting that signalling by PROTEIN3 is mediated by PROTEIN4 and/or PROTEIN5 receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, previous work indicated that PROTEIN0 and PROTEIN1 might be involved in PROTEIN2 repression.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In addition, previous work indicated that PROTEIN0 and PROTEIN1 might be involved in PROTEIN2 repression.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In addition, previous work indicated that PROTEIN0 and PROTEIN1 might be involved in PROTEIN2 repression.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We show that the physical interaction between PROTEIN0 and PROTEIN1 a coactivator for PROTEIN2 is not sufficient for repression of PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We show that the physical interaction between PROTEIN0 and PROTEIN1 a coactivator for PROTEIN2 is not sufficient for repression of PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We show that the physical interaction between PROTEIN0 and PROTEIN1 a coactivator for PROTEIN2 is not sufficient for repression of PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We show that the physical interaction between PROTEIN0 and PROTEIN1 a coactivator for PROTEIN2 is not sufficient for repression of PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We show that the physical interaction between PROTEIN0 and PROTEIN1 a coactivator for PROTEIN2 is not sufficient for repression of PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We show that the physical interaction between PROTEIN0 and PROTEIN1 a coactivator for PROTEIN2 is not sufficient for repression of PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Repression of PROTEIN0 transcription may be accomplished through direct physical interaction between PROTEIN1 and this activator.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, physical interaction is not necessary for PROTEIN0 to repress PROTEIN1 and PROTEIN2 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In contrast, physical interaction is not necessary for PROTEIN0 to repress PROTEIN1 and PROTEIN2 transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In contrast, physical interaction is not necessary for PROTEIN0 to repress PROTEIN1 and PROTEIN2 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Proto-oncoprotein PROTEIN0 interacts with PROTEIN1 in activated thymocytes and peripheral T cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The molecular adapter PROTEIN0 is rapidly tyrosine phosphorylated following stimulation through the PROTEIN1 and associates with PROTEIN2 homology domain-2 (SH2)/SH3 domain-containing adapters such as PROTEIN3 PROTEIN4 and PROTEIN5 which interact with guanine nucleotide exchange factors specific for the Ras family.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This suggests that PROTEIN0 may link PROTEIN1 activation to molecules that regulate GTP binding proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The SH2/SH3-containing protein PROTEIN0 also contains a guanine nucleotide exchange factor domain, and PROTEIN1 has a crucial role in thymocyte development and activation of peripheral T cells following stimulation through the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The SH2/SH3-containing protein PROTEIN0 also contains a guanine nucleotide exchange factor domain, and PROTEIN1 has a crucial role in thymocyte development and activation of peripheral T cells following stimulation through the PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The SH2/SH3-containing protein PROTEIN0 also contains a guanine nucleotide exchange factor domain, and PROTEIN1 has a crucial role in thymocyte development and activation of peripheral T cells following stimulation through the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we show that PROTEIN0 and PROTEIN1 form inducible molecular complexes in PROTEIN2 murine thymocytes and peripheral T cells as well as pervanadate-treated T cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we show that PROTEIN0 and PROTEIN1 form inducible molecular complexes in PROTEIN2 murine thymocytes and peripheral T cells as well as pervanadate-treated T cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we show that PROTEIN0 and PROTEIN1 form inducible molecular complexes in PROTEIN2 murine thymocytes and peripheral T cells as well as pervanadate-treated T cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 interactions are also detectable in freshly isolated T cells from gene-targeted mice that lack the T cell-specific inhibitory receptor PROTEIN2 in which PROTEIN3 is hyperphosphorylated on tyrosine residues.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 interactions are also detectable in freshly isolated T cells from gene-targeted mice that lack the T cell-specific inhibitory receptor PROTEIN2 in which PROTEIN3 is hyperphosphorylated on tyrosine residues.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 interactions are also detectable in freshly isolated T cells from gene-targeted mice that lack the T cell-specific inhibitory receptor PROTEIN2 in which PROTEIN3 is hyperphosphorylated on tyrosine residues.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 interactions are also detectable in freshly isolated T cells from gene-targeted mice that lack the T cell-specific inhibitory receptor PROTEIN2 in which PROTEIN3 is hyperphosphorylated on tyrosine residues.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 interactions are also detectable in freshly isolated T cells from gene-targeted mice that lack the T cell-specific inhibitory receptor PROTEIN2 in which PROTEIN3 is hyperphosphorylated on tyrosine residues.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 interactions are also detectable in freshly isolated T cells from gene-targeted mice that lack the T cell-specific inhibitory receptor PROTEIN2 in which PROTEIN3 is hyperphosphorylated on tyrosine residues.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The interaction between PROTEIN0 and PROTEIN1 is directly mediated via the SH2 domain of PROTEIN2 and is dependent on tyrosine phosphorylation of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The interaction between PROTEIN0 and PROTEIN1 is directly mediated via the SH2 domain of PROTEIN2 and is dependent on tyrosine phosphorylation of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The interaction between PROTEIN0 and PROTEIN1 is directly mediated via the SH2 domain of PROTEIN2 and is dependent on tyrosine phosphorylation of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The interaction between PROTEIN0 and PROTEIN1 is directly mediated via the SH2 domain of PROTEIN2 and is dependent on tyrosine phosphorylation of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The interaction between PROTEIN0 and PROTEIN1 is directly mediated via the SH2 domain of PROTEIN2 and is dependent on tyrosine phosphorylation of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The interaction between PROTEIN0 and PROTEIN1 is directly mediated via the SH2 domain of PROTEIN2 and is dependent on tyrosine phosphorylation of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, we show that the conserved motif Y699 MTP present in PROTEIN0 is the binding site for the PROTEIN1 SH2 domain in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These data imply that PROTEIN0 is a molecular adapter that regulates the function of PROTEIN1 in thymocytes and peripheral T cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Induction of cell migration by PROTEIN0 cleavage of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Specific cleavage of PROTEIN0 PROTEIN1 by PROTEIN2 PROTEIN3 was shown to induce migration of breast epithelial cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Specific cleavage of PROTEIN0 PROTEIN1 by PROTEIN2 PROTEIN3 was shown to induce migration of breast epithelial cells.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Specific cleavage of PROTEIN0 PROTEIN1 by PROTEIN2 PROTEIN3 was shown to induce migration of breast epithelial cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Specific cleavage of PROTEIN0 PROTEIN1 by PROTEIN2 PROTEIN3 was shown to induce migration of breast epithelial cells.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Specific cleavage of PROTEIN0 PROTEIN1 by PROTEIN2 PROTEIN3 was shown to induce migration of breast epithelial cells.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Specific cleavage of PROTEIN0 PROTEIN1 by PROTEIN2 PROTEIN3 was shown to induce migration of breast epithelial cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cleavage of PROTEIN0 by PROTEIN1 and the resulting activation of the PROTEIN2 cryptic site may provide new targets for modulation of tumor cell invasion and tissue remodeling.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Cleavage of PROTEIN0 by PROTEIN1 and the resulting activation of the PROTEIN2 cryptic site may provide new targets for modulation of tumor cell invasion and tissue remodeling.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Cleavage of PROTEIN0 by PROTEIN1 and the resulting activation of the PROTEIN2 cryptic site may provide new targets for modulation of tumor cell invasion and tissue remodeling.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 complex: structural basis for GTPase activation and its loss in oncogenic PROTEIN2 mutants.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 PROTEIN1 complex: structural basis for GTPase activation and its loss in oncogenic PROTEIN2 mutants.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 PROTEIN1 complex: structural basis for GTPase activation and its loss in oncogenic PROTEIN2 mutants.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The three-dimensional structure of the complex between human PROTEIN0 bound to guanosine diphosphate and the guanosine triphosphatase (GTPase)-activating domain of the human PROTEIN1 PROTEIN2 PROTEIN3 in the presence of aluminum fluoride was solved at a resolution of 2.5 angstroms.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The three-dimensional structure of the complex between human PROTEIN0 bound to guanosine diphosphate and the guanosine triphosphatase (GTPase)-activating domain of the human PROTEIN1 PROTEIN2 PROTEIN3 in the presence of aluminum fluoride was solved at a resolution of 2.5 angstroms.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The three-dimensional structure of the complex between human PROTEIN0 bound to guanosine diphosphate and the guanosine triphosphatase (GTPase)-activating domain of the human PROTEIN1 PROTEIN2 PROTEIN3 in the presence of aluminum fluoride was solved at a resolution of 2.5 angstroms.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The three-dimensional structure of the complex between human PROTEIN0 bound to guanosine diphosphate and the guanosine triphosphatase (GTPase)-activating domain of the human PROTEIN1 PROTEIN2 PROTEIN3 in the presence of aluminum fluoride was solved at a resolution of 2.5 angstroms.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The three-dimensional structure of the complex between human PROTEIN0 bound to guanosine diphosphate and the guanosine triphosphatase (GTPase)-activating domain of the human PROTEIN1 PROTEIN2 PROTEIN3 in the presence of aluminum fluoride was solved at a resolution of 2.5 angstroms.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The three-dimensional structure of the complex between human PROTEIN0 bound to guanosine diphosphate and the guanosine triphosphatase (GTPase)-activating domain of the human PROTEIN1 PROTEIN2 PROTEIN3 in the presence of aluminum fluoride was solved at a resolution of 2.5 angstroms.\n",
      "\n",
      "PROTEIN0 PROTEIN1 An arginine side chain (arginine-789) of PROTEIN0 is supplied into the active site of PROTEIN1 to neutralize developing charges in the transition state.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The switch II region of PROTEIN0 is stabilized by PROTEIN1 thus allowing glutamine-61 of PROTEIN2 mutation of which activates the oncogenic potential, to participate in catalysis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The switch II region of PROTEIN0 is stabilized by PROTEIN1 thus allowing glutamine-61 of PROTEIN2 mutation of which activates the oncogenic potential, to participate in catalysis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The switch II region of PROTEIN0 is stabilized by PROTEIN1 thus allowing glutamine-61 of PROTEIN2 mutation of which activates the oncogenic potential, to participate in catalysis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Glycine-12 in the transition state mimic is within van der Waals distance of both arginine-789 of PROTEIN0 and glutamine-61 of PROTEIN1 and even its mutation to alanine would disturb the arrangements of residues in the transition state.\n",
      "\n",
      "PROTEIN0 PROTEIN1 When human soluble PROTEIN0 (sFasL, containing the extracellular portion) was expressed in human embryo kidney 293 cells, the three N-linked glycans of each PROTEIN1 monomer were found to be essential for efficient secretion.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Based on the structure of the closely related PROTEIN0 PROTEIN1 complex, a molecular model of the PROTEIN2 homotrimer bound to three PROTEIN3 molecules was generated using knowledge-based protein modeling methods.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Based on the structure of the closely related PROTEIN0 PROTEIN1 complex, a molecular model of the PROTEIN2 homotrimer bound to three PROTEIN3 molecules was generated using knowledge-based protein modeling methods.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Based on the structure of the closely related PROTEIN0 PROTEIN1 complex, a molecular model of the PROTEIN2 homotrimer bound to three PROTEIN3 molecules was generated using knowledge-based protein modeling methods.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Based on the structure of the closely related PROTEIN0 PROTEIN1 complex, a molecular model of the PROTEIN2 homotrimer bound to three PROTEIN3 molecules was generated using knowledge-based protein modeling methods.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Based on the structure of the closely related PROTEIN0 PROTEIN1 complex, a molecular model of the PROTEIN2 homotrimer bound to three PROTEIN3 molecules was generated using knowledge-based protein modeling methods.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Based on the structure of the closely related PROTEIN0 PROTEIN1 complex, a molecular model of the PROTEIN2 homotrimer bound to three PROTEIN3 molecules was generated using knowledge-based protein modeling methods.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mutants P206R, P206D, and P206F displayed reduced cytotoxicity toward PROTEIN0 cells with a concomitant decrease in the binding affinity for the recombinant PROTEIN1 Fc fusion proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Although the cytotoxic activity of mutant Y218D was unaltered, mutant Y218R was inactive, correlating with the prediction that Tyr-218 of PROTEIN0 interacts with a cluster of three basic amino acid side chains of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 is a hematopoietin important for megakaryocyte proliferation and production of blood platelets.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We sought to characterize how PROTEIN0 binds and activates its receptor, PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 domain of PROTEIN1 (TPO1-153) has been fused to the PROTEIN2 coat protein of M13 bacteriophage.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 domain of PROTEIN1 (TPO1-153) has been fused to the PROTEIN2 coat protein of M13 bacteriophage.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 domain of PROTEIN1 (TPO1-153) has been fused to the PROTEIN2 coat protein of M13 bacteriophage.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Phage enzyme-linked immunosorbent assay was used to determine affinities for binding to both the PROTEIN0 receptor and five PROTEIN1 monoclonal antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, the other three monoclonal antibodies, which were effective at blocking PROTEIN0 activity but did not block initial binding of PROTEIN1 to its receptor, had epitopes predominantly on helix or 3.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 has two distinct receptor-binding sites that function to dimerize PROTEIN1 receptors in a sequential fashion.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 negatively regulates PROTEIN1 activity by interactions with the PROTEIN2 cysteine-rich domain.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 negatively regulates PROTEIN1 activity by interactions with the PROTEIN2 cysteine-rich domain.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 negatively regulates PROTEIN1 activity by interactions with the PROTEIN2 cysteine-rich domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Although PROTEIN0 is a critical effector of PROTEIN1 signaling and transformation, the mechanism by which PROTEIN2 promotes PROTEIN3 activation is complex and remains poorly understood.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Although PROTEIN0 is a critical effector of PROTEIN1 signaling and transformation, the mechanism by which PROTEIN2 promotes PROTEIN3 activation is complex and remains poorly understood.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Although PROTEIN0 is a critical effector of PROTEIN1 signaling and transformation, the mechanism by which PROTEIN2 promotes PROTEIN3 activation is complex and remains poorly understood.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Although PROTEIN0 is a critical effector of PROTEIN1 signaling and transformation, the mechanism by which PROTEIN2 promotes PROTEIN3 activation is complex and remains poorly understood.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Although PROTEIN0 is a critical effector of PROTEIN1 signaling and transformation, the mechanism by which PROTEIN2 promotes PROTEIN3 activation is complex and remains poorly understood.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Although PROTEIN0 is a critical effector of PROTEIN1 signaling and transformation, the mechanism by which PROTEIN2 promotes PROTEIN3 activation is complex and remains poorly understood.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 is located in the NH2-terminal negative regulatory domain of PROTEIN1 and is highly homologous to cysteine-rich domains found in protein kinase C family members.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Recent studies indicate that the structural integrity of the PROTEIN0 is also critical for PROTEIN1 interaction with 14-3-3 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, whether 14-3-3 proteins interact directly with the PROTEIN0 and how this interaction may mediate PROTEIN1 function has not been determined.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the present study, we demonstrate that PROTEIN0 binds directly to the isolated PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover, mutation of PROTEIN0 residues 143-145 impairs binding of 14-3-3, but not PROTEIN1 to the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Moreover, mutation of PROTEIN0 residues 143-145 impairs binding of 14-3-3, but not PROTEIN1 to the PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Moreover, mutation of PROTEIN0 residues 143-145 impairs binding of 14-3-3, but not PROTEIN1 to the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, 14-3-3 interaction with the PROTEIN0 may serve in negative regulation of PROTEIN1 function by facilitating dissociation of 14-3-3 from the NH2 terminus of PROTEIN2 to promote subsequent events necessary for full activation of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, 14-3-3 interaction with the PROTEIN0 may serve in negative regulation of PROTEIN1 function by facilitating dissociation of 14-3-3 from the NH2 terminus of PROTEIN2 to promote subsequent events necessary for full activation of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Thus, 14-3-3 interaction with the PROTEIN0 may serve in negative regulation of PROTEIN1 function by facilitating dissociation of 14-3-3 from the NH2 terminus of PROTEIN2 to promote subsequent events necessary for full activation of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, 14-3-3 interaction with the PROTEIN0 may serve in negative regulation of PROTEIN1 function by facilitating dissociation of 14-3-3 from the NH2 terminus of PROTEIN2 to promote subsequent events necessary for full activation of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Thus, 14-3-3 interaction with the PROTEIN0 may serve in negative regulation of PROTEIN1 function by facilitating dissociation of 14-3-3 from the NH2 terminus of PROTEIN2 to promote subsequent events necessary for full activation of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Thus, 14-3-3 interaction with the PROTEIN0 may serve in negative regulation of PROTEIN1 function by facilitating dissociation of 14-3-3 from the NH2 terminus of PROTEIN2 to promote subsequent events necessary for full activation of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Two distinct ligand binding domains of PROTEIN0 were isolated from proteolytic digests of the receptor by affinity separation on immobilized PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The protein product of PROTEIN0 proto-oncogene is known to interact with several proteins, including PROTEIN1 PROTEIN2 and PROTEIN3 and is thought to regulate signalling by many cell surface receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The protein product of PROTEIN0 proto-oncogene is known to interact with several proteins, including PROTEIN1 PROTEIN2 and PROTEIN3 and is thought to regulate signalling by many cell surface receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The protein product of PROTEIN0 proto-oncogene is known to interact with several proteins, including PROTEIN1 PROTEIN2 and PROTEIN3 and is thought to regulate signalling by many cell surface receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The protein product of PROTEIN0 proto-oncogene is known to interact with several proteins, including PROTEIN1 PROTEIN2 and PROTEIN3 and is thought to regulate signalling by many cell surface receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The protein product of PROTEIN0 proto-oncogene is known to interact with several proteins, including PROTEIN1 PROTEIN2 and PROTEIN3 and is thought to regulate signalling by many cell surface receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The protein product of PROTEIN0 proto-oncogene is known to interact with several proteins, including PROTEIN1 PROTEIN2 and PROTEIN3 and is thought to regulate signalling by many cell surface receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The screen identified interactions involving PROTEIN0 and two 14-3-3 isoforms, PROTEIN1 human unconventional myosin IC, and a recently identified SH3 domain containing protein, PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The screen identified interactions involving PROTEIN0 and two 14-3-3 isoforms, PROTEIN1 human unconventional myosin IC, and a recently identified SH3 domain containing protein, PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The screen identified interactions involving PROTEIN0 and two 14-3-3 isoforms, PROTEIN1 human unconventional myosin IC, and a recently identified SH3 domain containing protein, PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 PROTEIN2 surface expression correlates with the binding of an MHC class I leader peptide.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 and PROTEIN1 PROTEIN2 surface expression correlates with the binding of an MHC class I leader peptide.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 and PROTEIN1 PROTEIN2 surface expression correlates with the binding of an MHC class I leader peptide.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We demonstrate that PROTEIN0 surface expression is correlated with the presence of MHC class I molecules which provide suitable leader sequence peptides capable of binding to PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Further studies on the interaction of PROTEIN0 with molecules in the endoplasmic reticulum revealed that PROTEIN1 associates with the PROTEIN2 PROTEIN3 and PROTEIN4 and that PROTEIN5 expression is PROTEIN6 and tapasin-dependent.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, PROTEIN0 dissociates from PROTEIN1 upon binding of MHC class I leader sequence peptides.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The correlation between PROTEIN0 and MHC class I surface expression might be relevant to the function of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our results also show that, although these PROTEIN0 binding peptides are derived from signal sequences, they may be released back into the cytosol and subsequently translocated by the PROTEIN1 complex and loaded onto PROTEIN2 molecules.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Our results also show that, although these PROTEIN0 binding peptides are derived from signal sequences, they may be released back into the cytosol and subsequently translocated by the PROTEIN1 complex and loaded onto PROTEIN2 molecules.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Our results also show that, although these PROTEIN0 binding peptides are derived from signal sequences, they may be released back into the cytosol and subsequently translocated by the PROTEIN1 complex and loaded onto PROTEIN2 molecules.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 inhibits PROTEIN1 signaling by preventing formation of a PROTEIN2 PROTEIN3 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 inhibits PROTEIN1 signaling by preventing formation of a PROTEIN2 PROTEIN3 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 inhibits PROTEIN1 signaling by preventing formation of a PROTEIN2 PROTEIN3 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 inhibits PROTEIN1 signaling by preventing formation of a PROTEIN2 PROTEIN3 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 inhibits PROTEIN1 signaling by preventing formation of a PROTEIN2 PROTEIN3 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 inhibits PROTEIN1 signaling by preventing formation of a PROTEIN2 PROTEIN3 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A binary complex of birch PROTEIN0 and PROTEIN1 could be isolated by gel chromatography.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Birch PROTEIN0 increased the critical concentration required for PROTEIN1 and PROTEIN2 polymerization in a concentration-dependent manner, supporting the notion of the formation of a heterologous complex between the plant protein and animal PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Birch PROTEIN0 increased the critical concentration required for PROTEIN1 and PROTEIN2 polymerization in a concentration-dependent manner, supporting the notion of the formation of a heterologous complex between the plant protein and animal PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Birch PROTEIN0 increased the critical concentration required for PROTEIN1 and PROTEIN2 polymerization in a concentration-dependent manner, supporting the notion of the formation of a heterologous complex between the plant protein and animal PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Birch PROTEIN0 increased the critical concentration required for PROTEIN1 and PROTEIN2 polymerization in a concentration-dependent manner, supporting the notion of the formation of a heterologous complex between the plant protein and animal PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Birch PROTEIN0 increased the critical concentration required for PROTEIN1 and PROTEIN2 polymerization in a concentration-dependent manner, supporting the notion of the formation of a heterologous complex between the plant protein and animal PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Birch PROTEIN0 increased the critical concentration required for PROTEIN1 and PROTEIN2 polymerization in a concentration-dependent manner, supporting the notion of the formation of a heterologous complex between the plant protein and animal PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interaction of plant PROTEIN0 with mammalian PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In viscometric assays, the birch protein was seen to modulate PROTEIN0 filament formation analogous to animal PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 These data indicate that the PROTEIN0 domains in plant and animal PROTEIN1 are functionally highly conserved, although the overall sequence similarity is less than 25%.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Abnormal immunoreactivity in the tumor tissue was observed in 18 patients (34.0%) for PROTEIN0 in 35 (66.0%) for PROTEIN1 in 20 (37.7%) for PROTEIN2 and in 37 (69.8%) for PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Abnormal immunoreactivity in the tumor tissue was observed in 18 patients (34.0%) for PROTEIN0 in 35 (66.0%) for PROTEIN1 in 20 (37.7%) for PROTEIN2 and in 37 (69.8%) for PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Abnormal immunoreactivity in the tumor tissue was observed in 18 patients (34.0%) for PROTEIN0 in 35 (66.0%) for PROTEIN1 in 20 (37.7%) for PROTEIN2 and in 37 (69.8%) for PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Abnormal immunoreactivity in the tumor tissue was observed in 18 patients (34.0%) for PROTEIN0 in 35 (66.0%) for PROTEIN1 in 20 (37.7%) for PROTEIN2 and in 37 (69.8%) for PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Abnormal immunoreactivity in the tumor tissue was observed in 18 patients (34.0%) for PROTEIN0 in 35 (66.0%) for PROTEIN1 in 20 (37.7%) for PROTEIN2 and in 37 (69.8%) for PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Abnormal immunoreactivity in the tumor tissue was observed in 18 patients (34.0%) for PROTEIN0 in 35 (66.0%) for PROTEIN1 in 20 (37.7%) for PROTEIN2 and in 37 (69.8%) for PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Absence of PROTEIN0 leads to structurally aberrant neuromuscular synapses deficient in PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, PROTEIN0 has an important role in synapse formation and in the organization of PROTEIN1 PROTEIN2 and PROTEIN3 at the neuromuscular synapse.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, PROTEIN0 has an important role in synapse formation and in the organization of PROTEIN1 PROTEIN2 and PROTEIN3 at the neuromuscular synapse.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Thus, PROTEIN0 has an important role in synapse formation and in the organization of PROTEIN1 PROTEIN2 and PROTEIN3 at the neuromuscular synapse.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, PROTEIN0 has an important role in synapse formation and in the organization of PROTEIN1 PROTEIN2 and PROTEIN3 at the neuromuscular synapse.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Thus, PROTEIN0 has an important role in synapse formation and in the organization of PROTEIN1 PROTEIN2 and PROTEIN3 at the neuromuscular synapse.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Thus, PROTEIN0 has an important role in synapse formation and in the organization of PROTEIN1 PROTEIN2 and PROTEIN3 at the neuromuscular synapse.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN0 PROTEIN7 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN1 PROTEIN7 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN2 PROTEIN7 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN3 PROTEIN7 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN4 PROTEIN7 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN5 PROTEIN7 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN6 PROTEIN7 Abundance of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 is not affected by the differential PROTEIN7 expression.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Acanthamoeba PROTEIN0 and PROTEIN1 can be cross-linked between glutamic acid 364 of PROTEIN2 and lysine 115 of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Acanthamoeba PROTEIN0 and PROTEIN1 can be cross-linked between glutamic acid 364 of PROTEIN2 and lysine 115 of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Acanthamoeba PROTEIN0 and PROTEIN1 can be cross-linked between glutamic acid 364 of PROTEIN2 and lysine 115 of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Acanthamoeba PROTEIN0 and PROTEIN1 can be cross-linked between glutamic acid 364 of PROTEIN2 and lysine 115 of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Acanthamoeba PROTEIN0 and PROTEIN1 can be cross-linked between glutamic acid 364 of PROTEIN2 and lysine 115 of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Acanthamoeba PROTEIN0 and PROTEIN1 can be cross-linked between glutamic acid 364 of PROTEIN2 and lysine 115 of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Acanthamoeba PROTEIN0 was cross-linked to PROTEIN1 via a zero-length isopeptide bond using carbodiimide.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Since the NH2 terminus of PROTEIN0 is blocked, Edman degradation gave only the sequence of the conjugated PROTEIN1 CNBr fragment beginning with Trp-356.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Acanthamoeba PROTEIN0 affects the mechanical properties of nonfilamentous PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, PROTEIN0 had little effect on the rigidity and viscosity of PROTEIN1 filaments.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We speculate that nonfilamentous PROTEIN0 and PROTEIN1 both of which form shear-sensitive structures, can be modeled as flocculant materials.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Acanthamoeba PROTEIN0 has similar but weaker effects on PROTEIN1 requiring 5 to 10 times more PROTEIN2 than with amoeba PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Acanthamoeba PROTEIN0 has similar but weaker effects on PROTEIN1 requiring 5 to 10 times more PROTEIN2 than with amoeba PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Acanthamoeba PROTEIN0 has similar but weaker effects on PROTEIN1 requiring 5 to 10 times more PROTEIN2 than with amoeba PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Acanthamoeba PROTEIN0 has similar but weaker effects on PROTEIN1 requiring 5 to 10 times more PROTEIN2 than with amoeba PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Acanthamoeba PROTEIN0 has similar but weaker effects on PROTEIN1 requiring 5 to 10 times more PROTEIN2 than with amoeba PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Acanthamoeba PROTEIN0 has similar but weaker effects on PROTEIN1 requiring 5 to 10 times more PROTEIN2 than with amoeba PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Acanthamoeba PROTEIN0 strongly inhibits in a concentration-dependent fashion the rate and extent of Acanthamoeba PROTEIN1 polymerization in 50 mM KCl.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Addition of PROTEIN0 to polymerized PROTEIN1 causes it to depolymerize until a new steady-state, dependent on PROTEIN2 concentration, is achieved.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Addition of PROTEIN0 to polymerized PROTEIN1 causes it to depolymerize until a new steady-state, dependent on PROTEIN2 concentration, is achieved.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Addition of PROTEIN0 to polymerized PROTEIN1 causes it to depolymerize until a new steady-state, dependent on PROTEIN2 concentration, is achieved.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mechanism of action of Acanthamoeba PROTEIN0 demonstration of PROTEIN1 species specificity and regulation by micromolar concentrations of MgCl2.\n",
      "\n",
      "PROTEIN0 PROTEIN1 MgCl2 strongly inhibits these effects of PROTEIN0 most likely by binding to the high-affinity divalent cation site on the PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Accordingly, PROTEIN0 is also found in these structures, again in the absence of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition to its presence in cell-cell adhesion sites, we have found PROTEIN0 but not PROTEIN1 in the Z-discs of heart and skeletal striated muscles.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A chimera consisting of the PROTEIN0 region of PROTEIN1 linked to the amino terminus of PROTEIN2 behaves as a monomer in solution, as expected, since PROTEIN3 binding disrupts the PROTEIN4 dimer.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A chimera consisting of the PROTEIN0 region of PROTEIN1 linked to the amino terminus of PROTEIN2 behaves as a monomer in solution, as expected, since PROTEIN3 binding disrupts the PROTEIN4 dimer.\n",
      "\n",
      "PROTEIN0 PROTEIN3 A chimera consisting of the PROTEIN0 region of PROTEIN1 linked to the amino terminus of PROTEIN2 behaves as a monomer in solution, as expected, since PROTEIN3 binding disrupts the PROTEIN4 dimer.\n",
      "\n",
      "PROTEIN0 PROTEIN4 A chimera consisting of the PROTEIN0 region of PROTEIN1 linked to the amino terminus of PROTEIN2 behaves as a monomer in solution, as expected, since PROTEIN3 binding disrupts the PROTEIN4 dimer.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A chimera consisting of the PROTEIN0 region of PROTEIN1 linked to the amino terminus of PROTEIN2 behaves as a monomer in solution, as expected, since PROTEIN3 binding disrupts the PROTEIN4 dimer.\n",
      "\n",
      "PROTEIN1 PROTEIN3 A chimera consisting of the PROTEIN0 region of PROTEIN1 linked to the amino terminus of PROTEIN2 behaves as a monomer in solution, as expected, since PROTEIN3 binding disrupts the PROTEIN4 dimer.\n",
      "\n",
      "PROTEIN1 PROTEIN4 A chimera consisting of the PROTEIN0 region of PROTEIN1 linked to the amino terminus of PROTEIN2 behaves as a monomer in solution, as expected, since PROTEIN3 binding disrupts the PROTEIN4 dimer.\n",
      "\n",
      "PROTEIN2 PROTEIN3 A chimera consisting of the PROTEIN0 region of PROTEIN1 linked to the amino terminus of PROTEIN2 behaves as a monomer in solution, as expected, since PROTEIN3 binding disrupts the PROTEIN4 dimer.\n",
      "\n",
      "PROTEIN2 PROTEIN4 A chimera consisting of the PROTEIN0 region of PROTEIN1 linked to the amino terminus of PROTEIN2 behaves as a monomer in solution, as expected, since PROTEIN3 binding disrupts the PROTEIN4 dimer.\n",
      "\n",
      "PROTEIN3 PROTEIN4 A chimera consisting of the PROTEIN0 region of PROTEIN1 linked to the amino terminus of PROTEIN2 behaves as a monomer in solution, as expected, since PROTEIN3 binding disrupts the PROTEIN4 dimer.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A common feature of these PROTEIN0 domains is the deep membrane infoldings, where bundles of PROTEIN1 filaments are inserted into the membrane.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A core activity associated with the N terminus of the yeast PROTEIN0 protein is revealed by PROTEIN1 overexpression suppression of C-terminal PROTEIN2 truncation alleles.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A core activity associated with the N terminus of the yeast PROTEIN0 protein is revealed by PROTEIN1 overexpression suppression of C-terminal PROTEIN2 truncation alleles.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A core activity associated with the N terminus of the yeast PROTEIN0 protein is revealed by PROTEIN1 overexpression suppression of C-terminal PROTEIN2 truncation alleles.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 such as PROTEIN1 and PROTEIN2 bind to phosphatidylinositol (PI) 4,5-bisphosphate (PI 4,5-P2) and regulate the concentration of monomeric PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 such as PROTEIN1 and PROTEIN2 bind to phosphatidylinositol (PI) 4,5-bisphosphate (PI 4,5-P2) and regulate the concentration of monomeric PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 such as PROTEIN1 and PROTEIN2 bind to phosphatidylinositol (PI) 4,5-bisphosphate (PI 4,5-P2) and regulate the concentration of monomeric PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 such as PROTEIN1 and PROTEIN2 bind to phosphatidylinositol (PI) 4,5-bisphosphate (PI 4,5-P2) and regulate the concentration of monomeric PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 such as PROTEIN1 and PROTEIN2 bind to phosphatidylinositol (PI) 4,5-bisphosphate (PI 4,5-P2) and regulate the concentration of monomeric PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 such as PROTEIN1 and PROTEIN2 bind to phosphatidylinositol (PI) 4,5-bisphosphate (PI 4,5-P2) and regulate the concentration of monomeric PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 These studies suggest that PROTEIN0 and PROTEIN1 may control the generation of 3-OH phosphorylated phosphoinositides, which in turn may regulate the PROTEIN2 polymerization.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These studies suggest that PROTEIN0 and PROTEIN1 may control the generation of 3-OH phosphorylated phosphoinositides, which in turn may regulate the PROTEIN2 polymerization.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These studies suggest that PROTEIN0 and PROTEIN1 may control the generation of 3-OH phosphorylated phosphoinositides, which in turn may regulate the PROTEIN2 polymerization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This effect is specific to PROTEIN0 and PROTEIN1 because other cytoskeletal proteins such as PROTEIN2 or PROTEIN3 do not affect PROTEIN4 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 This effect is specific to PROTEIN0 and PROTEIN1 because other cytoskeletal proteins such as PROTEIN2 or PROTEIN3 do not affect PROTEIN4 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 This effect is specific to PROTEIN0 and PROTEIN1 because other cytoskeletal proteins such as PROTEIN2 or PROTEIN3 do not affect PROTEIN4 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN4 This effect is specific to PROTEIN0 and PROTEIN1 because other cytoskeletal proteins such as PROTEIN2 or PROTEIN3 do not affect PROTEIN4 activity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 This effect is specific to PROTEIN0 and PROTEIN1 because other cytoskeletal proteins such as PROTEIN2 or PROTEIN3 do not affect PROTEIN4 activity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 This effect is specific to PROTEIN0 and PROTEIN1 because other cytoskeletal proteins such as PROTEIN2 or PROTEIN3 do not affect PROTEIN4 activity.\n",
      "\n",
      "PROTEIN1 PROTEIN4 This effect is specific to PROTEIN0 and PROTEIN1 because other cytoskeletal proteins such as PROTEIN2 or PROTEIN3 do not affect PROTEIN4 activity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 This effect is specific to PROTEIN0 and PROTEIN1 because other cytoskeletal proteins such as PROTEIN2 or PROTEIN3 do not affect PROTEIN4 activity.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PROTEIN2 PROTEIN4 This effect is specific to PROTEIN0 and PROTEIN1 because other cytoskeletal proteins such as PROTEIN2 or PROTEIN3 do not affect PROTEIN4 activity.\n",
      "\n",
      "PROTEIN3 PROTEIN4 This effect is specific to PROTEIN0 and PROTEIN1 because other cytoskeletal proteins such as PROTEIN2 or PROTEIN3 do not affect PROTEIN4 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 and PROTEIN2 define a family of PROTEIN3 essential for the rapid turnover of filamentous PROTEIN4 in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 define a family of PROTEIN3 essential for the rapid turnover of filamentous PROTEIN4 in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 and PROTEIN2 define a family of PROTEIN3 essential for the rapid turnover of filamentous PROTEIN4 in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 PROTEIN1 and PROTEIN2 define a family of PROTEIN3 essential for the rapid turnover of filamentous PROTEIN4 in vivo.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 define a family of PROTEIN3 essential for the rapid turnover of filamentous PROTEIN4 in vivo.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 and PROTEIN2 define a family of PROTEIN3 essential for the rapid turnover of filamentous PROTEIN4 in vivo.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 PROTEIN1 and PROTEIN2 define a family of PROTEIN3 essential for the rapid turnover of filamentous PROTEIN4 in vivo.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 and PROTEIN2 define a family of PROTEIN3 essential for the rapid turnover of filamentous PROTEIN4 in vivo.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 PROTEIN1 and PROTEIN2 define a family of PROTEIN3 essential for the rapid turnover of filamentous PROTEIN4 in vivo.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 PROTEIN1 and PROTEIN2 define a family of PROTEIN3 essential for the rapid turnover of filamentous PROTEIN4 in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 expression was reduced by 25-50%, relative to that of PROTEIN1 in smooth muscle of KO mice, without any change in the relative distribution of the PROTEIN2 isoforms.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 expression was reduced by 25-50%, relative to that of PROTEIN1 in smooth muscle of KO mice, without any change in the relative distribution of the PROTEIN2 isoforms.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 expression was reduced by 25-50%, relative to that of PROTEIN1 in smooth muscle of KO mice, without any change in the relative distribution of the PROTEIN2 isoforms.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 isoforms from different species were also compared, and the effect of PROTEIN1 on the thermal stability of PROTEIN2 was studied.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 isoforms from different species were also compared, and the effect of PROTEIN1 on the thermal stability of PROTEIN2 was studied.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 isoforms from different species were also compared, and the effect of PROTEIN1 on the thermal stability of PROTEIN2 was studied.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Active, Ser133-phosphorylated PROTEIN0 effects transcription of CRE-dependent genes via interaction with the 265-kDa co-activator protein PROTEIN1 PROTEIN2 which bridges the PROTEIN3 complex to components of the basal transcriptional apparatus.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Active, Ser133-phosphorylated PROTEIN0 effects transcription of CRE-dependent genes via interaction with the 265-kDa co-activator protein PROTEIN1 PROTEIN2 which bridges the PROTEIN3 complex to components of the basal transcriptional apparatus.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Active, Ser133-phosphorylated PROTEIN0 effects transcription of CRE-dependent genes via interaction with the 265-kDa co-activator protein PROTEIN1 PROTEIN2 which bridges the PROTEIN3 complex to components of the basal transcriptional apparatus.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Active, Ser133-phosphorylated PROTEIN0 effects transcription of CRE-dependent genes via interaction with the 265-kDa co-activator protein PROTEIN1 PROTEIN2 which bridges the PROTEIN3 complex to components of the basal transcriptional apparatus.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Active, Ser133-phosphorylated PROTEIN0 effects transcription of CRE-dependent genes via interaction with the 265-kDa co-activator protein PROTEIN1 PROTEIN2 which bridges the PROTEIN3 complex to components of the basal transcriptional apparatus.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Active, Ser133-phosphorylated PROTEIN0 effects transcription of CRE-dependent genes via interaction with the 265-kDa co-activator protein PROTEIN1 PROTEIN2 which bridges the PROTEIN3 complex to components of the basal transcriptional apparatus.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 have opposite effects on the rate of exchange of nucleotide bound to PROTEIN2 monomers.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 and PROTEIN1 have opposite effects on the rate of exchange of nucleotide bound to PROTEIN2 monomers.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 and PROTEIN1 have opposite effects on the rate of exchange of nucleotide bound to PROTEIN2 monomers.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Addition of EGTA (3 mM) to the extracellular media completely inhibited the cleavage of PROTEIN0 PROTEIN1 and PROTEIN2 in response to A23187 (1 microM).\n",
      "\n",
      "PROTEIN0 PROTEIN2 Addition of EGTA (3 mM) to the extracellular media completely inhibited the cleavage of PROTEIN0 PROTEIN1 and PROTEIN2 in response to A23187 (1 microM).\n",
      "\n",
      "PROTEIN1 PROTEIN2 Addition of EGTA (3 mM) to the extracellular media completely inhibited the cleavage of PROTEIN0 PROTEIN1 and PROTEIN2 in response to A23187 (1 microM).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Adherens junction proteins PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and the proliferation marker PROTEIN6 were localized in the cultures by fluorescence microscopy, and in vitro wound healing was compared.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A distinct modulating domain in PROTEIN0 monomers in the repression of activity of the transcription factor PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 ADP-ribosylation of PROTEIN0 at Arg 177 by Clostridium perfringens PROTEIN1 increased the nucleotide dissociation rate from 2.2 x 10(-3) s-1 to 4.5 x 10(-3) s-1 without affecting the PROTEIN2 stimulation of nucleotide exchange.\n",
      "\n",
      "PROTEIN0 PROTEIN2 ADP-ribosylation of PROTEIN0 at Arg 177 by Clostridium perfringens PROTEIN1 increased the nucleotide dissociation rate from 2.2 x 10(-3) s-1 to 4.5 x 10(-3) s-1 without affecting the PROTEIN2 stimulation of nucleotide exchange.\n",
      "\n",
      "PROTEIN1 PROTEIN2 ADP-ribosylation of PROTEIN0 at Arg 177 by Clostridium perfringens PROTEIN1 increased the nucleotide dissociation rate from 2.2 x 10(-3) s-1 to 4.5 x 10(-3) s-1 without affecting the PROTEIN2 stimulation of nucleotide exchange.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, ADP-ribosylation of PROTEIN0 at Arg95/Arg372 induced by turkey PROTEIN1 decreased the nucleotide dissociation rate to 1.5 x 10(3) s-1 and inhibited the PROTEIN2 stimulation of nucleotide exchange.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In contrast, ADP-ribosylation of PROTEIN0 at Arg95/Arg372 induced by turkey PROTEIN1 decreased the nucleotide dissociation rate to 1.5 x 10(3) s-1 and inhibited the PROTEIN2 stimulation of nucleotide exchange.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In contrast, ADP-ribosylation of PROTEIN0 at Arg95/Arg372 induced by turkey PROTEIN1 decreased the nucleotide dissociation rate to 1.5 x 10(3) s-1 and inhibited the PROTEIN2 stimulation of nucleotide exchange.\n",
      "\n",
      "PROTEIN0 PROTEIN1 After 24 or 48 hours of shear, staining for PROTEIN0 PROTEIN1 and PROTEIN2 was intense and peripheral, forming a band of \"dashes\" (adherens plaques) that colocalized with the ends of stress fibers that inserted along the lateral membranes of cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 After 24 or 48 hours of shear, staining for PROTEIN0 PROTEIN1 and PROTEIN2 was intense and peripheral, forming a band of \"dashes\" (adherens plaques) that colocalized with the ends of stress fibers that inserted along the lateral membranes of cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 After 24 or 48 hours of shear, staining for PROTEIN0 PROTEIN1 and PROTEIN2 was intense and peripheral, forming a band of \"dashes\" (adherens plaques) that colocalized with the ends of stress fibers that inserted along the lateral membranes of cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 After digestion of the 60 kDa fragment with cyanogen bromide, the N-terminal 21-amino acid sequence of one of the resulting peptides was found to show sequence similarity to a region near the PROTEIN0 site (amino acid residues 260-281) of yeast PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 After elution of the three PROTEIN0 PROTEIN1 and PROTEIN2 were recovered from the PROTEIN3 column with 2 M urea solution.\n",
      "\n",
      "PROTEIN0 PROTEIN2 After elution of the three PROTEIN0 PROTEIN1 and PROTEIN2 were recovered from the PROTEIN3 column with 2 M urea solution.\n",
      "\n",
      "PROTEIN0 PROTEIN3 After elution of the three PROTEIN0 PROTEIN1 and PROTEIN2 were recovered from the PROTEIN3 column with 2 M urea solution.\n",
      "\n",
      "PROTEIN1 PROTEIN2 After elution of the three PROTEIN0 PROTEIN1 and PROTEIN2 were recovered from the PROTEIN3 column with 2 M urea solution.\n",
      "\n",
      "PROTEIN1 PROTEIN3 After elution of the three PROTEIN0 PROTEIN1 and PROTEIN2 were recovered from the PROTEIN3 column with 2 M urea solution.\n",
      "\n",
      "PROTEIN2 PROTEIN3 After elution of the three PROTEIN0 PROTEIN1 and PROTEIN2 were recovered from the PROTEIN3 column with 2 M urea solution.\n",
      "\n",
      "PROTEIN0 PROTEIN1 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN1 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN1 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN1 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN1 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN1 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN1 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN1 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN1 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN1 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN9 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN9 PROTEIN1 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN9 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN9 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN9 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN9 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN9 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN9 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN9 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN9 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN9 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN1 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN2 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN3 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN4 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN5 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN6 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN7 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN8 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN9 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN9 PROTEIN0 After in vitro labelling of isolated rat liver nuclei, the following ADP-ribosylated and unmodified PROTEIN0 were identified by HPLC and gel electrophoresis: PROTEIN1 PROTEIN2 four PROTEIN3 PROTEIN4 subfractions, PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 oxidized PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 three PROTEIN7 PROTEIN8 variants and PROTEIN9 PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN1 A further outcome of this study is that the PROTEIN0 filament core bundles in microvilli of chicken pigment epithelial cells are presumed to contain a crosslinking protein, which is not immunologically related to either PROTEIN1 PROTEIN2 or PROTEIN3 of the intestinal brush border.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A further outcome of this study is that the PROTEIN0 filament core bundles in microvilli of chicken pigment epithelial cells are presumed to contain a crosslinking protein, which is not immunologically related to either PROTEIN1 PROTEIN2 or PROTEIN3 of the intestinal brush border.\n",
      "\n",
      "PROTEIN0 PROTEIN3 A further outcome of this study is that the PROTEIN0 filament core bundles in microvilli of chicken pigment epithelial cells are presumed to contain a crosslinking protein, which is not immunologically related to either PROTEIN1 PROTEIN2 or PROTEIN3 of the intestinal brush border.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A further outcome of this study is that the PROTEIN0 filament core bundles in microvilli of chicken pigment epithelial cells are presumed to contain a crosslinking protein, which is not immunologically related to either PROTEIN1 PROTEIN2 or PROTEIN3 of the intestinal brush border.\n",
      "\n",
      "PROTEIN1 PROTEIN3 A further outcome of this study is that the PROTEIN0 filament core bundles in microvilli of chicken pigment epithelial cells are presumed to contain a crosslinking protein, which is not immunologically related to either PROTEIN1 PROTEIN2 or PROTEIN3 of the intestinal brush border.\n",
      "\n",
      "PROTEIN2 PROTEIN3 A further outcome of this study is that the PROTEIN0 filament core bundles in microvilli of chicken pigment epithelial cells are presumed to contain a crosslinking protein, which is not immunologically related to either PROTEIN1 PROTEIN2 or PROTEIN3 of the intestinal brush border.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A highly significant correlation was observed between the loss of expression of PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 with increased TNM stage (p <0.05).\n",
      "\n",
      "PROTEIN0 PROTEIN2 A highly significant correlation was observed between the loss of expression of PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 with increased TNM stage (p <0.05).\n",
      "\n",
      "PROTEIN0 PROTEIN3 A highly significant correlation was observed between the loss of expression of PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 with increased TNM stage (p <0.05).\n",
      "\n",
      "PROTEIN1 PROTEIN2 A highly significant correlation was observed between the loss of expression of PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 with increased TNM stage (p <0.05).\n",
      "\n",
      "PROTEIN1 PROTEIN3 A highly significant correlation was observed between the loss of expression of PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 with increased TNM stage (p <0.05).\n",
      "\n",
      "PROTEIN2 PROTEIN3 A highly significant correlation was observed between the loss of expression of PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 with increased TNM stage (p <0.05).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Loss of normal surface PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 expression was found in 52/68 (76.4%) tumors, 57/68 (83.8%) tumors, 54/68 (79.4%) tumors and 54/68 (79.4%) tumors (p <0.001).\n",
      "\n",
      "PROTEIN0 PROTEIN2 Loss of normal surface PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 expression was found in 52/68 (76.4%) tumors, 57/68 (83.8%) tumors, 54/68 (79.4%) tumors and 54/68 (79.4%) tumors (p <0.001).\n",
      "\n",
      "PROTEIN0 PROTEIN3 Loss of normal surface PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 expression was found in 52/68 (76.4%) tumors, 57/68 (83.8%) tumors, 54/68 (79.4%) tumors and 54/68 (79.4%) tumors (p <0.001).\n",
      "\n",
      "PROTEIN1 PROTEIN2 Loss of normal surface PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 expression was found in 52/68 (76.4%) tumors, 57/68 (83.8%) tumors, 54/68 (79.4%) tumors and 54/68 (79.4%) tumors (p <0.001).\n",
      "\n",
      "PROTEIN1 PROTEIN3 Loss of normal surface PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 expression was found in 52/68 (76.4%) tumors, 57/68 (83.8%) tumors, 54/68 (79.4%) tumors and 54/68 (79.4%) tumors (p <0.001).\n",
      "\n",
      "PROTEIN2 PROTEIN3 Loss of normal surface PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 expression was found in 52/68 (76.4%) tumors, 57/68 (83.8%) tumors, 54/68 (79.4%) tumors and 54/68 (79.4%) tumors (p <0.001).\n",
      "\n",
      "PROTEIN0 PROTEIN1 MATERIALS AND METHODS: We used an PROTEIN0 immunoperoxidase technique to examine the cellular localization of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 in 68 TCC and 14 normal bladder biopsies.\n",
      "\n",
      "PROTEIN0 PROTEIN2 MATERIALS AND METHODS: We used an PROTEIN0 immunoperoxidase technique to examine the cellular localization of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 in 68 TCC and 14 normal bladder biopsies.\n",
      "\n",
      "PROTEIN0 PROTEIN3 MATERIALS AND METHODS: We used an PROTEIN0 immunoperoxidase technique to examine the cellular localization of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 in 68 TCC and 14 normal bladder biopsies.\n",
      "\n",
      "PROTEIN0 PROTEIN4 MATERIALS AND METHODS: We used an PROTEIN0 immunoperoxidase technique to examine the cellular localization of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 in 68 TCC and 14 normal bladder biopsies.\n",
      "\n",
      "PROTEIN1 PROTEIN2 MATERIALS AND METHODS: We used an PROTEIN0 immunoperoxidase technique to examine the cellular localization of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 in 68 TCC and 14 normal bladder biopsies.\n",
      "\n",
      "PROTEIN1 PROTEIN3 MATERIALS AND METHODS: We used an PROTEIN0 immunoperoxidase technique to examine the cellular localization of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 in 68 TCC and 14 normal bladder biopsies.\n",
      "\n",
      "PROTEIN1 PROTEIN4 MATERIALS AND METHODS: We used an PROTEIN0 immunoperoxidase technique to examine the cellular localization of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 in 68 TCC and 14 normal bladder biopsies.\n",
      "\n",
      "PROTEIN2 PROTEIN3 MATERIALS AND METHODS: We used an PROTEIN0 immunoperoxidase technique to examine the cellular localization of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 in 68 TCC and 14 normal bladder biopsies.\n",
      "\n",
      "PROTEIN2 PROTEIN4 MATERIALS AND METHODS: We used an PROTEIN0 immunoperoxidase technique to examine the cellular localization of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 in 68 TCC and 14 normal bladder biopsies.\n",
      "\n",
      "PROTEIN3 PROTEIN4 MATERIALS AND METHODS: We used an PROTEIN0 immunoperoxidase technique to examine the cellular localization of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 in 68 TCC and 14 normal bladder biopsies.\n",
      "\n",
      "PROTEIN0 PROTEIN1 RESULTS: PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were expressed in a normal membranous pattern in all normal bladder epithelium specimens.\n",
      "\n",
      "PROTEIN0 PROTEIN2 RESULTS: PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were expressed in a normal membranous pattern in all normal bladder epithelium specimens.\n",
      "\n",
      "PROTEIN0 PROTEIN3 RESULTS: PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were expressed in a normal membranous pattern in all normal bladder epithelium specimens.\n",
      "\n",
      "PROTEIN1 PROTEIN2 RESULTS: PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were expressed in a normal membranous pattern in all normal bladder epithelium specimens.\n",
      "\n",
      "PROTEIN1 PROTEIN3 RESULTS: PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were expressed in a normal membranous pattern in all normal bladder epithelium specimens.\n",
      "\n",
      "PROTEIN2 PROTEIN3 RESULTS: PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were expressed in a normal membranous pattern in all normal bladder epithelium specimens.\n",
      "\n",
      "PROTEIN0 PROTEIN1 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN0 PROTEIN2 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN0 PROTEIN3 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN0 PROTEIN4 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN0 PROTEIN5 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN1 PROTEIN2 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN1 PROTEIN3 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN1 PROTEIN4 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN1 PROTEIN5 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN2 PROTEIN3 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN2 PROTEIN4 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN2 PROTEIN5 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN3 PROTEIN4 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN3 PROTEIN5 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN4 PROTEIN5 AIMS: To investigate the disturbance of intercellular adhesion in adenoid cystic carcinoma (ACC), we examined the ultrastructural localization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 in ACC, and compared it with that in the normal labial gland.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 interacts with PROTEIN1 to disassemble PROTEIN2 filaments.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 interacts with PROTEIN1 to disassemble PROTEIN2 filaments.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 interacts with PROTEIN1 to disassemble PROTEIN2 filaments.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 localizes to cortical PROTEIN1 patches in yeast cells, and this localization is disrupted by specific PROTEIN2 and PROTEIN3 mutations.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 localizes to cortical PROTEIN1 patches in yeast cells, and this localization is disrupted by specific PROTEIN2 and PROTEIN3 mutations.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 localizes to cortical PROTEIN1 patches in yeast cells, and this localization is disrupted by specific PROTEIN2 and PROTEIN3 mutations.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 localizes to cortical PROTEIN1 patches in yeast cells, and this localization is disrupted by specific PROTEIN2 and PROTEIN3 mutations.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 localizes to cortical PROTEIN1 patches in yeast cells, and this localization is disrupted by specific PROTEIN2 and PROTEIN3 mutations.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 localizes to cortical PROTEIN1 patches in yeast cells, and this localization is disrupted by specific PROTEIN2 and PROTEIN3 mutations.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A two-hybrid-based approach using PROTEIN0 and PROTEIN1 mutants identified residues necessary for the interaction of PROTEIN2 PROTEIN3 and PROTEIN4 in an apparent ternary complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A two-hybrid-based approach using PROTEIN0 and PROTEIN1 mutants identified residues necessary for the interaction of PROTEIN2 PROTEIN3 and PROTEIN4 in an apparent ternary complex.\n",
      "\n",
      "PROTEIN0 PROTEIN3 A two-hybrid-based approach using PROTEIN0 and PROTEIN1 mutants identified residues necessary for the interaction of PROTEIN2 PROTEIN3 and PROTEIN4 in an apparent ternary complex.\n",
      "\n",
      "PROTEIN0 PROTEIN4 A two-hybrid-based approach using PROTEIN0 and PROTEIN1 mutants identified residues necessary for the interaction of PROTEIN2 PROTEIN3 and PROTEIN4 in an apparent ternary complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A two-hybrid-based approach using PROTEIN0 and PROTEIN1 mutants identified residues necessary for the interaction of PROTEIN2 PROTEIN3 and PROTEIN4 in an apparent ternary complex.\n",
      "\n",
      "PROTEIN1 PROTEIN3 A two-hybrid-based approach using PROTEIN0 and PROTEIN1 mutants identified residues necessary for the interaction of PROTEIN2 PROTEIN3 and PROTEIN4 in an apparent ternary complex.\n",
      "\n",
      "PROTEIN1 PROTEIN4 A two-hybrid-based approach using PROTEIN0 and PROTEIN1 mutants identified residues necessary for the interaction of PROTEIN2 PROTEIN3 and PROTEIN4 in an apparent ternary complex.\n",
      "\n",
      "PROTEIN2 PROTEIN3 A two-hybrid-based approach using PROTEIN0 and PROTEIN1 mutants identified residues necessary for the interaction of PROTEIN2 PROTEIN3 and PROTEIN4 in an apparent ternary complex.\n",
      "\n",
      "PROTEIN2 PROTEIN4 A two-hybrid-based approach using PROTEIN0 and PROTEIN1 mutants identified residues necessary for the interaction of PROTEIN2 PROTEIN3 and PROTEIN4 in an apparent ternary complex.\n",
      "\n",
      "PROTEIN3 PROTEIN4 A two-hybrid-based approach using PROTEIN0 and PROTEIN1 mutants identified residues necessary for the interaction of PROTEIN2 PROTEIN3 and PROTEIN4 in an apparent ternary complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, we report that PROTEIN0 also interacts with the ubiquitous PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here, we report that PROTEIN0 also interacts with the ubiquitous PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here, we report that PROTEIN0 also interacts with the ubiquitous PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 We conclude that PROTEIN0 is a PROTEIN1 protein that enhances the filament disassembly activity of PROTEIN2 and restricts PROTEIN3 localization to cortical PROTEIN4 patches.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We conclude that PROTEIN0 is a PROTEIN1 protein that enhances the filament disassembly activity of PROTEIN2 and restricts PROTEIN3 localization to cortical PROTEIN4 patches.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We conclude that PROTEIN0 is a PROTEIN1 protein that enhances the filament disassembly activity of PROTEIN2 and restricts PROTEIN3 localization to cortical PROTEIN4 patches.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We conclude that PROTEIN0 is a PROTEIN1 protein that enhances the filament disassembly activity of PROTEIN2 and restricts PROTEIN3 localization to cortical PROTEIN4 patches.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We conclude that PROTEIN0 is a PROTEIN1 protein that enhances the filament disassembly activity of PROTEIN2 and restricts PROTEIN3 localization to cortical PROTEIN4 patches.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We conclude that PROTEIN0 is a PROTEIN1 protein that enhances the filament disassembly activity of PROTEIN2 and restricts PROTEIN3 localization to cortical PROTEIN4 patches.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We conclude that PROTEIN0 is a PROTEIN1 protein that enhances the filament disassembly activity of PROTEIN2 and restricts PROTEIN3 localization to cortical PROTEIN4 patches.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We conclude that PROTEIN0 is a PROTEIN1 protein that enhances the filament disassembly activity of PROTEIN2 and restricts PROTEIN3 localization to cortical PROTEIN4 patches.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We conclude that PROTEIN0 is a PROTEIN1 protein that enhances the filament disassembly activity of PROTEIN2 and restricts PROTEIN3 localization to cortical PROTEIN4 patches.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We conclude that PROTEIN0 is a PROTEIN1 protein that enhances the filament disassembly activity of PROTEIN2 and restricts PROTEIN3 localization to cortical PROTEIN4 patches.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 A PROTEIN0 mutant with replacement of threonine 505 by valine abolished PROTEIN1 activities for PROTEIN2 phosphorylation and PROTEIN3 cytoskeletal changes, whereas replacement by glutamate enhanced the PROTEIN4 activity and stress fiber formation by PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 All of the experimental data were quantitatively accounted for on the basis of a 1:1 complex between PROTEIN0 and monomeric PROTEIN1 with a Kr between 4 and 9 microM, the same value obtained previously in the absence of MgCl2.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The interaction of Acanthamoeba PROTEIN0 and Acanthamoeba PROTEIN1 was evaluated as a function of ionic conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Although binding patterns of different sera were not identical, almost all sera caused PROTEIN0 binding to platelet components of 87-90 kD, 140 kD (identified as PROTEIN1 and 220-240 kD (tentatively identified as PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Although binding patterns of different sera were not identical, almost all sera caused PROTEIN0 binding to platelet components of 87-90 kD, 140 kD (identified as PROTEIN1 and 220-240 kD (tentatively identified as PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Although binding patterns of different sera were not identical, almost all sera caused PROTEIN0 binding to platelet components of 87-90 kD, 140 kD (identified as PROTEIN1 and 220-240 kD (tentatively identified as PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Although binding patterns of different sera were not identical, almost all sera caused PROTEIN0 binding to platelet components of 87-90 kD, 140 kD (identified as PROTEIN1 and 220-240 kD (tentatively identified as PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Although binding patterns of different sera were not identical, almost all sera caused PROTEIN0 binding to platelet components of 87-90 kD, 140 kD (identified as PROTEIN1 and 220-240 kD (tentatively identified as PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Although binding patterns of different sera were not identical, almost all sera caused PROTEIN0 binding to platelet components of 87-90 kD, 140 kD (identified as PROTEIN1 and 220-240 kD (tentatively identified as PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Determination of the total PROTEIN0 and PROTEIN1 concentrations in PMN revealed that the molar ratio of PROTEIN2 to PROTEIN3 was 1 to 5.2.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Determination of the total PROTEIN0 and PROTEIN1 concentrations in PMN revealed that the molar ratio of PROTEIN2 to PROTEIN3 was 1 to 5.2.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Determination of the total PROTEIN0 and PROTEIN1 concentrations in PMN revealed that the molar ratio of PROTEIN2 to PROTEIN3 was 1 to 5.2.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Determination of the total PROTEIN0 and PROTEIN1 concentrations in PMN revealed that the molar ratio of PROTEIN2 to PROTEIN3 was 1 to 5.2.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Determination of the total PROTEIN0 and PROTEIN1 concentrations in PMN revealed that the molar ratio of PROTEIN2 to PROTEIN3 was 1 to 5.2.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Determination of the total PROTEIN0 and PROTEIN1 concentrations in PMN revealed that the molar ratio of PROTEIN2 to PROTEIN3 was 1 to 5.2.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We examined the role of the PROTEIN0 monomer sequestering protein, PROTEIN1 in the regulation of PROTEIN2 filament assembly during chemoattractant stimulation using a Triton extraction method.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We examined the role of the PROTEIN0 monomer sequestering protein, PROTEIN1 in the regulation of PROTEIN2 filament assembly during chemoattractant stimulation using a Triton extraction method.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We examined the role of the PROTEIN0 monomer sequestering protein, PROTEIN1 in the regulation of PROTEIN2 filament assembly during chemoattractant stimulation using a Triton extraction method.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Although co-localization of PROTEIN0 with PROTEIN1 filaments was not clearly demonstrated in the pancreatic beta-cell line MIN6, PROTEIN2 was found to be present in both the nucleus and the cytoplasm, though predominantly in the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Although co-localization of PROTEIN0 with PROTEIN1 filaments was not clearly demonstrated in the pancreatic beta-cell line MIN6, PROTEIN2 was found to be present in both the nucleus and the cytoplasm, though predominantly in the nucleus.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Although co-localization of PROTEIN0 with PROTEIN1 filaments was not clearly demonstrated in the pancreatic beta-cell line MIN6, PROTEIN2 was found to be present in both the nucleus and the cytoplasm, though predominantly in the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A portion of PROTEIN0 appeared to be co-localized with PROTEIN1 filaments in the cytoplasm of cultured rat hepatocytes incubated with 25 mM glucose.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Co-localization of PROTEIN0 with PROTEIN1 filaments.\n",
      "\n",
      "PROTEIN0 PROTEIN1 When PROTEIN0 or PROTEIN1 was transiently expressed in COS-7 cells, the expressed PROTEIN2 was also co-localized with PROTEIN3 filaments in the cytoplasm of these transfected cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 When PROTEIN0 or PROTEIN1 was transiently expressed in COS-7 cells, the expressed PROTEIN2 was also co-localized with PROTEIN3 filaments in the cytoplasm of these transfected cells.\n",
      "\n",
      "PROTEIN0 PROTEIN3 When PROTEIN0 or PROTEIN1 was transiently expressed in COS-7 cells, the expressed PROTEIN2 was also co-localized with PROTEIN3 filaments in the cytoplasm of these transfected cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 When PROTEIN0 or PROTEIN1 was transiently expressed in COS-7 cells, the expressed PROTEIN2 was also co-localized with PROTEIN3 filaments in the cytoplasm of these transfected cells.\n",
      "\n",
      "PROTEIN1 PROTEIN3 When PROTEIN0 or PROTEIN1 was transiently expressed in COS-7 cells, the expressed PROTEIN2 was also co-localized with PROTEIN3 filaments in the cytoplasm of these transfected cells.\n",
      "\n",
      "PROTEIN2 PROTEIN3 When PROTEIN0 or PROTEIN1 was transiently expressed in COS-7 cells, the expressed PROTEIN2 was also co-localized with PROTEIN3 filaments in the cytoplasm of these transfected cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A major point of interest was to clarify whether rheological, high precision measurements can yield quantitative information about the influence of PROTEIN0 and PROTEIN1 on the structure, chain dynamics, elasticity and viscoelasticity of PROTEIN2 filaments with time.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A major point of interest was to clarify whether rheological, high precision measurements can yield quantitative information about the influence of PROTEIN0 and PROTEIN1 on the structure, chain dynamics, elasticity and viscoelasticity of PROTEIN2 filaments with time.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A major point of interest was to clarify whether rheological, high precision measurements can yield quantitative information about the influence of PROTEIN0 and PROTEIN1 on the structure, chain dynamics, elasticity and viscoelasticity of PROTEIN2 filaments with time.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The viscoelasticity of entangled PROTEIN0 networks: the influence of defects and modulation by PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The viscoelasticity of entangled PROTEIN0 networks: the influence of defects and modulation by PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The viscoelasticity of entangled PROTEIN0 networks: the influence of defects and modulation by PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 A model is proposed for the function of PROTEIN0 in PROTEIN1 motility.\n",
      "\n",
      "PROTEIN0 PROTEIN1 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN3 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN4 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN5 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN3 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN4 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN5 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN3 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN4 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN5 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN3 PROTEIN4 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN3 PROTEIN5 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN4 PROTEIN5 As a consequence, in the presence of PROTEIN0 PROTEIN1 low amounts of PROTEIN2 promote extensive PROTEIN3 assembly off of the pool of PROTEIN4 PROTEIN5 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 How PROTEIN0 promotes PROTEIN1 filament assembly in the presence of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 How PROTEIN0 promotes PROTEIN1 filament assembly in the presence of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 How PROTEIN0 promotes PROTEIN1 filament assembly in the presence of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The role of PROTEIN0 in the regulation of PROTEIN1 assembly has been reexamined.\n",
      "\n",
      "PROTEIN0 PROTEIN1 When barbed ends are capped, PROTEIN0 simply sequesters globular PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binds PROTEIN1 likely to promote PROTEIN2 polymerization.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 binds PROTEIN1 likely to promote PROTEIN2 polymerization.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 binds PROTEIN1 likely to promote PROTEIN2 polymerization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 plays a fundamental role in mispair recognition whereas PROTEIN1 and PROTEIN2 appear to modify the specificity of this recognition.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 plays a fundamental role in mispair recognition whereas PROTEIN1 and PROTEIN2 appear to modify the specificity of this recognition.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 plays a fundamental role in mispair recognition whereas PROTEIN1 and PROTEIN2 appear to modify the specificity of this recognition.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a PROTEIN1 homologue enriched in epithelial cells and forms a complex with PROTEIN2 a PROTEIN3 localized to the apical plasma membrane.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is a PROTEIN1 homologue enriched in epithelial cells and forms a complex with PROTEIN2 a PROTEIN3 localized to the apical plasma membrane.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 is a PROTEIN1 homologue enriched in epithelial cells and forms a complex with PROTEIN2 a PROTEIN3 localized to the apical plasma membrane.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is a PROTEIN1 homologue enriched in epithelial cells and forms a complex with PROTEIN2 a PROTEIN3 localized to the apical plasma membrane.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 is a PROTEIN1 homologue enriched in epithelial cells and forms a complex with PROTEIN2 a PROTEIN3 localized to the apical plasma membrane.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 is a PROTEIN1 homologue enriched in epithelial cells and forms a complex with PROTEIN2 a PROTEIN3 localized to the apical plasma membrane.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Even the PROTEIN0 mutants that do not detectably bind PROTEIN1 were membrane associated in Caco-2 cells, suggesting that the PROTEIN2 interaction is not the sole determinant of PROTEIN3 protein membrane attachment.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Even the PROTEIN0 mutants that do not detectably bind PROTEIN1 were membrane associated in Caco-2 cells, suggesting that the PROTEIN2 interaction is not the sole determinant of PROTEIN3 protein membrane attachment.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Even the PROTEIN0 mutants that do not detectably bind PROTEIN1 were membrane associated in Caco-2 cells, suggesting that the PROTEIN2 interaction is not the sole determinant of PROTEIN3 protein membrane attachment.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Even the PROTEIN0 mutants that do not detectably bind PROTEIN1 were membrane associated in Caco-2 cells, suggesting that the PROTEIN2 interaction is not the sole determinant of PROTEIN3 protein membrane attachment.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Even the PROTEIN0 mutants that do not detectably bind PROTEIN1 were membrane associated in Caco-2 cells, suggesting that the PROTEIN2 interaction is not the sole determinant of PROTEIN3 protein membrane attachment.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Even the PROTEIN0 mutants that do not detectably bind PROTEIN1 were membrane associated in Caco-2 cells, suggesting that the PROTEIN2 interaction is not the sole determinant of PROTEIN3 protein membrane attachment.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The in vitro and in vivo binding patterns were highly similar, and the association of the PROTEIN0 variants with PROTEIN1 correlated well with their ability to displace PROTEIN2 from PROTEIN3 complexes when overexpressed in Caco-2 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The in vitro and in vivo binding patterns were highly similar, and the association of the PROTEIN0 variants with PROTEIN1 correlated well with their ability to displace PROTEIN2 from PROTEIN3 complexes when overexpressed in Caco-2 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The in vitro and in vivo binding patterns were highly similar, and the association of the PROTEIN0 variants with PROTEIN1 correlated well with their ability to displace PROTEIN2 from PROTEIN3 complexes when overexpressed in Caco-2 cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The in vitro and in vivo binding patterns were highly similar, and the association of the PROTEIN0 variants with PROTEIN1 correlated well with their ability to displace PROTEIN2 from PROTEIN3 complexes when overexpressed in Caco-2 cells.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The in vitro and in vivo binding patterns were highly similar, and the association of the PROTEIN0 variants with PROTEIN1 correlated well with their ability to displace PROTEIN2 from PROTEIN3 complexes when overexpressed in Caco-2 cells.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The in vitro and in vivo binding patterns were highly similar, and the association of the PROTEIN0 variants with PROTEIN1 correlated well with their ability to displace PROTEIN2 from PROTEIN3 complexes when overexpressed in Caco-2 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This implies that PROTEIN0 function in apical membrane trafficking involves aspects independent of the PROTEIN1 interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Analyses of PROTEIN0 or PROTEIN1 immunoprecipitates from BMMC lysate revealed that PROTEIN2 PROTEIN3 and PROTEIN4 were co-precipitated, suggesting that PROTEIN5 and PROTEIN6 form a complex in mast cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Analysis of PROTEIN0 PROTEIN1 crystals reveals an extensive intermolecular network, rather than a discrete \"monomeric complex\", comprising stacked PROTEIN2 ribbons held in place by columns of PROTEIN3 molecules, wedged in between neighboring PROTEIN4 subunits and running perpendicular to the ribbons.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Analysis of PROTEIN0 PROTEIN1 crystals reveals an extensive intermolecular network, rather than a discrete \"monomeric complex\", comprising stacked PROTEIN2 ribbons held in place by columns of PROTEIN3 molecules, wedged in between neighboring PROTEIN4 subunits and running perpendicular to the ribbons.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Analysis of PROTEIN0 PROTEIN1 crystals reveals an extensive intermolecular network, rather than a discrete \"monomeric complex\", comprising stacked PROTEIN2 ribbons held in place by columns of PROTEIN3 molecules, wedged in between neighboring PROTEIN4 subunits and running perpendicular to the ribbons.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Analysis of PROTEIN0 PROTEIN1 crystals reveals an extensive intermolecular network, rather than a discrete \"monomeric complex\", comprising stacked PROTEIN2 ribbons held in place by columns of PROTEIN3 molecules, wedged in between neighboring PROTEIN4 subunits and running perpendicular to the ribbons.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Analysis of PROTEIN0 PROTEIN1 crystals reveals an extensive intermolecular network, rather than a discrete \"monomeric complex\", comprising stacked PROTEIN2 ribbons held in place by columns of PROTEIN3 molecules, wedged in between neighboring PROTEIN4 subunits and running perpendicular to the ribbons.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Analysis of PROTEIN0 PROTEIN1 crystals reveals an extensive intermolecular network, rather than a discrete \"monomeric complex\", comprising stacked PROTEIN2 ribbons held in place by columns of PROTEIN3 molecules, wedged in between neighboring PROTEIN4 subunits and running perpendicular to the ribbons.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Analysis of PROTEIN0 PROTEIN1 crystals reveals an extensive intermolecular network, rather than a discrete \"monomeric complex\", comprising stacked PROTEIN2 ribbons held in place by columns of PROTEIN3 molecules, wedged in between neighboring PROTEIN4 subunits and running perpendicular to the ribbons.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Analysis of PROTEIN0 PROTEIN1 crystals reveals an extensive intermolecular network, rather than a discrete \"monomeric complex\", comprising stacked PROTEIN2 ribbons held in place by columns of PROTEIN3 molecules, wedged in between neighboring PROTEIN4 subunits and running perpendicular to the ribbons.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Analysis of PROTEIN0 PROTEIN1 crystals reveals an extensive intermolecular network, rather than a discrete \"monomeric complex\", comprising stacked PROTEIN2 ribbons held in place by columns of PROTEIN3 molecules, wedged in between neighboring PROTEIN4 subunits and running perpendicular to the ribbons.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Analysis of PROTEIN0 PROTEIN1 crystals reveals an extensive intermolecular network, rather than a discrete \"monomeric complex\", comprising stacked PROTEIN2 ribbons held in place by columns of PROTEIN3 molecules, wedged in between neighboring PROTEIN4 subunits and running perpendicular to the ribbons.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Molecular packing in PROTEIN0 PROTEIN1 crystals and its implications.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Analysis of changes in state suggests that PROTEIN0 protein has a role in the establishment but not the maintenance of repression of silent mating type genes, whereas PROTEIN1 PROTEIN2 and PROTEIN3 are required for maintenance.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Analysis of changes in state suggests that PROTEIN0 protein has a role in the establishment but not the maintenance of repression of silent mating type genes, whereas PROTEIN1 PROTEIN2 and PROTEIN3 are required for maintenance.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Analysis of changes in state suggests that PROTEIN0 protein has a role in the establishment but not the maintenance of repression of silent mating type genes, whereas PROTEIN1 PROTEIN2 and PROTEIN3 are required for maintenance.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Analysis of changes in state suggests that PROTEIN0 protein has a role in the establishment but not the maintenance of repression of silent mating type genes, whereas PROTEIN1 PROTEIN2 and PROTEIN3 are required for maintenance.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Analysis of changes in state suggests that PROTEIN0 protein has a role in the establishment but not the maintenance of repression of silent mating type genes, whereas PROTEIN1 PROTEIN2 and PROTEIN3 are required for maintenance.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Analysis of changes in state suggests that PROTEIN0 protein has a role in the establishment but not the maintenance of repression of silent mating type genes, whereas PROTEIN1 PROTEIN2 and PROTEIN3 are required for maintenance.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The interaction between calf PROTEIN0 and PROTEIN1 depends critically on the status of the C-terminus of the PROTEIN2 and in the case of PROTEIN3 the C-terminus is of great importance for the physiochemical behaviour of the protein.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The interaction between calf PROTEIN0 and PROTEIN1 depends critically on the status of the C-terminus of the PROTEIN2 and in the case of PROTEIN3 the C-terminus is of great importance for the physiochemical behaviour of the protein.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The interaction between calf PROTEIN0 and PROTEIN1 depends critically on the status of the C-terminus of the PROTEIN2 and in the case of PROTEIN3 the C-terminus is of great importance for the physiochemical behaviour of the protein.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The interaction between calf PROTEIN0 and PROTEIN1 depends critically on the status of the C-terminus of the PROTEIN2 and in the case of PROTEIN3 the C-terminus is of great importance for the physiochemical behaviour of the protein.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The interaction between calf PROTEIN0 and PROTEIN1 depends critically on the status of the C-terminus of the PROTEIN2 and in the case of PROTEIN3 the C-terminus is of great importance for the physiochemical behaviour of the protein.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The interaction between calf PROTEIN0 and PROTEIN1 depends critically on the status of the C-terminus of the PROTEIN2 and in the case of PROTEIN3 the C-terminus is of great importance for the physiochemical behaviour of the protein.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Analysis of the u.v. absorbance and far-u.v. circular dichroism spectra of PROTEIN0 and PROTEIN1 did not reveal any major changes in the conformation of the proteins accompanying the modifications at the C-terminal ends.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Analysis of this product by peptide mapping (Sutoh (1982) Biochemistry 21, 3654-3661) showed that PROTEIN0 was cross-linked with the N-terminal segment of PROTEIN1 containing residues 1-12.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Purification of PROTEIN0 a 21,000 molecular weight PROTEIN1 from porcine kidney and identification of the PROTEIN2 site in the PROTEIN3 sequence.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Purification of PROTEIN0 a 21,000 molecular weight PROTEIN1 from porcine kidney and identification of the PROTEIN2 site in the PROTEIN3 sequence.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Purification of PROTEIN0 a 21,000 molecular weight PROTEIN1 from porcine kidney and identification of the PROTEIN2 site in the PROTEIN3 sequence.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Purification of PROTEIN0 a 21,000 molecular weight PROTEIN1 from porcine kidney and identification of the PROTEIN2 site in the PROTEIN3 sequence.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PROTEIN1 PROTEIN3 Purification of PROTEIN0 a 21,000 molecular weight PROTEIN1 from porcine kidney and identification of the PROTEIN2 site in the PROTEIN3 sequence.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Purification of PROTEIN0 a 21,000 molecular weight PROTEIN1 from porcine kidney and identification of the PROTEIN2 site in the PROTEIN3 sequence.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Analysis of V3, a hamster equivalent of SCID, indicates that the protein level increases of PROTEIN0 and PROTEIN1 from G0 to G1/S/G2 do not require PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Analysis of V3, a hamster equivalent of SCID, indicates that the protein level increases of PROTEIN0 and PROTEIN1 from G0 to G1/S/G2 do not require PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Analysis of V3, a hamster equivalent of SCID, indicates that the protein level increases of PROTEIN0 and PROTEIN1 from G0 to G1/S/G2 do not require PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this study, cell cycle-dependent expression of human and rodent PROTEIN0 and PROTEIN1 proteins was monitored using two approaches.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Analysis of various truncated PROTEIN0 molecules revealed that amino-terminal residues 48-163 are able to bind to PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Analysis of various truncated PROTEIN0 molecules revealed that amino-terminal residues 48-163 are able to bind to PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Analysis of various truncated PROTEIN0 molecules revealed that amino-terminal residues 48-163 are able to bind to PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Consistent with the observation that PROTEIN0 and PROTEIN1 bind to the same region of PROTEIN2 PROTEIN3 competed with the binding of PROTEIN4 to PROTEIN5 and vice versa.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using an in vitro assay system involving bacterially expressed proteins, we localized a region in PROTEIN0 required for molecular interaction with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using an in vitro assay system involving bacterially expressed proteins, we localized a region in PROTEIN0 required for molecular interaction with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using an in vitro assay system involving bacterially expressed proteins, we localized a region in PROTEIN0 required for molecular interaction with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 When transfected into L cells expressing PROTEIN0 the amino-terminal region of PROTEIN1 (from residue 1 to 226) formed complexes with PROTEIN2 supporting the in vitro binding experiment results.\n",
      "\n",
      "PROTEIN0 PROTEIN2 When transfected into L cells expressing PROTEIN0 the amino-terminal region of PROTEIN1 (from residue 1 to 226) formed complexes with PROTEIN2 supporting the in vitro binding experiment results.\n",
      "\n",
      "PROTEIN1 PROTEIN2 When transfected into L cells expressing PROTEIN0 the amino-terminal region of PROTEIN1 (from residue 1 to 226) formed complexes with PROTEIN2 supporting the in vitro binding experiment results.\n",
      "\n",
      "PROTEIN0 PROTEIN1 An assembly of the PROTEIN0 and PROTEIN1 subunits retains both enzymic activities, and the PROTEIN2 protein has been implicated in modulating these functions, facilitating efficient nuclear uptake of the complex and interacting with other viral DNA replication proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 An assembly of the PROTEIN0 and PROTEIN1 subunits retains both enzymic activities, and the PROTEIN2 protein has been implicated in modulating these functions, facilitating efficient nuclear uptake of the complex and interacting with other viral DNA replication proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 An assembly of the PROTEIN0 and PROTEIN1 subunits retains both enzymic activities, and the PROTEIN2 protein has been implicated in modulating these functions, facilitating efficient nuclear uptake of the complex and interacting with other viral DNA replication proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Deletion mutants of the herpes simplex virus type 1 PROTEIN0 protein: effect on DNA synthesis and ability to interact with and influence the intracellular localization of the PROTEIN1 and PROTEIN2 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Deletion mutants of the herpes simplex virus type 1 PROTEIN0 protein: effect on DNA synthesis and ability to interact with and influence the intracellular localization of the PROTEIN1 and PROTEIN2 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Deletion mutants of the herpes simplex virus type 1 PROTEIN0 protein: effect on DNA synthesis and ability to interact with and influence the intracellular localization of the PROTEIN1 and PROTEIN2 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The herpes simplex virus type 1 (HSV-1) PROTEIN0 an essential component of the viral DNA replication machinery, is a trimeric complex of the virus-coded PROTEIN1 PROTEIN2 and PROTEIN3 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The herpes simplex virus type 1 (HSV-1) PROTEIN0 an essential component of the viral DNA replication machinery, is a trimeric complex of the virus-coded PROTEIN1 PROTEIN2 and PROTEIN3 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The herpes simplex virus type 1 (HSV-1) PROTEIN0 an essential component of the viral DNA replication machinery, is a trimeric complex of the virus-coded PROTEIN1 PROTEIN2 and PROTEIN3 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The herpes simplex virus type 1 (HSV-1) PROTEIN0 an essential component of the viral DNA replication machinery, is a trimeric complex of the virus-coded PROTEIN1 PROTEIN2 and PROTEIN3 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The herpes simplex virus type 1 (HSV-1) PROTEIN0 an essential component of the viral DNA replication machinery, is a trimeric complex of the virus-coded PROTEIN1 PROTEIN2 and PROTEIN3 proteins.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The herpes simplex virus type 1 (HSV-1) PROTEIN0 an essential component of the viral DNA replication machinery, is a trimeric complex of the virus-coded PROTEIN1 PROTEIN2 and PROTEIN3 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 An association was seen between PROTEIN0 and PROTEIN1 expression, in both effusions and solid tumours, and for PROTEIN2 in solid tumours (range p<0. 001 to p=0.014).\n",
      "\n",
      "PROTEIN0 PROTEIN2 An association was seen between PROTEIN0 and PROTEIN1 expression, in both effusions and solid tumours, and for PROTEIN2 in solid tumours (range p<0. 001 to p=0.014).\n",
      "\n",
      "PROTEIN1 PROTEIN2 An association was seen between PROTEIN0 and PROTEIN1 expression, in both effusions and solid tumours, and for PROTEIN2 in solid tumours (range p<0. 001 to p=0.014).\n",
      "\n",
      "PROTEIN0 PROTEIN1 An oligomeric form of simian virus 40 PROTEIN0 is immunologically related to the cellular tumor antigen PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The cellular tumor antigen PROTEIN0 is bound to the simian virus 40 (SV40) PROTEIN1 in SV40-infected and -transformed cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Another PROTEIN0 PROTEIN1 and PROTEIN2 increased slightly, whereas PROTEIN3 and PROTEIN4 remained unchanged.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Another PROTEIN0 PROTEIN1 and PROTEIN2 increased slightly, whereas PROTEIN3 and PROTEIN4 remained unchanged.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Another PROTEIN0 PROTEIN1 and PROTEIN2 increased slightly, whereas PROTEIN3 and PROTEIN4 remained unchanged.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Another PROTEIN0 PROTEIN1 and PROTEIN2 increased slightly, whereas PROTEIN3 and PROTEIN4 remained unchanged.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Another PROTEIN0 PROTEIN1 and PROTEIN2 increased slightly, whereas PROTEIN3 and PROTEIN4 remained unchanged.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Another PROTEIN0 PROTEIN1 and PROTEIN2 increased slightly, whereas PROTEIN3 and PROTEIN4 remained unchanged.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Another PROTEIN0 PROTEIN1 and PROTEIN2 increased slightly, whereas PROTEIN3 and PROTEIN4 remained unchanged.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Another PROTEIN0 PROTEIN1 and PROTEIN2 increased slightly, whereas PROTEIN3 and PROTEIN4 remained unchanged.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Another PROTEIN0 PROTEIN1 and PROTEIN2 increased slightly, whereas PROTEIN3 and PROTEIN4 remained unchanged.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Another PROTEIN0 PROTEIN1 and PROTEIN2 increased slightly, whereas PROTEIN3 and PROTEIN4 remained unchanged.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN0 PROTEIN3 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN0 PROTEIN4 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN0 PROTEIN5 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN0 PROTEIN6 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN0 PROTEIN7 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN0 PROTEIN8 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN0 PROTEIN9 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN0 PROTEIN0 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN1 PROTEIN3 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN1 PROTEIN4 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN1 PROTEIN5 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN1 PROTEIN6 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN1 PROTEIN7 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN1 PROTEIN8 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN1 PROTEIN9 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN1 PROTEIN0 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN2 PROTEIN3 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN2 PROTEIN4 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN2 PROTEIN5 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN2 PROTEIN6 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN2 PROTEIN7 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN2 PROTEIN8 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN2 PROTEIN9 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN2 PROTEIN0 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN3 PROTEIN4 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN3 PROTEIN5 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN3 PROTEIN6 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN3 PROTEIN7 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN3 PROTEIN8 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN3 PROTEIN9 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN3 PROTEIN0 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN4 PROTEIN5 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN4 PROTEIN6 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN4 PROTEIN7 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN4 PROTEIN8 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN4 PROTEIN9 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN4 PROTEIN0 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN5 PROTEIN6 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN5 PROTEIN7 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN5 PROTEIN8 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN5 PROTEIN9 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN5 PROTEIN0 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN6 PROTEIN7 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN6 PROTEIN8 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN6 PROTEIN9 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN6 PROTEIN0 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN7 PROTEIN8 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN7 PROTEIN9 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN7 PROTEIN0 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN8 PROTEIN9 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN8 PROTEIN0 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN9 PROTEIN0 A null mutation of the PROTEIN0 gene PROTEIN1 is lethal, but null mutations in the PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 PROTEIN9 and PROTEIN0 genes have relatively mild or no effects.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 and PROTEIN2 were stable (T1/2 >2 h) and associated with a chromatin-containing fraction throughout the cell cycle.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 were stable (T1/2 >2 h) and associated with a chromatin-containing fraction throughout the cell cycle.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 were stable (T1/2 >2 h) and associated with a chromatin-containing fraction throughout the cell cycle.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 associated with chromatin throughout mitosis in living cells and co-localized with PROTEIN2 in metaphase spreads.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 associated with chromatin throughout mitosis in living cells and co-localized with PROTEIN2 in metaphase spreads.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 associated with chromatin throughout mitosis in living cells and co-localized with PROTEIN2 in metaphase spreads.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 and PROTEIN2 encode proteins that are critical to the repair of double-strand DNA breaks by homologous recombination.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 encode proteins that are critical to the repair of double-strand DNA breaks by homologous recombination.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 encode proteins that are critical to the repair of double-strand DNA breaks by homologous recombination.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were required for efficient gap repair using either chromosomal or plasmid donors.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were required for efficient gap repair using either chromosomal or plasmid donors.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were required for efficient gap repair using either chromosomal or plasmid donors.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were required for efficient gap repair using either chromosomal or plasmid donors.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were required for efficient gap repair using either chromosomal or plasmid donors.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were required for efficient gap repair using either chromosomal or plasmid donors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are all required to facilitate strand invasion into otherwise inaccessible donor sequences.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are all required to facilitate strand invasion into otherwise inaccessible donor sequences.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are all required to facilitate strand invasion into otherwise inaccessible donor sequences.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are all required to facilitate strand invasion into otherwise inaccessible donor sequences.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are all required to facilitate strand invasion into otherwise inaccessible donor sequences.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are all required to facilitate strand invasion into otherwise inaccessible donor sequences.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are still required when the same transcribed donor is on the chromosome.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are still required when the same transcribed donor is on the chromosome.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are still required when the same transcribed donor is on the chromosome.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are still required when the same transcribed donor is on the chromosome.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are still required when the same transcribed donor is on the chromosome.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are still required when the same transcribed donor is on the chromosome.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we use physical monitoring of DNA to show that MAT switching is completely blocked at an early step in recombination in strains deleted for the DNA repair genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we use physical monitoring of DNA to show that MAT switching is completely blocked at an early step in recombination in strains deleted for the DNA repair genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here we use physical monitoring of DNA to show that MAT switching is completely blocked at an early step in recombination in strains deleted for the DNA repair genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Here we use physical monitoring of DNA to show that MAT switching is completely blocked at an early step in recombination in strains deleted for the DNA repair genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we use physical monitoring of DNA to show that MAT switching is completely blocked at an early step in recombination in strains deleted for the DNA repair genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here we use physical monitoring of DNA to show that MAT switching is completely blocked at an early step in recombination in strains deleted for the DNA repair genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Here we use physical monitoring of DNA to show that MAT switching is completely blocked at an early step in recombination in strains deleted for the DNA repair genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here we use physical monitoring of DNA to show that MAT switching is completely blocked at an early step in recombination in strains deleted for the DNA repair genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Here we use physical monitoring of DNA to show that MAT switching is completely blocked at an early step in recombination in strains deleted for the DNA repair genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Here we use physical monitoring of DNA to show that MAT switching is completely blocked at an early step in recombination in strains deleted for the DNA repair genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 A reconstitution is described that requires not only the growth factor, its receptor with PROTEIN0 activity, and the soluble PROTEIN1 but also the small soluble PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 A reconstitution is described that requires not only the growth factor, its receptor with PROTEIN0 activity, and the soluble PROTEIN1 but also the small soluble PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 A reconstitution is described that requires not only the growth factor, its receptor with PROTEIN0 activity, and the soluble PROTEIN1 but also the small soluble PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 A reconstitution is described that requires not only the growth factor, its receptor with PROTEIN0 activity, and the soluble PROTEIN1 but also the small soluble PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 A reconstitution is described that requires not only the growth factor, its receptor with PROTEIN0 activity, and the soluble PROTEIN1 but also the small soluble PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 A reconstitution is described that requires not only the growth factor, its receptor with PROTEIN0 activity, and the soluble PROTEIN1 but also the small soluble PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN6 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN6 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN6 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN6 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN6 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN5 PROTEIN6 PROTEIN0 PROTEIN1 repeat 10 to the COOH terminus of PROTEIN2 is involved in binding to PROTEIN3 whereas PROTEIN4 interacts directly with the PROTEIN5 domain of PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we show that the PROTEIN0 subunit of the complex can be chemically cross-linked to the PROTEIN1 site of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we show that the PROTEIN0 subunit of the complex can be chemically cross-linked to the PROTEIN1 site of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we show that the PROTEIN0 subunit of the complex can be chemically cross-linked to the PROTEIN1 site of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN2 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN3 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN4 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN5 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN1 PROTEIN2 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN1 PROTEIN3 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN1 PROTEIN4 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN1 PROTEIN5 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN2 PROTEIN3 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN2 PROTEIN4 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN2 PROTEIN5 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN3 PROTEIN4 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN3 PROTEIN5 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN4 PROTEIN5 As a nuclear transport signal sequence exists in PROTEIN0 and PROTEIN1 but not in PROTEIN2 PROTEIN3 and/or PROTEIN4 may be responsible for the nuclear import of PROTEIN5 in myogenic cells under certain conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunofluorescence microscopy revealed that two PROTEIN0 of low molecular weight with different functional activity, PROTEIN1 and PROTEIN2 are transported into nuclei of cultured myogenic cells to form rod structures there together with PROTEIN3 when the cells were incubated in medium containing dimethylsulfoxide.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunofluorescence microscopy revealed that two PROTEIN0 of low molecular weight with different functional activity, PROTEIN1 and PROTEIN2 are transported into nuclei of cultured myogenic cells to form rod structures there together with PROTEIN3 when the cells were incubated in medium containing dimethylsulfoxide.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunofluorescence microscopy revealed that two PROTEIN0 of low molecular weight with different functional activity, PROTEIN1 and PROTEIN2 are transported into nuclei of cultured myogenic cells to form rod structures there together with PROTEIN3 when the cells were incubated in medium containing dimethylsulfoxide.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunofluorescence microscopy revealed that two PROTEIN0 of low molecular weight with different functional activity, PROTEIN1 and PROTEIN2 are transported into nuclei of cultured myogenic cells to form rod structures there together with PROTEIN3 when the cells were incubated in medium containing dimethylsulfoxide.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunofluorescence microscopy revealed that two PROTEIN0 of low molecular weight with different functional activity, PROTEIN1 and PROTEIN2 are transported into nuclei of cultured myogenic cells to form rod structures there together with PROTEIN3 when the cells were incubated in medium containing dimethylsulfoxide.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunofluorescence microscopy revealed that two PROTEIN0 of low molecular weight with different functional activity, PROTEIN1 and PROTEIN2 are transported into nuclei of cultured myogenic cells to form rod structures there together with PROTEIN3 when the cells were incubated in medium containing dimethylsulfoxide.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In most cases, PROTEIN0 and PROTEIN1 colocalized in the same nuclear PROTEIN2 rods, but PROTEIN3 appeared to predominate in mononucleated cells, while PROTEIN4 was present in multinucleated myotubes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In most cases, PROTEIN0 and PROTEIN1 colocalized in the same nuclear PROTEIN2 rods, but PROTEIN3 appeared to predominate in mononucleated cells, while PROTEIN4 was present in multinucleated myotubes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In most cases, PROTEIN0 and PROTEIN1 colocalized in the same nuclear PROTEIN2 rods, but PROTEIN3 appeared to predominate in mononucleated cells, while PROTEIN4 was present in multinucleated myotubes.\n",
      "\n",
      "PROTEIN0 PROTEIN4 In most cases, PROTEIN0 and PROTEIN1 colocalized in the same nuclear PROTEIN2 rods, but PROTEIN3 appeared to predominate in mononucleated cells, while PROTEIN4 was present in multinucleated myotubes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In most cases, PROTEIN0 and PROTEIN1 colocalized in the same nuclear PROTEIN2 rods, but PROTEIN3 appeared to predominate in mononucleated cells, while PROTEIN4 was present in multinucleated myotubes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In most cases, PROTEIN0 and PROTEIN1 colocalized in the same nuclear PROTEIN2 rods, but PROTEIN3 appeared to predominate in mononucleated cells, while PROTEIN4 was present in multinucleated myotubes.\n",
      "\n",
      "PROTEIN1 PROTEIN4 In most cases, PROTEIN0 and PROTEIN1 colocalized in the same nuclear PROTEIN2 rods, but PROTEIN3 appeared to predominate in mononucleated cells, while PROTEIN4 was present in multinucleated myotubes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In most cases, PROTEIN0 and PROTEIN1 colocalized in the same nuclear PROTEIN2 rods, but PROTEIN3 appeared to predominate in mononucleated cells, while PROTEIN4 was present in multinucleated myotubes.\n",
      "\n",
      "PROTEIN2 PROTEIN4 In most cases, PROTEIN0 and PROTEIN1 colocalized in the same nuclear PROTEIN2 rods, but PROTEIN3 appeared to predominate in mononucleated cells, while PROTEIN4 was present in multinucleated myotubes.\n",
      "\n",
      "PROTEIN3 PROTEIN4 In most cases, PROTEIN0 and PROTEIN1 colocalized in the same nuclear PROTEIN2 rods, but PROTEIN3 appeared to predominate in mononucleated cells, while PROTEIN4 was present in multinucleated myotubes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A series of truncations was carried out at the C terminus of PROTEIN0 a region that has been implicated in PROTEIN1 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Since our earlier studies of Acanthamoeba PROTEIN0 suggested the importance of PIP2 binding for suppression, we conclude that both activities are implicated and that an interplay between PIP2 binding and PROTEIN1 binding may be important for PROTEIN2 function.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Since our earlier studies of Acanthamoeba PROTEIN0 suggested the importance of PIP2 binding for suppression, we conclude that both activities are implicated and that an interplay between PIP2 binding and PROTEIN1 binding may be important for PROTEIN2 function.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Since our earlier studies of Acanthamoeba PROTEIN0 suggested the importance of PIP2 binding for suppression, we conclude that both activities are implicated and that an interplay between PIP2 binding and PROTEIN1 binding may be important for PROTEIN2 function.\n",
      "\n",
      "PROTEIN0 PROTEIN1 As expected from immunofluorescence experiments, PROTEIN0 PROTEIN1 and PROTEIN2 colocalized in immuno-EM to the dense patchlike structures but not to the cables.\n",
      "\n",
      "PROTEIN0 PROTEIN2 As expected from immunofluorescence experiments, PROTEIN0 PROTEIN1 and PROTEIN2 colocalized in immuno-EM to the dense patchlike structures but not to the cables.\n",
      "\n",
      "PROTEIN1 PROTEIN2 As expected from immunofluorescence experiments, PROTEIN0 PROTEIN1 and PROTEIN2 colocalized in immuno-EM to the dense patchlike structures but not to the cables.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immuno-EM double-labeling experiments were conducted with antibodies to PROTEIN0 together with antibodies to the PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immuno-EM double-labeling experiments were conducted with antibodies to PROTEIN0 together with antibodies to the PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immuno-EM double-labeling experiments were conducted with antibodies to PROTEIN0 together with antibodies to the PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immuno-EM double-labeling experiments were conducted with antibodies to PROTEIN0 together with antibodies to the PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immuno-EM double-labeling experiments were conducted with antibodies to PROTEIN0 together with antibodies to the PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immuno-EM double-labeling experiments were conducted with antibodies to PROTEIN0 together with antibodies to the PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 A similar phenotype was seen in testes treated with cytochalasin B and has been noted previously in mutants at the PROTEIN0 locus, a gene that encodes a Drosophila member of the PROTEIN1 family of PROTEIN2 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A similar phenotype was seen in testes treated with cytochalasin B and has been noted previously in mutants at the PROTEIN0 locus, a gene that encodes a Drosophila member of the PROTEIN1 family of PROTEIN2 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A similar phenotype was seen in testes treated with cytochalasin B and has been noted previously in mutants at the PROTEIN0 locus, a gene that encodes a Drosophila member of the PROTEIN1 family of PROTEIN2 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 As previous studies indicated that the PROTEIN0 and PROTEIN1 gene products of Saccharomyces cerevisiae seem to be involved in the same cellular function, we considered the possibility of a complex between the proteins encoded by these two genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 As shown by in situ hybridization with cloned probes and analysis of in vitro translation products, M. occulta embryos do not accumulate high levels of PROTEIN0 or PROTEIN1 mRNA.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, PROTEIN0 and PROTEIN1 mRNA accumulation was not enhanced in hybrid embryos.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Associations of PROTEIN0 with PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 proteins in a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Associations of PROTEIN0 with PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 proteins in a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Associations of PROTEIN0 with PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 proteins in a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Associations of PROTEIN0 with PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 proteins in a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Associations of PROTEIN0 with PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 proteins in a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Associations of PROTEIN0 with PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 proteins in a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Associations of PROTEIN0 with PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 proteins in a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Associations of PROTEIN0 with PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 proteins in a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Associations of PROTEIN0 with PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 proteins in a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Associations of PROTEIN0 with PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 proteins in a yeast two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A truncated PROTEIN0 disrupts the interaction between PROTEIN1 and PROTEIN2 a cause of loss of intercellular adhesiveness in human cancer cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A truncated PROTEIN0 disrupts the interaction between PROTEIN1 and PROTEIN2 a cause of loss of intercellular adhesiveness in human cancer cell lines.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A truncated PROTEIN0 disrupts the interaction between PROTEIN1 and PROTEIN2 a cause of loss of intercellular adhesiveness in human cancer cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, the recombinant fusion protein containing wild-type PROTEIN0 precipitated PROTEIN1 from these cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that the dysfunction of PROTEIN0 in these cell lines is due primarily to its failure to interact with PROTEIN1 and that this defect results from the mutation in PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results suggest that the dysfunction of PROTEIN0 in these cell lines is due primarily to its failure to interact with PROTEIN1 and that this defect results from the mutation in PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results suggest that the dysfunction of PROTEIN0 in these cell lines is due primarily to its failure to interact with PROTEIN1 and that this defect results from the mutation in PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, it is most likely that the association between PROTEIN0 and PROTEIN1 is mediated by PROTEIN2 and that this process is blocked by NH2-terminal deletion in PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, it is most likely that the association between PROTEIN0 and PROTEIN1 is mediated by PROTEIN2 and that this process is blocked by NH2-terminal deletion in PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Thus, it is most likely that the association between PROTEIN0 and PROTEIN1 is mediated by PROTEIN2 and that this process is blocked by NH2-terminal deletion in PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, it is most likely that the association between PROTEIN0 and PROTEIN1 is mediated by PROTEIN2 and that this process is blocked by NH2-terminal deletion in PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Thus, it is most likely that the association between PROTEIN0 and PROTEIN1 is mediated by PROTEIN2 and that this process is blocked by NH2-terminal deletion in PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Thus, it is most likely that the association between PROTEIN0 and PROTEIN1 is mediated by PROTEIN2 and that this process is blocked by NH2-terminal deletion in PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 While PROTEIN0 has been demonstrated to be crucial for PROTEIN1 function, the role of PROTEIN2 is not yet fully understood.\n",
      "\n",
      "PROTEIN0 PROTEIN2 While PROTEIN0 has been demonstrated to be crucial for PROTEIN1 function, the role of PROTEIN2 is not yet fully understood.\n",
      "\n",
      "PROTEIN1 PROTEIN2 While PROTEIN0 has been demonstrated to be crucial for PROTEIN1 function, the role of PROTEIN2 is not yet fully understood.\n",
      "\n",
      "PROTEIN0 PROTEIN1 At the light microscope level, brush cells can be identified by antibodies against the PROTEIN0 filament crosslinking proteins PROTEIN1 and PROTEIN2 that not only stain the apical tuft of microvilli and their rootlets, but also label projections emanating from the basolateral surface of these cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 At the light microscope level, brush cells can be identified by antibodies against the PROTEIN0 filament crosslinking proteins PROTEIN1 and PROTEIN2 that not only stain the apical tuft of microvilli and their rootlets, but also label projections emanating from the basolateral surface of these cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 At the light microscope level, brush cells can be identified by antibodies against the PROTEIN0 filament crosslinking proteins PROTEIN1 and PROTEIN2 that not only stain the apical tuft of microvilli and their rootlets, but also label projections emanating from the basolateral surface of these cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 BACKGROUND: Although PROTEIN0 is believed to be an essential regulator of the PROTEIN1 cytoskeleton in most cells, its precise role in mammalian cells remains unknown.\n",
      "\n",
      "PROTEIN0 PROTEIN1 BACKGROUND: PROTEIN0 PROTEIN1 a PROTEIN2 negatively regulates the G1 phase progression of the cell cycle by binding to the PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 BACKGROUND: PROTEIN0 PROTEIN1 a PROTEIN2 negatively regulates the G1 phase progression of the cell cycle by binding to the PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN3 BACKGROUND: PROTEIN0 PROTEIN1 a PROTEIN2 negatively regulates the G1 phase progression of the cell cycle by binding to the PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN4 BACKGROUND: PROTEIN0 PROTEIN1 a PROTEIN2 negatively regulates the G1 phase progression of the cell cycle by binding to the PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 BACKGROUND: PROTEIN0 PROTEIN1 a PROTEIN2 negatively regulates the G1 phase progression of the cell cycle by binding to the PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN3 BACKGROUND: PROTEIN0 PROTEIN1 a PROTEIN2 negatively regulates the G1 phase progression of the cell cycle by binding to the PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN4 BACKGROUND: PROTEIN0 PROTEIN1 a PROTEIN2 negatively regulates the G1 phase progression of the cell cycle by binding to the PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN3 BACKGROUND: PROTEIN0 PROTEIN1 a PROTEIN2 negatively regulates the G1 phase progression of the cell cycle by binding to the PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN4 BACKGROUND: PROTEIN0 PROTEIN1 a PROTEIN2 negatively regulates the G1 phase progression of the cell cycle by binding to the PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN3 PROTEIN4 BACKGROUND: PROTEIN0 PROTEIN1 a PROTEIN2 negatively regulates the G1 phase progression of the cell cycle by binding to the PROTEIN3 PROTEIN4 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 Because PROTEIN0 is known to interact with PROTEIN1 which binds to PROTEIN2 we generated a fusion molecule consisting of the ectodomain of PROTEIN3 and the PROTEIN4 binding site of PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Because both proteins exchange rapidly between PROTEIN0 molecules, low concentrations of PROTEIN1 can overcome the inhibitory effects of high concentrations of PROTEIN2 on the nucleotide exchange.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Because both proteins exchange rapidly between PROTEIN0 molecules, low concentrations of PROTEIN1 can overcome the inhibitory effects of high concentrations of PROTEIN2 on the nucleotide exchange.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Because both proteins exchange rapidly between PROTEIN0 molecules, low concentrations of PROTEIN1 can overcome the inhibitory effects of high concentrations of PROTEIN2 on the nucleotide exchange.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Because other researchers have shown that the PROTEIN0 and PROTEIN1 proteins interact, PROTEIN2 on a high copy number plasmid was tested and found to suppress the PROTEIN3 allele, but PROTEIN4 PROTEIN5 and PROTEIN6 did not suppress.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Between day 1 and day 3 in culture, the specific synthetic activities of total proteins and of electrophoretically purified PROTEIN0 and PROTEIN1 ([14C] phenylalanine incorporation into protein, in disintegrations per minute per microgram protein) decreased (-19%,-32%, and -73%, respectively).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Binding of PROTEIN0 to PROTEIN1 occurs at a maximal molar ratio of 1:3 as determined by fluorescencetitration under G-buffer conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binds to PROTEIN1 in vitro and hence is an PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 binds to PROTEIN1 in vitro and hence is an PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 binds to PROTEIN1 in vitro and hence is an PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Biochemical analyses revealed a strong induction of PROTEIN0 PROTEIN1 PROTEIN2 tyrosine-autophosphorylation by PROTEIN3 which was maximal after 5 minutes and was followed by receptor downregulation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Biochemical analyses revealed a strong induction of PROTEIN0 PROTEIN1 PROTEIN2 tyrosine-autophosphorylation by PROTEIN3 which was maximal after 5 minutes and was followed by receptor downregulation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Biochemical analyses revealed a strong induction of PROTEIN0 PROTEIN1 PROTEIN2 tyrosine-autophosphorylation by PROTEIN3 which was maximal after 5 minutes and was followed by receptor downregulation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Biochemical analyses revealed a strong induction of PROTEIN0 PROTEIN1 PROTEIN2 tyrosine-autophosphorylation by PROTEIN3 which was maximal after 5 minutes and was followed by receptor downregulation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Biochemical analyses revealed a strong induction of PROTEIN0 PROTEIN1 PROTEIN2 tyrosine-autophosphorylation by PROTEIN3 which was maximal after 5 minutes and was followed by receptor downregulation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Biochemical analyses revealed a strong induction of PROTEIN0 PROTEIN1 PROTEIN2 tyrosine-autophosphorylation by PROTEIN3 which was maximal after 5 minutes and was followed by receptor downregulation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Biochemical complementation experiments also indicate that the PROTEIN0 and PROTEIN1 proteins interact.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Four yeast spliceosomal proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 interact to promote PROTEIN4 binding to pre-mRNA.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Four yeast spliceosomal proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 interact to promote PROTEIN4 binding to pre-mRNA.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Four yeast spliceosomal proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 interact to promote PROTEIN4 binding to pre-mRNA.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Four yeast spliceosomal proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 interact to promote PROTEIN4 binding to pre-mRNA.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Four yeast spliceosomal proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 interact to promote PROTEIN4 binding to pre-mRNA.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Four yeast spliceosomal proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 interact to promote PROTEIN4 binding to pre-mRNA.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Four yeast spliceosomal proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 interact to promote PROTEIN4 binding to pre-mRNA.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Four yeast spliceosomal proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 interact to promote PROTEIN4 binding to pre-mRNA.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Four yeast spliceosomal proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 interact to promote PROTEIN4 binding to pre-mRNA.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Four yeast spliceosomal proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 interact to promote PROTEIN4 binding to pre-mRNA.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, we show that PROTEIN0 (a PROTEIN1 protein), PROTEIN2 and PROTEIN3 are each required for the PROTEIN4 to bind to the pre-spliceosome during spliceosome assembly in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here, we show that PROTEIN0 (a PROTEIN1 protein), PROTEIN2 and PROTEIN3 are each required for the PROTEIN4 to bind to the pre-spliceosome during spliceosome assembly in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here, we show that PROTEIN0 (a PROTEIN1 protein), PROTEIN2 and PROTEIN3 are each required for the PROTEIN4 to bind to the pre-spliceosome during spliceosome assembly in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Here, we show that PROTEIN0 (a PROTEIN1 protein), PROTEIN2 and PROTEIN3 are each required for the PROTEIN4 to bind to the pre-spliceosome during spliceosome assembly in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here, we show that PROTEIN0 (a PROTEIN1 protein), PROTEIN2 and PROTEIN3 are each required for the PROTEIN4 to bind to the pre-spliceosome during spliceosome assembly in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here, we show that PROTEIN0 (a PROTEIN1 protein), PROTEIN2 and PROTEIN3 are each required for the PROTEIN4 to bind to the pre-spliceosome during spliceosome assembly in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Here, we show that PROTEIN0 (a PROTEIN1 protein), PROTEIN2 and PROTEIN3 are each required for the PROTEIN4 to bind to the pre-spliceosome during spliceosome assembly in vitro.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here, we show that PROTEIN0 (a PROTEIN1 protein), PROTEIN2 and PROTEIN3 are each required for the PROTEIN4 to bind to the pre-spliceosome during spliceosome assembly in vitro.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Here, we show that PROTEIN0 (a PROTEIN1 protein), PROTEIN2 and PROTEIN3 are each required for the PROTEIN4 to bind to the pre-spliceosome during spliceosome assembly in vitro.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Here, we show that PROTEIN0 (a PROTEIN1 protein), PROTEIN2 and PROTEIN3 are each required for the PROTEIN4 to bind to the pre-spliceosome during spliceosome assembly in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Birch PROTEIN0 and PROTEIN1 can be copurified as a complex, and purified recombinant birch PROTEIN2 can be used as an affinity matrix to obtain birch PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Birch PROTEIN0 and PROTEIN1 can be copurified as a complex, and purified recombinant birch PROTEIN2 can be used as an affinity matrix to obtain birch PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Birch PROTEIN0 and PROTEIN1 can be copurified as a complex, and purified recombinant birch PROTEIN2 can be used as an affinity matrix to obtain birch PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Birch PROTEIN0 and PROTEIN1 can be copurified as a complex, and purified recombinant birch PROTEIN2 can be used as an affinity matrix to obtain birch PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Birch PROTEIN0 and PROTEIN1 can be copurified as a complex, and purified recombinant birch PROTEIN2 can be used as an affinity matrix to obtain birch PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Birch PROTEIN0 and PROTEIN1 can be copurified as a complex, and purified recombinant birch PROTEIN2 can be used as an affinity matrix to obtain birch PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this study, PROTEIN0 is identified as an PROTEIN1 in higher plants which is present in monocot and dicot angiosperms.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The binding of 125I-labeled recombinant birch pollen PROTEIN0 to plant and animal PROTEIN1 can be blocked by PROTEIN2 antibodies that react with different epitopes of birch PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The binding of 125I-labeled recombinant birch pollen PROTEIN0 to plant and animal PROTEIN1 can be blocked by PROTEIN2 antibodies that react with different epitopes of birch PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The binding of 125I-labeled recombinant birch pollen PROTEIN0 to plant and animal PROTEIN1 can be blocked by PROTEIN2 antibodies that react with different epitopes of birch PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The binding of 125I-labeled recombinant birch pollen PROTEIN0 to plant and animal PROTEIN1 can be blocked by PROTEIN2 antibodies that react with different epitopes of birch PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The binding of 125I-labeled recombinant birch pollen PROTEIN0 to plant and animal PROTEIN1 can be blocked by PROTEIN2 antibodies that react with different epitopes of birch PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The binding of 125I-labeled recombinant birch pollen PROTEIN0 to plant and animal PROTEIN1 can be blocked by PROTEIN2 antibodies that react with different epitopes of birch PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Bistratene A causes phosphorylation of PROTEIN0 and redistribution of PROTEIN1 microfilaments in fibroblasts: possible role for PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Bistratene A causes phosphorylation of PROTEIN0 and redistribution of PROTEIN1 microfilaments in fibroblasts: possible role for PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Bistratene A causes phosphorylation of PROTEIN0 and redistribution of PROTEIN1 microfilaments in fibroblasts: possible role for PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 appears to be more closely related to the mouse PROTEIN1 gene and other similar eukaryotic PROTEIN2 homologues than to the yeast gene PROTEIN3 and other PROTEIN4 homologues that are involved in replication repair.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 appears to be more closely related to the mouse PROTEIN1 gene and other similar eukaryotic PROTEIN2 homologues than to the yeast gene PROTEIN3 and other PROTEIN4 homologues that are involved in replication repair.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 appears to be more closely related to the mouse PROTEIN1 gene and other similar eukaryotic PROTEIN2 homologues than to the yeast gene PROTEIN3 and other PROTEIN4 homologues that are involved in replication repair.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 appears to be more closely related to the mouse PROTEIN1 gene and other similar eukaryotic PROTEIN2 homologues than to the yeast gene PROTEIN3 and other PROTEIN4 homologues that are involved in replication repair.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 appears to be more closely related to the mouse PROTEIN1 gene and other similar eukaryotic PROTEIN2 homologues than to the yeast gene PROTEIN3 and other PROTEIN4 homologues that are involved in replication repair.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 appears to be more closely related to the mouse PROTEIN1 gene and other similar eukaryotic PROTEIN2 homologues than to the yeast gene PROTEIN3 and other PROTEIN4 homologues that are involved in replication repair.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 appears to be more closely related to the mouse PROTEIN1 gene and other similar eukaryotic PROTEIN2 homologues than to the yeast gene PROTEIN3 and other PROTEIN4 homologues that are involved in replication repair.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 appears to be more closely related to the mouse PROTEIN1 gene and other similar eukaryotic PROTEIN2 homologues than to the yeast gene PROTEIN3 and other PROTEIN4 homologues that are involved in replication repair.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 appears to be more closely related to the mouse PROTEIN1 gene and other similar eukaryotic PROTEIN2 homologues than to the yeast gene PROTEIN3 and other PROTEIN4 homologues that are involved in replication repair.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 appears to be more closely related to the mouse PROTEIN1 gene and other similar eukaryotic PROTEIN2 homologues than to the yeast gene PROTEIN3 and other PROTEIN4 homologues that are involved in replication repair.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is also expressed in endothelial cells and shown to bind PROTEIN1 PROTEIN2 a potent regulator for vasculogenesis and angiogenesis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is also expressed in endothelial cells and shown to bind PROTEIN1 PROTEIN2 a potent regulator for vasculogenesis and angiogenesis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is also expressed in endothelial cells and shown to bind PROTEIN1 PROTEIN2 a potent regulator for vasculogenesis and angiogenesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is expressed by endothelial and tumor cells as an isoform-specific receptor for PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 mediates PROTEIN1 PROTEIN2 inhibition of endothelial cell motility: functional competition of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 mediates PROTEIN1 PROTEIN2 inhibition of endothelial cell motility: functional competition of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 mediates PROTEIN1 PROTEIN2 inhibition of endothelial cell motility: functional competition of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 mediates PROTEIN1 PROTEIN2 inhibition of endothelial cell motility: functional competition of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 mediates PROTEIN1 PROTEIN2 inhibition of endothelial cell motility: functional competition of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 mediates PROTEIN1 PROTEIN2 inhibition of endothelial cell motility: functional competition of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 mediates PROTEIN1 PROTEIN2 inhibition of endothelial cell motility: functional competition of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 mediates PROTEIN1 PROTEIN2 inhibition of endothelial cell motility: functional competition of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 mediates PROTEIN1 PROTEIN2 inhibition of endothelial cell motility: functional competition of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 mediates PROTEIN1 PROTEIN2 inhibition of endothelial cell motility: functional competition of PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 Both pathway cells and superior tarsal smooth muscle cells expressed PROTEIN0 and PROTEIN1 throughout development.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Both types of beads induced the recruitment of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 by lateral mobilization of preexisting cell membrane complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Both types of beads induced the recruitment of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 by lateral mobilization of preexisting cell membrane complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Both types of beads induced the recruitment of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 by lateral mobilization of preexisting cell membrane complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Both types of beads induced the recruitment of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 by lateral mobilization of preexisting cell membrane complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Both types of beads induced the recruitment of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 by lateral mobilization of preexisting cell membrane complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Both types of beads induced the recruitment of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 by lateral mobilization of preexisting cell membrane complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Both tyrosine phosphorylation and association of PROTEIN0 with PROTEIN1 were inhibited by tyrphostin, a specific inhibitor of the PROTEIN2 PROTEIN3 whereas dissociation of PROTEIN4 from PROTEIN5 was not.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The results suggest that tyrosine phosphorylation of PROTEIN0 is achieved by PROTEIN1 upon tryptic digestion of cells and concurrent with but independent of dissociation of PROTEIN2 from PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The results suggest that tyrosine phosphorylation of PROTEIN0 is achieved by PROTEIN1 upon tryptic digestion of cells and concurrent with but independent of dissociation of PROTEIN2 from PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The results suggest that tyrosine phosphorylation of PROTEIN0 is achieved by PROTEIN1 upon tryptic digestion of cells and concurrent with but independent of dissociation of PROTEIN2 from PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The results suggest that tyrosine phosphorylation of PROTEIN0 is achieved by PROTEIN1 upon tryptic digestion of cells and concurrent with but independent of dissociation of PROTEIN2 from PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The results suggest that tyrosine phosphorylation of PROTEIN0 is achieved by PROTEIN1 upon tryptic digestion of cells and concurrent with but independent of dissociation of PROTEIN2 from PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The results suggest that tyrosine phosphorylation of PROTEIN0 is achieved by PROTEIN1 upon tryptic digestion of cells and concurrent with but independent of dissociation of PROTEIN2 from PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 Tyrosine phosphorylation of PROTEIN0 was concomitantly induced with association of PROTEIN1 with PROTEIN2 PROTEIN3 when quiescent cells at confluence were dissociated into single cells by tryptic digestion, being accompanied by dissociation of PROTEIN4 from PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 or PROTEIN1 recombination-deficient yeast strains stabilize otherwise unstable YACs containing ribosomal DNA or the human color vision locus (Kohno et al., 1994).\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 a new component of the yeast PROTEIN1 PROTEIN2 complex that interacts with a conserved region of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 a new component of the yeast PROTEIN1 PROTEIN2 complex that interacts with a conserved region of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 a new component of the yeast PROTEIN1 PROTEIN2 complex that interacts with a conserved region of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 a new component of the yeast PROTEIN1 PROTEIN2 complex that interacts with a conserved region of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 a new component of the yeast PROTEIN1 PROTEIN2 complex that interacts with a conserved region of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 a new component of the yeast PROTEIN1 PROTEIN2 complex that interacts with a conserved region of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 interacts with PROTEIN1 in vitro and copurifies with the PROTEIN2 PROTEIN3 complex from yeast cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 interacts with PROTEIN1 in vitro and copurifies with the PROTEIN2 PROTEIN3 complex from yeast cells.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 interacts with PROTEIN1 in vitro and copurifies with the PROTEIN2 PROTEIN3 complex from yeast cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 interacts with PROTEIN1 in vitro and copurifies with the PROTEIN2 PROTEIN3 complex from yeast cells.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 interacts with PROTEIN1 in vitro and copurifies with the PROTEIN2 PROTEIN3 complex from yeast cells.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 interacts with PROTEIN1 in vitro and copurifies with the PROTEIN2 PROTEIN3 complex from yeast cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 Bud-site selection directs both cell polarity and the cytoskeletal orientation during budding and is determined by at least five genes: PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 By 1 h the cells were well spread with straight PROTEIN0 bundles many of which ended at more central sites terminating on patches containing PROTEIN1 and PROTEIN2 thus the cells assemble typical stress fibers but do not appear to polarize.\n",
      "\n",
      "PROTEIN0 PROTEIN2 By 1 h the cells were well spread with straight PROTEIN0 bundles many of which ended at more central sites terminating on patches containing PROTEIN1 and PROTEIN2 thus the cells assemble typical stress fibers but do not appear to polarize.\n",
      "\n",
      "PROTEIN1 PROTEIN2 By 1 h the cells were well spread with straight PROTEIN0 bundles many of which ended at more central sites terminating on patches containing PROTEIN1 and PROTEIN2 thus the cells assemble typical stress fibers but do not appear to polarize.\n",
      "\n",
      "PROTEIN0 PROTEIN1 By indirect immunofluorescence, both R. rickettsii and Listeria monocytogenes PROTEIN0 tails were shown to contain the cytoskeletal proteins PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 By indirect immunofluorescence, both R. rickettsii and Listeria monocytogenes PROTEIN0 tails were shown to contain the cytoskeletal proteins PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 By indirect immunofluorescence, both R. rickettsii and Listeria monocytogenes PROTEIN0 tails were shown to contain the cytoskeletal proteins PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 By indirect immunofluorescence, both R. rickettsii and Listeria monocytogenes PROTEIN0 tails were shown to contain the cytoskeletal proteins PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 By indirect immunofluorescence, both R. rickettsii and Listeria monocytogenes PROTEIN0 tails were shown to contain the cytoskeletal proteins PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 By indirect immunofluorescence, both R. rickettsii and Listeria monocytogenes PROTEIN0 tails were shown to contain the cytoskeletal proteins PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 By indirect immunofluorescence, both R. rickettsii and Listeria monocytogenes PROTEIN0 tails were shown to contain the cytoskeletal proteins PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 By indirect immunofluorescence, both R. rickettsii and Listeria monocytogenes PROTEIN0 tails were shown to contain the cytoskeletal proteins PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 By indirect immunofluorescence, both R. rickettsii and Listeria monocytogenes PROTEIN0 tails were shown to contain the cytoskeletal proteins PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 By indirect immunofluorescence, both R. rickettsii and Listeria monocytogenes PROTEIN0 tails were shown to contain the cytoskeletal proteins PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 By site-directed mutagenesis of PROTEIN0 from Dictyostelium discoideum the point mutations K114E and W3N were generated by PCR thus changing PROTEIN1 and poly-(L)-proline-binding activity respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 exhibited a profound decrease in its ability to interact with PROTEIN1 whereas binding to poly-(L)-proline was essentially unchanged.\n",
      "\n",
      "PROTEIN0 PROTEIN1 By two-hybrid screening we have identified two partners that directly associate with PROTEIN0 PROTEIN1 a previously characterized PROTEIN2 subunit, and a novel protein of 33 kDa PROTEIN3 which is part of the PROTEIN4 complex and has an RNA binding domain.\n",
      "\n",
      "PROTEIN0 PROTEIN2 By two-hybrid screening we have identified two partners that directly associate with PROTEIN0 PROTEIN1 a previously characterized PROTEIN2 subunit, and a novel protein of 33 kDa PROTEIN3 which is part of the PROTEIN4 complex and has an RNA binding domain.\n",
      "\n",
      "PROTEIN0 PROTEIN3 By two-hybrid screening we have identified two partners that directly associate with PROTEIN0 PROTEIN1 a previously characterized PROTEIN2 subunit, and a novel protein of 33 kDa PROTEIN3 which is part of the PROTEIN4 complex and has an RNA binding domain.\n",
      "\n",
      "PROTEIN0 PROTEIN4 By two-hybrid screening we have identified two partners that directly associate with PROTEIN0 PROTEIN1 a previously characterized PROTEIN2 subunit, and a novel protein of 33 kDa PROTEIN3 which is part of the PROTEIN4 complex and has an RNA binding domain.\n",
      "\n",
      "PROTEIN1 PROTEIN2 By two-hybrid screening we have identified two partners that directly associate with PROTEIN0 PROTEIN1 a previously characterized PROTEIN2 subunit, and a novel protein of 33 kDa PROTEIN3 which is part of the PROTEIN4 complex and has an RNA binding domain.\n",
      "\n",
      "PROTEIN1 PROTEIN3 By two-hybrid screening we have identified two partners that directly associate with PROTEIN0 PROTEIN1 a previously characterized PROTEIN2 subunit, and a novel protein of 33 kDa PROTEIN3 which is part of the PROTEIN4 complex and has an RNA binding domain.\n",
      "\n",
      "PROTEIN1 PROTEIN4 By two-hybrid screening we have identified two partners that directly associate with PROTEIN0 PROTEIN1 a previously characterized PROTEIN2 subunit, and a novel protein of 33 kDa PROTEIN3 which is part of the PROTEIN4 complex and has an RNA binding domain.\n",
      "\n",
      "PROTEIN2 PROTEIN3 By two-hybrid screening we have identified two partners that directly associate with PROTEIN0 PROTEIN1 a previously characterized PROTEIN2 subunit, and a novel protein of 33 kDa PROTEIN3 which is part of the PROTEIN4 complex and has an RNA binding domain.\n",
      "\n",
      "PROTEIN2 PROTEIN4 By two-hybrid screening we have identified two partners that directly associate with PROTEIN0 PROTEIN1 a previously characterized PROTEIN2 subunit, and a novel protein of 33 kDa PROTEIN3 which is part of the PROTEIN4 complex and has an RNA binding domain.\n",
      "\n",
      "PROTEIN3 PROTEIN4 By two-hybrid screening we have identified two partners that directly associate with PROTEIN0 PROTEIN1 a previously characterized PROTEIN2 subunit, and a novel protein of 33 kDa PROTEIN3 which is part of the PROTEIN4 complex and has an RNA binding domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our results provide support for both physical and functional interactions between three subunits, PROTEIN0 PROTEIN1 and PROTEIN2 in the PROTEIN3 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Our results provide support for both physical and functional interactions between three subunits, PROTEIN0 PROTEIN1 and PROTEIN2 in the PROTEIN3 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Our results provide support for both physical and functional interactions between three subunits, PROTEIN0 PROTEIN1 and PROTEIN2 in the PROTEIN3 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Our results provide support for both physical and functional interactions between three subunits, PROTEIN0 PROTEIN1 and PROTEIN2 in the PROTEIN3 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Our results provide support for both physical and functional interactions between three subunits, PROTEIN0 PROTEIN1 and PROTEIN2 in the PROTEIN3 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Our results provide support for both physical and functional interactions between three subunits, PROTEIN0 PROTEIN1 and PROTEIN2 in the PROTEIN3 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 are essential for embryonic morphogenesis since they mediate calcium-dependent cell-cell adhesion and can modulate PROTEIN1 signaling.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 links PROTEIN1 to the PROTEIN2 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 links PROTEIN1 to the PROTEIN2 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 links PROTEIN1 to the PROTEIN2 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Calorimetric data show ternary complex formation between PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Calorimetric data show ternary complex formation between PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Calorimetric data show ternary complex formation between PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 activity in vivo was determined by observing proteolysis of PROTEIN1 and PROTEIN2 two known substrates of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 activity in vivo was determined by observing proteolysis of PROTEIN1 and PROTEIN2 two known substrates of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 activity in vivo was determined by observing proteolysis of PROTEIN1 and PROTEIN2 two known substrates of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 activity in vivo was determined by observing proteolysis of PROTEIN1 and PROTEIN2 two known substrates of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 activity in vivo was determined by observing proteolysis of PROTEIN1 and PROTEIN2 two known substrates of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 activity in vivo was determined by observing proteolysis of PROTEIN1 and PROTEIN2 two known substrates of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 a T cell activation Ag, also known as PROTEIN1 is directly associated with PROTEIN2 PROTEIN3 on the surface of T cells and T cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 a T cell activation Ag, also known as PROTEIN1 is directly associated with PROTEIN2 PROTEIN3 on the surface of T cells and T cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 a T cell activation Ag, also known as PROTEIN1 is directly associated with PROTEIN2 PROTEIN3 on the surface of T cells and T cell lines.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 a T cell activation Ag, also known as PROTEIN1 is directly associated with PROTEIN2 PROTEIN3 on the surface of T cells and T cell lines.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 a T cell activation Ag, also known as PROTEIN1 is directly associated with PROTEIN2 PROTEIN3 on the surface of T cells and T cell lines.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 a T cell activation Ag, also known as PROTEIN1 is directly associated with PROTEIN2 PROTEIN3 on the surface of T cells and T cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a T cell activation antigen known to bind PROTEIN1 and have PROTEIN2 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is a T cell activation antigen known to bind PROTEIN1 and have PROTEIN2 activity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is a T cell activation antigen known to bind PROTEIN1 and have PROTEIN2 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a T cell activation antigen that contains PROTEIN1 activity and is known to bind PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is a T cell activation antigen that contains PROTEIN1 activity and is known to bind PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is a T cell activation antigen that contains PROTEIN1 activity and is known to bind PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Changes in PROTEIN0 binding to PROTEIN1 or PROTEIN2 protein, and the ensuing effects on cell morphology and adhesion, are independent of PROTEIN3 binding to PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Changes in PROTEIN0 binding to PROTEIN1 or PROTEIN2 protein, and the ensuing effects on cell morphology and adhesion, are independent of PROTEIN3 binding to PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Changes in PROTEIN0 binding to PROTEIN1 or PROTEIN2 protein, and the ensuing effects on cell morphology and adhesion, are independent of PROTEIN3 binding to PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Changes in PROTEIN0 binding to PROTEIN1 or PROTEIN2 protein, and the ensuing effects on cell morphology and adhesion, are independent of PROTEIN3 binding to PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Changes in PROTEIN0 binding to PROTEIN1 or PROTEIN2 protein, and the ensuing effects on cell morphology and adhesion, are independent of PROTEIN3 binding to PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Changes in PROTEIN0 binding to PROTEIN1 or PROTEIN2 protein, and the ensuing effects on cell morphology and adhesion, are independent of PROTEIN3 binding to PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Changes in PROTEIN0 binding to PROTEIN1 or PROTEIN2 protein, and the ensuing effects on cell morphology and adhesion, are independent of PROTEIN3 binding to PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Changes in PROTEIN0 binding to PROTEIN1 or PROTEIN2 protein, and the ensuing effects on cell morphology and adhesion, are independent of PROTEIN3 binding to PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Changes in PROTEIN0 binding to PROTEIN1 or PROTEIN2 protein, and the ensuing effects on cell morphology and adhesion, are independent of PROTEIN3 binding to PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Changes in PROTEIN0 binding to PROTEIN1 or PROTEIN2 protein, and the ensuing effects on cell morphology and adhesion, are independent of PROTEIN3 binding to PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 Characterization of PROTEIN0 and poly-L-proline binding sites of Acanthamoeba PROTEIN1 with monoclonal antibodies and by mutagenesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 For example, PROTEIN0 with a deletion of the eight C-terminal residues has many of the physical properties of a molten globule, yet remarkably still binds to PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The results provide two independent lines of evidence about the binding sites for PROTEIN0 and poly-L-proline on the PROTEIN1 molecule.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Characterization of a mutant PROTEIN0 with reduced PROTEIN1 capacity: effects in vitro and in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Characterization of PROTEIN0 truncates that inhibit PROTEIN1 depolymerization by severing activity of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Characterization of PROTEIN0 truncates that inhibit PROTEIN1 depolymerization by severing activity of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Characterization of PROTEIN0 truncates that inhibit PROTEIN1 depolymerization by severing activity of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Characterization of PROTEIN0 truncates that inhibit PROTEIN1 depolymerization by severing activity of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Characterization of PROTEIN0 truncates that inhibit PROTEIN1 depolymerization by severing activity of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Characterization of PROTEIN0 truncates that inhibit PROTEIN1 depolymerization by severing activity of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Characterization of the action of porcine PROTEIN0 on PROTEIN1 polymerization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover, addition of PROTEIN0 to steady-state PROTEIN1 filaments causes slow depolymerization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Characterization of the interactions of PROTEIN0 with PROTEIN1 and PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Characterization of the interactions of PROTEIN0 with PROTEIN1 and PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Characterization of the interactions of PROTEIN0 with PROTEIN1 and PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Characterization of the interactions of PROTEIN0 with PROTEIN1 and PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Characterization of the interactions of PROTEIN0 with PROTEIN1 and PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Characterization of the interactions of PROTEIN0 with PROTEIN1 and PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this study we have used the yeast two-hybrid system to map the domains on PROTEIN0 that allow it to associate with PROTEIN1 PROTEIN2 and with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this study we have used the yeast two-hybrid system to map the domains on PROTEIN0 that allow it to associate with PROTEIN1 PROTEIN2 and with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 In this study we have used the yeast two-hybrid system to map the domains on PROTEIN0 that allow it to associate with PROTEIN1 PROTEIN2 and with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this study we have used the yeast two-hybrid system to map the domains on PROTEIN0 that allow it to associate with PROTEIN1 PROTEIN2 and with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 In this study we have used the yeast two-hybrid system to map the domains on PROTEIN0 that allow it to associate with PROTEIN1 PROTEIN2 and with PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 In this study we have used the yeast two-hybrid system to map the domains on PROTEIN0 that allow it to associate with PROTEIN1 PROTEIN2 and with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN6 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN7 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN8 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN6 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN7 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN8 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN6 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN7 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN8 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN3 PROTEIN6 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN3 PROTEIN7 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN3 PROTEIN8 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN4 PROTEIN6 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN4 PROTEIN7 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN4 PROTEIN8 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN5 PROTEIN6 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN5 PROTEIN7 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN5 PROTEIN8 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN6 PROTEIN7 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN6 PROTEIN8 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN7 PROTEIN8 PROTEIN0 or PROTEIN1 is associated directly with the PROTEIN2 PROTEIN3 binds to PROTEIN4 PROTEIN5 and serves to link the PROTEIN6 PROTEIN7 complex to the PROTEIN8 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Chemical modification of the cysteine residue 374 of PROTEIN0 both with N-ethylmaleimide and with the fluorescent probe N-(1-pyrenyl)iodoacetamide, is shown to counteract the inhibiting effect of PROTEIN1 on PROTEIN2 polymerization.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Chemical modification of the cysteine residue 374 of PROTEIN0 both with N-ethylmaleimide and with the fluorescent probe N-(1-pyrenyl)iodoacetamide, is shown to counteract the inhibiting effect of PROTEIN1 on PROTEIN2 polymerization.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Chemical modification of the cysteine residue 374 of PROTEIN0 both with N-ethylmaleimide and with the fluorescent probe N-(1-pyrenyl)iodoacetamide, is shown to counteract the inhibiting effect of PROTEIN1 on PROTEIN2 polymerization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN2 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN3 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN4 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN5 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN6 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN7 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN2 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN3 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN4 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN5 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN6 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN7 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN3 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN4 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN5 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN6 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN7 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN4 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN5 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN6 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN7 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN4 PROTEIN5 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN4 PROTEIN6 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN4 PROTEIN7 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN5 PROTEIN6 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN5 PROTEIN7 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN6 PROTEIN7 Chick embryo fibroblasts contain about 75-100 microM unpolymerized PROTEIN0 and at least four proteins which can bind PROTEIN1 monomers, PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 had higher affinity for poly-l-proline and sequestered more monomeric PROTEIN1 than did PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 had higher affinity for poly-l-proline and sequestered more monomeric PROTEIN1 than did PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 had higher affinity for poly-l-proline and sequestered more monomeric PROTEIN1 than did PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 activity of both maize PROTEIN1 classes was found to be dependent on the concentration of free calcium.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We propose a model in which PROTEIN0 alters cellular concentrations of PROTEIN1 polymers in response to fluctuations in cytosolic calcium concentration.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Coexpression of the PROTEIN0 and PROTEIN1 genes in Escherichia coli was required for in vitro activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Expression of the PROTEIN0 and PROTEIN1 genes occurred between 1 and 5 h after vaccinia virus infection and was not prevented by an inhibitor of DNA replication, consistent with a role for PROTEIN2 in specifically regulating intermediate transcription in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Expression of the PROTEIN0 and PROTEIN1 genes occurred between 1 and 5 h after vaccinia virus infection and was not prevented by an inhibitor of DNA replication, consistent with a role for PROTEIN2 in specifically regulating intermediate transcription in vivo.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Expression of the PROTEIN0 and PROTEIN1 genes occurred between 1 and 5 h after vaccinia virus infection and was not prevented by an inhibitor of DNA replication, consistent with a role for PROTEIN2 in specifically regulating intermediate transcription in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We found that the 34- and 45-kDa polypeptides encoded by vaccinia virus ORFs PROTEIN0 and PROTEIN1 respectively, were necessary to reconstitute transcription of a template with an intermediate stage promoter.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 was identified by peptide sequencing, and PROTEIN1 recruitment and Listeria tail length were found to be pH-dependent, in agreement with its recently reported role in enhancing PROTEIN2 filament turnover.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 was identified by peptide sequencing, and PROTEIN1 recruitment and Listeria tail length were found to be pH-dependent, in agreement with its recently reported role in enhancing PROTEIN2 filament turnover.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 was identified by peptide sequencing, and PROTEIN1 recruitment and Listeria tail length were found to be pH-dependent, in agreement with its recently reported role in enhancing PROTEIN2 filament turnover.\n",
      "\n",
      "PROTEIN0 PROTEIN1 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN2 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN3 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN4 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN5 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN6 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN7 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN8 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN9 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN0 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN2 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN3 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN4 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN5 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN6 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN7 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN8 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN9 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN1 PROTEIN0 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN3 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN4 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN5 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN6 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN7 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN8 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN9 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN2 PROTEIN0 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN4 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN5 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN6 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN7 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN8 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN9 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN3 PROTEIN0 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN5 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN6 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN7 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN8 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN9 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN4 PROTEIN0 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN6 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN7 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN8 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN9 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN5 PROTEIN0 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN7 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN8 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN9 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN6 PROTEIN0 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN8 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN9 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN7 PROTEIN0 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN9 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN8 PROTEIN0 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN9 PROTEIN0 To detect the expression of apoptosis-related molecules, PROTEIN0 protection assay was used with specific antisense RNA probes for PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0\n",
      "\n",
      "PROTEIN0 PROTEIN1 CONCLUSIONS: The expression of PROTEIN0 PROTEIN1 and PROTEIN2 is related to histological type and differentiation in NSCLC, although PROTEIN3 have no independent prognostic value.\n",
      "\n",
      "PROTEIN0 PROTEIN2 CONCLUSIONS: The expression of PROTEIN0 PROTEIN1 and PROTEIN2 is related to histological type and differentiation in NSCLC, although PROTEIN3 have no independent prognostic value.\n",
      "\n",
      "PROTEIN0 PROTEIN3 CONCLUSIONS: The expression of PROTEIN0 PROTEIN1 and PROTEIN2 is related to histological type and differentiation in NSCLC, although PROTEIN3 have no independent prognostic value.\n",
      "\n",
      "PROTEIN1 PROTEIN2 CONCLUSIONS: The expression of PROTEIN0 PROTEIN1 and PROTEIN2 is related to histological type and differentiation in NSCLC, although PROTEIN3 have no independent prognostic value.\n",
      "\n",
      "PROTEIN1 PROTEIN3 CONCLUSIONS: The expression of PROTEIN0 PROTEIN1 and PROTEIN2 is related to histological type and differentiation in NSCLC, although PROTEIN3 have no independent prognostic value.\n",
      "\n",
      "PROTEIN2 PROTEIN3 CONCLUSIONS: The expression of PROTEIN0 PROTEIN1 and PROTEIN2 is related to histological type and differentiation in NSCLC, although PROTEIN3 have no independent prognostic value.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Reduced expression of PROTEIN0 PROTEIN1 and PROTEIN2 is associated with high cell proliferative activity and poor differentiation in non-small cell lung cancer.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Reduced expression of PROTEIN0 PROTEIN1 and PROTEIN2 is associated with high cell proliferative activity and poor differentiation in non-small cell lung cancer.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Reduced expression of PROTEIN0 PROTEIN1 and PROTEIN2 is associated with high cell proliferative activity and poor differentiation in non-small cell lung cancer.\n",
      "\n",
      "PROTEIN0 PROTEIN1 CONCLUSIONS: These results suggest the cooperative modulation of the PROTEIN0 cytoskeleton by PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 CONCLUSIONS: These results suggest the cooperative modulation of the PROTEIN0 cytoskeleton by PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 CONCLUSIONS: These results suggest the cooperative modulation of the PROTEIN0 cytoskeleton by PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunofluorescence staining of a wild-type strain using PROTEIN0 antibodies revealed that PROTEIN1 was distributed in cortical PROTEIN2 patches where PROTEIN3 was also co-localized.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunofluorescence staining of a wild-type strain using PROTEIN0 antibodies revealed that PROTEIN1 was distributed in cortical PROTEIN2 patches where PROTEIN3 was also co-localized.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunofluorescence staining of a wild-type strain using PROTEIN0 antibodies revealed that PROTEIN1 was distributed in cortical PROTEIN2 patches where PROTEIN3 was also co-localized.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunofluorescence staining of a wild-type strain using PROTEIN0 antibodies revealed that PROTEIN1 was distributed in cortical PROTEIN2 patches where PROTEIN3 was also co-localized.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunofluorescence staining of a wild-type strain using PROTEIN0 antibodies revealed that PROTEIN1 was distributed in cortical PROTEIN2 patches where PROTEIN3 was also co-localized.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunofluorescence staining of a wild-type strain using PROTEIN0 antibodies revealed that PROTEIN1 was distributed in cortical PROTEIN2 patches where PROTEIN3 was also co-localized.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Confluent calf pulmonary artery endothelial monolayers exposed to 95% oxygen for 1, 2, or 3 days exhibit a time-dependent increase in adherence to substratum, which closely parallels changes in PROTEIN0 cytoarchitecture and the distribution of focal contact proteins PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Confluent calf pulmonary artery endothelial monolayers exposed to 95% oxygen for 1, 2, or 3 days exhibit a time-dependent increase in adherence to substratum, which closely parallels changes in PROTEIN0 cytoarchitecture and the distribution of focal contact proteins PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Confluent calf pulmonary artery endothelial monolayers exposed to 95% oxygen for 1, 2, or 3 days exhibit a time-dependent increase in adherence to substratum, which closely parallels changes in PROTEIN0 cytoarchitecture and the distribution of focal contact proteins PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Conversely, inhibition of PROTEIN0 activity by expressing a dominant negative construct, PROTEIN1 or expression of the constitutively active S3A PROTEIN2 mutant induces loss of PROTEIN3 filaments at the phagocytic cup and also inhibits phagocytosis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Conversely, inhibition of PROTEIN0 activity by expressing a dominant negative construct, PROTEIN1 or expression of the constitutively active S3A PROTEIN2 mutant induces loss of PROTEIN3 filaments at the phagocytic cup and also inhibits phagocytosis.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Conversely, inhibition of PROTEIN0 activity by expressing a dominant negative construct, PROTEIN1 or expression of the constitutively active S3A PROTEIN2 mutant induces loss of PROTEIN3 filaments at the phagocytic cup and also inhibits phagocytosis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Conversely, inhibition of PROTEIN0 activity by expressing a dominant negative construct, PROTEIN1 or expression of the constitutively active S3A PROTEIN2 mutant induces loss of PROTEIN3 filaments at the phagocytic cup and also inhibits phagocytosis.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Conversely, inhibition of PROTEIN0 activity by expressing a dominant negative construct, PROTEIN1 or expression of the constitutively active S3A PROTEIN2 mutant induces loss of PROTEIN3 filaments at the phagocytic cup and also inhibits phagocytosis.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Conversely, inhibition of PROTEIN0 activity by expressing a dominant negative construct, PROTEIN1 or expression of the constitutively active S3A PROTEIN2 mutant induces loss of PROTEIN3 filaments at the phagocytic cup and also inhibits phagocytosis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Co-precipitation experiments using whole cell lysates indicate that the mutant form of PROTEIN0 binds PROTEIN1 and PROTEIN2 and can form a structural complex with PROTEIN3 via these interactions.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Co-precipitation experiments using whole cell lysates indicate that the mutant form of PROTEIN0 binds PROTEIN1 and PROTEIN2 and can form a structural complex with PROTEIN3 via these interactions.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Co-precipitation experiments using whole cell lysates indicate that the mutant form of PROTEIN0 binds PROTEIN1 and PROTEIN2 and can form a structural complex with PROTEIN3 via these interactions.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Co-precipitation experiments using whole cell lysates indicate that the mutant form of PROTEIN0 binds PROTEIN1 and PROTEIN2 and can form a structural complex with PROTEIN3 via these interactions.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Co-precipitation experiments using whole cell lysates indicate that the mutant form of PROTEIN0 binds PROTEIN1 and PROTEIN2 and can form a structural complex with PROTEIN3 via these interactions.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Co-precipitation experiments using whole cell lysates indicate that the mutant form of PROTEIN0 binds PROTEIN1 and PROTEIN2 and can form a structural complex with PROTEIN3 via these interactions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Coprecipitations revealed that transfected PROTEIN0 molecules are complexed with PROTEIN1 and PROTEIN2 at plasma membranes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Coprecipitations revealed that transfected PROTEIN0 molecules are complexed with PROTEIN1 and PROTEIN2 at plasma membranes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Coprecipitations revealed that transfected PROTEIN0 molecules are complexed with PROTEIN1 and PROTEIN2 at plasma membranes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Cotransfections with different combinations of these genes demonstrated that a subset of four of them, coding for the HSV PROTEIN0 complex PROTEIN1 PROTEIN2 PROTEIN3 and the PROTEIN4 PROTEIN5 was already sufficient to mediate the helper effect.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Crystallization and preliminary crystallographic analysis of the N-terminal PROTEIN0 binding domain of human PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cytoplasmic staining included stress fibers that colocalized with PROTEIN0 probably as a consequence of the PROTEIN1 component of the fusion protein.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Data from affinity chromatography, analytical ultracentrifugation, covalent cross-linking, and fluorescence anisotropy show that PROTEIN0 PROTEIN1 and PROTEIN2 form a ternary complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Data from affinity chromatography, analytical ultracentrifugation, covalent cross-linking, and fluorescence anisotropy show that PROTEIN0 PROTEIN1 and PROTEIN2 form a ternary complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Data from affinity chromatography, analytical ultracentrifugation, covalent cross-linking, and fluorescence anisotropy show that PROTEIN0 PROTEIN1 and PROTEIN2 form a ternary complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 Experiments using a peptide that corresponds to the N-terminus of PROTEIN0 (residues 6-22) confirm the presence of an extensive binding surface between PROTEIN1 and PROTEIN2 and explain why PROTEIN3 and PROTEIN4 can bind simultaneously to PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Data is presented to suggest that the G1 PROTEIN0 and the PROTEIN1 PROTEIN2 may be involved in subversion of the G1/S traverse by signaling pathways activated by PROTEIN3 function.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Data is presented to suggest that the G1 PROTEIN0 and the PROTEIN1 PROTEIN2 may be involved in subversion of the G1/S traverse by signaling pathways activated by PROTEIN3 function.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Data is presented to suggest that the G1 PROTEIN0 and the PROTEIN1 PROTEIN2 may be involved in subversion of the G1/S traverse by signaling pathways activated by PROTEIN3 function.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Data is presented to suggest that the G1 PROTEIN0 and the PROTEIN1 PROTEIN2 may be involved in subversion of the G1/S traverse by signaling pathways activated by PROTEIN3 function.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Data is presented to suggest that the G1 PROTEIN0 and the PROTEIN1 PROTEIN2 may be involved in subversion of the G1/S traverse by signaling pathways activated by PROTEIN3 function.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Data is presented to suggest that the G1 PROTEIN0 and the PROTEIN1 PROTEIN2 may be involved in subversion of the G1/S traverse by signaling pathways activated by PROTEIN3 function.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Data presented here suggest that two of the repressed genes encode the proteins PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Selective repression of PROTEIN0 and PROTEIN1 expression during the programmed death of insect skeletal muscle.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The reduction in PROTEIN0 and PROTEIN1 synthesis presumably plays a role in the rapid dissolution of the muscles.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 belong to the PROTEIN1 family and are characterised by an intracellular death domain that serves to recruit adapter proteins such as PROTEIN2 and PROTEIN3 and PROTEIN4 such as PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Deletion and overexpression studies demonstrate that PROTEIN0 but not PROTEIN1 PROTEIN2 or PROTEIN3 is required for this rDNA position effect.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Deletion and overexpression studies demonstrate that PROTEIN0 but not PROTEIN1 PROTEIN2 or PROTEIN3 is required for this rDNA position effect.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Deletion and overexpression studies demonstrate that PROTEIN0 but not PROTEIN1 PROTEIN2 or PROTEIN3 is required for this rDNA position effect.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Deletion and overexpression studies demonstrate that PROTEIN0 but not PROTEIN1 PROTEIN2 or PROTEIN3 is required for this rDNA position effect.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Deletion and overexpression studies demonstrate that PROTEIN0 but not PROTEIN1 PROTEIN2 or PROTEIN3 is required for this rDNA position effect.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Deletion and overexpression studies demonstrate that PROTEIN0 but not PROTEIN1 PROTEIN2 or PROTEIN3 is required for this rDNA position effect.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Deletion of both PROTEIN0 and PROTEIN1 produces a phenotype similar to that produced by deletion of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Deletion of both PROTEIN0 and PROTEIN1 produces a phenotype similar to that produced by deletion of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Deletion of both PROTEIN0 and PROTEIN1 produces a phenotype similar to that produced by deletion of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 mutations completely abolished the ability to generate type II survivors, while PROTEIN1 mutations decreased the growth viability of type II survivors but did not completely eliminate their appearance.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Deletion of PROTEIN0 enhanced PROTEIN1 and PROTEIN2 silencing, but deletion of PROTEIN3 or PROTEIN4 did not affect silencing, indicating that the mechanism of silencing differs from that at telomeres and silent mating loci.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Deletion of PROTEIN0 enhanced PROTEIN1 and PROTEIN2 silencing, but deletion of PROTEIN3 or PROTEIN4 did not affect silencing, indicating that the mechanism of silencing differs from that at telomeres and silent mating loci.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Deletion of PROTEIN0 enhanced PROTEIN1 and PROTEIN2 silencing, but deletion of PROTEIN3 or PROTEIN4 did not affect silencing, indicating that the mechanism of silencing differs from that at telomeres and silent mating loci.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Deletion of PROTEIN0 enhanced PROTEIN1 and PROTEIN2 silencing, but deletion of PROTEIN3 or PROTEIN4 did not affect silencing, indicating that the mechanism of silencing differs from that at telomeres and silent mating loci.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Deletion of PROTEIN0 enhanced PROTEIN1 and PROTEIN2 silencing, but deletion of PROTEIN3 or PROTEIN4 did not affect silencing, indicating that the mechanism of silencing differs from that at telomeres and silent mating loci.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Deletion of PROTEIN0 enhanced PROTEIN1 and PROTEIN2 silencing, but deletion of PROTEIN3 or PROTEIN4 did not affect silencing, indicating that the mechanism of silencing differs from that at telomeres and silent mating loci.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Deletion of PROTEIN0 enhanced PROTEIN1 and PROTEIN2 silencing, but deletion of PROTEIN3 or PROTEIN4 did not affect silencing, indicating that the mechanism of silencing differs from that at telomeres and silent mating loci.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Deletion of PROTEIN0 enhanced PROTEIN1 and PROTEIN2 silencing, but deletion of PROTEIN3 or PROTEIN4 did not affect silencing, indicating that the mechanism of silencing differs from that at telomeres and silent mating loci.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Deletion of PROTEIN0 enhanced PROTEIN1 and PROTEIN2 silencing, but deletion of PROTEIN3 or PROTEIN4 did not affect silencing, indicating that the mechanism of silencing differs from that at telomeres and silent mating loci.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Deletion of PROTEIN0 enhanced PROTEIN1 and PROTEIN2 silencing, but deletion of PROTEIN3 or PROTEIN4 did not affect silencing, indicating that the mechanism of silencing differs from that at telomeres and silent mating loci.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Demembranated stereociliary cores consisted primarily of protein bands corresponding to PROTEIN0 and PROTEIN1 and several proteins ranging from 43 to 63 kDa.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Depending on the nature of the divalent cation, recombinant plant (birch) PROTEIN0 exhibited two different modes of interaction with PROTEIN1 like mammalian PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Depending on the nature of the divalent cation, recombinant plant (birch) PROTEIN0 exhibited two different modes of interaction with PROTEIN1 like mammalian PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Depending on the nature of the divalent cation, recombinant plant (birch) PROTEIN0 exhibited two different modes of interaction with PROTEIN1 like mammalian PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Recombinant plant (birch) PROTEIN0 was analyzed for its ability to promote PROTEIN1 polymerization from the PROTEIN2 PROTEIN3 and PROTEIN4 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Recombinant plant (birch) PROTEIN0 was analyzed for its ability to promote PROTEIN1 polymerization from the PROTEIN2 PROTEIN3 and PROTEIN4 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Recombinant plant (birch) PROTEIN0 was analyzed for its ability to promote PROTEIN1 polymerization from the PROTEIN2 PROTEIN3 and PROTEIN4 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Recombinant plant (birch) PROTEIN0 was analyzed for its ability to promote PROTEIN1 polymerization from the PROTEIN2 PROTEIN3 and PROTEIN4 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Recombinant plant (birch) PROTEIN0 was analyzed for its ability to promote PROTEIN1 polymerization from the PROTEIN2 PROTEIN3 and PROTEIN4 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Recombinant plant (birch) PROTEIN0 was analyzed for its ability to promote PROTEIN1 polymerization from the PROTEIN2 PROTEIN3 and PROTEIN4 complex.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Recombinant plant (birch) PROTEIN0 was analyzed for its ability to promote PROTEIN1 polymerization from the PROTEIN2 PROTEIN3 and PROTEIN4 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Recombinant plant (birch) PROTEIN0 was analyzed for its ability to promote PROTEIN1 polymerization from the PROTEIN2 PROTEIN3 and PROTEIN4 complex.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Recombinant plant (birch) PROTEIN0 was analyzed for its ability to promote PROTEIN1 polymerization from the PROTEIN2 PROTEIN3 and PROTEIN4 complex.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Recombinant plant (birch) PROTEIN0 was analyzed for its ability to promote PROTEIN1 polymerization from the PROTEIN2 PROTEIN3 and PROTEIN4 complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These data indicated a negative co-operativity between the PROTEIN0 and PROTEIN1 binding sites on PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 These data indicated a negative co-operativity between the PROTEIN0 and PROTEIN1 binding sites on PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 These data indicated a negative co-operativity between the PROTEIN0 and PROTEIN1 binding sites on PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Detection of a sequence involved in PROTEIN0 and phosphoinositide-binding in the N-terminal side of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The truncated PROTEIN0 molecules produced in E. coli were purified and examined for their PROTEIN1 and PIP2-binding ability.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Detection of loss of heterozygosity at PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and PROTEIN4 loci in breast cancer: pathological correlations.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Detection of loss of heterozygosity at PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and PROTEIN4 loci in breast cancer: pathological correlations.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Detection of loss of heterozygosity at PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and PROTEIN4 loci in breast cancer: pathological correlations.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Detection of loss of heterozygosity at PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and PROTEIN4 loci in breast cancer: pathological correlations.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Detection of loss of heterozygosity at PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and PROTEIN4 loci in breast cancer: pathological correlations.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Detection of loss of heterozygosity at PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and PROTEIN4 loci in breast cancer: pathological correlations.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Detection of loss of heterozygosity at PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and PROTEIN4 loci in breast cancer: pathological correlations.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Detection of loss of heterozygosity at PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and PROTEIN4 loci in breast cancer: pathological correlations.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Detection of loss of heterozygosity at PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and PROTEIN4 loci in breast cancer: pathological correlations.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Detection of loss of heterozygosity at PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and PROTEIN4 loci in breast cancer: pathological correlations.\n",
      "\n",
      "PROTEIN0 PROTEIN1 LOH was found in the PROTEIN0 region in 32% of tumours, in the PROTEIN1 region in 16%, in PROTEIN2 in 20% and in the PROTEIN3 and PROTEIN4 regions in 49% and 44% respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN2 LOH was found in the PROTEIN0 region in 32% of tumours, in the PROTEIN1 region in 16%, in PROTEIN2 in 20% and in the PROTEIN3 and PROTEIN4 regions in 49% and 44% respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN3 LOH was found in the PROTEIN0 region in 32% of tumours, in the PROTEIN1 region in 16%, in PROTEIN2 in 20% and in the PROTEIN3 and PROTEIN4 regions in 49% and 44% respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN4 LOH was found in the PROTEIN0 region in 32% of tumours, in the PROTEIN1 region in 16%, in PROTEIN2 in 20% and in the PROTEIN3 and PROTEIN4 regions in 49% and 44% respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN2 LOH was found in the PROTEIN0 region in 32% of tumours, in the PROTEIN1 region in 16%, in PROTEIN2 in 20% and in the PROTEIN3 and PROTEIN4 regions in 49% and 44% respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN3 LOH was found in the PROTEIN0 region in 32% of tumours, in the PROTEIN1 region in 16%, in PROTEIN2 in 20% and in the PROTEIN3 and PROTEIN4 regions in 49% and 44% respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN4 LOH was found in the PROTEIN0 region in 32% of tumours, in the PROTEIN1 region in 16%, in PROTEIN2 in 20% and in the PROTEIN3 and PROTEIN4 regions in 49% and 44% respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN3 LOH was found in the PROTEIN0 region in 32% of tumours, in the PROTEIN1 region in 16%, in PROTEIN2 in 20% and in the PROTEIN3 and PROTEIN4 regions in 49% and 44% respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN4 LOH was found in the PROTEIN0 region in 32% of tumours, in the PROTEIN1 region in 16%, in PROTEIN2 in 20% and in the PROTEIN3 and PROTEIN4 regions in 49% and 44% respectively.\n",
      "\n",
      "PROTEIN3 PROTEIN4 LOH was found in the PROTEIN0 region in 32% of tumours, in the PROTEIN1 region in 16%, in PROTEIN2 in 20% and in the PROTEIN3 and PROTEIN4 regions in 49% and 44% respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We investigate allelic losses in microsatellites of the PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 regions, and their correlations with nine pathologic parameters in 127 breast carcinomas.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We investigate allelic losses in microsatellites of the PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 regions, and their correlations with nine pathologic parameters in 127 breast carcinomas.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We investigate allelic losses in microsatellites of the PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 regions, and their correlations with nine pathologic parameters in 127 breast carcinomas.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We investigate allelic losses in microsatellites of the PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 regions, and their correlations with nine pathologic parameters in 127 breast carcinomas.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We investigate allelic losses in microsatellites of the PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 regions, and their correlations with nine pathologic parameters in 127 breast carcinomas.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We investigate allelic losses in microsatellites of the PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 regions, and their correlations with nine pathologic parameters in 127 breast carcinomas.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We investigate allelic losses in microsatellites of the PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 regions, and their correlations with nine pathologic parameters in 127 breast carcinomas.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We investigate allelic losses in microsatellites of the PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 regions, and their correlations with nine pathologic parameters in 127 breast carcinomas.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We investigate allelic losses in microsatellites of the PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 regions, and their correlations with nine pathologic parameters in 127 breast carcinomas.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We investigate allelic losses in microsatellites of the PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 regions, and their correlations with nine pathologic parameters in 127 breast carcinomas.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Developmental changes in PROTEIN0 and PROTEIN1 isoform expression in smooth muscle.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Digestion of isolated myofibrils with PROTEIN0 resulted in the degradation of PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Digestion of isolated myofibrils with PROTEIN0 resulted in the degradation of PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Digestion of isolated myofibrils with PROTEIN0 resulted in the degradation of PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Direct binding of the PROTEIN0 domain in PROTEIN1 to PROTEIN2 is essential for cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Direct binding of the PROTEIN0 domain in PROTEIN1 to PROTEIN2 is essential for cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Direct binding of the PROTEIN0 domain in PROTEIN1 to PROTEIN2 is essential for cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Recently, two mammalian proteins previously shown to regulate the PROTEIN0 cytoskeleton, PROTEIN1 PROTEIN2 and its homolog expressed in neurons PROTEIN3 were found to possess short peptide motifs homologous to one part of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Recently, two mammalian proteins previously shown to regulate the PROTEIN0 cytoskeleton, PROTEIN1 PROTEIN2 and its homolog expressed in neurons PROTEIN3 were found to possess short peptide motifs homologous to one part of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Recently, two mammalian proteins previously shown to regulate the PROTEIN0 cytoskeleton, PROTEIN1 PROTEIN2 and its homolog expressed in neurons PROTEIN3 were found to possess short peptide motifs homologous to one part of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Recently, two mammalian proteins previously shown to regulate the PROTEIN0 cytoskeleton, PROTEIN1 PROTEIN2 and its homolog expressed in neurons PROTEIN3 were found to possess short peptide motifs homologous to one part of PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Recently, two mammalian proteins previously shown to regulate the PROTEIN0 cytoskeleton, PROTEIN1 PROTEIN2 and its homolog expressed in neurons PROTEIN3 were found to possess short peptide motifs homologous to one part of PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Recently, two mammalian proteins previously shown to regulate the PROTEIN0 cytoskeleton, PROTEIN1 PROTEIN2 and its homolog expressed in neurons PROTEIN3 were found to possess short peptide motifs homologous to one part of PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Recently, two mammalian proteins previously shown to regulate the PROTEIN0 cytoskeleton, PROTEIN1 PROTEIN2 and its homolog expressed in neurons PROTEIN3 were found to possess short peptide motifs homologous to one part of PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Recently, two mammalian proteins previously shown to regulate the PROTEIN0 cytoskeleton, PROTEIN1 PROTEIN2 and its homolog expressed in neurons PROTEIN3 were found to possess short peptide motifs homologous to one part of PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Recently, two mammalian proteins previously shown to regulate the PROTEIN0 cytoskeleton, PROTEIN1 PROTEIN2 and its homolog expressed in neurons PROTEIN3 were found to possess short peptide motifs homologous to one part of PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Recently, two mammalian proteins previously shown to regulate the PROTEIN0 cytoskeleton, PROTEIN1 PROTEIN2 and its homolog expressed in neurons PROTEIN3 were found to possess short peptide motifs homologous to one part of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 Discrete segments (70-150 amino acids) of PROTEIN0 and PROTEIN1 were found to be responsible for their binding to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Discrete segments (70-150 amino acids) of PROTEIN0 and PROTEIN1 were found to be responsible for their binding to PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Discrete segments (70-150 amino acids) of PROTEIN0 and PROTEIN1 were found to be responsible for their binding to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins of Saccharomyces cerevisiae were found previously to copurify with PROTEIN4 PROTEIN5 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Disruption of the PROTEIN0 gene, which is homologous to PROTEIN1 PROTEIN2 and which is also required for correct PROTEIN3 localization, also blocks endocytosis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Disruption of the PROTEIN0 gene, which is homologous to PROTEIN1 PROTEIN2 and which is also required for correct PROTEIN3 localization, also blocks endocytosis.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Disruption of the PROTEIN0 gene, which is homologous to PROTEIN1 PROTEIN2 and which is also required for correct PROTEIN3 localization, also blocks endocytosis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Disruption of the PROTEIN0 gene, which is homologous to PROTEIN1 PROTEIN2 and which is also required for correct PROTEIN3 localization, also blocks endocytosis.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PROTEIN1 PROTEIN3 Disruption of the PROTEIN0 gene, which is homologous to PROTEIN1 PROTEIN2 and which is also required for correct PROTEIN3 localization, also blocks endocytosis.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Disruption of the PROTEIN0 gene, which is homologous to PROTEIN1 PROTEIN2 and which is also required for correct PROTEIN3 localization, also blocks endocytosis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 DNA sequence analysis of the PROTEIN0 gene reveals that it encodes PROTEIN1 an PROTEIN2 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN2 DNA sequence analysis of the PROTEIN0 gene reveals that it encodes PROTEIN1 an PROTEIN2 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN2 DNA sequence analysis of the PROTEIN0 gene reveals that it encodes PROTEIN1 an PROTEIN2 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We attribute these effects to potential sequestration of PROTEIN0 monomers by PROTEIN1 when present in excess.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Double mutants containing N-terminal domain deletions of both PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 were inviable.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Double mutants containing N-terminal domain deletions of both PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 were inviable.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Double mutants containing N-terminal domain deletions of both PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 were inviable.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Double mutants containing N-terminal domain deletions of both PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 were inviable.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Double mutants containing N-terminal domain deletions of both PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 were inviable.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Double mutants containing N-terminal domain deletions of both PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 were inviable.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 signal transduction is mediated by a complex of intracellular signaling molecules including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 signal transduction is mediated by a complex of intracellular signaling molecules including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 signal transduction is mediated by a complex of intracellular signaling molecules including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 signal transduction is mediated by a complex of intracellular signaling molecules including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 signal transduction is mediated by a complex of intracellular signaling molecules including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 signal transduction is mediated by a complex of intracellular signaling molecules including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 signal transduction is mediated by a complex of intracellular signaling molecules including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 signal transduction is mediated by a complex of intracellular signaling molecules including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 signal transduction is mediated by a complex of intracellular signaling molecules including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 signal transduction is mediated by a complex of intracellular signaling molecules including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 During apoptotic process induced by the agents, expression of PROTEIN0 PROTEIN1 and PROTEIN2 increased, whereas expression of PROTEIN3 decreased.\n",
      "\n",
      "PROTEIN0 PROTEIN2 During apoptotic process induced by the agents, expression of PROTEIN0 PROTEIN1 and PROTEIN2 increased, whereas expression of PROTEIN3 decreased.\n",
      "\n",
      "PROTEIN0 PROTEIN3 During apoptotic process induced by the agents, expression of PROTEIN0 PROTEIN1 and PROTEIN2 increased, whereas expression of PROTEIN3 decreased.\n",
      "\n",
      "PROTEIN1 PROTEIN2 During apoptotic process induced by the agents, expression of PROTEIN0 PROTEIN1 and PROTEIN2 increased, whereas expression of PROTEIN3 decreased.\n",
      "\n",
      "PROTEIN1 PROTEIN3 During apoptotic process induced by the agents, expression of PROTEIN0 PROTEIN1 and PROTEIN2 increased, whereas expression of PROTEIN3 decreased.\n",
      "\n",
      "PROTEIN2 PROTEIN3 During apoptotic process induced by the agents, expression of PROTEIN0 PROTEIN1 and PROTEIN2 increased, whereas expression of PROTEIN3 decreased.\n",
      "\n",
      "PROTEIN0 PROTEIN1 During incubation of whole myofibrils at 37 degrees C, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 suffered degradation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 During incubation of whole myofibrils at 37 degrees C, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 suffered degradation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 During incubation of whole myofibrils at 37 degrees C, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 suffered degradation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 During incubation of whole myofibrils at 37 degrees C, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 suffered degradation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 During incubation of whole myofibrils at 37 degrees C, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 suffered degradation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 During incubation of whole myofibrils at 37 degrees C, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 suffered degradation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Dynamic PROTEIN0 structures stabilized by PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The distribution of PROTEIN0 filaments is altered by PROTEIN1 overexpression.\n",
      "\n",
      "PROTEIN0 PROTEIN1 While parallel PROTEIN0 bundles crossing the cells are virtually absent in cells overexpressing PROTEIN1 the submembranous PROTEIN2 network of these cells is denser than in control cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 While parallel PROTEIN0 bundles crossing the cells are virtually absent in cells overexpressing PROTEIN1 the submembranous PROTEIN2 network of these cells is denser than in control cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 While parallel PROTEIN0 bundles crossing the cells are virtually absent in cells overexpressing PROTEIN1 the submembranous PROTEIN2 network of these cells is denser than in control cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Effects of PROTEIN0 PROTEIN1 association on some properties of both PROTEIN2 and PROTEIN3 modification of the inhibitory activity of PROTEIN4 on PROTEIN5 polymerization and inhibition of the self-association of PROTEIN6 and its interactions with liposomes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Effects of PROTEIN0 on PROTEIN1 in the presence and absence of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Effects of PROTEIN0 on PROTEIN1 in the presence and absence of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Effects of PROTEIN0 on PROTEIN1 in the presence and absence of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Effects of PROTEIN0 on PROTEIN1 in the presence and absence of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Effects of PROTEIN0 on PROTEIN1 in the presence and absence of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Effects of PROTEIN0 on PROTEIN1 in the presence and absence of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Effects of mutations of PROTEIN0 PROTEIN1 PROTEIN2 and related genes on illegitimate recombination in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Effects of mutations of PROTEIN0 PROTEIN1 PROTEIN2 and related genes on illegitimate recombination in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Effects of mutations of PROTEIN0 PROTEIN1 PROTEIN2 and related genes on illegitimate recombination in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Effects of single amino acid substitutions in the PROTEIN0 site on the biological activity of bovine PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, these mutants displayed a lower affinity for PROTEIN0 than wild-type PROTEIN1 suggesting that additional putative bonds created this way cannot increase PROTEIN2 affinity for PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, these mutants displayed a lower affinity for PROTEIN0 than wild-type PROTEIN1 suggesting that additional putative bonds created this way cannot increase PROTEIN2 affinity for PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 However, these mutants displayed a lower affinity for PROTEIN0 than wild-type PROTEIN1 suggesting that additional putative bonds created this way cannot increase PROTEIN2 affinity for PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, these mutants displayed a lower affinity for PROTEIN0 than wild-type PROTEIN1 suggesting that additional putative bonds created this way cannot increase PROTEIN2 affinity for PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 However, these mutants displayed a lower affinity for PROTEIN0 than wild-type PROTEIN1 suggesting that additional putative bonds created this way cannot increase PROTEIN2 affinity for PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 However, these mutants displayed a lower affinity for PROTEIN0 than wild-type PROTEIN1 suggesting that additional putative bonds created this way cannot increase PROTEIN2 affinity for PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results emphasize the highly conserved biological function of PROTEIN0 with low sequence homology, and correlate specifically their PROTEIN1 capacity with cell motility and proliferation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Elevation of cAMP induced morphologic alterations indicative of changes in the adhesive status of the macrophage, including cell rounding and disassembly of structures that represent points of contact with substrate containing PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 encodes a proline-rich protein PROTEIN1 or PROTEIN2 required for a polarised cortical PROTEIN3 cytoskeleton and endocytosis [2,3].\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 encodes a proline-rich protein PROTEIN1 or PROTEIN2 required for a polarised cortical PROTEIN3 cytoskeleton and endocytosis [2,3].\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 encodes a proline-rich protein PROTEIN1 or PROTEIN2 required for a polarised cortical PROTEIN3 cytoskeleton and endocytosis [2,3].\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 encodes a proline-rich protein PROTEIN1 or PROTEIN2 required for a polarised cortical PROTEIN3 cytoskeleton and endocytosis [2,3].\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 encodes a proline-rich protein PROTEIN1 or PROTEIN2 required for a polarised cortical PROTEIN3 cytoskeleton and endocytosis [2,3].\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 encodes a proline-rich protein PROTEIN1 or PROTEIN2 required for a polarised cortical PROTEIN3 cytoskeleton and endocytosis [2,3].\n",
      "\n",
      "PROTEIN0 PROTEIN1 Equine satellite cell clone SE-11 and ovine satellite cell clone I(1)were evaluated for expression of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 over a 240 h period in culture.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Equine satellite cell clone SE-11 and ovine satellite cell clone I(1)were evaluated for expression of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 over a 240 h period in culture.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Equine satellite cell clone SE-11 and ovine satellite cell clone I(1)were evaluated for expression of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 over a 240 h period in culture.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Equine satellite cell clone SE-11 and ovine satellite cell clone I(1)were evaluated for expression of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 over a 240 h period in culture.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Equine satellite cell clone SE-11 and ovine satellite cell clone I(1)were evaluated for expression of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 over a 240 h period in culture.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Equine satellite cell clone SE-11 and ovine satellite cell clone I(1)were evaluated for expression of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 over a 240 h period in culture.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Essential domains of the PROTEIN0 splicing factor are implicated in the binding to PROTEIN1 and PROTEIN2 proteins and are conserved through evolution.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Essential domains of the PROTEIN0 splicing factor are implicated in the binding to PROTEIN1 and PROTEIN2 proteins and are conserved through evolution.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Essential domains of the PROTEIN0 splicing factor are implicated in the binding to PROTEIN1 and PROTEIN2 proteins and are conserved through evolution.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Evaluation of the binding of Acanthamoeba PROTEIN0 to pyrene-labeled PROTEIN1 by fluorescence enhancement.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Evidence for a conformational change in PROTEIN0 induced by PROTEIN1 PROTEIN2 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Evidence for a conformational change in PROTEIN0 induced by PROTEIN1 PROTEIN2 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Evidence for a conformational change in PROTEIN0 induced by PROTEIN1 PROTEIN2 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Exclusion of PROTEIN0 and PROTEIN1 gene involvement in hypertrophic cardiomyopathy of several French families.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Exposure of the cells to tumor promoter did, however, result in a loss of PROTEIN0 and PROTEIN1 cell surface elaborations that resemble focal contact precursor structures.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Expression of PROTEIN0 and PROTEIN1 genes in skeletal, cardiac and uterine muscles of young and old rats.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Expression of PROTEIN0 was generally weak and did not correlate with the expression of either PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Expression of PROTEIN0 was generally weak and did not correlate with the expression of either PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Expression of PROTEIN0 was generally weak and did not correlate with the expression of either PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Expression of PROTEIN0 or PROTEIN1 was sufficient to activate PROTEIN2 PROTEIN3 by promoting the formation of high-molecular-weight complexes lacking the PROTEIN4 PROTEIN5 as has been described, following estrogen treatment.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Expression of PROTEIN0 PROTEIN1 and PROTEIN2 in normal ovarian surface epithelium and epithelial ovarian cancers.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Expression of PROTEIN0 PROTEIN1 and PROTEIN2 in normal ovarian surface epithelium and epithelial ovarian cancers.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Expression of PROTEIN0 PROTEIN1 and PROTEIN2 in normal ovarian surface epithelium and epithelial ovarian cancers.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Features studied were PROTEIN0 microtubules, PROTEIN1 and PROTEIN2 using immunofluorescence, sodium dodecyl sulfate-polyacrylamide gel electrophoresis, and immunoblotting; permeability of the cell sheet; wound healing; and phagocytic capacity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Features studied were PROTEIN0 microtubules, PROTEIN1 and PROTEIN2 using immunofluorescence, sodium dodecyl sulfate-polyacrylamide gel electrophoresis, and immunoblotting; permeability of the cell sheet; wound healing; and phagocytic capacity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Features studied were PROTEIN0 microtubules, PROTEIN1 and PROTEIN2 using immunofluorescence, sodium dodecyl sulfate-polyacrylamide gel electrophoresis, and immunoblotting; permeability of the cell sheet; wound healing; and phagocytic capacity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Finally, both receptors can interact with PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Finally, both receptors can interact with PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Finally, both receptors can interact with PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Thus, both PROTEIN0 and PROTEIN1 use PROTEIN2 PROTEIN3 and PROTEIN4 in their signal transduction pathways, and PROTEIN5 is the common mediator of apoptosis by all known death domain-containing receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Finally it was shown that the two isoforms of PROTEIN0 can be separated from each other in the absence of PROTEIN1 also by chromatography on hydroxyapatite.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN7 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN8 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN9 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN1 PROTEIN7 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN1 PROTEIN8 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN1 PROTEIN9 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN2 PROTEIN7 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN2 PROTEIN8 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN2 PROTEIN9 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN3 PROTEIN7 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN3 PROTEIN8 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN3 PROTEIN9 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN4 PROTEIN7 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN4 PROTEIN8 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN4 PROTEIN9 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN5 PROTEIN7 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN5 PROTEIN8 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN5 PROTEIN9 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN6 PROTEIN7 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN6 PROTEIN8 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN6 PROTEIN9 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN7 PROTEIN8 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN7 PROTEIN9 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN8 PROTEIN9 Finally, we evaluated the acetylation of three putative PROTEIN0 substrates, PROTEIN1 PROTEIN2 and PROTEIN3 and the transcription factor PROTEIN4 and have determined that PROTEIN5 PROTEIN6 is significantly preferred over the PROTEIN7 PROTEIN8 and PROTEIN9 substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN1 First, the C-terminal PROTEIN0 PROTEIN1 domain was shown to be essential for the regulation of PROTEIN2 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN2 First, the C-terminal PROTEIN0 PROTEIN1 domain was shown to be essential for the regulation of PROTEIN2 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN2 First, the C-terminal PROTEIN0 PROTEIN1 domain was shown to be essential for the regulation of PROTEIN2 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Fluorescent PROTEIN0 analogs with a high affinity for PROTEIN1 in vitro exhibit an enhanced gradient of assembly in living cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Three-dimensional fluorescence microscopy of the fluorescent analog of PROTEIN0 with a high affinity for PROTEIN1 revealed that it incorporated into cortical cytoplasmic fibers and was also distributed diffusely in the non-cortical cytoplasm consistent with a bias of PROTEIN2 assembly near the surface of the cell.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Three-dimensional fluorescence microscopy of the fluorescent analog of PROTEIN0 with a high affinity for PROTEIN1 revealed that it incorporated into cortical cytoplasmic fibers and was also distributed diffusely in the non-cortical cytoplasm consistent with a bias of PROTEIN2 assembly near the surface of the cell.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Three-dimensional fluorescence microscopy of the fluorescent analog of PROTEIN0 with a high affinity for PROTEIN1 revealed that it incorporated into cortical cytoplasmic fibers and was also distributed diffusely in the non-cortical cytoplasm consistent with a bias of PROTEIN2 assembly near the surface of the cell.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Fluorographic analysis showed that the specific activity of soluble PROTEIN0 was two to three times that of its particulate form and that soluble PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were transported at similar rates in slow component b of axonal flow.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Fluorographic analysis showed that the specific activity of soluble PROTEIN0 was two to three times that of its particulate form and that soluble PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were transported at similar rates in slow component b of axonal flow.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Fluorographic analysis showed that the specific activity of soluble PROTEIN0 was two to three times that of its particulate form and that soluble PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were transported at similar rates in slow component b of axonal flow.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Fluorographic analysis showed that the specific activity of soluble PROTEIN0 was two to three times that of its particulate form and that soluble PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were transported at similar rates in slow component b of axonal flow.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Fluorographic analysis showed that the specific activity of soluble PROTEIN0 was two to three times that of its particulate form and that soluble PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were transported at similar rates in slow component b of axonal flow.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Fluorographic analysis showed that the specific activity of soluble PROTEIN0 was two to three times that of its particulate form and that soluble PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were transported at similar rates in slow component b of axonal flow.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Fluorographic analysis showed that the specific activity of soluble PROTEIN0 was two to three times that of its particulate form and that soluble PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were transported at similar rates in slow component b of axonal flow.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Fluorographic analysis showed that the specific activity of soluble PROTEIN0 was two to three times that of its particulate form and that soluble PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were transported at similar rates in slow component b of axonal flow.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Fluorographic analysis showed that the specific activity of soluble PROTEIN0 was two to three times that of its particulate form and that soluble PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were transported at similar rates in slow component b of axonal flow.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Fluorographic analysis showed that the specific activity of soluble PROTEIN0 was two to three times that of its particulate form and that soluble PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were transported at similar rates in slow component b of axonal flow.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our data strongly support the view that the mobile form of PROTEIN0 in slow transport is soluble and that a substantial amount of this PROTEIN1 may travel as a complex with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Our data strongly support the view that the mobile form of PROTEIN0 in slow transport is soluble and that a substantial amount of this PROTEIN1 may travel as a complex with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Our data strongly support the view that the mobile form of PROTEIN0 in slow transport is soluble and that a substantial amount of this PROTEIN1 may travel as a complex with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Our data strongly support the view that the mobile form of PROTEIN0 in slow transport is soluble and that a substantial amount of this PROTEIN1 may travel as a complex with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Our data strongly support the view that the mobile form of PROTEIN0 in slow transport is soluble and that a substantial amount of this PROTEIN1 may travel as a complex with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Our data strongly support the view that the mobile form of PROTEIN0 in slow transport is soluble and that a substantial amount of this PROTEIN1 may travel as a complex with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Our data strongly support the view that the mobile form of PROTEIN0 in slow transport is soluble and that a substantial amount of this PROTEIN1 may travel as a complex with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Our data strongly support the view that the mobile form of PROTEIN0 in slow transport is soluble and that a substantial amount of this PROTEIN1 may travel as a complex with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Our data strongly support the view that the mobile form of PROTEIN0 in slow transport is soluble and that a substantial amount of this PROTEIN1 may travel as a complex with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Our data strongly support the view that the mobile form of PROTEIN0 in slow transport is soluble and that a substantial amount of this PROTEIN1 may travel as a complex with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 We examined the axonal transport of PROTEIN0 and its monomer binding proteins, PROTEIN1 PROTEIN2 and PROTEIN3 in the chicken sciatic nerve following injection of [35S]methionine into the lumbar spinal cord.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We examined the axonal transport of PROTEIN0 and its monomer binding proteins, PROTEIN1 PROTEIN2 and PROTEIN3 in the chicken sciatic nerve following injection of [35S]methionine into the lumbar spinal cord.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We examined the axonal transport of PROTEIN0 and its monomer binding proteins, PROTEIN1 PROTEIN2 and PROTEIN3 in the chicken sciatic nerve following injection of [35S]methionine into the lumbar spinal cord.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We examined the axonal transport of PROTEIN0 and its monomer binding proteins, PROTEIN1 PROTEIN2 and PROTEIN3 in the chicken sciatic nerve following injection of [35S]methionine into the lumbar spinal cord.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We examined the axonal transport of PROTEIN0 and its monomer binding proteins, PROTEIN1 PROTEIN2 and PROTEIN3 in the chicken sciatic nerve following injection of [35S]methionine into the lumbar spinal cord.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We examined the axonal transport of PROTEIN0 and its monomer binding proteins, PROTEIN1 PROTEIN2 and PROTEIN3 in the chicken sciatic nerve following injection of [35S]methionine into the lumbar spinal cord.\n",
      "\n",
      "PROTEIN0 PROTEIN1 For example, it is associated with PROTEIN0 and PROTEIN1 PROTEIN2 and protected from PROTEIN3 digestion only in the presence of Ca2+, as is the case for many of classic PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 For example, it is associated with PROTEIN0 and PROTEIN1 PROTEIN2 and protected from PROTEIN3 digestion only in the presence of Ca2+, as is the case for many of classic PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 For example, it is associated with PROTEIN0 and PROTEIN1 PROTEIN2 and protected from PROTEIN3 digestion only in the presence of Ca2+, as is the case for many of classic PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 For example, it is associated with PROTEIN0 and PROTEIN1 PROTEIN2 and protected from PROTEIN3 digestion only in the presence of Ca2+, as is the case for many of classic PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 For example, it is associated with PROTEIN0 and PROTEIN1 PROTEIN2 and protected from PROTEIN3 digestion only in the presence of Ca2+, as is the case for many of classic PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 For example, it is associated with PROTEIN0 and PROTEIN1 PROTEIN2 and protected from PROTEIN3 digestion only in the presence of Ca2+, as is the case for many of classic PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 For example, it is associated with PROTEIN0 and PROTEIN1 PROTEIN2 and protected from PROTEIN3 digestion only in the presence of Ca2+, as is the case for many of classic PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 For example, it is associated with PROTEIN0 and PROTEIN1 PROTEIN2 and protected from PROTEIN3 digestion only in the presence of Ca2+, as is the case for many of classic PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 For example, it is associated with PROTEIN0 and PROTEIN1 PROTEIN2 and protected from PROTEIN3 digestion only in the presence of Ca2+, as is the case for many of classic PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 For example, it is associated with PROTEIN0 and PROTEIN1 PROTEIN2 and protected from PROTEIN3 digestion only in the presence of Ca2+, as is the case for many of classic PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 From an analysis of PROTEIN0 mutants, whose PROTEIN1 cytoskeleton is disrupted, we found that PROTEIN2 also requires PROTEIN3 filaments to associate with adhesive cell membranes in the ovary.\n",
      "\n",
      "PROTEIN0 PROTEIN2 From an analysis of PROTEIN0 mutants, whose PROTEIN1 cytoskeleton is disrupted, we found that PROTEIN2 also requires PROTEIN3 filaments to associate with adhesive cell membranes in the ovary.\n",
      "\n",
      "PROTEIN0 PROTEIN3 From an analysis of PROTEIN0 mutants, whose PROTEIN1 cytoskeleton is disrupted, we found that PROTEIN2 also requires PROTEIN3 filaments to associate with adhesive cell membranes in the ovary.\n",
      "\n",
      "PROTEIN1 PROTEIN2 From an analysis of PROTEIN0 mutants, whose PROTEIN1 cytoskeleton is disrupted, we found that PROTEIN2 also requires PROTEIN3 filaments to associate with adhesive cell membranes in the ovary.\n",
      "\n",
      "PROTEIN1 PROTEIN3 From an analysis of PROTEIN0 mutants, whose PROTEIN1 cytoskeleton is disrupted, we found that PROTEIN2 also requires PROTEIN3 filaments to associate with adhesive cell membranes in the ovary.\n",
      "\n",
      "PROTEIN2 PROTEIN3 From an analysis of PROTEIN0 mutants, whose PROTEIN1 cytoskeleton is disrupted, we found that PROTEIN2 also requires PROTEIN3 filaments to associate with adhesive cell membranes in the ovary.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, a simultaneous expression of PROTEIN0 PROTEIN1 and PROTEIN2 was observed during the regeneration of rat liver, that is, all mRNAs of these proteins showed biphasic peaks around 6 h and 48 h after partial hepatectomy.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Furthermore, a simultaneous expression of PROTEIN0 PROTEIN1 and PROTEIN2 was observed during the regeneration of rat liver, that is, all mRNAs of these proteins showed biphasic peaks around 6 h and 48 h after partial hepatectomy.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Furthermore, a simultaneous expression of PROTEIN0 PROTEIN1 and PROTEIN2 was observed during the regeneration of rat liver, that is, all mRNAs of these proteins showed biphasic peaks around 6 h and 48 h after partial hepatectomy.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, the deletion of PROTEIN0 suppresses the temperature-sensitive growth defect of PROTEIN1 a mutant in yeast PROTEIN2 supporting a role for PROTEIN3 family members in PROTEIN4 function.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Furthermore, the deletion of PROTEIN0 suppresses the temperature-sensitive growth defect of PROTEIN1 a mutant in yeast PROTEIN2 supporting a role for PROTEIN3 family members in PROTEIN4 function.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Furthermore, the deletion of PROTEIN0 suppresses the temperature-sensitive growth defect of PROTEIN1 a mutant in yeast PROTEIN2 supporting a role for PROTEIN3 family members in PROTEIN4 function.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Furthermore, the deletion of PROTEIN0 suppresses the temperature-sensitive growth defect of PROTEIN1 a mutant in yeast PROTEIN2 supporting a role for PROTEIN3 family members in PROTEIN4 function.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Furthermore, the deletion of PROTEIN0 suppresses the temperature-sensitive growth defect of PROTEIN1 a mutant in yeast PROTEIN2 supporting a role for PROTEIN3 family members in PROTEIN4 function.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Furthermore, the deletion of PROTEIN0 suppresses the temperature-sensitive growth defect of PROTEIN1 a mutant in yeast PROTEIN2 supporting a role for PROTEIN3 family members in PROTEIN4 function.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Furthermore, the deletion of PROTEIN0 suppresses the temperature-sensitive growth defect of PROTEIN1 a mutant in yeast PROTEIN2 supporting a role for PROTEIN3 family members in PROTEIN4 function.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Furthermore, the deletion of PROTEIN0 suppresses the temperature-sensitive growth defect of PROTEIN1 a mutant in yeast PROTEIN2 supporting a role for PROTEIN3 family members in PROTEIN4 function.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Furthermore, the deletion of PROTEIN0 suppresses the temperature-sensitive growth defect of PROTEIN1 a mutant in yeast PROTEIN2 supporting a role for PROTEIN3 family members in PROTEIN4 function.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Furthermore, the deletion of PROTEIN0 suppresses the temperature-sensitive growth defect of PROTEIN1 a mutant in yeast PROTEIN2 supporting a role for PROTEIN3 family members in PROTEIN4 function.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, this inactivating effect of PROTEIN0 can be prevented by coexpressing a region of the cytoskeletal protein PROTEIN1 known to interact with the CO region of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Furthermore, this inactivating effect of PROTEIN0 can be prevented by coexpressing a region of the cytoskeletal protein PROTEIN1 known to interact with the CO region of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Furthermore, this inactivating effect of PROTEIN0 can be prevented by coexpressing a region of the cytoskeletal protein PROTEIN1 known to interact with the CO region of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Further, PRP incubated with PROTEIN0 showed a dose dependent increase in PROTEIN1 and PROTEIN2 secretion as measured by RIA and an increased incorporation of PROTEIN3 PROTEIN4 and PROTEIN5 into the cytoskeletal core (triton insoluble residue) as shown by SDS-PAGE.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Further, we show that PROTEIN0 a high molecular weight structural protein that links PROTEIN1 to the PROTEIN2 cytoskeleton, is proteolytically cleaved in fetal, but not in neonatal melanocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Further, we show that PROTEIN0 a high molecular weight structural protein that links PROTEIN1 to the PROTEIN2 cytoskeleton, is proteolytically cleaved in fetal, but not in neonatal melanocytes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Further, we show that PROTEIN0 a high molecular weight structural protein that links PROTEIN1 to the PROTEIN2 cytoskeleton, is proteolytically cleaved in fetal, but not in neonatal melanocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunofluorescence microscopy of cells grown on PROTEIN0 confirmed the presence of PROTEIN1 PROTEIN2 and PROTEIN3 at the ends of PROTEIN4 stress fibers at presumptive focal contacts in melanocytes.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PROTEIN0 PROTEIN2 Immunofluorescence microscopy of cells grown on PROTEIN0 confirmed the presence of PROTEIN1 PROTEIN2 and PROTEIN3 at the ends of PROTEIN4 stress fibers at presumptive focal contacts in melanocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunofluorescence microscopy of cells grown on PROTEIN0 confirmed the presence of PROTEIN1 PROTEIN2 and PROTEIN3 at the ends of PROTEIN4 stress fibers at presumptive focal contacts in melanocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Immunofluorescence microscopy of cells grown on PROTEIN0 confirmed the presence of PROTEIN1 PROTEIN2 and PROTEIN3 at the ends of PROTEIN4 stress fibers at presumptive focal contacts in melanocytes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunofluorescence microscopy of cells grown on PROTEIN0 confirmed the presence of PROTEIN1 PROTEIN2 and PROTEIN3 at the ends of PROTEIN4 stress fibers at presumptive focal contacts in melanocytes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunofluorescence microscopy of cells grown on PROTEIN0 confirmed the presence of PROTEIN1 PROTEIN2 and PROTEIN3 at the ends of PROTEIN4 stress fibers at presumptive focal contacts in melanocytes.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Immunofluorescence microscopy of cells grown on PROTEIN0 confirmed the presence of PROTEIN1 PROTEIN2 and PROTEIN3 at the ends of PROTEIN4 stress fibers at presumptive focal contacts in melanocytes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunofluorescence microscopy of cells grown on PROTEIN0 confirmed the presence of PROTEIN1 PROTEIN2 and PROTEIN3 at the ends of PROTEIN4 stress fibers at presumptive focal contacts in melanocytes.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Immunofluorescence microscopy of cells grown on PROTEIN0 confirmed the presence of PROTEIN1 PROTEIN2 and PROTEIN3 at the ends of PROTEIN4 stress fibers at presumptive focal contacts in melanocytes.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Immunofluorescence microscopy of cells grown on PROTEIN0 confirmed the presence of PROTEIN1 PROTEIN2 and PROTEIN3 at the ends of PROTEIN4 stress fibers at presumptive focal contacts in melanocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Fusiform RPE, however, were not deficient in these proteins, expressing amounts of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that were equivalent to epithelioid cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Gel electrophoresis was used to measure changes in the relative amounts of PROTEIN0 or PROTEIN1 and of PROTEIN2 PROTEIN3 isoforms.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Gel electrophoresis was used to measure changes in the relative amounts of PROTEIN0 or PROTEIN1 and of PROTEIN2 PROTEIN3 isoforms.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Gel electrophoresis was used to measure changes in the relative amounts of PROTEIN0 or PROTEIN1 and of PROTEIN2 PROTEIN3 isoforms.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Gel electrophoresis was used to measure changes in the relative amounts of PROTEIN0 or PROTEIN1 and of PROTEIN2 PROTEIN3 isoforms.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Gel electrophoresis was used to measure changes in the relative amounts of PROTEIN0 or PROTEIN1 and of PROTEIN2 PROTEIN3 isoforms.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Gel electrophoresis was used to measure changes in the relative amounts of PROTEIN0 or PROTEIN1 and of PROTEIN2 PROTEIN3 isoforms.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Generation of these membrane microparticles was dependent on the presence of extracellular calcium and was accompanied by proteolytic degradation of the cytoskeletal proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Generation of these membrane microparticles was dependent on the presence of extracellular calcium and was accompanied by proteolytic degradation of the cytoskeletal proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Generation of these membrane microparticles was dependent on the presence of extracellular calcium and was accompanied by proteolytic degradation of the cytoskeletal proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Generation of these membrane microparticles was dependent on the presence of extracellular calcium and was accompanied by proteolytic degradation of the cytoskeletal proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Generation of these membrane microparticles was dependent on the presence of extracellular calcium and was accompanied by proteolytic degradation of the cytoskeletal proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Generation of these membrane microparticles was dependent on the presence of extracellular calcium and was accompanied by proteolytic degradation of the cytoskeletal proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Genetic dissection of Drosophila myofibril formation: effects of PROTEIN0 and PROTEIN1 null alleles.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN0 PROTEIN5 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN1 PROTEIN5 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN2 PROTEIN5 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN3 PROTEIN5 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN4 PROTEIN5 We find that heterozygotes for PROTEIN0 PROTEIN1 or PROTEIN2 PROTEIN3 null alleles have complex myofibrillar defects, whereas PROTEIN4 PROTEIN5 double heterozygotes have nearly normal myofibrils.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Germline mutations in several members of these families, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but not PROTEIN4 are responsible for hereditary non-polyposis colorectal cancer.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Germline mutations in several members of these families, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but not PROTEIN4 are responsible for hereditary non-polyposis colorectal cancer.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Germline mutations in several members of these families, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but not PROTEIN4 are responsible for hereditary non-polyposis colorectal cancer.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Germline mutations in several members of these families, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but not PROTEIN4 are responsible for hereditary non-polyposis colorectal cancer.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Germline mutations in several members of these families, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but not PROTEIN4 are responsible for hereditary non-polyposis colorectal cancer.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Germline mutations in several members of these families, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but not PROTEIN4 are responsible for hereditary non-polyposis colorectal cancer.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Germline mutations in several members of these families, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but not PROTEIN4 are responsible for hereditary non-polyposis colorectal cancer.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Germline mutations in several members of these families, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but not PROTEIN4 are responsible for hereditary non-polyposis colorectal cancer.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Germline mutations in several members of these families, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but not PROTEIN4 are responsible for hereditary non-polyposis colorectal cancer.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Germline mutations in several members of these families, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but not PROTEIN4 are responsible for hereditary non-polyposis colorectal cancer.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Hence, we conclude that in its dimeric form, which is used in PROTEIN0 and lipid binding, PROTEIN1 is a dumbbell-shaped molecule built of two antiparallel subunits.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 dimer, which is crucial for PROTEIN1 and lipid binding, is built of a backbone containing the 200 kDa rod portions, at both ends of which a 47 kDa globular domain is attached.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, specific interactions between human PROTEIN0 and PROTEIN1 proteins are demonstrated both in vivo, using the yeast two-hybrid system and immunoprecipitation of insect cells co-infected with PROTEIN2 and PROTEIN3 recombinant viruses, and in vitro, using affinity chromatography with purified recombinant proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here, specific interactions between human PROTEIN0 and PROTEIN1 proteins are demonstrated both in vivo, using the yeast two-hybrid system and immunoprecipitation of insect cells co-infected with PROTEIN2 and PROTEIN3 recombinant viruses, and in vitro, using affinity chromatography with purified recombinant proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here, specific interactions between human PROTEIN0 and PROTEIN1 proteins are demonstrated both in vivo, using the yeast two-hybrid system and immunoprecipitation of insect cells co-infected with PROTEIN2 and PROTEIN3 recombinant viruses, and in vitro, using affinity chromatography with purified recombinant proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here, specific interactions between human PROTEIN0 and PROTEIN1 proteins are demonstrated both in vivo, using the yeast two-hybrid system and immunoprecipitation of insect cells co-infected with PROTEIN2 and PROTEIN3 recombinant viruses, and in vitro, using affinity chromatography with purified recombinant proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here, specific interactions between human PROTEIN0 and PROTEIN1 proteins are demonstrated both in vivo, using the yeast two-hybrid system and immunoprecipitation of insect cells co-infected with PROTEIN2 and PROTEIN3 recombinant viruses, and in vitro, using affinity chromatography with purified recombinant proteins.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here, specific interactions between human PROTEIN0 and PROTEIN1 proteins are demonstrated both in vivo, using the yeast two-hybrid system and immunoprecipitation of insect cells co-infected with PROTEIN2 and PROTEIN3 recombinant viruses, and in vitro, using affinity chromatography with purified recombinant proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interaction of the yeast PROTEIN0 and PROTEIN1 proteins plays a crucial role in yeast DNA homologous recombination and DNA double-strand break repair.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Specific interactions between the human PROTEIN0 and PROTEIN1 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 region (amino acids 291-330) of the human PROTEIN1 protein shows no homology with the yeast PROTEIN2 protein, indicating that the interaction between PROTEIN3 and PROTEIN4 is species-specific.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 region (amino acids 291-330) of the human PROTEIN1 protein shows no homology with the yeast PROTEIN2 protein, indicating that the interaction between PROTEIN3 and PROTEIN4 is species-specific.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 region (amino acids 291-330) of the human PROTEIN1 protein shows no homology with the yeast PROTEIN2 protein, indicating that the interaction between PROTEIN3 and PROTEIN4 is species-specific.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The PROTEIN0 region (amino acids 291-330) of the human PROTEIN1 protein shows no homology with the yeast PROTEIN2 protein, indicating that the interaction between PROTEIN3 and PROTEIN4 is species-specific.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 region (amino acids 291-330) of the human PROTEIN1 protein shows no homology with the yeast PROTEIN2 protein, indicating that the interaction between PROTEIN3 and PROTEIN4 is species-specific.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 region (amino acids 291-330) of the human PROTEIN1 protein shows no homology with the yeast PROTEIN2 protein, indicating that the interaction between PROTEIN3 and PROTEIN4 is species-specific.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The PROTEIN0 region (amino acids 291-330) of the human PROTEIN1 protein shows no homology with the yeast PROTEIN2 protein, indicating that the interaction between PROTEIN3 and PROTEIN4 is species-specific.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 region (amino acids 291-330) of the human PROTEIN1 protein shows no homology with the yeast PROTEIN2 protein, indicating that the interaction between PROTEIN3 and PROTEIN4 is species-specific.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The PROTEIN0 region (amino acids 291-330) of the human PROTEIN1 protein shows no homology with the yeast PROTEIN2 protein, indicating that the interaction between PROTEIN3 and PROTEIN4 is species-specific.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The PROTEIN0 region (amino acids 291-330) of the human PROTEIN1 protein shows no homology with the yeast PROTEIN2 protein, indicating that the interaction between PROTEIN3 and PROTEIN4 is species-specific.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 may modulate the catalytic activities of PROTEIN1 protein such as homologous pairing and strand exchange through a direct physical interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we demonstrate that accessory receptor triggering induces the transient association of PROTEIN0 with the PROTEIN1 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Recently, we have shown that the small PROTEIN0 protein PROTEIN1 represents a component of a co-stimulatory signaling pathway in human T cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 provides an as yet missing link between functionally crucial T cell surface receptors and rearrangements of the PROTEIN1 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN4 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN5 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN6 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN7 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN8 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN9 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN4 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN5 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN6 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN7 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN8 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN9 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN4 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN5 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN6 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN7 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN8 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN9 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN4 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN5 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PROTEIN3 PROTEIN6 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN7 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN8 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN9 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN5 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN6 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN7 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN8 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN9 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN5 PROTEIN6 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN5 PROTEIN7 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN5 PROTEIN8 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN5 PROTEIN9 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN6 PROTEIN7 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN6 PROTEIN8 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN6 PROTEIN9 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN7 PROTEIN8 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN7 PROTEIN9 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN8 PROTEIN9 Here we demonstrate that growth arrest and differentiation of NT2/D1 cells induced by HMBA involve increased expression of the PROTEIN0 PROTEIN1 enhanced association of PROTEIN2 with PROTEIN3 PROTEIN4 complexes and suppression of PROTEIN5 activity associated to PROTEIN6 PROTEIN7 (but not to PROTEIN8 PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we demonstrate that PROTEIN0 is concentrated in cultured skin fibroblasts and several other human mesenchymal cell lines in the focal adhesions at the ends of PROTEIN1 stress fibers, co-localizing with the known focal adhesion components PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we demonstrate that PROTEIN0 is concentrated in cultured skin fibroblasts and several other human mesenchymal cell lines in the focal adhesions at the ends of PROTEIN1 stress fibers, co-localizing with the known focal adhesion components PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here we demonstrate that PROTEIN0 is concentrated in cultured skin fibroblasts and several other human mesenchymal cell lines in the focal adhesions at the ends of PROTEIN1 stress fibers, co-localizing with the known focal adhesion components PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we demonstrate that PROTEIN0 is concentrated in cultured skin fibroblasts and several other human mesenchymal cell lines in the focal adhesions at the ends of PROTEIN1 stress fibers, co-localizing with the known focal adhesion components PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here we demonstrate that PROTEIN0 is concentrated in cultured skin fibroblasts and several other human mesenchymal cell lines in the focal adhesions at the ends of PROTEIN1 stress fibers, co-localizing with the known focal adhesion components PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here we demonstrate that PROTEIN0 is concentrated in cultured skin fibroblasts and several other human mesenchymal cell lines in the focal adhesions at the ends of PROTEIN1 stress fibers, co-localizing with the known focal adhesion components PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, we describe proline-rich proteins involved in regulating PROTEIN0 polymerization and classify them according to their role in recruiting PROTEIN1 to the membrane.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN4 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN5 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN6 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN4 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN5 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN6 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN4 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN5 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN6 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN4 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN5 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN6 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN5 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN6 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN5 PROTEIN6 Here we describe the PROTEIN0 activation of PROTEIN1 PROTEIN2 through the PROTEIN3 PROTEIN4 proteins PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 death domain-associated proteins PROTEIN1 and PROTEIN2 signal activation of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 death domain-associated proteins PROTEIN1 and PROTEIN2 signal activation of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 death domain-associated proteins PROTEIN1 and PROTEIN2 signal activation of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 death domain-associated proteins PROTEIN1 and PROTEIN2 signal activation of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 death domain-associated proteins PROTEIN1 and PROTEIN2 signal activation of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 death domain-associated proteins PROTEIN1 and PROTEIN2 signal activation of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we have now identified the reciprocal complementary binding site in PROTEIN0 which mediates its interaction with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we have now identified the reciprocal complementary binding site in PROTEIN0 which mediates its interaction with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we have now identified the reciprocal complementary binding site in PROTEIN0 which mediates its interaction with PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 It is well established that the cytoplasmic domain of PROTEIN0 binds either PROTEIN1 or PROTEIN2 which both can assemble PROTEIN3 into the complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 It is well established that the cytoplasmic domain of PROTEIN0 binds either PROTEIN1 or PROTEIN2 which both can assemble PROTEIN3 into the complex.\n",
      "\n",
      "PROTEIN0 PROTEIN3 It is well established that the cytoplasmic domain of PROTEIN0 binds either PROTEIN1 or PROTEIN2 which both can assemble PROTEIN3 into the complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 It is well established that the cytoplasmic domain of PROTEIN0 binds either PROTEIN1 or PROTEIN2 which both can assemble PROTEIN3 into the complex.\n",
      "\n",
      "PROTEIN1 PROTEIN3 It is well established that the cytoplasmic domain of PROTEIN0 binds either PROTEIN1 or PROTEIN2 which both can assemble PROTEIN3 into the complex.\n",
      "\n",
      "PROTEIN2 PROTEIN3 It is well established that the cytoplasmic domain of PROTEIN0 binds either PROTEIN1 or PROTEIN2 which both can assemble PROTEIN3 into the complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Recently we have identified an PROTEIN0 binding site in PROTEIN1 and PROTEIN2 and postulated, based on sequence analysis, that these protein-protein interactions are mediated by a hydrophobic interaction mechanism.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Recently we have identified an PROTEIN0 binding site in PROTEIN1 and PROTEIN2 and postulated, based on sequence analysis, that these protein-protein interactions are mediated by a hydrophobic interaction mechanism.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Recently we have identified an PROTEIN0 binding site in PROTEIN1 and PROTEIN2 and postulated, based on sequence analysis, that these protein-protein interactions are mediated by a hydrophobic interaction mechanism.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we identify a pathway for the regulation of PROTEIN0 a ubiquitous PROTEIN1 that is essential for effective depolymerization of PROTEIN2 filaments.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we identify a pathway for the regulation of PROTEIN0 a ubiquitous PROTEIN1 that is essential for effective depolymerization of PROTEIN2 filaments.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we identify a pathway for the regulation of PROTEIN0 a ubiquitous PROTEIN1 that is essential for effective depolymerization of PROTEIN2 filaments.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our results define a mechanism for the regulation of PROTEIN0 and hence of PROTEIN1 dynamics in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Phosphorylation by PROTEIN0 inactivates PROTEIN1 leading to accumulation of PROTEIN2 filaments.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Phosphorylation by PROTEIN0 inactivates PROTEIN1 leading to accumulation of PROTEIN2 filaments.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Phosphorylation by PROTEIN0 inactivates PROTEIN1 leading to accumulation of PROTEIN2 filaments.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Regulation of PROTEIN0 dynamics through phosphorylation of PROTEIN1 by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Regulation of PROTEIN0 dynamics through phosphorylation of PROTEIN1 by PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Regulation of PROTEIN0 dynamics through phosphorylation of PROTEIN1 by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The essential role of PROTEIN0 in the assembly of PROTEIN1 for microspike formation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These findings provide the first evidence that PROTEIN0 is a key molecule linking a signaling network to rapid PROTEIN1 polymerization in microspike formation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This mutant PROTEIN0 suppresses PROTEIN1 polymerization in microspike formation induced by PROTEIN2 the essential factor in microspike formation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 This mutant PROTEIN0 suppresses PROTEIN1 polymerization in microspike formation induced by PROTEIN2 the essential factor in microspike formation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 This mutant PROTEIN0 suppresses PROTEIN1 polymerization in microspike formation induced by PROTEIN2 the essential factor in microspike formation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we report that PROTEIN0 associates with a complex containing PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we report that PROTEIN0 associates with a complex containing PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here we report that PROTEIN0 associates with a complex containing PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Here we report that PROTEIN0 associates with a complex containing PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we report that PROTEIN0 associates with a complex containing PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here we report that PROTEIN0 associates with a complex containing PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Here we report that PROTEIN0 associates with a complex containing PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here we report that PROTEIN0 associates with a complex containing PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Here we report that PROTEIN0 associates with a complex containing PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Here we report that PROTEIN0 associates with a complex containing PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we show that PROTEIN0 is a member of a family of proteins including Drosophila PROTEIN1 Saccharomyces cerevisiae PROTEIN2 and S. pombe PROTEIN3 which are involved in cytokinesis or other PROTEIN4 processes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we show that PROTEIN0 is a member of a family of proteins including Drosophila PROTEIN1 Saccharomyces cerevisiae PROTEIN2 and S. pombe PROTEIN3 which are involved in cytokinesis or other PROTEIN4 processes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here we show that PROTEIN0 is a member of a family of proteins including Drosophila PROTEIN1 Saccharomyces cerevisiae PROTEIN2 and S. pombe PROTEIN3 which are involved in cytokinesis or other PROTEIN4 processes.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Here we show that PROTEIN0 is a member of a family of proteins including Drosophila PROTEIN1 Saccharomyces cerevisiae PROTEIN2 and S. pombe PROTEIN3 which are involved in cytokinesis or other PROTEIN4 processes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we show that PROTEIN0 is a member of a family of proteins including Drosophila PROTEIN1 Saccharomyces cerevisiae PROTEIN2 and S. pombe PROTEIN3 which are involved in cytokinesis or other PROTEIN4 processes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here we show that PROTEIN0 is a member of a family of proteins including Drosophila PROTEIN1 Saccharomyces cerevisiae PROTEIN2 and S. pombe PROTEIN3 which are involved in cytokinesis or other PROTEIN4 processes.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Here we show that PROTEIN0 is a member of a family of proteins including Drosophila PROTEIN1 Saccharomyces cerevisiae PROTEIN2 and S. pombe PROTEIN3 which are involved in cytokinesis or other PROTEIN4 processes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here we show that PROTEIN0 is a member of a family of proteins including Drosophila PROTEIN1 Saccharomyces cerevisiae PROTEIN2 and S. pombe PROTEIN3 which are involved in cytokinesis or other PROTEIN4 processes.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Here we show that PROTEIN0 is a member of a family of proteins including Drosophila PROTEIN1 Saccharomyces cerevisiae PROTEIN2 and S. pombe PROTEIN3 which are involved in cytokinesis or other PROTEIN4 processes.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Here we show that PROTEIN0 is a member of a family of proteins including Drosophila PROTEIN1 Saccharomyces cerevisiae PROTEIN2 and S. pombe PROTEIN3 which are involved in cytokinesis or other PROTEIN4 processes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, we show that PROTEIN0 is a potent inhibitor of differentiation and link this activity to its ability to inhibit PROTEIN1 (and the highly related molecule, PROTEIN2 or PROTEIN3 PROTEIN4 acetylation activity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here, we show that PROTEIN0 is a potent inhibitor of differentiation and link this activity to its ability to inhibit PROTEIN1 (and the highly related molecule, PROTEIN2 or PROTEIN3 PROTEIN4 acetylation activity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here, we show that PROTEIN0 is a potent inhibitor of differentiation and link this activity to its ability to inhibit PROTEIN1 (and the highly related molecule, PROTEIN2 or PROTEIN3 PROTEIN4 acetylation activity.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Here, we show that PROTEIN0 is a potent inhibitor of differentiation and link this activity to its ability to inhibit PROTEIN1 (and the highly related molecule, PROTEIN2 or PROTEIN3 PROTEIN4 acetylation activity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here, we show that PROTEIN0 is a potent inhibitor of differentiation and link this activity to its ability to inhibit PROTEIN1 (and the highly related molecule, PROTEIN2 or PROTEIN3 PROTEIN4 acetylation activity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here, we show that PROTEIN0 is a potent inhibitor of differentiation and link this activity to its ability to inhibit PROTEIN1 (and the highly related molecule, PROTEIN2 or PROTEIN3 PROTEIN4 acetylation activity.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Here, we show that PROTEIN0 is a potent inhibitor of differentiation and link this activity to its ability to inhibit PROTEIN1 (and the highly related molecule, PROTEIN2 or PROTEIN3 PROTEIN4 acetylation activity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here, we show that PROTEIN0 is a potent inhibitor of differentiation and link this activity to its ability to inhibit PROTEIN1 (and the highly related molecule, PROTEIN2 or PROTEIN3 PROTEIN4 acetylation activity.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Here, we show that PROTEIN0 is a potent inhibitor of differentiation and link this activity to its ability to inhibit PROTEIN1 (and the highly related molecule, PROTEIN2 or PROTEIN3 PROTEIN4 acetylation activity.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Here, we show that PROTEIN0 is a potent inhibitor of differentiation and link this activity to its ability to inhibit PROTEIN1 (and the highly related molecule, PROTEIN2 or PROTEIN3 PROTEIN4 acetylation activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, we show that the x-ray sensitivity of PROTEIN0 and PROTEIN1 mutant strains is suppressible by overexpression of PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here, we show that the x-ray sensitivity of PROTEIN0 and PROTEIN1 mutant strains is suppressible by overexpression of PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here, we show that the x-ray sensitivity of PROTEIN0 and PROTEIN1 mutant strains is suppressible by overexpression of PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here, we show that the x-ray sensitivity of PROTEIN0 and PROTEIN1 mutant strains is suppressible by overexpression of PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here, we show that the x-ray sensitivity of PROTEIN0 and PROTEIN1 mutant strains is suppressible by overexpression of PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here, we show that the x-ray sensitivity of PROTEIN0 and PROTEIN1 mutant strains is suppressible by overexpression of PROTEIN2 or PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binds to predicted alpha-helical coiled-coil regions of PROTEIN1 and PROTEIN2 shown previously to be engaged in their direct interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 binds to predicted alpha-helical coiled-coil regions of PROTEIN1 and PROTEIN2 shown previously to be engaged in their direct interaction.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 binds to predicted alpha-helical coiled-coil regions of PROTEIN1 and PROTEIN2 shown previously to be engaged in their direct interaction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, we sought to determine the spatial distributions of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and whether different complexes of these proteins accumulate at steady state in polarized Madin-Darby canine kidney cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here, we sought to determine the spatial distributions of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and whether different complexes of these proteins accumulate at steady state in polarized Madin-Darby canine kidney cells.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here, we sought to determine the spatial distributions of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and whether different complexes of these proteins accumulate at steady state in polarized Madin-Darby canine kidney cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here, we sought to determine the spatial distributions of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and whether different complexes of these proteins accumulate at steady state in polarized Madin-Darby canine kidney cells.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here, we sought to determine the spatial distributions of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and whether different complexes of these proteins accumulate at steady state in polarized Madin-Darby canine kidney cells.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here, we sought to determine the spatial distributions of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 and whether different complexes of these proteins accumulate at steady state in polarized Madin-Darby canine kidney cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Herpes simplex virus type 1 encodes a heterotrimeric PROTEIN0 complex that is composed of the products of the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Herpes simplex virus type 1 encodes a heterotrimeric PROTEIN0 complex that is composed of the products of the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Herpes simplex virus type 1 encodes a heterotrimeric PROTEIN0 complex that is composed of the products of the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Herpes simplex virus type 1 encodes a heterotrimeric PROTEIN0 complex that is composed of the products of the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Herpes simplex virus type 1 encodes a heterotrimeric PROTEIN0 complex that is composed of the products of the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Herpes simplex virus type 1 encodes a heterotrimeric PROTEIN0 complex that is composed of the products of the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Herpes simplex virus type 1 expresses a heterotrimeric PROTEIN0 the subunits of which are encoded by the viral PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Herpes simplex virus type 1 expresses a heterotrimeric PROTEIN0 the subunits of which are encoded by the viral PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Herpes simplex virus type 1 expresses a heterotrimeric PROTEIN0 the subunits of which are encoded by the viral PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Herpes simplex virus type 1 expresses a heterotrimeric PROTEIN0 the subunits of which are encoded by the viral PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Herpes simplex virus type 1 expresses a heterotrimeric PROTEIN0 the subunits of which are encoded by the viral PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Herpes simplex virus type 1 expresses a heterotrimeric PROTEIN0 the subunits of which are encoded by the viral PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The interactions of the PROTEIN0 protein with the PROTEIN1 and PROTEIN2 proteins were analysed by using the yeast two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The interactions of the PROTEIN0 protein with the PROTEIN1 and PROTEIN2 proteins were analysed by using the yeast two-hybrid system.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The interactions of the PROTEIN0 protein with the PROTEIN1 and PROTEIN2 proteins were analysed by using the yeast two-hybrid system.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Herpes simplex virus type 1 (HSV-1) encodes a PROTEIN0 that consists of three polypeptides encoded by the PROTEIN1 PROTEIN2 and PROTEIN3 genes (Crute, J.J., Tsurumi, T., Zhu, L., Weller, S.K., Olivo, P.D., Challberg, M.D., Mocarski, E.S., and Lehman, I.R. (1989) Proc. Natl.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Herpes simplex virus type 1 (HSV-1) encodes a PROTEIN0 that consists of three polypeptides encoded by the PROTEIN1 PROTEIN2 and PROTEIN3 genes (Crute, J.J., Tsurumi, T., Zhu, L., Weller, S.K., Olivo, P.D., Challberg, M.D., Mocarski, E.S., and Lehman, I.R. (1989) Proc. Natl.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Herpes simplex virus type 1 (HSV-1) encodes a PROTEIN0 that consists of three polypeptides encoded by the PROTEIN1 PROTEIN2 and PROTEIN3 genes (Crute, J.J., Tsurumi, T., Zhu, L., Weller, S.K., Olivo, P.D., Challberg, M.D., Mocarski, E.S., and Lehman, I.R. (1989) Proc. Natl.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Herpes simplex virus type 1 (HSV-1) encodes a PROTEIN0 that consists of three polypeptides encoded by the PROTEIN1 PROTEIN2 and PROTEIN3 genes (Crute, J.J., Tsurumi, T., Zhu, L., Weller, S.K., Olivo, P.D., Challberg, M.D., Mocarski, E.S., and Lehman, I.R. (1989) Proc. Natl.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Herpes simplex virus type 1 (HSV-1) encodes a PROTEIN0 that consists of three polypeptides encoded by the PROTEIN1 PROTEIN2 and PROTEIN3 genes (Crute, J.J., Tsurumi, T., Zhu, L., Weller, S.K., Olivo, P.D., Challberg, M.D., Mocarski, E.S., and Lehman, I.R. (1989) Proc. Natl.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Herpes simplex virus type 1 (HSV-1) encodes a PROTEIN0 that consists of three polypeptides encoded by the PROTEIN1 PROTEIN2 and PROTEIN3 genes (Crute, J.J., Tsurumi, T., Zhu, L., Weller, S.K., Olivo, P.D., Challberg, M.D., Mocarski, E.S., and Lehman, I.R. (1989) Proc. Natl.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Herpes simplex virus type 1 (HSV-1) encodes a heterotrimeric PROTEIN0 composed of the products of the three DNA replication-specific genes PROTEIN1 PROTEIN2 and PROTEIN3 (Crute, J. J., and Lehman, I. R. (1991) J. Biol. Chem. 266, 4484-4488).\n",
      "\n",
      "PROTEIN0 PROTEIN2 Herpes simplex virus type 1 (HSV-1) encodes a heterotrimeric PROTEIN0 composed of the products of the three DNA replication-specific genes PROTEIN1 PROTEIN2 and PROTEIN3 (Crute, J. J., and Lehman, I. R. (1991) J. Biol. Chem. 266, 4484-4488).\n",
      "\n",
      "PROTEIN0 PROTEIN3 Herpes simplex virus type 1 (HSV-1) encodes a heterotrimeric PROTEIN0 composed of the products of the three DNA replication-specific genes PROTEIN1 PROTEIN2 and PROTEIN3 (Crute, J. J., and Lehman, I. R. (1991) J. Biol. Chem. 266, 4484-4488).\n",
      "\n",
      "PROTEIN1 PROTEIN2 Herpes simplex virus type 1 (HSV-1) encodes a heterotrimeric PROTEIN0 composed of the products of the three DNA replication-specific genes PROTEIN1 PROTEIN2 and PROTEIN3 (Crute, J. J., and Lehman, I. R. (1991) J. Biol. Chem. 266, 4484-4488).\n",
      "\n",
      "PROTEIN1 PROTEIN3 Herpes simplex virus type 1 (HSV-1) encodes a heterotrimeric PROTEIN0 composed of the products of the three DNA replication-specific genes PROTEIN1 PROTEIN2 and PROTEIN3 (Crute, J. J., and Lehman, I. R. (1991) J. Biol. Chem. 266, 4484-4488).\n",
      "\n",
      "PROTEIN2 PROTEIN3 Herpes simplex virus type 1 (HSV-1) encodes a heterotrimeric PROTEIN0 composed of the products of the three DNA replication-specific genes PROTEIN1 PROTEIN2 and PROTEIN3 (Crute, J. J., and Lehman, I. R. (1991) J. Biol. Chem. 266, 4484-4488).\n",
      "\n",
      "PROTEIN0 PROTEIN1 High levels of PROTEIN0 suppress the lethality caused by overproduction of PROTEIN1 in yeast cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, overexpression of the PROTEIN0 gene, PROTEIN1 encoding an PROTEIN2 leads to no very obvious phenotype.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In contrast, overexpression of the PROTEIN0 gene, PROTEIN1 encoding an PROTEIN2 leads to no very obvious phenotype.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In contrast, overexpression of the PROTEIN0 gene, PROTEIN1 encoding an PROTEIN2 leads to no very obvious phenotype.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interestingly, PROTEIN0 overproduction can compensate for the deleterious effects of too much PROTEIN1 in a PROTEIN2 concentration-dependent manner.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Interestingly, PROTEIN0 overproduction can compensate for the deleterious effects of too much PROTEIN1 in a PROTEIN2 concentration-dependent manner.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Interestingly, PROTEIN0 overproduction can compensate for the deleterious effects of too much PROTEIN1 in a PROTEIN2 concentration-dependent manner.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 PROTEIN1 showed at least seven sites of acetylation, PROTEIN2 PROTEIN3 had six sites and PROTEIN4 PROTEIN5 had five sites.\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN3 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN4 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN5 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN6 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN7 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN3 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN4 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN5 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN6 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN7 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN3 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN4 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN5 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN6 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN7 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN4 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN5 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN6 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN7 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN4 PROTEIN5 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN4 PROTEIN6 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN4 PROTEIN7 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN5 PROTEIN6 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN5 PROTEIN7 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN6 PROTEIN7 However, although expression of PROTEIN0 alone will not induce G1 PROTEIN1 activity or S phase, coexpression of PROTEIN2 with PROTEIN3 allows the generation of PROTEIN4 activity and the induction of S phase, coincident with the loss of the PROTEIN5 PROTEIN6 PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN3 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN4 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN5 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN6 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN7 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN3 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN4 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN5 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN6 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN7 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN3 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN4 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN5 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN6 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN7 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN4 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN5 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN6 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN7 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN4 PROTEIN5 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN4 PROTEIN6 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN4 PROTEIN7 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN5 PROTEIN6 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN5 PROTEIN7 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN6 PROTEIN7 However, a number of mammalian DNA repair proteins lack NLS clusters; these proteins include PROTEIN0 PROTEIN1 PROTEIN2 mouse PROTEIN3 and the PROTEIN4 PROTEIN5 and PROTEIN6 subunits of PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN1 NLS-less S.cerevisiae proteins include both PROTEIN0 and PROTEIN1 that function in the recombination and in the repair of double-strand breaks as well as the PROTEIN2 and PROTEIN3 molecules.\n",
      "\n",
      "PROTEIN0 PROTEIN2 NLS-less S.cerevisiae proteins include both PROTEIN0 and PROTEIN1 that function in the recombination and in the repair of double-strand breaks as well as the PROTEIN2 and PROTEIN3 molecules.\n",
      "\n",
      "PROTEIN0 PROTEIN3 NLS-less S.cerevisiae proteins include both PROTEIN0 and PROTEIN1 that function in the recombination and in the repair of double-strand breaks as well as the PROTEIN2 and PROTEIN3 molecules.\n",
      "\n",
      "PROTEIN1 PROTEIN2 NLS-less S.cerevisiae proteins include both PROTEIN0 and PROTEIN1 that function in the recombination and in the repair of double-strand breaks as well as the PROTEIN2 and PROTEIN3 molecules.\n",
      "\n",
      "PROTEIN1 PROTEIN3 NLS-less S.cerevisiae proteins include both PROTEIN0 and PROTEIN1 that function in the recombination and in the repair of double-strand breaks as well as the PROTEIN2 and PROTEIN3 molecules.\n",
      "\n",
      "PROTEIN2 PROTEIN3 NLS-less S.cerevisiae proteins include both PROTEIN0 and PROTEIN1 that function in the recombination and in the repair of double-strand breaks as well as the PROTEIN2 and PROTEIN3 molecules.\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, disassembly of the PROTEIN0 cytoskeleton using Latrunculin-A does not alter PROTEIN1 location, indicating that PROTEIN2 establishes and maintains its location independent of the PROTEIN3 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, disassembly of the PROTEIN0 cytoskeleton using Latrunculin-A does not alter PROTEIN1 location, indicating that PROTEIN2 establishes and maintains its location independent of the PROTEIN3 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN3 However, disassembly of the PROTEIN0 cytoskeleton using Latrunculin-A does not alter PROTEIN1 location, indicating that PROTEIN2 establishes and maintains its location independent of the PROTEIN3 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, disassembly of the PROTEIN0 cytoskeleton using Latrunculin-A does not alter PROTEIN1 location, indicating that PROTEIN2 establishes and maintains its location independent of the PROTEIN3 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN3 However, disassembly of the PROTEIN0 cytoskeleton using Latrunculin-A does not alter PROTEIN1 location, indicating that PROTEIN2 establishes and maintains its location independent of the PROTEIN3 cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN3 However, disassembly of the PROTEIN0 cytoskeleton using Latrunculin-A does not alter PROTEIN1 location, indicating that PROTEIN2 establishes and maintains its location independent of the PROTEIN3 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using the two-hybrid system, we show that PROTEIN0 binds PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, immunofluorescence staining revealed a similar organisation of PROTEIN0 microfilaments, PROTEIN1 and phosphotyrosyl-containing proteins on both substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, increases were also selective in nature, with increases in certain individual proteins, including PROTEIN0 (twofold to threefold), PROTEIN1 (2.5-fold to sevenfold), PROTEIN2 (threefold to sixfold), and PROTEIN3 far exceeding overall increases in cellular protein content (20-40%).\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, increases were also selective in nature, with increases in certain individual proteins, including PROTEIN0 (twofold to threefold), PROTEIN1 (2.5-fold to sevenfold), PROTEIN2 (threefold to sixfold), and PROTEIN3 far exceeding overall increases in cellular protein content (20-40%).\n",
      "\n",
      "PROTEIN0 PROTEIN3 However, increases were also selective in nature, with increases in certain individual proteins, including PROTEIN0 (twofold to threefold), PROTEIN1 (2.5-fold to sevenfold), PROTEIN2 (threefold to sixfold), and PROTEIN3 far exceeding overall increases in cellular protein content (20-40%).\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, increases were also selective in nature, with increases in certain individual proteins, including PROTEIN0 (twofold to threefold), PROTEIN1 (2.5-fold to sevenfold), PROTEIN2 (threefold to sixfold), and PROTEIN3 far exceeding overall increases in cellular protein content (20-40%).\n",
      "\n",
      "PROTEIN1 PROTEIN3 However, increases were also selective in nature, with increases in certain individual proteins, including PROTEIN0 (twofold to threefold), PROTEIN1 (2.5-fold to sevenfold), PROTEIN2 (threefold to sixfold), and PROTEIN3 far exceeding overall increases in cellular protein content (20-40%).\n",
      "\n",
      "PROTEIN2 PROTEIN3 However, increases were also selective in nature, with increases in certain individual proteins, including PROTEIN0 (twofold to threefold), PROTEIN1 (2.5-fold to sevenfold), PROTEIN2 (threefold to sixfold), and PROTEIN3 far exceeding overall increases in cellular protein content (20-40%).\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, once initiated, the synthesis of PROTEIN0 mRNA continued in the presence of cycloheximide, while the expression of PROTEIN1 PROTEIN2 and PROTEIN3 mRNA was abolished.\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, once initiated, the synthesis of PROTEIN0 mRNA continued in the presence of cycloheximide, while the expression of PROTEIN1 PROTEIN2 and PROTEIN3 mRNA was abolished.\n",
      "\n",
      "PROTEIN0 PROTEIN3 However, once initiated, the synthesis of PROTEIN0 mRNA continued in the presence of cycloheximide, while the expression of PROTEIN1 PROTEIN2 and PROTEIN3 mRNA was abolished.\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, once initiated, the synthesis of PROTEIN0 mRNA continued in the presence of cycloheximide, while the expression of PROTEIN1 PROTEIN2 and PROTEIN3 mRNA was abolished.\n",
      "\n",
      "PROTEIN1 PROTEIN3 However, once initiated, the synthesis of PROTEIN0 mRNA continued in the presence of cycloheximide, while the expression of PROTEIN1 PROTEIN2 and PROTEIN3 mRNA was abolished.\n",
      "\n",
      "PROTEIN2 PROTEIN3 However, once initiated, the synthesis of PROTEIN0 mRNA continued in the presence of cycloheximide, while the expression of PROTEIN1 PROTEIN2 and PROTEIN3 mRNA was abolished.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that muscle-specific processing of PROTEIN0 transcripts can continue to occur in the absence of PROTEIN1 expression unlike the expression of PROTEIN2 and PROTEIN3 mRNAs.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results suggest that muscle-specific processing of PROTEIN0 transcripts can continue to occur in the absence of PROTEIN1 expression unlike the expression of PROTEIN2 and PROTEIN3 mRNAs.\n",
      "\n",
      "PROTEIN0 PROTEIN3 These results suggest that muscle-specific processing of PROTEIN0 transcripts can continue to occur in the absence of PROTEIN1 expression unlike the expression of PROTEIN2 and PROTEIN3 mRNAs.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results suggest that muscle-specific processing of PROTEIN0 transcripts can continue to occur in the absence of PROTEIN1 expression unlike the expression of PROTEIN2 and PROTEIN3 mRNAs.\n",
      "\n",
      "PROTEIN1 PROTEIN3 These results suggest that muscle-specific processing of PROTEIN0 transcripts can continue to occur in the absence of PROTEIN1 expression unlike the expression of PROTEIN2 and PROTEIN3 mRNAs.\n",
      "\n",
      "PROTEIN2 PROTEIN3 These results suggest that muscle-specific processing of PROTEIN0 transcripts can continue to occur in the absence of PROTEIN1 expression unlike the expression of PROTEIN2 and PROTEIN3 mRNAs.\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, similar to PROTEIN0 PROTEIN1 is only present at the neuromuscular junction in mdx mouse muscle in which PROTEIN2 is absent.\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, similar to PROTEIN0 PROTEIN1 is only present at the neuromuscular junction in mdx mouse muscle in which PROTEIN2 is absent.\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, similar to PROTEIN0 PROTEIN1 is only present at the neuromuscular junction in mdx mouse muscle in which PROTEIN2 is absent.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Identification of PROTEIN0 binding to PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Identification of PROTEIN0 binding to PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Identification of PROTEIN0 binding to PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Identification of PROTEIN0 binding to PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Identification of PROTEIN0 binding to PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Identification of PROTEIN0 binding to PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, smooth muscle contents of PROTEIN0 (26.0 +/- 1.8 vs. 19.1 +/- 2.2 micrograms/mg wet wt) and PROTEIN1 (5.5 +/- 0.4 vs. 2.0 +/- 0.3 microgram/mg wet wt) were greater (P < 0.04) in myometrium from late-pregnant vs. nonpregnant ewes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, some proteins (e.g., PROTEIN0 and other agents (e.g., cytochalasin D) that bind to PROTEIN1 are affected by the presence of fluorescent labels, making PROTEIN2 intrinsic fluorescence potentially useful in investigating the interaction of these agents with PROTEIN3 and in validating data obtained using labeled PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, some proteins (e.g., PROTEIN0 and other agents (e.g., cytochalasin D) that bind to PROTEIN1 are affected by the presence of fluorescent labels, making PROTEIN2 intrinsic fluorescence potentially useful in investigating the interaction of these agents with PROTEIN3 and in validating data obtained using labeled PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 However, some proteins (e.g., PROTEIN0 and other agents (e.g., cytochalasin D) that bind to PROTEIN1 are affected by the presence of fluorescent labels, making PROTEIN2 intrinsic fluorescence potentially useful in investigating the interaction of these agents with PROTEIN3 and in validating data obtained using labeled PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 However, some proteins (e.g., PROTEIN0 and other agents (e.g., cytochalasin D) that bind to PROTEIN1 are affected by the presence of fluorescent labels, making PROTEIN2 intrinsic fluorescence potentially useful in investigating the interaction of these agents with PROTEIN3 and in validating data obtained using labeled PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, some proteins (e.g., PROTEIN0 and other agents (e.g., cytochalasin D) that bind to PROTEIN1 are affected by the presence of fluorescent labels, making PROTEIN2 intrinsic fluorescence potentially useful in investigating the interaction of these agents with PROTEIN3 and in validating data obtained using labeled PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 However, some proteins (e.g., PROTEIN0 and other agents (e.g., cytochalasin D) that bind to PROTEIN1 are affected by the presence of fluorescent labels, making PROTEIN2 intrinsic fluorescence potentially useful in investigating the interaction of these agents with PROTEIN3 and in validating data obtained using labeled PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 However, some proteins (e.g., PROTEIN0 and other agents (e.g., cytochalasin D) that bind to PROTEIN1 are affected by the presence of fluorescent labels, making PROTEIN2 intrinsic fluorescence potentially useful in investigating the interaction of these agents with PROTEIN3 and in validating data obtained using labeled PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 However, some proteins (e.g., PROTEIN0 and other agents (e.g., cytochalasin D) that bind to PROTEIN1 are affected by the presence of fluorescent labels, making PROTEIN2 intrinsic fluorescence potentially useful in investigating the interaction of these agents with PROTEIN3 and in validating data obtained using labeled PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 However, some proteins (e.g., PROTEIN0 and other agents (e.g., cytochalasin D) that bind to PROTEIN1 are affected by the presence of fluorescent labels, making PROTEIN2 intrinsic fluorescence potentially useful in investigating the interaction of these agents with PROTEIN3 and in validating data obtained using labeled PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 However, some proteins (e.g., PROTEIN0 and other agents (e.g., cytochalasin D) that bind to PROTEIN1 are affected by the presence of fluorescent labels, making PROTEIN2 intrinsic fluorescence potentially useful in investigating the interaction of these agents with PROTEIN3 and in validating data obtained using labeled PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, these rod structures were not observed in response to PROTEIN0 forskolin, or TPA, suggesting that dephosphorylation of PROTEIN1 correlates with the reorganization of PROTEIN2 microfilaments but not with the nuclear transport of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, these rod structures were not observed in response to PROTEIN0 forskolin, or TPA, suggesting that dephosphorylation of PROTEIN1 correlates with the reorganization of PROTEIN2 microfilaments but not with the nuclear transport of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 However, these rod structures were not observed in response to PROTEIN0 forskolin, or TPA, suggesting that dephosphorylation of PROTEIN1 correlates with the reorganization of PROTEIN2 microfilaments but not with the nuclear transport of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, these rod structures were not observed in response to PROTEIN0 forskolin, or TPA, suggesting that dephosphorylation of PROTEIN1 correlates with the reorganization of PROTEIN2 microfilaments but not with the nuclear transport of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 However, these rod structures were not observed in response to PROTEIN0 forskolin, or TPA, suggesting that dephosphorylation of PROTEIN1 correlates with the reorganization of PROTEIN2 microfilaments but not with the nuclear transport of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 However, these rod structures were not observed in response to PROTEIN0 forskolin, or TPA, suggesting that dephosphorylation of PROTEIN1 correlates with the reorganization of PROTEIN2 microfilaments but not with the nuclear transport of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 This suggests that dephosphorylation of PROTEIN0 and PROTEIN1 PROTEIN2 by PROTEIN3 could be implicated in the disruption of PROTEIN4 stress fibers and in the reorganization of microfilaments induced by this hormone.\n",
      "\n",
      "PROTEIN0 PROTEIN2 This suggests that dephosphorylation of PROTEIN0 and PROTEIN1 PROTEIN2 by PROTEIN3 could be implicated in the disruption of PROTEIN4 stress fibers and in the reorganization of microfilaments induced by this hormone.\n",
      "\n",
      "PROTEIN0 PROTEIN3 This suggests that dephosphorylation of PROTEIN0 and PROTEIN1 PROTEIN2 by PROTEIN3 could be implicated in the disruption of PROTEIN4 stress fibers and in the reorganization of microfilaments induced by this hormone.\n",
      "\n",
      "PROTEIN0 PROTEIN4 This suggests that dephosphorylation of PROTEIN0 and PROTEIN1 PROTEIN2 by PROTEIN3 could be implicated in the disruption of PROTEIN4 stress fibers and in the reorganization of microfilaments induced by this hormone.\n",
      "\n",
      "PROTEIN1 PROTEIN2 This suggests that dephosphorylation of PROTEIN0 and PROTEIN1 PROTEIN2 by PROTEIN3 could be implicated in the disruption of PROTEIN4 stress fibers and in the reorganization of microfilaments induced by this hormone.\n",
      "\n",
      "PROTEIN1 PROTEIN3 This suggests that dephosphorylation of PROTEIN0 and PROTEIN1 PROTEIN2 by PROTEIN3 could be implicated in the disruption of PROTEIN4 stress fibers and in the reorganization of microfilaments induced by this hormone.\n",
      "\n",
      "PROTEIN1 PROTEIN4 This suggests that dephosphorylation of PROTEIN0 and PROTEIN1 PROTEIN2 by PROTEIN3 could be implicated in the disruption of PROTEIN4 stress fibers and in the reorganization of microfilaments induced by this hormone.\n",
      "\n",
      "PROTEIN2 PROTEIN3 This suggests that dephosphorylation of PROTEIN0 and PROTEIN1 PROTEIN2 by PROTEIN3 could be implicated in the disruption of PROTEIN4 stress fibers and in the reorganization of microfilaments induced by this hormone.\n",
      "\n",
      "PROTEIN2 PROTEIN4 This suggests that dephosphorylation of PROTEIN0 and PROTEIN1 PROTEIN2 by PROTEIN3 could be implicated in the disruption of PROTEIN4 stress fibers and in the reorganization of microfilaments induced by this hormone.\n",
      "\n",
      "PROTEIN3 PROTEIN4 This suggests that dephosphorylation of PROTEIN0 and PROTEIN1 PROTEIN2 by PROTEIN3 could be implicated in the disruption of PROTEIN4 stress fibers and in the reorganization of microfilaments induced by this hormone.\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, upon treatment with DMSO, cytoplasmic PROTEIN0 filaments were disrupted and intranuclear rod structures containing PROTEIN1 and PROTEIN2 were apparently larger and thicker in cells overexpressing PROTEIN3 than in normal cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, upon treatment with DMSO, cytoplasmic PROTEIN0 filaments were disrupted and intranuclear rod structures containing PROTEIN1 and PROTEIN2 were apparently larger and thicker in cells overexpressing PROTEIN3 than in normal cells.\n",
      "\n",
      "PROTEIN0 PROTEIN3 However, upon treatment with DMSO, cytoplasmic PROTEIN0 filaments were disrupted and intranuclear rod structures containing PROTEIN1 and PROTEIN2 were apparently larger and thicker in cells overexpressing PROTEIN3 than in normal cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, upon treatment with DMSO, cytoplasmic PROTEIN0 filaments were disrupted and intranuclear rod structures containing PROTEIN1 and PROTEIN2 were apparently larger and thicker in cells overexpressing PROTEIN3 than in normal cells.\n",
      "\n",
      "PROTEIN1 PROTEIN3 However, upon treatment with DMSO, cytoplasmic PROTEIN0 filaments were disrupted and intranuclear rod structures containing PROTEIN1 and PROTEIN2 were apparently larger and thicker in cells overexpressing PROTEIN3 than in normal cells.\n",
      "\n",
      "PROTEIN2 PROTEIN3 However, upon treatment with DMSO, cytoplasmic PROTEIN0 filaments were disrupted and intranuclear rod structures containing PROTEIN1 and PROTEIN2 were apparently larger and thicker in cells overexpressing PROTEIN3 than in normal cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In cultured cells, PROTEIN0 as well as PROTEIN1 translocates from the cytoplasm into the nucleus together with PROTEIN2 and forms rod-like structures in response to heat shock or dimethylsulfoxide (DMSO) treatment.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In cultured cells, PROTEIN0 as well as PROTEIN1 translocates from the cytoplasm into the nucleus together with PROTEIN2 and forms rod-like structures in response to heat shock or dimethylsulfoxide (DMSO) treatment.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In cultured cells, PROTEIN0 as well as PROTEIN1 translocates from the cytoplasm into the nucleus together with PROTEIN2 and forms rod-like structures in response to heat shock or dimethylsulfoxide (DMSO) treatment.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Stimulus-dependent disorganization of PROTEIN0 filaments induced by overexpression of PROTEIN1 in C2 myoblasts.\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, when all of the four proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were analysed as one group, a significant association was seen between reduction in immunoreactivity of at least one of these four proteins and the presence of metastases.\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, when all of the four proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were analysed as one group, a significant association was seen between reduction in immunoreactivity of at least one of these four proteins and the presence of metastases.\n",
      "\n",
      "PROTEIN0 PROTEIN3 However, when all of the four proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were analysed as one group, a significant association was seen between reduction in immunoreactivity of at least one of these four proteins and the presence of metastases.\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, when all of the four proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were analysed as one group, a significant association was seen between reduction in immunoreactivity of at least one of these four proteins and the presence of metastases.\n",
      "\n",
      "PROTEIN1 PROTEIN3 However, when all of the four proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were analysed as one group, a significant association was seen between reduction in immunoreactivity of at least one of these four proteins and the presence of metastases.\n",
      "\n",
      "PROTEIN2 PROTEIN3 However, when all of the four proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were analysed as one group, a significant association was seen between reduction in immunoreactivity of at least one of these four proteins and the presence of metastases.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Reduced or absent immunoreactivity in the tumour tissue was seen in 63 (70.0 per cent) for PROTEIN0 in 50 (55.6 per cent) for PROTEIN1 and in 50 (55.6 per cent) for PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Reduced or absent immunoreactivity in the tumour tissue was seen in 63 (70.0 per cent) for PROTEIN0 in 50 (55.6 per cent) for PROTEIN1 and in 50 (55.6 per cent) for PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Reduced or absent immunoreactivity in the tumour tissue was seen in 63 (70.0 per cent) for PROTEIN0 in 50 (55.6 per cent) for PROTEIN1 and in 50 (55.6 per cent) for PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Human PROTEIN0 are multifunctional, single-domain proteins which directly link the PROTEIN1 microfilament system to a variety of signalling pathways via two spatially distinct binding sites.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Human PROTEIN0 a PROTEIN1 PROTEIN2 binds selectively to mixed lipid bilayers.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Human PROTEIN0 a PROTEIN1 PROTEIN2 binds selectively to mixed lipid bilayers.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Human PROTEIN0 a PROTEIN1 PROTEIN2 binds selectively to mixed lipid bilayers.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Human platelets contain PROTEIN0 a potential regulator of PROTEIN1 polymerisability.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Hypertonicity provoked PROTEIN0 tyrosine phosphorylation in PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 and caused the dissociation of PROTEIN5 from the contacts.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immediately after synthesis, PROTEIN0 PROTEIN1 and PROTEIN2 cosedimented as complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immediately after synthesis, PROTEIN0 PROTEIN1 and PROTEIN2 cosedimented as complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immediately after synthesis, PROTEIN0 PROTEIN1 and PROTEIN2 cosedimented as complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 was not associated with these complexes after synthesis, but a subpopulation of PROTEIN1 joined the complex at a time coincident with the arrival of PROTEIN2 at the plasma membrane.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 was not associated with these complexes after synthesis, but a subpopulation of PROTEIN1 joined the complex at a time coincident with the arrival of PROTEIN2 at the plasma membrane.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 was not associated with these complexes after synthesis, but a subpopulation of PROTEIN1 joined the complex at a time coincident with the arrival of PROTEIN2 at the plasma membrane.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunocytochemical localisation of PROTEIN0 and PROTEIN1 in the generative cell of angiosperm pollen: TEM studies on high-pressure frozen and freeze-substituted Ledebouria socialis Roth (Hyacinthaceae).\n",
      "\n",
      "PROTEIN0 PROTEIN1 We attribute the detection of PROTEIN0 and PROTEIN1 to the applied cryomethods which yield a much better preservation of ultrastructure and antigenicity of delicate cytoskeletal constituents than conventional fixation techniques.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunofluorescence experiments showed that, like PROTEIN0 PROTEIN1 is associated with the membrane PROTEIN2 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunofluorescence experiments showed that, like PROTEIN0 PROTEIN1 is associated with the membrane PROTEIN2 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunofluorescence experiments showed that, like PROTEIN0 PROTEIN1 is associated with the membrane PROTEIN2 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this study, we demonstrate using viscosity, sedimentation, and PROTEIN0 assembly rate assays that yeast PROTEIN1 (16 kD) possesses all of these properties.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The NH2-terminal 16kD of PROTEIN0 a 65-kD yeast protein identified by its ability to bind to PROTEIN1 filaments, is 23% identical to yeast PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The NH2-terminal 16kD of PROTEIN0 a 65-kD yeast protein identified by its ability to bind to PROTEIN1 filaments, is 23% identical to yeast PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The NH2-terminal 16kD of PROTEIN0 a 65-kD yeast protein identified by its ability to bind to PROTEIN1 filaments, is 23% identical to yeast PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have biochemically identified the Saccharomyces cerevisiae homologue of the mammalian PROTEIN0 PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunofluorescence microscopy of monolayers between 2 and 3 h PROTEIN0 showed that PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were colocalized both at the cell periphery and in newly formed bands that are perpendicular to the cell-cell border.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunofluorescence microscopy of monolayers between 2 and 3 h PROTEIN0 showed that PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were colocalized both at the cell periphery and in newly formed bands that are perpendicular to the cell-cell border.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunofluorescence microscopy of monolayers between 2 and 3 h PROTEIN0 showed that PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were colocalized both at the cell periphery and in newly formed bands that are perpendicular to the cell-cell border.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Immunofluorescence microscopy of monolayers between 2 and 3 h PROTEIN0 showed that PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were colocalized both at the cell periphery and in newly formed bands that are perpendicular to the cell-cell border.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunofluorescence microscopy of monolayers between 2 and 3 h PROTEIN0 showed that PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were colocalized both at the cell periphery and in newly formed bands that are perpendicular to the cell-cell border.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunofluorescence microscopy of monolayers between 2 and 3 h PROTEIN0 showed that PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were colocalized both at the cell periphery and in newly formed bands that are perpendicular to the cell-cell border.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Immunofluorescence microscopy of monolayers between 2 and 3 h PROTEIN0 showed that PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were colocalized both at the cell periphery and in newly formed bands that are perpendicular to the cell-cell border.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunofluorescence microscopy of monolayers between 2 and 3 h PROTEIN0 showed that PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were colocalized both at the cell periphery and in newly formed bands that are perpendicular to the cell-cell border.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Immunofluorescence microscopy of monolayers between 2 and 3 h PROTEIN0 showed that PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were colocalized both at the cell periphery and in newly formed bands that are perpendicular to the cell-cell border.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Immunofluorescence microscopy of monolayers between 2 and 3 h PROTEIN0 showed that PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 were colocalized both at the cell periphery and in newly formed bands that are perpendicular to the cell-cell border.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunohistochemical identification and localization of PROTEIN0 and PROTEIN1 in vestibular hair cells in the normal guinea pig and in a strain of the waltzing guinea pig.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunohistochemical investigation of PROTEIN0 proteins PROTEIN1 PROTEIN2 and PROTEIN3 in rat brain following lesion: a moderate reaction, confined to the astroglia of brain tracts.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunohistochemical investigation of PROTEIN0 proteins PROTEIN1 PROTEIN2 and PROTEIN3 in rat brain following lesion: a moderate reaction, confined to the astroglia of brain tracts.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunohistochemical investigation of PROTEIN0 proteins PROTEIN1 PROTEIN2 and PROTEIN3 in rat brain following lesion: a moderate reaction, confined to the astroglia of brain tracts.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunohistochemical investigation of PROTEIN0 proteins PROTEIN1 PROTEIN2 and PROTEIN3 in rat brain following lesion: a moderate reaction, confined to the astroglia of brain tracts.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunohistochemical investigation of PROTEIN0 proteins PROTEIN1 PROTEIN2 and PROTEIN3 in rat brain following lesion: a moderate reaction, confined to the astroglia of brain tracts.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunohistochemical investigation of PROTEIN0 proteins PROTEIN1 PROTEIN2 and PROTEIN3 in rat brain following lesion: a moderate reaction, confined to the astroglia of brain tracts.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunohistochemical studies showed a strong reactivity of most LAM cells for PROTEIN0 and PROTEIN1 and a weak to moderate reactivity of a lesser number of cells for PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunohistochemical studies showed a strong reactivity of most LAM cells for PROTEIN0 and PROTEIN1 and a weak to moderate reactivity of a lesser number of cells for PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunohistochemical studies showed a strong reactivity of most LAM cells for PROTEIN0 and PROTEIN1 and a weak to moderate reactivity of a lesser number of cells for PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunohistochemical studies showed a strong reactivity of most LAM cells for PROTEIN0 and PROTEIN1 and a weak to moderate reactivity of a lesser number of cells for PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunohistochemical studies showed a strong reactivity of most LAM cells for PROTEIN0 and PROTEIN1 and a weak to moderate reactivity of a lesser number of cells for PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunohistochemical studies showed a strong reactivity of most LAM cells for PROTEIN0 and PROTEIN1 and a weak to moderate reactivity of a lesser number of cells for PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunohistochemistry revealed that arteries had three distinct populations of cells in respect to PROTEIN0 PROTEIN1 and PROTEIN2 (staining intensities '-', '+' or '++' for each protein), but only two populations in respect to PROTEIN3 ('-' and '+').\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunohistochemistry revealed that arteries had three distinct populations of cells in respect to PROTEIN0 PROTEIN1 and PROTEIN2 (staining intensities '-', '+' or '++' for each protein), but only two populations in respect to PROTEIN3 ('-' and '+').\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunohistochemistry revealed that arteries had three distinct populations of cells in respect to PROTEIN0 PROTEIN1 and PROTEIN2 (staining intensities '-', '+' or '++' for each protein), but only two populations in respect to PROTEIN3 ('-' and '+').\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunohistochemistry revealed that arteries had three distinct populations of cells in respect to PROTEIN0 PROTEIN1 and PROTEIN2 (staining intensities '-', '+' or '++' for each protein), but only two populations in respect to PROTEIN3 ('-' and '+').\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunohistochemistry revealed that arteries had three distinct populations of cells in respect to PROTEIN0 PROTEIN1 and PROTEIN2 (staining intensities '-', '+' or '++' for each protein), but only two populations in respect to PROTEIN3 ('-' and '+').\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunohistochemistry revealed that arteries had three distinct populations of cells in respect to PROTEIN0 PROTEIN1 and PROTEIN2 (staining intensities '-', '+' or '++' for each protein), but only two populations in respect to PROTEIN3 ('-' and '+').\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunolocalization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 has rarely been investigated in human cholangiocarcinoma (CC).\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunolocalization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 has rarely been investigated in human cholangiocarcinoma (CC).\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunolocalization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 has rarely been investigated in human cholangiocarcinoma (CC).\n",
      "\n",
      "PROTEIN0 PROTEIN4 Immunolocalization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 has rarely been investigated in human cholangiocarcinoma (CC).\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunolocalization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 has rarely been investigated in human cholangiocarcinoma (CC).\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunolocalization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 has rarely been investigated in human cholangiocarcinoma (CC).\n",
      "\n",
      "PROTEIN1 PROTEIN4 Immunolocalization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 has rarely been investigated in human cholangiocarcinoma (CC).\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunolocalization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 has rarely been investigated in human cholangiocarcinoma (CC).\n",
      "\n",
      "PROTEIN2 PROTEIN4 Immunolocalization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 has rarely been investigated in human cholangiocarcinoma (CC).\n",
      "\n",
      "PROTEIN3 PROTEIN4 Immunolocalization of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 has rarely been investigated in human cholangiocarcinoma (CC).\n",
      "\n",
      "PROTEIN0 PROTEIN1 In CC, membranous expression of PROTEIN0 PROTEIN1 and PROTEIN2 was the same or reduced when compared with non-cancerous bile ducts in the majority of CC.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In CC, membranous expression of PROTEIN0 PROTEIN1 and PROTEIN2 was the same or reduced when compared with non-cancerous bile ducts in the majority of CC.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In CC, membranous expression of PROTEIN0 PROTEIN1 and PROTEIN2 was the same or reduced when compared with non-cancerous bile ducts in the majority of CC.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN0 PROTEIN4 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN0 PROTEIN5 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN0 PROTEIN6 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN0 PROTEIN7 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN1 PROTEIN4 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN1 PROTEIN5 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN1 PROTEIN6 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN1 PROTEIN7 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN2 PROTEIN4 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN2 PROTEIN5 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN2 PROTEIN6 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN2 PROTEIN7 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN3 PROTEIN4 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN3 PROTEIN5 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN3 PROTEIN6 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN3 PROTEIN7 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN4 PROTEIN5 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN4 PROTEIN6 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN4 PROTEIN7 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN5 PROTEIN6 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN5 PROTEIN7 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN6 PROTEIN7 In normal livers, PROTEIN0 PROTEIN1 and PROTEIN2 but not PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 were expressed at the cell membrane of normal intrahepatic bile ducts.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN0 PROTEIN5 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN0 PROTEIN6 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN0 PROTEIN7 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN1 PROTEIN5 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN1 PROTEIN6 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN1 PROTEIN7 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN2 PROTEIN5 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN2 PROTEIN6 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN2 PROTEIN7 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN3 PROTEIN5 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN3 PROTEIN6 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN3 PROTEIN7 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN4 PROTEIN5 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN4 PROTEIN6 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN4 PROTEIN7 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN5 PROTEIN6 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN5 PROTEIN7 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN6 PROTEIN7 We found that the down-regulation of PROTEIN0 PROTEIN1 and PROTEIN2 expression significantly correlated with tumor high grade, but not with vascular invasion, metastasis, PROTEIN3 expression, PROTEIN4 labeling, or PROTEIN5 expression, except for PROTEIN6 the down-regulation of which was associated with PROTEIN7 down-regulation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoprecipitation of PROTEIN0 was increased in response to both doses of CdCl2, while the immunoprecipitation of PROTEIN1 was little changed by either cadmium dose.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 Immunoprecipitation of metabolically labeled proteins with PROTEIN0 revealed three bands corresponding to PROTEIN1 PROTEIN2 and PROTEIN3 and a 79-kDa band which was apparently smaller than that of normal PROTEIN4 indicating truncated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN4 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN5 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN6 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN7 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN4 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN5 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN6 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN1 PROTEIN7 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN4 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN5 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN6 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN2 PROTEIN7 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN4 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN5 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN6 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN3 PROTEIN7 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN4 PROTEIN5 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN4 PROTEIN6 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN4 PROTEIN7 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN5 PROTEIN6 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN5 PROTEIN7 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN6 PROTEIN7 Immunoprecipitations showed in both cell lines the presence of two different PROTEIN0 PROTEIN1 complexes, one composed of PROTEIN2 PROTEIN3 and PROTEIN4 and the other of PROTEIN5 PROTEIN6 and PROTEIN7\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 Immunoprecipitation with an PROTEIN0 mAb, which is known to co-precipitate the PROTEIN1 demonstrated that the three poorly differentiated cell lines expressing PROTEIN2 did not co-precipitate PROTEIN3 although all of the moderately-well differentiated cell lines expressed both PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoreactive PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins in hepatocellular carcinoma: relationships with tumor grade, clinicopathologic parameters, and patients' survival.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunoreactive PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins in hepatocellular carcinoma: relationships with tumor grade, clinicopathologic parameters, and patients' survival.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunoreactive PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins in hepatocellular carcinoma: relationships with tumor grade, clinicopathologic parameters, and patients' survival.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunoreactive PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins in hepatocellular carcinoma: relationships with tumor grade, clinicopathologic parameters, and patients' survival.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunoreactive PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins in hepatocellular carcinoma: relationships with tumor grade, clinicopathologic parameters, and patients' survival.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunoreactive PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 proteins in hepatocellular carcinoma: relationships with tumor grade, clinicopathologic parameters, and patients' survival.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunoreactivity for PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was observed in all 15 specimens.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immunoreactivity for PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was observed in all 15 specimens.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immunoreactivity for PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was observed in all 15 specimens.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Immunoreactivity for PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was observed in all 15 specimens.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immunoreactivity for PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was observed in all 15 specimens.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immunoreactivity for PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was observed in all 15 specimens.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Immunoreactivity for PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was observed in all 15 specimens.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immunoreactivity for PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was observed in all 15 specimens.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Immunoreactivity for PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was observed in all 15 specimens.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Immunoreactivity for PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was observed in all 15 specimens.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Impact of PROTEIN0 on PROTEIN1 nucleotide exchange and PROTEIN2 polymerization dynamics.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Impact of PROTEIN0 on PROTEIN1 nucleotide exchange and PROTEIN2 polymerization dynamics.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Impact of PROTEIN0 on PROTEIN1 nucleotide exchange and PROTEIN2 polymerization dynamics.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In 50 mM KCl, high concentrations of Acanthamoeba PROTEIN0 inhibit the elongation rate of PROTEIN1 filaments measured directly by electron microscopy, but the effect is minimal in KCl with 2 MgCl2.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In a blot overlay experiment, purified 125I-labeled recombinant PROTEIN0 bound not only to plant PROTEIN1 but also to mammalian PROTEIN2 demonstrating that cytoskeletal components from distantly related organisms with divergent primary structures can be compatible.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In a blot overlay experiment, purified 125I-labeled recombinant PROTEIN0 bound not only to plant PROTEIN1 but also to mammalian PROTEIN2 demonstrating that cytoskeletal components from distantly related organisms with divergent primary structures can be compatible.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In a blot overlay experiment, purified 125I-labeled recombinant PROTEIN0 bound not only to plant PROTEIN1 but also to mammalian PROTEIN2 demonstrating that cytoskeletal components from distantly related organisms with divergent primary structures can be compatible.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In Acanthamoeba PROTEIN0 polymerization is regulated, at least in part, by PROTEIN1 which binds to PROTEIN2 monomers, and by PROTEIN3 which both nucleates polymerization and blocks monomer addition at the 'barbed' end of the filament.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In Acanthamoeba PROTEIN0 polymerization is regulated, at least in part, by PROTEIN1 which binds to PROTEIN2 monomers, and by PROTEIN3 which both nucleates polymerization and blocks monomer addition at the 'barbed' end of the filament.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In Acanthamoeba PROTEIN0 polymerization is regulated, at least in part, by PROTEIN1 which binds to PROTEIN2 monomers, and by PROTEIN3 which both nucleates polymerization and blocks monomer addition at the 'barbed' end of the filament.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In Acanthamoeba PROTEIN0 polymerization is regulated, at least in part, by PROTEIN1 which binds to PROTEIN2 monomers, and by PROTEIN3 which both nucleates polymerization and blocks monomer addition at the 'barbed' end of the filament.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In Acanthamoeba PROTEIN0 polymerization is regulated, at least in part, by PROTEIN1 which binds to PROTEIN2 monomers, and by PROTEIN3 which both nucleates polymerization and blocks monomer addition at the 'barbed' end of the filament.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In Acanthamoeba PROTEIN0 polymerization is regulated, at least in part, by PROTEIN1 which binds to PROTEIN2 monomers, and by PROTEIN3 which both nucleates polymerization and blocks monomer addition at the 'barbed' end of the filament.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In Acanthamoeba, the two isoforms of PROTEIN0 may have specialized functions on the basis of their identical (approximately 10 microM) affinities for PROTEIN1 monomers and different affinities for PIP2.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Inactivation of the PROTEIN0 pathway in non-small cell lung carcinoma (NSCLC) occurs mostly through inactivation of the PROTEIN1 PROTEIN2 PROTEIN3 and/or up-regulation of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Inactivation of the PROTEIN0 pathway in non-small cell lung carcinoma (NSCLC) occurs mostly through inactivation of the PROTEIN1 PROTEIN2 PROTEIN3 and/or up-regulation of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Inactivation of the PROTEIN0 pathway in non-small cell lung carcinoma (NSCLC) occurs mostly through inactivation of the PROTEIN1 PROTEIN2 PROTEIN3 and/or up-regulation of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Inactivation of the PROTEIN0 pathway in non-small cell lung carcinoma (NSCLC) occurs mostly through inactivation of the PROTEIN1 PROTEIN2 PROTEIN3 and/or up-regulation of PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Inactivation of the PROTEIN0 pathway in non-small cell lung carcinoma (NSCLC) occurs mostly through inactivation of the PROTEIN1 PROTEIN2 PROTEIN3 and/or up-regulation of PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Inactivation of the PROTEIN0 pathway in non-small cell lung carcinoma (NSCLC) occurs mostly through inactivation of the PROTEIN1 PROTEIN2 PROTEIN3 and/or up-regulation of PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Inactivation of the PROTEIN0 pathway in non-small cell lung carcinoma (NSCLC) occurs mostly through inactivation of the PROTEIN1 PROTEIN2 PROTEIN3 and/or up-regulation of PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Inactivation of the PROTEIN0 pathway in non-small cell lung carcinoma (NSCLC) occurs mostly through inactivation of the PROTEIN1 PROTEIN2 PROTEIN3 and/or up-regulation of PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Inactivation of the PROTEIN0 pathway in non-small cell lung carcinoma (NSCLC) occurs mostly through inactivation of the PROTEIN1 PROTEIN2 PROTEIN3 and/or up-regulation of PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Inactivation of the PROTEIN0 pathway in non-small cell lung carcinoma (NSCLC) occurs mostly through inactivation of the PROTEIN1 PROTEIN2 PROTEIN3 and/or up-regulation of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition to their role in mismatch repair (MMR), the PROTEIN0 and PROTEIN1 gene products are required to remove 3' nonhomologous DNA tails during genetic recombination.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition to the PROTEIN0 increase in the general protein synthesis, stimulation with PROTEIN1 led to a 2.4-fold increase in net PROTEIN2 protein synthesis and a 3.3-fold increase in net PROTEIN3 synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In addition to the PROTEIN0 increase in the general protein synthesis, stimulation with PROTEIN1 led to a 2.4-fold increase in net PROTEIN2 protein synthesis and a 3.3-fold increase in net PROTEIN3 synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In addition to the PROTEIN0 increase in the general protein synthesis, stimulation with PROTEIN1 led to a 2.4-fold increase in net PROTEIN2 protein synthesis and a 3.3-fold increase in net PROTEIN3 synthesis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In addition to the PROTEIN0 increase in the general protein synthesis, stimulation with PROTEIN1 led to a 2.4-fold increase in net PROTEIN2 protein synthesis and a 3.3-fold increase in net PROTEIN3 synthesis.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In addition to the PROTEIN0 increase in the general protein synthesis, stimulation with PROTEIN1 led to a 2.4-fold increase in net PROTEIN2 protein synthesis and a 3.3-fold increase in net PROTEIN3 synthesis.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In addition to the PROTEIN0 increase in the general protein synthesis, stimulation with PROTEIN1 led to a 2.4-fold increase in net PROTEIN2 protein synthesis and a 3.3-fold increase in net PROTEIN3 synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 In addition, we also provide new evidence to suggest that PROTEIN0 stimulation of the PROTEIN1 promoter involves increased physical interactions with acetylated PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, we demonstrate that PROTEIN0 and its PROTEIN1 binding peptide compete with PROTEIN2 segments 2-3 for binding to PROTEIN3 filaments.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In addition, we demonstrate that PROTEIN0 and its PROTEIN1 binding peptide compete with PROTEIN2 segments 2-3 for binding to PROTEIN3 filaments.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In addition, we demonstrate that PROTEIN0 and its PROTEIN1 binding peptide compete with PROTEIN2 segments 2-3 for binding to PROTEIN3 filaments.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In addition, we demonstrate that PROTEIN0 and its PROTEIN1 binding peptide compete with PROTEIN2 segments 2-3 for binding to PROTEIN3 filaments.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In addition, we demonstrate that PROTEIN0 and its PROTEIN1 binding peptide compete with PROTEIN2 segments 2-3 for binding to PROTEIN3 filaments.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In addition, we demonstrate that PROTEIN0 and its PROTEIN1 binding peptide compete with PROTEIN2 segments 2-3 for binding to PROTEIN3 filaments.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, we have demonstrated that increased gene dosage of either PROTEIN0 or PROTEIN1 two other factors required for silencing, suppresses the silencing defect of the PROTEIN2 mutants.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In addition, we have demonstrated that increased gene dosage of either PROTEIN0 or PROTEIN1 two other factors required for silencing, suppresses the silencing defect of the PROTEIN2 mutants.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In addition, we have demonstrated that increased gene dosage of either PROTEIN0 or PROTEIN1 two other factors required for silencing, suppresses the silencing defect of the PROTEIN2 mutants.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN2 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN3 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN4 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN5 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN6 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN7 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN8 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN2 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN3 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN4 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN5 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN6 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN7 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN8 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN3 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN4 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN5 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN6 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN7 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN8 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN4 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN5 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN6 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN7 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN8 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN5 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN6 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN7 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN8 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN5 PROTEIN6 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN5 PROTEIN7 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN5 PROTEIN8 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN6 PROTEIN7 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN6 PROTEIN8 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN7 PROTEIN8 In addition, we performed immunohistochemistry on sectioned PROTEIN0 and PROTEIN1 mutant embryos with antibodies reactive with PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, we report that radiation induced transient activation of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 activation and the transcription factor, PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 In addition, we report that radiation induced transient activation of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 activation and the transcription factor, PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 In addition, we report that radiation induced transient activation of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 activation and the transcription factor, PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 In addition, we report that radiation induced transient activation of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 activation and the transcription factor, PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 In addition, we report that radiation induced transient activation of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 activation and the transcription factor, PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 In addition, we report that radiation induced transient activation of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 activation and the transcription factor, PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 In addition, we report that radiation induced transient activation of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 activation and the transcription factor, PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 In addition, we report that radiation induced transient activation of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 activation and the transcription factor, PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 In addition, we report that radiation induced transient activation of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 activation and the transcription factor, PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 In addition, we report that radiation induced transient activation of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 activation and the transcription factor, PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 In a model system, the interacting proteins recruiting PROTEIN0 to DNA were PROTEIN1 and PROTEIN2 or PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 In a model system, the interacting proteins recruiting PROTEIN0 to DNA were PROTEIN1 and PROTEIN2 or PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 In a model system, the interacting proteins recruiting PROTEIN0 to DNA were PROTEIN1 and PROTEIN2 or PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 In a model system, the interacting proteins recruiting PROTEIN0 to DNA were PROTEIN1 and PROTEIN2 or PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 In a model system, the interacting proteins recruiting PROTEIN0 to DNA were PROTEIN1 and PROTEIN2 or PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 In a model system, the interacting proteins recruiting PROTEIN0 to DNA were PROTEIN1 and PROTEIN2 or PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 In a model system, the interacting proteins recruiting PROTEIN0 to DNA were PROTEIN1 and PROTEIN2 or PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 In a model system, the interacting proteins recruiting PROTEIN0 to DNA were PROTEIN1 and PROTEIN2 or PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 In a model system, the interacting proteins recruiting PROTEIN0 to DNA were PROTEIN1 and PROTEIN2 or PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 In a model system, the interacting proteins recruiting PROTEIN0 to DNA were PROTEIN1 and PROTEIN2 or PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 In brush cells, PROTEIN0 and PROTEIN1 are not only present in the PROTEIN2 filament core bundles of apical microvilli and their long rootlets but, in addition, both proteins are also associated with microvilli extending from the basolateral cell surface of the brush cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In brush cells, PROTEIN0 and PROTEIN1 are not only present in the PROTEIN2 filament core bundles of apical microvilli and their long rootlets but, in addition, both proteins are also associated with microvilli extending from the basolateral cell surface of the brush cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In brush cells, PROTEIN0 and PROTEIN1 are not only present in the PROTEIN2 filament core bundles of apical microvilli and their long rootlets but, in addition, both proteins are also associated with microvilli extending from the basolateral cell surface of the brush cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the present study we show that brush cells can be identified in tissue sections even at the light microscopic level by immunostaining with antibodies against PROTEIN0 and PROTEIN1 two proteins that crosslink PROTEIN2 filaments to form bundles.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the present study we show that brush cells can be identified in tissue sections even at the light microscopic level by immunostaining with antibodies against PROTEIN0 and PROTEIN1 two proteins that crosslink PROTEIN2 filaments to form bundles.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the present study we show that brush cells can be identified in tissue sections even at the light microscopic level by immunostaining with antibodies against PROTEIN0 and PROTEIN1 two proteins that crosslink PROTEIN2 filaments to form bundles.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In cancerous tissues, the frequency of reduced expression of PROTEIN0 (28 of 86, 33%) was similar to that of PROTEIN1 (19 of 86, 22%), but less than that of PROTEIN2 (47 of 86, 55%).\n",
      "\n",
      "PROTEIN0 PROTEIN2 In cancerous tissues, the frequency of reduced expression of PROTEIN0 (28 of 86, 33%) was similar to that of PROTEIN1 (19 of 86, 22%), but less than that of PROTEIN2 (47 of 86, 55%).\n",
      "\n",
      "PROTEIN1 PROTEIN2 In cancerous tissues, the frequency of reduced expression of PROTEIN0 (28 of 86, 33%) was similar to that of PROTEIN1 (19 of 86, 22%), but less than that of PROTEIN2 (47 of 86, 55%).\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this study, we immunohistochemically investigated the expression of PROTEIN0 as well as PROTEIN1 and PROTEIN2 in 86 human colorectal cancers, and we analysed their coexpression pattern and relationship to clinicopathological factors.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this study, we immunohistochemically investigated the expression of PROTEIN0 as well as PROTEIN1 and PROTEIN2 in 86 human colorectal cancers, and we analysed their coexpression pattern and relationship to clinicopathological factors.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this study, we immunohistochemically investigated the expression of PROTEIN0 as well as PROTEIN1 and PROTEIN2 in 86 human colorectal cancers, and we analysed their coexpression pattern and relationship to clinicopathological factors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In cells lacking the PROTEIN0 the bundles, which were induced by an over-expression of PROTEIN1 shortened and became straight following hyperosmotic stress, forming a polygonal structure.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN4 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN5 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN4 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN5 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN2 PROTEIN4 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN2 PROTEIN5 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN3 PROTEIN4 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN3 PROTEIN5 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN4 PROTEIN5 In contrast, PROTEIN0 partially suppresses the temperature sensitivity of a PROTEIN1 mutant, and the loss of cytoplasmic cables seen in PROTEIN2 PROTEIN3 and PROTEIN4 null mutants, suggesting filament stabilizing functions for these PROTEIN5 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, PROTEIN0 another putative PROTEIN1 linker protein, could not be detected in significant amounts in human neutrophils using a polyclonal antibody raised against chicken PROTEIN2 which reacted with human PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 In contrast, PROTEIN0 another putative PROTEIN1 linker protein, could not be detected in significant amounts in human neutrophils using a polyclonal antibody raised against chicken PROTEIN2 which reacted with human PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 In contrast, PROTEIN0 another putative PROTEIN1 linker protein, could not be detected in significant amounts in human neutrophils using a polyclonal antibody raised against chicken PROTEIN2 which reacted with human PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 In contrast, PROTEIN0 another putative PROTEIN1 linker protein, could not be detected in significant amounts in human neutrophils using a polyclonal antibody raised against chicken PROTEIN2 which reacted with human PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 In contrast, PROTEIN0 another putative PROTEIN1 linker protein, could not be detected in significant amounts in human neutrophils using a polyclonal antibody raised against chicken PROTEIN2 which reacted with human PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 In contrast, PROTEIN0 another putative PROTEIN1 linker protein, could not be detected in significant amounts in human neutrophils using a polyclonal antibody raised against chicken PROTEIN2 which reacted with human PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 In contrast, PROTEIN0 another putative PROTEIN1 linker protein, could not be detected in significant amounts in human neutrophils using a polyclonal antibody raised against chicken PROTEIN2 which reacted with human PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 In contrast, PROTEIN0 another putative PROTEIN1 linker protein, could not be detected in significant amounts in human neutrophils using a polyclonal antibody raised against chicken PROTEIN2 which reacted with human PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 In contrast, PROTEIN0 another putative PROTEIN1 linker protein, could not be detected in significant amounts in human neutrophils using a polyclonal antibody raised against chicken PROTEIN2 which reacted with human PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 In contrast, PROTEIN0 another putative PROTEIN1 linker protein, could not be detected in significant amounts in human neutrophils using a polyclonal antibody raised against chicken PROTEIN2 which reacted with human PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, the PROTEIN0 and PROTEIN1 genes are essential for homothallic mating-type switching.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast to the model of PROTEIN0 binding proposed for PROTEIN1 the PROTEIN2 PROTEIN3 domain appears to associate with PROTEIN4 in an extended conformation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In contrast to the model of PROTEIN0 binding proposed for PROTEIN1 the PROTEIN2 PROTEIN3 domain appears to associate with PROTEIN4 in an extended conformation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In contrast to the model of PROTEIN0 binding proposed for PROTEIN1 the PROTEIN2 PROTEIN3 domain appears to associate with PROTEIN4 in an extended conformation.\n",
      "\n",
      "PROTEIN0 PROTEIN4 In contrast to the model of PROTEIN0 binding proposed for PROTEIN1 the PROTEIN2 PROTEIN3 domain appears to associate with PROTEIN4 in an extended conformation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In contrast to the model of PROTEIN0 binding proposed for PROTEIN1 the PROTEIN2 PROTEIN3 domain appears to associate with PROTEIN4 in an extended conformation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In contrast to the model of PROTEIN0 binding proposed for PROTEIN1 the PROTEIN2 PROTEIN3 domain appears to associate with PROTEIN4 in an extended conformation.\n",
      "\n",
      "PROTEIN1 PROTEIN4 In contrast to the model of PROTEIN0 binding proposed for PROTEIN1 the PROTEIN2 PROTEIN3 domain appears to associate with PROTEIN4 in an extended conformation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In contrast to the model of PROTEIN0 binding proposed for PROTEIN1 the PROTEIN2 PROTEIN3 domain appears to associate with PROTEIN4 in an extended conformation.\n",
      "\n",
      "PROTEIN2 PROTEIN4 In contrast to the model of PROTEIN0 binding proposed for PROTEIN1 the PROTEIN2 PROTEIN3 domain appears to associate with PROTEIN4 in an extended conformation.\n",
      "\n",
      "PROTEIN3 PROTEIN4 In contrast to the model of PROTEIN0 binding proposed for PROTEIN1 the PROTEIN2 PROTEIN3 domain appears to associate with PROTEIN4 in an extended conformation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Increased interaction of PROTEIN0 with the PROTEIN1 cytoskeleton is consistent with and might be mediated by its enhanced binding to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Increased interaction of PROTEIN0 with the PROTEIN1 cytoskeleton is consistent with and might be mediated by its enhanced binding to PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Increased interaction of PROTEIN0 with the PROTEIN1 cytoskeleton is consistent with and might be mediated by its enhanced binding to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Increasing the ionic strength within a relatively narrow range significantly decreased ability of PROTEIN0 to bind to PROTEIN1 regardless of pH.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interaction of PROTEIN0 with PROTEIN1 sensitive modulation of filament crosslinking activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In enriched cell lines and in a pure myocyte cell strain, PROTEIN0 inhibited increases in immunoreactive PROTEIN1 and PROTEIN2 PROTEIN3 normally seen after serum withdrawal.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In enriched cell lines and in a pure myocyte cell strain, PROTEIN0 inhibited increases in immunoreactive PROTEIN1 and PROTEIN2 PROTEIN3 normally seen after serum withdrawal.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In enriched cell lines and in a pure myocyte cell strain, PROTEIN0 inhibited increases in immunoreactive PROTEIN1 and PROTEIN2 PROTEIN3 normally seen after serum withdrawal.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In enriched cell lines and in a pure myocyte cell strain, PROTEIN0 inhibited increases in immunoreactive PROTEIN1 and PROTEIN2 PROTEIN3 normally seen after serum withdrawal.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In enriched cell lines and in a pure myocyte cell strain, PROTEIN0 inhibited increases in immunoreactive PROTEIN1 and PROTEIN2 PROTEIN3 normally seen after serum withdrawal.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In enriched cell lines and in a pure myocyte cell strain, PROTEIN0 inhibited increases in immunoreactive PROTEIN1 and PROTEIN2 PROTEIN3 normally seen after serum withdrawal.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In extracts from mouse brain, PROTEIN0 and PROTEIN1 can form complexes with regulators of endocytosis, synaptic vesicle recycling and PROTEIN2 assembly.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In extracts from mouse brain, PROTEIN0 and PROTEIN1 can form complexes with regulators of endocytosis, synaptic vesicle recycling and PROTEIN2 assembly.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In extracts from mouse brain, PROTEIN0 and PROTEIN1 can form complexes with regulators of endocytosis, synaptic vesicle recycling and PROTEIN2 assembly.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In mouse brain PROTEIN0 and PROTEIN1 associate with regulators of the endocytic pathway and PROTEIN2 assembly.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In mouse brain PROTEIN0 and PROTEIN1 associate with regulators of the endocytic pathway and PROTEIN2 assembly.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In mouse brain PROTEIN0 and PROTEIN1 associate with regulators of the endocytic pathway and PROTEIN2 assembly.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our findings strongly suggest that in brain PROTEIN0 and PROTEIN1 complexes link the PROTEIN2 cytoskeleton and endocytic membrane flow, directing PROTEIN3 and PROTEIN4 assembly to distinct membrane domains.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Our findings strongly suggest that in brain PROTEIN0 and PROTEIN1 complexes link the PROTEIN2 cytoskeleton and endocytic membrane flow, directing PROTEIN3 and PROTEIN4 assembly to distinct membrane domains.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Our findings strongly suggest that in brain PROTEIN0 and PROTEIN1 complexes link the PROTEIN2 cytoskeleton and endocytic membrane flow, directing PROTEIN3 and PROTEIN4 assembly to distinct membrane domains.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Our findings strongly suggest that in brain PROTEIN0 and PROTEIN1 complexes link the PROTEIN2 cytoskeleton and endocytic membrane flow, directing PROTEIN3 and PROTEIN4 assembly to distinct membrane domains.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Our findings strongly suggest that in brain PROTEIN0 and PROTEIN1 complexes link the PROTEIN2 cytoskeleton and endocytic membrane flow, directing PROTEIN3 and PROTEIN4 assembly to distinct membrane domains.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Our findings strongly suggest that in brain PROTEIN0 and PROTEIN1 complexes link the PROTEIN2 cytoskeleton and endocytic membrane flow, directing PROTEIN3 and PROTEIN4 assembly to distinct membrane domains.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Our findings strongly suggest that in brain PROTEIN0 and PROTEIN1 complexes link the PROTEIN2 cytoskeleton and endocytic membrane flow, directing PROTEIN3 and PROTEIN4 assembly to distinct membrane domains.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Our findings strongly suggest that in brain PROTEIN0 and PROTEIN1 complexes link the PROTEIN2 cytoskeleton and endocytic membrane flow, directing PROTEIN3 and PROTEIN4 assembly to distinct membrane domains.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Our findings strongly suggest that in brain PROTEIN0 and PROTEIN1 complexes link the PROTEIN2 cytoskeleton and endocytic membrane flow, directing PROTEIN3 and PROTEIN4 assembly to distinct membrane domains.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Our findings strongly suggest that in brain PROTEIN0 and PROTEIN1 complexes link the PROTEIN2 cytoskeleton and endocytic membrane flow, directing PROTEIN3 and PROTEIN4 assembly to distinct membrane domains.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In flattened cells under starvation stress, PROTEIN0 localized at dramatically reorganizing PROTEIN1 in ruffling membranes of the leading edge, but not at rigid PROTEIN2 meshwork in focal adhesion plaques.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In flattened cells under starvation stress, PROTEIN0 localized at dramatically reorganizing PROTEIN1 in ruffling membranes of the leading edge, but not at rigid PROTEIN2 meshwork in focal adhesion plaques.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In flattened cells under starvation stress, PROTEIN0 localized at dramatically reorganizing PROTEIN1 in ruffling membranes of the leading edge, but not at rigid PROTEIN2 meshwork in focal adhesion plaques.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 may be involved in dynamic reorganization of membranous PROTEIN1 cytoskeletons.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In growth cones, PROTEIN0 was most concentrated in the margin bordering the thickened, organelle-rich central region and the thin, PROTEIN1 peripheral region.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 staining was dimmer and its colocalization with filamentous PROTEIN1 bundles in growth cones was less striking than that of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 staining was dimmer and its colocalization with filamentous PROTEIN1 bundles in growth cones was less striking than that of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 staining was dimmer and its colocalization with filamentous PROTEIN1 bundles in growth cones was less striking than that of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Inhibition of 12P-CRE binding was also observed in the presence of an antibody to PROTEIN0 (265-kDa PROTEIN1 an integrator and coactivator of cAMP-responsive genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Inhibition of PROTEIN0 polymerization by a synthetic dodecapeptide patterned on the sequence around the PROTEIN1 site of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Inhibition of PROTEIN0 polymerization by a synthetic dodecapeptide patterned on the sequence around the PROTEIN1 site of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Inhibition of PROTEIN0 polymerization by a synthetic dodecapeptide patterned on the sequence around the PROTEIN1 site of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Inhibition of PROTEIN0 polymerization in KCl was stronger than that of the parental PROTEIN1 and the Kd value of its interaction with rabbit PROTEIN2 as determined by falling ball viscometry, was smaller (mean value 0.5 x 10(-6) M, as compared to 1.9 x 10(-6) M for bovine PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Inhibition of PROTEIN0 polymerization in KCl was stronger than that of the parental PROTEIN1 and the Kd value of its interaction with rabbit PROTEIN2 as determined by falling ball viscometry, was smaller (mean value 0.5 x 10(-6) M, as compared to 1.9 x 10(-6) M for bovine PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Inhibition of PROTEIN0 polymerization in KCl was stronger than that of the parental PROTEIN1 and the Kd value of its interaction with rabbit PROTEIN2 as determined by falling ball viscometry, was smaller (mean value 0.5 x 10(-6) M, as compared to 1.9 x 10(-6) M for bovine PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Inhibition of PROTEIN0 polymerization in KCl was stronger than that of the parental PROTEIN1 and the Kd value of its interaction with rabbit PROTEIN2 as determined by falling ball viscometry, was smaller (mean value 0.5 x 10(-6) M, as compared to 1.9 x 10(-6) M for bovine PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Inhibition of PROTEIN0 polymerization in KCl was stronger than that of the parental PROTEIN1 and the Kd value of its interaction with rabbit PROTEIN2 as determined by falling ball viscometry, was smaller (mean value 0.5 x 10(-6) M, as compared to 1.9 x 10(-6) M for bovine PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Inhibition of PROTEIN0 polymerization in KCl was stronger than that of the parental PROTEIN1 and the Kd value of its interaction with rabbit PROTEIN2 as determined by falling ball viscometry, was smaller (mean value 0.5 x 10(-6) M, as compared to 1.9 x 10(-6) M for bovine PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Inhibition of PROTEIN0 subfragment 1 PROTEIN1 activity by analog peptides of the PROTEIN2 site around the Cys(SH1) of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Inhibition of PROTEIN0 subfragment 1 PROTEIN1 activity by analog peptides of the PROTEIN2 site around the Cys(SH1) of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Inhibition of PROTEIN0 subfragment 1 PROTEIN1 activity by analog peptides of the PROTEIN2 site around the Cys(SH1) of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Inhibition of PROTEIN0 subfragment 1 PROTEIN1 activity by analog peptides of the PROTEIN2 site around the Cys(SH1) of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Inhibition of PROTEIN0 subfragment 1 PROTEIN1 activity by analog peptides of the PROTEIN2 site around the Cys(SH1) of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Inhibition of PROTEIN0 subfragment 1 PROTEIN1 activity by analog peptides of the PROTEIN2 site around the Cys(SH1) of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Inhibition of the interactions of PROTEIN0 PROTEIN1 and PROTEIN2 with PROTEIN3 by phosphoinositides.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Inhibition of the interactions of PROTEIN0 PROTEIN1 and PROTEIN2 with PROTEIN3 by phosphoinositides.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Inhibition of the interactions of PROTEIN0 PROTEIN1 and PROTEIN2 with PROTEIN3 by phosphoinositides.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Inhibition of the interactions of PROTEIN0 PROTEIN1 and PROTEIN2 with PROTEIN3 by phosphoinositides.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Inhibition of the interactions of PROTEIN0 PROTEIN1 and PROTEIN2 with PROTEIN3 by phosphoinositides.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Inhibition of the interactions of PROTEIN0 PROTEIN1 and PROTEIN2 with PROTEIN3 by phosphoinositides.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In higher plants, a large number of isoforms for the PROTEIN0 monomer-binding protein PROTEIN1 have been identified, whereas other organisms express only few PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In higher plants, a large number of isoforms for the PROTEIN0 monomer-binding protein PROTEIN1 have been identified, whereas other organisms express only few PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In higher plants, a large number of isoforms for the PROTEIN0 monomer-binding protein PROTEIN1 have been identified, whereas other organisms express only few PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 In isolated hearts, PD 81,723 alone produced only a small stimulus to His bundle (S-H) interval prolongation of 1.5 to 4 msec, which was completely reversed by the PROTEIN0 antagonist 8-cyclopentyltheophylline and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In localization studies with mammalian cells, all fusion proteins showed the localization expected for PROTEIN0 in areas of high PROTEIN1 dynamics, such as leading lamellae and ruffles induced by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In localization studies with mammalian cells, all fusion proteins showed the localization expected for PROTEIN0 in areas of high PROTEIN1 dynamics, such as leading lamellae and ruffles induced by PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In localization studies with mammalian cells, all fusion proteins showed the localization expected for PROTEIN0 in areas of high PROTEIN1 dynamics, such as leading lamellae and ruffles induced by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 In order to clarify PROTEIN0 regulation of PROTEIN1 assembly in muscle cells, PROTEIN2 tagged with fluorescence dyes was introduced into C2 myoblasts by a micro injection method.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In order to clarify PROTEIN0 regulation of PROTEIN1 assembly in muscle cells, PROTEIN2 tagged with fluorescence dyes was introduced into C2 myoblasts by a micro injection method.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In order to clarify PROTEIN0 regulation of PROTEIN1 assembly in muscle cells, PROTEIN2 tagged with fluorescence dyes was introduced into C2 myoblasts by a micro injection method.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The exogeneous PROTEIN0 but not PROTEIN1 caused quick disassembly of PROTEIN2 filaments and accumulated in furrow region of dividing cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The exogeneous PROTEIN0 but not PROTEIN1 caused quick disassembly of PROTEIN2 filaments and accumulated in furrow region of dividing cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The exogeneous PROTEIN0 but not PROTEIN1 caused quick disassembly of PROTEIN2 filaments and accumulated in furrow region of dividing cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that the subcellular distribution of PROTEIN0 as well as its interaction with PROTEIN1 in vivo is regulated by its phosphorylation and dephosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In order to help explain the apparent beneficial effects of PROTEIN0 over-expression, a number of closely associated cell cycle proteins have also been evaluated, including the PROTEIN1 PROTEIN2 which blocks the activating effects of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 In order to help explain the apparent beneficial effects of PROTEIN0 over-expression, a number of closely associated cell cycle proteins have also been evaluated, including the PROTEIN1 PROTEIN2 which blocks the activating effects of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 In order to help explain the apparent beneficial effects of PROTEIN0 over-expression, a number of closely associated cell cycle proteins have also been evaluated, including the PROTEIN1 PROTEIN2 which blocks the activating effects of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 In order to help explain the apparent beneficial effects of PROTEIN0 over-expression, a number of closely associated cell cycle proteins have also been evaluated, including the PROTEIN1 PROTEIN2 which blocks the activating effects of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 In order to help explain the apparent beneficial effects of PROTEIN0 over-expression, a number of closely associated cell cycle proteins have also been evaluated, including the PROTEIN1 PROTEIN2 which blocks the activating effects of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 In order to help explain the apparent beneficial effects of PROTEIN0 over-expression, a number of closely associated cell cycle proteins have also been evaluated, including the PROTEIN1 PROTEIN2 which blocks the activating effects of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 In reconstitution experiments, PROTEIN0 filaments incubated in EGTA with purified PROTEIN1 and PROTEIN2 form smooth-sided bundles containing an apparently random number of filaments.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In reconstitution experiments, PROTEIN0 filaments incubated in EGTA with purified PROTEIN1 and PROTEIN2 form smooth-sided bundles containing an apparently random number of filaments.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In reconstitution experiments, PROTEIN0 filaments incubated in EGTA with purified PROTEIN1 and PROTEIN2 form smooth-sided bundles containing an apparently random number of filaments.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In resting leukocytes, PROTEIN0 are constitutively linked to the PROTEIN1 cytoskeleton via the protein PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In resting leukocytes, PROTEIN0 are constitutively linked to the PROTEIN1 cytoskeleton via the protein PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In resting leukocytes, PROTEIN0 are constitutively linked to the PROTEIN1 cytoskeleton via the protein PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 In search for genes which might participate in chicken PROTEIN0 gene conversion, we have identified chicken counterparts of the yeast PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In search for genes which might participate in chicken PROTEIN0 gene conversion, we have identified chicken counterparts of the yeast PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In search for genes which might participate in chicken PROTEIN0 gene conversion, we have identified chicken counterparts of the yeast PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In search for genes which might participate in chicken PROTEIN0 gene conversion, we have identified chicken counterparts of the yeast PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In search for genes which might participate in chicken PROTEIN0 gene conversion, we have identified chicken counterparts of the yeast PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In search for genes which might participate in chicken PROTEIN0 gene conversion, we have identified chicken counterparts of the yeast PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 and PROTEIN2 messenger RNAs in skeletal muscle after an injection of growth hormone in subjects over 60 years old.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 messenger RNAs in skeletal muscle after an injection of growth hormone in subjects over 60 years old.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 messenger RNAs in skeletal muscle after an injection of growth hormone in subjects over 60 years old.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 stimulates PROTEIN1 and PROTEIN2 synthesis in the awake rat.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 stimulates PROTEIN1 and PROTEIN2 synthesis in the awake rat.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 stimulates PROTEIN1 and PROTEIN2 synthesis in the awake rat.\n",
      "\n",
      "PROTEIN0 PROTEIN1 To determine the effect of PROTEIN0 PROTEIN1 on cardiac contractile protein synthesis in vivo, we measured L-[ring-2, 6-3H]phenylalanine incorporation into PROTEIN2 and PROTEIN3 during intravenous infusions (4 h) of either saline or PROTEIN4 (1 microgram. kg-1. min-1) in awake rats.\n",
      "\n",
      "PROTEIN0 PROTEIN2 To determine the effect of PROTEIN0 PROTEIN1 on cardiac contractile protein synthesis in vivo, we measured L-[ring-2, 6-3H]phenylalanine incorporation into PROTEIN2 and PROTEIN3 during intravenous infusions (4 h) of either saline or PROTEIN4 (1 microgram. kg-1. min-1) in awake rats.\n",
      "\n",
      "PROTEIN0 PROTEIN3 To determine the effect of PROTEIN0 PROTEIN1 on cardiac contractile protein synthesis in vivo, we measured L-[ring-2, 6-3H]phenylalanine incorporation into PROTEIN2 and PROTEIN3 during intravenous infusions (4 h) of either saline or PROTEIN4 (1 microgram. kg-1. min-1) in awake rats.\n",
      "\n",
      "PROTEIN0 PROTEIN4 To determine the effect of PROTEIN0 PROTEIN1 on cardiac contractile protein synthesis in vivo, we measured L-[ring-2, 6-3H]phenylalanine incorporation into PROTEIN2 and PROTEIN3 during intravenous infusions (4 h) of either saline or PROTEIN4 (1 microgram. kg-1. min-1) in awake rats.\n",
      "\n",
      "PROTEIN1 PROTEIN2 To determine the effect of PROTEIN0 PROTEIN1 on cardiac contractile protein synthesis in vivo, we measured L-[ring-2, 6-3H]phenylalanine incorporation into PROTEIN2 and PROTEIN3 during intravenous infusions (4 h) of either saline or PROTEIN4 (1 microgram. kg-1. min-1) in awake rats.\n",
      "\n",
      "PROTEIN1 PROTEIN3 To determine the effect of PROTEIN0 PROTEIN1 on cardiac contractile protein synthesis in vivo, we measured L-[ring-2, 6-3H]phenylalanine incorporation into PROTEIN2 and PROTEIN3 during intravenous infusions (4 h) of either saline or PROTEIN4 (1 microgram. kg-1. min-1) in awake rats.\n",
      "\n",
      "PROTEIN1 PROTEIN4 To determine the effect of PROTEIN0 PROTEIN1 on cardiac contractile protein synthesis in vivo, we measured L-[ring-2, 6-3H]phenylalanine incorporation into PROTEIN2 and PROTEIN3 during intravenous infusions (4 h) of either saline or PROTEIN4 (1 microgram. kg-1. min-1) in awake rats.\n",
      "\n",
      "PROTEIN2 PROTEIN3 To determine the effect of PROTEIN0 PROTEIN1 on cardiac contractile protein synthesis in vivo, we measured L-[ring-2, 6-3H]phenylalanine incorporation into PROTEIN2 and PROTEIN3 during intravenous infusions (4 h) of either saline or PROTEIN4 (1 microgram. kg-1. min-1) in awake rats.\n",
      "\n",
      "PROTEIN2 PROTEIN4 To determine the effect of PROTEIN0 PROTEIN1 on cardiac contractile protein synthesis in vivo, we measured L-[ring-2, 6-3H]phenylalanine incorporation into PROTEIN2 and PROTEIN3 during intravenous infusions (4 h) of either saline or PROTEIN4 (1 microgram. kg-1. min-1) in awake rats.\n",
      "\n",
      "PROTEIN3 PROTEIN4 To determine the effect of PROTEIN0 PROTEIN1 on cardiac contractile protein synthesis in vivo, we measured L-[ring-2, 6-3H]phenylalanine incorporation into PROTEIN2 and PROTEIN3 during intravenous infusions (4 h) of either saline or PROTEIN4 (1 microgram. kg-1. min-1) in awake rats.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Integrating the PROTEIN0 and PROTEIN1 cytoskeletons.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binding may be promoted by disruption of links to the cytoskeleton, effected through depolymerisation of PROTEIN1 or cleavage of linking protein PROTEIN2 by PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 binding may be promoted by disruption of links to the cytoskeleton, effected through depolymerisation of PROTEIN1 or cleavage of linking protein PROTEIN2 by PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 binding may be promoted by disruption of links to the cytoskeleton, effected through depolymerisation of PROTEIN1 or cleavage of linking protein PROTEIN2 by PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 binding may be promoted by disruption of links to the cytoskeleton, effected through depolymerisation of PROTEIN1 or cleavage of linking protein PROTEIN2 by PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 binding may be promoted by disruption of links to the cytoskeleton, effected through depolymerisation of PROTEIN1 or cleavage of linking protein PROTEIN2 by PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 binding may be promoted by disruption of links to the cytoskeleton, effected through depolymerisation of PROTEIN1 or cleavage of linking protein PROTEIN2 by PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interaction of PROTEIN0 with PROTEIN1 controls the association of apical PROTEIN2 in epithelial cells.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PROTEIN0 PROTEIN2 Interaction of PROTEIN0 with PROTEIN1 controls the association of apical PROTEIN2 in epithelial cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Interaction of PROTEIN0 with PROTEIN1 controls the association of apical PROTEIN2 in epithelial cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The quantity of the complex is reduced by treatment of Caco-2 cells with the alkylating agent N-ethylmaleimide which also has an inhibitory effect on the ability of PROTEIN0 to associate with PROTEIN1 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the PROTEIN0 assay system, silencing depends completely upon the function of the PROTEIN1 genes, known trans-acting regulators of the silent loci, and is sensitive to the gene dosage of two PROTEIN2 genes, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the PROTEIN0 assay system, silencing depends completely upon the function of the PROTEIN1 genes, known trans-acting regulators of the silent loci, and is sensitive to the gene dosage of two PROTEIN2 genes, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 In the PROTEIN0 assay system, silencing depends completely upon the function of the PROTEIN1 genes, known trans-acting regulators of the silent loci, and is sensitive to the gene dosage of two PROTEIN2 genes, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 In the PROTEIN0 assay system, silencing depends completely upon the function of the PROTEIN1 genes, known trans-acting regulators of the silent loci, and is sensitive to the gene dosage of two PROTEIN2 genes, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the PROTEIN0 assay system, silencing depends completely upon the function of the PROTEIN1 genes, known trans-acting regulators of the silent loci, and is sensitive to the gene dosage of two PROTEIN2 genes, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 In the PROTEIN0 assay system, silencing depends completely upon the function of the PROTEIN1 genes, known trans-acting regulators of the silent loci, and is sensitive to the gene dosage of two PROTEIN2 genes, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 In the PROTEIN0 assay system, silencing depends completely upon the function of the PROTEIN1 genes, known trans-acting regulators of the silent loci, and is sensitive to the gene dosage of two PROTEIN2 genes, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 In the PROTEIN0 assay system, silencing depends completely upon the function of the PROTEIN1 genes, known trans-acting regulators of the silent loci, and is sensitive to the gene dosage of two PROTEIN2 genes, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 In the PROTEIN0 assay system, silencing depends completely upon the function of the PROTEIN1 genes, known trans-acting regulators of the silent loci, and is sensitive to the gene dosage of two PROTEIN2 genes, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 In the PROTEIN0 assay system, silencing depends completely upon the function of the PROTEIN1 genes, known trans-acting regulators of the silent loci, and is sensitive to the gene dosage of two PROTEIN2 genes, PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN4 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN5 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN6 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN1 PROTEIN4 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN1 PROTEIN5 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN1 PROTEIN6 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN2 PROTEIN4 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN2 PROTEIN5 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN2 PROTEIN6 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN3 PROTEIN4 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN3 PROTEIN5 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN3 PROTEIN6 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN4 PROTEIN5 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN4 PROTEIN6 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN5 PROTEIN6 In the basal layer, patches of brightly labeled cells were detected with antibodies to PROTEIN0 PROTEIN1 and PROTEIN2 but not with antibodies to PROTEIN3 PROTEIN4 or with PROTEIN5 and PROTEIN6 antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 In the budding yeast Saccharomyces cerevisiae, the DNA damage-induced G2 arrest requires the checkpoint control genes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 In the cytoplasm, PROTEIN0 was shown to stimulate the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 cascade, which is essential for activation of transcription factor PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN2 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN3 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN4 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN5 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN6 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN2 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN3 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN4 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN5 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN6 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN3 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN4 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN5 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN6 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN4 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN5 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN6 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN5 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN6 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN5 PROTEIN6 In these investigations, we determine that recruitment of a coactivator protein, PROTEIN0 (the PROTEIN1 to the PROTEIN2 or PROTEIN3 promoter is the molecular mechanism of transcriptional activation by PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this PROTEIN0 clustering, PROTEIN1 a monomeric PROTEIN2 that has been suggested to be involved in PROTEIN3 polymerization, was shown to be essential.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this PROTEIN0 clustering, PROTEIN1 a monomeric PROTEIN2 that has been suggested to be involved in PROTEIN3 polymerization, was shown to be essential.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In this PROTEIN0 clustering, PROTEIN1 a monomeric PROTEIN2 that has been suggested to be involved in PROTEIN3 polymerization, was shown to be essential.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this PROTEIN0 clustering, PROTEIN1 a monomeric PROTEIN2 that has been suggested to be involved in PROTEIN3 polymerization, was shown to be essential.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In this PROTEIN0 clustering, PROTEIN1 a monomeric PROTEIN2 that has been suggested to be involved in PROTEIN3 polymerization, was shown to be essential.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In this PROTEIN0 clustering, PROTEIN1 a monomeric PROTEIN2 that has been suggested to be involved in PROTEIN3 polymerization, was shown to be essential.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this case, efficient repair does not require PROTEIN0 and PROTEIN1 consistent with our previous observations.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this model, PROTEIN0 can bind both to PROTEIN1 monomers with a Kd of about 5 microM and to the barbed end of PROTEIN2 filaments with a Kd of about 50-100 microM.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this model, PROTEIN0 can bind both to PROTEIN1 monomers with a Kd of about 5 microM and to the barbed end of PROTEIN2 filaments with a Kd of about 50-100 microM.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this model, PROTEIN0 can bind both to PROTEIN1 monomers with a Kd of about 5 microM and to the barbed end of PROTEIN2 filaments with a Kd of about 50-100 microM.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Quantitative analysis of the effect of Acanthamoeba PROTEIN0 on PROTEIN1 filament nucleation and elongation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This model includes two different Kd's--one for PROTEIN0 bound to PROTEIN1 monomers and one for PROTEIN2 bound to an PROTEIN3 molecule at the barbed end of a filament.\n",
      "\n",
      "PROTEIN0 PROTEIN2 This model includes two different Kd's--one for PROTEIN0 bound to PROTEIN1 monomers and one for PROTEIN2 bound to an PROTEIN3 molecule at the barbed end of a filament.\n",
      "\n",
      "PROTEIN0 PROTEIN3 This model includes two different Kd's--one for PROTEIN0 bound to PROTEIN1 monomers and one for PROTEIN2 bound to an PROTEIN3 molecule at the barbed end of a filament.\n",
      "\n",
      "PROTEIN1 PROTEIN2 This model includes two different Kd's--one for PROTEIN0 bound to PROTEIN1 monomers and one for PROTEIN2 bound to an PROTEIN3 molecule at the barbed end of a filament.\n",
      "\n",
      "PROTEIN1 PROTEIN3 This model includes two different Kd's--one for PROTEIN0 bound to PROTEIN1 monomers and one for PROTEIN2 bound to an PROTEIN3 molecule at the barbed end of a filament.\n",
      "\n",
      "PROTEIN2 PROTEIN3 This model includes two different Kd's--one for PROTEIN0 bound to PROTEIN1 monomers and one for PROTEIN2 bound to an PROTEIN3 molecule at the barbed end of a filament.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 In this recombinant expression system, the PROTEIN0 relatives, human PROTEIN1 PROTEIN2 or PROTEIN3 and the PROTEIN4 from Torpedo electric organ, also bind to translated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this study, by immunoprecipitation and affinity chromatography it is shown that PROTEIN0 and PROTEIN1 interact in pig brain cortical membranes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN0 PROTEIN3 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN0 PROTEIN4 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN0 PROTEIN5 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN0 PROTEIN6 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN1 PROTEIN3 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN1 PROTEIN4 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN1 PROTEIN5 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN1 PROTEIN6 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN2 PROTEIN3 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN2 PROTEIN4 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN2 PROTEIN5 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN2 PROTEIN6 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN3 PROTEIN4 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN3 PROTEIN5 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN3 PROTEIN6 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN4 PROTEIN5 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN4 PROTEIN6 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN5 PROTEIN6 In this study, the expressions of smooth muscle-specific proteins PROTEIN0 PROTEIN1 PROTEIN2 and the PROTEIN3 isoforms of PROTEIN4 and PROTEIN5 in three parent and four cloned neuroblastoma cell lines, composed of S-type cells, were examined by indirect immunofluorescence, Western blot and/or by reverse PROTEIN6 chain reaction (RT-PCR).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Redundancy of Saccharomyces cerevisiae PROTEIN0 and PROTEIN1 in PROTEIN2 mismatch repair.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Redundancy of Saccharomyces cerevisiae PROTEIN0 and PROTEIN1 in PROTEIN2 mismatch repair.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Redundancy of Saccharomyces cerevisiae PROTEIN0 and PROTEIN1 in PROTEIN2 mismatch repair.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN2 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN3 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN4 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN5 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN6 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN2 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN3 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN4 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN5 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN6 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN3 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN4 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN5 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN6 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN4 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN5 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN6 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN5 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN6 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN5 PROTEIN6 The results indicate that S. cerevisiae has two pathways of PROTEIN0 mismatch repair: one that recognized single-base mispairs and requires PROTEIN1 and PROTEIN2 and a second that recognizes insertion/deletion mispairs and requires a combination of either PROTEIN3 and PROTEIN4 or PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this study we demonstrate that PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 synthesized in young (5 hr) conjugating Tetrahymena are deposited into micronuclei in acetylated forms.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this study we demonstrate that PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 synthesized in young (5 hr) conjugating Tetrahymena are deposited into micronuclei in acetylated forms.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In this study we demonstrate that PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 synthesized in young (5 hr) conjugating Tetrahymena are deposited into micronuclei in acetylated forms.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this study we demonstrate that PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 synthesized in young (5 hr) conjugating Tetrahymena are deposited into micronuclei in acetylated forms.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In this study we demonstrate that PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 synthesized in young (5 hr) conjugating Tetrahymena are deposited into micronuclei in acetylated forms.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In this study we demonstrate that PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 synthesized in young (5 hr) conjugating Tetrahymena are deposited into micronuclei in acetylated forms.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In three of the cell lines with PROTEIN0 mutations, complexes of PROTEIN1 with PROTEIN2 and PROTEIN3 were detectable.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In three of the cell lines with PROTEIN0 mutations, complexes of PROTEIN1 with PROTEIN2 and PROTEIN3 were detectable.\n",
      "\n",
      "PROTEIN0 PROTEIN3 In three of the cell lines with PROTEIN0 mutations, complexes of PROTEIN1 with PROTEIN2 and PROTEIN3 were detectable.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In three of the cell lines with PROTEIN0 mutations, complexes of PROTEIN1 with PROTEIN2 and PROTEIN3 were detectable.\n",
      "\n",
      "PROTEIN1 PROTEIN3 In three of the cell lines with PROTEIN0 mutations, complexes of PROTEIN1 with PROTEIN2 and PROTEIN3 were detectable.\n",
      "\n",
      "PROTEIN2 PROTEIN3 In three of the cell lines with PROTEIN0 mutations, complexes of PROTEIN1 with PROTEIN2 and PROTEIN3 were detectable.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Introduction of high concentrations of PROTEIN0 (estimated injected intracellular concentration 11-22 microM) into infected PtK2 cells causes a marked slowing of PROTEIN1 tail elongation and bacterial migration.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In vivo importance of PROTEIN0 nucleotide exchange catalyzed by PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 an PROTEIN1 mutant with an increased intrinsic rate of nucleotide exchange, suppressed defects in PROTEIN2 organization, cell growth, and fluid-phase endocytosis of PROTEIN3 a PROTEIN4 mutant defective in PROTEIN5 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 monomer-binding protein, PROTEIN1 influences the dynamics of PROTEIN2 filaments in vitro by suppressing nucleation, enhancing nucleotide exchange on PROTEIN3 and promoting barbed-end assembly.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 monomer-binding protein, PROTEIN1 influences the dynamics of PROTEIN2 filaments in vitro by suppressing nucleation, enhancing nucleotide exchange on PROTEIN3 and promoting barbed-end assembly.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 monomer-binding protein, PROTEIN1 influences the dynamics of PROTEIN2 filaments in vitro by suppressing nucleation, enhancing nucleotide exchange on PROTEIN3 and promoting barbed-end assembly.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 monomer-binding protein, PROTEIN1 influences the dynamics of PROTEIN2 filaments in vitro by suppressing nucleation, enhancing nucleotide exchange on PROTEIN3 and promoting barbed-end assembly.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 monomer-binding protein, PROTEIN1 influences the dynamics of PROTEIN2 filaments in vitro by suppressing nucleation, enhancing nucleotide exchange on PROTEIN3 and promoting barbed-end assembly.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 monomer-binding protein, PROTEIN1 influences the dynamics of PROTEIN2 filaments in vitro by suppressing nucleation, enhancing nucleotide exchange on PROTEIN3 and promoting barbed-end assembly.\n",
      "\n",
      "PROTEIN0 PROTEIN1 To study PROTEIN0 function, we extensively mutagenized the Saccharomyces cerevisiae PROTEIN1 gene PROTEIN2 and examined the consequences of specific point mutations on growth and PROTEIN3 organization.\n",
      "\n",
      "PROTEIN0 PROTEIN2 To study PROTEIN0 function, we extensively mutagenized the Saccharomyces cerevisiae PROTEIN1 gene PROTEIN2 and examined the consequences of specific point mutations on growth and PROTEIN3 organization.\n",
      "\n",
      "PROTEIN0 PROTEIN3 To study PROTEIN0 function, we extensively mutagenized the Saccharomyces cerevisiae PROTEIN1 gene PROTEIN2 and examined the consequences of specific point mutations on growth and PROTEIN3 organization.\n",
      "\n",
      "PROTEIN1 PROTEIN2 To study PROTEIN0 function, we extensively mutagenized the Saccharomyces cerevisiae PROTEIN1 gene PROTEIN2 and examined the consequences of specific point mutations on growth and PROTEIN3 organization.\n",
      "\n",
      "PROTEIN1 PROTEIN3 To study PROTEIN0 function, we extensively mutagenized the Saccharomyces cerevisiae PROTEIN1 gene PROTEIN2 and examined the consequences of specific point mutations on growth and PROTEIN3 organization.\n",
      "\n",
      "PROTEIN2 PROTEIN3 To study PROTEIN0 function, we extensively mutagenized the Saccharomyces cerevisiae PROTEIN1 gene PROTEIN2 and examined the consequences of specific point mutations on growth and PROTEIN3 organization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Involvement of PROTEIN0 in the PROTEIN1 motility of L. monocytogenes in cells and in cell-free extracts.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN7 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN8 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN9 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN1 PROTEIN7 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PROTEIN1 PROTEIN8 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN1 PROTEIN9 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN2 PROTEIN7 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN2 PROTEIN8 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN2 PROTEIN9 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN3 PROTEIN7 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN3 PROTEIN8 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN3 PROTEIN9 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN4 PROTEIN7 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN4 PROTEIN8 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN4 PROTEIN9 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN5 PROTEIN7 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN5 PROTEIN8 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN5 PROTEIN9 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN6 PROTEIN7 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN6 PROTEIN8 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN6 PROTEIN9 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN7 PROTEIN8 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN7 PROTEIN9 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN8 PROTEIN9 Isolation and characterization of death domain PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 domain-containing proteins PROTEIN7 PROTEIN8 PROTEIN9 have partially bridged a large molecular gap within one of several signaling pathways which originate at the plasma membrane and terminate in the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Isolation of PROTEIN0 from embryonic chicken skeletal muscle and evaluation of its interaction with different PROTEIN1 isoforms.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results indicate that the assembly of PROTEIN0 and PROTEIN1 is regulated differently by PROTEIN2 in the developing skeletal muscle, and that the former may not be involved in myofibril assembly.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results indicate that the assembly of PROTEIN0 and PROTEIN1 is regulated differently by PROTEIN2 in the developing skeletal muscle, and that the former may not be involved in myofibril assembly.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results indicate that the assembly of PROTEIN0 and PROTEIN1 is regulated differently by PROTEIN2 in the developing skeletal muscle, and that the former may not be involved in myofibril assembly.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Isolation of human PROTEIN0 and its binding specificity with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Isolation of human PROTEIN0 and PROTEIN1 for measurement of fractional synthesis rates.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE), we have developed a simple method to isolate PROTEIN0 PROTEIN1 and PROTEIN2 from small (60-80 mg) human skeletal muscle samples for the determination of their fractional synthesis rates.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE), we have developed a simple method to isolate PROTEIN0 PROTEIN1 and PROTEIN2 from small (60-80 mg) human skeletal muscle samples for the determination of their fractional synthesis rates.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE), we have developed a simple method to isolate PROTEIN0 PROTEIN1 and PROTEIN2 from small (60-80 mg) human skeletal muscle samples for the determination of their fractional synthesis rates.\n",
      "\n",
      "PROTEIN0 PROTEIN1 It is also possible that PtdIns-4,5-P2 has other roles, such as promoting the release of G PROTEIN0 from PROTEIN1 or promoting the cross-linking of PROTEIN2 or its anchorage to the plasma membrane.\n",
      "\n",
      "PROTEIN0 PROTEIN2 It is also possible that PtdIns-4,5-P2 has other roles, such as promoting the release of G PROTEIN0 from PROTEIN1 or promoting the cross-linking of PROTEIN2 or its anchorage to the plasma membrane.\n",
      "\n",
      "PROTEIN1 PROTEIN2 It is also possible that PtdIns-4,5-P2 has other roles, such as promoting the release of G PROTEIN0 from PROTEIN1 or promoting the cross-linking of PROTEIN2 or its anchorage to the plasma membrane.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Native PROTEIN0 purified from extracts of lung epithelial cells by affinity binding to a poly-L-proline matrix, stimulated the PROTEIN1 RSV transcription by 2.5- to 3-fold.\n",
      "\n",
      "PROTEIN0 PROTEIN1 It reduced the rate of PROTEIN0 polymerization as reported in the cases of PROTEIN1 from other organisms.\n",
      "\n",
      "PROTEIN0 PROTEIN1 It therefore appears that PROTEIN0 and PROTEIN1 multigene families are both expressed in a species specific fashion but are independently regulated within a species.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Regulation of PROTEIN0 and PROTEIN1 isogenes expression during cardiac growth.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 phenotype is developmentally and hormonally regulated, but less is known concerning the PROTEIN1 phenotype.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Kinetic measurements indicate that PROTEIN0 binding to PROTEIN1 weakens the affinity of PROTEIN2 for nucleotides primarily due to an increased nucleotide dissociation rate constant, but the nucleotide association rate constant is also increased about 2-fold.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Kinetic measurements indicate that PROTEIN0 binding to PROTEIN1 weakens the affinity of PROTEIN2 for nucleotides primarily due to an increased nucleotide dissociation rate constant, but the nucleotide association rate constant is also increased about 2-fold.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Kinetic measurements indicate that PROTEIN0 binding to PROTEIN1 weakens the affinity of PROTEIN2 for nucleotides primarily due to an increased nucleotide dissociation rate constant, but the nucleotide association rate constant is also increased about 2-fold.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The data suggest that PROTEIN0 binding to PROTEIN1 weakens nucleotide binding to PROTEIN2 by disrupting Mg(2+) coordination in the PROTEIN3 central cleft.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The data suggest that PROTEIN0 binding to PROTEIN1 weakens nucleotide binding to PROTEIN2 by disrupting Mg(2+) coordination in the PROTEIN3 central cleft.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The data suggest that PROTEIN0 binding to PROTEIN1 weakens nucleotide binding to PROTEIN2 by disrupting Mg(2+) coordination in the PROTEIN3 central cleft.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The data suggest that PROTEIN0 binding to PROTEIN1 weakens nucleotide binding to PROTEIN2 by disrupting Mg(2+) coordination in the PROTEIN3 central cleft.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The data suggest that PROTEIN0 binding to PROTEIN1 weakens nucleotide binding to PROTEIN2 by disrupting Mg(2+) coordination in the PROTEIN3 central cleft.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The data suggest that PROTEIN0 binding to PROTEIN1 weakens nucleotide binding to PROTEIN2 by disrupting Mg(2+) coordination in the PROTEIN3 central cleft.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Kinetic studies on the effect of yeast PROTEIN0 on yeast PROTEIN1 polymerization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The results suggest some cooperativity with respect to PROTEIN0 binding to filamentous PROTEIN1 which may be pH dependent.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Lack of detection in these preparations of a constituent that binds RBC PROTEIN0 antibody plus the presence of significant quantities of PROTEIN1 further suggests that the principal membrane skeletal element is PROTEIN2 with perhaps smaller quantities of a 240-kd component, identified by others as the protein PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Lack of detection in these preparations of a constituent that binds RBC PROTEIN0 antibody plus the presence of significant quantities of PROTEIN1 further suggests that the principal membrane skeletal element is PROTEIN2 with perhaps smaller quantities of a 240-kd component, identified by others as the protein PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Lack of detection in these preparations of a constituent that binds RBC PROTEIN0 antibody plus the presence of significant quantities of PROTEIN1 further suggests that the principal membrane skeletal element is PROTEIN2 with perhaps smaller quantities of a 240-kd component, identified by others as the protein PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Lack of detection in these preparations of a constituent that binds RBC PROTEIN0 antibody plus the presence of significant quantities of PROTEIN1 further suggests that the principal membrane skeletal element is PROTEIN2 with perhaps smaller quantities of a 240-kd component, identified by others as the protein PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Lack of detection in these preparations of a constituent that binds RBC PROTEIN0 antibody plus the presence of significant quantities of PROTEIN1 further suggests that the principal membrane skeletal element is PROTEIN2 with perhaps smaller quantities of a 240-kd component, identified by others as the protein PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Lack of detection in these preparations of a constituent that binds RBC PROTEIN0 antibody plus the presence of significant quantities of PROTEIN1 further suggests that the principal membrane skeletal element is PROTEIN2 with perhaps smaller quantities of a 240-kd component, identified by others as the protein PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN0 PROTEIN7 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN0 PROTEIN8 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN0 PROTEIN9 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN0 PROTEIN0 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN1 PROTEIN7 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN1 PROTEIN8 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN1 PROTEIN9 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN1 PROTEIN0 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN1 PROTEIN1 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN2 PROTEIN7 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN2 PROTEIN8 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN2 PROTEIN9 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN2 PROTEIN0 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN2 PROTEIN1 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN3 PROTEIN7 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN3 PROTEIN8 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN3 PROTEIN9 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN3 PROTEIN0 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN3 PROTEIN1 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN4 PROTEIN7 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN4 PROTEIN8 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN4 PROTEIN9 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN4 PROTEIN0 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN4 PROTEIN1 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN5 PROTEIN7 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN5 PROTEIN8 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN5 PROTEIN9 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN5 PROTEIN0 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN5 PROTEIN1 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN6 PROTEIN7 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN6 PROTEIN8 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN6 PROTEIN9 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN6 PROTEIN0 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN6 PROTEIN1 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN7 PROTEIN8 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN7 PROTEIN9 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN7 PROTEIN0 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN7 PROTEIN1 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN8 PROTEIN9 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN8 PROTEIN0 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN8 PROTEIN1 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN9 PROTEIN0 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN9 PROTEIN1 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Lethality of the PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 double mutants can also be rescued by mutations in the DNA damage checkpoint functions PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 indicating that the PROTEIN8 PROTEIN9 and PROTEIN0 PROTEIN1 mutant combinations lead to an intermediate that is sensed by these checkpoint functions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Like the PROTEIN0 the major central part of the PROTEIN1 is made up of repeat units of 106 amino-acids.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 phosphorylates PROTEIN2 an PROTEIN3 and regulates PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 phosphorylates PROTEIN2 an PROTEIN3 and regulates PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 phosphorylates PROTEIN2 an PROTEIN3 and regulates PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 PROTEIN1 phosphorylates PROTEIN2 an PROTEIN3 and regulates PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 phosphorylates PROTEIN2 an PROTEIN3 and regulates PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 phosphorylates PROTEIN2 an PROTEIN3 and regulates PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 PROTEIN1 phosphorylates PROTEIN2 an PROTEIN3 and regulates PROTEIN4 cytoskeletal reorganization.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 phosphorylates PROTEIN2 an PROTEIN3 and regulates PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 PROTEIN1 phosphorylates PROTEIN2 an PROTEIN3 and regulates PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 PROTEIN1 phosphorylates PROTEIN2 an PROTEIN3 and regulates PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Linkage between cancer susceptibility and the candidate DNA mismatch repair genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 was investigated.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Linkage between cancer susceptibility and the candidate DNA mismatch repair genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 was investigated.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Linkage between cancer susceptibility and the candidate DNA mismatch repair genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 was investigated.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Linkage between cancer susceptibility and the candidate DNA mismatch repair genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 was investigated.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Linkage between cancer susceptibility and the candidate DNA mismatch repair genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 was investigated.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Linkage between cancer susceptibility and the candidate DNA mismatch repair genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 was investigated.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Linkage between cancer susceptibility and the candidate DNA mismatch repair genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 was investigated.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Linkage between cancer susceptibility and the candidate DNA mismatch repair genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 was investigated.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Linkage between cancer susceptibility and the candidate DNA mismatch repair genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 was investigated.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Linkage between cancer susceptibility and the candidate DNA mismatch repair genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 PROTEIN4 was investigated.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Live dynamics of Dictyostelium PROTEIN0 suggests a role in remodeling PROTEIN1 latticework into bundles.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results demonstrate that PROTEIN0 plays a crucial role in vivo in rapid remodeling of the cortical PROTEIN1 meshwork into bundles.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Localization of PROTEIN0 to the leading edge of a migrating cell can lead to local accumulation of PROTEIN1 which in turn can supply PROTEIN2 monomers to growing filament ends.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Localization of PROTEIN0 to the leading edge of a migrating cell can lead to local accumulation of PROTEIN1 which in turn can supply PROTEIN2 monomers to growing filament ends.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Localization of PROTEIN0 to the leading edge of a migrating cell can lead to local accumulation of PROTEIN1 which in turn can supply PROTEIN2 monomers to growing filament ends.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Location of PROTEIN0 at presynaptic sites in the cerebellar cortex; implication for the regulation of the PROTEIN1 state during axonal elongation and synaptogenesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Loss of memberanous expression of PROTEIN0 PROTEIN1 and PROTEIN2 was demonstrated in 52%, 85% and 40% of tumours respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Loss of memberanous expression of PROTEIN0 PROTEIN1 and PROTEIN2 was demonstrated in 52%, 85% and 40% of tumours respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Loss of memberanous expression of PROTEIN0 PROTEIN1 and PROTEIN2 was demonstrated in 52%, 85% and 40% of tumours respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN1 There was a trend towards an association between advanced tumour stage and loss of membranous expression of PROTEIN0 or PROTEIN1 although these associations were not statistically significant.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mammalian homologues of two important yeast genes involved in DNA double-strand break repair and recombination, PROTEIN0 and PROTEIN1 have been isolated.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mammalian PROTEIN0 PROTEIN1 phosphorylate PROTEIN2 and induce PROTEIN3 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Mammalian PROTEIN0 PROTEIN1 phosphorylate PROTEIN2 and induce PROTEIN3 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Mammalian PROTEIN0 PROTEIN1 phosphorylate PROTEIN2 and induce PROTEIN3 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Mammalian PROTEIN0 PROTEIN1 phosphorylate PROTEIN2 and induce PROTEIN3 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Mammalian PROTEIN0 PROTEIN1 phosphorylate PROTEIN2 and induce PROTEIN3 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Mammalian PROTEIN0 PROTEIN1 phosphorylate PROTEIN2 and induce PROTEIN3 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 MATERIALS AND METHODS: A mutation in the PROTEIN0 PROTEIN1 predicted to interfere strongly with PROTEIN2 binding to PROTEIN3 was designed and used to create an animal model for HCM.\n",
      "\n",
      "PROTEIN0 PROTEIN2 MATERIALS AND METHODS: A mutation in the PROTEIN0 PROTEIN1 predicted to interfere strongly with PROTEIN2 binding to PROTEIN3 was designed and used to create an animal model for HCM.\n",
      "\n",
      "PROTEIN0 PROTEIN3 MATERIALS AND METHODS: A mutation in the PROTEIN0 PROTEIN1 predicted to interfere strongly with PROTEIN2 binding to PROTEIN3 was designed and used to create an animal model for HCM.\n",
      "\n",
      "PROTEIN1 PROTEIN2 MATERIALS AND METHODS: A mutation in the PROTEIN0 PROTEIN1 predicted to interfere strongly with PROTEIN2 binding to PROTEIN3 was designed and used to create an animal model for HCM.\n",
      "\n",
      "PROTEIN1 PROTEIN3 MATERIALS AND METHODS: A mutation in the PROTEIN0 PROTEIN1 predicted to interfere strongly with PROTEIN2 binding to PROTEIN3 was designed and used to create an animal model for HCM.\n",
      "\n",
      "PROTEIN2 PROTEIN3 MATERIALS AND METHODS: A mutation in the PROTEIN0 PROTEIN1 predicted to interfere strongly with PROTEIN2 binding to PROTEIN3 was designed and used to create an animal model for HCM.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mechanism of inhibition of PROTEIN0 DNA synthesis by the PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Mechanism of inhibition of PROTEIN0 DNA synthesis by the PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Mechanism of inhibition of PROTEIN0 DNA synthesis by the PROTEIN1 PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mechanism of regulation of PROTEIN0 polymerization by Physarum PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The apparent critical concentration for polymerization of PROTEIN0 is increased by the addition of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Membranous staining and concomitant cytoplasmic localization of PROTEIN0 PROTEIN1 and PROTEIN2 were seen in one case with abnormal PROTEIN3 immunoreactivity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Membranous staining and concomitant cytoplasmic localization of PROTEIN0 PROTEIN1 and PROTEIN2 were seen in one case with abnormal PROTEIN3 immunoreactivity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Membranous staining and concomitant cytoplasmic localization of PROTEIN0 PROTEIN1 and PROTEIN2 were seen in one case with abnormal PROTEIN3 immunoreactivity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Membranous staining and concomitant cytoplasmic localization of PROTEIN0 PROTEIN1 and PROTEIN2 were seen in one case with abnormal PROTEIN3 immunoreactivity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Membranous staining and concomitant cytoplasmic localization of PROTEIN0 PROTEIN1 and PROTEIN2 were seen in one case with abnormal PROTEIN3 immunoreactivity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Membranous staining and concomitant cytoplasmic localization of PROTEIN0 PROTEIN1 and PROTEIN2 were seen in one case with abnormal PROTEIN3 immunoreactivity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 METHODS: Ileal and colonic biopsy specimens from 19 SpA patients with subclinical inflammatory gut lesions and from seven controls were stained with monoclonal antibodies against PROTEIN0 PROTEIN1 and PROTEIN2 and a polyclonal antibody against PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 METHODS: Ileal and colonic biopsy specimens from 19 SpA patients with subclinical inflammatory gut lesions and from seven controls were stained with monoclonal antibodies against PROTEIN0 PROTEIN1 and PROTEIN2 and a polyclonal antibody against PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 METHODS: Ileal and colonic biopsy specimens from 19 SpA patients with subclinical inflammatory gut lesions and from seven controls were stained with monoclonal antibodies against PROTEIN0 PROTEIN1 and PROTEIN2 and a polyclonal antibody against PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 METHODS: Ileal and colonic biopsy specimens from 19 SpA patients with subclinical inflammatory gut lesions and from seven controls were stained with monoclonal antibodies against PROTEIN0 PROTEIN1 and PROTEIN2 and a polyclonal antibody against PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 METHODS: Ileal and colonic biopsy specimens from 19 SpA patients with subclinical inflammatory gut lesions and from seven controls were stained with monoclonal antibodies against PROTEIN0 PROTEIN1 and PROTEIN2 and a polyclonal antibody against PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 METHODS: Ileal and colonic biopsy specimens from 19 SpA patients with subclinical inflammatory gut lesions and from seven controls were stained with monoclonal antibodies against PROTEIN0 PROTEIN1 and PROTEIN2 and a polyclonal antibody against PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 METHODS: Paraffin-embedded sections of five AVMs, CCMs, and control brain tissues were stained immunohistochemically with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 a novel marker for contractile VSMC phenotype.\n",
      "\n",
      "PROTEIN0 PROTEIN2 METHODS: Paraffin-embedded sections of five AVMs, CCMs, and control brain tissues were stained immunohistochemically with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 a novel marker for contractile VSMC phenotype.\n",
      "\n",
      "PROTEIN0 PROTEIN3 METHODS: Paraffin-embedded sections of five AVMs, CCMs, and control brain tissues were stained immunohistochemically with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 a novel marker for contractile VSMC phenotype.\n",
      "\n",
      "PROTEIN1 PROTEIN2 METHODS: Paraffin-embedded sections of five AVMs, CCMs, and control brain tissues were stained immunohistochemically with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 a novel marker for contractile VSMC phenotype.\n",
      "\n",
      "PROTEIN1 PROTEIN3 METHODS: Paraffin-embedded sections of five AVMs, CCMs, and control brain tissues were stained immunohistochemically with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 a novel marker for contractile VSMC phenotype.\n",
      "\n",
      "PROTEIN2 PROTEIN3 METHODS: Paraffin-embedded sections of five AVMs, CCMs, and control brain tissues were stained immunohistochemically with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 a novel marker for contractile VSMC phenotype.\n",
      "\n",
      "PROTEIN0 PROTEIN1 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN0 PROTEIN2 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN0 PROTEIN3 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN0 PROTEIN4 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN0 PROTEIN5 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN0 PROTEIN6 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN0 PROTEIN7 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN0 PROTEIN8 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN0 PROTEIN9 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN0 PROTEIN0 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN1 PROTEIN2 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN1 PROTEIN3 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN1 PROTEIN4 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN1 PROTEIN5 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN1 PROTEIN6 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN1 PROTEIN7 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN1 PROTEIN8 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN1 PROTEIN9 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN1 PROTEIN0 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN2 PROTEIN3 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN2 PROTEIN4 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN2 PROTEIN5 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN2 PROTEIN6 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN2 PROTEIN7 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN2 PROTEIN8 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN2 PROTEIN9 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN2 PROTEIN0 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN3 PROTEIN4 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN3 PROTEIN5 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN3 PROTEIN6 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN3 PROTEIN7 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN3 PROTEIN8 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN3 PROTEIN9 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN3 PROTEIN0 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN4 PROTEIN5 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN4 PROTEIN6 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN4 PROTEIN7 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN4 PROTEIN8 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN4 PROTEIN9 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN4 PROTEIN0 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN5 PROTEIN6 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN5 PROTEIN7 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN5 PROTEIN8 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN5 PROTEIN9 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN5 PROTEIN0 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN6 PROTEIN7 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN6 PROTEIN8 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN6 PROTEIN9 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN6 PROTEIN0 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN7 PROTEIN8 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN7 PROTEIN9 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN7 PROTEIN0 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN8 PROTEIN9 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN8 PROTEIN0 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN9 PROTEIN0 METHODS: We have performed a detailed study of the pattern of expression of PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 isoforms, PROTEIN6 PROTEIN7 PROTEIN8 and PROTEIN9 PROTEIN0 in the ventricular conduction system of normal and congenitally malformed mouse hearts (iv background) from embryonic day 14.5 to 19.5.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Microinjection or stable expression of PROTEIN0 into keratinocytes promotes the loss of PROTEIN1 and PROTEIN2 and relocalization of PROTEIN3 to the cytoplasm and nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Microinjection or stable expression of PROTEIN0 into keratinocytes promotes the loss of PROTEIN1 and PROTEIN2 and relocalization of PROTEIN3 to the cytoplasm and nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Microinjection or stable expression of PROTEIN0 into keratinocytes promotes the loss of PROTEIN1 and PROTEIN2 and relocalization of PROTEIN3 to the cytoplasm and nucleus.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Microinjection or stable expression of PROTEIN0 into keratinocytes promotes the loss of PROTEIN1 and PROTEIN2 and relocalization of PROTEIN3 to the cytoplasm and nucleus.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Microinjection or stable expression of PROTEIN0 into keratinocytes promotes the loss of PROTEIN1 and PROTEIN2 and relocalization of PROTEIN3 to the cytoplasm and nucleus.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Microinjection or stable expression of PROTEIN0 into keratinocytes promotes the loss of PROTEIN1 and PROTEIN2 and relocalization of PROTEIN3 to the cytoplasm and nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 is fused in-frame to a different exon of PROTEIN1 in two patients producing chimeric proteins containing the AT-hooks, PROTEIN2 homology domain, and transcriptional repression domain of PROTEIN3 fused to the PROTEIN4 binding domain or to the bromodomain of PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Modifying preselected sites on proteins: the stretch of residues 633-642 of the PROTEIN0 is part of the PROTEIN1 site.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Molecular analysis of the PROTEIN0 PROTEIN1 complex elucidated the central role of PROTEIN2 in this adhesion complex, as it binds to the cytoplasmic domain of PROTEIN3 and to PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Molecular analysis of the PROTEIN0 PROTEIN1 complex elucidated the central role of PROTEIN2 in this adhesion complex, as it binds to the cytoplasmic domain of PROTEIN3 and to PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Molecular analysis of the PROTEIN0 PROTEIN1 complex elucidated the central role of PROTEIN2 in this adhesion complex, as it binds to the cytoplasmic domain of PROTEIN3 and to PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Molecular analysis of the PROTEIN0 PROTEIN1 complex elucidated the central role of PROTEIN2 in this adhesion complex, as it binds to the cytoplasmic domain of PROTEIN3 and to PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Molecular analysis of the PROTEIN0 PROTEIN1 complex elucidated the central role of PROTEIN2 in this adhesion complex, as it binds to the cytoplasmic domain of PROTEIN3 and to PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Molecular analysis of the PROTEIN0 PROTEIN1 complex elucidated the central role of PROTEIN2 in this adhesion complex, as it binds to the cytoplasmic domain of PROTEIN3 and to PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Molecular analysis of the PROTEIN0 PROTEIN1 complex elucidated the central role of PROTEIN2 in this adhesion complex, as it binds to the cytoplasmic domain of PROTEIN3 and to PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Molecular analysis of the PROTEIN0 PROTEIN1 complex elucidated the central role of PROTEIN2 in this adhesion complex, as it binds to the cytoplasmic domain of PROTEIN3 and to PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Molecular analysis of the PROTEIN0 PROTEIN1 complex elucidated the central role of PROTEIN2 in this adhesion complex, as it binds to the cytoplasmic domain of PROTEIN3 and to PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Molecular analysis of the PROTEIN0 PROTEIN1 complex elucidated the central role of PROTEIN2 in this adhesion complex, as it binds to the cytoplasmic domain of PROTEIN3 and to PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 may also function in signalling pathways, given its homology to the gene product of the Drosophila segment polarity gene PROTEIN1 which is known to be involved in the wingless signalling cascade.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Molecular cloning and primary structure analysis demonstrated that PROTEIN0 is homologous to PROTEIN1 and the PROTEIN2 is homologous to human PROTEIN3 and the Drosophila gene product PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Molecular cloning and primary structure analysis demonstrated that PROTEIN0 is homologous to PROTEIN1 and the PROTEIN2 is homologous to human PROTEIN3 and the Drosophila gene product PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Molecular cloning and primary structure analysis demonstrated that PROTEIN0 is homologous to PROTEIN1 and the PROTEIN2 is homologous to human PROTEIN3 and the Drosophila gene product PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Molecular cloning and primary structure analysis demonstrated that PROTEIN0 is homologous to PROTEIN1 and the PROTEIN2 is homologous to human PROTEIN3 and the Drosophila gene product PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Molecular cloning and primary structure analysis demonstrated that PROTEIN0 is homologous to PROTEIN1 and the PROTEIN2 is homologous to human PROTEIN3 and the Drosophila gene product PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Molecular cloning and primary structure analysis demonstrated that PROTEIN0 is homologous to PROTEIN1 and the PROTEIN2 is homologous to human PROTEIN3 and the Drosophila gene product PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Molecular cloning and primary structure analysis demonstrated that PROTEIN0 is homologous to PROTEIN1 and the PROTEIN2 is homologous to human PROTEIN3 and the Drosophila gene product PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Molecular cloning and primary structure analysis demonstrated that PROTEIN0 is homologous to PROTEIN1 and the PROTEIN2 is homologous to human PROTEIN3 and the Drosophila gene product PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Molecular cloning and primary structure analysis demonstrated that PROTEIN0 is homologous to PROTEIN1 and the PROTEIN2 is homologous to human PROTEIN3 and the Drosophila gene product PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Molecular cloning and primary structure analysis demonstrated that PROTEIN0 is homologous to PROTEIN1 and the PROTEIN2 is homologous to human PROTEIN3 and the Drosophila gene product PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 One complex is composed of PROTEIN0 PROTEIN1 and PROTEIN2 the other of PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 [Molecular functions of PROTEIN0 which regulates reorganization of PROTEIN1 cytoskeleton].\n",
      "\n",
      "PROTEIN0 PROTEIN1 Molecules known to link PROTEIN0 filaments to membrane were also examined, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 but none was identified at the host-parasite junction.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Molecules known to link PROTEIN0 filaments to membrane were also examined, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 but none was identified at the host-parasite junction.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Molecules known to link PROTEIN0 filaments to membrane were also examined, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 but none was identified at the host-parasite junction.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Molecules known to link PROTEIN0 filaments to membrane were also examined, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 but none was identified at the host-parasite junction.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Molecules known to link PROTEIN0 filaments to membrane were also examined, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 but none was identified at the host-parasite junction.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Molecules known to link PROTEIN0 filaments to membrane were also examined, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 but none was identified at the host-parasite junction.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Molecules known to link PROTEIN0 filaments to membrane were also examined, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 but none was identified at the host-parasite junction.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Molecules known to link PROTEIN0 filaments to membrane were also examined, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 but none was identified at the host-parasite junction.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Molecules known to link PROTEIN0 filaments to membrane were also examined, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 but none was identified at the host-parasite junction.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Molecules known to link PROTEIN0 filaments to membrane were also examined, including PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 but none was identified at the host-parasite junction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Other PROTEIN0 proteins, including PROTEIN1 PROTEIN2 and PROTEIN3 are not present.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Other PROTEIN0 proteins, including PROTEIN1 PROTEIN2 and PROTEIN3 are not present.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Other PROTEIN0 proteins, including PROTEIN1 PROTEIN2 and PROTEIN3 are not present.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Other PROTEIN0 proteins, including PROTEIN1 PROTEIN2 and PROTEIN3 are not present.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Other PROTEIN0 proteins, including PROTEIN1 PROTEIN2 and PROTEIN3 are not present.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Other PROTEIN0 proteins, including PROTEIN1 PROTEIN2 and PROTEIN3 are not present.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Monoclonal antibodies recognizing the N- and C-terminal regions of PROTEIN0 disrupt PROTEIN1 stress fibers when microinjected into human fibroblasts.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results show that the C-terminal PROTEIN0 site is distinct from the region recognized by the anti-functional antibody PROTEIN1 raising the possibility that it binds to a novel functionally important ligand-binding site in the PROTEIN2 molecule.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results show that the C-terminal PROTEIN0 site is distinct from the region recognized by the anti-functional antibody PROTEIN1 raising the possibility that it binds to a novel functionally important ligand-binding site in the PROTEIN2 molecule.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results show that the C-terminal PROTEIN0 site is distinct from the region recognized by the anti-functional antibody PROTEIN1 raising the possibility that it binds to a novel functionally important ligand-binding site in the PROTEIN2 molecule.\n",
      "\n",
      "PROTEIN0 PROTEIN1 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN2 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN3 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN4 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN5 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN2 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN3 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN4 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN1 PROTEIN5 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN2 PROTEIN3 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN2 PROTEIN4 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN2 PROTEIN5 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN3 PROTEIN4 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN3 PROTEIN5 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN4 PROTEIN5 To investigate the possibility that PROTEIN0 disrupts PROTEIN1 stress fibers by binding directly to the C-terminal PROTEIN2 binding site, additional PROTEIN3 fusion proteins were generated and analyzed for PROTEIN4 and PROTEIN5 binding.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Monocytes and in vitro differentiated U937 cells induce focal loss in the staining of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 during transendothelial migration under physiological flow conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Monocytes and in vitro differentiated U937 cells induce focal loss in the staining of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 during transendothelial migration under physiological flow conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Monocytes and in vitro differentiated U937 cells induce focal loss in the staining of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 during transendothelial migration under physiological flow conditions.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Monocytes and in vitro differentiated U937 cells induce focal loss in the staining of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 during transendothelial migration under physiological flow conditions.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Monocytes and in vitro differentiated U937 cells induce focal loss in the staining of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 during transendothelial migration under physiological flow conditions.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Monocytes and in vitro differentiated U937 cells induce focal loss in the staining of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 during transendothelial migration under physiological flow conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover, PROTEIN0 but not PROTEIN1 or PROTEIN2 appears to co-localize with PROTEIN3 microfilaments in the membrane ruffles of NK cells that undergo cytoskeleton rearrangement following PROTEIN4 cross-linking.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Moreover, PROTEIN0 but not PROTEIN1 or PROTEIN2 appears to co-localize with PROTEIN3 microfilaments in the membrane ruffles of NK cells that undergo cytoskeleton rearrangement following PROTEIN4 cross-linking.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Moreover, PROTEIN0 but not PROTEIN1 or PROTEIN2 appears to co-localize with PROTEIN3 microfilaments in the membrane ruffles of NK cells that undergo cytoskeleton rearrangement following PROTEIN4 cross-linking.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Moreover, PROTEIN0 but not PROTEIN1 or PROTEIN2 appears to co-localize with PROTEIN3 microfilaments in the membrane ruffles of NK cells that undergo cytoskeleton rearrangement following PROTEIN4 cross-linking.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Moreover, PROTEIN0 but not PROTEIN1 or PROTEIN2 appears to co-localize with PROTEIN3 microfilaments in the membrane ruffles of NK cells that undergo cytoskeleton rearrangement following PROTEIN4 cross-linking.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Moreover, PROTEIN0 but not PROTEIN1 or PROTEIN2 appears to co-localize with PROTEIN3 microfilaments in the membrane ruffles of NK cells that undergo cytoskeleton rearrangement following PROTEIN4 cross-linking.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Moreover, PROTEIN0 but not PROTEIN1 or PROTEIN2 appears to co-localize with PROTEIN3 microfilaments in the membrane ruffles of NK cells that undergo cytoskeleton rearrangement following PROTEIN4 cross-linking.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Moreover, PROTEIN0 but not PROTEIN1 or PROTEIN2 appears to co-localize with PROTEIN3 microfilaments in the membrane ruffles of NK cells that undergo cytoskeleton rearrangement following PROTEIN4 cross-linking.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Moreover, PROTEIN0 but not PROTEIN1 or PROTEIN2 appears to co-localize with PROTEIN3 microfilaments in the membrane ruffles of NK cells that undergo cytoskeleton rearrangement following PROTEIN4 cross-linking.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Moreover, PROTEIN0 but not PROTEIN1 or PROTEIN2 appears to co-localize with PROTEIN3 microfilaments in the membrane ruffles of NK cells that undergo cytoskeleton rearrangement following PROTEIN4 cross-linking.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover, in the presence of PROTEIN0 rapid interconversion of monomeric and polymeric forms of PROTEIN1 can be induced by simply changing the pH of the medium.\n",
      "\n",
      "PROTEIN0 PROTEIN1 pH control of PROTEIN0 polymerization by PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover, the virus-encoded PROTEIN0 homolog was not required for PROTEIN1 events, including intracellular virus movement to the periphery of the cell, formation of specialized microvilli, or release of mature virions, as shown by electron microscopy and yields of infectious intra- and extracellular virus.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Most previous studies have focused on PROTEIN0 itself rather than its complex with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Motile areas of leech neurites are rich in microfilaments and two PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Motile areas of leech neurites are rich in microfilaments and two PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Motile areas of leech neurites are rich in microfilaments and two PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The colocalization of PROTEIN0 and PROTEIN1 in motile, microfilament-rich areas supports the hypothesis that they synergistically regulate the PROTEIN2 dynamics that underlie neurite growth.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The colocalization of PROTEIN0 and PROTEIN1 in motile, microfilament-rich areas supports the hypothesis that they synergistically regulate the PROTEIN2 dynamics that underlie neurite growth.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The colocalization of PROTEIN0 and PROTEIN1 in motile, microfilament-rich areas supports the hypothesis that they synergistically regulate the PROTEIN2 dynamics that underlie neurite growth.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mutants of PROTEIN0 and PROTEIN1 which encode PROTEIN2 and PROTEIN3 respectively, display disorganized PROTEIN4 patches in all cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Mutants of PROTEIN0 and PROTEIN1 which encode PROTEIN2 and PROTEIN3 respectively, display disorganized PROTEIN4 patches in all cells.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Mutants of PROTEIN0 and PROTEIN1 which encode PROTEIN2 and PROTEIN3 respectively, display disorganized PROTEIN4 patches in all cells.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Mutants of PROTEIN0 and PROTEIN1 which encode PROTEIN2 and PROTEIN3 respectively, display disorganized PROTEIN4 patches in all cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Mutants of PROTEIN0 and PROTEIN1 which encode PROTEIN2 and PROTEIN3 respectively, display disorganized PROTEIN4 patches in all cells.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Mutants of PROTEIN0 and PROTEIN1 which encode PROTEIN2 and PROTEIN3 respectively, display disorganized PROTEIN4 patches in all cells.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Mutants of PROTEIN0 and PROTEIN1 which encode PROTEIN2 and PROTEIN3 respectively, display disorganized PROTEIN4 patches in all cells.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Mutants of PROTEIN0 and PROTEIN1 which encode PROTEIN2 and PROTEIN3 respectively, display disorganized PROTEIN4 patches in all cells.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Mutants of PROTEIN0 and PROTEIN1 which encode PROTEIN2 and PROTEIN3 respectively, display disorganized PROTEIN4 patches in all cells.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Mutants of PROTEIN0 and PROTEIN1 which encode PROTEIN2 and PROTEIN3 respectively, display disorganized PROTEIN4 patches in all cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mutations in PROTEIN0 and PROTEIN1 the two yeast PROTEIN2 homologs, cause very similar growth phenotypes, a depolarized PROTEIN3 cytoskeleton, and a defect in the internalization step of endocytosis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Mutations in PROTEIN0 and PROTEIN1 the two yeast PROTEIN2 homologs, cause very similar growth phenotypes, a depolarized PROTEIN3 cytoskeleton, and a defect in the internalization step of endocytosis.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Mutations in PROTEIN0 and PROTEIN1 the two yeast PROTEIN2 homologs, cause very similar growth phenotypes, a depolarized PROTEIN3 cytoskeleton, and a defect in the internalization step of endocytosis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Mutations in PROTEIN0 and PROTEIN1 the two yeast PROTEIN2 homologs, cause very similar growth phenotypes, a depolarized PROTEIN3 cytoskeleton, and a defect in the internalization step of endocytosis.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Mutations in PROTEIN0 and PROTEIN1 the two yeast PROTEIN2 homologs, cause very similar growth phenotypes, a depolarized PROTEIN3 cytoskeleton, and a defect in the internalization step of endocytosis.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Mutations in PROTEIN0 and PROTEIN1 the two yeast PROTEIN2 homologs, cause very similar growth phenotypes, a depolarized PROTEIN3 cytoskeleton, and a defect in the internalization step of endocytosis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN7 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN8 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN9 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN0 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN7 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN8 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN9 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN0 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN1 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN2 PROTEIN7 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN2 PROTEIN8 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN2 PROTEIN9 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN2 PROTEIN0 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN2 PROTEIN1 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN2 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN3 PROTEIN7 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN3 PROTEIN8 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN3 PROTEIN9 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN3 PROTEIN0 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN3 PROTEIN1 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN3 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN3 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN4 PROTEIN7 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN4 PROTEIN8 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN4 PROTEIN9 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN4 PROTEIN0 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN4 PROTEIN1 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN4 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN4 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN5 PROTEIN7 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN5 PROTEIN8 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN5 PROTEIN9 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN5 PROTEIN0 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN5 PROTEIN1 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN5 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN5 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN6 PROTEIN7 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN6 PROTEIN8 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN6 PROTEIN9 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN6 PROTEIN0 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN6 PROTEIN1 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN6 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN6 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN7 PROTEIN8 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN7 PROTEIN9 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN7 PROTEIN0 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN7 PROTEIN1 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN7 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN7 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN8 PROTEIN9 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN8 PROTEIN0 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN8 PROTEIN1 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN8 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN8 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN9 PROTEIN0 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN9 PROTEIN1 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN9 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN9 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Mutations in Saccharomyces cerevisiae PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 increased the rate of genome rearrangements up to 200-fold whereas mutations in PROTEIN9 PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 had little effect.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mutations that alter putative RNA-binding residues of either PROTEIN0 RRM are lethal in vivo, but do not prevent binding to PROTEIN1 in vitro, suggesting that the predicted RNA-binding surfaces of PROTEIN2 are not required for interaction with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Mutations that alter putative RNA-binding residues of either PROTEIN0 RRM are lethal in vivo, but do not prevent binding to PROTEIN1 in vitro, suggesting that the predicted RNA-binding surfaces of PROTEIN2 are not required for interaction with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Mutations that alter putative RNA-binding residues of either PROTEIN0 RRM are lethal in vivo, but do not prevent binding to PROTEIN1 in vitro, suggesting that the predicted RNA-binding surfaces of PROTEIN2 are not required for interaction with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Mutations that alter putative RNA-binding residues of either PROTEIN0 RRM are lethal in vivo, but do not prevent binding to PROTEIN1 in vitro, suggesting that the predicted RNA-binding surfaces of PROTEIN2 are not required for interaction with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Mutations that alter putative RNA-binding residues of either PROTEIN0 RRM are lethal in vivo, but do not prevent binding to PROTEIN1 in vitro, suggesting that the predicted RNA-binding surfaces of PROTEIN2 are not required for interaction with PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Mutations that alter putative RNA-binding residues of either PROTEIN0 RRM are lethal in vivo, but do not prevent binding to PROTEIN1 in vitro, suggesting that the predicted RNA-binding surfaces of PROTEIN2 are not required for interaction with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Recombinant PROTEIN0 protein has a general RNA-binding activity that does not require PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 shows homology to the head domains and the coiledcoil tail of the conventional PROTEIN1 and carries putative binding sites for ATP and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 shows homology to the head domains and the coiledcoil tail of the conventional PROTEIN1 and carries putative binding sites for ATP and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 shows homology to the head domains and the coiledcoil tail of the conventional PROTEIN1 and carries putative binding sites for ATP and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 mRNAs were detected precociously, before the young myotube stage.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Neither can PROTEIN0 in the presence of the peptide, promote PROTEIN1 polymerization during its early phase consistent with a lower affinity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The resulting PROTEIN0 complex overcomes the combined capacity of PROTEIN1 and PROTEIN2 to inhibit PROTEIN3 nucleation and restores the extent of filament formation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The resulting PROTEIN0 complex overcomes the combined capacity of PROTEIN1 and PROTEIN2 to inhibit PROTEIN3 nucleation and restores the extent of filament formation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The resulting PROTEIN0 complex overcomes the combined capacity of PROTEIN1 and PROTEIN2 to inhibit PROTEIN3 nucleation and restores the extent of filament formation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The resulting PROTEIN0 complex overcomes the combined capacity of PROTEIN1 and PROTEIN2 to inhibit PROTEIN3 nucleation and restores the extent of filament formation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The resulting PROTEIN0 complex overcomes the combined capacity of PROTEIN1 and PROTEIN2 to inhibit PROTEIN3 nucleation and restores the extent of filament formation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The resulting PROTEIN0 complex overcomes the combined capacity of PROTEIN1 and PROTEIN2 to inhibit PROTEIN3 nucleation and restores the extent of filament formation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 N-formyl peptide chemoattractants in neutrophils stimulate the formation of phosphatidylinositol-4,5-bisphosphate (PIP2), a reservoir for second messenger molecules and regulator of PROTEIN0 assembly through its association with the PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 N-formyl peptide chemoattractants in neutrophils stimulate the formation of phosphatidylinositol-4,5-bisphosphate (PIP2), a reservoir for second messenger molecules and regulator of PROTEIN0 assembly through its association with the PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 N-formyl peptide chemoattractants in neutrophils stimulate the formation of phosphatidylinositol-4,5-bisphosphate (PIP2), a reservoir for second messenger molecules and regulator of PROTEIN0 assembly through its association with the PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 N-formyl peptide chemoattractants in neutrophils stimulate the formation of phosphatidylinositol-4,5-bisphosphate (PIP2), a reservoir for second messenger molecules and regulator of PROTEIN0 assembly through its association with the PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 N-formyl peptide chemoattractants in neutrophils stimulate the formation of phosphatidylinositol-4,5-bisphosphate (PIP2), a reservoir for second messenger molecules and regulator of PROTEIN0 assembly through its association with the PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 N-formyl peptide chemoattractants in neutrophils stimulate the formation of phosphatidylinositol-4,5-bisphosphate (PIP2), a reservoir for second messenger molecules and regulator of PROTEIN0 assembly through its association with the PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 No significant changes in glomerular PROTEIN0 PROTEIN1 PROTEIN2 or total PROTEIN3 expression occurred at any time point during disease development.\n",
      "\n",
      "PROTEIN0 PROTEIN2 No significant changes in glomerular PROTEIN0 PROTEIN1 PROTEIN2 or total PROTEIN3 expression occurred at any time point during disease development.\n",
      "\n",
      "PROTEIN0 PROTEIN3 No significant changes in glomerular PROTEIN0 PROTEIN1 PROTEIN2 or total PROTEIN3 expression occurred at any time point during disease development.\n",
      "\n",
      "PROTEIN1 PROTEIN2 No significant changes in glomerular PROTEIN0 PROTEIN1 PROTEIN2 or total PROTEIN3 expression occurred at any time point during disease development.\n",
      "\n",
      "PROTEIN1 PROTEIN3 No significant changes in glomerular PROTEIN0 PROTEIN1 PROTEIN2 or total PROTEIN3 expression occurred at any time point during disease development.\n",
      "\n",
      "PROTEIN2 PROTEIN3 No significant changes in glomerular PROTEIN0 PROTEIN1 PROTEIN2 or total PROTEIN3 expression occurred at any time point during disease development.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Notably, the two pro-apoptotic adapter proteins PROTEIN0 and PROTEIN1 are also involved in the activation of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Notably, the two pro-apoptotic adapter proteins PROTEIN0 and PROTEIN1 are also involved in the activation of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Notably, the two pro-apoptotic adapter proteins PROTEIN0 and PROTEIN1 are also involved in the activation of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN0 PROTEIN2 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN0 PROTEIN3 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN0 PROTEIN4 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN0 PROTEIN5 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN0 PROTEIN6 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN0 PROTEIN7 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN0 PROTEIN8 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN1 PROTEIN2 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN1 PROTEIN3 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN1 PROTEIN4 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN1 PROTEIN5 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN1 PROTEIN6 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN1 PROTEIN7 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN1 PROTEIN8 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN2 PROTEIN3 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN2 PROTEIN4 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN2 PROTEIN5 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN2 PROTEIN6 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN2 PROTEIN7 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN2 PROTEIN8 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN3 PROTEIN4 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN3 PROTEIN5 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN3 PROTEIN6 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN3 PROTEIN7 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN3 PROTEIN8 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN4 PROTEIN5 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN4 PROTEIN6 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN4 PROTEIN7 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN4 PROTEIN8 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN5 PROTEIN6 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN5 PROTEIN7 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN5 PROTEIN8 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN6 PROTEIN7 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN6 PROTEIN8 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN7 PROTEIN8 OBJECTIVE: To determine whether other PROTEIN0 PROTEIN1 binding PROTEIN2 can mimic the effects of PROTEIN3 PROTEIN4 in acting synergistically with PROTEIN5 PROTEIN6 to induce cartilage PROTEIN7 breakdown and PROTEIN8 expression, and to determine which receptors mediate these effects.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Of the five major structural proteins of MV, only PROTEIN0 _ PROTEIN1 and PROTEIN2 PROTEIN3 were consistently detected in diseased brain areas.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Of the five major structural proteins of MV, only PROTEIN0 _ PROTEIN1 and PROTEIN2 PROTEIN3 were consistently detected in diseased brain areas.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Of the five major structural proteins of MV, only PROTEIN0 _ PROTEIN1 and PROTEIN2 PROTEIN3 were consistently detected in diseased brain areas.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Of the five major structural proteins of MV, only PROTEIN0 _ PROTEIN1 and PROTEIN2 PROTEIN3 were consistently detected in diseased brain areas.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Of the five major structural proteins of MV, only PROTEIN0 _ PROTEIN1 and PROTEIN2 PROTEIN3 were consistently detected in diseased brain areas.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Of the five major structural proteins of MV, only PROTEIN0 _ PROTEIN1 and PROTEIN2 PROTEIN3 were consistently detected in diseased brain areas.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Once the PROTEIN0 monomer is bound to the filament, the PROTEIN1 is released, and is available to bind to additional PROTEIN2 monomers.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Once the PROTEIN0 monomer is bound to the filament, the PROTEIN1 is released, and is available to bind to additional PROTEIN2 monomers.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Once the PROTEIN0 monomer is bound to the filament, the PROTEIN1 is released, and is available to bind to additional PROTEIN2 monomers.\n",
      "\n",
      "PROTEIN0 PROTEIN1 One involves PROTEIN0 which has recently been shown to bind PROTEIN1 directly.\n",
      "\n",
      "PROTEIN0 PROTEIN1 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN2 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN3 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN4 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN5 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN6 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN7 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN8 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN9 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN0 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN2 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN3 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN4 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN5 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN6 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN7 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN8 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN9 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN1 PROTEIN0 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN3 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN4 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN5 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN6 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN7 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN8 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN9 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN2 PROTEIN0 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN3 PROTEIN4 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN3 PROTEIN5 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN3 PROTEIN6 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN3 PROTEIN7 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN3 PROTEIN8 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN3 PROTEIN9 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN3 PROTEIN0 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN4 PROTEIN5 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN4 PROTEIN6 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN4 PROTEIN7 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN4 PROTEIN8 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN4 PROTEIN9 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN4 PROTEIN0 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN5 PROTEIN6 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN5 PROTEIN7 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN5 PROTEIN8 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN5 PROTEIN9 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN5 PROTEIN0 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN6 PROTEIN7 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN6 PROTEIN8 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN6 PROTEIN9 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN6 PROTEIN0 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN7 PROTEIN8 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN7 PROTEIN9 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN7 PROTEIN0 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN8 PROTEIN9 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN8 PROTEIN0 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN9 PROTEIN0 One, PROTEIN0 (also called PROTEIN1 binds to PROTEIN2 PROTEIN3 but not to PROTEIN4 another, PROTEIN5 binds to the PROTEIN6 PROTEIN7 but not to PROTEIN8 PROTEIN9 and the third, PROTEIN0 binds weakly to both receptors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 One such protein is the yeast PROTEIN0 PROTEIN1 which is homologous to vertebrate PROTEIN2 (Adams, A. E. M., D. Botstein, and D. G. Drubin. 1991. Nature (Lond.). 354:404-408.).\n",
      "\n",
      "PROTEIN0 PROTEIN2 One such protein is the yeast PROTEIN0 PROTEIN1 which is homologous to vertebrate PROTEIN2 (Adams, A. E. M., D. Botstein, and D. G. Drubin. 1991. Nature (Lond.). 354:404-408.).\n",
      "\n",
      "PROTEIN1 PROTEIN2 One such protein is the yeast PROTEIN0 PROTEIN1 which is homologous to vertebrate PROTEIN2 (Adams, A. E. M., D. Botstein, and D. G. Drubin. 1991. Nature (Lond.). 354:404-408.).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Only the unphosphorylated form of PROTEIN0 is an active form that binds PROTEIN1 whereas the regulatory mechanisms of PROTEIN2 have not been elucidated.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Only the unphosphorylated form of PROTEIN0 is an active form that binds PROTEIN1 whereas the regulatory mechanisms of PROTEIN2 have not been elucidated.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Only the unphosphorylated form of PROTEIN0 is an active form that binds PROTEIN1 whereas the regulatory mechanisms of PROTEIN2 have not been elucidated.\n",
      "\n",
      "PROTEIN0 PROTEIN1 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN0 PROTEIN2 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN0 PROTEIN3 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN0 PROTEIN4 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN0 PROTEIN5 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN0 PROTEIN6 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN0 PROTEIN7 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN1 PROTEIN2 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN1 PROTEIN3 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN1 PROTEIN4 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN1 PROTEIN5 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN1 PROTEIN6 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN1 PROTEIN7 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN2 PROTEIN3 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN2 PROTEIN4 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN2 PROTEIN5 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN2 PROTEIN6 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN2 PROTEIN7 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN3 PROTEIN4 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN3 PROTEIN5 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN3 PROTEIN6 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN3 PROTEIN7 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN4 PROTEIN5 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN4 PROTEIN6 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN4 PROTEIN7 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN5 PROTEIN6 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN5 PROTEIN7 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN6 PROTEIN7 On the basis of these results, we present a model for prereplicative site formation in infected cells in which the PROTEIN0 components PROTEIN1 PROTEIN2 and PROTEIN3 the PROTEIN4 PROTEIN5 and the viral PROTEIN6 PROTEIN7 assemble together to initiate the process.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN5 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN1 PROTEIN5 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN2 PROTEIN5 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN3 PROTEIN5 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN4 PROTEIN5 We observed that four replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are necessary for the localization of PROTEIN4 PROTEIN5 to prereplicative sites natural infection conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 On the internal (cytoplasmic) side of focal contacts, several proteins, including PROTEIN0 and PROTEIN1 mediate interactions with the PROTEIN2 filament bundles of the cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN2 On the internal (cytoplasmic) side of focal contacts, several proteins, including PROTEIN0 and PROTEIN1 mediate interactions with the PROTEIN2 filament bundles of the cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN2 On the internal (cytoplasmic) side of focal contacts, several proteins, including PROTEIN0 and PROTEIN1 mediate interactions with the PROTEIN2 filament bundles of the cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN1 On the other hand, we have recently proposed the importance of PROTEIN0 an PROTEIN1 phosphoprotein, in phagocyte functions through dephosphorylation and translocation to the plasma membrane regions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Opposite effects of PROTEIN0 and PROTEIN1 from porcine brain on rate of exchange of PROTEIN2 adenosine 5'-triphosphate.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Opposite effects of PROTEIN0 and PROTEIN1 from porcine brain on rate of exchange of PROTEIN2 adenosine 5'-triphosphate.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Opposite effects of PROTEIN0 and PROTEIN1 from porcine brain on rate of exchange of PROTEIN2 adenosine 5'-triphosphate.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Other mammalian PROTEIN0 such as PROTEIN1 and PROTEIN2 but also PROTEIN3 from Physarum polycephalum are similar targets for PIP2-stimulated PROTEIN4 PROTEIN5 phosphorylation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 cell injury exhibited proteolysis of the PROTEIN1 protein substrate PROTEIN2 without appreciable breakdown of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 cell injury exhibited proteolysis of the PROTEIN1 protein substrate PROTEIN2 without appreciable breakdown of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 cell injury exhibited proteolysis of the PROTEIN1 protein substrate PROTEIN2 without appreciable breakdown of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 cell injury exhibited proteolysis of the PROTEIN1 protein substrate PROTEIN2 without appreciable breakdown of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 cell injury exhibited proteolysis of the PROTEIN1 protein substrate PROTEIN2 without appreciable breakdown of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 cell injury exhibited proteolysis of the PROTEIN1 protein substrate PROTEIN2 without appreciable breakdown of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Other than monomeric PROTEIN0 no major PROTEIN1 ligands are detected in crude extracts.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our results show that the major pool of polymerizable PROTEIN0 monomers is complexed with PROTEIN1 and spread throughout the cytoplasm.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Selective monoclonal antibodies confirm that most of the PROTEIN0 is bound to PROTEIN1 65% in extract immunoadsorption assays and 74-89% by fluorescent antibody staining.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our data, which suggest that pollen PROTEIN0 can regulate PROTEIN1 cytoskeletal protein assembly and PROTEIN2 or PROTEIN3 activity, indicate a possible role for the involvement of PROTEIN4 in signaling pathways that may regulate pollen tube growth.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Our data, which suggest that pollen PROTEIN0 can regulate PROTEIN1 cytoskeletal protein assembly and PROTEIN2 or PROTEIN3 activity, indicate a possible role for the involvement of PROTEIN4 in signaling pathways that may regulate pollen tube growth.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Our data, which suggest that pollen PROTEIN0 can regulate PROTEIN1 cytoskeletal protein assembly and PROTEIN2 or PROTEIN3 activity, indicate a possible role for the involvement of PROTEIN4 in signaling pathways that may regulate pollen tube growth.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Our data, which suggest that pollen PROTEIN0 can regulate PROTEIN1 cytoskeletal protein assembly and PROTEIN2 or PROTEIN3 activity, indicate a possible role for the involvement of PROTEIN4 in signaling pathways that may regulate pollen tube growth.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Our data, which suggest that pollen PROTEIN0 can regulate PROTEIN1 cytoskeletal protein assembly and PROTEIN2 or PROTEIN3 activity, indicate a possible role for the involvement of PROTEIN4 in signaling pathways that may regulate pollen tube growth.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Our data, which suggest that pollen PROTEIN0 can regulate PROTEIN1 cytoskeletal protein assembly and PROTEIN2 or PROTEIN3 activity, indicate a possible role for the involvement of PROTEIN4 in signaling pathways that may regulate pollen tube growth.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Our data, which suggest that pollen PROTEIN0 can regulate PROTEIN1 cytoskeletal protein assembly and PROTEIN2 or PROTEIN3 activity, indicate a possible role for the involvement of PROTEIN4 in signaling pathways that may regulate pollen tube growth.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Our data, which suggest that pollen PROTEIN0 can regulate PROTEIN1 cytoskeletal protein assembly and PROTEIN2 or PROTEIN3 activity, indicate a possible role for the involvement of PROTEIN4 in signaling pathways that may regulate pollen tube growth.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Our data, which suggest that pollen PROTEIN0 can regulate PROTEIN1 cytoskeletal protein assembly and PROTEIN2 or PROTEIN3 activity, indicate a possible role for the involvement of PROTEIN4 in signaling pathways that may regulate pollen tube growth.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Our data, which suggest that pollen PROTEIN0 can regulate PROTEIN1 cytoskeletal protein assembly and PROTEIN2 or PROTEIN3 activity, indicate a possible role for the involvement of PROTEIN4 in signaling pathways that may regulate pollen tube growth.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our in vitro recombinant protein-protein interaction studies demonstrated that PROTEIN0 could directly interact with PROTEIN1 and PROTEIN2 but not with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Our in vitro recombinant protein-protein interaction studies demonstrated that PROTEIN0 could directly interact with PROTEIN1 and PROTEIN2 but not with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Our in vitro recombinant protein-protein interaction studies demonstrated that PROTEIN0 could directly interact with PROTEIN1 and PROTEIN2 but not with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Our in vitro recombinant protein-protein interaction studies demonstrated that PROTEIN0 could directly interact with PROTEIN1 and PROTEIN2 but not with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Our in vitro recombinant protein-protein interaction studies demonstrated that PROTEIN0 could directly interact with PROTEIN1 and PROTEIN2 but not with PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Our in vitro recombinant protein-protein interaction studies demonstrated that PROTEIN0 could directly interact with PROTEIN1 and PROTEIN2 but not with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 a PROTEIN3 induces G1 arrest and can inhibit the activity of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Particle aggregates associated with this array of PROTEIN0 microfilaments also labeled with antibodies to PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Particle aggregates associated with this array of PROTEIN0 microfilaments also labeled with antibodies to PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Particle aggregates associated with this array of PROTEIN0 microfilaments also labeled with antibodies to PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Particle aggregates associated with this array of PROTEIN0 microfilaments also labeled with antibodies to PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Particle aggregates associated with this array of PROTEIN0 microfilaments also labeled with antibodies to PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Particle aggregates associated with this array of PROTEIN0 microfilaments also labeled with antibodies to PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 expression is a better indicator of clinical outcome than alterations in the expression of PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 expression is a better indicator of clinical outcome than alterations in the expression of PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 expression is a better indicator of clinical outcome than alterations in the expression of PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 expression is a better indicator of clinical outcome than alterations in the expression of PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 expression is a better indicator of clinical outcome than alterations in the expression of PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 expression is a better indicator of clinical outcome than alterations in the expression of PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 expression is a better indicator of clinical outcome than alterations in the expression of PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 expression is a better indicator of clinical outcome than alterations in the expression of PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 expression is a better indicator of clinical outcome than alterations in the expression of PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 expression is a better indicator of clinical outcome than alterations in the expression of PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 Pervanadate caused a dramatic augmentation of the phosphorylation of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 but PROTEIN4 was not detectably phosphorylated.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Pervanadate caused a dramatic augmentation of the phosphorylation of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 but PROTEIN4 was not detectably phosphorylated.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Pervanadate caused a dramatic augmentation of the phosphorylation of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 but PROTEIN4 was not detectably phosphorylated.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Pervanadate caused a dramatic augmentation of the phosphorylation of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 but PROTEIN4 was not detectably phosphorylated.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Pervanadate caused a dramatic augmentation of the phosphorylation of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 but PROTEIN4 was not detectably phosphorylated.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Pervanadate caused a dramatic augmentation of the phosphorylation of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 but PROTEIN4 was not detectably phosphorylated.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Pervanadate caused a dramatic augmentation of the phosphorylation of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 but PROTEIN4 was not detectably phosphorylated.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Pervanadate caused a dramatic augmentation of the phosphorylation of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 but PROTEIN4 was not detectably phosphorylated.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Pervanadate caused a dramatic augmentation of the phosphorylation of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 but PROTEIN4 was not detectably phosphorylated.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Pervanadate caused a dramatic augmentation of the phosphorylation of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 but PROTEIN4 was not detectably phosphorylated.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Phosphoinositides bind to PROTEIN0 and regulate PROTEIN1 cytoskeletal protein assembly.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Phosphorylation of PROTEIN0 by PROTEIN1 was not affected by the presence of various concentrations of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Phosphorylation of PROTEIN0 by PROTEIN1 was not affected by the presence of various concentrations of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Phosphorylation of PROTEIN0 by PROTEIN1 was not affected by the presence of various concentrations of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Phosphorylation of the headpiece domain could regulate the PROTEIN0 binding and bundling properties of PROTEIN1 or it could regulate the interaction of PROTEIN2 with other proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Phosphorylation of the headpiece domain could regulate the PROTEIN0 binding and bundling properties of PROTEIN1 or it could regulate the interaction of PROTEIN2 with other proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Phosphorylation of the headpiece domain could regulate the PROTEIN0 binding and bundling properties of PROTEIN1 or it could regulate the interaction of PROTEIN2 with other proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Photochemical cleavage of PROTEIN0 and the effect on the interaction with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Polyacrylamide gels of platelets heated at 45 degrees C for 90 minutes showed an increase in PROTEIN0 incorporation into heated platelet cytoskeletons but no increase in filamentous PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Previously, we have shown that in the absence of PROTEIN0 repair is nearly absent and diploid cells lose the broken chromosome; however, in cells lacking PROTEIN1 gene conversion is absent but cells can repair the DSB by BIR.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Previous studies have yielded conflicting results concerning the physiological role of PROTEIN0 a 12-15-kD PROTEIN1 and phosphoinositide-binding protein, as a regulator of PROTEIN2 polymerization.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Previous studies have yielded conflicting results concerning the physiological role of PROTEIN0 a 12-15-kD PROTEIN1 and phosphoinositide-binding protein, as a regulator of PROTEIN2 polymerization.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Previous studies have yielded conflicting results concerning the physiological role of PROTEIN0 a 12-15-kD PROTEIN1 and phosphoinositide-binding protein, as a regulator of PROTEIN2 polymerization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Production of PIP and PIP2 may be important downstream signals since these polyphosphoinositides are able to regulate the interaction of PROTEIN0 and PROTEIN1 with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Production of PIP and PIP2 may be important downstream signals since these polyphosphoinositides are able to regulate the interaction of PROTEIN0 and PROTEIN1 with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Production of PIP and PIP2 may be important downstream signals since these polyphosphoinositides are able to regulate the interaction of PROTEIN0 and PROTEIN1 with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 have similar affinities for PtdIns(4,5)P2 and poly(L-proline), and both accelerate nucleotide exchange on monomeric PROTEIN2 to the same extent.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 and PROTEIN1 have similar affinities for PtdIns(4,5)P2 and poly(L-proline), and both accelerate nucleotide exchange on monomeric PROTEIN2 to the same extent.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 and PROTEIN1 have similar affinities for PtdIns(4,5)P2 and poly(L-proline), and both accelerate nucleotide exchange on monomeric PROTEIN2 to the same extent.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binds PROTEIN1 with a similar affinity to that of PROTEIN2 although it inhibits PROTEIN3 polymerization more strongly than PROTEIN4 under non-equilibrium conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 binds PROTEIN1 with a similar affinity to that of PROTEIN2 although it inhibits PROTEIN3 polymerization more strongly than PROTEIN4 under non-equilibrium conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 binds PROTEIN1 with a similar affinity to that of PROTEIN2 although it inhibits PROTEIN3 polymerization more strongly than PROTEIN4 under non-equilibrium conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 binds PROTEIN1 with a similar affinity to that of PROTEIN2 although it inhibits PROTEIN3 polymerization more strongly than PROTEIN4 under non-equilibrium conditions.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 binds PROTEIN1 with a similar affinity to that of PROTEIN2 although it inhibits PROTEIN3 polymerization more strongly than PROTEIN4 under non-equilibrium conditions.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 binds PROTEIN1 with a similar affinity to that of PROTEIN2 although it inhibits PROTEIN3 polymerization more strongly than PROTEIN4 under non-equilibrium conditions.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 binds PROTEIN1 with a similar affinity to that of PROTEIN2 although it inhibits PROTEIN3 polymerization more strongly than PROTEIN4 under non-equilibrium conditions.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 binds PROTEIN1 with a similar affinity to that of PROTEIN2 although it inhibits PROTEIN3 polymerization more strongly than PROTEIN4 under non-equilibrium conditions.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 binds PROTEIN1 with a similar affinity to that of PROTEIN2 although it inhibits PROTEIN3 polymerization more strongly than PROTEIN4 under non-equilibrium conditions.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 binds PROTEIN1 with a similar affinity to that of PROTEIN2 although it inhibits PROTEIN3 polymerization more strongly than PROTEIN4 under non-equilibrium conditions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Since functional characteristics for PROTEIN0 are lacking, we assayed the PROTEIN1 the phosphatidylinositol 4,5-bisphosphate and the poly(L-proline) binding properties of this isoform.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is generally thought to regulate PROTEIN1 polymerization, but the observation that acidic phospholipids dissociate the complex of PROTEIN2 and PROTEIN3 raised the possibility that PROTEIN4 might also regulate lipid metabolism.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is generally thought to regulate PROTEIN1 polymerization, but the observation that acidic phospholipids dissociate the complex of PROTEIN2 and PROTEIN3 raised the possibility that PROTEIN4 might also regulate lipid metabolism.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 is generally thought to regulate PROTEIN1 polymerization, but the observation that acidic phospholipids dissociate the complex of PROTEIN2 and PROTEIN3 raised the possibility that PROTEIN4 might also regulate lipid metabolism.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 is generally thought to regulate PROTEIN1 polymerization, but the observation that acidic phospholipids dissociate the complex of PROTEIN2 and PROTEIN3 raised the possibility that PROTEIN4 might also regulate lipid metabolism.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is generally thought to regulate PROTEIN1 polymerization, but the observation that acidic phospholipids dissociate the complex of PROTEIN2 and PROTEIN3 raised the possibility that PROTEIN4 might also regulate lipid metabolism.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 is generally thought to regulate PROTEIN1 polymerization, but the observation that acidic phospholipids dissociate the complex of PROTEIN2 and PROTEIN3 raised the possibility that PROTEIN4 might also regulate lipid metabolism.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 is generally thought to regulate PROTEIN1 polymerization, but the observation that acidic phospholipids dissociate the complex of PROTEIN2 and PROTEIN3 raised the possibility that PROTEIN4 might also regulate lipid metabolism.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 is generally thought to regulate PROTEIN1 polymerization, but the observation that acidic phospholipids dissociate the complex of PROTEIN2 and PROTEIN3 raised the possibility that PROTEIN4 might also regulate lipid metabolism.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 is generally thought to regulate PROTEIN1 polymerization, but the observation that acidic phospholipids dissociate the complex of PROTEIN2 and PROTEIN3 raised the possibility that PROTEIN4 might also regulate lipid metabolism.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 is generally thought to regulate PROTEIN1 polymerization, but the observation that acidic phospholipids dissociate the complex of PROTEIN2 and PROTEIN3 raised the possibility that PROTEIN4 might also regulate lipid metabolism.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 binds to PIP2 and inhibits its hydrolysis by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 PROTEIN1 binds to PIP2 and inhibits its hydrolysis by PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 PROTEIN1 binds to PIP2 and inhibits its hydrolysis by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The cellular concentrations and binding characteristics of these molecules are consistent with PROTEIN0 being a negative regulator of the phosphoinositide signaling pathway in addition to its established function as an inhibitor of PROTEIN1 polymerization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 also bind polyphosphoinositides, which can disrupt the PROTEIN1 PROTEIN2 complex, and proline-rich ligands which localize PROTEIN3 to sites requiring extensive PROTEIN4 filament accumulation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 also bind polyphosphoinositides, which can disrupt the PROTEIN1 PROTEIN2 complex, and proline-rich ligands which localize PROTEIN3 to sites requiring extensive PROTEIN4 filament accumulation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 also bind polyphosphoinositides, which can disrupt the PROTEIN1 PROTEIN2 complex, and proline-rich ligands which localize PROTEIN3 to sites requiring extensive PROTEIN4 filament accumulation.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 also bind polyphosphoinositides, which can disrupt the PROTEIN1 PROTEIN2 complex, and proline-rich ligands which localize PROTEIN3 to sites requiring extensive PROTEIN4 filament accumulation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 also bind polyphosphoinositides, which can disrupt the PROTEIN1 PROTEIN2 complex, and proline-rich ligands which localize PROTEIN3 to sites requiring extensive PROTEIN4 filament accumulation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 also bind polyphosphoinositides, which can disrupt the PROTEIN1 PROTEIN2 complex, and proline-rich ligands which localize PROTEIN3 to sites requiring extensive PROTEIN4 filament accumulation.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 also bind polyphosphoinositides, which can disrupt the PROTEIN1 PROTEIN2 complex, and proline-rich ligands which localize PROTEIN3 to sites requiring extensive PROTEIN4 filament accumulation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 also bind polyphosphoinositides, which can disrupt the PROTEIN1 PROTEIN2 complex, and proline-rich ligands which localize PROTEIN3 to sites requiring extensive PROTEIN4 filament accumulation.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 also bind polyphosphoinositides, which can disrupt the PROTEIN1 PROTEIN2 complex, and proline-rich ligands which localize PROTEIN3 to sites requiring extensive PROTEIN4 filament accumulation.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 also bind polyphosphoinositides, which can disrupt the PROTEIN1 PROTEIN2 complex, and proline-rich ligands which localize PROTEIN3 to sites requiring extensive PROTEIN4 filament accumulation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Proteolytic activity was not detected with PROTEIN0 PROTEIN1 or PROTEIN2 substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Proteolytic activity was not detected with PROTEIN0 PROTEIN1 or PROTEIN2 substrates.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Proteolytic activity was not detected with PROTEIN0 PROTEIN1 or PROTEIN2 substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 fragments form complexes with PROTEIN1 PROTEIN2 and PROTEIN3 all components of adherens junctions.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 fragments form complexes with PROTEIN1 PROTEIN2 and PROTEIN3 all components of adherens junctions.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 fragments form complexes with PROTEIN1 PROTEIN2 and PROTEIN3 all components of adherens junctions.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 fragments form complexes with PROTEIN1 PROTEIN2 and PROTEIN3 all components of adherens junctions.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 fragments form complexes with PROTEIN1 PROTEIN2 and PROTEIN3 all components of adherens junctions.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 fragments form complexes with PROTEIN1 PROTEIN2 and PROTEIN3 all components of adherens junctions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PtK2 cells of exceptionally large size were microinjected with fluorescently labeled probes for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in order to follow the assembly of the contractile proteins into the cleavage furrows.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PtK2 cells of exceptionally large size were microinjected with fluorescently labeled probes for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in order to follow the assembly of the contractile proteins into the cleavage furrows.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PtK2 cells of exceptionally large size were microinjected with fluorescently labeled probes for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in order to follow the assembly of the contractile proteins into the cleavage furrows.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PtK2 cells of exceptionally large size were microinjected with fluorescently labeled probes for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in order to follow the assembly of the contractile proteins into the cleavage furrows.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PtK2 cells of exceptionally large size were microinjected with fluorescently labeled probes for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in order to follow the assembly of the contractile proteins into the cleavage furrows.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PtK2 cells of exceptionally large size were microinjected with fluorescently labeled probes for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in order to follow the assembly of the contractile proteins into the cleavage furrows.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The presence of PROTEIN0 in the cleavage furrows also suggests the possibility of an overlapping mechanism in addition to that of a PROTEIN1 mediated mechanism for the attachment of PROTEIN2 filaments to the cell surfaces in the cleavage furrow.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The presence of PROTEIN0 in the cleavage furrows also suggests the possibility of an overlapping mechanism in addition to that of a PROTEIN1 mediated mechanism for the attachment of PROTEIN2 filaments to the cell surfaces in the cleavage furrow.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The presence of PROTEIN0 in the cleavage furrows also suggests the possibility of an overlapping mechanism in addition to that of a PROTEIN1 mediated mechanism for the attachment of PROTEIN2 filaments to the cell surfaces in the cleavage furrow.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Pyrenyliodoacetamide labeling of cysteine 374 of PROTEIN0 reduces the affinity for PROTEIN1 10-fold.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Replacement of serine 38 with cysteine created a unique site where labeling with rhodamine did not alter the affinity of PROTEIN0 for PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Three methods, fluorescence anisotropy of rhodamine-labeled PROTEIN0 intrinsic fluorescence and nucleotide exchange, give the same affinity, Kd = 0.1 microM, for Acanthamoeba PROTEIN1 binding amoeba PROTEIN2 monomers with bound Mg-ATP.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Three methods, fluorescence anisotropy of rhodamine-labeled PROTEIN0 intrinsic fluorescence and nucleotide exchange, give the same affinity, Kd = 0.1 microM, for Acanthamoeba PROTEIN1 binding amoeba PROTEIN2 monomers with bound Mg-ATP.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Three methods, fluorescence anisotropy of rhodamine-labeled PROTEIN0 intrinsic fluorescence and nucleotide exchange, give the same affinity, Kd = 0.1 microM, for Acanthamoeba PROTEIN1 binding amoeba PROTEIN2 monomers with bound Mg-ATP.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 Quantitation of the appearance of PROTEIN0 banding in primary cultures of myotubes indicates that it precedes that of other myofibrillar proteins and that assembly takes place in the following order: PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Recently it has been established, through a detailed biochemical analysis, that recombinant Arabidopsis thaliana PROTEIN0 PROTEIN1 is a member of the PROTEIN2 family of PROTEIN3 filament bundling or cross-linking proteins [D.R. Kovar et al. (2000) Plant J 24:625-636].\n",
      "\n",
      "PROTEIN0 PROTEIN2 Recently it has been established, through a detailed biochemical analysis, that recombinant Arabidopsis thaliana PROTEIN0 PROTEIN1 is a member of the PROTEIN2 family of PROTEIN3 filament bundling or cross-linking proteins [D.R. Kovar et al. (2000) Plant J 24:625-636].\n",
      "\n",
      "PROTEIN0 PROTEIN3 Recently it has been established, through a detailed biochemical analysis, that recombinant Arabidopsis thaliana PROTEIN0 PROTEIN1 is a member of the PROTEIN2 family of PROTEIN3 filament bundling or cross-linking proteins [D.R. Kovar et al. (2000) Plant J 24:625-636].\n",
      "\n",
      "PROTEIN1 PROTEIN2 Recently it has been established, through a detailed biochemical analysis, that recombinant Arabidopsis thaliana PROTEIN0 PROTEIN1 is a member of the PROTEIN2 family of PROTEIN3 filament bundling or cross-linking proteins [D.R. Kovar et al. (2000) Plant J 24:625-636].\n",
      "\n",
      "PROTEIN1 PROTEIN3 Recently it has been established, through a detailed biochemical analysis, that recombinant Arabidopsis thaliana PROTEIN0 PROTEIN1 is a member of the PROTEIN2 family of PROTEIN3 filament bundling or cross-linking proteins [D.R. Kovar et al. (2000) Plant J 24:625-636].\n",
      "\n",
      "PROTEIN2 PROTEIN3 Recently it has been established, through a detailed biochemical analysis, that recombinant Arabidopsis thaliana PROTEIN0 PROTEIN1 is a member of the PROTEIN2 family of PROTEIN3 filament bundling or cross-linking proteins [D.R. Kovar et al. (2000) Plant J 24:625-636].\n",
      "\n",
      "PROTEIN0 PROTEIN1 Recombinant human PROTEIN0 can simultaneously bind to the PROTEIN1 PROTEIN2 complex but does not bind PROTEIN3 directly.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Recombinant human PROTEIN0 can simultaneously bind to the PROTEIN1 PROTEIN2 complex but does not bind PROTEIN3 directly.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Recombinant human PROTEIN0 can simultaneously bind to the PROTEIN1 PROTEIN2 complex but does not bind PROTEIN3 directly.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Recombinant human PROTEIN0 can simultaneously bind to the PROTEIN1 PROTEIN2 complex but does not bind PROTEIN3 directly.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Recombinant human PROTEIN0 can simultaneously bind to the PROTEIN1 PROTEIN2 complex but does not bind PROTEIN3 directly.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Recombinant human PROTEIN0 can simultaneously bind to the PROTEIN1 PROTEIN2 complex but does not bind PROTEIN3 directly.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Recruitment of a PROTEIN0 to PROTEIN1 protrusions in Dictyostelium.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Reduced PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 expression correlated closely with the differentiation grade of the esophageal squamous cell carcinoma (p < 0.05).\n",
      "\n",
      "PROTEIN0 PROTEIN2 Reduced PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 expression correlated closely with the differentiation grade of the esophageal squamous cell carcinoma (p < 0.05).\n",
      "\n",
      "PROTEIN0 PROTEIN3 Reduced PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 expression correlated closely with the differentiation grade of the esophageal squamous cell carcinoma (p < 0.05).\n",
      "\n",
      "PROTEIN1 PROTEIN2 Reduced PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 expression correlated closely with the differentiation grade of the esophageal squamous cell carcinoma (p < 0.05).\n",
      "\n",
      "PROTEIN1 PROTEIN3 Reduced PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 expression correlated closely with the differentiation grade of the esophageal squamous cell carcinoma (p < 0.05).\n",
      "\n",
      "PROTEIN2 PROTEIN3 Reduced PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 expression correlated closely with the differentiation grade of the esophageal squamous cell carcinoma (p < 0.05).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Reduced expression of PROTEIN0 and PROTEIN1 as well as mutations in the PROTEIN2 gene, have been found in various carcinomas, whereas mutations in the PROTEIN3 and PROTEIN4 genes have been described only in carcinoma cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Reduced expression of PROTEIN0 and PROTEIN1 as well as mutations in the PROTEIN2 gene, have been found in various carcinomas, whereas mutations in the PROTEIN3 and PROTEIN4 genes have been described only in carcinoma cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Reduced expression of PROTEIN0 and PROTEIN1 as well as mutations in the PROTEIN2 gene, have been found in various carcinomas, whereas mutations in the PROTEIN3 and PROTEIN4 genes have been described only in carcinoma cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Reduced expression of PROTEIN0 and PROTEIN1 as well as mutations in the PROTEIN2 gene, have been found in various carcinomas, whereas mutations in the PROTEIN3 and PROTEIN4 genes have been described only in carcinoma cell lines.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Reduced expression of PROTEIN0 and PROTEIN1 as well as mutations in the PROTEIN2 gene, have been found in various carcinomas, whereas mutations in the PROTEIN3 and PROTEIN4 genes have been described only in carcinoma cell lines.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Reduced expression of PROTEIN0 and PROTEIN1 as well as mutations in the PROTEIN2 gene, have been found in various carcinomas, whereas mutations in the PROTEIN3 and PROTEIN4 genes have been described only in carcinoma cell lines.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Reduced expression of PROTEIN0 and PROTEIN1 as well as mutations in the PROTEIN2 gene, have been found in various carcinomas, whereas mutations in the PROTEIN3 and PROTEIN4 genes have been described only in carcinoma cell lines.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Reduced expression of PROTEIN0 and PROTEIN1 as well as mutations in the PROTEIN2 gene, have been found in various carcinomas, whereas mutations in the PROTEIN3 and PROTEIN4 genes have been described only in carcinoma cell lines.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Reduced expression of PROTEIN0 and PROTEIN1 as well as mutations in the PROTEIN2 gene, have been found in various carcinomas, whereas mutations in the PROTEIN3 and PROTEIN4 genes have been described only in carcinoma cell lines.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Reduced expression of PROTEIN0 and PROTEIN1 as well as mutations in the PROTEIN2 gene, have been found in various carcinomas, whereas mutations in the PROTEIN3 and PROTEIN4 genes have been described only in carcinoma cell lines.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Regulation of PROTEIN0 and PROTEIN1 isogenes during cardiac growth and hypertrophy.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Removal of one nonhomologous DNA end during gene conversion by a PROTEIN0 and PROTEIN1 pathway.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Requirement of the yeast PROTEIN0 and PROTEIN1 genes for PROTEIN2 genomic stability.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Requirement of the yeast PROTEIN0 and PROTEIN1 genes for PROTEIN2 genomic stability.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Requirement of the yeast PROTEIN0 and PROTEIN1 genes for PROTEIN2 genomic stability.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Results from epistasis analyses indicate that PROTEIN0 functions in mismatch repair in conjunction with PROTEIN1 or PROTEIN2 and that PROTEIN3 and PROTEIN4 constitute alternate pathways of PROTEIN5 mismatch repair.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Requirement of yeast PROTEIN0 for PROTEIN1 organization and morphogenesis in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Resonance energy transfer determination of the following distances, using a hybrid PROTEIN0 those between Cys-55 on the Mercenaria PROTEIN1 SH-1 on the Aequipecten PROTEIN2 and Cys-374 of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Resonance energy transfer determination of the following distances, using a hybrid PROTEIN0 those between Cys-55 on the Mercenaria PROTEIN1 SH-1 on the Aequipecten PROTEIN2 and Cys-374 of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Resonance energy transfer determination of the following distances, using a hybrid PROTEIN0 those between Cys-55 on the Mercenaria PROTEIN1 SH-1 on the Aequipecten PROTEIN2 and Cys-374 of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Resonance energy transfer determination of the following distances, using a hybrid PROTEIN0 those between Cys-55 on the Mercenaria PROTEIN1 SH-1 on the Aequipecten PROTEIN2 and Cys-374 of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Resonance energy transfer determination of the following distances, using a hybrid PROTEIN0 those between Cys-55 on the Mercenaria PROTEIN1 SH-1 on the Aequipecten PROTEIN2 and Cys-374 of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Resonance energy transfer determination of the following distances, using a hybrid PROTEIN0 those between Cys-55 on the Mercenaria PROTEIN1 SH-1 on the Aequipecten PROTEIN2 and Cys-374 of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Reversible binding of PROTEIN0 to PROTEIN1 and PROTEIN2 in human platelet extracts.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Reversible binding of PROTEIN0 to PROTEIN1 and PROTEIN2 in human platelet extracts.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Reversible binding of PROTEIN0 to PROTEIN1 and PROTEIN2 in human platelet extracts.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Sepharose beads coupled to either monoclonal PROTEIN0 antibodies or to polyproline were used to extract PROTEIN1 and PROTEIN2 respectively, from EGTA-containing platelet extracts and determine the proportion of these molecules bound to PROTEIN3 with sufficient affinity to withstand dilution (high-affinity complexes).\n",
      "\n",
      "PROTEIN0 PROTEIN2 Sepharose beads coupled to either monoclonal PROTEIN0 antibodies or to polyproline were used to extract PROTEIN1 and PROTEIN2 respectively, from EGTA-containing platelet extracts and determine the proportion of these molecules bound to PROTEIN3 with sufficient affinity to withstand dilution (high-affinity complexes).\n",
      "\n",
      "PROTEIN0 PROTEIN3 Sepharose beads coupled to either monoclonal PROTEIN0 antibodies or to polyproline were used to extract PROTEIN1 and PROTEIN2 respectively, from EGTA-containing platelet extracts and determine the proportion of these molecules bound to PROTEIN3 with sufficient affinity to withstand dilution (high-affinity complexes).\n",
      "\n",
      "PROTEIN1 PROTEIN2 Sepharose beads coupled to either monoclonal PROTEIN0 antibodies or to polyproline were used to extract PROTEIN1 and PROTEIN2 respectively, from EGTA-containing platelet extracts and determine the proportion of these molecules bound to PROTEIN3 with sufficient affinity to withstand dilution (high-affinity complexes).\n",
      "\n",
      "PROTEIN1 PROTEIN3 Sepharose beads coupled to either monoclonal PROTEIN0 antibodies or to polyproline were used to extract PROTEIN1 and PROTEIN2 respectively, from EGTA-containing platelet extracts and determine the proportion of these molecules bound to PROTEIN3 with sufficient affinity to withstand dilution (high-affinity complexes).\n",
      "\n",
      "PROTEIN2 PROTEIN3 Sepharose beads coupled to either monoclonal PROTEIN0 antibodies or to polyproline were used to extract PROTEIN1 and PROTEIN2 respectively, from EGTA-containing platelet extracts and determine the proportion of these molecules bound to PROTEIN3 with sufficient affinity to withstand dilution (high-affinity complexes).\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 within seconds, caused quantitative conversion of PROTEIN1 and PROTEIN2 to high-affinity complexes with PROTEIN3 but these complexes were not present 5 min after stimulation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 within seconds, caused quantitative conversion of PROTEIN1 and PROTEIN2 to high-affinity complexes with PROTEIN3 but these complexes were not present 5 min after stimulation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 within seconds, caused quantitative conversion of PROTEIN1 and PROTEIN2 to high-affinity complexes with PROTEIN3 but these complexes were not present 5 min after stimulation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 within seconds, caused quantitative conversion of PROTEIN1 and PROTEIN2 to high-affinity complexes with PROTEIN3 but these complexes were not present 5 min after stimulation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 within seconds, caused quantitative conversion of PROTEIN1 and PROTEIN2 to high-affinity complexes with PROTEIN3 but these complexes were not present 5 min after stimulation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 within seconds, caused quantitative conversion of PROTEIN1 and PROTEIN2 to high-affinity complexes with PROTEIN3 but these complexes were not present 5 min after stimulation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 one of the cleavage products, enhanced interaction between PROTEIN1 and PROTEIN2 PROTEIN3 and increased cells' sensitivity to PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 one of the cleavage products, enhanced interaction between PROTEIN1 and PROTEIN2 PROTEIN3 and increased cells' sensitivity to PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 one of the cleavage products, enhanced interaction between PROTEIN1 and PROTEIN2 PROTEIN3 and increased cells' sensitivity to PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 one of the cleavage products, enhanced interaction between PROTEIN1 and PROTEIN2 PROTEIN3 and increased cells' sensitivity to PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 one of the cleavage products, enhanced interaction between PROTEIN1 and PROTEIN2 PROTEIN3 and increased cells' sensitivity to PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 one of the cleavage products, enhanced interaction between PROTEIN1 and PROTEIN2 PROTEIN3 and increased cells' sensitivity to PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 one of the cleavage products, enhanced interaction between PROTEIN1 and PROTEIN2 PROTEIN3 and increased cells' sensitivity to PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 one of the cleavage products, enhanced interaction between PROTEIN1 and PROTEIN2 PROTEIN3 and increased cells' sensitivity to PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 one of the cleavage products, enhanced interaction between PROTEIN1 and PROTEIN2 PROTEIN3 and increased cells' sensitivity to PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 one of the cleavage products, enhanced interaction between PROTEIN1 and PROTEIN2 PROTEIN3 and increased cells' sensitivity to PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 SDS-PAGE analysis of growth cone cytoskeletons revealed the presence of several major bands, identified by their mobility as PROTEIN0 (43 kDa Mr), PROTEIN1 (195 kDa Mr), PROTEIN2 (235 and 240 kDa Mr), and PROTEIN3 (51-54 kDa Mr).\n",
      "\n",
      "PROTEIN0 PROTEIN2 SDS-PAGE analysis of growth cone cytoskeletons revealed the presence of several major bands, identified by their mobility as PROTEIN0 (43 kDa Mr), PROTEIN1 (195 kDa Mr), PROTEIN2 (235 and 240 kDa Mr), and PROTEIN3 (51-54 kDa Mr).\n",
      "\n",
      "PROTEIN0 PROTEIN3 SDS-PAGE analysis of growth cone cytoskeletons revealed the presence of several major bands, identified by their mobility as PROTEIN0 (43 kDa Mr), PROTEIN1 (195 kDa Mr), PROTEIN2 (235 and 240 kDa Mr), and PROTEIN3 (51-54 kDa Mr).\n",
      "\n",
      "PROTEIN1 PROTEIN2 SDS-PAGE analysis of growth cone cytoskeletons revealed the presence of several major bands, identified by their mobility as PROTEIN0 (43 kDa Mr), PROTEIN1 (195 kDa Mr), PROTEIN2 (235 and 240 kDa Mr), and PROTEIN3 (51-54 kDa Mr).\n",
      "\n",
      "PROTEIN1 PROTEIN3 SDS-PAGE analysis of growth cone cytoskeletons revealed the presence of several major bands, identified by their mobility as PROTEIN0 (43 kDa Mr), PROTEIN1 (195 kDa Mr), PROTEIN2 (235 and 240 kDa Mr), and PROTEIN3 (51-54 kDa Mr).\n",
      "\n",
      "PROTEIN2 PROTEIN3 SDS-PAGE analysis of growth cone cytoskeletons revealed the presence of several major bands, identified by their mobility as PROTEIN0 (43 kDa Mr), PROTEIN1 (195 kDa Mr), PROTEIN2 (235 and 240 kDa Mr), and PROTEIN3 (51-54 kDa Mr).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN2 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN3 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN4 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN5 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN6 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN2 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN3 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN4 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN5 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN6 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN3 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN4 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN5 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN6 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN4 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN5 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN6 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN5 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN6 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN5 PROTEIN6 Semiserial sections were subjected to immunohistochemical staining with antibodies to PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 isoforms: PROTEIN4 PROTEIN5 and PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN1 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN0 PROTEIN7 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN0 PROTEIN8 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN0 PROTEIN9 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN1 PROTEIN7 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN1 PROTEIN8 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN1 PROTEIN9 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN2 PROTEIN7 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN2 PROTEIN8 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN2 PROTEIN9 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN3 PROTEIN7 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN3 PROTEIN8 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN3 PROTEIN9 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN4 PROTEIN7 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN4 PROTEIN8 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN4 PROTEIN9 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN5 PROTEIN7 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN5 PROTEIN8 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN5 PROTEIN9 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN6 PROTEIN7 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN6 PROTEIN8 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN6 PROTEIN9 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN7 PROTEIN8 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN7 PROTEIN9 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN8 PROTEIN9 Seventeen embryonic genes were identified as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and nine novel sequences designated as A1-9.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Several intracellular proteins termed PROTEIN0 including PROTEIN1 PROTEIN2 and PROTEIN3 are tightly associated with these PROTEIN4 and serve to link them to the cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Several intracellular proteins termed PROTEIN0 including PROTEIN1 PROTEIN2 and PROTEIN3 are tightly associated with these PROTEIN4 and serve to link them to the cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Several intracellular proteins termed PROTEIN0 including PROTEIN1 PROTEIN2 and PROTEIN3 are tightly associated with these PROTEIN4 and serve to link them to the cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Several intracellular proteins termed PROTEIN0 including PROTEIN1 PROTEIN2 and PROTEIN3 are tightly associated with these PROTEIN4 and serve to link them to the cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Several intracellular proteins termed PROTEIN0 including PROTEIN1 PROTEIN2 and PROTEIN3 are tightly associated with these PROTEIN4 and serve to link them to the cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Several intracellular proteins termed PROTEIN0 including PROTEIN1 PROTEIN2 and PROTEIN3 are tightly associated with these PROTEIN4 and serve to link them to the cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Several intracellular proteins termed PROTEIN0 including PROTEIN1 PROTEIN2 and PROTEIN3 are tightly associated with these PROTEIN4 and serve to link them to the cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Several intracellular proteins termed PROTEIN0 including PROTEIN1 PROTEIN2 and PROTEIN3 are tightly associated with these PROTEIN4 and serve to link them to the cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Several intracellular proteins termed PROTEIN0 including PROTEIN1 PROTEIN2 and PROTEIN3 are tightly associated with these PROTEIN4 and serve to link them to the cytoskeleton.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Several intracellular proteins termed PROTEIN0 including PROTEIN1 PROTEIN2 and PROTEIN3 are tightly associated with these PROTEIN4 and serve to link them to the cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Several proteins, such as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 have previously been described.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Several proteins, such as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 have previously been described.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Several proteins, such as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 have previously been described.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Several proteins, such as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 have previously been described.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Several proteins, such as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 have previously been described.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Several proteins, such as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 have previously been described.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Several proteins, such as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 have previously been described.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Several proteins, such as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 have previously been described.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Several proteins, such as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 have previously been described.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Several proteins, such as PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 have previously been described.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Signaling pathways involved in dephosphorylation and localization of the PROTEIN0 PROTEIN1 in stimulated human neutrophils.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These findings suggest a role of PROTEIN0 in stimulus-dependent PROTEIN1 remodeling in motile neutrophils.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Significantly, those PROTEIN0 mutants exhibiting the most severe phenotypes in all three processes have altered residues that cluster to a small region of the PROTEIN1 crystal structure previously defined as the PROTEIN2 PROTEIN3 site.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Significantly, those PROTEIN0 mutants exhibiting the most severe phenotypes in all three processes have altered residues that cluster to a small region of the PROTEIN1 crystal structure previously defined as the PROTEIN2 PROTEIN3 site.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Significantly, those PROTEIN0 mutants exhibiting the most severe phenotypes in all three processes have altered residues that cluster to a small region of the PROTEIN1 crystal structure previously defined as the PROTEIN2 PROTEIN3 site.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Significantly, those PROTEIN0 mutants exhibiting the most severe phenotypes in all three processes have altered residues that cluster to a small region of the PROTEIN1 crystal structure previously defined as the PROTEIN2 PROTEIN3 site.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Significantly, those PROTEIN0 mutants exhibiting the most severe phenotypes in all three processes have altered residues that cluster to a small region of the PROTEIN1 crystal structure previously defined as the PROTEIN2 PROTEIN3 site.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Significantly, those PROTEIN0 mutants exhibiting the most severe phenotypes in all three processes have altered residues that cluster to a small region of the PROTEIN1 crystal structure previously defined as the PROTEIN2 PROTEIN3 site.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN2 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN3 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN4 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN5 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN6 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN2 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN3 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN4 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN5 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN6 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN3 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN4 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN5 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN6 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN4 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN5 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN6 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN5 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN6 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN5 PROTEIN6 Silencing mediated by PROTEIN0 PROTEIN1 requires the trans-acting factors that normally participate in repression, namely, PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN1 Since PROTEIN0 is required for PROTEIN1 adhesion, the PROTEIN2 repeat region alone probably cannot promote cell adhesion, making it unlikely that PROTEIN3 induces axis duplication by increasing cell adhesion.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Since PROTEIN0 is required for PROTEIN1 adhesion, the PROTEIN2 repeat region alone probably cannot promote cell adhesion, making it unlikely that PROTEIN3 induces axis duplication by increasing cell adhesion.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Since PROTEIN0 is required for PROTEIN1 adhesion, the PROTEIN2 repeat region alone probably cannot promote cell adhesion, making it unlikely that PROTEIN3 induces axis duplication by increasing cell adhesion.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Since PROTEIN0 is required for PROTEIN1 adhesion, the PROTEIN2 repeat region alone probably cannot promote cell adhesion, making it unlikely that PROTEIN3 induces axis duplication by increasing cell adhesion.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Since PROTEIN0 is required for PROTEIN1 adhesion, the PROTEIN2 repeat region alone probably cannot promote cell adhesion, making it unlikely that PROTEIN3 induces axis duplication by increasing cell adhesion.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Since PROTEIN0 is required for PROTEIN1 adhesion, the PROTEIN2 repeat region alone probably cannot promote cell adhesion, making it unlikely that PROTEIN3 induces axis duplication by increasing cell adhesion.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Since PROTEIN0 is a recombinational protein associated with PROTEIN1 this is the first genetic evidence that cancer arises from a defect in repair processes involving homologous recombination.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PROTEIN0 PROTEIN1 PROTEIN0 which consists of a full length PROTEIN1 and a tagged tail, was isolated on the basis of the affinities for Nickel agarose and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 which consists of a full length PROTEIN1 and a tagged tail, was isolated on the basis of the affinities for Nickel agarose and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 which consists of a full length PROTEIN1 and a tagged tail, was isolated on the basis of the affinities for Nickel agarose and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 expression in the vascular smooth muscle cells surrounding human endometrial arterioles.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 prepared from chicken gizzard binds to PROTEIN1 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Specific antibodies to PROTEIN0 isoforms PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 and cDNAs for PROTEIN6 were used.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Spontaneous and mitomycin C-induced SCE levels were significantly reduced for chicken DT40 B cells lacking the key HR genes PROTEIN0 and PROTEIN1 but not for nonhomologous DNA end-joining (NHEJ)-defective KU70(-/-) cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Stable transfection of mutant PROTEIN0 molecules showed that deletion of the PROTEIN1 binding domain, but not the PROTEIN2 binding domain, abolished PROTEIN3 downregulation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Stable transfection of mutant PROTEIN0 molecules showed that deletion of the PROTEIN1 binding domain, but not the PROTEIN2 binding domain, abolished PROTEIN3 downregulation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Stable transfection of mutant PROTEIN0 molecules showed that deletion of the PROTEIN1 binding domain, but not the PROTEIN2 binding domain, abolished PROTEIN3 downregulation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Stable transfection of mutant PROTEIN0 molecules showed that deletion of the PROTEIN1 binding domain, but not the PROTEIN2 binding domain, abolished PROTEIN3 downregulation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Stable transfection of mutant PROTEIN0 molecules showed that deletion of the PROTEIN1 binding domain, but not the PROTEIN2 binding domain, abolished PROTEIN3 downregulation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Stable transfection of mutant PROTEIN0 molecules showed that deletion of the PROTEIN1 binding domain, but not the PROTEIN2 binding domain, abolished PROTEIN3 downregulation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Structural studies on the ribbon-to-helix transition in PROTEIN0 PROTEIN1 crystals.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Studies on cultured cells have suggested that both PROTEIN0 and PROTEIN1 are important for the adhesive function of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Studies on cultured cells have suggested that both PROTEIN0 and PROTEIN1 are important for the adhesive function of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Studies on cultured cells have suggested that both PROTEIN0 and PROTEIN1 are important for the adhesive function of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Studies on the interaction between PROTEIN0 and PROTEIN1 purified by a new method.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Study of mutated yeast PROTEIN0 and PROTEIN1 from Acanthamoeba suggests that the ability of PROTEIN2 to suppress PROTEIN3 cells is dependent upon a property other than, or in addition to, its ability to bind PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Study of mutated yeast PROTEIN0 and PROTEIN1 from Acanthamoeba suggests that the ability of PROTEIN2 to suppress PROTEIN3 cells is dependent upon a property other than, or in addition to, its ability to bind PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Study of mutated yeast PROTEIN0 and PROTEIN1 from Acanthamoeba suggests that the ability of PROTEIN2 to suppress PROTEIN3 cells is dependent upon a property other than, or in addition to, its ability to bind PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Study of mutated yeast PROTEIN0 and PROTEIN1 from Acanthamoeba suggests that the ability of PROTEIN2 to suppress PROTEIN3 cells is dependent upon a property other than, or in addition to, its ability to bind PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Study of mutated yeast PROTEIN0 and PROTEIN1 from Acanthamoeba suggests that the ability of PROTEIN2 to suppress PROTEIN3 cells is dependent upon a property other than, or in addition to, its ability to bind PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Study of mutated yeast PROTEIN0 and PROTEIN1 from Acanthamoeba suggests that the ability of PROTEIN2 to suppress PROTEIN3 cells is dependent upon a property other than, or in addition to, its ability to bind PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Study of mutated yeast PROTEIN0 and PROTEIN1 from Acanthamoeba suggests that the ability of PROTEIN2 to suppress PROTEIN3 cells is dependent upon a property other than, or in addition to, its ability to bind PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Study of mutated yeast PROTEIN0 and PROTEIN1 from Acanthamoeba suggests that the ability of PROTEIN2 to suppress PROTEIN3 cells is dependent upon a property other than, or in addition to, its ability to bind PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Study of mutated yeast PROTEIN0 and PROTEIN1 from Acanthamoeba suggests that the ability of PROTEIN2 to suppress PROTEIN3 cells is dependent upon a property other than, or in addition to, its ability to bind PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Study of mutated yeast PROTEIN0 and PROTEIN1 from Acanthamoeba suggests that the ability of PROTEIN2 to suppress PROTEIN3 cells is dependent upon a property other than, or in addition to, its ability to bind PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 Suppression of cell transformation by the PROTEIN0 PROTEIN1 requires binding to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Suppression of cell transformation by the PROTEIN0 PROTEIN1 requires binding to PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Suppression of cell transformation by the PROTEIN0 PROTEIN1 requires binding to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Surprisingly, although maintained in mitogen-rich medium, this ectopic expression was associated with a transactivation of the endogenous PROTEIN0 and PROTEIN1 gene but not the endogenous PROTEIN2 PROTEIN3 PROTEIN4 the PROTEIN5 or the PROTEIN6 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 inhibition also led to the formation of less prominent focal adhesions as revealed by immunofluorescence staining against PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 inhibition also led to the formation of less prominent focal adhesions as revealed by immunofluorescence staining against PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 inhibition also led to the formation of less prominent focal adhesions as revealed by immunofluorescence staining against PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 inhibition also led to the formation of less prominent focal adhesions as revealed by immunofluorescence staining against PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 inhibition also led to the formation of less prominent focal adhesions as revealed by immunofluorescence staining against PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 inhibition also led to the formation of less prominent focal adhesions as revealed by immunofluorescence staining against PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 is a PROTEIN2 highly expressed in testicular germ cells and has the potential to phosphorylate PROTEIN3 and induce PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 is a PROTEIN2 highly expressed in testicular germ cells and has the potential to phosphorylate PROTEIN3 and induce PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 is a PROTEIN2 highly expressed in testicular germ cells and has the potential to phosphorylate PROTEIN3 and induce PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 PROTEIN1 is a PROTEIN2 highly expressed in testicular germ cells and has the potential to phosphorylate PROTEIN3 and induce PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 is a PROTEIN2 highly expressed in testicular germ cells and has the potential to phosphorylate PROTEIN3 and induce PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 is a PROTEIN2 highly expressed in testicular germ cells and has the potential to phosphorylate PROTEIN3 and induce PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 PROTEIN1 is a PROTEIN2 highly expressed in testicular germ cells and has the potential to phosphorylate PROTEIN3 and induce PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 is a PROTEIN2 highly expressed in testicular germ cells and has the potential to phosphorylate PROTEIN3 and induce PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 PROTEIN1 is a PROTEIN2 highly expressed in testicular germ cells and has the potential to phosphorylate PROTEIN3 and induce PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 PROTEIN1 is a PROTEIN2 highly expressed in testicular germ cells and has the potential to phosphorylate PROTEIN3 and induce PROTEIN4 cytoskeletal reorganization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 was isolated from Tetrahymena thermophila by affinity chromatography, and the peptide sequence was determined for part of the protein.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 bundle formation induced by overexpression of PROTEIN1 is accompanied with an increase in cell motility of Dictyostelium cells. 4.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 cytoskeleton-associated protein PROTEIN1 binds to PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 cytoskeleton-associated protein PROTEIN1 binds to PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 cytoskeleton-associated protein PROTEIN1 binds to PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 The PROTEIN0 cytoskeleton-associated protein PROTEIN1 binds to PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 cytoskeleton-associated protein PROTEIN1 binds to PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 cytoskeleton-associated protein PROTEIN1 binds to PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 The PROTEIN0 cytoskeleton-associated protein PROTEIN1 binds to PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 cytoskeleton-associated protein PROTEIN1 binds to PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 The PROTEIN0 cytoskeleton-associated protein PROTEIN1 binds to PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 The PROTEIN0 cytoskeleton-associated protein PROTEIN1 binds to PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The PROTEIN0 molecule links PROTEIN1 PROTEIN2 or PROTEIN3 PROTEIN4 complexes to the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The ability of PROTEIN0 to replace PROTEIN1 is dependent on its WH2 PROTEIN2 binding domain and a putative PROTEIN3 binding domain.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The ability of PROTEIN0 to replace PROTEIN1 is dependent on its WH2 PROTEIN2 binding domain and a putative PROTEIN3 binding domain.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The ability of PROTEIN0 to replace PROTEIN1 is dependent on its WH2 PROTEIN2 binding domain and a putative PROTEIN3 binding domain.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The ability of PROTEIN0 to replace PROTEIN1 is dependent on its WH2 PROTEIN2 binding domain and a putative PROTEIN3 binding domain.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The ability of PROTEIN0 to replace PROTEIN1 is dependent on its WH2 PROTEIN2 binding domain and a putative PROTEIN3 binding domain.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The ability of PROTEIN0 to replace PROTEIN1 is dependent on its WH2 PROTEIN2 binding domain and a putative PROTEIN3 binding domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, like PROTEIN0 PROTEIN1 functions in yeast to link the polarity development pathway and the PROTEIN2 cytoskeleton to generate cytoskeletal asymmetry.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, like PROTEIN0 PROTEIN1 functions in yeast to link the polarity development pathway and the PROTEIN2 cytoskeleton to generate cytoskeletal asymmetry.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, like PROTEIN0 PROTEIN1 functions in yeast to link the polarity development pathway and the PROTEIN2 cytoskeleton to generate cytoskeletal asymmetry.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 binding domain of PROTEIN1 is sufficient to contact to PROTEIN2 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 binding domain of PROTEIN1 is sufficient to contact to PROTEIN2 in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 binding domain of PROTEIN1 is sufficient to contact to PROTEIN2 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The activation of the PROTEIN0 PROTEIN1 protein leads to an accumulation of high levels of PROTEIN2 protein and a repression of the PROTEIN3 PROTEIN4 under all culture conditions tested.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The activation of the PROTEIN0 PROTEIN1 protein leads to an accumulation of high levels of PROTEIN2 protein and a repression of the PROTEIN3 PROTEIN4 under all culture conditions tested.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The activation of the PROTEIN0 PROTEIN1 protein leads to an accumulation of high levels of PROTEIN2 protein and a repression of the PROTEIN3 PROTEIN4 under all culture conditions tested.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The activation of the PROTEIN0 PROTEIN1 protein leads to an accumulation of high levels of PROTEIN2 protein and a repression of the PROTEIN3 PROTEIN4 under all culture conditions tested.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The activation of the PROTEIN0 PROTEIN1 protein leads to an accumulation of high levels of PROTEIN2 protein and a repression of the PROTEIN3 PROTEIN4 under all culture conditions tested.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The activation of the PROTEIN0 PROTEIN1 protein leads to an accumulation of high levels of PROTEIN2 protein and a repression of the PROTEIN3 PROTEIN4 under all culture conditions tested.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The activation of the PROTEIN0 PROTEIN1 protein leads to an accumulation of high levels of PROTEIN2 protein and a repression of the PROTEIN3 PROTEIN4 under all culture conditions tested.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The activation of the PROTEIN0 PROTEIN1 protein leads to an accumulation of high levels of PROTEIN2 protein and a repression of the PROTEIN3 PROTEIN4 under all culture conditions tested.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The activation of the PROTEIN0 PROTEIN1 protein leads to an accumulation of high levels of PROTEIN2 protein and a repression of the PROTEIN3 PROTEIN4 under all culture conditions tested.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The activation of the PROTEIN0 PROTEIN1 protein leads to an accumulation of high levels of PROTEIN2 protein and a repression of the PROTEIN3 PROTEIN4 under all culture conditions tested.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The adventitial tissue underlying PTCA-induced lesions temporarily expressed PROTEIN0 PROTEIN1 and PROTEIN2 isoforms, but not PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The adventitial tissue underlying PTCA-induced lesions temporarily expressed PROTEIN0 PROTEIN1 and PROTEIN2 isoforms, but not PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The adventitial tissue underlying PTCA-induced lesions temporarily expressed PROTEIN0 PROTEIN1 and PROTEIN2 isoforms, but not PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The adventitial tissue underlying PTCA-induced lesions temporarily expressed PROTEIN0 PROTEIN1 and PROTEIN2 isoforms, but not PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The adventitial tissue underlying PTCA-induced lesions temporarily expressed PROTEIN0 PROTEIN1 and PROTEIN2 isoforms, but not PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The adventitial tissue underlying PTCA-induced lesions temporarily expressed PROTEIN0 PROTEIN1 and PROTEIN2 isoforms, but not PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The amount of unassembled PROTEIN0 (12 microM) is accounted for by the sequestering functions of PROTEIN1 (20 microM) and PROTEIN2 (5 microM), the barbed ends being capped.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The amount of unassembled PROTEIN0 (12 microM) is accounted for by the sequestering functions of PROTEIN1 (20 microM) and PROTEIN2 (5 microM), the barbed ends being capped.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The amount of unassembled PROTEIN0 (12 microM) is accounted for by the sequestering functions of PROTEIN1 (20 microM) and PROTEIN2 (5 microM), the barbed ends being capped.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 binding site on PROTEIN1 may be discontinuous since neither the carboxyl- nor amino-terminal halves of PROTEIN2 will independently associate with PROTEIN3 although the amino-terminal half independently binds PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 binding site on PROTEIN1 may be discontinuous since neither the carboxyl- nor amino-terminal halves of PROTEIN2 will independently associate with PROTEIN3 although the amino-terminal half independently binds PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 binding site on PROTEIN1 may be discontinuous since neither the carboxyl- nor amino-terminal halves of PROTEIN2 will independently associate with PROTEIN3 although the amino-terminal half independently binds PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 The PROTEIN0 binding site on PROTEIN1 may be discontinuous since neither the carboxyl- nor amino-terminal halves of PROTEIN2 will independently associate with PROTEIN3 although the amino-terminal half independently binds PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 binding site on PROTEIN1 may be discontinuous since neither the carboxyl- nor amino-terminal halves of PROTEIN2 will independently associate with PROTEIN3 although the amino-terminal half independently binds PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 binding site on PROTEIN1 may be discontinuous since neither the carboxyl- nor amino-terminal halves of PROTEIN2 will independently associate with PROTEIN3 although the amino-terminal half independently binds PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 The PROTEIN0 binding site on PROTEIN1 may be discontinuous since neither the carboxyl- nor amino-terminal halves of PROTEIN2 will independently associate with PROTEIN3 although the amino-terminal half independently binds PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 binding site on PROTEIN1 may be discontinuous since neither the carboxyl- nor amino-terminal halves of PROTEIN2 will independently associate with PROTEIN3 although the amino-terminal half independently binds PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 The PROTEIN0 binding site on PROTEIN1 may be discontinuous since neither the carboxyl- nor amino-terminal halves of PROTEIN2 will independently associate with PROTEIN3 although the amino-terminal half independently binds PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 The PROTEIN0 binding site on PROTEIN1 may be discontinuous since neither the carboxyl- nor amino-terminal halves of PROTEIN2 will independently associate with PROTEIN3 although the amino-terminal half independently binds PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 protein and PROTEIN1 form similar but independent complexes with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 protein and PROTEIN1 form similar but independent complexes with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 protein and PROTEIN1 form similar but independent complexes with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 The PROTEIN0 protein and PROTEIN1 form similar but independent complexes with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 protein and PROTEIN1 form similar but independent complexes with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 protein and PROTEIN1 form similar but independent complexes with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 The PROTEIN0 protein and PROTEIN1 form similar but independent complexes with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 protein and PROTEIN1 form similar but independent complexes with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 The PROTEIN0 protein and PROTEIN1 form similar but independent complexes with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 The PROTEIN0 protein and PROTEIN1 form similar but independent complexes with PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Thus, PROTEIN0 forms distinct heteromeric complexes containing combinations of PROTEIN1 PROTEIN2 and PROTEIN3 which are independent from the PROTEIN4 PROTEIN5 complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN0 PROTEIN6 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN1 PROTEIN6 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN2 PROTEIN6 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN3 PROTEIN6 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN4 PROTEIN6 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN5 PROTEIN6 The assembly of complete HSV replication compartments and incorporation of BrdU were both abolished by treatment with phosphonoacetic acid (PAA) and by omission of any one of the seven viral replication proteins, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 that are essential for viral DNA replication.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 component of the herpes simplex virus PROTEIN1 complex stimulates primer synthesis by a subassembly of the PROTEIN2 and PROTEIN3 components.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 component of the herpes simplex virus PROTEIN1 complex stimulates primer synthesis by a subassembly of the PROTEIN2 and PROTEIN3 components.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 component of the herpes simplex virus PROTEIN1 complex stimulates primer synthesis by a subassembly of the PROTEIN2 and PROTEIN3 components.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 component of the herpes simplex virus PROTEIN1 complex stimulates primer synthesis by a subassembly of the PROTEIN2 and PROTEIN3 components.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 component of the herpes simplex virus PROTEIN1 complex stimulates primer synthesis by a subassembly of the PROTEIN2 and PROTEIN3 components.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 component of the herpes simplex virus PROTEIN1 complex stimulates primer synthesis by a subassembly of the PROTEIN2 and PROTEIN3 components.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The herpes simplex virus type 1 (HSV) PROTEIN0 PROTEIN1 and PROTEIN2 proteins form a helicase-primase complex in infected cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The herpes simplex virus type 1 (HSV) PROTEIN0 PROTEIN1 and PROTEIN2 proteins form a helicase-primase complex in infected cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The herpes simplex virus type 1 (HSV) PROTEIN0 PROTEIN1 and PROTEIN2 proteins form a helicase-primase complex in infected cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 region is distinct from major PROTEIN1 mAb epitopes on the human PROTEIN2 PROTEIN3 molecule.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 region is distinct from major PROTEIN1 mAb epitopes on the human PROTEIN2 PROTEIN3 molecule.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 region is distinct from major PROTEIN1 mAb epitopes on the human PROTEIN2 PROTEIN3 molecule.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 region is distinct from major PROTEIN1 mAb epitopes on the human PROTEIN2 PROTEIN3 molecule.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 region is distinct from major PROTEIN1 mAb epitopes on the human PROTEIN2 PROTEIN3 molecule.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 region is distinct from major PROTEIN1 mAb epitopes on the human PROTEIN2 PROTEIN3 molecule.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The binding surfaces for segment 1 and PROTEIN0 are different, although they peripherally overlap on PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 This review describes three structures of PROTEIN0 complexed with different monomer-binding proteins, namely with PROTEIN1 PROTEIN2 segment 1, and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 This review describes three structures of PROTEIN0 complexed with different monomer-binding proteins, namely with PROTEIN1 PROTEIN2 segment 1, and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 This review describes three structures of PROTEIN0 complexed with different monomer-binding proteins, namely with PROTEIN1 PROTEIN2 segment 1, and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 This review describes three structures of PROTEIN0 complexed with different monomer-binding proteins, namely with PROTEIN1 PROTEIN2 segment 1, and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 This review describes three structures of PROTEIN0 complexed with different monomer-binding proteins, namely with PROTEIN1 PROTEIN2 segment 1, and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 This review describes three structures of PROTEIN0 complexed with different monomer-binding proteins, namely with PROTEIN1 PROTEIN2 segment 1, and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 contains the PROTEIN2 and ATP-binding sites and represents the molecular motor of muscle contraction.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 PROTEIN1 contains the PROTEIN2 and ATP-binding sites and represents the molecular motor of muscle contraction.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 PROTEIN1 contains the PROTEIN2 and ATP-binding sites and represents the molecular motor of muscle contraction.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 and PROTEIN1 genes are involved in genetic recombination and double-strand break repair in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The C374S mutation had the most pronounced effect; it reduced the polymerizability of the PROTEIN0 abolished its binding to PROTEIN1 and filaments containing this mutation moved at reduced rates in the in vitro 'motility assay'.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 Arg-403-->Gln mutation that causes hypertrophic cardiomyopathy does not affect the PROTEIN1 or ATP-binding capacities of two size-limited recombinant PROTEIN2 fragments.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 Arg-403-->Gln mutation that causes hypertrophic cardiomyopathy does not affect the PROTEIN1 or ATP-binding capacities of two size-limited recombinant PROTEIN2 fragments.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 Arg-403-->Gln mutation that causes hypertrophic cardiomyopathy does not affect the PROTEIN1 or ATP-binding capacities of two size-limited recombinant PROTEIN2 fragments.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The cellular morphological changes were analyzed and correlated with the distribution of cell-substratum contacts viewed by confocal images obtained after immunostaining with antibodies raised against the PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The cellular morphological changes were analyzed and correlated with the distribution of cell-substratum contacts viewed by confocal images obtained after immunostaining with antibodies raised against the PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The cellular morphological changes were analyzed and correlated with the distribution of cell-substratum contacts viewed by confocal images obtained after immunostaining with antibodies raised against the PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The cellular morphological changes were analyzed and correlated with the distribution of cell-substratum contacts viewed by confocal images obtained after immunostaining with antibodies raised against the PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PROTEIN1 PROTEIN3 The cellular morphological changes were analyzed and correlated with the distribution of cell-substratum contacts viewed by confocal images obtained after immunostaining with antibodies raised against the PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The cellular morphological changes were analyzed and correlated with the distribution of cell-substratum contacts viewed by confocal images obtained after immunostaining with antibodies raised against the PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The cellular transcription co-activators PROTEIN0 and the PROTEIN1 PROTEIN2 are cellular targets for transformation by the PROTEIN3 proteins of non-oncogenic adenovirus 5 (Ad5).\n",
      "\n",
      "PROTEIN0 PROTEIN2 The cellular transcription co-activators PROTEIN0 and the PROTEIN1 PROTEIN2 are cellular targets for transformation by the PROTEIN3 proteins of non-oncogenic adenovirus 5 (Ad5).\n",
      "\n",
      "PROTEIN0 PROTEIN3 The cellular transcription co-activators PROTEIN0 and the PROTEIN1 PROTEIN2 are cellular targets for transformation by the PROTEIN3 proteins of non-oncogenic adenovirus 5 (Ad5).\n",
      "\n",
      "PROTEIN1 PROTEIN2 The cellular transcription co-activators PROTEIN0 and the PROTEIN1 PROTEIN2 are cellular targets for transformation by the PROTEIN3 proteins of non-oncogenic adenovirus 5 (Ad5).\n",
      "\n",
      "PROTEIN1 PROTEIN3 The cellular transcription co-activators PROTEIN0 and the PROTEIN1 PROTEIN2 are cellular targets for transformation by the PROTEIN3 proteins of non-oncogenic adenovirus 5 (Ad5).\n",
      "\n",
      "PROTEIN2 PROTEIN3 The cellular transcription co-activators PROTEIN0 and the PROTEIN1 PROTEIN2 are cellular targets for transformation by the PROTEIN3 proteins of non-oncogenic adenovirus 5 (Ad5).\n",
      "\n",
      "PROTEIN0 PROTEIN1 The common denominator is impaired PROTEIN0 association with either PROTEIN1 (PaTuII) or PROTEIN2 (BxPc3 and T3M4).\n",
      "\n",
      "PROTEIN0 PROTEIN2 The common denominator is impaired PROTEIN0 association with either PROTEIN1 (PaTuII) or PROTEIN2 (BxPc3 and T3M4).\n",
      "\n",
      "PROTEIN1 PROTEIN2 The common denominator is impaired PROTEIN0 association with either PROTEIN1 (PaTuII) or PROTEIN2 (BxPc3 and T3M4).\n",
      "\n",
      "PROTEIN0 PROTEIN1 The comparison of PROTEIN0 proteins synthesized by E62 and E24 cell lines revealed no structural or functional differences because both localized at cell-cell contacts and associated with PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The comparison of PROTEIN0 proteins synthesized by E62 and E24 cell lines revealed no structural or functional differences because both localized at cell-cell contacts and associated with PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The comparison of PROTEIN0 proteins synthesized by E62 and E24 cell lines revealed no structural or functional differences because both localized at cell-cell contacts and associated with PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The comparison of PROTEIN0 proteins synthesized by E62 and E24 cell lines revealed no structural or functional differences because both localized at cell-cell contacts and associated with PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The comparison of PROTEIN0 proteins synthesized by E62 and E24 cell lines revealed no structural or functional differences because both localized at cell-cell contacts and associated with PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The comparison of PROTEIN0 proteins synthesized by E62 and E24 cell lines revealed no structural or functional differences because both localized at cell-cell contacts and associated with PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The concentration of PROTEIN0 was constant during the follicular phase, whereas the content of PROTEIN1 decreased in granulosa cells after treatment with diethylstilboestrol or hCG.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 were present in most ovarian cells at all stages of folliculogenesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The C-terminal region of PROTEIN0 was, however, necessary to bring PROTEIN1 and PROTEIN2 into the core complex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The C-terminal region of PROTEIN0 was, however, necessary to bring PROTEIN1 and PROTEIN2 into the core complex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The C-terminal region of PROTEIN0 was, however, necessary to bring PROTEIN1 and PROTEIN2 into the core complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The cytoplasmic domain of PROTEIN0 cannot directly associate with PROTEIN1 but interacts with high affinity with PROTEIN2 whereas the binding of PROTEIN3 PROTEIN4 to PROTEIN5 is less efficient.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 assemble into a 1:1 heterodimeric complex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The deduced peptide sequence contained a nuclear localization signal and a putative PROTEIN0 sequence as reported in PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The domain consists of a mixed alpha-helical/beta-sheet fold that resembles a circular permutation of the PROTEIN0 binding protein, PROTEIN1 and the PROTEIN2 family of regulatory modules.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The domain consists of a mixed alpha-helical/beta-sheet fold that resembles a circular permutation of the PROTEIN0 binding protein, PROTEIN1 and the PROTEIN2 family of regulatory modules.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The domain consists of a mixed alpha-helical/beta-sheet fold that resembles a circular permutation of the PROTEIN0 binding protein, PROTEIN1 and the PROTEIN2 family of regulatory modules.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The domains in PROTEIN0 that are involved in PROTEIN1 binding and transcriptional activation are highly related to the adenoviral PROTEIN2 cellular protein PROTEIN3 (refs 2, 3), and to two hypothetical proteins from Caenorhabditis elegans, PROTEIN4 and PROTEIN5 (refs 4 and 5, respectively), whose functions are unknown.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The drastic effects of the exogenous PROTEIN0 on PROTEIN1 assembly were suppressed by phosphatidylinositol 4,5-bisphosphate (PIP2).\n",
      "\n",
      "PROTEIN0 PROTEIN1 The previous investigation (Abe et al. (1989) J. Biochem. 106, 696-702) suggested that PROTEIN0 is deeply involved in the regulation of PROTEIN1 assembly in developing skeletal muscle.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 proteins PROTEIN1 and PROTEIN2 were not significantly affected by treatment with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 proteins PROTEIN1 and PROTEIN2 were not significantly affected by treatment with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 proteins PROTEIN1 and PROTEIN2 were not significantly affected by treatment with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 proteins PROTEIN1 and PROTEIN2 were not significantly affected by treatment with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 proteins PROTEIN1 and PROTEIN2 were not significantly affected by treatment with PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 proteins PROTEIN1 and PROTEIN2 were not significantly affected by treatment with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The effects of two adhesion substrates (serum and PROTEIN0 and time in culture on the expression of genes encoding PROTEIN1 PROTEIN2 isoforms and PROTEIN3 were analysed in myocytes isolated from adult rat heart and maintained in serum-free culture.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The effects of two adhesion substrates (serum and PROTEIN0 and time in culture on the expression of genes encoding PROTEIN1 PROTEIN2 isoforms and PROTEIN3 were analysed in myocytes isolated from adult rat heart and maintained in serum-free culture.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The effects of two adhesion substrates (serum and PROTEIN0 and time in culture on the expression of genes encoding PROTEIN1 PROTEIN2 isoforms and PROTEIN3 were analysed in myocytes isolated from adult rat heart and maintained in serum-free culture.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The effects of two adhesion substrates (serum and PROTEIN0 and time in culture on the expression of genes encoding PROTEIN1 PROTEIN2 isoforms and PROTEIN3 were analysed in myocytes isolated from adult rat heart and maintained in serum-free culture.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The effects of two adhesion substrates (serum and PROTEIN0 and time in culture on the expression of genes encoding PROTEIN1 PROTEIN2 isoforms and PROTEIN3 were analysed in myocytes isolated from adult rat heart and maintained in serum-free culture.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The effects of two adhesion substrates (serum and PROTEIN0 and time in culture on the expression of genes encoding PROTEIN1 PROTEIN2 isoforms and PROTEIN3 were analysed in myocytes isolated from adult rat heart and maintained in serum-free culture.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The enzyme showed hardly any activity below 50 degrees C but considerable activity at around 60 degrees C against myofibrils, digesting PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The enzyme showed hardly any activity below 50 degrees C but considerable activity at around 60 degrees C against myofibrils, digesting PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The enzyme showed hardly any activity below 50 degrees C but considerable activity at around 60 degrees C against myofibrils, digesting PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The existence of this pathway helps to explain some of the effects of PROTEIN0 and PROTEIN1 in the control of PROTEIN2 dynamics.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The existence of this pathway helps to explain some of the effects of PROTEIN0 and PROTEIN1 in the control of PROTEIN2 dynamics.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The existence of this pathway helps to explain some of the effects of PROTEIN0 and PROTEIN1 in the control of PROTEIN2 dynamics.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The expression of PROTEIN0 PROTEIN1 and PROTEIN2 also was decreased in HT29 cells, with a translocation of PROTEIN3 into the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The expression of PROTEIN0 PROTEIN1 and PROTEIN2 also was decreased in HT29 cells, with a translocation of PROTEIN3 into the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The expression of PROTEIN0 PROTEIN1 and PROTEIN2 also was decreased in HT29 cells, with a translocation of PROTEIN3 into the nucleus.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The expression of PROTEIN0 PROTEIN1 and PROTEIN2 also was decreased in HT29 cells, with a translocation of PROTEIN3 into the nucleus.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The expression of PROTEIN0 PROTEIN1 and PROTEIN2 also was decreased in HT29 cells, with a translocation of PROTEIN3 into the nucleus.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The expression of PROTEIN0 PROTEIN1 and PROTEIN2 also was decreased in HT29 cells, with a translocation of PROTEIN3 into the nucleus.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The gene for the human PROTEIN0 the transcriptional coactivator PROTEIN1 is included in the RT1 cosmid, and mutations in PROTEIN2 have recently been identified in nondeleted RTS patients.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The gene for the human PROTEIN0 the transcriptional coactivator PROTEIN1 is included in the RT1 cosmid, and mutations in PROTEIN2 have recently been identified in nondeleted RTS patients.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The gene for the human PROTEIN0 the transcriptional coactivator PROTEIN1 is included in the RT1 cosmid, and mutations in PROTEIN2 have recently been identified in nondeleted RTS patients.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The head domain of PROTEIN0 thus binds to PROTEIN1 to form a link to the PROTEIN2 cytoskeleton and can thus regulate PROTEIN3 function.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The head domain of PROTEIN0 thus binds to PROTEIN1 to form a link to the PROTEIN2 cytoskeleton and can thus regulate PROTEIN3 function.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The head domain of PROTEIN0 thus binds to PROTEIN1 to form a link to the PROTEIN2 cytoskeleton and can thus regulate PROTEIN3 function.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The head domain of PROTEIN0 thus binds to PROTEIN1 to form a link to the PROTEIN2 cytoskeleton and can thus regulate PROTEIN3 function.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The head domain of PROTEIN0 thus binds to PROTEIN1 to form a link to the PROTEIN2 cytoskeleton and can thus regulate PROTEIN3 function.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The head domain of PROTEIN0 thus binds to PROTEIN1 to form a link to the PROTEIN2 cytoskeleton and can thus regulate PROTEIN3 function.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The herpes simplex virus type-1 PROTEIN0 is a heterotrimer encoded by the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The herpes simplex virus type-1 PROTEIN0 is a heterotrimer encoded by the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The herpes simplex virus type-1 PROTEIN0 is a heterotrimer encoded by the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The herpes simplex virus type-1 PROTEIN0 is a heterotrimer encoded by the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The herpes simplex virus type-1 PROTEIN0 is a heterotrimer encoded by the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The herpes simplex virus type-1 PROTEIN0 is a heterotrimer encoded by the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The Herpes simplex virus type 1 PROTEIN0 consists of three subunits that are the products of the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The Herpes simplex virus type 1 PROTEIN0 consists of three subunits that are the products of the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The Herpes simplex virus type 1 PROTEIN0 consists of three subunits that are the products of the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The Herpes simplex virus type 1 PROTEIN0 consists of three subunits that are the products of the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The Herpes simplex virus type 1 PROTEIN0 consists of three subunits that are the products of the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The Herpes simplex virus type 1 PROTEIN0 consists of three subunits that are the products of the PROTEIN1 PROTEIN2 and PROTEIN3 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The herpes simplex virus type-1 PROTEIN0 PROTEIN1 and PROTEIN2 genes encode an essential heterotrimeric PROTEIN3 that is responsible for concomitant DNA unwinding and primer synthesis at the viral DNA replication fork.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The herpes simplex virus type-1 PROTEIN0 PROTEIN1 and PROTEIN2 genes encode an essential heterotrimeric PROTEIN3 that is responsible for concomitant DNA unwinding and primer synthesis at the viral DNA replication fork.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The herpes simplex virus type-1 PROTEIN0 PROTEIN1 and PROTEIN2 genes encode an essential heterotrimeric PROTEIN3 that is responsible for concomitant DNA unwinding and primer synthesis at the viral DNA replication fork.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The herpes simplex virus type-1 PROTEIN0 PROTEIN1 and PROTEIN2 genes encode an essential heterotrimeric PROTEIN3 that is responsible for concomitant DNA unwinding and primer synthesis at the viral DNA replication fork.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The herpes simplex virus type-1 PROTEIN0 PROTEIN1 and PROTEIN2 genes encode an essential heterotrimeric PROTEIN3 that is responsible for concomitant DNA unwinding and primer synthesis at the viral DNA replication fork.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The herpes simplex virus type-1 PROTEIN0 PROTEIN1 and PROTEIN2 genes encode an essential heterotrimeric PROTEIN3 that is responsible for concomitant DNA unwinding and primer synthesis at the viral DNA replication fork.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The higher affinity of vaccinia PROTEIN0 for polyphosphoinositides (Kd = 0.2-8.5 microM) than for PROTEIN1 or poly(L-proline) and the concentration of vaccinia PROTEIN2 expressed in infected HeLa cells (approximately 20 microM) suggest that vaccinia PROTEIN3 binds preferentially to PIP and PIP2 in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The higher affinity of vaccinia PROTEIN0 for polyphosphoinositides (Kd = 0.2-8.5 microM) than for PROTEIN1 or poly(L-proline) and the concentration of vaccinia PROTEIN2 expressed in infected HeLa cells (approximately 20 microM) suggest that vaccinia PROTEIN3 binds preferentially to PIP and PIP2 in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The higher affinity of vaccinia PROTEIN0 for polyphosphoinositides (Kd = 0.2-8.5 microM) than for PROTEIN1 or poly(L-proline) and the concentration of vaccinia PROTEIN2 expressed in infected HeLa cells (approximately 20 microM) suggest that vaccinia PROTEIN3 binds preferentially to PIP and PIP2 in vivo.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The higher affinity of vaccinia PROTEIN0 for polyphosphoinositides (Kd = 0.2-8.5 microM) than for PROTEIN1 or poly(L-proline) and the concentration of vaccinia PROTEIN2 expressed in infected HeLa cells (approximately 20 microM) suggest that vaccinia PROTEIN3 binds preferentially to PIP and PIP2 in vivo.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The higher affinity of vaccinia PROTEIN0 for polyphosphoinositides (Kd = 0.2-8.5 microM) than for PROTEIN1 or poly(L-proline) and the concentration of vaccinia PROTEIN2 expressed in infected HeLa cells (approximately 20 microM) suggest that vaccinia PROTEIN3 binds preferentially to PIP and PIP2 in vivo.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The higher affinity of vaccinia PROTEIN0 for polyphosphoinositides (Kd = 0.2-8.5 microM) than for PROTEIN1 or poly(L-proline) and the concentration of vaccinia PROTEIN2 expressed in infected HeLa cells (approximately 20 microM) suggest that vaccinia PROTEIN3 binds preferentially to PIP and PIP2 in vivo.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We expressed in Escherichia coli the vaccinia virus gene for a protein similar to vertebrate PROTEIN0 purified the recombinant viral PROTEIN1 and characterized its interactions with PROTEIN2 and polyphosphoinositides.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We expressed in Escherichia coli the vaccinia virus gene for a protein similar to vertebrate PROTEIN0 purified the recombinant viral PROTEIN1 and characterized its interactions with PROTEIN2 and polyphosphoinositides.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We expressed in Escherichia coli the vaccinia virus gene for a protein similar to vertebrate PROTEIN0 purified the recombinant viral PROTEIN1 and characterized its interactions with PROTEIN2 and polyphosphoinositides.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PROTEIN0 PROTEIN1 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The high recombination levels seen in PROTEIN0 and PROTEIN1 mutants is dependent on the PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 genes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 The hyperproliferative response observed in the gastric mucosa is secondary to this initial insult and is associated with increased expression of PROTEIN0 the PROTEIN1 PROTEIN2 and of PROTEIN3 and decreased expression of the PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 The influence of these PROTEIN0 on cell morphology and the distribution pattern of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was analyzed by light microscopy, conventional electron microscopy, immunofluorescence, and immunogold labeling after lysis-squirting.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The influence of these PROTEIN0 on cell morphology and the distribution pattern of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was analyzed by light microscopy, conventional electron microscopy, immunofluorescence, and immunogold labeling after lysis-squirting.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The influence of these PROTEIN0 on cell morphology and the distribution pattern of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was analyzed by light microscopy, conventional electron microscopy, immunofluorescence, and immunogold labeling after lysis-squirting.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The influence of these PROTEIN0 on cell morphology and the distribution pattern of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was analyzed by light microscopy, conventional electron microscopy, immunofluorescence, and immunogold labeling after lysis-squirting.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The influence of these PROTEIN0 on cell morphology and the distribution pattern of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was analyzed by light microscopy, conventional electron microscopy, immunofluorescence, and immunogold labeling after lysis-squirting.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The influence of these PROTEIN0 on cell morphology and the distribution pattern of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was analyzed by light microscopy, conventional electron microscopy, immunofluorescence, and immunogold labeling after lysis-squirting.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The influence of these PROTEIN0 on cell morphology and the distribution pattern of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was analyzed by light microscopy, conventional electron microscopy, immunofluorescence, and immunogold labeling after lysis-squirting.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The influence of these PROTEIN0 on cell morphology and the distribution pattern of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was analyzed by light microscopy, conventional electron microscopy, immunofluorescence, and immunogold labeling after lysis-squirting.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The influence of these PROTEIN0 on cell morphology and the distribution pattern of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was analyzed by light microscopy, conventional electron microscopy, immunofluorescence, and immunogold labeling after lysis-squirting.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The influence of these PROTEIN0 on cell morphology and the distribution pattern of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 was analyzed by light microscopy, conventional electron microscopy, immunofluorescence, and immunogold labeling after lysis-squirting.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The inhibitory effects of rapamycin on bombesin-stimulated cell cycle progression did not involve accumulation of the PROTEIN0 PROTEIN1 PROTEIN2 but a striking inhibition of the expression of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The inhibitory effects of rapamycin on bombesin-stimulated cell cycle progression did not involve accumulation of the PROTEIN0 PROTEIN1 PROTEIN2 but a striking inhibition of the expression of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The inhibitory effects of rapamycin on bombesin-stimulated cell cycle progression did not involve accumulation of the PROTEIN0 PROTEIN1 PROTEIN2 but a striking inhibition of the expression of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The inhibitory effects of rapamycin on bombesin-stimulated cell cycle progression did not involve accumulation of the PROTEIN0 PROTEIN1 PROTEIN2 but a striking inhibition of the expression of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The inhibitory effects of rapamycin on bombesin-stimulated cell cycle progression did not involve accumulation of the PROTEIN0 PROTEIN1 PROTEIN2 but a striking inhibition of the expression of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The inhibitory effects of rapamycin on bombesin-stimulated cell cycle progression did not involve accumulation of the PROTEIN0 PROTEIN1 PROTEIN2 but a striking inhibition of the expression of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The interactions of PROTEIN0 with PROTEIN1 PROTEIN2 and PROTEIN3 are compared.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The interactions of PROTEIN0 with PROTEIN1 PROTEIN2 and PROTEIN3 are compared.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The interactions of PROTEIN0 with PROTEIN1 PROTEIN2 and PROTEIN3 are compared.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The interactions of PROTEIN0 with PROTEIN1 PROTEIN2 and PROTEIN3 are compared.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The interactions of PROTEIN0 with PROTEIN1 PROTEIN2 and PROTEIN3 are compared.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The interactions of PROTEIN0 with PROTEIN1 PROTEIN2 and PROTEIN3 are compared.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The intracellular molecules PROTEIN0 and PROTEIN1 actively regulate PROTEIN2 motility and participate in the signaling pathways used to steer growth cones.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The intracellular molecules PROTEIN0 and PROTEIN1 actively regulate PROTEIN2 motility and participate in the signaling pathways used to steer growth cones.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The intracellular molecules PROTEIN0 and PROTEIN1 actively regulate PROTEIN2 motility and participate in the signaling pathways used to steer growth cones.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Their abilities to store lipids and express PROTEIN0 intermediate filaments, PROTEIN1 and PROTEIN2 in contractile filaments in vitro illustrate similarities among the pulmonary LF, the hepatic lipocyte, and the contractile interstitial cell, which contribute to the repair reaction in the lung after pulmonary injury.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Their abilities to store lipids and express PROTEIN0 intermediate filaments, PROTEIN1 and PROTEIN2 in contractile filaments in vitro illustrate similarities among the pulmonary LF, the hepatic lipocyte, and the contractile interstitial cell, which contribute to the repair reaction in the lung after pulmonary injury.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Their abilities to store lipids and express PROTEIN0 intermediate filaments, PROTEIN1 and PROTEIN2 in contractile filaments in vitro illustrate similarities among the pulmonary LF, the hepatic lipocyte, and the contractile interstitial cell, which contribute to the repair reaction in the lung after pulmonary injury.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The isolated intestinal microvillus cytoskeleton (core) consists of four major proteins: PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The isolated intestinal microvillus cytoskeleton (core) consists of four major proteins: PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The isolated intestinal microvillus cytoskeleton (core) consists of four major proteins: PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The isolated intestinal microvillus cytoskeleton (core) consists of four major proteins: PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The isolated intestinal microvillus cytoskeleton (core) consists of four major proteins: PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The isolated intestinal microvillus cytoskeleton (core) consists of four major proteins: PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The junctional multidomain protein PROTEIN0 is a binding partner of the PROTEIN1 PROTEIN2 and associates with the PROTEIN3 cytoskeletal regulator PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 The junctional multidomain protein PROTEIN0 is a binding partner of the PROTEIN1 PROTEIN2 and associates with the PROTEIN3 cytoskeletal regulator PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 The junctional multidomain protein PROTEIN0 is a binding partner of the PROTEIN1 PROTEIN2 and associates with the PROTEIN3 cytoskeletal regulator PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 The junctional multidomain protein PROTEIN0 is a binding partner of the PROTEIN1 PROTEIN2 and associates with the PROTEIN3 cytoskeletal regulator PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 The junctional multidomain protein PROTEIN0 is a binding partner of the PROTEIN1 PROTEIN2 and associates with the PROTEIN3 cytoskeletal regulator PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 The junctional multidomain protein PROTEIN0 is a binding partner of the PROTEIN1 PROTEIN2 and associates with the PROTEIN3 cytoskeletal regulator PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 The junctional multidomain protein PROTEIN0 is a binding partner of the PROTEIN1 PROTEIN2 and associates with the PROTEIN3 cytoskeletal regulator PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 The junctional multidomain protein PROTEIN0 is a binding partner of the PROTEIN1 PROTEIN2 and associates with the PROTEIN3 cytoskeletal regulator PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 The junctional multidomain protein PROTEIN0 is a binding partner of the PROTEIN1 PROTEIN2 and associates with the PROTEIN3 cytoskeletal regulator PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 The junctional multidomain protein PROTEIN0 is a binding partner of the PROTEIN1 PROTEIN2 and associates with the PROTEIN3 cytoskeletal regulator PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 To our knowledge, PROTEIN0 is the only integral component in cell-cell junctions discovered thus far that interacts with PROTEIN1 and thus could modulate PROTEIN2 modeling proximal to adhesion complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 To our knowledge, PROTEIN0 is the only integral component in cell-cell junctions discovered thus far that interacts with PROTEIN1 and thus could modulate PROTEIN2 modeling proximal to adhesion complexes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 To our knowledge, PROTEIN0 is the only integral component in cell-cell junctions discovered thus far that interacts with PROTEIN1 and thus could modulate PROTEIN2 modeling proximal to adhesion complexes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The long repeats, which are interdigitated between the PRRs, increased the frequency with which PROTEIN0 motility occurred by a mechanism independent of the PRRs, PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The long repeats, which are interdigitated between the PRRs, increased the frequency with which PROTEIN0 motility occurred by a mechanism independent of the PRRs, PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The long repeats, which are interdigitated between the PRRs, increased the frequency with which PROTEIN0 motility occurred by a mechanism independent of the PRRs, PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The tandem repeat domain in the Listeria monocytogenes PROTEIN0 protein controls the rate of PROTEIN1 motility, the percentage of moving bacteria, and the localization of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The tandem repeat domain in the Listeria monocytogenes PROTEIN0 protein controls the rate of PROTEIN1 motility, the percentage of moving bacteria, and the localization of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The tandem repeat domain in the Listeria monocytogenes PROTEIN0 protein controls the rate of PROTEIN1 motility, the percentage of moving bacteria, and the localization of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The tandem repeat domain in the Listeria monocytogenes PROTEIN0 protein controls the rate of PROTEIN1 motility, the percentage of moving bacteria, and the localization of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The tandem repeat domain in the Listeria monocytogenes PROTEIN0 protein controls the rate of PROTEIN1 motility, the percentage of moving bacteria, and the localization of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The tandem repeat domain in the Listeria monocytogenes PROTEIN0 protein controls the rate of PROTEIN1 motility, the percentage of moving bacteria, and the localization of PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The main inhibitory action of PROTEIN0 a PROTEIN1 PROTEIN2 arises from its binding with the PROTEIN3 PROTEIN4 PROTEIN5 complex that results in G(1)-S arrest.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The majority of the aromatic residues in PROTEIN0 are exposed to solvent and lie in either of two hydrophobic patches, neither of which takes part in an interface with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The microvillus cytoskeleton, isolated from chicken intestinal epithelial cell brush borders, is known to contain five major protein components, the 110,000-dalton polypeptide, PROTEIN0 (95,000 daltons), PROTEIN1 (68,000 daltons), PROTEIN2 (43,000 daltons), and PROTEIN3 (17,000 daltons).\n",
      "\n",
      "PROTEIN0 PROTEIN2 The microvillus cytoskeleton, isolated from chicken intestinal epithelial cell brush borders, is known to contain five major protein components, the 110,000-dalton polypeptide, PROTEIN0 (95,000 daltons), PROTEIN1 (68,000 daltons), PROTEIN2 (43,000 daltons), and PROTEIN3 (17,000 daltons).\n",
      "\n",
      "PROTEIN0 PROTEIN3 The microvillus cytoskeleton, isolated from chicken intestinal epithelial cell brush borders, is known to contain five major protein components, the 110,000-dalton polypeptide, PROTEIN0 (95,000 daltons), PROTEIN1 (68,000 daltons), PROTEIN2 (43,000 daltons), and PROTEIN3 (17,000 daltons).\n",
      "\n",
      "PROTEIN1 PROTEIN2 The microvillus cytoskeleton, isolated from chicken intestinal epithelial cell brush borders, is known to contain five major protein components, the 110,000-dalton polypeptide, PROTEIN0 (95,000 daltons), PROTEIN1 (68,000 daltons), PROTEIN2 (43,000 daltons), and PROTEIN3 (17,000 daltons).\n",
      "\n",
      "PROTEIN1 PROTEIN3 The microvillus cytoskeleton, isolated from chicken intestinal epithelial cell brush borders, is known to contain five major protein components, the 110,000-dalton polypeptide, PROTEIN0 (95,000 daltons), PROTEIN1 (68,000 daltons), PROTEIN2 (43,000 daltons), and PROTEIN3 (17,000 daltons).\n",
      "\n",
      "PROTEIN2 PROTEIN3 The microvillus cytoskeleton, isolated from chicken intestinal epithelial cell brush borders, is known to contain five major protein components, the 110,000-dalton polypeptide, PROTEIN0 (95,000 daltons), PROTEIN1 (68,000 daltons), PROTEIN2 (43,000 daltons), and PROTEIN3 (17,000 daltons).\n",
      "\n",
      "PROTEIN0 PROTEIN1 The monomeric PROTEIN0 PROTEIN1 is a key regulator of PROTEIN2 dynamics in animal cells and it has recently been identified in plants as a pollen allergen.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The monomeric PROTEIN0 PROTEIN1 is a key regulator of PROTEIN2 dynamics in animal cells and it has recently been identified in plants as a pollen allergen.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The monomeric PROTEIN0 PROTEIN1 is a key regulator of PROTEIN2 dynamics in animal cells and it has recently been identified in plants as a pollen allergen.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The most significant finding was the re-expression of PROTEIN0 PROTEIN1 and PROTEIN2 and increased down-regulation of PROTEIN3 in metastatic lesions.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The most significant finding was the re-expression of PROTEIN0 PROTEIN1 and PROTEIN2 and increased down-regulation of PROTEIN3 in metastatic lesions.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The most significant finding was the re-expression of PROTEIN0 PROTEIN1 and PROTEIN2 and increased down-regulation of PROTEIN3 in metastatic lesions.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The most significant finding was the re-expression of PROTEIN0 PROTEIN1 and PROTEIN2 and increased down-regulation of PROTEIN3 in metastatic lesions.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The most significant finding was the re-expression of PROTEIN0 PROTEIN1 and PROTEIN2 and increased down-regulation of PROTEIN3 in metastatic lesions.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The most significant finding was the re-expression of PROTEIN0 PROTEIN1 and PROTEIN2 and increased down-regulation of PROTEIN3 in metastatic lesions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN2 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN3 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN4 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN5 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN6 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN7 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN8 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN9 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN2 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN3 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN4 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN5 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN6 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN7 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN8 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN1 PROTEIN9 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN3 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN4 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN5 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN6 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN7 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN8 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN2 PROTEIN9 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN4 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN5 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN6 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN7 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN8 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN3 PROTEIN9 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN5 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN6 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN7 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN8 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN4 PROTEIN9 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN5 PROTEIN6 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN5 PROTEIN7 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN5 PROTEIN8 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN5 PROTEIN9 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN6 PROTEIN7 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN6 PROTEIN8 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN6 PROTEIN9 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN7 PROTEIN8 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN7 PROTEIN9 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN8 PROTEIN9 The most striking difference was in the relative preference for acetylation of PROTEIN0 PROTEIN1 versus acetylation of PROTEIN2 PROTEIN3 with the purified PROTEIN4 PROTEIN5 PROTEIN6 in nucleosomes was acetylated to a much greater extent than was PROTEIN7 PROTEIN8 whereas the reverse preference was found with the endogenous PROTEIN9\n",
      "\n",
      "PROTEIN0 PROTEIN1 The multifunctional protein PROTEIN0 is one of the most abundant proteins in the cytoplasm and is thought to regulate PROTEIN1 assembly and the phosphoinositide signaling pathway.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN0 PROTEIN6 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN0 PROTEIN7 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN0 PROTEIN8 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN0 PROTEIN9 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN0 PROTEIN0 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN1 PROTEIN6 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN1 PROTEIN7 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN1 PROTEIN8 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN1 PROTEIN9 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN1 PROTEIN0 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN1 PROTEIN1 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN2 PROTEIN6 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN2 PROTEIN7 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN2 PROTEIN8 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN2 PROTEIN9 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN2 PROTEIN0 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN2 PROTEIN1 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN3 PROTEIN6 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN3 PROTEIN7 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN3 PROTEIN8 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN3 PROTEIN9 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN3 PROTEIN0 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN3 PROTEIN1 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN4 PROTEIN6 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN4 PROTEIN7 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN4 PROTEIN8 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN4 PROTEIN9 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN4 PROTEIN0 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN4 PROTEIN1 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN5 PROTEIN6 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN5 PROTEIN7 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN5 PROTEIN8 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN5 PROTEIN9 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN5 PROTEIN0 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN5 PROTEIN1 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN6 PROTEIN7 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN6 PROTEIN8 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN6 PROTEIN9 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN6 PROTEIN0 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN6 PROTEIN1 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN7 PROTEIN8 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN7 PROTEIN9 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN7 PROTEIN0 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN7 PROTEIN1 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN8 PROTEIN9 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN8 PROTEIN0 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN8 PROTEIN1 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN9 PROTEIN0 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN9 PROTEIN1 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The muscle-related genes included PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 PROTEIN0 protein, and PROTEIN1 suggesting that vertebrate skeletal and smooth muscle-type genes are simultaneously expressed in the amphioxus notochord.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The mutants ranged in affinity, from those that only weakly affected polymerization of PROTEIN0 to one that bound PROTEIN1 more strongly than wild-type PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The mutants ranged in affinity, from those that only weakly affected polymerization of PROTEIN0 to one that bound PROTEIN1 more strongly than wild-type PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The mutants ranged in affinity, from those that only weakly affected polymerization of PROTEIN0 to one that bound PROTEIN1 more strongly than wild-type PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The role of PROTEIN0 in PROTEIN1 polymerization and nucleotide exchange.\n",
      "\n",
      "PROTEIN0 PROTEIN1 With PROTEIN0 whose sequestering activity was low, the concentration of free PROTEIN1 monomers observed at steady-state of polymerization [Afree], was close to that seen with PROTEIN2 alone ([Acc], critical concentration of polymerization).\n",
      "\n",
      "PROTEIN0 PROTEIN2 With PROTEIN0 whose sequestering activity was low, the concentration of free PROTEIN1 monomers observed at steady-state of polymerization [Afree], was close to that seen with PROTEIN2 alone ([Acc], critical concentration of polymerization).\n",
      "\n",
      "PROTEIN1 PROTEIN2 With PROTEIN0 whose sequestering activity was low, the concentration of free PROTEIN1 monomers observed at steady-state of polymerization [Afree], was close to that seen with PROTEIN2 alone ([Acc], critical concentration of polymerization).\n",
      "\n",
      "PROTEIN0 PROTEIN1 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN6 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN7 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN8 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN9 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN0 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN6 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN7 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN8 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN9 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN1 PROTEIN0 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN2 PROTEIN6 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN2 PROTEIN7 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN2 PROTEIN8 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN2 PROTEIN9 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN2 PROTEIN0 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN3 PROTEIN6 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN3 PROTEIN7 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN3 PROTEIN8 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN3 PROTEIN9 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN3 PROTEIN0 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN4 PROTEIN6 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN4 PROTEIN7 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN4 PROTEIN8 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN4 PROTEIN9 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN4 PROTEIN0 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN5 PROTEIN6 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN5 PROTEIN7 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN5 PROTEIN8 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN5 PROTEIN9 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN5 PROTEIN0 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN6 PROTEIN7 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN6 PROTEIN8 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN6 PROTEIN9 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN6 PROTEIN0 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN7 PROTEIN8 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN7 PROTEIN9 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN7 PROTEIN0 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN8 PROTEIN9 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN8 PROTEIN0 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN9 PROTEIN0 The neoplastic cells were positive for PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 but negative for PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 PROTEIN8 PROTEIN9 and PROTEIN0 protein.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The N-terminal PROTEIN0 polypeptide eventually disrupted PROTEIN1 stress fibres whereas the C-terminal polypeptide was without effect.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The Pro1176 to Thr substitution found in PROTEIN0 from Wistar-Furth rats does not destroy the capacity of this region of the protein to bind PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The Pro1176 to Thr substitution found in PROTEIN0 from Wistar-Furth rats does not destroy the capacity of this region of the protein to bind PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The Pro1176 to Thr substitution found in PROTEIN0 from Wistar-Furth rats does not destroy the capacity of this region of the protein to bind PROTEIN1 or PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The observed in vivo expression patterns of PROTEIN0 PROTEIN1 and PROTEIN2 suggest that these proteins are directly linked with the developmental regulation of cell junctions, as cardiac cells become stably committed and phenotypically differentiated to eventually form a mature myocardium.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The observed in vivo expression patterns of PROTEIN0 PROTEIN1 and PROTEIN2 suggest that these proteins are directly linked with the developmental regulation of cell junctions, as cardiac cells become stably committed and phenotypically differentiated to eventually form a mature myocardium.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The observed in vivo expression patterns of PROTEIN0 PROTEIN1 and PROTEIN2 suggest that these proteins are directly linked with the developmental regulation of cell junctions, as cardiac cells become stably committed and phenotypically differentiated to eventually form a mature myocardium.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This study details the expression patterns of PROTEIN0 PROTEIN1 and PROTEIN2 during definition of the cardiac cell population as distinct compartments in the anterior regions of the chick embryo between stages 5 and 9.\n",
      "\n",
      "PROTEIN0 PROTEIN2 This study details the expression patterns of PROTEIN0 PROTEIN1 and PROTEIN2 during definition of the cardiac cell population as distinct compartments in the anterior regions of the chick embryo between stages 5 and 9.\n",
      "\n",
      "PROTEIN1 PROTEIN2 This study details the expression patterns of PROTEIN0 PROTEIN1 and PROTEIN2 during definition of the cardiac cell population as distinct compartments in the anterior regions of the chick embryo between stages 5 and 9.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 The PROTEIN0 gene encodes a PROTEIN1 substrate that associates with the cell-cell adhesion protein complex containing PROTEIN2 and its cytoplasmic cofactors PROTEIN3 PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN6 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN1 PROTEIN6 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN2 PROTEIN6 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN3 PROTEIN6 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN4 PROTEIN6 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN5 PROTEIN6 PROTEIN0 binds to and inhibits complexes formed by PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 PROTEIN6\n",
      "\n",
      "PROTEIN0 PROTEIN1 The phosphoprotein was identified as PROTEIN0 an PROTEIN1 and the activation-induced changes in its intracellular distribution have been described elsewhere (Suzuki et al., 1995, J Biol Chem 270:19551-19556).\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 gene product mediates PROTEIN1 and PROTEIN2 activation of the transcription factor PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 gene product mediates PROTEIN1 and PROTEIN2 activation of the transcription factor PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 gene product mediates PROTEIN1 and PROTEIN2 activation of the transcription factor PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 gene product mediates PROTEIN1 and PROTEIN2 activation of the transcription factor PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 gene product mediates PROTEIN1 and PROTEIN2 activation of the transcription factor PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 gene product mediates PROTEIN1 and PROTEIN2 activation of the transcription factor PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The posttranslational metabolism of PROTEIN0 PROTEIN1 and PROTEIN2 was regulated in parallel with the total contractile protein pool.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The posttranslational metabolism of PROTEIN0 PROTEIN1 and PROTEIN2 was regulated in parallel with the total contractile protein pool.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The posttranslational metabolism of PROTEIN0 PROTEIN1 and PROTEIN2 was regulated in parallel with the total contractile protein pool.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The potentiating effect of PROTEIN0 appears to involve increases in the expression of PROTEIN1 and decreases in the expression of the PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 The potentiating effect of PROTEIN0 appears to involve increases in the expression of PROTEIN1 and decreases in the expression of the PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 The potentiating effect of PROTEIN0 appears to involve increases in the expression of PROTEIN1 and decreases in the expression of the PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 The potentiating effect of PROTEIN0 appears to involve increases in the expression of PROTEIN1 and decreases in the expression of the PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 The potentiating effect of PROTEIN0 appears to involve increases in the expression of PROTEIN1 and decreases in the expression of the PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 The potentiating effect of PROTEIN0 appears to involve increases in the expression of PROTEIN1 and decreases in the expression of the PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 The potentiating effect of PROTEIN0 appears to involve increases in the expression of PROTEIN1 and decreases in the expression of the PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 The potentiating effect of PROTEIN0 appears to involve increases in the expression of PROTEIN1 and decreases in the expression of the PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 The potentiating effect of PROTEIN0 appears to involve increases in the expression of PROTEIN1 and decreases in the expression of the PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 The potentiating effect of PROTEIN0 appears to involve increases in the expression of PROTEIN1 and decreases in the expression of the PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 The presence, in varying quantities, of PROTEIN0 and PROTEIN1 isoforms of smooth muscle type in the different vessels reflected their degree of differentiation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The present study was designed to determine if cholesterol feeding results in alterations in the isoforms of PROTEIN0 and/or PROTEIN1 in gallbladder smooth muscle.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The primary purpose of this study was to determine the relationship between PROTEIN0 PROTEIN1 and PROTEIN2 contents and maximum isometric tetanic force (Po) in mouse extensor digitorum longus (EDL) muscles following eccentric contraction-induced injury.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The primary purpose of this study was to determine the relationship between PROTEIN0 PROTEIN1 and PROTEIN2 contents and maximum isometric tetanic force (Po) in mouse extensor digitorum longus (EDL) muscles following eccentric contraction-induced injury.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The primary purpose of this study was to determine the relationship between PROTEIN0 PROTEIN1 and PROTEIN2 contents and maximum isometric tetanic force (Po) in mouse extensor digitorum longus (EDL) muscles following eccentric contraction-induced injury.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The procedure allows for the simultaneous quantification of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 from as little as 50 mg of tissue.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The procedure allows for the simultaneous quantification of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 from as little as 50 mg of tissue.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The procedure allows for the simultaneous quantification of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 from as little as 50 mg of tissue.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The procedure allows for the simultaneous quantification of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 from as little as 50 mg of tissue.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The procedure allows for the simultaneous quantification of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 from as little as 50 mg of tissue.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The procedure allows for the simultaneous quantification of PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 from as little as 50 mg of tissue.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The product of PROTEIN0 has been shown to be essential for capsid assembly, but no role has previously been assigned to the product of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The products of the yeast mismatch repair genes PROTEIN0 and PROTEIN1 participate in the inhibition of genetic recombination between homeologous (divergent) DNA sequences.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 and PROTEIN1 can function differentially to control homeologous exchanges.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The prominent 125 A transverse stripping pattern characteristic of the PROTEIN0 cross-bridges in a bundle is also absent in hair borders suggesting PROTEIN1 as the component that gives rise to the transverse stripes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The protein PROTEIN0 binds to both PROTEIN1 and the head groups of poly)phosphoinositide)s and may regulate both PROTEIN2 assembly and the phosphoinositide signaling pathway.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The protein PROTEIN0 binds to both PROTEIN1 and the head groups of poly)phosphoinositide)s and may regulate both PROTEIN2 assembly and the phosphoinositide signaling pathway.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The protein PROTEIN0 binds to both PROTEIN1 and the head groups of poly)phosphoinositide)s and may regulate both PROTEIN2 assembly and the phosphoinositide signaling pathway.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The protein has the two conserved domains identified as PROTEIN0 and phosphatidylinositol 4,5-biphosphate (PIP2) binding sites found in members of the PROTEIN1 family.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The proteins encoded by PROTEIN0 and PROTEIN1 which display 35%, sequence identity are both located in the mitochondrial intermembrane space.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The proximity of the semi-conserved PROTEIN0 site and the binding pocket characteristic of plant PROTEIN1 suggests that epitopes encompassing both features are responsible for the cross-reactivity of antibodies between human and plant PROTEIN2 thought to be responsible for type I allergies.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The proximity of the semi-conserved PROTEIN0 site and the binding pocket characteristic of plant PROTEIN1 suggests that epitopes encompassing both features are responsible for the cross-reactivity of antibodies between human and plant PROTEIN2 thought to be responsible for type I allergies.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The proximity of the semi-conserved PROTEIN0 site and the binding pocket characteristic of plant PROTEIN1 suggests that epitopes encompassing both features are responsible for the cross-reactivity of antibodies between human and plant PROTEIN2 thought to be responsible for type I allergies.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The radioresistant N10 and parental KB cell lines were examined for the expression of human DNA repair genes which were related to the repair of radiation-induced DNA damage by northern blot analysis using five kinds of DNA probes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 The radioresistant N10 and parental KB cell lines were examined for the expression of human DNA repair genes which were related to the repair of radiation-induced DNA damage by northern blot analysis using five kinds of DNA probes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 The radioresistant N10 and parental KB cell lines were examined for the expression of human DNA repair genes which were related to the repair of radiation-induced DNA damage by northern blot analysis using five kinds of DNA probes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 The radioresistant N10 and parental KB cell lines were examined for the expression of human DNA repair genes which were related to the repair of radiation-induced DNA damage by northern blot analysis using five kinds of DNA probes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 The radioresistant N10 and parental KB cell lines were examined for the expression of human DNA repair genes which were related to the repair of radiation-induced DNA damage by northern blot analysis using five kinds of DNA probes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 The radioresistant N10 and parental KB cell lines were examined for the expression of human DNA repair genes which were related to the repair of radiation-induced DNA damage by northern blot analysis using five kinds of DNA probes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 The radioresistant N10 and parental KB cell lines were examined for the expression of human DNA repair genes which were related to the repair of radiation-induced DNA damage by northern blot analysis using five kinds of DNA probes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 The radioresistant N10 and parental KB cell lines were examined for the expression of human DNA repair genes which were related to the repair of radiation-induced DNA damage by northern blot analysis using five kinds of DNA probes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 The radioresistant N10 and parental KB cell lines were examined for the expression of human DNA repair genes which were related to the repair of radiation-induced DNA damage by northern blot analysis using five kinds of DNA probes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 The radioresistant N10 and parental KB cell lines were examined for the expression of human DNA repair genes which were related to the repair of radiation-induced DNA damage by northern blot analysis using five kinds of DNA probes PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 There are only 14 non-conservative substitutions in the two PROTEIN0 three of which are located in the secondary PROTEIN1 loop and flanking regions and others correspond to residues so far not assigned a functional role, including two residues in the proximal S2 domain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The recombinant material is similar to authentic PROTEIN0 from Acanthamoeba-based on fluorescence monitored urea denaturation, circular dichroism, PROTEIN1 exchange rate and the Kd for rabbit PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The recombinant material is similar to authentic PROTEIN0 from Acanthamoeba-based on fluorescence monitored urea denaturation, circular dichroism, PROTEIN1 exchange rate and the Kd for rabbit PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The recombinant material is similar to authentic PROTEIN0 from Acanthamoeba-based on fluorescence monitored urea denaturation, circular dichroism, PROTEIN1 exchange rate and the Kd for rabbit PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The recombinant material is similar to authentic human PROTEIN0 based on the measured Kd for rabbit PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The resulting PROTEIN0 product is unable to bind PROTEIN1 that is responsible for PROTEIN2 filament binding and organization.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The resulting PROTEIN0 product is unable to bind PROTEIN1 that is responsible for PROTEIN2 filament binding and organization.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The resulting PROTEIN0 product is unable to bind PROTEIN1 that is responsible for PROTEIN2 filament binding and organization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The results clearly showed that the mutated PROTEIN0 possessing KTLKK instead of KKRKK did not translocate into the nuclei in response to heat shock whereas a recombinant PROTEIN1 with the unaltered sequence of KKRKK responded to heat shock and formed intranuclear rods together with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The results clearly showed that the mutated PROTEIN0 possessing KTLKK instead of KKRKK did not translocate into the nuclei in response to heat shock whereas a recombinant PROTEIN1 with the unaltered sequence of KKRKK responded to heat shock and formed intranuclear rods together with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The results clearly showed that the mutated PROTEIN0 possessing KTLKK instead of KKRKK did not translocate into the nuclei in response to heat shock whereas a recombinant PROTEIN1 with the unaltered sequence of KKRKK responded to heat shock and formed intranuclear rods together with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN0 PROTEIN6 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN0 PROTEIN7 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN0 PROTEIN8 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN1 PROTEIN6 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN1 PROTEIN7 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN1 PROTEIN8 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN2 PROTEIN6 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN2 PROTEIN7 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN2 PROTEIN8 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN3 PROTEIN6 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN3 PROTEIN7 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN3 PROTEIN8 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN4 PROTEIN6 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN4 PROTEIN7 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN4 PROTEIN8 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN5 PROTEIN6 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN5 PROTEIN7 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN5 PROTEIN8 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN6 PROTEIN7 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN6 PROTEIN8 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN7 PROTEIN8 The results indicate that (a) PROTEIN0 and PROTEIN1 differ in their nuclear translocation and complexing with PROTEIN2 and PROTEIN3 (b) PROTEIN4 transactivation is preferentially driven by PROTEIN5 and (c) the cytoplasmic partners of PROTEIN6 PROTEIN7 and PROTEIN8 can sequester it to the cytoplasm and inhibit its transcriptional activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The Schizosaccharomyces pombe PROTEIN0 PROTEIN1 and PROTEIN2 genes are homologous to PROTEIN3 PROTEIN4 and PROTEIN5 respectively, which are indispensable in the recombinational repair of double-strand breaks (DSBs) in Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These cells have both PROTEIN0 and PROTEIN1 display unusual expression of PROTEIN2 and have the typical fibroblastic phenotype of transformed cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These cells have both PROTEIN0 and PROTEIN1 display unusual expression of PROTEIN2 and have the typical fibroblastic phenotype of transformed cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These cells have both PROTEIN0 and PROTEIN1 display unusual expression of PROTEIN2 and have the typical fibroblastic phenotype of transformed cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These data suggest that nuclear PROTEIN0 can mediate a stimulus-response action on the PROTEIN1 cytoskeleton which is somehow linked to a phosphoinositide-signaling cascade.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These differences in contractile protein profiles between single and cultured cells were collaborated with the observation of cells using immunofluorescence microscope with responsible antibodies to isoforms of PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 These differences in contractile protein profiles between single and cultured cells were collaborated with the observation of cells using immunofluorescence microscope with responsible antibodies to isoforms of PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 These differences in contractile protein profiles between single and cultured cells were collaborated with the observation of cells using immunofluorescence microscope with responsible antibodies to isoforms of PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 These findings reveal striking differences between S. cerevisiae and vertebrates in the functions of PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 These genes were designated PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 and were predicted to encode proteins with molecular weights of, respectively, 99,000, 80,000, 94,000, 51,000, and 114,000.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These genes were designated PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 and were predicted to encode proteins with molecular weights of, respectively, 99,000, 80,000, 94,000, 51,000, and 114,000.\n",
      "\n",
      "PROTEIN0 PROTEIN3 These genes were designated PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 and were predicted to encode proteins with molecular weights of, respectively, 99,000, 80,000, 94,000, 51,000, and 114,000.\n",
      "\n",
      "PROTEIN0 PROTEIN4 These genes were designated PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 and were predicted to encode proteins with molecular weights of, respectively, 99,000, 80,000, 94,000, 51,000, and 114,000.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These genes were designated PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 and were predicted to encode proteins with molecular weights of, respectively, 99,000, 80,000, 94,000, 51,000, and 114,000.\n",
      "\n",
      "PROTEIN1 PROTEIN3 These genes were designated PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 and were predicted to encode proteins with molecular weights of, respectively, 99,000, 80,000, 94,000, 51,000, and 114,000.\n",
      "\n",
      "PROTEIN1 PROTEIN4 These genes were designated PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 and were predicted to encode proteins with molecular weights of, respectively, 99,000, 80,000, 94,000, 51,000, and 114,000.\n",
      "\n",
      "PROTEIN2 PROTEIN3 These genes were designated PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 and were predicted to encode proteins with molecular weights of, respectively, 99,000, 80,000, 94,000, 51,000, and 114,000.\n",
      "\n",
      "PROTEIN2 PROTEIN4 These genes were designated PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 and were predicted to encode proteins with molecular weights of, respectively, 99,000, 80,000, 94,000, 51,000, and 114,000.\n",
      "\n",
      "PROTEIN3 PROTEIN4 These genes were designated PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 and were predicted to encode proteins with molecular weights of, respectively, 99,000, 80,000, 94,000, 51,000, and 114,000.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These increases were specific; DNA, total protein, PROTEIN0 PROTEIN1 and PROTEIN2 levels were unchanged in the overexpressing cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These increases were specific; DNA, total protein, PROTEIN0 PROTEIN1 and PROTEIN2 levels were unchanged in the overexpressing cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These increases were specific; DNA, total protein, PROTEIN0 PROTEIN1 and PROTEIN2 levels were unchanged in the overexpressing cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These observations demonstrate that cardiac development in man, in contrast to rodent heart, is characterized by an up-regulation of the PROTEIN0 gene, the expression of which does not change in hypertrophied and failing hearts, and suggest that the PROTEIN1 and PROTEIN2 families are independently regulated in human heart.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These observations demonstrate that cardiac development in man, in contrast to rodent heart, is characterized by an up-regulation of the PROTEIN0 gene, the expression of which does not change in hypertrophied and failing hearts, and suggest that the PROTEIN1 and PROTEIN2 families are independently regulated in human heart.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These observations demonstrate that cardiac development in man, in contrast to rodent heart, is characterized by an up-regulation of the PROTEIN0 gene, the expression of which does not change in hypertrophied and failing hearts, and suggest that the PROTEIN1 and PROTEIN2 families are independently regulated in human heart.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These phenotypes are similar to those of the mutant of PROTEIN0 which encodes PROTEIN1 an PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 These phenotypes are similar to those of the mutant of PROTEIN0 which encodes PROTEIN1 an PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 These phenotypes are similar to those of the mutant of PROTEIN0 which encodes PROTEIN1 an PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The sequence of the inhibitory region is similar to that of a beta-hairpin region of the PROTEIN0 PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN6 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN7 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN6 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN1 PROTEIN7 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN6 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN2 PROTEIN7 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN3 PROTEIN6 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN3 PROTEIN7 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN4 PROTEIN6 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN4 PROTEIN7 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN5 PROTEIN6 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN5 PROTEIN7 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN6 PROTEIN7 The sequences of the predicted PROTEIN0 PROTEIN1 and PROTEIN2 proteins are related to the PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 and PROTEIN7 respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These regions also interact with the physiologically relevant ligands of PROTEIN0 PROTEIN1 and proline-rich peptides.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN3 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN4 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN5 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN3 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN4 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN1 PROTEIN5 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN3 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN4 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN2 PROTEIN5 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN3 PROTEIN4 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN3 PROTEIN5 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN4 PROTEIN5 These results indicate that phosphorylation of PROTEIN0 by PROTEIN1 and consequently increased phosphorylation of PROTEIN2 by PROTEIN3 contribute to PROTEIN4 reorganization of the PROTEIN5 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results show that PROTEIN0 PROTEIN1 is an important regulator of PROTEIN2 cell motility during Drosophila development.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results show that PROTEIN0 PROTEIN1 is an important regulator of PROTEIN2 cell motility during Drosophila development.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results show that PROTEIN0 PROTEIN1 is an important regulator of PROTEIN2 cell motility during Drosophila development.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 may be involved in the pathogenesis of glomerulonephritis by reorganizing PROTEIN1 cytoskeleton.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 functions downstream of the PROTEIN1 PROTEIN2 signalling pathway and plays a role in reorganization of PROTEIN3 filaments and membrane structures, by phosphorylating PROTEIN4 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results suggest that PROTEIN0 functions downstream of the PROTEIN1 PROTEIN2 signalling pathway and plays a role in reorganization of PROTEIN3 filaments and membrane structures, by phosphorylating PROTEIN4 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN3 These results suggest that PROTEIN0 functions downstream of the PROTEIN1 PROTEIN2 signalling pathway and plays a role in reorganization of PROTEIN3 filaments and membrane structures, by phosphorylating PROTEIN4 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN4 These results suggest that PROTEIN0 functions downstream of the PROTEIN1 PROTEIN2 signalling pathway and plays a role in reorganization of PROTEIN3 filaments and membrane structures, by phosphorylating PROTEIN4 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results suggest that PROTEIN0 functions downstream of the PROTEIN1 PROTEIN2 signalling pathway and plays a role in reorganization of PROTEIN3 filaments and membrane structures, by phosphorylating PROTEIN4 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN3 These results suggest that PROTEIN0 functions downstream of the PROTEIN1 PROTEIN2 signalling pathway and plays a role in reorganization of PROTEIN3 filaments and membrane structures, by phosphorylating PROTEIN4 proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN4 These results suggest that PROTEIN0 functions downstream of the PROTEIN1 PROTEIN2 signalling pathway and plays a role in reorganization of PROTEIN3 filaments and membrane structures, by phosphorylating PROTEIN4 proteins.\n",
      "\n",
      "PROTEIN2 PROTEIN3 These results suggest that PROTEIN0 functions downstream of the PROTEIN1 PROTEIN2 signalling pathway and plays a role in reorganization of PROTEIN3 filaments and membrane structures, by phosphorylating PROTEIN4 proteins.\n",
      "\n",
      "PROTEIN2 PROTEIN4 These results suggest that PROTEIN0 functions downstream of the PROTEIN1 PROTEIN2 signalling pathway and plays a role in reorganization of PROTEIN3 filaments and membrane structures, by phosphorylating PROTEIN4 proteins.\n",
      "\n",
      "PROTEIN3 PROTEIN4 These results suggest that PROTEIN0 functions downstream of the PROTEIN1 PROTEIN2 signalling pathway and plays a role in reorganization of PROTEIN3 filaments and membrane structures, by phosphorylating PROTEIN4 proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN5 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN1 PROTEIN5 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN2 PROTEIN5 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN3 PROTEIN5 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN4 PROTEIN5 We report here that PROTEIN0 and PROTEIN1 phosphorylate both PROTEIN2 and PROTEIN3 PROTEIN4 specifically at Ser-3 and exhibit partially distinct substrate specificity when tested using site-directed PROTEIN5 mutants as substrates.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that the heptapeptide sequence is specific for the interaction with PROTEIN0 and, therefore, may constitute part of the PROTEIN1 domain of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results suggest that the heptapeptide sequence is specific for the interaction with PROTEIN0 and, therefore, may constitute part of the PROTEIN1 domain of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results suggest that the heptapeptide sequence is specific for the interaction with PROTEIN0 and, therefore, may constitute part of the PROTEIN1 domain of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN3 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN4 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN5 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN1 PROTEIN3 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN1 PROTEIN4 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN1 PROTEIN5 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN2 PROTEIN3 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN2 PROTEIN4 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN2 PROTEIN5 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN3 PROTEIN4 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN3 PROTEIN5 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN4 PROTEIN5 These results suggest that PROTEIN0 can cause apoptosis in pancreatic beta cells through PROTEIN1 apoptotic factors, PROTEIN2 PROTEIN3 and PROTEIN4 and PROTEIN5 ceramide production may be involved in the pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 and PROTEIN2 associate with and activate the PROTEIN3 PROTEIN4 in human T lymphocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 PROTEIN1 and PROTEIN2 associate with and activate the PROTEIN3 PROTEIN4 in human T lymphocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 PROTEIN1 and PROTEIN2 associate with and activate the PROTEIN3 PROTEIN4 in human T lymphocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The PROTEIN0 PROTEIN1 and PROTEIN2 associate with and activate the PROTEIN3 PROTEIN4 in human T lymphocytes.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 PROTEIN1 and PROTEIN2 associate with and activate the PROTEIN3 PROTEIN4 in human T lymphocytes.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 PROTEIN1 and PROTEIN2 associate with and activate the PROTEIN3 PROTEIN4 in human T lymphocytes.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The PROTEIN0 PROTEIN1 and PROTEIN2 associate with and activate the PROTEIN3 PROTEIN4 in human T lymphocytes.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 PROTEIN1 and PROTEIN2 associate with and activate the PROTEIN3 PROTEIN4 in human T lymphocytes.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The PROTEIN0 PROTEIN1 and PROTEIN2 associate with and activate the PROTEIN3 PROTEIN4 in human T lymphocytes.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The PROTEIN0 PROTEIN1 and PROTEIN2 associate with and activate the PROTEIN3 PROTEIN4 in human T lymphocytes.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These two proteins were identified as the phosphorylated and non-phosphorylated forms of the pH-sensitive PROTEIN0 protein, PROTEIN1 by sequencing of peptide fragments and immunoblotting with a monoclonal antibody specific for PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 These two proteins were identified as the phosphorylated and non-phosphorylated forms of the pH-sensitive PROTEIN0 protein, PROTEIN1 by sequencing of peptide fragments and immunoblotting with a monoclonal antibody specific for PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 These two proteins were identified as the phosphorylated and non-phosphorylated forms of the pH-sensitive PROTEIN0 protein, PROTEIN1 by sequencing of peptide fragments and immunoblotting with a monoclonal antibody specific for PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 homology domain 3 (SH3) of a yeast PROTEIN1 PROTEIN2 binds to PROTEIN3 and is required for targeting to sites of PROTEIN4 polarization.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 homology domain 3 (SH3) of a yeast PROTEIN1 PROTEIN2 binds to PROTEIN3 and is required for targeting to sites of PROTEIN4 polarization.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 homology domain 3 (SH3) of a yeast PROTEIN1 PROTEIN2 binds to PROTEIN3 and is required for targeting to sites of PROTEIN4 polarization.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The PROTEIN0 homology domain 3 (SH3) of a yeast PROTEIN1 PROTEIN2 binds to PROTEIN3 and is required for targeting to sites of PROTEIN4 polarization.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 homology domain 3 (SH3) of a yeast PROTEIN1 PROTEIN2 binds to PROTEIN3 and is required for targeting to sites of PROTEIN4 polarization.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 homology domain 3 (SH3) of a yeast PROTEIN1 PROTEIN2 binds to PROTEIN3 and is required for targeting to sites of PROTEIN4 polarization.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The PROTEIN0 homology domain 3 (SH3) of a yeast PROTEIN1 PROTEIN2 binds to PROTEIN3 and is required for targeting to sites of PROTEIN4 polarization.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 homology domain 3 (SH3) of a yeast PROTEIN1 PROTEIN2 binds to PROTEIN3 and is required for targeting to sites of PROTEIN4 polarization.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The PROTEIN0 homology domain 3 (SH3) of a yeast PROTEIN1 PROTEIN2 binds to PROTEIN3 and is required for targeting to sites of PROTEIN4 polarization.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The PROTEIN0 homology domain 3 (SH3) of a yeast PROTEIN1 PROTEIN2 binds to PROTEIN3 and is required for targeting to sites of PROTEIN4 polarization.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The structural homology between PROTEIN0 and PROTEIN1 segment-1 binding to PROTEIN2 was confirmed experimentally by two types of competitive binding assays.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The structural homology between PROTEIN0 and PROTEIN1 segment-1 binding to PROTEIN2 was confirmed experimentally by two types of competitive binding assays.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The structural homology between PROTEIN0 and PROTEIN1 segment-1 binding to PROTEIN2 was confirmed experimentally by two types of competitive binding assays.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The structure of an PROTEIN0 domain from human PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The structure of PROTEIN0 differed entirely from that of PROTEIN1 and resembled the overall fold of the PROTEIN2 regulatory protein, PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The structure of PROTEIN0 differed entirely from that of PROTEIN1 and resembled the overall fold of the PROTEIN2 regulatory protein, PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The structure of PROTEIN0 differed entirely from that of PROTEIN1 and resembled the overall fold of the PROTEIN2 regulatory protein, PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The structure of PROTEIN0 differed entirely from that of PROTEIN1 and resembled the overall fold of the PROTEIN2 regulatory protein, PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The structure of PROTEIN0 differed entirely from that of PROTEIN1 and resembled the overall fold of the PROTEIN2 regulatory protein, PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The structure of PROTEIN0 differed entirely from that of PROTEIN1 and resembled the overall fold of the PROTEIN2 regulatory protein, PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The Tetrahymena PROTEIN0 has two PROTEIN1 domains, but lacks the EF-hand Ca(2+)-binding motif, suggesting that Tetrahymena PROTEIN2 probably cross-links PROTEIN3 filaments in a Ca(2+)- insensitive manner during cytokinesis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The Tetrahymena PROTEIN0 has two PROTEIN1 domains, but lacks the EF-hand Ca(2+)-binding motif, suggesting that Tetrahymena PROTEIN2 probably cross-links PROTEIN3 filaments in a Ca(2+)- insensitive manner during cytokinesis.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The Tetrahymena PROTEIN0 has two PROTEIN1 domains, but lacks the EF-hand Ca(2+)-binding motif, suggesting that Tetrahymena PROTEIN2 probably cross-links PROTEIN3 filaments in a Ca(2+)- insensitive manner during cytokinesis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The Tetrahymena PROTEIN0 has two PROTEIN1 domains, but lacks the EF-hand Ca(2+)-binding motif, suggesting that Tetrahymena PROTEIN2 probably cross-links PROTEIN3 filaments in a Ca(2+)- insensitive manner during cytokinesis.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The Tetrahymena PROTEIN0 has two PROTEIN1 domains, but lacks the EF-hand Ca(2+)-binding motif, suggesting that Tetrahymena PROTEIN2 probably cross-links PROTEIN3 filaments in a Ca(2+)- insensitive manner during cytokinesis.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The Tetrahymena PROTEIN0 has two PROTEIN1 domains, but lacks the EF-hand Ca(2+)-binding motif, suggesting that Tetrahymena PROTEIN2 probably cross-links PROTEIN3 filaments in a Ca(2+)- insensitive manner during cytokinesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN0 PROTEIN4 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN0 PROTEIN5 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN0 PROTEIN6 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN1 PROTEIN4 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN1 PROTEIN5 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN1 PROTEIN6 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN2 PROTEIN4 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN2 PROTEIN5 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN2 PROTEIN6 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN3 PROTEIN4 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN3 PROTEIN5 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN3 PROTEIN6 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN4 PROTEIN5 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN4 PROTEIN6 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN5 PROTEIN6 The PROTEIN0 PROTEIN1 and the PROTEIN2 receptor recruit complexes formed by the interactions between PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 - adaptor proteins that contain death domains - which in turn recruit other proteins to initiate signaling [1][2][3][4][5].\n",
      "\n",
      "PROTEIN0 PROTEIN1 The two proteins were found to differ in their interaction with PROTEIN0 like PROTEIN1 and PROTEIN2 isolated from porcine brain.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The two proteins were found to differ in their interaction with PROTEIN0 like PROTEIN1 and PROTEIN2 isolated from porcine brain.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The two proteins were found to differ in their interaction with PROTEIN0 like PROTEIN1 and PROTEIN2 isolated from porcine brain.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The vaccinia virus PROTEIN0 gene product, PROTEIN1 is a dsRNA binding protein that prevents activation of the PROTEIN2 dsRNA-activated PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 The vaccinia virus PROTEIN0 gene product, PROTEIN1 is a dsRNA binding protein that prevents activation of the PROTEIN2 dsRNA-activated PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 The vaccinia virus PROTEIN0 gene product, PROTEIN1 is a dsRNA binding protein that prevents activation of the PROTEIN2 dsRNA-activated PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 The vaccinia virus PROTEIN0 gene product, PROTEIN1 is a dsRNA binding protein that prevents activation of the PROTEIN2 dsRNA-activated PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 The vaccinia virus PROTEIN0 gene product, PROTEIN1 is a dsRNA binding protein that prevents activation of the PROTEIN2 dsRNA-activated PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 The vaccinia virus PROTEIN0 gene product, PROTEIN1 is a dsRNA binding protein that prevents activation of the PROTEIN2 dsRNA-activated PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 The vaccinia virus PROTEIN0 gene product, PROTEIN1 is a dsRNA binding protein that prevents activation of the PROTEIN2 dsRNA-activated PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 The vaccinia virus PROTEIN0 gene product, PROTEIN1 is a dsRNA binding protein that prevents activation of the PROTEIN2 dsRNA-activated PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 The vaccinia virus PROTEIN0 gene product, PROTEIN1 is a dsRNA binding protein that prevents activation of the PROTEIN2 dsRNA-activated PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 The vaccinia virus PROTEIN0 gene product, PROTEIN1 is a dsRNA binding protein that prevents activation of the PROTEIN2 dsRNA-activated PROTEIN3 PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 They exhibited strong synergistic increases in PROTEIN0 duplication mutations, intrachromosomal and interchromosomal recombination, and required the wild-type double-strand break repair genes PROTEIN1 PROTEIN2 and PROTEIN3 for viability.\n",
      "\n",
      "PROTEIN0 PROTEIN2 They exhibited strong synergistic increases in PROTEIN0 duplication mutations, intrachromosomal and interchromosomal recombination, and required the wild-type double-strand break repair genes PROTEIN1 PROTEIN2 and PROTEIN3 for viability.\n",
      "\n",
      "PROTEIN0 PROTEIN3 They exhibited strong synergistic increases in PROTEIN0 duplication mutations, intrachromosomal and interchromosomal recombination, and required the wild-type double-strand break repair genes PROTEIN1 PROTEIN2 and PROTEIN3 for viability.\n",
      "\n",
      "PROTEIN1 PROTEIN2 They exhibited strong synergistic increases in PROTEIN0 duplication mutations, intrachromosomal and interchromosomal recombination, and required the wild-type double-strand break repair genes PROTEIN1 PROTEIN2 and PROTEIN3 for viability.\n",
      "\n",
      "PROTEIN1 PROTEIN3 They exhibited strong synergistic increases in PROTEIN0 duplication mutations, intrachromosomal and interchromosomal recombination, and required the wild-type double-strand break repair genes PROTEIN1 PROTEIN2 and PROTEIN3 for viability.\n",
      "\n",
      "PROTEIN2 PROTEIN3 They exhibited strong synergistic increases in PROTEIN0 duplication mutations, intrachromosomal and interchromosomal recombination, and required the wild-type double-strand break repair genes PROTEIN1 PROTEIN2 and PROTEIN3 for viability.\n",
      "\n",
      "PROTEIN0 PROTEIN1 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 They had altered adhesive properties as measured by their ability to bind to a PROTEIN0 glass surface, suggesting that the expression of a PROTEIN1 alters the assembly of PROTEIN2 receptors, PROTEIN3 and cytoskeletal proteins such as PROTEIN4 and PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 This activity is independent of PROTEIN0 ability to bind to three known interacting proteins, PROTEIN1 PROTEIN2 or PROTEIN3 suggesting that it is distinct from PROTEIN4 functions at activated PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 This approach is based on the assumption that an PROTEIN0 mutation that specifically impairs the interaction with an PROTEIN1 will cause a phenotype similar to a null mutation in the gene that encodes the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 This approach is based on the assumption that an PROTEIN0 mutation that specifically impairs the interaction with an PROTEIN1 will cause a phenotype similar to a null mutation in the gene that encodes the PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 This approach is based on the assumption that an PROTEIN0 mutation that specifically impairs the interaction with an PROTEIN1 will cause a phenotype similar to a null mutation in the gene that encodes the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 21 PROTEIN0 mutations were analyzed in budding yeast, and specific regions of PROTEIN1 subdomain 1 were implicated in the interaction with PROTEIN2 an PROTEIN3 filament-bundling protein.\n",
      "\n",
      "PROTEIN0 PROTEIN2 21 PROTEIN0 mutations were analyzed in budding yeast, and specific regions of PROTEIN1 subdomain 1 were implicated in the interaction with PROTEIN2 an PROTEIN3 filament-bundling protein.\n",
      "\n",
      "PROTEIN0 PROTEIN3 21 PROTEIN0 mutations were analyzed in budding yeast, and specific regions of PROTEIN1 subdomain 1 were implicated in the interaction with PROTEIN2 an PROTEIN3 filament-bundling protein.\n",
      "\n",
      "PROTEIN1 PROTEIN2 21 PROTEIN0 mutations were analyzed in budding yeast, and specific regions of PROTEIN1 subdomain 1 were implicated in the interaction with PROTEIN2 an PROTEIN3 filament-bundling protein.\n",
      "\n",
      "PROTEIN1 PROTEIN3 21 PROTEIN0 mutations were analyzed in budding yeast, and specific regions of PROTEIN1 subdomain 1 were implicated in the interaction with PROTEIN2 an PROTEIN3 filament-bundling protein.\n",
      "\n",
      "PROTEIN2 PROTEIN3 21 PROTEIN0 mutations were analyzed in budding yeast, and specific regions of PROTEIN1 subdomain 1 were implicated in the interaction with PROTEIN2 an PROTEIN3 filament-bundling protein.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This arrest can be suppressed by mutations in PROTEIN0 PROTEIN1 and PROTEIN2 suggesting that the cell cycle defect in PROTEIN3 PROTEIN4 mutants results from inappropriate homologous recombination.\n",
      "\n",
      "PROTEIN0 PROTEIN2 This arrest can be suppressed by mutations in PROTEIN0 PROTEIN1 and PROTEIN2 suggesting that the cell cycle defect in PROTEIN3 PROTEIN4 mutants results from inappropriate homologous recombination.\n",
      "\n",
      "PROTEIN0 PROTEIN3 This arrest can be suppressed by mutations in PROTEIN0 PROTEIN1 and PROTEIN2 suggesting that the cell cycle defect in PROTEIN3 PROTEIN4 mutants results from inappropriate homologous recombination.\n",
      "\n",
      "PROTEIN0 PROTEIN4 This arrest can be suppressed by mutations in PROTEIN0 PROTEIN1 and PROTEIN2 suggesting that the cell cycle defect in PROTEIN3 PROTEIN4 mutants results from inappropriate homologous recombination.\n",
      "\n",
      "PROTEIN1 PROTEIN2 This arrest can be suppressed by mutations in PROTEIN0 PROTEIN1 and PROTEIN2 suggesting that the cell cycle defect in PROTEIN3 PROTEIN4 mutants results from inappropriate homologous recombination.\n",
      "\n",
      "PROTEIN1 PROTEIN3 This arrest can be suppressed by mutations in PROTEIN0 PROTEIN1 and PROTEIN2 suggesting that the cell cycle defect in PROTEIN3 PROTEIN4 mutants results from inappropriate homologous recombination.\n",
      "\n",
      "PROTEIN1 PROTEIN4 This arrest can be suppressed by mutations in PROTEIN0 PROTEIN1 and PROTEIN2 suggesting that the cell cycle defect in PROTEIN3 PROTEIN4 mutants results from inappropriate homologous recombination.\n",
      "\n",
      "PROTEIN2 PROTEIN3 This arrest can be suppressed by mutations in PROTEIN0 PROTEIN1 and PROTEIN2 suggesting that the cell cycle defect in PROTEIN3 PROTEIN4 mutants results from inappropriate homologous recombination.\n",
      "\n",
      "PROTEIN2 PROTEIN4 This arrest can be suppressed by mutations in PROTEIN0 PROTEIN1 and PROTEIN2 suggesting that the cell cycle defect in PROTEIN3 PROTEIN4 mutants results from inappropriate homologous recombination.\n",
      "\n",
      "PROTEIN3 PROTEIN4 This arrest can be suppressed by mutations in PROTEIN0 PROTEIN1 and PROTEIN2 suggesting that the cell cycle defect in PROTEIN3 PROTEIN4 mutants results from inappropriate homologous recombination.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This filament growth is driven by a large storage pool of PROTEIN0 bound to the sequestering protein, PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 This is consistent with PROTEIN0 activation being required for PROTEIN1 reorganization during exocytosis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This is the first report describing the coexistence of PROTEIN0 with PROTEIN1 filaments in the division furrow, implying the possible involvement of PROTEIN2 in assembly and disassembly of contractile ring microfilaments in the process of cytokinesis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 This is the first report describing the coexistence of PROTEIN0 with PROTEIN1 filaments in the division furrow, implying the possible involvement of PROTEIN2 in assembly and disassembly of contractile ring microfilaments in the process of cytokinesis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 This is the first report describing the coexistence of PROTEIN0 with PROTEIN1 filaments in the division furrow, implying the possible involvement of PROTEIN2 in assembly and disassembly of contractile ring microfilaments in the process of cytokinesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This novel mutation was predicted to disrupt the binding of PROTEIN0 to PROTEIN1 and may be related to the diffuse type morphology.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This shows that structural changes in the poly(L-proline)-binding region of PROTEIN0 can affect the distantly located PROTEIN1 site.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 This technique was employed to quantify changes caused by the lack of PROTEIN0 a protein that couples the PROTEIN1 network to the plasma membrane, or by the lack of PROTEIN2 or PROTEIN3 two isoforms of the PROTEIN4 protein PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Three PROTEIN0 proteins, PROTEIN1 PROTEIN2 and PROTEIN3 influence gelation, solation, and polymerization, respectively, of PROTEIN4 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Three PROTEIN0 proteins, PROTEIN1 PROTEIN2 and PROTEIN3 influence gelation, solation, and polymerization, respectively, of PROTEIN4 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Three PROTEIN0 proteins, PROTEIN1 PROTEIN2 and PROTEIN3 influence gelation, solation, and polymerization, respectively, of PROTEIN4 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Three PROTEIN0 proteins, PROTEIN1 PROTEIN2 and PROTEIN3 influence gelation, solation, and polymerization, respectively, of PROTEIN4 in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Three PROTEIN0 proteins, PROTEIN1 PROTEIN2 and PROTEIN3 influence gelation, solation, and polymerization, respectively, of PROTEIN4 in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Three PROTEIN0 proteins, PROTEIN1 PROTEIN2 and PROTEIN3 influence gelation, solation, and polymerization, respectively, of PROTEIN4 in vitro.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Three PROTEIN0 proteins, PROTEIN1 PROTEIN2 and PROTEIN3 influence gelation, solation, and polymerization, respectively, of PROTEIN4 in vitro.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Three PROTEIN0 proteins, PROTEIN1 PROTEIN2 and PROTEIN3 influence gelation, solation, and polymerization, respectively, of PROTEIN4 in vitro.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Three PROTEIN0 proteins, PROTEIN1 PROTEIN2 and PROTEIN3 influence gelation, solation, and polymerization, respectively, of PROTEIN4 in vitro.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Three PROTEIN0 proteins, PROTEIN1 PROTEIN2 and PROTEIN3 influence gelation, solation, and polymerization, respectively, of PROTEIN4 in vitro.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Three components of Drosophila adherens junctions, analogous to those in vertebrates, have been identified: PROTEIN0 (homolog of PROTEIN1 Drosophila PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Three components of Drosophila adherens junctions, analogous to those in vertebrates, have been identified: PROTEIN0 (homolog of PROTEIN1 Drosophila PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Three components of Drosophila adherens junctions, analogous to those in vertebrates, have been identified: PROTEIN0 (homolog of PROTEIN1 Drosophila PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Three components of Drosophila adherens junctions, analogous to those in vertebrates, have been identified: PROTEIN0 (homolog of PROTEIN1 Drosophila PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Three components of Drosophila adherens junctions, analogous to those in vertebrates, have been identified: PROTEIN0 (homolog of PROTEIN1 Drosophila PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Three components of Drosophila adherens junctions, analogous to those in vertebrates, have been identified: PROTEIN0 (homolog of PROTEIN1 Drosophila PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Three components of Drosophila adherens junctions, analogous to those in vertebrates, have been identified: PROTEIN0 (homolog of PROTEIN1 Drosophila PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Three components of Drosophila adherens junctions, analogous to those in vertebrates, have been identified: PROTEIN0 (homolog of PROTEIN1 Drosophila PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Three components of Drosophila adherens junctions, analogous to those in vertebrates, have been identified: PROTEIN0 (homolog of PROTEIN1 Drosophila PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Three components of Drosophila adherens junctions, analogous to those in vertebrates, have been identified: PROTEIN0 (homolog of PROTEIN1 Drosophila PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, PROTEIN0 was found in association with PROTEIN1 in the mechanoreceptive region of hair cells, whereas supporting cells, although rich in PROTEIN2 did not reveal PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, PROTEIN0 was found in association with PROTEIN1 in the mechanoreceptive region of hair cells, whereas supporting cells, although rich in PROTEIN2 did not reveal PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Thus, PROTEIN0 was found in association with PROTEIN1 in the mechanoreceptive region of hair cells, whereas supporting cells, although rich in PROTEIN2 did not reveal PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, PROTEIN0 was found in association with PROTEIN1 in the mechanoreceptive region of hair cells, whereas supporting cells, although rich in PROTEIN2 did not reveal PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Thus, PROTEIN0 was found in association with PROTEIN1 in the mechanoreceptive region of hair cells, whereas supporting cells, although rich in PROTEIN2 did not reveal PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Thus, PROTEIN0 was found in association with PROTEIN1 in the mechanoreceptive region of hair cells, whereas supporting cells, although rich in PROTEIN2 did not reveal PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, PROTEIN0 PROTEIN1 and PROTEIN2 have similar prognostic values.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, PROTEIN0 PROTEIN1 and PROTEIN2 have similar prognostic values.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, PROTEIN0 PROTEIN1 and PROTEIN2 have similar prognostic values.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, PROTEIN0 contains an NH(2)-terminal domain homologous to that implicated in PROTEIN1 endocytic function, three predicted coiled-coils, a leucine zipper, and a PROTEIN2 PROTEIN3 domain at the COOH terminus.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, PROTEIN0 contains an NH(2)-terminal domain homologous to that implicated in PROTEIN1 endocytic function, three predicted coiled-coils, a leucine zipper, and a PROTEIN2 PROTEIN3 domain at the COOH terminus.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Thus, PROTEIN0 contains an NH(2)-terminal domain homologous to that implicated in PROTEIN1 endocytic function, three predicted coiled-coils, a leucine zipper, and a PROTEIN2 PROTEIN3 domain at the COOH terminus.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, PROTEIN0 contains an NH(2)-terminal domain homologous to that implicated in PROTEIN1 endocytic function, three predicted coiled-coils, a leucine zipper, and a PROTEIN2 PROTEIN3 domain at the COOH terminus.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Thus, PROTEIN0 contains an NH(2)-terminal domain homologous to that implicated in PROTEIN1 endocytic function, three predicted coiled-coils, a leucine zipper, and a PROTEIN2 PROTEIN3 domain at the COOH terminus.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Thus, PROTEIN0 contains an NH(2)-terminal domain homologous to that implicated in PROTEIN1 endocytic function, three predicted coiled-coils, a leucine zipper, and a PROTEIN2 PROTEIN3 domain at the COOH terminus.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, poly-L-proline binding, PROTEIN0 binding, and PROTEIN1 nucleotide exchange are each independent requirements for PROTEIN2 function in fission yeast.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, poly-L-proline binding, PROTEIN0 binding, and PROTEIN1 nucleotide exchange are each independent requirements for PROTEIN2 function in fission yeast.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, poly-L-proline binding, PROTEIN0 binding, and PROTEIN1 nucleotide exchange are each independent requirements for PROTEIN2 function in fission yeast.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, the PROTEIN0 dodecapeptide sequence of PROTEIN1 may constitute a multifunctional domain in PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, the PROTEIN0 dodecapeptide sequence of PROTEIN1 may constitute a multifunctional domain in PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, the PROTEIN0 dodecapeptide sequence of PROTEIN1 may constitute a multifunctional domain in PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have previously shown that the synthetic dodecapeptide corresponding to Trp104-Met115 of PROTEIN0 is a potent inhibitor of PROTEIN1 polymerization (Yonezawa, N., Nishida, E., Iida, K., Kumagai, H., Yahara, I., and Sakai, H. (1991) J. Biol. Chem. 266, 10485-10489).\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 also inhibited muscle differentiation as measured by several parameters, including cell fusion and the expression of other muscle-specific genes, such as PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 also inhibited muscle differentiation as measured by several parameters, including cell fusion and the expression of other muscle-specific genes, such as PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 also inhibited muscle differentiation as measured by several parameters, including cell fusion and the expression of other muscle-specific genes, such as PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 To characterize the phenotypic alteration in mesangial cells in human glomerulonephritis, we investigated the expression of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in PROTEIN4 nephropathy.\n",
      "\n",
      "PROTEIN0 PROTEIN2 To characterize the phenotypic alteration in mesangial cells in human glomerulonephritis, we investigated the expression of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in PROTEIN4 nephropathy.\n",
      "\n",
      "PROTEIN0 PROTEIN3 To characterize the phenotypic alteration in mesangial cells in human glomerulonephritis, we investigated the expression of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in PROTEIN4 nephropathy.\n",
      "\n",
      "PROTEIN0 PROTEIN4 To characterize the phenotypic alteration in mesangial cells in human glomerulonephritis, we investigated the expression of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in PROTEIN4 nephropathy.\n",
      "\n",
      "PROTEIN1 PROTEIN2 To characterize the phenotypic alteration in mesangial cells in human glomerulonephritis, we investigated the expression of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in PROTEIN4 nephropathy.\n",
      "\n",
      "PROTEIN1 PROTEIN3 To characterize the phenotypic alteration in mesangial cells in human glomerulonephritis, we investigated the expression of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in PROTEIN4 nephropathy.\n",
      "\n",
      "PROTEIN1 PROTEIN4 To characterize the phenotypic alteration in mesangial cells in human glomerulonephritis, we investigated the expression of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in PROTEIN4 nephropathy.\n",
      "\n",
      "PROTEIN2 PROTEIN3 To characterize the phenotypic alteration in mesangial cells in human glomerulonephritis, we investigated the expression of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in PROTEIN4 nephropathy.\n",
      "\n",
      "PROTEIN2 PROTEIN4 To characterize the phenotypic alteration in mesangial cells in human glomerulonephritis, we investigated the expression of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in PROTEIN4 nephropathy.\n",
      "\n",
      "PROTEIN3 PROTEIN4 To characterize the phenotypic alteration in mesangial cells in human glomerulonephritis, we investigated the expression of PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 in PROTEIN4 nephropathy.\n",
      "\n",
      "PROTEIN0 PROTEIN1 To check its applicability, well characterized, commercially available antibodies (against PROTEIN0 PROTEIN1 and PROTEIN2 were used on sections of human small intestine.\n",
      "\n",
      "PROTEIN0 PROTEIN2 To check its applicability, well characterized, commercially available antibodies (against PROTEIN0 PROTEIN1 and PROTEIN2 were used on sections of human small intestine.\n",
      "\n",
      "PROTEIN1 PROTEIN2 To check its applicability, well characterized, commercially available antibodies (against PROTEIN0 PROTEIN1 and PROTEIN2 were used on sections of human small intestine.\n",
      "\n",
      "PROTEIN0 PROTEIN1 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN0 PROTEIN2 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN0 PROTEIN3 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN0 PROTEIN4 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN0 PROTEIN5 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN1 PROTEIN2 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN1 PROTEIN3 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN1 PROTEIN4 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN1 PROTEIN5 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN2 PROTEIN3 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN2 PROTEIN4 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN2 PROTEIN5 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN3 PROTEIN4 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN3 PROTEIN5 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN4 PROTEIN5 To clarify the relation between alteration of expression of PROTEIN0 and progression of extrahepatic bile duct carcinomas 55 cases were immunohistochemically examined for PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 with additional reverse PROTEIN5 chain reaction and Southern blotting hybridization (RT-PCR/SBH) assays.\n",
      "\n",
      "PROTEIN0 PROTEIN1 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN2 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN3 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN4 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN5 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN1 PROTEIN2 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN1 PROTEIN3 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN1 PROTEIN4 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN1 PROTEIN5 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN2 PROTEIN3 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN2 PROTEIN4 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN2 PROTEIN5 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN3 PROTEIN4 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN3 PROTEIN5 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN4 PROTEIN5 To determine the phenotypes of smooth muscle cells (SMCs) in such lesions, the authors conducted an immunohistochemical analysis of lung tissues from two patients with PPH, using two PROTEIN0 antibodies, PROTEIN1 and PROTEIN2 and two PROTEIN3 markers, PROTEIN4 and PROTEIN5 antibodies and related antibodies.\n",
      "\n",
      "PROTEIN0 PROTEIN1 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN0 PROTEIN2 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN0 PROTEIN3 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN0 PROTEIN4 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN0 PROTEIN5 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN0 PROTEIN6 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN1 PROTEIN2 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN1 PROTEIN3 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PROTEIN1 PROTEIN4 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN1 PROTEIN5 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN1 PROTEIN6 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN2 PROTEIN3 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN2 PROTEIN4 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN2 PROTEIN5 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN2 PROTEIN6 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN3 PROTEIN4 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN3 PROTEIN5 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN3 PROTEIN6 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN4 PROTEIN5 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN4 PROTEIN6 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN5 PROTEIN6 To determine the relationship between cell cycle regulation and differentiation, the spatiotemporal expression of PROTEIN0 PROTEIN1 PROTEIN2 the PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and markers of differentiating podocytes in developing human kidneys was investigated by immunohistochemistry.\n",
      "\n",
      "PROTEIN0 PROTEIN1 To determine the relative importance of protein degradation in the development of starvation-induced cardiac atrophy, in vivo fractional synthetic rates of total cardiac protein, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were measured in fed and fasted rabbits by continuous infusion of [3H] leucine.\n",
      "\n",
      "PROTEIN0 PROTEIN2 To determine the relative importance of protein degradation in the development of starvation-induced cardiac atrophy, in vivo fractional synthetic rates of total cardiac protein, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were measured in fed and fasted rabbits by continuous infusion of [3H] leucine.\n",
      "\n",
      "PROTEIN0 PROTEIN3 To determine the relative importance of protein degradation in the development of starvation-induced cardiac atrophy, in vivo fractional synthetic rates of total cardiac protein, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were measured in fed and fasted rabbits by continuous infusion of [3H] leucine.\n",
      "\n",
      "PROTEIN1 PROTEIN2 To determine the relative importance of protein degradation in the development of starvation-induced cardiac atrophy, in vivo fractional synthetic rates of total cardiac protein, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were measured in fed and fasted rabbits by continuous infusion of [3H] leucine.\n",
      "\n",
      "PROTEIN1 PROTEIN3 To determine the relative importance of protein degradation in the development of starvation-induced cardiac atrophy, in vivo fractional synthetic rates of total cardiac protein, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were measured in fed and fasted rabbits by continuous infusion of [3H] leucine.\n",
      "\n",
      "PROTEIN2 PROTEIN3 To determine the relative importance of protein degradation in the development of starvation-induced cardiac atrophy, in vivo fractional synthetic rates of total cardiac protein, PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 were measured in fed and fasted rabbits by continuous infusion of [3H] leucine.\n",
      "\n",
      "PROTEIN0 PROTEIN1 To determine whether alpha 1-adrenergic stimulation produced similar effects on the turnover of myofibrillar proteins, rates of synthesis and degradation were estimated for a myofibrillar-enriched protein fraction and for PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Together these results indicate that PROTEIN0 regulates PROTEIN1 gene expression through an PROTEIN2 site and its cognate transcription factor PROTEIN3 and requires the involvement of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Together these results indicate that PROTEIN0 regulates PROTEIN1 gene expression through an PROTEIN2 site and its cognate transcription factor PROTEIN3 and requires the involvement of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Together these results indicate that PROTEIN0 regulates PROTEIN1 gene expression through an PROTEIN2 site and its cognate transcription factor PROTEIN3 and requires the involvement of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Together these results indicate that PROTEIN0 regulates PROTEIN1 gene expression through an PROTEIN2 site and its cognate transcription factor PROTEIN3 and requires the involvement of PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Together these results indicate that PROTEIN0 regulates PROTEIN1 gene expression through an PROTEIN2 site and its cognate transcription factor PROTEIN3 and requires the involvement of PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Together these results indicate that PROTEIN0 regulates PROTEIN1 gene expression through an PROTEIN2 site and its cognate transcription factor PROTEIN3 and requires the involvement of PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Together these results indicate that PROTEIN0 regulates PROTEIN1 gene expression through an PROTEIN2 site and its cognate transcription factor PROTEIN3 and requires the involvement of PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Together these results indicate that PROTEIN0 regulates PROTEIN1 gene expression through an PROTEIN2 site and its cognate transcription factor PROTEIN3 and requires the involvement of PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Together these results indicate that PROTEIN0 regulates PROTEIN1 gene expression through an PROTEIN2 site and its cognate transcription factor PROTEIN3 and requires the involvement of PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Together these results indicate that PROTEIN0 regulates PROTEIN1 gene expression through an PROTEIN2 site and its cognate transcription factor PROTEIN3 and requires the involvement of PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 To improve our ability to study these proteins, we have expressed PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in insect cells by using the baculovirus expression system.\n",
      "\n",
      "PROTEIN0 PROTEIN2 To improve our ability to study these proteins, we have expressed PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in insect cells by using the baculovirus expression system.\n",
      "\n",
      "PROTEIN0 PROTEIN3 To improve our ability to study these proteins, we have expressed PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in insect cells by using the baculovirus expression system.\n",
      "\n",
      "PROTEIN1 PROTEIN2 To improve our ability to study these proteins, we have expressed PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in insect cells by using the baculovirus expression system.\n",
      "\n",
      "PROTEIN1 PROTEIN3 To improve our ability to study these proteins, we have expressed PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in insect cells by using the baculovirus expression system.\n",
      "\n",
      "PROTEIN2 PROTEIN3 To improve our ability to study these proteins, we have expressed PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in insect cells by using the baculovirus expression system.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Various immunological assays suggest that four of these proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are made in infected cells in very low abundance relative to the other three.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Various immunological assays suggest that four of these proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are made in infected cells in very low abundance relative to the other three.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Various immunological assays suggest that four of these proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are made in infected cells in very low abundance relative to the other three.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Various immunological assays suggest that four of these proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are made in infected cells in very low abundance relative to the other three.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Various immunological assays suggest that four of these proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are made in infected cells in very low abundance relative to the other three.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Various immunological assays suggest that four of these proteins PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 are made in infected cells in very low abundance relative to the other three.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN0 PROTEIN7 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN0 PROTEIN8 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN0 PROTEIN9 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN1 PROTEIN7 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN1 PROTEIN8 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN1 PROTEIN9 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN2 PROTEIN7 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN2 PROTEIN8 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN2 PROTEIN9 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN3 PROTEIN7 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN3 PROTEIN8 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN3 PROTEIN9 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN4 PROTEIN7 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN4 PROTEIN8 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN4 PROTEIN9 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN5 PROTEIN7 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN5 PROTEIN8 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN5 PROTEIN9 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN6 PROTEIN7 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN6 PROTEIN8 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN6 PROTEIN9 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN7 PROTEIN8 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN7 PROTEIN9 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN8 PROTEIN9 Transcriptional expression of PROTEIN0 PROTEIN1 as well as that of PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 PROTEIN6 PROTEIN7 and PROTEIN8 was similar in these cell lines, indicating that the susceptibility to PROTEIN9 apoptosis may not be determined by the constitutive expression level of these factors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Transcriptional regulation in the galactose regulon of yeast is determined by an interplay between a positive regulatory protein, PROTEIN0 and a negative regulatory protein, PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Transfection of PROTEIN0 colon carcinoma cells recruited PROTEIN1 and PROTEIN2 to cell-cell contacts and functional PROTEIN3 cell-cell adhesion was restored in this way.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Transfection of PROTEIN0 colon carcinoma cells recruited PROTEIN1 and PROTEIN2 to cell-cell contacts and functional PROTEIN3 cell-cell adhesion was restored in this way.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Transfection of PROTEIN0 colon carcinoma cells recruited PROTEIN1 and PROTEIN2 to cell-cell contacts and functional PROTEIN3 cell-cell adhesion was restored in this way.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Transfection of PROTEIN0 colon carcinoma cells recruited PROTEIN1 and PROTEIN2 to cell-cell contacts and functional PROTEIN3 cell-cell adhesion was restored in this way.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Transfection of PROTEIN0 colon carcinoma cells recruited PROTEIN1 and PROTEIN2 to cell-cell contacts and functional PROTEIN3 cell-cell adhesion was restored in this way.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Transfection of PROTEIN0 colon carcinoma cells recruited PROTEIN1 and PROTEIN2 to cell-cell contacts and functional PROTEIN3 cell-cell adhesion was restored in this way.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Treatment of cells with gamma linolenic acid (GLA) increased PROTEIN0 expression in most cell lines, while PROTEIN1 levels were reduced, and PROTEIN2 expression was unchanged.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Treatment of cells with gamma linolenic acid (GLA) increased PROTEIN0 expression in most cell lines, while PROTEIN1 levels were reduced, and PROTEIN2 expression was unchanged.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Treatment of cells with gamma linolenic acid (GLA) increased PROTEIN0 expression in most cell lines, while PROTEIN1 levels were reduced, and PROTEIN2 expression was unchanged.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Treatment with HNE resulted in activation of PROTEIN0 PROTEIN1 and PROTEIN2 induction of PROTEIN3 and PROTEIN4 protein expression, and an increase in transcription factor PROTEIN5 DNA binding activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Two cDNAs, isolated from a Xenopus laevis embryonic library, encode proteins of 168 amino acids, both of which are 77% identical to chick PROTEIN0 and 66% identical to chick PROTEIN1 PROTEIN2 two structurally and functionally related proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Two cDNAs, isolated from a Xenopus laevis embryonic library, encode proteins of 168 amino acids, both of which are 77% identical to chick PROTEIN0 and 66% identical to chick PROTEIN1 PROTEIN2 two structurally and functionally related proteins.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Two cDNAs, isolated from a Xenopus laevis embryonic library, encode proteins of 168 amino acids, both of which are 77% identical to chick PROTEIN0 and 66% identical to chick PROTEIN1 PROTEIN2 two structurally and functionally related proteins.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Under all conditions, platelets retained 40-50% of their total PROTEIN0 and greater than 70% of their PROTEIN1 PROTEIN2 but lost greater than 80% of PROTEIN3 and PROTEIN4 to the supernatant.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Under all conditions, platelets retained 40-50% of their total PROTEIN0 and greater than 70% of their PROTEIN1 PROTEIN2 but lost greater than 80% of PROTEIN3 and PROTEIN4 to the supernatant.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Under all conditions, platelets retained 40-50% of their total PROTEIN0 and greater than 70% of their PROTEIN1 PROTEIN2 but lost greater than 80% of PROTEIN3 and PROTEIN4 to the supernatant.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Under all conditions, platelets retained 40-50% of their total PROTEIN0 and greater than 70% of their PROTEIN1 PROTEIN2 but lost greater than 80% of PROTEIN3 and PROTEIN4 to the supernatant.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Under all conditions, platelets retained 40-50% of their total PROTEIN0 and greater than 70% of their PROTEIN1 PROTEIN2 but lost greater than 80% of PROTEIN3 and PROTEIN4 to the supernatant.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Under all conditions, platelets retained 40-50% of their total PROTEIN0 and greater than 70% of their PROTEIN1 PROTEIN2 but lost greater than 80% of PROTEIN3 and PROTEIN4 to the supernatant.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Under all conditions, platelets retained 40-50% of their total PROTEIN0 and greater than 70% of their PROTEIN1 PROTEIN2 but lost greater than 80% of PROTEIN3 and PROTEIN4 to the supernatant.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Under all conditions, platelets retained 40-50% of their total PROTEIN0 and greater than 70% of their PROTEIN1 PROTEIN2 but lost greater than 80% of PROTEIN3 and PROTEIN4 to the supernatant.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Under all conditions, platelets retained 40-50% of their total PROTEIN0 and greater than 70% of their PROTEIN1 PROTEIN2 but lost greater than 80% of PROTEIN3 and PROTEIN4 to the supernatant.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Under all conditions, platelets retained 40-50% of their total PROTEIN0 and greater than 70% of their PROTEIN1 PROTEIN2 but lost greater than 80% of PROTEIN3 and PROTEIN4 to the supernatant.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Unexpectedly, treatment with the PROTEIN0 drug latrunculin-A disrupted the medial region targeting pattern, and cells deficient in the PROTEIN1 PROTEIN2 and PROTEIN3 also did not exhibit medial PROTEIN4 PROTEIN5 staining.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Unlike PROTEIN0 PROTEIN1 PROTEIN2 action was strictly calcium-dependent: the bundles were formed at pCa 7, but not at pCa 6.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Unlike PROTEIN0 PROTEIN1 PROTEIN2 action was strictly calcium-dependent: the bundles were formed at pCa 7, but not at pCa 6.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Unlike PROTEIN0 PROTEIN1 PROTEIN2 action was strictly calcium-dependent: the bundles were formed at pCa 7, but not at pCa 6.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Unmasking of this site serves as a molecular switch that initiates assembly of an PROTEIN0 motility complex containing PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Unmasking of this site serves as a molecular switch that initiates assembly of an PROTEIN0 motility complex containing PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Unmasking of this site serves as a molecular switch that initiates assembly of an PROTEIN0 motility complex containing PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Using ITP (a substrate for PROTEIN0 PROTEIN1 and/or ATP gamma S (a substrate for PROTEIN2 PROTEIN3 we demonstrated that phosphorylation of PROTEIN4 occurred at the foci within the PROTEIN5 network, a result that suggests that PROTEIN6 was localized at the foci.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using the known three-dimensional structure of the homologous PROTEIN0 domain of PROTEIN1 these results have enabled us to determine the likely orientation of the PROTEIN2 PROTEIN3 domain with respect to the PROTEIN4 filament.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using the known three-dimensional structure of the homologous PROTEIN0 domain of PROTEIN1 these results have enabled us to determine the likely orientation of the PROTEIN2 PROTEIN3 domain with respect to the PROTEIN4 filament.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Using the known three-dimensional structure of the homologous PROTEIN0 domain of PROTEIN1 these results have enabled us to determine the likely orientation of the PROTEIN2 PROTEIN3 domain with respect to the PROTEIN4 filament.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Using the known three-dimensional structure of the homologous PROTEIN0 domain of PROTEIN1 these results have enabled us to determine the likely orientation of the PROTEIN2 PROTEIN3 domain with respect to the PROTEIN4 filament.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using the known three-dimensional structure of the homologous PROTEIN0 domain of PROTEIN1 these results have enabled us to determine the likely orientation of the PROTEIN2 PROTEIN3 domain with respect to the PROTEIN4 filament.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Using the known three-dimensional structure of the homologous PROTEIN0 domain of PROTEIN1 these results have enabled us to determine the likely orientation of the PROTEIN2 PROTEIN3 domain with respect to the PROTEIN4 filament.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Using the known three-dimensional structure of the homologous PROTEIN0 domain of PROTEIN1 these results have enabled us to determine the likely orientation of the PROTEIN2 PROTEIN3 domain with respect to the PROTEIN4 filament.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Using the known three-dimensional structure of the homologous PROTEIN0 domain of PROTEIN1 these results have enabled us to determine the likely orientation of the PROTEIN2 PROTEIN3 domain with respect to the PROTEIN4 filament.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Using the known three-dimensional structure of the homologous PROTEIN0 domain of PROTEIN1 these results have enabled us to determine the likely orientation of the PROTEIN2 PROTEIN3 domain with respect to the PROTEIN4 filament.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Using the known three-dimensional structure of the homologous PROTEIN0 domain of PROTEIN1 these results have enabled us to determine the likely orientation of the PROTEIN2 PROTEIN3 domain with respect to the PROTEIN4 filament.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using the 'nuclear displacement assay' as a measure of the integrity of the PROTEIN0 cytoskeleton in living stamen hair cells, we demonstrated that PROTEIN1 protects PROTEIN2 filaments in these cells from Z. mays PROTEIN3 PROTEIN4 depolymerization, in a dose-dependent manner.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Using the 'nuclear displacement assay' as a measure of the integrity of the PROTEIN0 cytoskeleton in living stamen hair cells, we demonstrated that PROTEIN1 protects PROTEIN2 filaments in these cells from Z. mays PROTEIN3 PROTEIN4 depolymerization, in a dose-dependent manner.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Using the 'nuclear displacement assay' as a measure of the integrity of the PROTEIN0 cytoskeleton in living stamen hair cells, we demonstrated that PROTEIN1 protects PROTEIN2 filaments in these cells from Z. mays PROTEIN3 PROTEIN4 depolymerization, in a dose-dependent manner.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Using the 'nuclear displacement assay' as a measure of the integrity of the PROTEIN0 cytoskeleton in living stamen hair cells, we demonstrated that PROTEIN1 protects PROTEIN2 filaments in these cells from Z. mays PROTEIN3 PROTEIN4 depolymerization, in a dose-dependent manner.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Using the 'nuclear displacement assay' as a measure of the integrity of the PROTEIN0 cytoskeleton in living stamen hair cells, we demonstrated that PROTEIN1 protects PROTEIN2 filaments in these cells from Z. mays PROTEIN3 PROTEIN4 depolymerization, in a dose-dependent manner.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Using the 'nuclear displacement assay' as a measure of the integrity of the PROTEIN0 cytoskeleton in living stamen hair cells, we demonstrated that PROTEIN1 protects PROTEIN2 filaments in these cells from Z. mays PROTEIN3 PROTEIN4 depolymerization, in a dose-dependent manner.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Using the 'nuclear displacement assay' as a measure of the integrity of the PROTEIN0 cytoskeleton in living stamen hair cells, we demonstrated that PROTEIN1 protects PROTEIN2 filaments in these cells from Z. mays PROTEIN3 PROTEIN4 depolymerization, in a dose-dependent manner.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Using the 'nuclear displacement assay' as a measure of the integrity of the PROTEIN0 cytoskeleton in living stamen hair cells, we demonstrated that PROTEIN1 protects PROTEIN2 filaments in these cells from Z. mays PROTEIN3 PROTEIN4 depolymerization, in a dose-dependent manner.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Using the 'nuclear displacement assay' as a measure of the integrity of the PROTEIN0 cytoskeleton in living stamen hair cells, we demonstrated that PROTEIN1 protects PROTEIN2 filaments in these cells from Z. mays PROTEIN3 PROTEIN4 depolymerization, in a dose-dependent manner.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Using the 'nuclear displacement assay' as a measure of the integrity of the PROTEIN0 cytoskeleton in living stamen hair cells, we demonstrated that PROTEIN1 protects PROTEIN2 filaments in these cells from Z. mays PROTEIN3 PROTEIN4 depolymerization, in a dose-dependent manner.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Using these methods, we confirm previous reports that intact PROTEIN0 induces cross-linking as well as filament shortening on PROTEIN1 networks.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 clusters were co-localized with PROTEIN2 an important signal transduction ligand, and with PROTEIN3 which is thought to link the complex to the peri-junctional PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 clusters were co-localized with PROTEIN2 an important signal transduction ligand, and with PROTEIN3 which is thought to link the complex to the peri-junctional PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 clusters were co-localized with PROTEIN2 an important signal transduction ligand, and with PROTEIN3 which is thought to link the complex to the peri-junctional PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 PROTEIN1 clusters were co-localized with PROTEIN2 an important signal transduction ligand, and with PROTEIN3 which is thought to link the complex to the peri-junctional PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 clusters were co-localized with PROTEIN2 an important signal transduction ligand, and with PROTEIN3 which is thought to link the complex to the peri-junctional PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 clusters were co-localized with PROTEIN2 an important signal transduction ligand, and with PROTEIN3 which is thought to link the complex to the peri-junctional PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 PROTEIN1 clusters were co-localized with PROTEIN2 an important signal transduction ligand, and with PROTEIN3 which is thought to link the complex to the peri-junctional PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 clusters were co-localized with PROTEIN2 an important signal transduction ligand, and with PROTEIN3 which is thought to link the complex to the peri-junctional PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 PROTEIN1 clusters were co-localized with PROTEIN2 an important signal transduction ligand, and with PROTEIN3 which is thought to link the complex to the peri-junctional PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 PROTEIN1 clusters were co-localized with PROTEIN2 an important signal transduction ligand, and with PROTEIN3 which is thought to link the complex to the peri-junctional PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 in turn, provides the ABM-2 sequences [XPPPPP, X = G, P, L, S, A] for binding PROTEIN1 an PROTEIN2 protein that stimulates PROTEIN3 filament assembly.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 in turn, provides the ABM-2 sequences [XPPPPP, X = G, P, L, S, A] for binding PROTEIN1 an PROTEIN2 protein that stimulates PROTEIN3 filament assembly.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 in turn, provides the ABM-2 sequences [XPPPPP, X = G, P, L, S, A] for binding PROTEIN1 an PROTEIN2 protein that stimulates PROTEIN3 filament assembly.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 in turn, provides the ABM-2 sequences [XPPPPP, X = G, P, L, S, A] for binding PROTEIN1 an PROTEIN2 protein that stimulates PROTEIN3 filament assembly.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 in turn, provides the ABM-2 sequences [XPPPPP, X = G, P, L, S, A] for binding PROTEIN1 an PROTEIN2 protein that stimulates PROTEIN3 filament assembly.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 in turn, provides the ABM-2 sequences [XPPPPP, X = G, P, L, S, A] for binding PROTEIN1 an PROTEIN2 protein that stimulates PROTEIN3 filament assembly.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 are two PROTEIN2 that bundle PROTEIN3 filaments in the intestine and kidney brush border epithelium.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 and PROTEIN1 are two PROTEIN2 that bundle PROTEIN3 filaments in the intestine and kidney brush border epithelium.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 and PROTEIN1 are two PROTEIN2 that bundle PROTEIN3 filaments in the intestine and kidney brush border epithelium.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 and PROTEIN1 are two PROTEIN2 that bundle PROTEIN3 filaments in the intestine and kidney brush border epithelium.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 and PROTEIN1 are two PROTEIN2 that bundle PROTEIN3 filaments in the intestine and kidney brush border epithelium.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 and PROTEIN1 are two PROTEIN2 that bundle PROTEIN3 filaments in the intestine and kidney brush border epithelium.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN0 PROTEIN6 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN0 PROTEIN7 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN0 PROTEIN8 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN1 PROTEIN6 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN1 PROTEIN7 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN1 PROTEIN8 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN2 PROTEIN6 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN2 PROTEIN7 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN2 PROTEIN8 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN3 PROTEIN6 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN3 PROTEIN7 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN3 PROTEIN8 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN4 PROTEIN6 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN4 PROTEIN7 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN4 PROTEIN8 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN5 PROTEIN6 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN5 PROTEIN7 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN5 PROTEIN8 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN6 PROTEIN7 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN6 PROTEIN8 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN7 PROTEIN8 PROTEIN0 contains a binding motif for the PROTEIN1 PROTEIN2 as well as PROTEIN3 PROTEIN4 PROTEIN5 domains, but no specific PROTEIN6 structure regulated by PROTEIN7 PROTEIN8 has been identified.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We also analysed NHEJ in other DNA damage response mutants and showed that the checkpoint mutant PROTEIN0 and two recombination mutants defective in PROTEIN1 and PROTEIN2 (homologues of S.cerevisiae PROTEIN3 and PROTEIN4 respectively) are not affected.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We also analysed NHEJ in other DNA damage response mutants and showed that the checkpoint mutant PROTEIN0 and two recombination mutants defective in PROTEIN1 and PROTEIN2 (homologues of S.cerevisiae PROTEIN3 and PROTEIN4 respectively) are not affected.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We also analysed NHEJ in other DNA damage response mutants and showed that the checkpoint mutant PROTEIN0 and two recombination mutants defective in PROTEIN1 and PROTEIN2 (homologues of S.cerevisiae PROTEIN3 and PROTEIN4 respectively) are not affected.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We also analysed NHEJ in other DNA damage response mutants and showed that the checkpoint mutant PROTEIN0 and two recombination mutants defective in PROTEIN1 and PROTEIN2 (homologues of S.cerevisiae PROTEIN3 and PROTEIN4 respectively) are not affected.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We also analysed NHEJ in other DNA damage response mutants and showed that the checkpoint mutant PROTEIN0 and two recombination mutants defective in PROTEIN1 and PROTEIN2 (homologues of S.cerevisiae PROTEIN3 and PROTEIN4 respectively) are not affected.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We also analysed NHEJ in other DNA damage response mutants and showed that the checkpoint mutant PROTEIN0 and two recombination mutants defective in PROTEIN1 and PROTEIN2 (homologues of S.cerevisiae PROTEIN3 and PROTEIN4 respectively) are not affected.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We also analysed NHEJ in other DNA damage response mutants and showed that the checkpoint mutant PROTEIN0 and two recombination mutants defective in PROTEIN1 and PROTEIN2 (homologues of S.cerevisiae PROTEIN3 and PROTEIN4 respectively) are not affected.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We also analysed NHEJ in other DNA damage response mutants and showed that the checkpoint mutant PROTEIN0 and two recombination mutants defective in PROTEIN1 and PROTEIN2 (homologues of S.cerevisiae PROTEIN3 and PROTEIN4 respectively) are not affected.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We also analysed NHEJ in other DNA damage response mutants and showed that the checkpoint mutant PROTEIN0 and two recombination mutants defective in PROTEIN1 and PROTEIN2 (homologues of S.cerevisiae PROTEIN3 and PROTEIN4 respectively) are not affected.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We also analysed NHEJ in other DNA damage response mutants and showed that the checkpoint mutant PROTEIN0 and two recombination mutants defective in PROTEIN1 and PROTEIN2 (homologues of S.cerevisiae PROTEIN3 and PROTEIN4 respectively) are not affected.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We believe that this complex may mediate the cortical functions of PROTEIN0 at PROTEIN1 patches in S. pombe.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We constructed recombinant viruses based on the herpes simplex virus type 1 mutant tsK which individually were able to express the products of four viral DNA replication genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in the absence of any of the other proteins required for viral DNA synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We constructed recombinant viruses based on the herpes simplex virus type 1 mutant tsK which individually were able to express the products of four viral DNA replication genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in the absence of any of the other proteins required for viral DNA synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We constructed recombinant viruses based on the herpes simplex virus type 1 mutant tsK which individually were able to express the products of four viral DNA replication genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in the absence of any of the other proteins required for viral DNA synthesis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We constructed recombinant viruses based on the herpes simplex virus type 1 mutant tsK which individually were able to express the products of four viral DNA replication genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in the absence of any of the other proteins required for viral DNA synthesis.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We constructed recombinant viruses based on the herpes simplex virus type 1 mutant tsK which individually were able to express the products of four viral DNA replication genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in the absence of any of the other proteins required for viral DNA synthesis.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We constructed recombinant viruses based on the herpes simplex virus type 1 mutant tsK which individually were able to express the products of four viral DNA replication genes PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 in the absence of any of the other proteins required for viral DNA synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN0 PROTEIN5 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN0 PROTEIN6 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN0 PROTEIN7 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN1 PROTEIN5 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN1 PROTEIN6 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN1 PROTEIN7 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN2 PROTEIN5 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN2 PROTEIN6 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "PROTEIN2 PROTEIN7 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN3 PROTEIN5 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN3 PROTEIN6 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN3 PROTEIN7 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN4 PROTEIN5 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN4 PROTEIN6 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN4 PROTEIN7 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN5 PROTEIN6 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN5 PROTEIN7 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN6 PROTEIN7 We describe the localization of PROTEIN0 and PROTEIN1 along the cell margins of mouse osteoblast-like cells, the colocalization of PROTEIN2 with PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 and the association of these complexes with the PROTEIN7 microfilaments.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We determined whether short-term weight-lifting exercise increases the synthesis rate of the major contractile proteins, PROTEIN0 PROTEIN1 PROTEIN2 and mixed muscle proteins in nonfrail elders and younger women and men.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We determined whether short-term weight-lifting exercise increases the synthesis rate of the major contractile proteins, PROTEIN0 PROTEIN1 PROTEIN2 and mixed muscle proteins in nonfrail elders and younger women and men.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We determined whether short-term weight-lifting exercise increases the synthesis rate of the major contractile proteins, PROTEIN0 PROTEIN1 PROTEIN2 and mixed muscle proteins in nonfrail elders and younger women and men.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We examined whether levels of PROTEIN0 or PROTEIN1 correlated with PROTEIN2 levels in the glomeruli of rats with puromycin aminonucleoside nephrosis.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We examined whether levels of PROTEIN0 or PROTEIN1 correlated with PROTEIN2 levels in the glomeruli of rats with puromycin aminonucleoside nephrosis.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We examined whether levels of PROTEIN0 or PROTEIN1 correlated with PROTEIN2 levels in the glomeruli of rats with puromycin aminonucleoside nephrosis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We find that PROTEIN0 and PROTEIN1 regulate PROTEIN2 formation throughout the cell cortex.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We find that PROTEIN0 and PROTEIN1 regulate PROTEIN2 formation throughout the cell cortex.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We find that PROTEIN0 and PROTEIN1 regulate PROTEIN2 formation throughout the cell cortex.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We find that PROTEIN0 contributes in several ways to PROTEIN1 nucleation of PROTEIN2 filaments in high speed supernatant of lysed neutrophils.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We find that PROTEIN0 contributes in several ways to PROTEIN1 nucleation of PROTEIN2 filaments in high speed supernatant of lysed neutrophils.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We find that PROTEIN0 contributes in several ways to PROTEIN1 nucleation of PROTEIN2 filaments in high speed supernatant of lysed neutrophils.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We found normal levels of PROTEIN0 and PROTEIN1 in BT549 and HS578t cells; however, low levels of PROTEIN2 were expressed in these cells compared to those found in weakly invasive MCF-7 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We found normal levels of PROTEIN0 and PROTEIN1 in BT549 and HS578t cells; however, low levels of PROTEIN2 were expressed in these cells compared to those found in weakly invasive MCF-7 cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We found normal levels of PROTEIN0 and PROTEIN1 in BT549 and HS578t cells; however, low levels of PROTEIN2 were expressed in these cells compared to those found in weakly invasive MCF-7 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have analyzed the expression levels of the genes encoding PROTEIN0 PROTEIN1 and PROTEIN2 in correlation to the PROTEIN3 expression levels in cell lines derived from human cervical carcinomas.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have analyzed the expression levels of the genes encoding PROTEIN0 PROTEIN1 and PROTEIN2 in correlation to the PROTEIN3 expression levels in cell lines derived from human cervical carcinomas.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We have analyzed the expression levels of the genes encoding PROTEIN0 PROTEIN1 and PROTEIN2 in correlation to the PROTEIN3 expression levels in cell lines derived from human cervical carcinomas.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have analyzed the expression levels of the genes encoding PROTEIN0 PROTEIN1 and PROTEIN2 in correlation to the PROTEIN3 expression levels in cell lines derived from human cervical carcinomas.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We have analyzed the expression levels of the genes encoding PROTEIN0 PROTEIN1 and PROTEIN2 in correlation to the PROTEIN3 expression levels in cell lines derived from human cervical carcinomas.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We have analyzed the expression levels of the genes encoding PROTEIN0 PROTEIN1 and PROTEIN2 in correlation to the PROTEIN3 expression levels in cell lines derived from human cervical carcinomas.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have characterized this PROTEIN0 with respect to its ability to interact with yeast PROTEIN1 and PROTEIN2 the only yeast PROTEIN3 so far purified and characterized.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have characterized this PROTEIN0 with respect to its ability to interact with yeast PROTEIN1 and PROTEIN2 the only yeast PROTEIN3 so far purified and characterized.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We have characterized this PROTEIN0 with respect to its ability to interact with yeast PROTEIN1 and PROTEIN2 the only yeast PROTEIN3 so far purified and characterized.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have characterized this PROTEIN0 with respect to its ability to interact with yeast PROTEIN1 and PROTEIN2 the only yeast PROTEIN3 so far purified and characterized.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We have characterized this PROTEIN0 with respect to its ability to interact with yeast PROTEIN1 and PROTEIN2 the only yeast PROTEIN3 so far purified and characterized.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We have characterized this PROTEIN0 with respect to its ability to interact with yeast PROTEIN1 and PROTEIN2 the only yeast PROTEIN3 so far purified and characterized.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Yeast PROTEIN0 can sequester yeast PROTEIN1 monomers and thereby reduce the ability of yeast PROTEIN2 to polymerize, whereas it has little effect on the degree of polymerization of rabbit PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Yeast PROTEIN0 can sequester yeast PROTEIN1 monomers and thereby reduce the ability of yeast PROTEIN2 to polymerize, whereas it has little effect on the degree of polymerization of rabbit PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Yeast PROTEIN0 can sequester yeast PROTEIN1 monomers and thereby reduce the ability of yeast PROTEIN2 to polymerize, whereas it has little effect on the degree of polymerization of rabbit PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Yeast PROTEIN0 can sequester yeast PROTEIN1 monomers and thereby reduce the ability of yeast PROTEIN2 to polymerize, whereas it has little effect on the degree of polymerization of rabbit PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Yeast PROTEIN0 can sequester yeast PROTEIN1 monomers and thereby reduce the ability of yeast PROTEIN2 to polymerize, whereas it has little effect on the degree of polymerization of rabbit PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Yeast PROTEIN0 can sequester yeast PROTEIN1 monomers and thereby reduce the ability of yeast PROTEIN2 to polymerize, whereas it has little effect on the degree of polymerization of rabbit PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have cloned a Saccharomyces cerevisiae gene PROTEIN0 encoding a low-M(r) PROTEIN1 of 143 amino acid (aa) residues (yeast PROTEIN2 PROTEIN3 its aa sequence is 35% identical to porcine PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have cloned a Saccharomyces cerevisiae gene PROTEIN0 encoding a low-M(r) PROTEIN1 of 143 amino acid (aa) residues (yeast PROTEIN2 PROTEIN3 its aa sequence is 35% identical to porcine PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 We have cloned a Saccharomyces cerevisiae gene PROTEIN0 encoding a low-M(r) PROTEIN1 of 143 amino acid (aa) residues (yeast PROTEIN2 PROTEIN3 its aa sequence is 35% identical to porcine PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 We have cloned a Saccharomyces cerevisiae gene PROTEIN0 encoding a low-M(r) PROTEIN1 of 143 amino acid (aa) residues (yeast PROTEIN2 PROTEIN3 its aa sequence is 35% identical to porcine PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have cloned a Saccharomyces cerevisiae gene PROTEIN0 encoding a low-M(r) PROTEIN1 of 143 amino acid (aa) residues (yeast PROTEIN2 PROTEIN3 its aa sequence is 35% identical to porcine PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 We have cloned a Saccharomyces cerevisiae gene PROTEIN0 encoding a low-M(r) PROTEIN1 of 143 amino acid (aa) residues (yeast PROTEIN2 PROTEIN3 its aa sequence is 35% identical to porcine PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 We have cloned a Saccharomyces cerevisiae gene PROTEIN0 encoding a low-M(r) PROTEIN1 of 143 amino acid (aa) residues (yeast PROTEIN2 PROTEIN3 its aa sequence is 35% identical to porcine PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 We have cloned a Saccharomyces cerevisiae gene PROTEIN0 encoding a low-M(r) PROTEIN1 of 143 amino acid (aa) residues (yeast PROTEIN2 PROTEIN3 its aa sequence is 35% identical to porcine PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 We have cloned a Saccharomyces cerevisiae gene PROTEIN0 encoding a low-M(r) PROTEIN1 of 143 amino acid (aa) residues (yeast PROTEIN2 PROTEIN3 its aa sequence is 35% identical to porcine PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 We have cloned a Saccharomyces cerevisiae gene PROTEIN0 encoding a low-M(r) PROTEIN1 of 143 amino acid (aa) residues (yeast PROTEIN2 PROTEIN3 its aa sequence is 35% identical to porcine PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have examined the level of PROTEIN0 a PROTEIN1 and PROTEIN2 a PROTEIN3 in BALB/c 3T3 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have examined the level of PROTEIN0 a PROTEIN1 and PROTEIN2 a PROTEIN3 in BALB/c 3T3 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We have examined the level of PROTEIN0 a PROTEIN1 and PROTEIN2 a PROTEIN3 in BALB/c 3T3 cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have examined the level of PROTEIN0 a PROTEIN1 and PROTEIN2 a PROTEIN3 in BALB/c 3T3 cells.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We have examined the level of PROTEIN0 a PROTEIN1 and PROTEIN2 a PROTEIN3 in BALB/c 3T3 cells.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We have examined the level of PROTEIN0 a PROTEIN1 and PROTEIN2 a PROTEIN3 in BALB/c 3T3 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have found that PROTEIN0 a 21-kDa PROTEIN1 is a component of these rods.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN5 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN6 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN7 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN1 PROTEIN5 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN1 PROTEIN6 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN1 PROTEIN7 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN2 PROTEIN5 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN2 PROTEIN6 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN2 PROTEIN7 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN3 PROTEIN5 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN3 PROTEIN6 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN3 PROTEIN7 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN4 PROTEIN5 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN4 PROTEIN6 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN4 PROTEIN7 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN5 PROTEIN6 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN5 PROTEIN7 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN6 PROTEIN7 We have shown that the PROTEIN0 PROTEIN1 and PROTEIN2 are potential targets of the PROTEIN3 family PROTEIN4 and bind to an PROTEIN5 PROTEIN6 at their proline-rich FH1 domains to regulate reorganization of the PROTEIN7 cytoskeleton in the yeast Saccharomyces cerevisiae.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 We have shown that these proteins are indeed PROTEIN0 the 14 kd is the PROTEIN1 PROTEIN2 the 19 kd is the PROTEIN3 PROTEIN4 and the 25 kd is the sperm-specific protein PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 We previously showed that PROTEIN0 is transported in an unassembled form with its associated proteins PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 We previously showed that PROTEIN0 is transported in an unassembled form with its associated proteins PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 We previously showed that PROTEIN0 is transported in an unassembled form with its associated proteins PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 We previously showed that PROTEIN0 is transported in an unassembled form with its associated proteins PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 We previously showed that PROTEIN0 is transported in an unassembled form with its associated proteins PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 We previously showed that PROTEIN0 is transported in an unassembled form with its associated proteins PROTEIN1 PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN0 PROTEIN5 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN0 PROTEIN6 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN1 PROTEIN5 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN1 PROTEIN6 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN2 PROTEIN5 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN2 PROTEIN6 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN3 PROTEIN5 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN3 PROTEIN6 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN4 PROTEIN5 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN4 PROTEIN6 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN5 PROTEIN6 We show that the HSV-1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 and PROTEIN5 gene products along with the AAV PROTEIN6 protein are sufficient to initiate replication on duplex DNA containing the AAV origins of replication, resulting in products several hundred nucleotides in length.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We speculate that PROTEIN0 may help maintain the parallel growth of PROTEIN1 filaments within the stereocilia.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN2 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN3 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN4 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN5 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN6 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN7 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN8 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN2 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN3 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN4 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN5 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN6 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN7 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN1 PROTEIN8 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN3 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN4 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN5 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN6 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN7 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN2 PROTEIN8 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN4 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN5 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN6 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN7 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN3 PROTEIN8 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN5 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN6 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN7 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN4 PROTEIN8 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN5 PROTEIN6 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN5 PROTEIN7 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN5 PROTEIN8 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN6 PROTEIN7 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN6 PROTEIN8 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN7 PROTEIN8 Western analysis of gel-purified PROTEIN0 PROTEIN1 and PROTEIN2 PROTEIN3 PROTEIN4 complexes demonstrated the predominant presence of acetylated PROTEIN5 PROTEIN6 relative to acetylated PROTEIN7 PROTEIN8\n",
      "\n",
      "PROTEIN0 PROTEIN1 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN0 PROTEIN2 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN0 PROTEIN3 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN0 PROTEIN4 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN0 PROTEIN5 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN0 PROTEIN6 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN1 PROTEIN2 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN1 PROTEIN3 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN1 PROTEIN4 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN1 PROTEIN5 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN1 PROTEIN6 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN2 PROTEIN3 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN2 PROTEIN4 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN2 PROTEIN5 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN2 PROTEIN6 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN3 PROTEIN4 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN3 PROTEIN5 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN3 PROTEIN6 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN4 PROTEIN5 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN4 PROTEIN6 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN5 PROTEIN6 We used yeast as a model to test this hypothesis and found that chromosome deletion of any known nuclear mitotic mismatch repair genes, including PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 did not rescue PROTEIN5 O6 MeG DNA repair PROTEIN6 cells from killing by MNNG.\n",
      "\n",
      "PROTEIN0 PROTEIN1 When a lyophilized Mf and AO mixture was incubated at 40 degrees C and 65% relative humidity, the conjugation of AO was confirmed at PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 When a lyophilized Mf and AO mixture was incubated at 40 degrees C and 65% relative humidity, the conjugation of AO was confirmed at PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 When a lyophilized Mf and AO mixture was incubated at 40 degrees C and 65% relative humidity, the conjugation of AO was confirmed at PROTEIN0 PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 When the PROTEIN0 monomer pool in untransfected myoblasts is increased 70% by treatment with latrunculin A, synthesis of PROTEIN1 and PROTEIN2 are down-regulated compared with PROTEIN3 and 19 other proteins selected at random.\n",
      "\n",
      "PROTEIN0 PROTEIN2 When the PROTEIN0 monomer pool in untransfected myoblasts is increased 70% by treatment with latrunculin A, synthesis of PROTEIN1 and PROTEIN2 are down-regulated compared with PROTEIN3 and 19 other proteins selected at random.\n",
      "\n",
      "PROTEIN0 PROTEIN3 When the PROTEIN0 monomer pool in untransfected myoblasts is increased 70% by treatment with latrunculin A, synthesis of PROTEIN1 and PROTEIN2 are down-regulated compared with PROTEIN3 and 19 other proteins selected at random.\n",
      "\n",
      "PROTEIN1 PROTEIN2 When the PROTEIN0 monomer pool in untransfected myoblasts is increased 70% by treatment with latrunculin A, synthesis of PROTEIN1 and PROTEIN2 are down-regulated compared with PROTEIN3 and 19 other proteins selected at random.\n",
      "\n",
      "PROTEIN1 PROTEIN3 When the PROTEIN0 monomer pool in untransfected myoblasts is increased 70% by treatment with latrunculin A, synthesis of PROTEIN1 and PROTEIN2 are down-regulated compared with PROTEIN3 and 19 other proteins selected at random.\n",
      "\n",
      "PROTEIN2 PROTEIN3 When the PROTEIN0 monomer pool in untransfected myoblasts is increased 70% by treatment with latrunculin A, synthesis of PROTEIN1 and PROTEIN2 are down-regulated compared with PROTEIN3 and 19 other proteins selected at random.\n",
      "\n",
      "PROTEIN0 PROTEIN1 While levels of most of these proteins decrease, accumulation of PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 is observed.\n",
      "\n",
      "PROTEIN0 PROTEIN2 While levels of most of these proteins decrease, accumulation of PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 is observed.\n",
      "\n",
      "PROTEIN0 PROTEIN3 While levels of most of these proteins decrease, accumulation of PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 is observed.\n",
      "\n",
      "PROTEIN0 PROTEIN4 While levels of most of these proteins decrease, accumulation of PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 is observed.\n",
      "\n",
      "PROTEIN1 PROTEIN2 While levels of most of these proteins decrease, accumulation of PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 is observed.\n",
      "\n",
      "PROTEIN1 PROTEIN3 While levels of most of these proteins decrease, accumulation of PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 is observed.\n",
      "\n",
      "PROTEIN1 PROTEIN4 While levels of most of these proteins decrease, accumulation of PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 is observed.\n",
      "\n",
      "PROTEIN2 PROTEIN3 While levels of most of these proteins decrease, accumulation of PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 is observed.\n",
      "\n",
      "PROTEIN2 PROTEIN4 While levels of most of these proteins decrease, accumulation of PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 is observed.\n",
      "\n",
      "PROTEIN3 PROTEIN4 While levels of most of these proteins decrease, accumulation of PROTEIN0 and the PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 is observed.\n",
      "\n",
      "PROTEIN0 PROTEIN1 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN2 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN3 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN4 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN5 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN2 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN3 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN4 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN1 PROTEIN5 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN3 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN4 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN2 PROTEIN5 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN4 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN3 PROTEIN5 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN4 PROTEIN5 While poly(ADP-ribose) effectively competed with DNA for binding of PROTEIN0 PROTEIN1 it equally competed with DNA for binding of PROTEIN2 PROTEIN3 and only inefficiently competed with DNA for binding of PROTEIN4 PROTEIN5\n",
      "\n",
      "PROTEIN0 PROTEIN1 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN4 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN5 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN6 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN7 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN8 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN4 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN5 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN6 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN7 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN1 PROTEIN8 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN2 PROTEIN4 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN2 PROTEIN5 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN2 PROTEIN6 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN2 PROTEIN7 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN2 PROTEIN8 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN3 PROTEIN4 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN3 PROTEIN5 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN3 PROTEIN6 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN3 PROTEIN7 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN3 PROTEIN8 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN4 PROTEIN5 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN4 PROTEIN6 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN4 PROTEIN7 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN4 PROTEIN8 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN5 PROTEIN6 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PROTEIN5 PROTEIN7 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN5 PROTEIN8 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN6 PROTEIN7 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN6 PROTEIN8 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN7 PROTEIN8 Within 1 hour of raising the concentration of calcium ions, PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 PROTEIN4 PROTEIN5 and PROTEIN6 appeared to accumulate at cell-cell borders, whereas the focal contact proteins, PROTEIN7 and PROTEIN8 did not.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Within the sieve-tube exudate, PROTEIN0 was present in 15-fold molar excess to PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 YACs containing PROTEIN0 gene DNA or 1.5 rDNA tandem repeat units were transformed into hosts bearing lesions at the PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4 loci.\n",
      "\n",
      "PROTEIN0 PROTEIN2 YACs containing PROTEIN0 gene DNA or 1.5 rDNA tandem repeat units were transformed into hosts bearing lesions at the PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4 loci.\n",
      "\n",
      "PROTEIN0 PROTEIN3 YACs containing PROTEIN0 gene DNA or 1.5 rDNA tandem repeat units were transformed into hosts bearing lesions at the PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4 loci.\n",
      "\n",
      "PROTEIN0 PROTEIN4 YACs containing PROTEIN0 gene DNA or 1.5 rDNA tandem repeat units were transformed into hosts bearing lesions at the PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4 loci.\n",
      "\n",
      "PROTEIN1 PROTEIN2 YACs containing PROTEIN0 gene DNA or 1.5 rDNA tandem repeat units were transformed into hosts bearing lesions at the PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4 loci.\n",
      "\n",
      "PROTEIN1 PROTEIN3 YACs containing PROTEIN0 gene DNA or 1.5 rDNA tandem repeat units were transformed into hosts bearing lesions at the PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4 loci.\n",
      "\n",
      "PROTEIN1 PROTEIN4 YACs containing PROTEIN0 gene DNA or 1.5 rDNA tandem repeat units were transformed into hosts bearing lesions at the PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4 loci.\n",
      "\n",
      "PROTEIN2 PROTEIN3 YACs containing PROTEIN0 gene DNA or 1.5 rDNA tandem repeat units were transformed into hosts bearing lesions at the PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4 loci.\n",
      "\n",
      "PROTEIN2 PROTEIN4 YACs containing PROTEIN0 gene DNA or 1.5 rDNA tandem repeat units were transformed into hosts bearing lesions at the PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4 loci.\n",
      "\n",
      "PROTEIN3 PROTEIN4 YACs containing PROTEIN0 gene DNA or 1.5 rDNA tandem repeat units were transformed into hosts bearing lesions at the PROTEIN1 PROTEIN2 PROTEIN3 or PROTEIN4 loci.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cloning and characterization of two evolutionarily conserved subunits PROTEIN0 and PROTEIN1 of human PROTEIN2 and their involvement in functional interactions with TFIIIB and RNA polymerase III\n",
      "\n",
      "PROTEIN0 PROTEIN2 Cloning and characterization of two evolutionarily conserved subunits PROTEIN0 and PROTEIN1 of human PROTEIN2 and their involvement in functional interactions with TFIIIB and RNA polymerase III\n",
      "\n",
      "PROTEIN1 PROTEIN2 Cloning and characterization of two evolutionarily conserved subunits PROTEIN0 and PROTEIN1 of human PROTEIN2 and their involvement in functional interactions with TFIIIB and RNA polymerase III\n",
      "\n",
      "PROTEIN0 PROTEIN1 Identification of residues in the PROTEIN0 that contact the MCP-1 receptor, PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The receptor binding site of PROTEIN0 also is significantly different from the binding sites of PROTEIN1 and PROTEIN2 providing insight into the issue of receptor specificity\n",
      "\n",
      "PROTEIN0 PROTEIN2 The receptor binding site of PROTEIN0 also is significantly different from the binding sites of PROTEIN1 and PROTEIN2 providing insight into the issue of receptor specificity\n",
      "\n",
      "PROTEIN1 PROTEIN2 The receptor binding site of PROTEIN0 also is significantly different from the binding sites of PROTEIN1 and PROTEIN2 providing insight into the issue of receptor specificity\n",
      "\n",
      "PROTEIN0 PROTEIN1 It was previously shown that the N-terminus of PROTEIN0 is critical for binding PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 To identify the regions of PROTEIN0 that contact its receptor, PROTEIN1 we substituted all surface-exposed residues with alanine\n",
      "\n",
      "PROTEIN0 PROTEIN1 We tested this point by cotransfecting CHO cells with the genes encoding F beta alpha and the PROTEIN0 subunit or the PROTEIN1 and PROTEIN2 monomer\n",
      "\n",
      "PROTEIN0 PROTEIN2 We tested this point by cotransfecting CHO cells with the genes encoding F beta alpha and the PROTEIN0 subunit or the PROTEIN1 and PROTEIN2 monomer\n",
      "\n",
      "PROTEIN1 PROTEIN2 We tested this point by cotransfecting CHO cells with the genes encoding F beta alpha and the PROTEIN0 subunit or the PROTEIN1 and PROTEIN2 monomer\n",
      "\n",
      "PROTEIN0 PROTEIN1 The formation of a functional single chain/subunit complex was not restricted to the FSH single PROTEIN0 subunit since CG single chain interacts with the monomeric PROTEIN1 subunit and exhibits FSH activity\n",
      "\n",
      "PROTEIN0 PROTEIN1 This is relevant for the case of LH and FSH, because both are synthesized in the same cell (i.e., pituitary gonadotrophs) and several of the alpha subunit sequences required for association with either the PROTEIN0 or PROTEIN1 subunits are different\n",
      "\n",
      "PROTEIN0 PROTEIN1 Previously, we showed that the PROTEIN0 or PROTEIN1 subunit genes can be genetically fused to the alpha gene to produce biologically active single chains, PROTEIN2 and F beta alpha, respectively\n",
      "\n",
      "PROTEIN0 PROTEIN2 Previously, we showed that the PROTEIN0 or PROTEIN1 subunit genes can be genetically fused to the alpha gene to produce biologically active single chains, PROTEIN2 and F beta alpha, respectively\n",
      "\n",
      "PROTEIN1 PROTEIN2 Previously, we showed that the PROTEIN0 or PROTEIN1 subunit genes can be genetically fused to the alpha gene to produce biologically active single chains, PROTEIN2 and F beta alpha, respectively\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 activates PROTEIN1 which inhibits apoptosis in human prostate cancer cell lines\n",
      "\n",
      "PROTEIN0 PROTEIN1 RESULTS: Tyrosine phosphorylation of p85 is upregulated by PROTEIN0 in both LNCaP and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 promotes coprecipitation of p85 with PROTEIN1 the signal-transducing component of the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 promotes coprecipitation of p85 with PROTEIN1 the signal-transducing component of the PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 promotes coprecipitation of p85 with PROTEIN1 the signal-transducing component of the PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 In solution, FGFR4ed formed complexes with PROTEIN0 and PROTEIN1 both in the presence and absence of heparin\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immobilized PROTEIN0 also bound PROTEIN1 besides PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Immobilized PROTEIN0 also bound PROTEIN1 besides PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Immobilized PROTEIN0 also bound PROTEIN1 besides PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Immobilized PROTEIN0 also bound PROTEIN1 besides PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Immobilized PROTEIN0 also bound PROTEIN1 besides PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Immobilized PROTEIN0 also bound PROTEIN1 besides PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 regulates beta-2 integrin-mediated adhesion through both ARF-GEF function and interaction with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we provide evidence of how PROTEIN0 an integrin-binding protein and guanine-nucleotide exchange factor (GEF) for PROTEIN1 GTPases, regulates cell adhesion\n",
      "\n",
      "PROTEIN0 PROTEIN1 Mutational analyses of the beta-2 cytoplasmic domain revealed that the adhesive function of PROTEIN0 depends on its interaction with PROTEIN1 unless the integrin is activated by exogenous divalent cations\n",
      "\n",
      "PROTEIN0 PROTEIN1 Secondly, PROTEIN0 induces expression of an extracellular activation epitope of PROTEIN1 and the exchange factor function is not essential for this activity\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, LFA-1-mediated cell adhesion and spreading on intercellular cell adhesion molecule 1 is strongly inhibited by a PROTEIN0 mutant, which fails to catalyze PROTEIN1 GDP-GTP exchange in vitro\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, PROTEIN0 is involved in the activation of PROTEIN1 most probably through direct interaction with the integrin, and induces cell spreading by its ARF-GEF activity\n",
      "\n",
      "PROTEIN0 PROTEIN1 Suppression of rat thromboxane synthase gene transcription by PROTEIN0 in macrophages via an interaction with PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 By electrophoretic mobility shift assay using the PROTEIN0 PROTEIN1 site and nuclear extracts from macrophages, we observed a specific protein-DNA complex formation, which was inhibited by a specific antibody against the transcription factor PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 By electrophoretic mobility shift assay using the PROTEIN0 PROTEIN1 site and nuclear extracts from macrophages, we observed a specific protein-DNA complex formation, which was inhibited by a specific antibody against the transcription factor PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 By electrophoretic mobility shift assay using the PROTEIN0 PROTEIN1 site and nuclear extracts from macrophages, we observed a specific protein-DNA complex formation, which was inhibited by a specific antibody against the transcription factor PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 By electrophoretic mobility shift assay using the PROTEIN0 PROTEIN1 site and nuclear extracts from macrophages, we observed a specific protein-DNA complex formation, which was inhibited by a specific antibody against the transcription factor PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 By electrophoretic mobility shift assay using the PROTEIN0 PROTEIN1 site and nuclear extracts from macrophages, we observed a specific protein-DNA complex formation, which was inhibited by a specific antibody against the transcription factor PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 By electrophoretic mobility shift assay using the PROTEIN0 PROTEIN1 site and nuclear extracts from macrophages, we observed a specific protein-DNA complex formation, which was inhibited by a specific antibody against the transcription factor PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Finally, a direct interaction between PPARgamma and PROTEIN0 was confirmed by PROTEIN1 pull-down assay\n",
      "\n",
      "PROTEIN0 PROTEIN1 By constructing deletion mutants, we identified that amino acid residues 229-309 of Akt were involved in the binding to PROTEIN0 and amino acid residues 327-340 of PROTEIN1 were involved in the binding to Akt\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, we cloned and characterized a mouse orthologue of human PROTEIN0 protein as an atypical PROTEIN1 (named PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here, we cloned and characterized a mouse orthologue of human PROTEIN0 protein as an atypical PROTEIN1 (named PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here, we cloned and characterized a mouse orthologue of human PROTEIN0 protein as an atypical PROTEIN1 (named PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Subcellular fractionation and antibody-uptake experiments indicate that PROTEIN0 is indeed a type-I membrane protein, but, unlike other PROTEIN1 isoforms, lacks an N-terminal extracellular domain\n",
      "\n",
      "PROTEIN0 PROTEIN1 Direct interaction between the cytoplasmic tail of PROTEIN0 and the Src homology 3 domain of PROTEIN1 activates PROTEIN2 in C2C12 cells\n",
      "\n",
      "PROTEIN0 PROTEIN2 Direct interaction between the cytoplasmic tail of PROTEIN0 and the Src homology 3 domain of PROTEIN1 activates PROTEIN2 in C2C12 cells\n",
      "\n",
      "PROTEIN1 PROTEIN2 Direct interaction between the cytoplasmic tail of PROTEIN0 and the Src homology 3 domain of PROTEIN1 activates PROTEIN2 in C2C12 cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 This suggests that transmembrane PROTEIN0 by providing docking sites for the Src homology 3 domain of PROTEIN1 activates PI 3-kinase by mediating its recruitment to the membrane\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binds to the C-terminal SH2 domain of PROTEIN1 and inhibits its enzymatic activity\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this report, we demonstrate the stable association of PROTEIN0 with PROTEIN1 in a clathrin-coated vesicle complex, which not only binds to the carboxyl-terminal SH2 domain of PROTEIN2 but also inhibits its enzymatic activity in a dose-dependent manner\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this report, we demonstrate the stable association of PROTEIN0 with PROTEIN1 in a clathrin-coated vesicle complex, which not only binds to the carboxyl-terminal SH2 domain of PROTEIN2 but also inhibits its enzymatic activity in a dose-dependent manner\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this report, we demonstrate the stable association of PROTEIN0 with PROTEIN1 in a clathrin-coated vesicle complex, which not only binds to the carboxyl-terminal SH2 domain of PROTEIN2 but also inhibits its enzymatic activity in a dose-dependent manner\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we demonstrate colocalization and direct interaction between PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Overexpression of PROTEIN0 results in a zinc finger-dependent localization of a PROTEIN1 isoform and dynactin to MT bundles, raising the possibility that PROTEIN2 and PROTEIN3 regulate dynein/dynactin binding to MTs\n",
      "\n",
      "PROTEIN0 PROTEIN2 Overexpression of PROTEIN0 results in a zinc finger-dependent localization of a PROTEIN1 isoform and dynactin to MT bundles, raising the possibility that PROTEIN2 and PROTEIN3 regulate dynein/dynactin binding to MTs\n",
      "\n",
      "PROTEIN0 PROTEIN3 Overexpression of PROTEIN0 results in a zinc finger-dependent localization of a PROTEIN1 isoform and dynactin to MT bundles, raising the possibility that PROTEIN2 and PROTEIN3 regulate dynein/dynactin binding to MTs\n",
      "\n",
      "PROTEIN1 PROTEIN2 Overexpression of PROTEIN0 results in a zinc finger-dependent localization of a PROTEIN1 isoform and dynactin to MT bundles, raising the possibility that PROTEIN2 and PROTEIN3 regulate dynein/dynactin binding to MTs\n",
      "\n",
      "PROTEIN1 PROTEIN3 Overexpression of PROTEIN0 results in a zinc finger-dependent localization of a PROTEIN1 isoform and dynactin to MT bundles, raising the possibility that PROTEIN2 and PROTEIN3 regulate dynein/dynactin binding to MTs\n",
      "\n",
      "PROTEIN2 PROTEIN3 Overexpression of PROTEIN0 results in a zinc finger-dependent localization of a PROTEIN1 isoform and dynactin to MT bundles, raising the possibility that PROTEIN2 and PROTEIN3 regulate dynein/dynactin binding to MTs\n",
      "\n",
      "PROTEIN0 PROTEIN1 This work suggests that PROTEIN0 is a regulated adapter between PROTEIN1 and cytoplasmic dynein at sites involved in cargo-MT loading, and/or in the control of MT dynamics\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 multifunctional endocytic receptors\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 are two structurally different endocytic receptors that interact to serve such functions\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 are PROTEIN2 kinase inhibitors with a role in S and G2/M cell cycle checkpoints induced by genotoxic stress\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 and PROTEIN1 are PROTEIN2 kinase inhibitors with a role in S and G2/M cell cycle checkpoints induced by genotoxic stress\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 and PROTEIN1 are PROTEIN2 kinase inhibitors with a role in S and G2/M cell cycle checkpoints induced by genotoxic stress\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4 PROTEIN5 constitute a family of evolutionarily conserved, small, acidic, nuclear proteins, which have been implicated in terminal differentiation, growth suppression, and apoptosis\n",
      "\n",
      "PROTEIN0 PROTEIN1 Recent evidence has implicated PROTEIN0 in inhibition of PROTEIN1 kinase and in G2/M cell cycle arrest\n",
      "\n",
      "PROTEIN0 PROTEIN1 Yet, whether PROTEIN0 and/or PROTEIN1 function as inhibitors of PROTEIN2 kinase and/or play a role in G2/M cell cycle arrest has not been fully established\n",
      "\n",
      "PROTEIN0 PROTEIN2 Yet, whether PROTEIN0 and/or PROTEIN1 function as inhibitors of PROTEIN2 kinase and/or play a role in G2/M cell cycle arrest has not been fully established\n",
      "\n",
      "PROTEIN1 PROTEIN2 Yet, whether PROTEIN0 and/or PROTEIN1 function as inhibitors of PROTEIN2 kinase and/or play a role in G2/M cell cycle arrest has not been fully established\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this work, we show that PROTEIN0 and PROTEIN1 specifically interact with the PROTEIN2 complex, but not with other Cdk/Cyclin complexes, in vitro and in vivo\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this work, we show that PROTEIN0 and PROTEIN1 specifically interact with the PROTEIN2 complex, but not with other Cdk/Cyclin complexes, in vitro and in vivo\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this work, we show that PROTEIN0 and PROTEIN1 specifically interact with the PROTEIN2 complex, but not with other Cdk/Cyclin complexes, in vitro and in vivo\n",
      "\n",
      "PROTEIN0 PROTEIN1 Data also has been obtained that PROTEIN0 and PROTEIN1 as well as PROTEIN2 interact with both PROTEIN3 and cyclinB1, resulting in inhibition of the kinase activity of the PROTEIN4 complex\n",
      "\n",
      "PROTEIN0 PROTEIN2 Data also has been obtained that PROTEIN0 and PROTEIN1 as well as PROTEIN2 interact with both PROTEIN3 and cyclinB1, resulting in inhibition of the kinase activity of the PROTEIN4 complex\n",
      "\n",
      "PROTEIN0 PROTEIN3 Data also has been obtained that PROTEIN0 and PROTEIN1 as well as PROTEIN2 interact with both PROTEIN3 and cyclinB1, resulting in inhibition of the kinase activity of the PROTEIN4 complex\n",
      "\n",
      "PROTEIN0 PROTEIN4 Data also has been obtained that PROTEIN0 and PROTEIN1 as well as PROTEIN2 interact with both PROTEIN3 and cyclinB1, resulting in inhibition of the kinase activity of the PROTEIN4 complex\n",
      "\n",
      "PROTEIN1 PROTEIN2 Data also has been obtained that PROTEIN0 and PROTEIN1 as well as PROTEIN2 interact with both PROTEIN3 and cyclinB1, resulting in inhibition of the kinase activity of the PROTEIN4 complex\n",
      "\n",
      "PROTEIN1 PROTEIN3 Data also has been obtained that PROTEIN0 and PROTEIN1 as well as PROTEIN2 interact with both PROTEIN3 and cyclinB1, resulting in inhibition of the kinase activity of the PROTEIN4 complex\n",
      "\n",
      "PROTEIN1 PROTEIN4 Data also has been obtained that PROTEIN0 and PROTEIN1 as well as PROTEIN2 interact with both PROTEIN3 and cyclinB1, resulting in inhibition of the kinase activity of the PROTEIN4 complex\n",
      "\n",
      "PROTEIN2 PROTEIN3 Data also has been obtained that PROTEIN0 and PROTEIN1 as well as PROTEIN2 interact with both PROTEIN3 and cyclinB1, resulting in inhibition of the kinase activity of the PROTEIN4 complex\n",
      "\n",
      "PROTEIN2 PROTEIN4 Data also has been obtained that PROTEIN0 and PROTEIN1 as well as PROTEIN2 interact with both PROTEIN3 and cyclinB1, resulting in inhibition of the kinase activity of the PROTEIN4 complex\n",
      "\n",
      "PROTEIN3 PROTEIN4 Data also has been obtained that PROTEIN0 and PROTEIN1 as well as PROTEIN2 interact with both PROTEIN3 and cyclinB1, resulting in inhibition of the kinase activity of the PROTEIN4 complex\n",
      "\n",
      "PROTEIN0 PROTEIN1 Inhibition of PROTEIN0 kinase activity by PROTEIN1 and PROTEIN2 was found to involve disruption of the complex, whereas PROTEIN3 did not disrupt the complex\n",
      "\n",
      "PROTEIN0 PROTEIN2 Inhibition of PROTEIN0 kinase activity by PROTEIN1 and PROTEIN2 was found to involve disruption of the complex, whereas PROTEIN3 did not disrupt the complex\n",
      "\n",
      "PROTEIN0 PROTEIN3 Inhibition of PROTEIN0 kinase activity by PROTEIN1 and PROTEIN2 was found to involve disruption of the complex, whereas PROTEIN3 did not disrupt the complex\n",
      "\n",
      "PROTEIN1 PROTEIN2 Inhibition of PROTEIN0 kinase activity by PROTEIN1 and PROTEIN2 was found to involve disruption of the complex, whereas PROTEIN3 did not disrupt the complex\n",
      "\n",
      "PROTEIN1 PROTEIN3 Inhibition of PROTEIN0 kinase activity by PROTEIN1 and PROTEIN2 was found to involve disruption of the complex, whereas PROTEIN3 did not disrupt the complex\n",
      "\n",
      "PROTEIN2 PROTEIN3 Inhibition of PROTEIN0 kinase activity by PROTEIN1 and PROTEIN2 was found to involve disruption of the complex, whereas PROTEIN3 did not disrupt the complex\n",
      "\n",
      "PROTEIN0 PROTEIN1 The transmembrane PROTEIN0 PROTEIN1 interacts with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The transmembrane PROTEIN0 PROTEIN1 interacts with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The transmembrane PROTEIN0 PROTEIN1 interacts with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 PROTEIN1 also known as PROTEIN2 is expressed predominantly in epithelial cells, in a variety of tumor cell lines, and in lymphocytes\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 PROTEIN1 also known as PROTEIN2 is expressed predominantly in epithelial cells, in a variety of tumor cell lines, and in lymphocytes\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 PROTEIN1 also known as PROTEIN2 is expressed predominantly in epithelial cells, in a variety of tumor cell lines, and in lymphocytes\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 also interacted with other members of the catenin gene family including PROTEIN1 and gamma-catenin\n",
      "\n",
      "PROTEIN0 PROTEIN1 The interaction with PROTEIN0 is likely to be direct, as the interaction occurs in K562 cells lacking functional adherens junctions and PROTEIN1 expression\n",
      "\n",
      "PROTEIN0 PROTEIN1 Catalytic domains of the tyrosine phosphatases PROTEIN0 PROTEIN1 and PTPbeta did not interact with proteins of the catenin family to detectable levels, suggesting that the interaction of PROTEIN2 with these proteins is specific\n",
      "\n",
      "PROTEIN0 PROTEIN2 Catalytic domains of the tyrosine phosphatases PROTEIN0 PROTEIN1 and PTPbeta did not interact with proteins of the catenin family to detectable levels, suggesting that the interaction of PROTEIN2 with these proteins is specific\n",
      "\n",
      "PROTEIN1 PROTEIN2 Catalytic domains of the tyrosine phosphatases PROTEIN0 PROTEIN1 and PTPbeta did not interact with proteins of the catenin family to detectable levels, suggesting that the interaction of PROTEIN2 with these proteins is specific\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 PROTEIN2 protein (CBP) modulates co-operation between myocyte enhancer factor 2A PROTEIN3 and thyroid hormone receptor-retinoid X receptor\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 protein (CBP) modulates co-operation between myocyte enhancer factor 2A PROTEIN3 and thyroid hormone receptor-retinoid X receptor\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 protein (CBP) modulates co-operation between myocyte enhancer factor 2A PROTEIN3 and thyroid hormone receptor-retinoid X receptor\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 protein (CBP) modulates co-operation between myocyte enhancer factor 2A PROTEIN3 and thyroid hormone receptor-retinoid X receptor\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 protein (CBP) modulates co-operation between myocyte enhancer factor 2A PROTEIN3 and thyroid hormone receptor-retinoid X receptor\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 PROTEIN2 protein (CBP) modulates co-operation between myocyte enhancer factor 2A PROTEIN3 and thyroid hormone receptor-retinoid X receptor\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 PROTEIN2 protein (CBP) interacting with transcription factors is able to increase their activity on target gene promoters\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 protein (CBP) interacting with transcription factors is able to increase their activity on target gene promoters\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 PROTEIN2 protein (CBP) interacting with transcription factors is able to increase their activity on target gene promoters\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our findings showed that the same C-terminal portion of PROTEIN0 binds the N-terminal domains of both TR and PROTEIN1 and our in vivo studies demonstrated that TR, PROTEIN2 and PROTEIN3 form a ternary complex\n",
      "\n",
      "PROTEIN0 PROTEIN2 Our findings showed that the same C-terminal portion of PROTEIN0 binds the N-terminal domains of both TR and PROTEIN1 and our in vivo studies demonstrated that TR, PROTEIN2 and PROTEIN3 form a ternary complex\n",
      "\n",
      "PROTEIN0 PROTEIN3 Our findings showed that the same C-terminal portion of PROTEIN0 binds the N-terminal domains of both TR and PROTEIN1 and our in vivo studies demonstrated that TR, PROTEIN2 and PROTEIN3 form a ternary complex\n",
      "\n",
      "PROTEIN1 PROTEIN2 Our findings showed that the same C-terminal portion of PROTEIN0 binds the N-terminal domains of both TR and PROTEIN1 and our in vivo studies demonstrated that TR, PROTEIN2 and PROTEIN3 form a ternary complex\n",
      "\n",
      "PROTEIN1 PROTEIN3 Our findings showed that the same C-terminal portion of PROTEIN0 binds the N-terminal domains of both TR and PROTEIN1 and our in vivo studies demonstrated that TR, PROTEIN2 and PROTEIN3 form a ternary complex\n",
      "\n",
      "PROTEIN2 PROTEIN3 Our findings showed that the same C-terminal portion of PROTEIN0 binds the N-terminal domains of both TR and PROTEIN1 and our in vivo studies demonstrated that TR, PROTEIN2 and PROTEIN3 form a ternary complex\n",
      "\n",
      "PROTEIN0 PROTEIN1 The human PROTEIN0 and PROTEIN1 proteins form a ternary complex with hMpp10, which only interacts with the U3 PROTEIN2 in 60-80S ribonucleoprotein complexes\n",
      "\n",
      "PROTEIN0 PROTEIN2 The human PROTEIN0 and PROTEIN1 proteins form a ternary complex with hMpp10, which only interacts with the U3 PROTEIN2 in 60-80S ribonucleoprotein complexes\n",
      "\n",
      "PROTEIN1 PROTEIN2 The human PROTEIN0 and PROTEIN1 proteins form a ternary complex with hMpp10, which only interacts with the U3 PROTEIN2 in 60-80S ribonucleoprotein complexes\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have cloned cDNAs encoding the human and mouse homologs of the yeast U3 snoRNP-associated proteins PROTEIN0 and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The results of complementation experiments show that, in contrast to mouse PROTEIN0 mouse PROTEIN1 can partially alleviate the growth defect of the corresponding yeast null strain, indicating that the role of PROTEIN2 in pre-rRNA processing is evolutionarily conserved\n",
      "\n",
      "PROTEIN0 PROTEIN2 The results of complementation experiments show that, in contrast to mouse PROTEIN0 mouse PROTEIN1 can partially alleviate the growth defect of the corresponding yeast null strain, indicating that the role of PROTEIN2 in pre-rRNA processing is evolutionarily conserved\n",
      "\n",
      "PROTEIN1 PROTEIN2 The results of complementation experiments show that, in contrast to mouse PROTEIN0 mouse PROTEIN1 can partially alleviate the growth defect of the corresponding yeast null strain, indicating that the role of PROTEIN2 in pre-rRNA processing is evolutionarily conserved\n",
      "\n",
      "PROTEIN0 PROTEIN1 The results of density gradient centrifugation experiments show that, in contrast to hU3-55K, the human PROTEIN0 and PROTEIN1 proteins predominantly interact with the U3 PROTEIN2 in 60-80S ribonucleoprotein complexes\n",
      "\n",
      "PROTEIN0 PROTEIN2 The results of density gradient centrifugation experiments show that, in contrast to hU3-55K, the human PROTEIN0 and PROTEIN1 proteins predominantly interact with the U3 PROTEIN2 in 60-80S ribonucleoprotein complexes\n",
      "\n",
      "PROTEIN1 PROTEIN2 The results of density gradient centrifugation experiments show that, in contrast to hU3-55K, the human PROTEIN0 and PROTEIN1 proteins predominantly interact with the U3 PROTEIN2 in 60-80S ribonucleoprotein complexes\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the present study, we observed evidence of cross-talk between the cannabinoid receptor PROTEIN0 and the PROTEIN1 PROTEIN2 using a heterologous system\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the present study, we observed evidence of cross-talk between the cannabinoid receptor PROTEIN0 and the PROTEIN1 PROTEIN2 using a heterologous system\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the present study, we observed evidence of cross-talk between the cannabinoid receptor PROTEIN0 and the PROTEIN1 PROTEIN2 using a heterologous system\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast to PROTEIN0 the potency of direct activation of PROTEIN1 was not affected by co-expression with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In contrast to PROTEIN0 the potency of direct activation of PROTEIN1 was not affected by co-expression with PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In contrast to PROTEIN0 the potency of direct activation of PROTEIN1 was not affected by co-expression with PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, electron microscopy experiments revealed that PROTEIN0 and PROTEIN1 are closely apposed at the plasma membrane level; they are close enough to form hetero-oligomers\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 a Rho family PROTEIN1 that interacts with signaling adapters PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 a Rho family PROTEIN1 that interacts with signaling adapters PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 a Rho family PROTEIN1 that interacts with signaling adapters PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 a Rho family PROTEIN1 that interacts with signaling adapters PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 a Rho family PROTEIN1 that interacts with signaling adapters PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 a Rho family PROTEIN1 that interacts with signaling adapters PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 are scaffolding proteins acting downstream of cell surface receptors and interact with a variety of cytoplasmic signaling proteins such as PROTEIN2 PROTEIN3 PROTEIN4 Shc, and Crk\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 and PROTEIN1 are scaffolding proteins acting downstream of cell surface receptors and interact with a variety of cytoplasmic signaling proteins such as PROTEIN2 PROTEIN3 PROTEIN4 Shc, and Crk\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 and PROTEIN1 are scaffolding proteins acting downstream of cell surface receptors and interact with a variety of cytoplasmic signaling proteins such as PROTEIN2 PROTEIN3 PROTEIN4 Shc, and Crk\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 and PROTEIN1 are scaffolding proteins acting downstream of cell surface receptors and interact with a variety of cytoplasmic signaling proteins such as PROTEIN2 PROTEIN3 PROTEIN4 Shc, and Crk\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 and PROTEIN1 are scaffolding proteins acting downstream of cell surface receptors and interact with a variety of cytoplasmic signaling proteins such as PROTEIN2 PROTEIN3 PROTEIN4 Shc, and Crk\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 and PROTEIN1 are scaffolding proteins acting downstream of cell surface receptors and interact with a variety of cytoplasmic signaling proteins such as PROTEIN2 PROTEIN3 PROTEIN4 Shc, and Crk\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 and PROTEIN1 are scaffolding proteins acting downstream of cell surface receptors and interact with a variety of cytoplasmic signaling proteins such as PROTEIN2 PROTEIN3 PROTEIN4 Shc, and Crk\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 and PROTEIN1 are scaffolding proteins acting downstream of cell surface receptors and interact with a variety of cytoplasmic signaling proteins such as PROTEIN2 PROTEIN3 PROTEIN4 Shc, and Crk\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 and PROTEIN1 are scaffolding proteins acting downstream of cell surface receptors and interact with a variety of cytoplasmic signaling proteins such as PROTEIN2 PROTEIN3 PROTEIN4 Shc, and Crk\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 and PROTEIN1 are scaffolding proteins acting downstream of cell surface receptors and interact with a variety of cytoplasmic signaling proteins such as PROTEIN2 PROTEIN3 PROTEIN4 Shc, and Crk\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, PROTEIN0 contains several classic proline-rich motifs, and it interacts with the first SH3 domain of Crk and full-length PROTEIN1 in vitro\n",
      "\n",
      "PROTEIN0 PROTEIN1 We propose that PROTEIN0 and PROTEIN1 in cooperation with other adapter molecules might regulate the cellular localization of PROTEIN2 under specific stimuli, acting to regulate precisely Rac and PROTEIN3 activities\n",
      "\n",
      "PROTEIN0 PROTEIN2 We propose that PROTEIN0 and PROTEIN1 in cooperation with other adapter molecules might regulate the cellular localization of PROTEIN2 under specific stimuli, acting to regulate precisely Rac and PROTEIN3 activities\n",
      "\n",
      "PROTEIN0 PROTEIN3 We propose that PROTEIN0 and PROTEIN1 in cooperation with other adapter molecules might regulate the cellular localization of PROTEIN2 under specific stimuli, acting to regulate precisely Rac and PROTEIN3 activities\n",
      "\n",
      "PROTEIN1 PROTEIN2 We propose that PROTEIN0 and PROTEIN1 in cooperation with other adapter molecules might regulate the cellular localization of PROTEIN2 under specific stimuli, acting to regulate precisely Rac and PROTEIN3 activities\n",
      "\n",
      "PROTEIN1 PROTEIN3 We propose that PROTEIN0 and PROTEIN1 in cooperation with other adapter molecules might regulate the cellular localization of PROTEIN2 under specific stimuli, acting to regulate precisely Rac and PROTEIN3 activities\n",
      "\n",
      "PROTEIN2 PROTEIN3 We propose that PROTEIN0 and PROTEIN1 in cooperation with other adapter molecules might regulate the cellular localization of PROTEIN2 under specific stimuli, acting to regulate precisely Rac and PROTEIN3 activities\n",
      "\n",
      "PROTEIN0 PROTEIN1 This work led to identification of a novel PROTEIN0 PROTEIN1 for Rho family GTPases\n",
      "\n",
      "PROTEIN0 PROTEIN1 The GAP domain shows high similarity to the recently cloned PROTEIN0 and displays activity toward RhoA, PROTEIN1 and PROTEIN2 in vitro\n",
      "\n",
      "PROTEIN0 PROTEIN2 The GAP domain shows high similarity to the recently cloned PROTEIN0 and displays activity toward RhoA, PROTEIN1 and PROTEIN2 in vitro\n",
      "\n",
      "PROTEIN1 PROTEIN2 The GAP domain shows high similarity to the recently cloned PROTEIN0 and displays activity toward RhoA, PROTEIN1 and PROTEIN2 in vitro\n",
      "\n",
      "PROTEIN0 PROTEIN1 The protein was named PROTEIN0 for its ability to interact with PROTEIN1 proteins and its activity toward Rac and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The protein was named PROTEIN0 for its ability to interact with PROTEIN1 proteins and its activity toward Rac and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The protein was named PROTEIN0 for its ability to interact with PROTEIN1 proteins and its activity toward Rac and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Expression of PROTEIN0 in 293T cells led to a reduction in active PROTEIN1 and PROTEIN2 levels but not RhoA\n",
      "\n",
      "PROTEIN0 PROTEIN2 Expression of PROTEIN0 in 293T cells led to a reduction in active PROTEIN1 and PROTEIN2 levels but not RhoA\n",
      "\n",
      "PROTEIN1 PROTEIN2 Expression of PROTEIN0 in 293T cells led to a reduction in active PROTEIN1 and PROTEIN2 levels but not RhoA\n",
      "\n",
      "PROTEIN0 PROTEIN1 A role for PROTEIN0 phosphorylation of PROTEIN1 in cytokinesis\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN0 PROTEIN4 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN0 PROTEIN5 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN1 PROTEIN4 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN1 PROTEIN5 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN2 PROTEIN4 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN2 PROTEIN5 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN3 PROTEIN4 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN3 PROTEIN5 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN4 PROTEIN5 Here, we characterize the interaction between PROTEIN0 and PROTEIN1 show that PROTEIN2 phosphorylates PROTEIN3 at conserved S274 and S326 residues in vitro, and present evidence that PROTEIN4 is also a substrate for PROTEIN5 in vivo\n",
      "\n",
      "PROTEIN0 PROTEIN1 Downregulation of PROTEIN0 by RNA interference results in multiple mitotic defects, including multinucleation and cells arrested at the midbody stage, which are rescued by ectopic expression of wild-type PROTEIN1 but not by PROTEIN2 with mutations in the PROTEIN3 phosphorylation sites\n",
      "\n",
      "PROTEIN0 PROTEIN2 Downregulation of PROTEIN0 by RNA interference results in multiple mitotic defects, including multinucleation and cells arrested at the midbody stage, which are rescued by ectopic expression of wild-type PROTEIN1 but not by PROTEIN2 with mutations in the PROTEIN3 phosphorylation sites\n",
      "\n",
      "PROTEIN0 PROTEIN3 Downregulation of PROTEIN0 by RNA interference results in multiple mitotic defects, including multinucleation and cells arrested at the midbody stage, which are rescued by ectopic expression of wild-type PROTEIN1 but not by PROTEIN2 with mutations in the PROTEIN3 phosphorylation sites\n",
      "\n",
      "PROTEIN1 PROTEIN2 Downregulation of PROTEIN0 by RNA interference results in multiple mitotic defects, including multinucleation and cells arrested at the midbody stage, which are rescued by ectopic expression of wild-type PROTEIN1 but not by PROTEIN2 with mutations in the PROTEIN3 phosphorylation sites\n",
      "\n",
      "PROTEIN1 PROTEIN3 Downregulation of PROTEIN0 by RNA interference results in multiple mitotic defects, including multinucleation and cells arrested at the midbody stage, which are rescued by ectopic expression of wild-type PROTEIN1 but not by PROTEIN2 with mutations in the PROTEIN3 phosphorylation sites\n",
      "\n",
      "PROTEIN2 PROTEIN3 Downregulation of PROTEIN0 by RNA interference results in multiple mitotic defects, including multinucleation and cells arrested at the midbody stage, which are rescued by ectopic expression of wild-type PROTEIN1 but not by PROTEIN2 with mutations in the PROTEIN3 phosphorylation sites\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 phosphorylation of PROTEIN1 may influence cytokinesis\n",
      "\n",
      "PROTEIN0 PROTEIN1 Specific interaction of PROTEIN0 and c/EBPbeta leads to the transcriptional activation of the human PROTEIN1 gene\n",
      "\n",
      "PROTEIN0 PROTEIN1 The sterol-independent regulatory element (SIRE) of the PROTEIN0 PROTEIN1 promoter mediates PROTEIN2 (OM)-induced transcription of the PROTEIN3 gene through a cholesterol-independent pathway\n",
      "\n",
      "PROTEIN0 PROTEIN2 The sterol-independent regulatory element (SIRE) of the PROTEIN0 PROTEIN1 promoter mediates PROTEIN2 (OM)-induced transcription of the PROTEIN3 gene through a cholesterol-independent pathway\n",
      "\n",
      "PROTEIN0 PROTEIN3 The sterol-independent regulatory element (SIRE) of the PROTEIN0 PROTEIN1 promoter mediates PROTEIN2 (OM)-induced transcription of the PROTEIN3 gene through a cholesterol-independent pathway\n",
      "\n",
      "PROTEIN1 PROTEIN2 The sterol-independent regulatory element (SIRE) of the PROTEIN0 PROTEIN1 promoter mediates PROTEIN2 (OM)-induced transcription of the PROTEIN3 gene through a cholesterol-independent pathway\n",
      "\n",
      "PROTEIN1 PROTEIN3 The sterol-independent regulatory element (SIRE) of the PROTEIN0 PROTEIN1 promoter mediates PROTEIN2 (OM)-induced transcription of the PROTEIN3 gene through a cholesterol-independent pathway\n",
      "\n",
      "PROTEIN2 PROTEIN3 The sterol-independent regulatory element (SIRE) of the PROTEIN0 PROTEIN1 promoter mediates PROTEIN2 (OM)-induced transcription of the PROTEIN3 gene through a cholesterol-independent pathway\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 orphan receptor functions as negative modulator for PROTEIN1 in prostate cancer cells PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 orphan receptor functions as negative modulator for PROTEIN1 in prostate cancer cells PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 orphan receptor functions as negative modulator for PROTEIN1 in prostate cancer cells PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 BACKGROUND: Both PROTEIN0 (AR) and orphan receptor PROTEIN1 PROTEIN2 belong to the steroid nuclear receptor superfamily and are expressed in prostate cancer tissue and cell lines\n",
      "\n",
      "PROTEIN0 PROTEIN2 BACKGROUND: Both PROTEIN0 (AR) and orphan receptor PROTEIN1 PROTEIN2 belong to the steroid nuclear receptor superfamily and are expressed in prostate cancer tissue and cell lines\n",
      "\n",
      "PROTEIN1 PROTEIN2 BACKGROUND: Both PROTEIN0 (AR) and orphan receptor PROTEIN1 PROTEIN2 belong to the steroid nuclear receptor superfamily and are expressed in prostate cancer tissue and cell lines\n",
      "\n",
      "PROTEIN0 PROTEIN1 The interaction between AR and PROTEIN0 was assessed by PROTEIN1 PROTEIN2 pull-down assay and mammalian two-hybrid system assay\n",
      "\n",
      "PROTEIN0 PROTEIN2 The interaction between AR and PROTEIN0 was assessed by PROTEIN1 PROTEIN2 pull-down assay and mammalian two-hybrid system assay\n",
      "\n",
      "PROTEIN1 PROTEIN2 The interaction between AR and PROTEIN0 was assessed by PROTEIN1 PROTEIN2 pull-down assay and mammalian two-hybrid system assay\n",
      "\n",
      "PROTEIN0 PROTEIN1 The protein PROTEIN0 functions in a multi-subunit E3 ubiquitin ligase that targets the hypoxia-inducible transcription factor PROTEIN1 for proteasomal degradation during normoxia\n",
      "\n",
      "PROTEIN0 PROTEIN1 We establish that PROTEIN0 binds to PROTEIN1 PROTEIN2 a component of the 19S regulatory complex of the proteasome\n",
      "\n",
      "PROTEIN0 PROTEIN2 We establish that PROTEIN0 binds to PROTEIN1 PROTEIN2 a component of the 19S regulatory complex of the proteasome\n",
      "\n",
      "PROTEIN1 PROTEIN2 We establish that PROTEIN0 binds to PROTEIN1 PROTEIN2 a component of the 19S regulatory complex of the proteasome\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 associates with the beta-domain of PROTEIN1 and complexes with PROTEIN2 and PROTEIN3 in vivo\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 associates with the beta-domain of PROTEIN1 and complexes with PROTEIN2 and PROTEIN3 in vivo\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 associates with the beta-domain of PROTEIN1 and complexes with PROTEIN2 and PROTEIN3 in vivo\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 associates with the beta-domain of PROTEIN1 and complexes with PROTEIN2 and PROTEIN3 in vivo\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 associates with the beta-domain of PROTEIN1 and complexes with PROTEIN2 and PROTEIN3 in vivo\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 associates with the beta-domain of PROTEIN1 and complexes with PROTEIN2 and PROTEIN3 in vivo\n",
      "\n",
      "PROTEIN0 PROTEIN1 Overexpression of PROTEIN0 promotes degradation of PROTEIN1 in a pVHL-dependent manner that requires the ATPase domain of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Overexpression of PROTEIN0 promotes degradation of PROTEIN1 in a pVHL-dependent manner that requires the ATPase domain of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Overexpression of PROTEIN0 promotes degradation of PROTEIN1 in a pVHL-dependent manner that requires the ATPase domain of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Blockade of PROTEIN0 expression by small interfering RNA (siRNA) causes prolonged degradation kinetics of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Several distinct mutations in exon 2 of PROTEIN0 disrupt binding of PROTEIN1 to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Several distinct mutations in exon 2 of PROTEIN0 disrupt binding of PROTEIN1 to PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Several distinct mutations in exon 2 of PROTEIN0 disrupt binding of PROTEIN1 to PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 A PROTEIN0 mutant containing a P154L substitution coimmunoprecipitates with PROTEIN1 but not PROTEIN2 and does not promote degradation of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 A PROTEIN0 mutant containing a P154L substitution coimmunoprecipitates with PROTEIN1 but not PROTEIN2 and does not promote degradation of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 A PROTEIN0 mutant containing a P154L substitution coimmunoprecipitates with PROTEIN1 but not PROTEIN2 and does not promote degradation of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 A PROTEIN0 mutant containing a P154L substitution coimmunoprecipitates with PROTEIN1 but not PROTEIN2 and does not promote degradation of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 A PROTEIN0 mutant containing a P154L substitution coimmunoprecipitates with PROTEIN1 but not PROTEIN2 and does not promote degradation of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 A PROTEIN0 mutant containing a P154L substitution coimmunoprecipitates with PROTEIN1 but not PROTEIN2 and does not promote degradation of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, the ability of PROTEIN0 to degrade PROTEIN1 depends in part on its interaction with PROTEIN2 and suggests a new mechanism for PROTEIN3 stabilization in some PROTEIN4 tumors\n",
      "\n",
      "PROTEIN0 PROTEIN2 Thus, the ability of PROTEIN0 to degrade PROTEIN1 depends in part on its interaction with PROTEIN2 and suggests a new mechanism for PROTEIN3 stabilization in some PROTEIN4 tumors\n",
      "\n",
      "PROTEIN0 PROTEIN3 Thus, the ability of PROTEIN0 to degrade PROTEIN1 depends in part on its interaction with PROTEIN2 and suggests a new mechanism for PROTEIN3 stabilization in some PROTEIN4 tumors\n",
      "\n",
      "PROTEIN0 PROTEIN4 Thus, the ability of PROTEIN0 to degrade PROTEIN1 depends in part on its interaction with PROTEIN2 and suggests a new mechanism for PROTEIN3 stabilization in some PROTEIN4 tumors\n",
      "\n",
      "PROTEIN1 PROTEIN2 Thus, the ability of PROTEIN0 to degrade PROTEIN1 depends in part on its interaction with PROTEIN2 and suggests a new mechanism for PROTEIN3 stabilization in some PROTEIN4 tumors\n",
      "\n",
      "PROTEIN1 PROTEIN3 Thus, the ability of PROTEIN0 to degrade PROTEIN1 depends in part on its interaction with PROTEIN2 and suggests a new mechanism for PROTEIN3 stabilization in some PROTEIN4 tumors\n",
      "\n",
      "PROTEIN1 PROTEIN4 Thus, the ability of PROTEIN0 to degrade PROTEIN1 depends in part on its interaction with PROTEIN2 and suggests a new mechanism for PROTEIN3 stabilization in some PROTEIN4 tumors\n",
      "\n",
      "PROTEIN2 PROTEIN3 Thus, the ability of PROTEIN0 to degrade PROTEIN1 depends in part on its interaction with PROTEIN2 and suggests a new mechanism for PROTEIN3 stabilization in some PROTEIN4 tumors\n",
      "\n",
      "PROTEIN2 PROTEIN4 Thus, the ability of PROTEIN0 to degrade PROTEIN1 depends in part on its interaction with PROTEIN2 and suggests a new mechanism for PROTEIN3 stabilization in some PROTEIN4 tumors\n",
      "\n",
      "PROTEIN3 PROTEIN4 Thus, the ability of PROTEIN0 to degrade PROTEIN1 depends in part on its interaction with PROTEIN2 and suggests a new mechanism for PROTEIN3 stabilization in some PROTEIN4 tumors\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 interacts with PROTEIN2 PROTEIN3 and SRrp130 in corneal epithelial cells\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 interacts with PROTEIN2 PROTEIN3 and SRrp130 in corneal epithelial cells\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 interacts with PROTEIN2 PROTEIN3 and SRrp130 in corneal epithelial cells\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 interacts with PROTEIN2 PROTEIN3 and SRrp130 in corneal epithelial cells\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 interacts with PROTEIN2 PROTEIN3 and SRrp130 in corneal epithelial cells\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 interacts with PROTEIN2 PROTEIN3 and SRrp130 in corneal epithelial cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 Three SR-rich proteins were identified that interact with the C-terminus of Pnn: PROTEIN0 and PROTEIN1 known components of spliceosome machinery, and a novel 130-kDa nuclear protein, SRrp130\n",
      "\n",
      "PROTEIN0 PROTEIN1 Modification of de novo DNA methyltransferase 3a PROTEIN0 by PROTEIN1 modulates its interaction with histone deacetylases (HDACs) and its capacity to repress transcription\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we show that PROTEIN0 interacts with multiple components of the sumoylation machinery, namely the E2 sumo conjugating enzyme PROTEIN1 and the E3 sumo ligases PROTEIN2 and PROTEIN3 all of which are involved in conjugating the small ubiquitin-like modifier polypeptide, PROTEIN4 to its target proteins\n",
      "\n",
      "PROTEIN0 PROTEIN2 Here we show that PROTEIN0 interacts with multiple components of the sumoylation machinery, namely the E2 sumo conjugating enzyme PROTEIN1 and the E3 sumo ligases PROTEIN2 and PROTEIN3 all of which are involved in conjugating the small ubiquitin-like modifier polypeptide, PROTEIN4 to its target proteins\n",
      "\n",
      "PROTEIN0 PROTEIN3 Here we show that PROTEIN0 interacts with multiple components of the sumoylation machinery, namely the E2 sumo conjugating enzyme PROTEIN1 and the E3 sumo ligases PROTEIN2 and PROTEIN3 all of which are involved in conjugating the small ubiquitin-like modifier polypeptide, PROTEIN4 to its target proteins\n",
      "\n",
      "PROTEIN0 PROTEIN4 Here we show that PROTEIN0 interacts with multiple components of the sumoylation machinery, namely the E2 sumo conjugating enzyme PROTEIN1 and the E3 sumo ligases PROTEIN2 and PROTEIN3 all of which are involved in conjugating the small ubiquitin-like modifier polypeptide, PROTEIN4 to its target proteins\n",
      "\n",
      "PROTEIN1 PROTEIN2 Here we show that PROTEIN0 interacts with multiple components of the sumoylation machinery, namely the E2 sumo conjugating enzyme PROTEIN1 and the E3 sumo ligases PROTEIN2 and PROTEIN3 all of which are involved in conjugating the small ubiquitin-like modifier polypeptide, PROTEIN4 to its target proteins\n",
      "\n",
      "PROTEIN1 PROTEIN3 Here we show that PROTEIN0 interacts with multiple components of the sumoylation machinery, namely the E2 sumo conjugating enzyme PROTEIN1 and the E3 sumo ligases PROTEIN2 and PROTEIN3 all of which are involved in conjugating the small ubiquitin-like modifier polypeptide, PROTEIN4 to its target proteins\n",
      "\n",
      "PROTEIN1 PROTEIN4 Here we show that PROTEIN0 interacts with multiple components of the sumoylation machinery, namely the E2 sumo conjugating enzyme PROTEIN1 and the E3 sumo ligases PROTEIN2 and PROTEIN3 all of which are involved in conjugating the small ubiquitin-like modifier polypeptide, PROTEIN4 to its target proteins\n",
      "\n",
      "PROTEIN2 PROTEIN3 Here we show that PROTEIN0 interacts with multiple components of the sumoylation machinery, namely the E2 sumo conjugating enzyme PROTEIN1 and the E3 sumo ligases PROTEIN2 and PROTEIN3 all of which are involved in conjugating the small ubiquitin-like modifier polypeptide, PROTEIN4 to its target proteins\n",
      "\n",
      "PROTEIN2 PROTEIN4 Here we show that PROTEIN0 interacts with multiple components of the sumoylation machinery, namely the E2 sumo conjugating enzyme PROTEIN1 and the E3 sumo ligases PROTEIN2 and PROTEIN3 all of which are involved in conjugating the small ubiquitin-like modifier polypeptide, PROTEIN4 to its target proteins\n",
      "\n",
      "PROTEIN3 PROTEIN4 Here we show that PROTEIN0 interacts with multiple components of the sumoylation machinery, namely the E2 sumo conjugating enzyme PROTEIN1 and the E3 sumo ligases PROTEIN2 and PROTEIN3 all of which are involved in conjugating the small ubiquitin-like modifier polypeptide, PROTEIN4 to its target proteins\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is modified by PROTEIN1 in vivo and in vitro and the region of PROTEIN2 responsible for interaction maps to the N-terminal regulatory domain\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is modified by PROTEIN1 in vivo and in vitro and the region of PROTEIN2 responsible for interaction maps to the N-terminal regulatory domain\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is modified by PROTEIN1 in vivo and in vitro and the region of PROTEIN2 responsible for interaction maps to the N-terminal regulatory domain\n",
      "\n",
      "PROTEIN0 PROTEIN1 Functionally, sumoylation of PROTEIN0 disrupts its ability to interact with histone deacetylases PROTEIN1 but not with another interaction partner, PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Functionally, sumoylation of PROTEIN0 disrupts its ability to interact with histone deacetylases PROTEIN1 but not with another interaction partner, PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Functionally, sumoylation of PROTEIN0 disrupts its ability to interact with histone deacetylases PROTEIN1 but not with another interaction partner, PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The tyrosine kinase PROTEIN0 is physically associated with the Type I interferon PROTEIN1 receptor complex and is rapidly activated during PROTEIN2 stimulation\n",
      "\n",
      "PROTEIN0 PROTEIN2 The tyrosine kinase PROTEIN0 is physically associated with the Type I interferon PROTEIN1 receptor complex and is rapidly activated during PROTEIN2 stimulation\n",
      "\n",
      "PROTEIN1 PROTEIN2 The tyrosine kinase PROTEIN0 is physically associated with the Type I interferon PROTEIN1 receptor complex and is rapidly activated during PROTEIN2 stimulation\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 interaction with PROTEIN1 phosphatase ties mitogenic signal transduction to cell cycle activation\n",
      "\n",
      "PROTEIN0 PROTEIN1 We show here that PROTEIN0 phosphatase associates with PROTEIN1 in somatic mammalian cells and in meiotic frog oocytes\n",
      "\n",
      "PROTEIN0 PROTEIN1 Differential binding of vascular cell-derived proteoglycans PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 to the beta-amyloid protein of Alzheimer's disease\n",
      "\n",
      "PROTEIN0 PROTEIN2 Differential binding of vascular cell-derived proteoglycans PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 to the beta-amyloid protein of Alzheimer's disease\n",
      "\n",
      "PROTEIN0 PROTEIN3 Differential binding of vascular cell-derived proteoglycans PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 to the beta-amyloid protein of Alzheimer's disease\n",
      "\n",
      "PROTEIN1 PROTEIN2 Differential binding of vascular cell-derived proteoglycans PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 to the beta-amyloid protein of Alzheimer's disease\n",
      "\n",
      "PROTEIN1 PROTEIN3 Differential binding of vascular cell-derived proteoglycans PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 to the beta-amyloid protein of Alzheimer's disease\n",
      "\n",
      "PROTEIN2 PROTEIN3 Differential binding of vascular cell-derived proteoglycans PROTEIN0 PROTEIN1 PROTEIN2 and PROTEIN3 to the beta-amyloid protein of Alzheimer's disease\n",
      "\n",
      "PROTEIN0 PROTEIN1 Previous studies have demonstrated the immunolocalization of PROTEIN0 a specific PROTEIN1 to the PROTEIN2 protein (A beta)-containing PROTEIN3 deposits within the walls of blood vessels (i.e., congophilic angiopathy) in Alzheimer's disease (AD) brain\n",
      "\n",
      "PROTEIN0 PROTEIN2 Previous studies have demonstrated the immunolocalization of PROTEIN0 a specific PROTEIN1 to the PROTEIN2 protein (A beta)-containing PROTEIN3 deposits within the walls of blood vessels (i.e., congophilic angiopathy) in Alzheimer's disease (AD) brain\n",
      "\n",
      "PROTEIN0 PROTEIN3 Previous studies have demonstrated the immunolocalization of PROTEIN0 a specific PROTEIN1 to the PROTEIN2 protein (A beta)-containing PROTEIN3 deposits within the walls of blood vessels (i.e., congophilic angiopathy) in Alzheimer's disease (AD) brain\n",
      "\n",
      "PROTEIN1 PROTEIN2 Previous studies have demonstrated the immunolocalization of PROTEIN0 a specific PROTEIN1 to the PROTEIN2 protein (A beta)-containing PROTEIN3 deposits within the walls of blood vessels (i.e., congophilic angiopathy) in Alzheimer's disease (AD) brain\n",
      "\n",
      "PROTEIN1 PROTEIN3 Previous studies have demonstrated the immunolocalization of PROTEIN0 a specific PROTEIN1 to the PROTEIN2 protein (A beta)-containing PROTEIN3 deposits within the walls of blood vessels (i.e., congophilic angiopathy) in Alzheimer's disease (AD) brain\n",
      "\n",
      "PROTEIN2 PROTEIN3 Previous studies have demonstrated the immunolocalization of PROTEIN0 a specific PROTEIN1 to the PROTEIN2 protein (A beta)-containing PROTEIN3 deposits within the walls of blood vessels (i.e., congophilic angiopathy) in Alzheimer's disease (AD) brain\n",
      "\n",
      "PROTEIN0 PROTEIN1 Overall, the results indicate that specific vascular cell-derived PROTEIN0 differentially interact with A beta, and that the interactions of highest affinity occur between A beta and binding sites on both the core protein and glycosaminoglycan chains of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the present investigation, the differential binding of previously characterized endothelial cell (EC)- and smooth muscle cell PROTEIN0 PROTEIN1 to A beta was examined to determine whether the accumulation of A beta in cerebrovascular PROTEIN2 deposits may be due to its interactions with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the present investigation, the differential binding of previously characterized endothelial cell (EC)- and smooth muscle cell PROTEIN0 PROTEIN1 to A beta was examined to determine whether the accumulation of A beta in cerebrovascular PROTEIN2 deposits may be due to its interactions with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 In the present investigation, the differential binding of previously characterized endothelial cell (EC)- and smooth muscle cell PROTEIN0 PROTEIN1 to A beta was examined to determine whether the accumulation of A beta in cerebrovascular PROTEIN2 deposits may be due to its interactions with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the present investigation, the differential binding of previously characterized endothelial cell (EC)- and smooth muscle cell PROTEIN0 PROTEIN1 to A beta was examined to determine whether the accumulation of A beta in cerebrovascular PROTEIN2 deposits may be due to its interactions with PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 In the present investigation, the differential binding of previously characterized endothelial cell (EC)- and smooth muscle cell PROTEIN0 PROTEIN1 to A beta was examined to determine whether the accumulation of A beta in cerebrovascular PROTEIN2 deposits may be due to its interactions with PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 In the present investigation, the differential binding of previously characterized endothelial cell (EC)- and smooth muscle cell PROTEIN0 PROTEIN1 to A beta was examined to determine whether the accumulation of A beta in cerebrovascular PROTEIN2 deposits may be due to its interactions with PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Pretreatment of AA amyloidotic splenic and liver tissue sections with synthetic A beta (1-28) produced strong immunoreactivity with A beta antibodies at tissue sites enriched in PROTEIN0 which was partially removed by pretreatment with heparitinase, but not by chondroitin PROTEIN1 lyase\n",
      "\n",
      "PROTEIN0 PROTEIN1 [35S]-Sulfate labeled proteoglycans PROTEIN0 derived from cultured ECs and SMCs bound to affinity columns containing A beta (1-28) or (1-40), with virtually no binding to A beta (40-1) (reverse peptide), beta-amyloid precursor protein (410-429), or bovine PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN0 PROTEIN2 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN0 PROTEIN3 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN0 PROTEIN4 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN0 PROTEIN5 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN0 PROTEIN6 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN1 PROTEIN2 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN1 PROTEIN3 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN1 PROTEIN4 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN1 PROTEIN5 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN1 PROTEIN6 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN2 PROTEIN3 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN2 PROTEIN4 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN2 PROTEIN5 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN2 PROTEIN6 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN3 PROTEIN4 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN3 PROTEIN5 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN3 PROTEIN6 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN4 PROTEIN5 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN4 PROTEIN6 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN5 PROTEIN6 Characterization of EC and PROTEIN0 PROTEIN1 bound to A beta (1-28) revealed strong binding by PROTEIN2 weak binding by PROTEIN3 and PROTEIN4 two dermatan sulfate proteoglycans, and lack of binding by PROTEIN5 PROTEIN6 a large chondroitin sulfate proteoglycan\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, cytokines, in particular PROTEIN0 and PROTEIN1 potentiated the DNA synthesis induced by cross-linking of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, cytokines, in particular PROTEIN0 and PROTEIN1 potentiated the DNA synthesis induced by cross-linking of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, cytokines, in particular PROTEIN0 and PROTEIN1 potentiated the DNA synthesis induced by cross-linking of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The human PROTEIN0 mRNA editing protein is a PROTEIN1 showing structural homology to some known mammalian and bacteriophage deoxycytidylate deaminases\n",
      "\n",
      "PROTEIN0 PROTEIN1 The fact that the PROTEIN0 mRNA editing protein also exists as a PROTEIN1 has important implications for the mechanism of PROTEIN2 mRNA editing in humans\n",
      "\n",
      "PROTEIN0 PROTEIN2 The fact that the PROTEIN0 mRNA editing protein also exists as a PROTEIN1 has important implications for the mechanism of PROTEIN2 mRNA editing in humans\n",
      "\n",
      "PROTEIN1 PROTEIN2 The fact that the PROTEIN0 mRNA editing protein also exists as a PROTEIN1 has important implications for the mechanism of PROTEIN2 mRNA editing in humans\n",
      "\n",
      "PROTEIN0 PROTEIN1 Assignment of PROTEIN0 PROTEIN1 and Etfa to chromosomes 3, 7, and 13: the mouse homologs of genes responsible for glutaric acidemia type II in human\n",
      "\n",
      "PROTEIN0 PROTEIN1 We used cDNA probes for the PROTEIN0 PROTEIN1 and Etfa genes to determine localization of these mouse genes to chromosomes 3, 7, and 13\n",
      "\n",
      "PROTEIN0 PROTEIN1 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN0 PROTEIN2 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN0 PROTEIN3 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN0 PROTEIN4 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN0 PROTEIN5 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN1 PROTEIN2 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN1 PROTEIN3 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN1 PROTEIN4 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN1 PROTEIN5 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN2 PROTEIN3 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN2 PROTEIN4 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN2 PROTEIN5 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN3 PROTEIN4 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN3 PROTEIN5 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN4 PROTEIN5 Subsequent amino acid sequencing revealed many peptides involving PROTEIN0 cross-linked either to itself or to a variety of other known CE protein components, including PROTEIN1 PROTEIN2 PROTEIN3 keratins, members of the small proline-rich superfamily, PROTEIN4 and unknown proteins related to the PROTEIN5 family\n",
      "\n",
      "PROTEIN0 PROTEIN1 Specific glutamines or lysines of PROTEIN0 were used to cross-link the different proteins, such as glutamines 495 and 496 to PROTEIN1 glutamine 288 to keratins, and lysines 468, 485, and 508 and glutamines 465 and 489 for interchain PROTEIN2 cross-links\n",
      "\n",
      "PROTEIN0 PROTEIN2 Specific glutamines or lysines of PROTEIN0 were used to cross-link the different proteins, such as glutamines 495 and 496 to PROTEIN1 glutamine 288 to keratins, and lysines 468, 485, and 508 and glutamines 465 and 489 for interchain PROTEIN2 cross-links\n",
      "\n",
      "PROTEIN1 PROTEIN2 Specific glutamines or lysines of PROTEIN0 were used to cross-link the different proteins, such as glutamines 495 and 496 to PROTEIN1 glutamine 288 to keratins, and lysines 468, 485, and 508 and glutamines 465 and 489 for interchain PROTEIN2 cross-links\n",
      "\n",
      "PROTEIN0 PROTEIN1 A prominent isolate, designated PROTEIN0 PROTEIN1 specifically interacts with the PROTEIN2 not only in vivo but also in vitro\n",
      "\n",
      "PROTEIN0 PROTEIN2 A prominent isolate, designated PROTEIN0 PROTEIN1 specifically interacts with the PROTEIN2 not only in vivo but also in vitro\n",
      "\n",
      "PROTEIN1 PROTEIN2 A prominent isolate, designated PROTEIN0 PROTEIN1 specifically interacts with the PROTEIN2 not only in vivo but also in vitro\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a nuclear protein with a characteristic distribution in large irregularly shaped nuclear speckles and co-localizes perfectly with the SR domain-containing PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Transient expression of the full-length PROTEIN0 without RSRS in U1240MG glioblastoma cells resulted in repression of PROTEIN1 promoter activity\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that the relative abundance of the two splice variants of PROTEIN0 in brain could be an important determinant for the expression of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 stimulates PROTEIN1 production in dose-dependent fashion as well\n",
      "\n",
      "PROTEIN0 PROTEIN1 Phosphoinositide hydrolysis, resulting in PROTEIN0 (IP3) and diacylglycerol (DG) formation, has been implicated in PROTEIN1 pulsatile secretion of prostaglandin F2 alpha (PGF2 alpha) from uterine endometrium of sheep and other mammals\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results indicate that oTP-1 may prevent luteolysis by inhibiting development of endometrial responsiveness to PROTEIN0 and, therefore, reduce PROTEIN1 synthesis of PROTEIN2 and PGF2 alpha\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results indicate that oTP-1 may prevent luteolysis by inhibiting development of endometrial responsiveness to PROTEIN0 and, therefore, reduce PROTEIN1 synthesis of PROTEIN2 and PGF2 alpha\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results indicate that oTP-1 may prevent luteolysis by inhibiting development of endometrial responsiveness to PROTEIN0 and, therefore, reduce PROTEIN1 synthesis of PROTEIN2 and PGF2 alpha\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 binding in human myometrium\n",
      "\n",
      "PROTEIN0 PROTEIN1 The effects of prostaglandin F2 alpha (PGF2 alpha) on intracellular Ca2+ concentration ([Ca2+]i) and PROTEIN0 (IP) generation in human myometrial cells were evaluated and compared to the effects of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, the potencies of PROTEIN0 to raise PROTEIN1 and [Ca2+]i were similar and independent of extracellular Ca2+, and could be suppressed by pertussis toxin and phorbol ester, but not by verapamil\n",
      "\n",
      "PROTEIN0 PROTEIN1 The action of PGF2 alpha depends on extracellular Ca2+, whereas PROTEIN0 activates the G-protein-dependent PROTEIN1 signal-transducing pathway, complemented by the influx of extracellular Ca2+\n",
      "\n",
      "PROTEIN0 PROTEIN1 The objectives of this study were to evaluate and compare the actions of endothelin-1 (ET-1), PROTEIN0 prostaglandin F2 alpha (PGF2 alpha) and PROTEIN1 (IP3) on 45Ca2+ mobilization in permeabilized rat myometrial cells and to examine the activation of the inositol lipid cycle in intact myocytes\n",
      "\n",
      "PROTEIN0 PROTEIN1 All four agonists caused a biphasic release of 45Ca2+ from non-mitochondrial pool(s), with the rank order of potency: PROTEIN0 > PGF2 alpha > ET-1 > PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Endothelin-1 and PROTEIN0 stimulated PROTEIN1 accumulation at concentrations similar to those that promoted 45Ca2+ efflux, whereas about 100 times higher concentrations of PGF2 alpha were needed to activate this signaling pathway in intact cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 It is concluded that the primary action of PGF2 alpha in myometrial cells is to enhance Ca2+ influx, whereas PROTEIN0 and ET-1 receptors are coupled to phospholipase C, generating PROTEIN1 and raising the intracellular concentration of free Ca2+ from intracellular as well as extracellular sources\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 induced a biphasic increase in the intracellular Ca2+ concentration of porcine myometrial cells: participation of a pertussis toxin-insensitive G-protein, PROTEIN1 Ca2+ pool, and Ca2+ channels\n",
      "\n",
      "PROTEIN0 PROTEIN1 Administration of PROTEIN0 (1 microM) for 15 sec increased PROTEIN1 (IP3) formation by 61%\n",
      "\n",
      "PROTEIN0 PROTEIN1 Pretreatment with pertussis toxin (PTX, 1 microgram/ml) for 2 hr failed to alter the PROTEIN0 increase in [Ca2+]i and PROTEIN1 formation\n",
      "\n",
      "PROTEIN0 PROTEIN1 U-73122 (3 microM) also abolished the PROTEIN0 PROTEIN1 formation\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 beta protein disruption of cholinergic and growth factor PROTEIN1 signals could underlie cognitive and neurodegerative aspects of Alzheimer's disease\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 beta protein (25-35) stimulation of PROTEIN1 A, C and D activities of LA-N-2 cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 [3H]Inositol prelabeled LA-N-2 cells were exposed to varying concentrations of PROTEIN0 from 20 to 125 micrograms/ml, and PROTEIN1 activation was measured\n",
      "\n",
      "PROTEIN0 PROTEIN1 The effects of adrenergic, metabotropic amino acid and bombesin antagonists on the PROTEIN0 mediated stimulation of PROTEIN1 activity was investigated\n",
      "\n",
      "PROTEIN0 PROTEIN1 Propranolol, a beta adrenergic antagonist, 7-chloro-kynurenic acid, a metabotropic amino acid antagonist, and [Tyr4-D-Phe12]bombesin, a bombesin antagonist, blunted the PROTEIN0 stimulation of PROTEIN1 activity in [3H]inositol prelabeled LA-N-2 cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 This suggests that PROTEIN0 beta protein activation of PROTEIN1 may be receptor mediated\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 inhibitor U 71322 prevented the activation of PROTEIN1 by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 inhibitor U 71322 prevented the activation of PROTEIN1 by PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 inhibitor U 71322 prevented the activation of PROTEIN1 by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 secretion and PROTEIN1 production in NT2N neurons are regulated by the PROTEIN2 signal transduction pathway\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results suggest that PROTEIN0 secretion and PROTEIN1 production in NT2N neurons are regulated by the PROTEIN2 signal transduction pathway\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results suggest that PROTEIN0 secretion and PROTEIN1 production in NT2N neurons are regulated by the PROTEIN2 signal transduction pathway\n",
      "\n",
      "PROTEIN0 PROTEIN1 Current interest in reducing heart disease risks by diet involves attention to total fat; saturated, monounsaturated, polyunsaturated and trans fatty acids, as well as dietary PROTEIN0 soluble fiber, salt, alcohol, antioxidants, dietary alterations causing homocysteinemia and other dietary constituents, such as PROTEIN1 compounds in some soy products\n",
      "\n",
      "PROTEIN0 PROTEIN1 This indicates that the calcium response to PROTEIN0 application was principally associated with activation of the PROTEIN1 calcium stores\n",
      "\n",
      "PROTEIN0 PROTEIN1 The beta PROTEIN0 protein fragments had diverse effects on phosphoinositide-specific PROTEIN1 (PI-PLC) as assayed in rat cortical membranes\n",
      "\n",
      "PROTEIN0 PROTEIN1 They are needed for the catabolism of PROTEIN0 bile acids, and steroid hormones; they hydrolyze a number of PROTEIN1 glycosides to anticarcinogens; and they detoxify certain carcinogens\n",
      "\n",
      "PROTEIN0 PROTEIN1 The aim of this study was to investigate the effects of hCG, hCG plus PROTEIN0 and PROTEIN1 on [3H] PROTEIN2 (IP) formations in porcine myometrial cells obtained from ovariectomized and cyclic gilts\n",
      "\n",
      "PROTEIN0 PROTEIN2 The aim of this study was to investigate the effects of hCG, hCG plus PROTEIN0 and PROTEIN1 on [3H] PROTEIN2 (IP) formations in porcine myometrial cells obtained from ovariectomized and cyclic gilts\n",
      "\n",
      "PROTEIN1 PROTEIN2 The aim of this study was to investigate the effects of hCG, hCG plus PROTEIN0 and PROTEIN1 on [3H] PROTEIN2 (IP) formations in porcine myometrial cells obtained from ovariectomized and cyclic gilts\n",
      "\n",
      "PROTEIN0 PROTEIN1 There was also a very significant increase of PROTEIN0 after the addition of 1000 mU hCG (p < 0.001) and PROTEIN1 and PROTEIN2 when 1000 mU hCG plus PROTEIN3 were added (p < 0.001 and p < 0.01, respectively)\n",
      "\n",
      "PROTEIN0 PROTEIN2 There was also a very significant increase of PROTEIN0 after the addition of 1000 mU hCG (p < 0.001) and PROTEIN1 and PROTEIN2 when 1000 mU hCG plus PROTEIN3 were added (p < 0.001 and p < 0.01, respectively)\n",
      "\n",
      "PROTEIN0 PROTEIN3 There was also a very significant increase of PROTEIN0 after the addition of 1000 mU hCG (p < 0.001) and PROTEIN1 and PROTEIN2 when 1000 mU hCG plus PROTEIN3 were added (p < 0.001 and p < 0.01, respectively)\n",
      "\n",
      "PROTEIN1 PROTEIN2 There was also a very significant increase of PROTEIN0 after the addition of 1000 mU hCG (p < 0.001) and PROTEIN1 and PROTEIN2 when 1000 mU hCG plus PROTEIN3 were added (p < 0.001 and p < 0.01, respectively)\n",
      "\n",
      "PROTEIN1 PROTEIN3 There was also a very significant increase of PROTEIN0 after the addition of 1000 mU hCG (p < 0.001) and PROTEIN1 and PROTEIN2 when 1000 mU hCG plus PROTEIN3 were added (p < 0.001 and p < 0.01, respectively)\n",
      "\n",
      "PROTEIN2 PROTEIN3 There was also a very significant increase of PROTEIN0 after the addition of 1000 mU hCG (p < 0.001) and PROTEIN1 and PROTEIN2 when 1000 mU hCG plus PROTEIN3 were added (p < 0.001 and p < 0.01, respectively)\n",
      "\n",
      "PROTEIN0 PROTEIN1 Only PROTEIN0 alone increased the formation of PROTEIN1 in cells from ovariectomized pigs treated with estradiol in vivo (p < 0.01)\n",
      "\n",
      "PROTEIN0 PROTEIN1 The highest dose of hCG plus The highest dose of hCG plus PROTEIN0 provoked accumulation of total PROTEIN1 (p < 0.05) 53% over the basal level on days 21/1 of the estrous cycle\n",
      "\n",
      "PROTEIN0 PROTEIN1 The present study demonstrates that hCG and PROTEIN0 can increase the accumulation of PROTEIN1 in porcine myometrial cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 The relative risks between highest and lowest quarters of PROTEIN0 intake adjusted for age, smoking, serum PROTEIN1 concentration, blood pressure, and body mass index were 0.69 (95% confidence interval 0.53 to 0.90) and 0.54 (0.33 to 0.87) for total and coronary mortality, respectively\n",
      "\n",
      "PROTEIN0 PROTEIN1 We propose that PROTEIN0 binds to specific OT receptors (OTR) on the endometrium to stimulate phosphoinositide (PI) hydrolysis, thereby activating the inositol trisphosphate PROTEIN1 (DAG) second-messenger system to promote pulsatile PGF2 alpha secretion\n",
      "\n",
      "PROTEIN0 PROTEIN1 Endometrial PROTEIN0 was increased within 30 seconds after PROTEIN1 treatment and preceded the increase in PGF2 alpha release stimulated by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Endometrial PROTEIN0 was increased within 30 seconds after PROTEIN1 treatment and preceded the increase in PGF2 alpha release stimulated by PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Endometrial PROTEIN0 was increased within 30 seconds after PROTEIN1 treatment and preceded the increase in PGF2 alpha release stimulated by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results support the hypothesis that PROTEIN0 stimulates phospholipase C to hydrolyze PI, yielding PROTEIN1 and DAG second-messengers which promote endometrial PGF2 alpha release during CL regression in pigs\n",
      "\n",
      "PROTEIN0 PROTEIN1 These data suggest that cAMP-dependent phosphorylation at a step involving GTP-binding protein/PLC coupling can exert a negative effect on the stimulation of PROTEIN0 formation by PROTEIN1 and thereby affect contraction/relaxation in the myometrium\n",
      "\n",
      "PROTEIN0 PROTEIN1 The resulting abnormally phosphorylated PROTEIN0 aggregates to freshly synthesized PROTEIN1 transforming it into same; due to a system of positive feedback invoked by the organophosphate-induced blockage of a PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The resulting abnormally phosphorylated PROTEIN0 aggregates to freshly synthesized PROTEIN1 transforming it into same; due to a system of positive feedback invoked by the organophosphate-induced blockage of a PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The resulting abnormally phosphorylated PROTEIN0 aggregates to freshly synthesized PROTEIN1 transforming it into same; due to a system of positive feedback invoked by the organophosphate-induced blockage of a PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The resulting abnormally phosphorylated PROTEIN0 aggregates to freshly synthesized PROTEIN1 transforming it into same; due to a system of positive feedback invoked by the organophosphate-induced blockage of a PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The resulting abnormally phosphorylated PROTEIN0 aggregates to freshly synthesized PROTEIN1 transforming it into same; due to a system of positive feedback invoked by the organophosphate-induced blockage of a PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The resulting abnormally phosphorylated PROTEIN0 aggregates to freshly synthesized PROTEIN1 transforming it into same; due to a system of positive feedback invoked by the organophosphate-induced blockage of a PROTEIN2 PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The aim of the present study was to investigate how PROTEIN0 given subcutaneously (SC) and intracerebroventricularly (ICV) influences the secretion of PROTEIN1 glucagon and glucose and to investigate whether the effect on these variables of suckling in lactating rats is mediated by oxytocinergic mechanisms\n",
      "\n",
      "PROTEIN0 PROTEIN1 Subcutaneous injections of PROTEIN0 increased PROTEIN1 glucagon and glucose levels significantly\n",
      "\n",
      "PROTEIN0 PROTEIN1 Two nanograms PROTEIN0 given ICV had no effect on glucagon and glucose levels but caused a significant rise in PROTEIN1 levels at this time point\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 antagonist 1-deamino-2-D-Tyr-(OEt)-4-Thr-8-Orn-oxytocin administered ICV increased PROTEIN1 levels itself and therefore the effect on PROTEIN2 PROTEIN3 secretion was difficult to evaluate\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 antagonist 1-deamino-2-D-Tyr-(OEt)-4-Thr-8-Orn-oxytocin administered ICV increased PROTEIN1 levels itself and therefore the effect on PROTEIN2 PROTEIN3 secretion was difficult to evaluate\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 antagonist 1-deamino-2-D-Tyr-(OEt)-4-Thr-8-Orn-oxytocin administered ICV increased PROTEIN1 levels itself and therefore the effect on PROTEIN2 PROTEIN3 secretion was difficult to evaluate\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 antagonist 1-deamino-2-D-Tyr-(OEt)-4-Thr-8-Orn-oxytocin administered ICV increased PROTEIN1 levels itself and therefore the effect on PROTEIN2 PROTEIN3 secretion was difficult to evaluate\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 antagonist 1-deamino-2-D-Tyr-(OEt)-4-Thr-8-Orn-oxytocin administered ICV increased PROTEIN1 levels itself and therefore the effect on PROTEIN2 PROTEIN3 secretion was difficult to evaluate\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 antagonist 1-deamino-2-D-Tyr-(OEt)-4-Thr-8-Orn-oxytocin administered ICV increased PROTEIN1 levels itself and therefore the effect on PROTEIN2 PROTEIN3 secretion was difficult to evaluate\n",
      "\n",
      "PROTEIN0 PROTEIN1 Intracerebroventricular injections of 200 ng PROTEIN0 caused a significant rise not only of PROTEIN1 but also of glucagon and glucose levels\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, nanogram amounts of PROTEIN0 administered ICV cause a rise of PROTEIN1 levels\n",
      "\n",
      "PROTEIN0 PROTEIN1 Stimulation of 5HT-1D receptors with sumatriptan was unable to change neither AVP nor PROTEIN0 response to PROTEIN1 hypoglycemia\n",
      "\n",
      "PROTEIN0 PROTEIN1 The SGR and RGR also had different endocrine profiles with, for example, twice as high PROTEIN0 (p < 0.01) and PROTEIN1 levels (p < 0.01) in RGR compared to the SGR\n",
      "\n",
      "PROTEIN0 PROTEIN1 8-OH-DPAT also increased PROTEIN0 and decreased CCK and somatostatin levels, effects that were blocked by pretreatment with the PROTEIN1 antagonist\n",
      "\n",
      "PROTEIN0 PROTEIN1 Taken together, these data suggest that the effect of 8-OH-DPAT on plasma levels of PROTEIN0 somatostatin and CCK may be mediated by PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In previous experiments, we have shown that following i.c.v. application of PROTEIN0 plasma levels of PROTEIN1 are increased through a cholinergic mechanism\n",
      "\n",
      "PROTEIN0 PROTEIN1 In HVSMC both PROTEIN0 and AVP increased PROTEIN1 (IP3) production and [Ca2+]i response, but the efficacy of the responses was greater for PROTEIN2 than AVP\n",
      "\n",
      "PROTEIN0 PROTEIN2 In HVSMC both PROTEIN0 and AVP increased PROTEIN1 (IP3) production and [Ca2+]i response, but the efficacy of the responses was greater for PROTEIN2 than AVP\n",
      "\n",
      "PROTEIN1 PROTEIN2 In HVSMC both PROTEIN0 and AVP increased PROTEIN1 (IP3) production and [Ca2+]i response, but the efficacy of the responses was greater for PROTEIN2 than AVP\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 may rapidly stimulate PROTEIN1 (IP3) and diacylglycerol (DAG) formation, consistent with the concept of rapid activation of a second-messenger system\n",
      "\n",
      "PROTEIN0 PROTEIN1 In support of this hypothesis, endometrial PROTEIN0 levels were increased (P < 0.05) within 0.5 min after treatment with 0.1 microM PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results indicate that PROTEIN0 induces endometrial PROTEIN1 production in a rapid manner indicative of a second-messenger system\n",
      "\n",
      "PROTEIN0 PROTEIN1 Gamma-aminobutyric acid mediation of the inhibitory effect of nitric oxide on the arginine vasopressin and PROTEIN0 responses to PROTEIN1 hypoglycemia\n",
      "\n",
      "PROTEIN0 PROTEIN1 Previous studies have demonstrated that the nitric oxide (NO) synthase inhibitor L-NAME exerts positive effects on the arginine vasopressin (AVP) and PROTEIN0 (OT) responses to PROTEIN1 hypoglycemia, suggesting inhibitory actions of NO\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 hypoglycemia increased by 2-fold (peak vs. baseline) plasma AVP and PROTEIN1 levels\n",
      "\n",
      "PROTEIN0 PROTEIN1 These data indicate a GABAergic mediation of the inhibitory modulation by NO of the AVP and PROTEIN0 responses to PROTEIN1 hypoglycemia\n",
      "\n",
      "PROTEIN0 PROTEIN1 Correlation between the stimulatory effect of PROTEIN0 on the formation of PROTEIN1 and the oxytocin receptor level in the pregnant rabbit myometrium\n",
      "\n",
      "PROTEIN0 PROTEIN1 OBJECTIVE: In order to elucidate the roles of PROTEIN0 (IP3) and oxytocin (OT) receptors in rabbit parturition, the concentration of PROTEIN1 induced by PROTEIN2 and the OT receptor levels were determined in rabbit myometria before and after parturition\n",
      "\n",
      "PROTEIN0 PROTEIN2 OBJECTIVE: In order to elucidate the roles of PROTEIN0 (IP3) and oxytocin (OT) receptors in rabbit parturition, the concentration of PROTEIN1 induced by PROTEIN2 and the OT receptor levels were determined in rabbit myometria before and after parturition\n",
      "\n",
      "PROTEIN1 PROTEIN2 OBJECTIVE: In order to elucidate the roles of PROTEIN0 (IP3) and oxytocin (OT) receptors in rabbit parturition, the concentration of PROTEIN1 induced by PROTEIN2 and the OT receptor levels were determined in rabbit myometria before and after parturition\n",
      "\n",
      "PROTEIN0 PROTEIN1 METHODS: The effects of PROTEIN0 on PROTEIN1 formation and OT receptor levels were determined in the myometria of non-pregnant rabbits, Days 26, 28 and 30 of pregnancy rabbits, postpartum rabbits within 12 hours and steroid-treated ovariectomized rabbits\n",
      "\n",
      "PROTEIN0 PROTEIN1 RESULTS: OT receptors were not detectable in the myometria of non-pregnant rabbits, and PROTEIN0 had no effect on the formation of PROTEIN1 (IPs)\n",
      "\n",
      "PROTEIN0 PROTEIN1 On Day 28 of pregnancy, OT receptors became detectable, and then PROTEIN0 could induce the formation of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thereafter, the stimulatory effects of PROTEIN0 on PROTEIN1 formation and the OT receptor levels dramatically increased toward the end of pregnancy and reduced rapidly after parturition\n",
      "\n",
      "PROTEIN0 PROTEIN1 When the ovariectomized pregnant rabbits were treated with estrogen, OT receptors in the myometrium were induced, and PROTEIN0 acquired the ability to stimulate PROTEIN1 formation\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, an OT receptor antagonist inhibited the stimulatory effects of PROTEIN0 on PROTEIN1 formation\n",
      "\n",
      "PROTEIN0 PROTEIN1 CONCLUSIONS: These results suggest that the formation of PROTEIN0 by PROTEIN1 the OT receptor levels in the myometrium, and the production of PGF2 in the decidua might play crucial roles in parturition\n",
      "\n",
      "PROTEIN0 PROTEIN1 Immunohistochemical localization of PROTEIN0 and PROTEIN1 in white and brown adipose tissue\n",
      "\n",
      "PROTEIN0 PROTEIN1 To establish the cell type expressing PROTEIN0 we also assessed the size and organization of lipid droplets, the ultrastructural features of mitochondria, and the presence or absence of PROTEIN1 a brown fat-specific marker\n",
      "\n",
      "PROTEIN0 PROTEIN1 In brown adipose tissue of lean animals, multilocular PROTEIN0 (UCP)-positive brown adipocytes had typical brown mitochondria and were PROTEIN1 both in fed and fasted conditions\n",
      "\n",
      "PROTEIN0 PROTEIN1 At the periphery of the interscapular brown adipose tissue depot, unilocular, PROTEIN0 adipocytes (mean diameter: 41.55 microns) with white-type mitochondria were observed, and these cells were PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In obese (db/db) animals, brown fat was composed mainly of small unilocular, PROTEIN0 adipocytes (mean diameter: 40.08 microns), which were also PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In summary, classical brown adipocytes differ from white adipocytes, not only by their morphology and PROTEIN0 expression, but also by their apparent lack of detectable PROTEIN1 expression\n",
      "\n",
      "PROTEIN0 PROTEIN1 Wortmannin converts PROTEIN0 but not PROTEIN1 from an antilipolytic to a lipolytic agent in the presence of forskolin\n",
      "\n",
      "PROTEIN0 PROTEIN1 The present studies compared the effects of PROTEIN0 in rat adipocytes with the effects of PROTEIN1 and peroxovanadate, which mimic some effects of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The present studies compared the effects of PROTEIN0 in rat adipocytes with the effects of PROTEIN1 and peroxovanadate, which mimic some effects of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The present studies compared the effects of PROTEIN0 in rat adipocytes with the effects of PROTEIN1 and peroxovanadate, which mimic some effects of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 The antilipolytic effects of peroxovanadate and PROTEIN0 were unaffected by 500 nmol/L wortmannin, which blocked the antilipolytic action of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The data provide additional support for the hypothesis that PROTEIN0 and peroxovanadate affect adipose tissue metabolism by mechanisms distinctly different from those involved in PROTEIN1 action\n",
      "\n",
      "PROTEIN0 PROTEIN1 These findings support the concept that the mitochondrial metabolism of nutrient molecules is an event sufficient to acutely augment PROTEIN0 release from the beta cell, to increase PROTEIN1 phosphoinositide hydrolysis, and to induce time-dependent potentiation of PROTEIN2 secretion\n",
      "\n",
      "PROTEIN0 PROTEIN2 These findings support the concept that the mitochondrial metabolism of nutrient molecules is an event sufficient to acutely augment PROTEIN0 release from the beta cell, to increase PROTEIN1 phosphoinositide hydrolysis, and to induce time-dependent potentiation of PROTEIN2 secretion\n",
      "\n",
      "PROTEIN1 PROTEIN2 These findings support the concept that the mitochondrial metabolism of nutrient molecules is an event sufficient to acutely augment PROTEIN0 release from the beta cell, to increase PROTEIN1 phosphoinositide hydrolysis, and to induce time-dependent potentiation of PROTEIN2 secretion\n",
      "\n",
      "PROTEIN0 PROTEIN1 2) IGF-1, IGF-2, and PROTEIN0 also induced a Ca2+ mobilization from the endoplasmic reticulum: PROTEIN1 (PLC) inhibitors, neomycin, or U-73122 partially blocked the intracellular [Ca2+]i increase induced by IGF-1 and PROTEIN2 and totally inhibited the effect of IGF-2\n",
      "\n",
      "PROTEIN0 PROTEIN2 2) IGF-1, IGF-2, and PROTEIN0 also induced a Ca2+ mobilization from the endoplasmic reticulum: PROTEIN1 (PLC) inhibitors, neomycin, or U-73122 partially blocked the intracellular [Ca2+]i increase induced by IGF-1 and PROTEIN2 and totally inhibited the effect of IGF-2\n",
      "\n",
      "PROTEIN1 PROTEIN2 2) IGF-1, IGF-2, and PROTEIN0 also induced a Ca2+ mobilization from the endoplasmic reticulum: PROTEIN1 (PLC) inhibitors, neomycin, or U-73122 partially blocked the intracellular [Ca2+]i increase induced by IGF-1 and PROTEIN2 and totally inhibited the effect of IGF-2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Homogeneous liposome immunoassay for PROTEIN0 using PROTEIN1 from Clostridium perfringens\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 was conjugated to PROTEIN1 by a three-step procedure with hetero-bifunctional crosslinking reagents, succinimidyl 4-(N-maleimidomethyl)cyclohexane-1-carboxylate and 3-(2-pyridyldithio)propionic acid N-hydroxysuccinimide ester\n",
      "\n",
      "PROTEIN0 PROTEIN1 Both p-nitrophenylphosphatidylcholine hydrolytic activity and liposome lytic activity of PROTEIN0 PROTEIN1 conjugate were inhibited in the presence of PROTEIN2 antiserum\n",
      "\n",
      "PROTEIN0 PROTEIN2 Both p-nitrophenylphosphatidylcholine hydrolytic activity and liposome lytic activity of PROTEIN0 PROTEIN1 conjugate were inhibited in the presence of PROTEIN2 antiserum\n",
      "\n",
      "PROTEIN1 PROTEIN2 Both p-nitrophenylphosphatidylcholine hydrolytic activity and liposome lytic activity of PROTEIN0 PROTEIN1 conjugate were inhibited in the presence of PROTEIN2 antiserum\n",
      "\n",
      "PROTEIN0 PROTEIN1 Roles of GTP and PROTEIN0 in the potentiation of Ca(2+)-induced PROTEIN1 secretion by glucose in rat pancreatic islets\n",
      "\n",
      "PROTEIN0 PROTEIN1 There was a 98% greater augmentation of PROTEIN0 secretion by 16.7 mM glucose (in the presence of diazoxide and 40 mM K+) in primed islets; however, the ability of high glucose to augment PROTEIN1 activity bore no relationship to the secretory response\n",
      "\n",
      "PROTEIN0 PROTEIN1 MPA markedly inhibited PROTEIN0 in both conditions; however, PROTEIN1 secretion was only inhibited (by 46%) in primed islets\n",
      "\n",
      "PROTEIN0 PROTEIN1 These data indicate that an adequate level of GTP is critical for glucose's potentiation of Ca(2+)-induced PROTEIN0 secretion in primed islets but that PROTEIN1 activation can clearly be dissociated from PROTEIN2 secretion and therefore cannot be the major cause of glucose's augmentation of Ca(2+)-induced PROTEIN3 secretion\n",
      "\n",
      "PROTEIN0 PROTEIN2 These data indicate that an adequate level of GTP is critical for glucose's potentiation of Ca(2+)-induced PROTEIN0 secretion in primed islets but that PROTEIN1 activation can clearly be dissociated from PROTEIN2 secretion and therefore cannot be the major cause of glucose's augmentation of Ca(2+)-induced PROTEIN3 secretion\n",
      "\n",
      "PROTEIN0 PROTEIN3 These data indicate that an adequate level of GTP is critical for glucose's potentiation of Ca(2+)-induced PROTEIN0 secretion in primed islets but that PROTEIN1 activation can clearly be dissociated from PROTEIN2 secretion and therefore cannot be the major cause of glucose's augmentation of Ca(2+)-induced PROTEIN3 secretion\n",
      "\n",
      "PROTEIN1 PROTEIN2 These data indicate that an adequate level of GTP is critical for glucose's potentiation of Ca(2+)-induced PROTEIN0 secretion in primed islets but that PROTEIN1 activation can clearly be dissociated from PROTEIN2 secretion and therefore cannot be the major cause of glucose's augmentation of Ca(2+)-induced PROTEIN3 secretion\n",
      "\n",
      "PROTEIN1 PROTEIN3 These data indicate that an adequate level of GTP is critical for glucose's potentiation of Ca(2+)-induced PROTEIN0 secretion in primed islets but that PROTEIN1 activation can clearly be dissociated from PROTEIN2 secretion and therefore cannot be the major cause of glucose's augmentation of Ca(2+)-induced PROTEIN3 secretion\n",
      "\n",
      "PROTEIN2 PROTEIN3 These data indicate that an adequate level of GTP is critical for glucose's potentiation of Ca(2+)-induced PROTEIN0 secretion in primed islets but that PROTEIN1 activation can clearly be dissociated from PROTEIN2 secretion and therefore cannot be the major cause of glucose's augmentation of Ca(2+)-induced PROTEIN3 secretion\n",
      "\n",
      "PROTEIN0 PROTEIN1 Regulation of PROTEIN0 secretion via ATP-sensitive K+ channel independent mechanisms: role of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The further addition of carbachol, an agonist that activates an isozyme of PROTEIN0 distinct from that activated by glucose, together with K+, 20 mM glucose, plus diazoxide resulted in a sustained amplification of PROTEIN1 secretion from mouse but not rat islets\n",
      "\n",
      "PROTEIN0 PROTEIN1 The inability to activate a nutrient- and calcium-regulated PROTEIN0 isozyme in mouse islets to the same extent as in rat islets appears to account, at least in part, for these different PROTEIN1 secretory responses under these unique conditions\n",
      "\n",
      "PROTEIN0 PROTEIN1 Induction by PROTEIN0 of PROTEIN1 and enzymes of fatty acid oxidation\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here we show that PROTEIN0 alters in pancreatic islets the mRNA of the genes encoding enzymes of free fatty acid metabolism and PROTEIN1 (UCP-2)\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 overexpression increased PROTEIN1 mRNA by more than 10-fold in epididymal, retroperitoneal, and subcutaneous fat tissue of normal, but not of leptin-receptor-defective obese rats\n",
      "\n",
      "PROTEIN0 PROTEIN1 By directly regulating the expression of enzymes of free fatty acid metabolism and of PROTEIN0 PROTEIN1 controls intracellular triglyceride content of certain nonadipocytes, as well as adipocytes\n",
      "\n",
      "PROTEIN0 PROTEIN1 Further, the PROTEIN0 activation of IK(sl) was effectively antagonized by 5 x 10(-8) mol/l U-73122 or 5 x 10(-6) mol/l 2-nitro-4-carboxyphenyl N,N-diphenylcarbamate (inhibitors of the cell membrane phospholipase C), as well as by intracellularly applied heparin (selective inhibitor of PROTEIN1 (IP3)-induced Ca2+ release channels)\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 increases PROTEIN1 expression and energy expenditure\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 increased BAT PROTEIN1 mRNA levels greater than twofold in both ad libitum-fed and food-restricted rats\n",
      "\n",
      "PROTEIN0 PROTEIN1 These data demonstrate a PROTEIN0 increase in energy expenditure in nonmutant rodents and suggest that one mechanism by which PROTEIN1 increases energy expenditure is through increased thermogenesis in BAT, including increased expression of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 These data demonstrate a PROTEIN0 increase in energy expenditure in nonmutant rodents and suggest that one mechanism by which PROTEIN1 increases energy expenditure is through increased thermogenesis in BAT, including increased expression of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 These data demonstrate a PROTEIN0 increase in energy expenditure in nonmutant rodents and suggest that one mechanism by which PROTEIN1 increases energy expenditure is through increased thermogenesis in BAT, including increased expression of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a mediator of thermogenesis regulated by thyroid hormone, beta3-adrenergic agonists, and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 mRNA levels were also regulated by dexamethasone, PROTEIN1 and starvation, albeit differently in muscle and brown adipose tissue\n",
      "\n",
      "PROTEIN0 PROTEIN1 This effect is maximal at 10(-8) M PROTEIN0 (a concentration clearly above the normal physiological range of PROTEIN1 concentrations) and involves the V2 receptor pathway, while activation of PROTEIN2 or changes in intracellular calcium are ineffective\n",
      "\n",
      "PROTEIN0 PROTEIN2 This effect is maximal at 10(-8) M PROTEIN0 (a concentration clearly above the normal physiological range of PROTEIN1 concentrations) and involves the V2 receptor pathway, while activation of PROTEIN2 or changes in intracellular calcium are ineffective\n",
      "\n",
      "PROTEIN1 PROTEIN2 This effect is maximal at 10(-8) M PROTEIN0 (a concentration clearly above the normal physiological range of PROTEIN1 concentrations) and involves the V2 receptor pathway, while activation of PROTEIN2 or changes in intracellular calcium are ineffective\n",
      "\n",
      "PROTEIN0 PROTEIN1 After PROTEIN0 activity was functionally depleted by treating cells with phorbol 12-myristate 13-acetate for 24 hours, PROTEIN1 did not augment IL-1beta-induced NO production\n",
      "\n",
      "PROTEIN0 PROTEIN1 The effect of PROTEIN0 was also inhibited in the presence of the PROTEIN1 inhibitor calphostin C\n",
      "\n",
      "PROTEIN0 PROTEIN1 The addition of PROTEIN0 increased PROTEIN1 activity in cardiac myocytes, and its effect was significantly inhibited in the presence of calphostin C\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results support the hypothesis that the heart may be a target organ for PROTEIN0 and that PROTEIN1 modulates IL-1beta-induced iNOS expression in myocytes through the V1a receptor, which is mediated at least partially via activation of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results support the hypothesis that the heart may be a target organ for PROTEIN0 and that PROTEIN1 modulates IL-1beta-induced iNOS expression in myocytes through the V1a receptor, which is mediated at least partially via activation of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results support the hypothesis that the heart may be a target organ for PROTEIN0 and that PROTEIN1 modulates IL-1beta-induced iNOS expression in myocytes through the V1a receptor, which is mediated at least partially via activation of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Activation of EGFR overexpressing 3T3-L1 adipocytes leads to a 3.4 +/- 1.2-fold stimulation of PROTEIN0 activity over basal levels vs. only 1.06 +/- 0.01-fold stimulation by PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Despite the low levels of PROTEIN0 PROTEIN1 activity, PROTEIN2 glucose transport activity was similarly inhibited by U73122 (55.9 +/- 13.1% inhibition)\n",
      "\n",
      "PROTEIN0 PROTEIN2 Despite the low levels of PROTEIN0 PROTEIN1 activity, PROTEIN2 glucose transport activity was similarly inhibited by U73122 (55.9 +/- 13.1% inhibition)\n",
      "\n",
      "PROTEIN1 PROTEIN2 Despite the low levels of PROTEIN0 PROTEIN1 activity, PROTEIN2 glucose transport activity was similarly inhibited by U73122 (55.9 +/- 13.1% inhibition)\n",
      "\n",
      "PROTEIN0 PROTEIN1 Inhibition of PROTEIN0 activation did not impair either EGF- or PROTEIN1 activation of glycogen synthase or incorporation of glucose into lipid, supporting the hypothesis that both EGF- and PROTEIN2 glucose disposal can be independent of GLUT4-mediated glucose transport.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Inhibition of PROTEIN0 activation did not impair either EGF- or PROTEIN1 activation of glycogen synthase or incorporation of glucose into lipid, supporting the hypothesis that both EGF- and PROTEIN2 glucose disposal can be independent of GLUT4-mediated glucose transport.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Inhibition of PROTEIN0 activation did not impair either EGF- or PROTEIN1 activation of glycogen synthase or incorporation of glucose into lipid, supporting the hypothesis that both EGF- and PROTEIN2 glucose disposal can be independent of GLUT4-mediated glucose transport.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The diminution of glucose transport secondary to inhibition of PROTEIN0 activity was reflected by a decrease in GLUT4 translocation to the plasma membrane upon either EGF or PROTEIN1 stimulation\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have recently shown that PROTEIN0 causes a PROTEIN1 (PKC)-dependent increase in ACTH release and biosynthesis in ovine anterior pituitary cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 In these cells, PROTEIN0 also causes the translocation of PROTEIN1 from the cytosol to the cell membrane which is maximal at 5 min, but the intracellular events distal to PROTEIN2 activation that underlie ACTH secretion have not been well characterized to date\n",
      "\n",
      "PROTEIN0 PROTEIN2 In these cells, PROTEIN0 also causes the translocation of PROTEIN1 from the cytosol to the cell membrane which is maximal at 5 min, but the intracellular events distal to PROTEIN2 activation that underlie ACTH secretion have not been well characterized to date\n",
      "\n",
      "PROTEIN1 PROTEIN2 In these cells, PROTEIN0 also causes the translocation of PROTEIN1 from the cytosol to the cell membrane which is maximal at 5 min, but the intracellular events distal to PROTEIN2 activation that underlie ACTH secretion have not been well characterized to date\n",
      "\n",
      "PROTEIN0 PROTEIN1 The activity of the MAPKs, induced by PROTEIN0 or PMA was inhibited by downregulation of PROTEIN1 (PKC), by the tyrosine kinase inhibitor genistein and by MAPK kinase (MEK) inhibitor, PD98059\n",
      "\n",
      "PROTEIN0 PROTEIN1 We suggest that PROTEIN0 activates the 42/44kDa MAPKs through a signal transduction pathway that involves stimulation of AVP-V1 receptor, tyrosine kinase, PROTEIN1 and MEK\n",
      "\n",
      "PROTEIN0 PROTEIN1 Secreted form of PROTEIN0 precursor protein activates protein kinase C and PROTEIN1 in cultured embryonic rat neocortical cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, we demonstrate that tyrosine phosphorylation of PROTEIN0 and formation of inositol 1,4,5-trisphosphate were increased by PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 male rats had increased circulating levels of cholecystokinin, a tendency to increased plasma levels of PROTEIN1 (p = 0.066), and relatively more adipose tissue in the thigh and interscapular region, compared with controls\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, in brown fat, we observed a 2-3-fold increase in the expression of PROTEIN0 in response to dietary fat challenge, which may be related to diet-induced elevations in plasma PROTEIN1 levels\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 activation of PROTEIN1 in renal epithelial cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 induced a biphasic increase in diacylglycerol generation, characterized by an initial rapid rise and then a sustained elevation, and PROTEIN1 activation, reflected by phosphorylation of a specific 80 kDa myristoylated alanine-rich PKC substrate (MARCKS)\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results argue that Ca(2+)-dependent PROTEIN0 is involved in the action of PROTEIN1 and that of other agonists, which stimulate sodium transport\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, the PROTEIN0 stimulation required activation of PROTEIN1 (PKC), whereas the inhibition was PROTEIN2 independent, indicating that the NGF-induced signaling pathways leading to inhibition and stimulation of HCO3- absorption are distinct\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, the PROTEIN0 stimulation required activation of PROTEIN1 (PKC), whereas the inhibition was PROTEIN2 independent, indicating that the NGF-induced signaling pathways leading to inhibition and stimulation of HCO3- absorption are distinct\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, the PROTEIN0 stimulation required activation of PROTEIN1 (PKC), whereas the inhibition was PROTEIN2 independent, indicating that the NGF-induced signaling pathways leading to inhibition and stimulation of HCO3- absorption are distinct\n",
      "\n",
      "PROTEIN0 PROTEIN1 Association of the insulin receptor with PROTEIN0 (PLCgamma) in 3T3-L1 adipocytes suggests a role for PROTEIN1 in metabolic signaling by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Association of the insulin receptor with PROTEIN0 (PLCgamma) in 3T3-L1 adipocytes suggests a role for PROTEIN1 in metabolic signaling by PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Association of the insulin receptor with PROTEIN0 (PLCgamma) in 3T3-L1 adipocytes suggests a role for PROTEIN1 in metabolic signaling by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 To determine the functional significance of the interaction of PROTEIN0 and the IR, we used a specific inhibitor of PROTEIN1 U73122, or microinjection of SH2 domain glutathione S-transferase fusion proteins derived from PROTEIN2 to block PROTEIN3 GLUT4 translocation\n",
      "\n",
      "PROTEIN0 PROTEIN2 To determine the functional significance of the interaction of PROTEIN0 and the IR, we used a specific inhibitor of PROTEIN1 U73122, or microinjection of SH2 domain glutathione S-transferase fusion proteins derived from PROTEIN2 to block PROTEIN3 GLUT4 translocation\n",
      "\n",
      "PROTEIN0 PROTEIN3 To determine the functional significance of the interaction of PROTEIN0 and the IR, we used a specific inhibitor of PROTEIN1 U73122, or microinjection of SH2 domain glutathione S-transferase fusion proteins derived from PROTEIN2 to block PROTEIN3 GLUT4 translocation\n",
      "\n",
      "PROTEIN1 PROTEIN2 To determine the functional significance of the interaction of PROTEIN0 and the IR, we used a specific inhibitor of PROTEIN1 U73122, or microinjection of SH2 domain glutathione S-transferase fusion proteins derived from PROTEIN2 to block PROTEIN3 GLUT4 translocation\n",
      "\n",
      "PROTEIN1 PROTEIN3 To determine the functional significance of the interaction of PROTEIN0 and the IR, we used a specific inhibitor of PROTEIN1 U73122, or microinjection of SH2 domain glutathione S-transferase fusion proteins derived from PROTEIN2 to block PROTEIN3 GLUT4 translocation\n",
      "\n",
      "PROTEIN2 PROTEIN3 To determine the functional significance of the interaction of PROTEIN0 and the IR, we used a specific inhibitor of PROTEIN1 U73122, or microinjection of SH2 domain glutathione S-transferase fusion proteins derived from PROTEIN2 to block PROTEIN3 GLUT4 translocation\n",
      "\n",
      "PROTEIN0 PROTEIN1 Inhibition of 5-HT3 serotonergic receptors with ondansetron (4 or 8 mg) did not modify the basal secretion of AVP and PROTEIN0 and the PROTEIN1 response to PROTEIN2 hypoglycemia\n",
      "\n",
      "PROTEIN0 PROTEIN2 Inhibition of 5-HT3 serotonergic receptors with ondansetron (4 or 8 mg) did not modify the basal secretion of AVP and PROTEIN0 and the PROTEIN1 response to PROTEIN2 hypoglycemia\n",
      "\n",
      "PROTEIN1 PROTEIN2 Inhibition of 5-HT3 serotonergic receptors with ondansetron (4 or 8 mg) did not modify the basal secretion of AVP and PROTEIN0 and the PROTEIN1 response to PROTEIN2 hypoglycemia\n",
      "\n",
      "PROTEIN0 PROTEIN1 HvSPY coexpression largely abolished PROTEIN0 activity of an PROTEIN1 promoter.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Treatment with (+)-trifluoro-ABA strongly inhibited the PROTEIN0 expression of PROTEIN1 I-1 encoded by RAmy1A in the aleurone layers of embryoless half-seeds at the levels of transcription, protein synthesis, and enzyme activity\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, treatment of isolated rat diaphragms and adipocytes with PIG-P as well as with other agents exerting partially PROTEIN0 activity, such as PI-specific PROTEIN1 (PLC) and the sulfonylurea glimepiride, triggered tyrosine phosphorylation of the caveolar marker protein caveolin, which was apparently correlated with stimulation of lipogenesis\n",
      "\n",
      "PROTEIN0 PROTEIN1 Strikingly, in adipocytes subjected to combined trypsin/salt treatment, PIG-P, PI-specific PROTEIN0 and glimepiride failed completely to provoke PROTEIN1 effects\n",
      "\n",
      "PROTEIN0 PROTEIN1 Chronic central PROTEIN0 infusion enhances insulin-stimulated glucose metabolism and favors the expression of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In marked contrast, intracerebroventricular PROTEIN0 administration was accompanied by the maintenance of high PROTEIN1 PROTEIN2 and PROTEIN3 expression in all these tissues\n",
      "\n",
      "PROTEIN0 PROTEIN2 In marked contrast, intracerebroventricular PROTEIN0 administration was accompanied by the maintenance of high PROTEIN1 PROTEIN2 and PROTEIN3 expression in all these tissues\n",
      "\n",
      "PROTEIN0 PROTEIN3 In marked contrast, intracerebroventricular PROTEIN0 administration was accompanied by the maintenance of high PROTEIN1 PROTEIN2 and PROTEIN3 expression in all these tissues\n",
      "\n",
      "PROTEIN1 PROTEIN2 In marked contrast, intracerebroventricular PROTEIN0 administration was accompanied by the maintenance of high PROTEIN1 PROTEIN2 and PROTEIN3 expression in all these tissues\n",
      "\n",
      "PROTEIN1 PROTEIN3 In marked contrast, intracerebroventricular PROTEIN0 administration was accompanied by the maintenance of high PROTEIN1 PROTEIN2 and PROTEIN3 expression in all these tissues\n",
      "\n",
      "PROTEIN2 PROTEIN3 In marked contrast, intracerebroventricular PROTEIN0 administration was accompanied by the maintenance of high PROTEIN1 PROTEIN2 and PROTEIN3 expression in all these tissues\n",
      "\n",
      "PROTEIN0 PROTEIN1 While PROTEIN0 maintains or favors energy-dissipating mechanisms PROTEIN1 PROTEIN2 and PROTEIN3 the latter are markedly depressed in pair-fed rats\n",
      "\n",
      "PROTEIN0 PROTEIN2 While PROTEIN0 maintains or favors energy-dissipating mechanisms PROTEIN1 PROTEIN2 and PROTEIN3 the latter are markedly depressed in pair-fed rats\n",
      "\n",
      "PROTEIN0 PROTEIN3 While PROTEIN0 maintains or favors energy-dissipating mechanisms PROTEIN1 PROTEIN2 and PROTEIN3 the latter are markedly depressed in pair-fed rats\n",
      "\n",
      "PROTEIN1 PROTEIN2 While PROTEIN0 maintains or favors energy-dissipating mechanisms PROTEIN1 PROTEIN2 and PROTEIN3 the latter are markedly depressed in pair-fed rats\n",
      "\n",
      "PROTEIN1 PROTEIN3 While PROTEIN0 maintains or favors energy-dissipating mechanisms PROTEIN1 PROTEIN2 and PROTEIN3 the latter are markedly depressed in pair-fed rats\n",
      "\n",
      "PROTEIN2 PROTEIN3 While PROTEIN0 maintains or favors energy-dissipating mechanisms PROTEIN1 PROTEIN2 and PROTEIN3 the latter are markedly depressed in pair-fed rats\n",
      "\n",
      "PROTEIN0 PROTEIN1 Melatonin inhibits PROTEIN0 response to PROTEIN1 hypoglycemia, but not to angiotensin II in normal men\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, the PROTEIN0 response to PROTEIN1 hypoglycemia was significantly reduced by melatonin treatment\n",
      "\n",
      "PROTEIN0 PROTEIN1 Three weeks after administration of STZ, the diabetic rats were subcutaneously treated with PROTEIN0 for 1 week, which resulted in significant suppression of the induction of nNOS, AVP and PROTEIN1 gene expression in the PVN and SON\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 evokes the release of Ca2+ from PROTEIN1 intracellular stores\n",
      "\n",
      "PROTEIN0 PROTEIN1 A single i.c.v. injection of PROTEIN0 decreased cumulative food intake and body weight gain, and increased PROTEIN1 mRNA expression during 3 h at the onset of the dark phase\n",
      "\n",
      "PROTEIN0 PROTEIN1 Co-injection of SHU9119 also inhibited completely the PROTEIN0 increase in PROTEIN1 mRNA expression in the BAT\n",
      "\n",
      "PROTEIN0 PROTEIN1 It has been established that PROTEIN0 beta peptide (AbetaP) activates phospholipase A2, PROTEIN1 and phospholipase D of LA-N-2 cells and other cell types\n",
      "\n",
      "PROTEIN0 PROTEIN1 (-)Nicotine did not blunt the PROTEIN0 activation of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Neural site of PROTEIN0 influence on neuropeptide Y signaling pathways altering feeding and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 NPY in the PVN increases feeding and decreases PROTEIN0 (UCP) activity in brown fat, whereas PROTEIN1 decreases NPY biosynthesis in the Arc, which presumably decreases PVN NPY\n",
      "\n",
      "PROTEIN0 PROTEIN1 Next, we determined that PROTEIN0 decreases NPY and increases PROTEIN1 gene expression\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, immunocytochemical studies showed the PROTEIN0 neuronal MAC expression on the SH-SY5Y cells after CD59 was removed by PROTEIN1 or blocked by anti-CD59 antibody\n",
      "\n",
      "PROTEIN0 PROTEIN1 Effect of residual endogenous PROTEIN0 secretion on the abnormal PROTEIN1 response to hypoglycaemia in PROTEIN2 diabetics\n",
      "\n",
      "PROTEIN0 PROTEIN2 Effect of residual endogenous PROTEIN0 secretion on the abnormal PROTEIN1 response to hypoglycaemia in PROTEIN2 diabetics\n",
      "\n",
      "PROTEIN1 PROTEIN2 Effect of residual endogenous PROTEIN0 secretion on the abnormal PROTEIN1 response to hypoglycaemia in PROTEIN2 diabetics\n",
      "\n",
      "PROTEIN0 PROTEIN1 Since previous studies showed that AVP secretion is influenced by the persistence of residual endogenous PROTEIN0 secretion, we wondered whether this factor also regulates PROTEIN1 secretion\n",
      "\n",
      "PROTEIN0 PROTEIN1 DESIGN: Case-control study: the PROTEIN0 response to PROTEIN1 hypoglycaemia was measured in normal and diabetic patients with or without residual endogenous PROTEIN2 secretion\n",
      "\n",
      "PROTEIN0 PROTEIN2 DESIGN: Case-control study: the PROTEIN0 response to PROTEIN1 hypoglycaemia was measured in normal and diabetic patients with or without residual endogenous PROTEIN2 secretion\n",
      "\n",
      "PROTEIN1 PROTEIN2 DESIGN: Case-control study: the PROTEIN0 response to PROTEIN1 hypoglycaemia was measured in normal and diabetic patients with or without residual endogenous PROTEIN2 secretion\n",
      "\n",
      "PROTEIN0 PROTEIN1 Blood samples for PROTEIN0 assay were taken just before the rapid injection of PROTEIN1 (time 0) and at time 15, 30, 45 and 60 min\n",
      "\n",
      "PROTEIN0 PROTEIN1 CONCLUSIONS: These data indicate that a residual endogenous PROTEIN0 secretion exerts a partial protective action against the hypothalamic-pituitary disorder affecting the PROTEIN1 secretory system in IDDM\n",
      "\n",
      "PROTEIN0 PROTEIN1 A barley gene encoding a novel DNA-binding protein (HRT) was identified by southwestern screening with baits containing a PROTEIN0 phytohormone response element from an PROTEIN1 promoter\n",
      "\n",
      "PROTEIN0 PROTEIN1 These observations suggest that bradykinin, like PROTEIN0 activates phospholipase C which generates PROTEIN1 with a subsequent release of Ca2+ from intracellular stores followed by store-operated Ca2+ influx\n",
      "\n",
      "PROTEIN0 PROTEIN1 These findings led us to conclude that inositol trisphosphate [corrected] causes Ca2+ mobilization by muscarinic activation of PROTEIN0 leading to intracellular translocation of PROTEIN1 granules to the ready-releasable pool in pancreatic beta-cells via Ca2+/calmodulin-dependent phosphorylation of myosin light chains\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results clearly demonstrate that PROTEIN0 gene expression is upregulated by TZDs in the WAT and BAT in Wistar fatty rats, an obese model with PROTEIN1 receptor defect, and that adipose PROTEIN2 gene expression is increased in response to TZDs in vitro\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results clearly demonstrate that PROTEIN0 gene expression is upregulated by TZDs in the WAT and BAT in Wistar fatty rats, an obese model with PROTEIN1 receptor defect, and that adipose PROTEIN2 gene expression is increased in response to TZDs in vitro\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results clearly demonstrate that PROTEIN0 gene expression is upregulated by TZDs in the WAT and BAT in Wistar fatty rats, an obese model with PROTEIN1 receptor defect, and that adipose PROTEIN2 gene expression is increased in response to TZDs in vitro\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 and PROTEIN2 gene expression: modulation by food restriction and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 gene expression: modulation by food restriction and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 PROTEIN1 and PROTEIN2 gene expression: modulation by food restriction and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 PROTEIN1 and PROTEIN2 gene expression: modulation by food restriction and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 PROTEIN1 and PROTEIN2 gene expression: modulation by food restriction and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 PROTEIN1 and PROTEIN2 gene expression: modulation by food restriction and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 increased LPL mRNA by 80%, PROTEIN1 mRNA twofold, and PROTEIN2 mRNA levels by 62% in BAT, and increased PROTEIN3 mRNA levels twofold in EWAT\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 increased LPL mRNA by 80%, PROTEIN1 mRNA twofold, and PROTEIN2 mRNA levels by 62% in BAT, and increased PROTEIN3 mRNA levels twofold in EWAT\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 increased LPL mRNA by 80%, PROTEIN1 mRNA twofold, and PROTEIN2 mRNA levels by 62% in BAT, and increased PROTEIN3 mRNA levels twofold in EWAT\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 increased LPL mRNA by 80%, PROTEIN1 mRNA twofold, and PROTEIN2 mRNA levels by 62% in BAT, and increased PROTEIN3 mRNA levels twofold in EWAT\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 increased LPL mRNA by 80%, PROTEIN1 mRNA twofold, and PROTEIN2 mRNA levels by 62% in BAT, and increased PROTEIN3 mRNA levels twofold in EWAT\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 increased LPL mRNA by 80%, PROTEIN1 mRNA twofold, and PROTEIN2 mRNA levels by 62% in BAT, and increased PROTEIN3 mRNA levels twofold in EWAT\n",
      "\n",
      "PROTEIN0 PROTEIN1 The present study indicates that PROTEIN0 increases the gene expression of PROTEIN1 in EWAT and that of PROTEIN2 PROTEIN3 and LPL in BAT, whereas reduced food consumption but not leptin, decreases LPL expression in WAT\n",
      "\n",
      "PROTEIN0 PROTEIN2 The present study indicates that PROTEIN0 increases the gene expression of PROTEIN1 in EWAT and that of PROTEIN2 PROTEIN3 and LPL in BAT, whereas reduced food consumption but not leptin, decreases LPL expression in WAT\n",
      "\n",
      "PROTEIN0 PROTEIN3 The present study indicates that PROTEIN0 increases the gene expression of PROTEIN1 in EWAT and that of PROTEIN2 PROTEIN3 and LPL in BAT, whereas reduced food consumption but not leptin, decreases LPL expression in WAT\n",
      "\n",
      "PROTEIN1 PROTEIN2 The present study indicates that PROTEIN0 increases the gene expression of PROTEIN1 in EWAT and that of PROTEIN2 PROTEIN3 and LPL in BAT, whereas reduced food consumption but not leptin, decreases LPL expression in WAT\n",
      "\n",
      "PROTEIN1 PROTEIN3 The present study indicates that PROTEIN0 increases the gene expression of PROTEIN1 in EWAT and that of PROTEIN2 PROTEIN3 and LPL in BAT, whereas reduced food consumption but not leptin, decreases LPL expression in WAT\n",
      "\n",
      "PROTEIN2 PROTEIN3 The present study indicates that PROTEIN0 increases the gene expression of PROTEIN1 in EWAT and that of PROTEIN2 PROTEIN3 and LPL in BAT, whereas reduced food consumption but not leptin, decreases LPL expression in WAT\n",
      "\n",
      "PROTEIN0 PROTEIN1 DNase1 footprints suggest the involvement of at least three types of transcription factors in the regulation of PROTEIN0 by PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The regulation of PROTEIN0 genes by PROTEIN1 therefore involves an interplay of at least three different types of transcription factor\n",
      "\n",
      "PROTEIN0 PROTEIN1 Oxytocin-stimulated PROTEIN0 release in a clonal beta-cell line RINm5F: involvement of PROTEIN1 and -independent pathways.\n",
      "\n",
      "PROTEIN0 PROTEIN1 CONCLUSION: Oxy increases PROTEIN0 release through both PROTEIN1 and non-PLC mediated signal transduction mechanisms.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, U-73122 diminished the PROTEIN0 increase in intracellular concentration of PROTEIN1 (IP3)\n",
      "\n",
      "PROTEIN0 PROTEIN1 U-73122 at 8 mumol.L-1 totally abolished the PROTEIN0 increases in [Ca2+]i and PROTEIN1 however it reduced the PROTEIN2 increase in insulin release only by 36% and 63% in the monolayer and suspended cell preparations, respectively\n",
      "\n",
      "PROTEIN0 PROTEIN2 U-73122 at 8 mumol.L-1 totally abolished the PROTEIN0 increases in [Ca2+]i and PROTEIN1 however it reduced the PROTEIN2 increase in insulin release only by 36% and 63% in the monolayer and suspended cell preparations, respectively\n",
      "\n",
      "PROTEIN1 PROTEIN2 U-73122 at 8 mumol.L-1 totally abolished the PROTEIN0 increases in [Ca2+]i and PROTEIN1 however it reduced the PROTEIN2 increase in insulin release only by 36% and 63% in the monolayer and suspended cell preparations, respectively\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 release in a clonal beta-cell line RINm5F: involvement of phospholipase C-dependent and -independent pathways\n",
      "\n",
      "PROTEIN0 PROTEIN1 AIM: To study the mechanisms underlying PROTEIN0 (Oxy)-induced PROTEIN1 release\n",
      "\n",
      "PROTEIN0 PROTEIN1 RESULTS: PROTEIN0 increased PROTEIN1 release and [Ca2+]i in a concentration-dependent manner\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 release was not altered by pretreatment with pertussis toxin (PT)\n",
      "\n",
      "PROTEIN0 PROTEIN1 U-73122 at 8 mumol.L-1 totally abolished the PROTEIN0 increases in [Ca2+]i and IP3; however it reduced the PROTEIN1 increase in PROTEIN2 release only by 36% and 63% in the monolayer and suspended cell preparations, respectively\n",
      "\n",
      "PROTEIN0 PROTEIN2 U-73122 at 8 mumol.L-1 totally abolished the PROTEIN0 increases in [Ca2+]i and IP3; however it reduced the PROTEIN1 increase in PROTEIN2 release only by 36% and 63% in the monolayer and suspended cell preparations, respectively\n",
      "\n",
      "PROTEIN1 PROTEIN2 U-73122 at 8 mumol.L-1 totally abolished the PROTEIN0 increases in [Ca2+]i and IP3; however it reduced the PROTEIN1 increase in PROTEIN2 release only by 36% and 63% in the monolayer and suspended cell preparations, respectively\n",
      "\n",
      "PROTEIN0 PROTEIN1 CONCLUSION: PROTEIN0 increases PROTEIN1 release through both PLC and non-PLC mediated signal transduction mechanisms\n",
      "\n",
      "PROTEIN0 PROTEIN1 In isolated adipocytes, the loss of almost half the 18S RNA content over a 24-hour incubation was prevented in the presence of PROTEIN0 but not PROTEIN1 or epidermal growth factor (EGF)\n",
      "\n",
      "PROTEIN0 PROTEIN1 Further investigation suggested that activation of PROTEIN0 (PKC), which is part of the PROTEIN1 intracellular signalling pathway, is necessary and sufficient for the generation of the synergistic response, although it is not obligatory for PROTEIN2 ACTH release\n",
      "\n",
      "PROTEIN0 PROTEIN2 Further investigation suggested that activation of PROTEIN0 (PKC), which is part of the PROTEIN1 intracellular signalling pathway, is necessary and sufficient for the generation of the synergistic response, although it is not obligatory for PROTEIN2 ACTH release\n",
      "\n",
      "PROTEIN1 PROTEIN2 Further investigation suggested that activation of PROTEIN0 (PKC), which is part of the PROTEIN1 intracellular signalling pathway, is necessary and sufficient for the generation of the synergistic response, although it is not obligatory for PROTEIN2 ACTH release\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 (OT) is more expressed in the TEC/TNC than vasopressin (VP); insulin-like growth factor 2 (IGF-2) thymic expression predominates over IGF-1, and much more over PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, PROTEIN0 was proposed to be the self antigen of the neurohypophysial family, and IGF-2 the self antigen precursor of the PROTEIN1 family\n",
      "\n",
      "PROTEIN0 PROTEIN1 Downregulation of PROTEIN0 mRNA in white adipose tissue and PROTEIN1 mRNA in skeletal muscle during the early stages of PROTEIN2 treatment\n",
      "\n",
      "PROTEIN0 PROTEIN2 Downregulation of PROTEIN0 mRNA in white adipose tissue and PROTEIN1 mRNA in skeletal muscle during the early stages of PROTEIN2 treatment\n",
      "\n",
      "PROTEIN1 PROTEIN2 Downregulation of PROTEIN0 mRNA in white adipose tissue and PROTEIN1 mRNA in skeletal muscle during the early stages of PROTEIN2 treatment\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 by intravenous or intracerebroventricular infusion for 5 h was associated with a decrease in PROTEIN1 mRNA in WAT (47-52%) and PROTEIN2 mRNA in SM (33-37%)\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 by intravenous or intracerebroventricular infusion for 5 h was associated with a decrease in PROTEIN1 mRNA in WAT (47-52%) and PROTEIN2 mRNA in SM (33-37%)\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 by intravenous or intracerebroventricular infusion for 5 h was associated with a decrease in PROTEIN1 mRNA in WAT (47-52%) and PROTEIN2 mRNA in SM (33-37%)\n",
      "\n",
      "PROTEIN0 PROTEIN1 Denervation suppressed mRNA levels for PROTEIN0 (49%), PROTEIN1 (36%), and COX-IV (59%) and eliminated the acute response to PROTEIN2 in SM\n",
      "\n",
      "PROTEIN0 PROTEIN2 Denervation suppressed mRNA levels for PROTEIN0 (49%), PROTEIN1 (36%), and COX-IV (59%) and eliminated the acute response to PROTEIN2 in SM\n",
      "\n",
      "PROTEIN1 PROTEIN2 Denervation suppressed mRNA levels for PROTEIN0 (49%), PROTEIN1 (36%), and COX-IV (59%) and eliminated the acute response to PROTEIN2 in SM\n",
      "\n",
      "PROTEIN0 PROTEIN1 The specific PROTEIN0 (PKC) inhibitor chelerythrine (5 micrometers) inhibited PTH-, PROTEIN1 PGE2-, and adenosine-stimulated Ca2+ reabsorption by 77%, 67%, 79%, and 100%, respectively\n",
      "\n",
      "PROTEIN0 PROTEIN1 CONCLUSION: PTH, PROTEIN0 PGE2, and adenosine stimulate Ca2+ reabsorption via a pathway that is independent of cAMP and that involves a phorbol ester-insensitive PROTEIN1 isotype\n",
      "\n",
      "PROTEIN0 PROTEIN1 We observed that primary microglial cultures and the THP-1 monocytic cell line are stimulated by fibrillar beta-amyloid and PROTEIN0 peptides to activate identical PROTEIN1 inflammatory signal transduction cascades\n",
      "\n",
      "PROTEIN0 PROTEIN1 Enhanced phosphoinositide hydrolysis via overexpression of PROTEIN0 beta1 or delta1 inhibits stimulus-induced PROTEIN1 release in insulinoma MIN6 cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 To study the effects of enhanced phosphoinositide hydrolysis on PROTEIN0 secretion, phosphoinositide-specific PROTEIN1 (PLCbeta1) or PROTEIN2 was overexpressed in insulinoma MIN6 cells via adenoviral vectors.\n",
      "\n",
      "PROTEIN0 PROTEIN2 To study the effects of enhanced phosphoinositide hydrolysis on PROTEIN0 secretion, phosphoinositide-specific PROTEIN1 (PLCbeta1) or PROTEIN2 was overexpressed in insulinoma MIN6 cells via adenoviral vectors.\n",
      "\n",
      "PROTEIN1 PROTEIN2 To study the effects of enhanced phosphoinositide hydrolysis on PROTEIN0 secretion, phosphoinositide-specific PROTEIN1 (PLCbeta1) or PROTEIN2 was overexpressed in insulinoma MIN6 cells via adenoviral vectors.\n",
      "\n",
      "PROTEIN0 PROTEIN1 If daily PROTEIN0 injections are repeated over a 5-day period, blood pressure is decreased by 10-20 mmHg, the withdrawal latency to heat stimuli is prolonged, cortisol levels are decreased and PROTEIN1 and cholecystokinin levels are increased\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition, abscisic acid-mediated inhibition of PROTEIN0 responses seems to depend on the activation of a phospholipase D during induction of PROTEIN1 in barley aleurone cells as well as on a putative acetyltransferase involved in elongation growth.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 (PKC) inhibitors reversed PROTEIN1 inhibition, whereas PROTEIN2 activator inhibited nitrite production\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 (PKC) inhibitors reversed PROTEIN1 inhibition, whereas PROTEIN2 activator inhibited nitrite production\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 (PKC) inhibitors reversed PROTEIN1 inhibition, whereas PROTEIN2 activator inhibited nitrite production\n",
      "\n",
      "PROTEIN0 PROTEIN1 The inhibitory action of PROTEIN0 involves both the activation of PROTEIN1 and the transcription of iNOS mRNA in cultured rat GMC\n",
      "\n",
      "PROTEIN0 PROTEIN1 Molecular characterization of catalytic-subunit cDNA sequences encoding protein phosphatases 1 and 2A and study of their roles in the PROTEIN0 PROTEIN1 expression in rice\n",
      "\n",
      "PROTEIN0 PROTEIN1 To understand the molecular mechanism of PROTEIN0 gene regulation, the effect of three phosphatase inhibitors on the germination of rice seeds and the expression of a target gene, the PROTEIN1 gene, Osamy-c, were measured\n",
      "\n",
      "PROTEIN0 PROTEIN1 To further understand the possible role of protein phosphatases 1 and 2A in the PROTEIN0 expression of PROTEIN1 we isolated cDNA clones encoding protein phosphatase 1 and protein phosphatase 2A from a rice aleurone cDNA library\n",
      "\n",
      "PROTEIN0 PROTEIN1 Taken together, our results suggest that protein phosphatases PP1 or PP2A are involved in the PROTEIN0 expression of the rice PROTEIN1 gene, though the PP1 or/and PP2A enzymatic activities as well as mRNA levels do not increase upon GA3 treatment\n",
      "\n",
      "PROTEIN0 PROTEIN1 Fasting and PROTEIN0 modulate adipose and muscle PROTEIN1 divergent effects between messenger ribonucleic acid and protein expression\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this study, we examined the effects of fasting for 2 days and exogenous s.c. PROTEIN0 200 microg every 8 h for 2 days, on the regulation of PROTEIN1 (UCP) subtypes in brown adipose tissue (BAT) and gastrocnemius muscle\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 compared with vehicle, did not alter BAT PROTEIN1 or PROTEIN2 mRNA or protein expression when administered to normal ad libitum fed rats\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 compared with vehicle, did not alter BAT PROTEIN1 or PROTEIN2 mRNA or protein expression when administered to normal ad libitum fed rats\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 compared with vehicle, did not alter BAT PROTEIN1 or PROTEIN2 mRNA or protein expression when administered to normal ad libitum fed rats\n",
      "\n",
      "PROTEIN0 PROTEIN1 Fasting significantly decreased BAT PROTEIN0 and PROTEIN1 mRNA expression, to 31% and 30% of ad libitum fed controls, respectively, effects which were prevented by administration of PROTEIN2 to fasted rats\n",
      "\n",
      "PROTEIN0 PROTEIN2 Fasting significantly decreased BAT PROTEIN0 and PROTEIN1 mRNA expression, to 31% and 30% of ad libitum fed controls, respectively, effects which were prevented by administration of PROTEIN2 to fasted rats\n",
      "\n",
      "PROTEIN1 PROTEIN2 Fasting significantly decreased BAT PROTEIN0 and PROTEIN1 mRNA expression, to 31% and 30% of ad libitum fed controls, respectively, effects which were prevented by administration of PROTEIN2 to fasted rats\n",
      "\n",
      "PROTEIN0 PROTEIN1 Fasting also significantly decreased BAT PROTEIN0 protein expression, to 67% of control; however, that effect was not prevented by PROTEIN1 treatment\n",
      "\n",
      "PROTEIN0 PROTEIN1 Fasting, with or without PROTEIN0 administration, did not affect BAT PROTEIN1 mRNA; however, PROTEIN2 administration to ad libitum fed rats significantly increased BAT PROTEIN3 mRNA, to 138% of control\n",
      "\n",
      "PROTEIN0 PROTEIN2 Fasting, with or without PROTEIN0 administration, did not affect BAT PROTEIN1 mRNA; however, PROTEIN2 administration to ad libitum fed rats significantly increased BAT PROTEIN3 mRNA, to 138% of control\n",
      "\n",
      "PROTEIN0 PROTEIN3 Fasting, with or without PROTEIN0 administration, did not affect BAT PROTEIN1 mRNA; however, PROTEIN2 administration to ad libitum fed rats significantly increased BAT PROTEIN3 mRNA, to 138% of control\n",
      "\n",
      "PROTEIN1 PROTEIN2 Fasting, with or without PROTEIN0 administration, did not affect BAT PROTEIN1 mRNA; however, PROTEIN2 administration to ad libitum fed rats significantly increased BAT PROTEIN3 mRNA, to 138% of control\n",
      "\n",
      "PROTEIN1 PROTEIN3 Fasting, with or without PROTEIN0 administration, did not affect BAT PROTEIN1 mRNA; however, PROTEIN2 administration to ad libitum fed rats significantly increased BAT PROTEIN3 mRNA, to 138% of control\n",
      "\n",
      "PROTEIN2 PROTEIN3 Fasting, with or without PROTEIN0 administration, did not affect BAT PROTEIN1 mRNA; however, PROTEIN2 administration to ad libitum fed rats significantly increased BAT PROTEIN3 mRNA, to 138% of control\n",
      "\n",
      "PROTEIN0 PROTEIN1 Fasting significantly enhanced gastrocnemius muscle PROTEIN0 mRNA (411% of control) and protein expression (168% of control), whereas PROTEIN1 administration to fasted rats did not alter either of these effects\n",
      "\n",
      "PROTEIN0 PROTEIN1 In summary, PROTEIN0 subtype mRNA and protein are regulated in tissue- and subtype-specific fashion by PROTEIN1 and food restriction\n",
      "\n",
      "PROTEIN0 PROTEIN1 Plasma PROTEIN0 and some functionally related peptides (CCK, gastrin, somatostatin and PROTEIN1 were measured by standard radioimmunoassay techniques\n",
      "\n",
      "PROTEIN0 PROTEIN1 Analyses of hormone regulation of expression of mRNA for the aleurone RNase revealed that, like the pattern for PROTEIN0 mRNA levels increased in the presence of PROTEIN1 and its antagonist abscisic acid prevented this effect\n",
      "\n",
      "PROTEIN0 PROTEIN1 Quantitative studies at early times demonstrated that cycloheximide treatment of aleurone layers increased mRNA levels 4-fold, whereas a combination of PROTEIN0 plus cycloheximide treatment was required to increase PROTEIN1 mRNA levels to the same extent\n",
      "\n",
      "PROTEIN0 PROTEIN1 The experiments were concluded by taking blood samples for later analysis of plasma glucose and plasma levels of the following hormones: PROTEIN0 gastrin, CCK, glucagon, somatostatin, PROTEIN1 and corticosterone\n",
      "\n",
      "PROTEIN0 PROTEIN1 In normal pancreatic islets, PROTEIN0 is upregulated by PROTEIN1 and is low in PROTEIN2 islets of ZDF rats\n",
      "\n",
      "PROTEIN0 PROTEIN2 In normal pancreatic islets, PROTEIN0 is upregulated by PROTEIN1 and is low in PROTEIN2 islets of ZDF rats\n",
      "\n",
      "PROTEIN1 PROTEIN2 In normal pancreatic islets, PROTEIN0 is upregulated by PROTEIN1 and is low in PROTEIN2 islets of ZDF rats\n",
      "\n",
      "PROTEIN0 PROTEIN1 To determine whether PROTEIN0 does, in fact, have uncoupling activity and, if so, whether such activity would favorably influence the abnormalities in PROTEIN1 PROTEIN2 islets of diabetic ZDF rats, we transferred the PROTEIN3 gene to the islets of diabetic ZDF rats and lean (+/+) ZDF control rats\n",
      "\n",
      "PROTEIN0 PROTEIN2 To determine whether PROTEIN0 does, in fact, have uncoupling activity and, if so, whether such activity would favorably influence the abnormalities in PROTEIN1 PROTEIN2 islets of diabetic ZDF rats, we transferred the PROTEIN3 gene to the islets of diabetic ZDF rats and lean (+/+) ZDF control rats\n",
      "\n",
      "PROTEIN0 PROTEIN3 To determine whether PROTEIN0 does, in fact, have uncoupling activity and, if so, whether such activity would favorably influence the abnormalities in PROTEIN1 PROTEIN2 islets of diabetic ZDF rats, we transferred the PROTEIN3 gene to the islets of diabetic ZDF rats and lean (+/+) ZDF control rats\n",
      "\n",
      "PROTEIN1 PROTEIN2 To determine whether PROTEIN0 does, in fact, have uncoupling activity and, if so, whether such activity would favorably influence the abnormalities in PROTEIN1 PROTEIN2 islets of diabetic ZDF rats, we transferred the PROTEIN3 gene to the islets of diabetic ZDF rats and lean (+/+) ZDF control rats\n",
      "\n",
      "PROTEIN1 PROTEIN3 To determine whether PROTEIN0 does, in fact, have uncoupling activity and, if so, whether such activity would favorably influence the abnormalities in PROTEIN1 PROTEIN2 islets of diabetic ZDF rats, we transferred the PROTEIN3 gene to the islets of diabetic ZDF rats and lean (+/+) ZDF control rats\n",
      "\n",
      "PROTEIN2 PROTEIN3 To determine whether PROTEIN0 does, in fact, have uncoupling activity and, if so, whether such activity would favorably influence the abnormalities in PROTEIN1 PROTEIN2 islets of diabetic ZDF rats, we transferred the PROTEIN3 gene to the islets of diabetic ZDF rats and lean (+/+) ZDF control rats\n",
      "\n",
      "PROTEIN0 PROTEIN1 We conclude that PROTEIN0 has uncoupling function when overexpressed in PROTEIN1 islets and that its overexpression corrects the underexpression of the insulin gene and ameliorates glucose-stimulated insulin secretion, possibly by increasing the ATP:ADP ratio\n",
      "\n",
      "PROTEIN0 PROTEIN1 The second class of mutant, gse PROTEIN0 sensitivity), differed principally in PROTEIN1 sensitivity, requiring approximately 100-fold higher PROTEIN2 for both leaf elongation and PROTEIN3 production by aleurone\n",
      "\n",
      "PROTEIN0 PROTEIN2 The second class of mutant, gse PROTEIN0 sensitivity), differed principally in PROTEIN1 sensitivity, requiring approximately 100-fold higher PROTEIN2 for both leaf elongation and PROTEIN3 production by aleurone\n",
      "\n",
      "PROTEIN0 PROTEIN3 The second class of mutant, gse PROTEIN0 sensitivity), differed principally in PROTEIN1 sensitivity, requiring approximately 100-fold higher PROTEIN2 for both leaf elongation and PROTEIN3 production by aleurone\n",
      "\n",
      "PROTEIN1 PROTEIN2 The second class of mutant, gse PROTEIN0 sensitivity), differed principally in PROTEIN1 sensitivity, requiring approximately 100-fold higher PROTEIN2 for both leaf elongation and PROTEIN3 production by aleurone\n",
      "\n",
      "PROTEIN1 PROTEIN3 The second class of mutant, gse PROTEIN0 sensitivity), differed principally in PROTEIN1 sensitivity, requiring approximately 100-fold higher PROTEIN2 for both leaf elongation and PROTEIN3 production by aleurone\n",
      "\n",
      "PROTEIN2 PROTEIN3 The second class of mutant, gse PROTEIN0 sensitivity), differed principally in PROTEIN1 sensitivity, requiring approximately 100-fold higher PROTEIN2 for both leaf elongation and PROTEIN3 production by aleurone\n",
      "\n",
      "PROTEIN0 PROTEIN1 Phorbol myristate acetate (TPA), an activator of PROTEIN0 elicited an increase of MAPK activity, but did not further influence the level of PROTEIN1 MAPK activity; Nevertheless, the extent of CaMKII activation was attenuated by TPA\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 elements conserved in cereal PROTEIN1 genes are not included in the 5'-upstream region of Rep1 or RepA\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, our 14C-acetate-labeling studies showed a 50% reduction in PROTEIN0 synthesis in the presence of either PROTEIN1 which could account for the reduction in net apoB secretion caused by incubation with these compounds\n",
      "\n",
      "PROTEIN0 PROTEIN1 An inhibitor of PROTEIN0 Ro31-8220 (10 microM), abolished the ability of PDBu to decrease [Ca(2+)](i), without affecting the response to maximal or submaximal concentrations of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Effects of intravenously infused PROTEIN0 on insulin sensitivity and on the expression of PROTEIN1 in brown adipose tissue\n",
      "\n",
      "PROTEIN0 PROTEIN1 Centrally administered PROTEIN0 has been shown to increase insulin-stimulated glucose utilization and to favor the expression of PROTEIN1 (UCPs)\n",
      "\n",
      "PROTEIN0 PROTEIN1 I.v. PROTEIN0 infusion also favored the expression of PROTEIN1 in brown adipose tissue, either by increasing their expression or preventing the fall occurring during the pair-feeding regimen.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Relative PROTEIN0 expression levels were 100, 104, and 33 for PROTEIN1 100, 191, and 125 for PROTEIN2 and 100, 107, and 29 for PROTEIN3 in ad libitum fed control rats, in PROTEIN4 rats and in pair-fed control rats, respectively\n",
      "\n",
      "PROTEIN0 PROTEIN2 Relative PROTEIN0 expression levels were 100, 104, and 33 for PROTEIN1 100, 191, and 125 for PROTEIN2 and 100, 107, and 29 for PROTEIN3 in ad libitum fed control rats, in PROTEIN4 rats and in pair-fed control rats, respectively\n",
      "\n",
      "PROTEIN0 PROTEIN3 Relative PROTEIN0 expression levels were 100, 104, and 33 for PROTEIN1 100, 191, and 125 for PROTEIN2 and 100, 107, and 29 for PROTEIN3 in ad libitum fed control rats, in PROTEIN4 rats and in pair-fed control rats, respectively\n",
      "\n",
      "PROTEIN0 PROTEIN4 Relative PROTEIN0 expression levels were 100, 104, and 33 for PROTEIN1 100, 191, and 125 for PROTEIN2 and 100, 107, and 29 for PROTEIN3 in ad libitum fed control rats, in PROTEIN4 rats and in pair-fed control rats, respectively\n",
      "\n",
      "PROTEIN1 PROTEIN2 Relative PROTEIN0 expression levels were 100, 104, and 33 for PROTEIN1 100, 191, and 125 for PROTEIN2 and 100, 107, and 29 for PROTEIN3 in ad libitum fed control rats, in PROTEIN4 rats and in pair-fed control rats, respectively\n",
      "\n",
      "PROTEIN1 PROTEIN3 Relative PROTEIN0 expression levels were 100, 104, and 33 for PROTEIN1 100, 191, and 125 for PROTEIN2 and 100, 107, and 29 for PROTEIN3 in ad libitum fed control rats, in PROTEIN4 rats and in pair-fed control rats, respectively\n",
      "\n",
      "PROTEIN1 PROTEIN4 Relative PROTEIN0 expression levels were 100, 104, and 33 for PROTEIN1 100, 191, and 125 for PROTEIN2 and 100, 107, and 29 for PROTEIN3 in ad libitum fed control rats, in PROTEIN4 rats and in pair-fed control rats, respectively\n",
      "\n",
      "PROTEIN2 PROTEIN3 Relative PROTEIN0 expression levels were 100, 104, and 33 for PROTEIN1 100, 191, and 125 for PROTEIN2 and 100, 107, and 29 for PROTEIN3 in ad libitum fed control rats, in PROTEIN4 rats and in pair-fed control rats, respectively\n",
      "\n",
      "PROTEIN2 PROTEIN4 Relative PROTEIN0 expression levels were 100, 104, and 33 for PROTEIN1 100, 191, and 125 for PROTEIN2 and 100, 107, and 29 for PROTEIN3 in ad libitum fed control rats, in PROTEIN4 rats and in pair-fed control rats, respectively\n",
      "\n",
      "PROTEIN3 PROTEIN4 Relative PROTEIN0 expression levels were 100, 104, and 33 for PROTEIN1 100, 191, and 125 for PROTEIN2 and 100, 107, and 29 for PROTEIN3 in ad libitum fed control rats, in PROTEIN4 rats and in pair-fed control rats, respectively\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that the overall effect of PROTEIN0 on glucose utilization and on the expression of PROTEIN1 may be mediated through central mechanism\n",
      "\n",
      "PROTEIN0 PROTEIN1 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN0 PROTEIN2 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN0 PROTEIN3 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN0 PROTEIN4 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN0 PROTEIN5 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN1 PROTEIN2 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN1 PROTEIN3 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN1 PROTEIN4 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN1 PROTEIN5 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN2 PROTEIN3 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN2 PROTEIN4 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN2 PROTEIN5 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN3 PROTEIN4 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN3 PROTEIN5 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN4 PROTEIN5 The expression of uterine PROTEIN0 and PROTEIN1 were up-regulated by 3.2- and 1.5-fold, respectively, during the late stage of pregnancy with an increase of WAT PROTEIN2 mRNA expression and exogenous administration of PROTEIN3 resulted in induction of the uterine PROTEIN4 and PROTEIN5 levels\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results indicate that PROTEIN0 gene expressions during pregnancy are regulated tissue-dependently, and up-regulation of uterine PROTEIN1 and PROTEIN2 mRNA may be due to increased PROTEIN3 levels\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results indicate that PROTEIN0 gene expressions during pregnancy are regulated tissue-dependently, and up-regulation of uterine PROTEIN1 and PROTEIN2 mRNA may be due to increased PROTEIN3 levels\n",
      "\n",
      "PROTEIN0 PROTEIN3 These results indicate that PROTEIN0 gene expressions during pregnancy are regulated tissue-dependently, and up-regulation of uterine PROTEIN1 and PROTEIN2 mRNA may be due to increased PROTEIN3 levels\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results indicate that PROTEIN0 gene expressions during pregnancy are regulated tissue-dependently, and up-regulation of uterine PROTEIN1 and PROTEIN2 mRNA may be due to increased PROTEIN3 levels\n",
      "\n",
      "PROTEIN1 PROTEIN3 These results indicate that PROTEIN0 gene expressions during pregnancy are regulated tissue-dependently, and up-regulation of uterine PROTEIN1 and PROTEIN2 mRNA may be due to increased PROTEIN3 levels\n",
      "\n",
      "PROTEIN2 PROTEIN3 These results indicate that PROTEIN0 gene expressions during pregnancy are regulated tissue-dependently, and up-regulation of uterine PROTEIN1 and PROTEIN2 mRNA may be due to increased PROTEIN3 levels\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 treatment of fetal guinea pig brain cultures substantially reduced the amount of PROTEIN1 and PROTEIN2 in the medium but had no effect on sAPPalpha\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 treatment of fetal guinea pig brain cultures substantially reduced the amount of PROTEIN1 and PROTEIN2 in the medium but had no effect on sAPPalpha\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 treatment of fetal guinea pig brain cultures substantially reduced the amount of PROTEIN1 and PROTEIN2 in the medium but had no effect on sAPPalpha\n",
      "\n",
      "PROTEIN0 PROTEIN1 When this parental line was treated with PROTEIN0 PROTEIN1 PROTEIN2 and sAPPbeta decreased to levels similar to those observed in the mutant line, and the mutant line was resistant to these effects of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 When this parental line was treated with PROTEIN0 PROTEIN1 PROTEIN2 and sAPPbeta decreased to levels similar to those observed in the mutant line, and the mutant line was resistant to these effects of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 When this parental line was treated with PROTEIN0 PROTEIN1 PROTEIN2 and sAPPbeta decreased to levels similar to those observed in the mutant line, and the mutant line was resistant to these effects of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 When this parental line was treated with PROTEIN0 PROTEIN1 PROTEIN2 and sAPPbeta decreased to levels similar to those observed in the mutant line, and the mutant line was resistant to these effects of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 When this parental line was treated with PROTEIN0 PROTEIN1 PROTEIN2 and sAPPbeta decreased to levels similar to those observed in the mutant line, and the mutant line was resistant to these effects of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 When this parental line was treated with PROTEIN0 PROTEIN1 PROTEIN2 and sAPPbeta decreased to levels similar to those observed in the mutant line, and the mutant line was resistant to these effects of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 A neurotoxic fragment of PROTEIN0 Abeta 25-35, incubated in the presence of endogenous Ca2+, increased significantly the PROTEIN1 activity of normoxic brain\n",
      "\n",
      "PROTEIN0 PROTEIN1 In its non-aggregated form, PROTEIN0 activates PROTEIN1 but in the aggregated form the enzymatic activity decreased\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, PROTEIN0 exerts a similar effect on the membrane-bound PROTEIN1 from normoxic brain or subjected to ischemia reperfusion injury\n",
      "\n",
      "PROTEIN0 PROTEIN1 Ischemia-reperfusion injury had no effect on PROTEIN0 alterations of synaptic plasma membrane-bound PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Sympathetic inhibition, PROTEIN0 and PROTEIN1 subtype expression in normal fasting rats\n",
      "\n",
      "PROTEIN0 PROTEIN1 To further investigate neural effects on PROTEIN0 and PROTEIN1 (UCPs), we studied in vivo perturbations intended to block adrenergic input to peripheral tissues\n",
      "\n",
      "PROTEIN0 PROTEIN1 We examined plasma PROTEIN0 PROTEIN1 mRNA, and adipose and muscle PROTEIN2 subtype mRNA in rats treated with alpha-methyl-p-tyrosine methyl ester (AMPT-ME), which inhibits catecholamine synthesis and 6-hydroxydopamine (6HDA), which is toxic to catecholinergic nerve terminals but, unlike AMPT-ME, does not enter the central nervous system\n",
      "\n",
      "PROTEIN0 PROTEIN2 We examined plasma PROTEIN0 PROTEIN1 mRNA, and adipose and muscle PROTEIN2 subtype mRNA in rats treated with alpha-methyl-p-tyrosine methyl ester (AMPT-ME), which inhibits catecholamine synthesis and 6-hydroxydopamine (6HDA), which is toxic to catecholinergic nerve terminals but, unlike AMPT-ME, does not enter the central nervous system\n",
      "\n",
      "PROTEIN1 PROTEIN2 We examined plasma PROTEIN0 PROTEIN1 mRNA, and adipose and muscle PROTEIN2 subtype mRNA in rats treated with alpha-methyl-p-tyrosine methyl ester (AMPT-ME), which inhibits catecholamine synthesis and 6-hydroxydopamine (6HDA), which is toxic to catecholinergic nerve terminals but, unlike AMPT-ME, does not enter the central nervous system\n",
      "\n",
      "PROTEIN0 PROTEIN1 Intraperitoneal 6HDA, 100 mg/kg, as a single dose at 1800, increased plasma PROTEIN0 approximately twofold after 18-20 h, increased IBAT (but not epididymal fat) PROTEIN1 mRNA by two- to threefold, and decreased IBAT PROTEIN2 mRNA to 30-40% of control\n",
      "\n",
      "PROTEIN0 PROTEIN2 Intraperitoneal 6HDA, 100 mg/kg, as a single dose at 1800, increased plasma PROTEIN0 approximately twofold after 18-20 h, increased IBAT (but not epididymal fat) PROTEIN1 mRNA by two- to threefold, and decreased IBAT PROTEIN2 mRNA to 30-40% of control\n",
      "\n",
      "PROTEIN1 PROTEIN2 Intraperitoneal 6HDA, 100 mg/kg, as a single dose at 1800, increased plasma PROTEIN0 approximately twofold after 18-20 h, increased IBAT (but not epididymal fat) PROTEIN1 mRNA by two- to threefold, and decreased IBAT PROTEIN2 mRNA to 30-40% of control\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, the in vivo modulation of PROTEIN0 and PROTEIN1 appears complex and, beyond a causal effect of SNA per se, may depend on concurrent changes in plasma insulin, glucose, and circulatory hemodynamics\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 beta peptide 25-35 modulates hydrolysis of phosphoinositides by membrane PROTEIN1 of adult brain cortex\n",
      "\n",
      "PROTEIN0 PROTEIN1 Fresh-water-soluble PROTEIN0 25-35 activated PROTEIN1 in SPM markedly by two- to threefold, but this effect was absent in the presence of 2 mM CaCl2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Moreover, PROTEIN0 25-35 had no effect on basal PROTEIN1 activity and cytosolic PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Moreover, PROTEIN0 25-35 had no effect on basal PROTEIN1 activity and cytosolic PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Moreover, PROTEIN0 25-35 had no effect on basal PROTEIN1 activity and cytosolic PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Moreover, PROTEIN0 25-35 had no effect on basal PROTEIN1 activity and cytosolic PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Moreover, PROTEIN0 25-35 had no effect on basal PROTEIN1 activity and cytosolic PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Moreover, PROTEIN0 25-35 had no effect on basal PROTEIN1 activity and cytosolic PROTEIN2 and PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The aggregated form of PROTEIN0 25-35 significantly inhibited PROTEIN1 only in the presence of endogenous CaCl2\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 stimulates tyrosine phosphorylation by activation of PROTEIN1 in the rat smooth muscle cell line, A-10\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 (PKC) inhibitors: staurosporine, H7 and GF109203X are able to block the PROTEIN1 phosphorylation\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results indicate that PROTEIN0 is upstream of the phosphorylation, a motion which is supported by the fact that the PROTEIN1 phosphorylation was downregulated by phorbol esters\n",
      "\n",
      "PROTEIN0 PROTEIN1 The effects of the fluorinated analogues were evaluated on inhibition of cress seed germination and inhibition of PROTEIN0 PROTEIN1 induction in embryoless barley half-seeds\n",
      "\n",
      "PROTEIN0 PROTEIN1 (2E, 4E)-2-Fluoro-5-(1'-hydroxy-2',6', 6'-trimethyl-2'-cyclohexen-1'-yl)-3-methyl-2,4-pentadienoic acid (5b) showed potent inhibitory activity at the same level as ABA in the cress seed germination test, and 5b also inhibited PROTEIN0 PROTEIN1 induction at 4 x 10(-)(6), 3 times the concentration of ABA (1 x 10(-)(6)) for 50% inhibition of PROTEIN2 production\n",
      "\n",
      "PROTEIN0 PROTEIN2 (2E, 4E)-2-Fluoro-5-(1'-hydroxy-2',6', 6'-trimethyl-2'-cyclohexen-1'-yl)-3-methyl-2,4-pentadienoic acid (5b) showed potent inhibitory activity at the same level as ABA in the cress seed germination test, and 5b also inhibited PROTEIN0 PROTEIN1 induction at 4 x 10(-)(6), 3 times the concentration of ABA (1 x 10(-)(6)) for 50% inhibition of PROTEIN2 production\n",
      "\n",
      "PROTEIN1 PROTEIN2 (2E, 4E)-2-Fluoro-5-(1'-hydroxy-2',6', 6'-trimethyl-2'-cyclohexen-1'-yl)-3-methyl-2,4-pentadienoic acid (5b) showed potent inhibitory activity at the same level as ABA in the cress seed germination test, and 5b also inhibited PROTEIN0 PROTEIN1 induction at 4 x 10(-)(6), 3 times the concentration of ABA (1 x 10(-)(6)) for 50% inhibition of PROTEIN2 production\n",
      "\n",
      "PROTEIN0 PROTEIN1 Neurotransmitters and hormones, such as arginine vasopressin (AVP) and bombesin, evoke frequency-modulated repetitive Ca2+ transients in PROTEIN0 HIT-T15 cells by binding to receptors linked to PROTEIN1 (PLC)\n",
      "\n",
      "PROTEIN0 PROTEIN1 Isolation and characterization of gibbestatin B, an inhibitor of PROTEIN0 expression of PROTEIN1 and gibbestatin C from streptomycetaes\n",
      "\n",
      "PROTEIN0 PROTEIN1 We previously showed that PROTEIN0 (AVP) stimulates heat shock protein 27 (HSP27) induction through PROTEIN1 activation in aortic smooth muscle A10 cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 Calphostin C and ET-18-OCH(3), inhibitors of PROTEIN0 reduced the phosphorylation of p38 MAP kinase by PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 signaling in yeast: modulation of protein phosphatase 2A, protein kinase A, cAMP-specific phosphodiesterase, and glycosyl-phosphatidylinositol-specific PROTEIN1 activities\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 was most pronouncedly stimulated by authentic human PROTEIN1 compared to various insulin analogues and insulin-like growth factor I\n",
      "\n",
      "PROTEIN0 PROTEIN1 As the PROTEIN0 reaction is rate limiting, the efficiency of the two-step anchor cleavage was significantly increased when PROTEIN1 was present together with glucose as compared to glucose alone\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 concentrations effective in modulating PP2A, PKA, cAMP-PDE, and PROTEIN1 activities correlate well with those required for half-saturation of the specific binding sites as well as for stimulation of protein phosphorylation and glycogen accumulation\n",
      "\n",
      "PROTEIN0 PROTEIN1 The data suggest that mammalian PROTEIN0 cells and yeast share (part of) the key regulatory mechanism (consisting of PP2A, PKA, cAMP-PDE, and PROTEIN1 involved in the transduction of the PROTEIN2 signal from the respective receptor systems to glycogen synthase and phosphorylase\n",
      "\n",
      "PROTEIN0 PROTEIN2 The data suggest that mammalian PROTEIN0 cells and yeast share (part of) the key regulatory mechanism (consisting of PP2A, PKA, cAMP-PDE, and PROTEIN1 involved in the transduction of the PROTEIN2 signal from the respective receptor systems to glycogen synthase and phosphorylase\n",
      "\n",
      "PROTEIN1 PROTEIN2 The data suggest that mammalian PROTEIN0 cells and yeast share (part of) the key regulatory mechanism (consisting of PP2A, PKA, cAMP-PDE, and PROTEIN1 involved in the transduction of the PROTEIN2 signal from the respective receptor systems to glycogen synthase and phosphorylase\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 and PROTEIN1 (1 and 2) mRNA levels were determined by reverse transcription-polymerase chain reaction (RT-PCR) methodology in adipose tissue and gastrocnemius muscle\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 (AVP) and lysophosphatidic acid (LPA) have been shown to stimulate PROTEIN1 (PKC) and mitogen-activated protein (MAP) kinases and the proliferation of vascular smooth muscle cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, PROTEIN0 activation of RSK2, a downstream substrate of ERK1 and ERK2, was PROTEIN1 and PI 3-kinase-independent\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 was necessary in PROTEIN1 but not in LPA-induced activation of p70 S6K\n",
      "\n",
      "PROTEIN0 PROTEIN1 Tolbutamide and diazoxide modulate PROTEIN0 Ca(2+) signaling and PROTEIN1 secretion in beta-cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 We examined whether tolbutamide and diazoxide, which close or open ATP-sensitive K(+) channels (K(ATP) channels), respectively, interact with PROTEIN0 Ca(2+) signals in HIT-T15 and mouse beta-cells and with PROTEIN1 PROTEIN2 secretion from HIT-T15 cells\n",
      "\n",
      "PROTEIN0 PROTEIN2 We examined whether tolbutamide and diazoxide, which close or open ATP-sensitive K(+) channels (K(ATP) channels), respectively, interact with PROTEIN0 Ca(2+) signals in HIT-T15 and mouse beta-cells and with PROTEIN1 PROTEIN2 secretion from HIT-T15 cells\n",
      "\n",
      "PROTEIN1 PROTEIN2 We examined whether tolbutamide and diazoxide, which close or open ATP-sensitive K(+) channels (K(ATP) channels), respectively, interact with PROTEIN0 Ca(2+) signals in HIT-T15 and mouse beta-cells and with PROTEIN1 PROTEIN2 secretion from HIT-T15 cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus tolbutamide and diazoxide regulate both PROTEIN0 Ca(2+) signaling and PROTEIN1 secretion from pancreatic beta-cells by modulating K(ATP) channels, thereby determining voltage-sensitive Ca(2+) influx\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 central injections to intact animals stimulated PROTEIN1 synthesis and secretion without changing of islets area\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, PROTEIN0 administration stimulates PROTEIN1 synthesis and secretion in intact and diabetic animals and this effect is more expressed with central injections\n",
      "\n",
      "PROTEIN0 PROTEIN1 Hypothalamic PROTEIN0 and its neuroendocrine regulation by PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Ca2+ signalling in rat vascular smooth muscle cells: a role for PROTEIN0 at physiological vasoconstrictor concentrations of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 (PKC) activator 4beta-phorbol 12-myristate 13-acetate (PMA, 100 pM to 200 nM), also stimulated Ca2+ spiking and this effect was additive with a submaximal concentration of PROTEIN1 (50 pM)\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 Ro-31-8220 (1 microM) and calphostin C (250 nM) completely blocked the stimulation of Ca2+ spiking by either PMA or PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Time-dependent redistribution of PROTEIN0 -delta and -epsilon isoforms between the membrane and cytosolic fractions occurred in response to 100 pM PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Pretreatment for 24 h with 1 microM PMA downregulated expression of PROTEIN0 and -delta, but not PROTEIN1 and prevented the Ca2+-spiking responses to either 1 nM PMA or 100 pM PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Pretreatment for 24 h with 1 microM PMA downregulated expression of PROTEIN0 and -delta, but not PROTEIN1 and prevented the Ca2+-spiking responses to either 1 nM PMA or 100 pM PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Pretreatment for 24 h with 1 microM PMA downregulated expression of PROTEIN0 and -delta, but not PROTEIN1 and prevented the Ca2+-spiking responses to either 1 nM PMA or 100 pM PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 We conclude that PROTEIN0 activation is a necessary step in the signal transduction pathway linking low concentrations of PROTEIN1 to Ca2+ spiking in A7r5 cells\n",
      "\n",
      "PROTEIN0 PROTEIN1 Ten PROTEIN0 have been assayed for their ability to phosphorylate in vitro the PROTEIN1 (25-242) (rbPrP)\n",
      "\n",
      "PROTEIN0 PROTEIN1 With regard to PROTEIN0 phosphorylation occurs at Ser 154 with a stoichiometry of about 0.1 mol phosphate/mol PROTEIN1 which is doubled by mild heat treatment of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 With regard to PROTEIN0 phosphorylation occurs at Ser 154 with a stoichiometry of about 0.1 mol phosphate/mol PROTEIN1 which is doubled by mild heat treatment of PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 With regard to PROTEIN0 phosphorylation occurs at Ser 154 with a stoichiometry of about 0.1 mol phosphate/mol PROTEIN1 which is doubled by mild heat treatment of PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 In vivo effects of CGP-12177 on the expression of PROTEIN0 and PROTEIN1 genes in mouse brown and white adipose tissues\n",
      "\n",
      "PROTEIN0 PROTEIN1 OBJECTIVE: To assess the effect of chronic treatment with CGP-12177 a beta3-adrenergic receptor (AR) agonist with beta2/beta1-AR antagonist action, on the expression of the PROTEIN0 gene and of genes coding for PROTEIN1 (ucp1, ucp2 and ucp3) in brown and white adipose tissues\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, in SN-56/OBR cells both CNTF and PROTEIN0 produce changes in neurotransmitter and neuropeptide phenotype characteristic of cholinergic neurons, such as an increase in choline acetyltransferase and vasoactive intestinal polypeptide, and a decrease in PROTEIN1 expression\n",
      "\n",
      "PROTEIN0 PROTEIN1 Possible central nervous system (CNS) mechanisms could mediate these responses in that leptin receptors are located on hypothalamic neurons that coexpress PROTEIN0 (NPY) or proopiomelanocortin (POMC) and both peptides that have been implicated as mediators of the CNS action of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 has been demonstrated to decrease or down regulate PROTEIN1 expression and increase POMC expression\n",
      "\n",
      "PROTEIN0 PROTEIN1 Relationship among PROTEIN0 PROTEIN1 and galanin in young women and in postmenopausal women [see comments]\n",
      "\n",
      "PROTEIN0 PROTEIN1 OBJECTIVE: To determine whether hormonal status may affect PROTEIN0 (NPY), galanin, and PROTEIN1 release in postmenopausal women and in young women\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interestingly, both 5-HT and PROTEIN0 modulate the action of PROTEIN1 which may form a part of a common output pathway for the expression of appetite\n",
      "\n",
      "PROTEIN0 PROTEIN1 The glycosyl-phosphatidylinositol anchored protein, membrane dipeptidase (EC 3.4.13.19) is released from the surface of 3T3-L1 adipocytes in response to PROTEIN0 treatment through the action of a PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 Although X-Pro aminopeptidase (EC 3.4.11.9) is GPI-anchored in 3T3-L1 adipocytes as shown by digestion with bacterial phosphatidylinositol-specific PROTEIN0 it was not released upon PROTEIN1 treatment of the cells, indicating that only a subset of the GPI-anchored proteins are susceptible to PROTEIN2 release\n",
      "\n",
      "PROTEIN0 PROTEIN2 Although X-Pro aminopeptidase (EC 3.4.11.9) is GPI-anchored in 3T3-L1 adipocytes as shown by digestion with bacterial phosphatidylinositol-specific PROTEIN0 it was not released upon PROTEIN1 treatment of the cells, indicating that only a subset of the GPI-anchored proteins are susceptible to PROTEIN2 release\n",
      "\n",
      "PROTEIN1 PROTEIN2 Although X-Pro aminopeptidase (EC 3.4.11.9) is GPI-anchored in 3T3-L1 adipocytes as shown by digestion with bacterial phosphatidylinositol-specific PROTEIN0 it was not released upon PROTEIN1 treatment of the cells, indicating that only a subset of the GPI-anchored proteins are susceptible to PROTEIN2 release\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, neither progesterone nor PROTEIN0 increased phosphotyrosine on PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is a satiety factor which acts within the hypothalamus to decrease the levels of several neuropeptides stimulating food intake (among them, PROTEIN1 [NPY]), while increasing those that inhibit food intake\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have studied the effects of neomycin, a potent inhibitor of inositol phospholipid-specific phospholipase C (PLC), on the germination of rice seed and the PROTEIN0 expression of PROTEIN1 in the aleurone layer and the scutellar tissues\n",
      "\n",
      "PROTEIN0 PROTEIN1 It was shown that, in the absence of exogenous Ca2+, neomycin markedly reduced the germination speed and seedling growth of rice seeds and inhibited the PROTEIN0 expression of PROTEIN1 in both secretory tissues\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results strongly suggest that the phosphoinositide-Ca2+ signal transduction pathway may play an important role in the PROTEIN0 expression of PROTEIN1 molecules closely related to the germination processes of rice seed\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 PROTEIN1 opposing effects on appetitive and consummatory ingestive behavior and sexual behavior\n",
      "\n",
      "PROTEIN0 PROTEIN1 Many studies have indicated that PROTEIN0 (NPY) stimulates and PROTEIN1 inhibits food intake\n",
      "\n",
      "PROTEIN0 PROTEIN1 In line with this, intracerebroventricular injection of PROTEIN0 (10 microg) stimulated and PROTEIN1 (10 microg) inhibited intake of a sucrose solution when female rats were required to obtain the solution from a bottle\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, PROTEIN0 inhibited and PROTEIN1 stimulated intake if the solution was infused intraorally\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus PROTEIN0 stimulates the responses used to obtain food but inhibits those used to consume food, and PROTEIN1 has the opposite effects\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 expression remained inversely related with plasma PROTEIN1 after controlling for percent body (r = -0.45; P = 0.038)\n",
      "\n",
      "PROTEIN0 PROTEIN1 In conclusion, 1) PROTEIN0 messenger ribonucleic acid expression in sc adipose tissue is inversely related to adiposity and independently linked to local plasma PROTEIN1 levels; and 2) PROTEIN2 expression is not acutely regulated by food intake, insulin, or fatty acids\n",
      "\n",
      "PROTEIN0 PROTEIN2 In conclusion, 1) PROTEIN0 messenger ribonucleic acid expression in sc adipose tissue is inversely related to adiposity and independently linked to local plasma PROTEIN1 levels; and 2) PROTEIN2 expression is not acutely regulated by food intake, insulin, or fatty acids\n",
      "\n",
      "PROTEIN1 PROTEIN2 In conclusion, 1) PROTEIN0 messenger ribonucleic acid expression in sc adipose tissue is inversely related to adiposity and independently linked to local plasma PROTEIN1 levels; and 2) PROTEIN2 expression is not acutely regulated by food intake, insulin, or fatty acids\n",
      "\n",
      "PROTEIN0 PROTEIN1 Thus, highly expressed in human adipocytes, the Y(1) receptor sustains the strong antilipolytic effect of PROTEIN0 and exerts a positive action on PROTEIN1 secretion\n",
      "\n",
      "PROTEIN0 PROTEIN1 These findings suggest that the regulation of hypothalamic PROTEIN0 mRNA by PROTEIN1 may be impaired with age\n",
      "\n",
      "PROTEIN0 PROTEIN1 We examined daily food consumption, body weight, whole-body oxygen consumption, serum PROTEIN0 and PROTEIN1 mRNA in the hypothalamus\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 infusion diminished hypothalamic PROTEIN1 levels by nearly 50% compared with pair-fed young rats, whereas there were no changes in the hypothalamic PROTEIN2 mRNA levels in senescent rats\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 infusion diminished hypothalamic PROTEIN1 levels by nearly 50% compared with pair-fed young rats, whereas there were no changes in the hypothalamic PROTEIN2 mRNA levels in senescent rats\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 infusion diminished hypothalamic PROTEIN1 levels by nearly 50% compared with pair-fed young rats, whereas there were no changes in the hypothalamic PROTEIN2 mRNA levels in senescent rats\n",
      "\n",
      "PROTEIN0 PROTEIN1 These diminished responses to PROTEIN0 were associated with and may be the result of an impaired suppression of hypothalamic PROTEIN1 mRNA levels\n",
      "\n",
      "PROTEIN0 PROTEIN1 C75 inhibited expression of the prophagic signal PROTEIN0 in the hypothalamus and acted in a PROTEIN1 manner that appears to be mediated by malonyl-coenzyme A\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interactions between PROTEIN0 and PROTEIN1 affecting lipid mobilization in adipose tissue\n",
      "\n",
      "PROTEIN0 PROTEIN1 The aim of the current investigations was focused on the study of the involvement of PROTEIN0 and PROTEIN1 in lipid mobilization\n",
      "\n",
      "PROTEIN0 PROTEIN1 The lipolytic rate was assessed through glycerol release after incubation with PROTEIN0 and PROTEIN1 at concentrations ranging from 10(-6) to 10(-12) M in isolated adipocytes obtained from female rats\n",
      "\n",
      "PROTEIN0 PROTEIN1 At equimolar concentrations of PROTEIN0 and PROTEIN1 (10(-12) to 10(-6) M) the observed lipolytic activity is comparable to the basal lipolytic activity, except at a concentration of 10(-9) M where upon a significant increase in lipolysis is observed\n",
      "\n",
      "PROTEIN0 PROTEIN1 Summing up, new evidence suggests that PROTEIN0 and PROTEIN1 may interact in a homeostatic loop to regulate body-fat mass and energy balance not only at the central nervous system level, but also directly at the adipocyte level\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 specific inhibitor Go-6983 further enhanced the inhibitory effect of VACM-1 on PROTEIN1 cAMP production\n",
      "\n",
      "PROTEIN0 PROTEIN1 Involvement of thyroid hormones in the effect of intracerebroventricular PROTEIN0 infusion on PROTEIN1 expression in rat muscle\n",
      "\n",
      "PROTEIN0 PROTEIN1 A reduction of food intake imposed on control rats (pair-feeding), aimed at mimicking PROTEIN0 hyperphagia, produced a marked decrease in the expression of muscle PROTEIN1 (UCP-3), whereas ICV infusion of PROTEIN2 prevented such a decrease in PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 A reduction of food intake imposed on control rats (pair-feeding), aimed at mimicking PROTEIN0 hyperphagia, produced a marked decrease in the expression of muscle PROTEIN1 (UCP-3), whereas ICV infusion of PROTEIN2 prevented such a decrease in PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 A reduction of food intake imposed on control rats (pair-feeding), aimed at mimicking PROTEIN0 hyperphagia, produced a marked decrease in the expression of muscle PROTEIN1 (UCP-3), whereas ICV infusion of PROTEIN2 prevented such a decrease in PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 A reduction of food intake imposed on control rats (pair-feeding), aimed at mimicking PROTEIN0 hyperphagia, produced a marked decrease in the expression of muscle PROTEIN1 (UCP-3), whereas ICV infusion of PROTEIN2 prevented such a decrease in PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 A reduction of food intake imposed on control rats (pair-feeding), aimed at mimicking PROTEIN0 hyperphagia, produced a marked decrease in the expression of muscle PROTEIN1 (UCP-3), whereas ICV infusion of PROTEIN2 prevented such a decrease in PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 A reduction of food intake imposed on control rats (pair-feeding), aimed at mimicking PROTEIN0 hyperphagia, produced a marked decrease in the expression of muscle PROTEIN1 (UCP-3), whereas ICV infusion of PROTEIN2 prevented such a decrease in PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 To further substantiate an involvement of thyroid hormones in the effect of PROTEIN0 on muscle PROTEIN1 expression, hypothyroid rats were ICV infused with PROTEIN2 or vehicle\n",
      "\n",
      "PROTEIN0 PROTEIN2 To further substantiate an involvement of thyroid hormones in the effect of PROTEIN0 on muscle PROTEIN1 expression, hypothyroid rats were ICV infused with PROTEIN2 or vehicle\n",
      "\n",
      "PROTEIN1 PROTEIN2 To further substantiate an involvement of thyroid hormones in the effect of PROTEIN0 on muscle PROTEIN1 expression, hypothyroid rats were ICV infused with PROTEIN2 or vehicle\n",
      "\n",
      "PROTEIN0 PROTEIN1 It was observed that in hypothyroid rats, ICV PROTEIN0 was unable to maintain muscle PROTEIN1 expression at values measured in ad libitum fed controls\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that central PROTEIN0 stimulates T3 production via an activation of T4 to T3 conversion, and that this stimulation could be responsible for the effect of PROTEIN1 on muscle PROTEIN2 expression\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results suggest that central PROTEIN0 stimulates T3 production via an activation of T4 to T3 conversion, and that this stimulation could be responsible for the effect of PROTEIN1 on muscle PROTEIN2 expression\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results suggest that central PROTEIN0 stimulates T3 production via an activation of T4 to T3 conversion, and that this stimulation could be responsible for the effect of PROTEIN1 on muscle PROTEIN2 expression\n",
      "\n",
      "PROTEIN0 PROTEIN1 The body mass index (BMI), PROTEIN0 plasma PROTEIN1 (NPY), serotonin, thyroxine (T4), triiodothyronine (T3) and reverse triiodothyronine (rT3) in serum were evaluated for each subject\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 (PLC) activity was investigated by stimulation of membrane preparations obtained from PROTEIN1 (beta-TC3)-, somatostatin (Rin 1027-B2)-, and glucagon (INR1-G9)-producing pancreatic cell lines using the non-hydrolyzable GTP analogue GTPgammaS alone, the C-terminal octapeptide cholecystokinin (CCK-8), or gastrin\n",
      "\n",
      "PROTEIN0 PROTEIN1 Therefore, it was examined whether stimulation of Swiss 3T3 cells with PROTEIN0 causes changes in the subcellular distribution of EGF receptors and PROTEIN1 that could potentially explain the observed synergy or costimulation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, both EGF and PROTEIN0 stimulated the accumulation of PROTEIN1 at the actin arc, which was coincident with the EGF receptor in the case of PROTEIN2 stimulated cells.\n",
      "\n",
      "PROTEIN0 PROTEIN2 However, both EGF and PROTEIN0 stimulated the accumulation of PROTEIN1 at the actin arc, which was coincident with the EGF receptor in the case of PROTEIN2 stimulated cells.\n",
      "\n",
      "PROTEIN1 PROTEIN2 However, both EGF and PROTEIN0 stimulated the accumulation of PROTEIN1 at the actin arc, which was coincident with the EGF receptor in the case of PROTEIN2 stimulated cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Therefore, it is suggested that the PROTEIN0 coclustering of the EGF receptor with PROTEIN1 at the actin arc may allow for greater efficiency of signal transduction, resulting in the synergy observed for these two hormones in stimulation of DNA synthesis.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have established that the recovery of secreted PROTEIN0 is increased by the PROTEIN1 agonists PDBu and PMA in a time- and dose-dependent manner in both cell lines\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, secretion of PROTEIN0 remains unaffected by the inactive PDBu analog alphaPDD and by the PROTEIN1 effectors dibutyryl cAMP and forskolin\n",
      "\n",
      "PROTEIN0 PROTEIN1 Overall, our data indicate that the normal processing of PROTEIN0 is up-regulated by PROTEIN1 but not PROTEIN2 in human cells and murine neurons\n",
      "\n",
      "PROTEIN0 PROTEIN2 Overall, our data indicate that the normal processing of PROTEIN0 is up-regulated by PROTEIN1 but not PROTEIN2 in human cells and murine neurons\n",
      "\n",
      "PROTEIN1 PROTEIN2 Overall, our data indicate that the normal processing of PROTEIN0 is up-regulated by PROTEIN1 but not PROTEIN2 in human cells and murine neurons\n",
      "\n",
      "PROTEIN0 PROTEIN1 Dietary intakes of PROTEIN0 PROTEIN1 and PROTEIN2 by Japanese women and the inverse correlation between quercetin intake and plasma LDL PROTEIN3 concentration\n",
      "\n",
      "PROTEIN0 PROTEIN2 Dietary intakes of PROTEIN0 PROTEIN1 and PROTEIN2 by Japanese women and the inverse correlation between quercetin intake and plasma LDL PROTEIN3 concentration\n",
      "\n",
      "PROTEIN0 PROTEIN3 Dietary intakes of PROTEIN0 PROTEIN1 and PROTEIN2 by Japanese women and the inverse correlation between quercetin intake and plasma LDL PROTEIN3 concentration\n",
      "\n",
      "PROTEIN1 PROTEIN2 Dietary intakes of PROTEIN0 PROTEIN1 and PROTEIN2 by Japanese women and the inverse correlation between quercetin intake and plasma LDL PROTEIN3 concentration\n",
      "\n",
      "PROTEIN1 PROTEIN3 Dietary intakes of PROTEIN0 PROTEIN1 and PROTEIN2 by Japanese women and the inverse correlation between quercetin intake and plasma LDL PROTEIN3 concentration\n",
      "\n",
      "PROTEIN2 PROTEIN3 Dietary intakes of PROTEIN0 PROTEIN1 and PROTEIN2 by Japanese women and the inverse correlation between quercetin intake and plasma LDL PROTEIN3 concentration\n",
      "\n",
      "PROTEIN0 PROTEIN1 The total intake of PROTEIN0 was inversely correlated with the plasma total PROTEIN1 concentration (TC) (r = -0.236, P: < 0.05) and plasma LDL PROTEIN2 concentration (LDL-C) (r = -0.220, P: < 0.05), after the adjustment for age, body mass index and total energy intake\n",
      "\n",
      "PROTEIN0 PROTEIN2 The total intake of PROTEIN0 was inversely correlated with the plasma total PROTEIN1 concentration (TC) (r = -0.236, P: < 0.05) and plasma LDL PROTEIN2 concentration (LDL-C) (r = -0.220, P: < 0.05), after the adjustment for age, body mass index and total energy intake\n",
      "\n",
      "PROTEIN1 PROTEIN2 The total intake of PROTEIN0 was inversely correlated with the plasma total PROTEIN1 concentration (TC) (r = -0.236, P: < 0.05) and plasma LDL PROTEIN2 concentration (LDL-C) (r = -0.220, P: < 0.05), after the adjustment for age, body mass index and total energy intake\n",
      "\n",
      "PROTEIN0 PROTEIN1 Although the above data are consistent with a direct ability of PROTEIN0 to promote TRH biosynthesis through actions on TRH neurons, addition of a-MSH produced a 3.5 fold increase in TRH biosynthesis and release, whereas PROTEIN1 treatment suppressed ~3 fold proTRH biosynthesis\n",
      "\n",
      "PROTEIN0 PROTEIN1 For example, PROTEIN0 suppresses PROTEIN1 (NPY) expression in the arcuate nucleus\n",
      "\n",
      "PROTEIN0 PROTEIN1 As expected, in fusion of PROTEIN0 alone (18 microg/day) augmented food intake (191.6% over the initial control, P < 0.05) and produced a 25.1% weight gain in conjunction with a 10-fold increase in serum PROTEIN1 concentrations at the end of the 7-day period\n",
      "\n",
      "PROTEIN0 PROTEIN1 Interestingly, coinfusion of this regimen of PROTEIN0 with the highly effective anorectic and body reducing effects of CNTF (5.0 microg/day) not only prevented the CNTF-induced anorexia and weight loss, but also normalized serum PROTEIN1 concentrations and hypothalamic PROTEIN2 gene expression\n",
      "\n",
      "PROTEIN0 PROTEIN2 Interestingly, coinfusion of this regimen of PROTEIN0 with the highly effective anorectic and body reducing effects of CNTF (5.0 microg/day) not only prevented the CNTF-induced anorexia and weight loss, but also normalized serum PROTEIN1 concentrations and hypothalamic PROTEIN2 gene expression\n",
      "\n",
      "PROTEIN1 PROTEIN2 Interestingly, coinfusion of this regimen of PROTEIN0 with the highly effective anorectic and body reducing effects of CNTF (5.0 microg/day) not only prevented the CNTF-induced anorexia and weight loss, but also normalized serum PROTEIN1 concentrations and hypothalamic PROTEIN2 gene expression\n",
      "\n",
      "PROTEIN0 PROTEIN1 A caveolin-1-dependent coupling of PROTEIN0 to the PROTEIN1 was observed\n",
      "\n",
      "PROTEIN0 PROTEIN1 Recently several significant advances have been made in this area, including the identification of the appetite regulating hormone, PROTEIN0 and a detailed understanding of its targets in the central nervous system (CNS), such as PROTEIN1 (NPY) and the melanocortin-4 receptor\n",
      "\n",
      "PROTEIN0 PROTEIN1 A loop system exists between hypothalamic PROTEIN0 (NPY) and peripheral adipose tissue PROTEIN1 to maintain normal body homeostasis\n",
      "\n",
      "PROTEIN0 PROTEIN1 When infused i.c.v. to normal rats to mimic its central effects, PROTEIN0 decreases PROTEIN1 levels, thus food intake and body weight\n",
      "\n",
      "PROTEIN0 PROTEIN1 Circadian fluctuations of PROTEIN0 and PROTEIN1 levels maintain normal body homeostasis\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 increases the expression of PROTEIN1 (UCP-1, -2, -3) and thus energy dissipation\n",
      "\n",
      "PROTEIN0 PROTEIN1 Activity and expression of the three PROTEIN0 are stimulated by several neuromediators and hormones such as noradrenaline, tri-iodothyronine and PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The effect of PROTEIN0 in adult rats appears to be exerted at hypothalamic level by regulating growth hormone releasing hormone (GHRH), somatostatin and PROTEIN1 (NPY)-producing neurones\n",
      "\n",
      "PROTEIN0 PROTEIN1 Plasma insulin, cholecystokinin, galanin, PROTEIN0 and PROTEIN1 levels in obese women with and without type 2 diabetes mellitus\n",
      "\n",
      "PROTEIN0 PROTEIN1 Experimental studies provided evidences that PROTEIN0 cholecystokinin (CCK), galanin (GAL), PROTEIN1 (NPY) and insulin are involved in feeding behaviour\n",
      "\n",
      "PROTEIN0 PROTEIN1 Plasma concentrations of FSH, PROTEIN0 CCK, GAL, PROTEIN1 and insulin were determined using commercial RIA kits\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our results, ie higher mean plasma PROTEIN0 levels in postmenopausal diabetic women and positive correlation of CCK with PROTEIN1 and insulin, may suggest the role of these neuropeptides in metabolic disorders leading to type 2 diabetes mellitus\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that T3 may increase PROTEIN0 family expression independent of PROTEIN1 action\n",
      "\n",
      "PROTEIN0 PROTEIN1 The ARC PROTEIN0 neurones are stimulated by starvation, probably mediated by falls in circulating PROTEIN1 and insulin (which both inhibit these neurones), and contribute to the increased hunger in this and other conditions of energy deficit\n",
      "\n",
      "PROTEIN0 PROTEIN1 ARC PROTEIN0 neurones also mediate hyperphagia and obesity in the ob/ob and db/db mice and fa/fa rat, in which PROTEIN1 inhibition is lost through mutations affecting PROTEIN2 or its receptor\n",
      "\n",
      "PROTEIN0 PROTEIN2 ARC PROTEIN0 neurones also mediate hyperphagia and obesity in the ob/ob and db/db mice and fa/fa rat, in which PROTEIN1 inhibition is lost through mutations affecting PROTEIN2 or its receptor\n",
      "\n",
      "PROTEIN1 PROTEIN2 ARC PROTEIN0 neurones also mediate hyperphagia and obesity in the ob/ob and db/db mice and fa/fa rat, in which PROTEIN1 inhibition is lost through mutations affecting PROTEIN2 or its receptor\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 secretion, inositol phosphate levels, and PROTEIN1 isozymes in rodent pancreatic islets\n",
      "\n",
      "PROTEIN0 PROTEIN1 These findings are consistent with the concept that the underexpression of the nutrient-activated PROTEIN0 isozyme may account for the minimal inositol phosphate (IP) and second-phase PROTEIN1 secretory response from mouse islets\n",
      "\n",
      "PROTEIN0 PROTEIN1 Gibbestatin B inhibits the PROTEIN0 expression of PROTEIN1 expression in cereal seeds [In Process Citation]\n",
      "\n",
      "PROTEIN0 PROTEIN1 The expression of PROTEIN0 in aleurone layers of barley is known to be induced by PROTEIN1 (GA).\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the present study, gibbestatin B (GNB) was isolated from Streptomyces sp. C-39 as an inhibitor of the PROTEIN0 expression of PROTEIN1 in barley and rice, with IC50 values of 125 and 70 microM, respectively.\n",
      "\n",
      "PROTEIN0 PROTEIN1 GNB suppressed accumulation of PROTEIN0 barley high-pI type B and rice RAmylA PROTEIN1 transcripts.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These analyses indicate that GNB inhibits the PROTEIN0 expression of PROTEIN1 by regulating one of the steps involved in ABA signaling, but not by acting as a weak ABA analog.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We also determined whether well known hypothalamic neuropeptide targets, e.g. PROTEIN0 (NPY), proopiomelanocortin (POMC), agouti-related peptide (AGRP) and cocaine and amphetamine-regulated transcript (CART) were regulated in a pattern consistent with their presumed roles as mediators of PROTEIN1 action\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, PROTEIN0 decreased body weight and adiposity, increased CART and suppressed PROTEIN1 and AGRP mRNA expression in adult mice\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 effects on pulsatile gonadotropin releasing hormone secretion from the adult rat hypothalamus and interaction with cocaine and amphetamine regulated transcript peptide and PROTEIN1 [In Process Citation]\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 may act as a negative feedback signal to the hypothalamic control of appetite through suppression of PROTEIN1 (NPY) secretion and stimulation of cocaine and amphetamine regulated transcript (CART).\n",
      "\n",
      "PROTEIN0 PROTEIN1 We aimed at studying the effects of PROTEIN0 CART and PROTEIN1 on the hypothalamic control of the pituitary-gonadal system.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The increase in GnRH pulse amplitude caused by PROTEIN0 was totally prevented by coincubation with an anti-CART antiserum whereas it was not affected by coincubation with the PROTEIN1 Y5-receptor antagonist (10(-7) M).\n",
      "\n",
      "PROTEIN0 PROTEIN1 In conclusion, PROTEIN0 and PROTEIN1 show separate permissive effects on GnRH secretion in the adult rat hypothalamus.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Similarly, central PROTEIN0 treatment also had a greater effect in suppressing hypothalamic PROTEIN1 mRNA expression in young (-23+/-4%) than in old (-8+/-4%) rats compared with their age-matched pair-fed treated rats\n",
      "\n",
      "PROTEIN0 PROTEIN1 Compared with the wild type, PROTEIN0 induction of PROTEIN1 activity in aleurone cells of d1 was greatly reduced\n",
      "\n",
      "PROTEIN0 PROTEIN1 Relative to the wild type, the PROTEIN0 aleurone layer of d1 had lower expression of Ramy1A, which encodes PROTEIN1 and OsGAMYB, which encodes a PROTEIN2 transcriptional factor, and no increase in expression of Ca(2 +)-ATPase\n",
      "\n",
      "PROTEIN0 PROTEIN2 Relative to the wild type, the PROTEIN0 aleurone layer of d1 had lower expression of Ramy1A, which encodes PROTEIN1 and OsGAMYB, which encodes a PROTEIN2 transcriptional factor, and no increase in expression of Ca(2 +)-ATPase\n",
      "\n",
      "PROTEIN1 PROTEIN2 Relative to the wild type, the PROTEIN0 aleurone layer of d1 had lower expression of Ramy1A, which encodes PROTEIN1 and OsGAMYB, which encodes a PROTEIN2 transcriptional factor, and no increase in expression of Ca(2 +)-ATPase\n",
      "\n",
      "PROTEIN0 PROTEIN1 However, in the presence of high PROTEIN0 concentrations, PROTEIN1 induction occurred even in d1\n",
      "\n",
      "PROTEIN0 PROTEIN1 All these results suggest that d1 affects a part of the PROTEIN0 pathway, namely the induction of PROTEIN1 in the aleurone layer and internode elongation\n",
      "\n",
      "PROTEIN0 PROTEIN1 Finally, recent evidences suggest that melanocortins could be involved in mediating the effects of PROTEIN0 and in controlling the expression of PROTEIN1 (NPY)\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 phospholipase D1 and phospholipase D2 activity in human embryonic kidney-293 cells mediated by the PROTEIN1 and protein kinase Calpha signalling cascade.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Co-expression of PLD1 with PROTEIN0 has the same effect, while co-expression of PLD2 with PROTEIN1 allows PLD2 activity to be stimulated in an PROTEIN2 manner.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Co-expression of PLD1 with PROTEIN0 has the same effect, while co-expression of PLD2 with PROTEIN1 allows PLD2 activity to be stimulated in an PROTEIN2 manner.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Co-expression of PLD1 with PROTEIN0 has the same effect, while co-expression of PLD2 with PROTEIN1 allows PLD2 activity to be stimulated in an PROTEIN2 manner.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PKC-specific inhibitors bisindolylmaleimide and Go 6976 abolish PROTEIN0 PLD2 activation in HEK-293 cells co-expressing the insulin receptor, PROTEIN1 and PLD2, confirming that not only PLD1, but PLD2 as well, is regulated in a PKC-dependent manner.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In summary, we demonstrate that PROTEIN0 treatment results in activation of both PLD1 and PLD2 in appropriate cell types when the appropriate upstream intermediate signalling components, i.e. PKCalpha and PROTEIN1 are expressed at sufficient levels.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Both receive information about the nutritional status and the level of energy storage through insulin and PROTEIN0 signaling mediated by specific receptors located on POMC and PROTEIN1 neurons present predominantly in the arcuate nucleus (ARC).\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 as a modulator of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 On the basis of far-Western blot and plasmon resonance (BIAcore) experiments, we show here that recombinant PROTEIN0 (bPrP) (25-242) strongly interacts with the catalytic alpha/alpha' subunits of PROTEIN1 (also termed 'casein kinase 2')\n",
      "\n",
      "PROTEIN0 PROTEIN1 We also show that PROTEIN0 counteracts the inhibition of calmodulin phosphorylation promoted by the regulatory beta subunits of PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 A truncated form of PROTEIN0 encompassing the C-terminal domain (residues 105-242) interacts with PROTEIN1 but does not affect its catalytic activity\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results disclose the potential of the PROTEIN0 to modulate the activity of PROTEIN1 a pleiotropic PROTEIN2 that is particularly abundant in the brain\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results disclose the potential of the PROTEIN0 to modulate the activity of PROTEIN1 a pleiotropic PROTEIN2 that is particularly abundant in the brain\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results disclose the potential of the PROTEIN0 to modulate the activity of PROTEIN1 a pleiotropic PROTEIN2 that is particularly abundant in the brain\n",
      "\n",
      "PROTEIN0 PROTEIN1 Arcuate nucleus PROTEIN0 mRNA increased in rats given protein-free diet, correlating with PROTEIN1 rather than decreased feeding\n",
      "\n",
      "PROTEIN0 PROTEIN1 The induction of PROTEIN0 triggered by PROTEIN1 in barley embryos is repressed by sugars\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our results indicate that glucose represses PROTEIN0 signalling late along this hormone transduction pathway, downstream of transcription of the PROTEIN1 transcriptional activator (GAMYB) needed for PROTEIN2 induction\n",
      "\n",
      "PROTEIN0 PROTEIN2 Our results indicate that glucose represses PROTEIN0 signalling late along this hormone transduction pathway, downstream of transcription of the PROTEIN1 transcriptional activator (GAMYB) needed for PROTEIN2 induction\n",
      "\n",
      "PROTEIN1 PROTEIN2 Our results indicate that glucose represses PROTEIN0 signalling late along this hormone transduction pathway, downstream of transcription of the PROTEIN1 transcriptional activator (GAMYB) needed for PROTEIN2 induction\n",
      "\n",
      "PROTEIN0 PROTEIN1 In this mutant, expression of the PROTEIN0 gene, whose transcription depends on both PROTEIN1 and the phosphorylated PROTEIN2 protein, Spo0A~P, a major transcription factor during early stages of sporulation, was greatly reduced at 43 degrees C.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In this mutant, expression of the PROTEIN0 gene, whose transcription depends on both PROTEIN1 and the phosphorylated PROTEIN2 protein, Spo0A~P, a major transcription factor during early stages of sporulation, was greatly reduced at 43 degrees C.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In this mutant, expression of the PROTEIN0 gene, whose transcription depends on both PROTEIN1 and the phosphorylated PROTEIN2 protein, Spo0A~P, a major transcription factor during early stages of sporulation, was greatly reduced at 43 degrees C.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Transcription of PROTEIN0 was dependent on PROTEIN1 and the mRNA was detectable from 2 h after the cessation of logarithmic growth (T2 of sporulation).\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 was transcribed by PROTEIN1 RNA polymerase from T4 of sporulation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Both PROTEIN0 and PROTEIN1 were essential for PROTEIN2 expression, and this gene was transcribed from T5 of sporulation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Both PROTEIN0 and PROTEIN1 were essential for PROTEIN2 expression, and this gene was transcribed from T5 of sporulation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Both PROTEIN0 and PROTEIN1 were essential for PROTEIN2 expression, and this gene was transcribed from T5 of sporulation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Analysis of the expression of a translational ywhE-lacZ fusion showed that PROTEIN0 expression is sporulation-specific, and is controlled predominantly by the forespore-specific sigma factor PROTEIN1 and to a lesser extent by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Analysis of the expression of a translational ywhE-lacZ fusion showed that PROTEIN0 expression is sporulation-specific, and is controlled predominantly by the forespore-specific sigma factor PROTEIN1 and to a lesser extent by PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Analysis of the expression of a translational ywhE-lacZ fusion showed that PROTEIN0 expression is sporulation-specific, and is controlled predominantly by the forespore-specific sigma factor PROTEIN1 and to a lesser extent by PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Secondly, PROTEIN0 is essential for the activation of the first compartment-specific transcription factor PROTEIN1 in the prespore.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Localization of PROTEIN0 was shown to be dependent on the essential cell division protein PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In the shaA mutant, PROTEIN0 expression of PROTEIN1 and PROTEIN2 at an early stage of sporulation was sensitive to external NaCl.\n",
      "\n",
      "PROTEIN0 PROTEIN2 In the shaA mutant, PROTEIN0 expression of PROTEIN1 and PROTEIN2 at an early stage of sporulation was sensitive to external NaCl.\n",
      "\n",
      "PROTEIN1 PROTEIN2 In the shaA mutant, PROTEIN0 expression of PROTEIN1 and PROTEIN2 at an early stage of sporulation was sensitive to external NaCl.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The expression of PROTEIN0 is under the control of both PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 The expression of PROTEIN0 is under the control of both PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 The expression of PROTEIN0 is under the control of both PROTEIN1 and PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Expression of the PROTEIN0 PROTEIN1 gene depended on PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 Expression of the PROTEIN0 PROTEIN1 gene depended on PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 Expression of the PROTEIN0 PROTEIN1 gene depended on PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 Our results indicate that the level of PROTEIN0 level in outgrowing spores depends mainly on PROTEIN1 because sigB mutants show normal KatX activity in dormant and outgrowing spores.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Therefore, the physiological role of PROTEIN0 PROTEIN1 expression remains obscure.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The -10 and -35 sequences of the PROTEIN0 and PROTEIN1 promoters of PROTEIN2 overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The -10 and -35 sequences of the PROTEIN0 and PROTEIN1 promoters of PROTEIN2 overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The -10 and -35 sequences of the PROTEIN0 and PROTEIN1 promoters of PROTEIN2 overlap, and the transcriptional start points used by EsigmaB and EsigmaF differ by only one nucleotide.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We demonstrated that PROTEIN0 is also a PROTEIN1 general stress gene, since it is strongly induced by heat, salt and ethanol stress, as well as by energy depletion.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Taken together, the results support the idea that sequestration of PROTEIN0 protein into the prespore plays an important role in the control of PROTEIN1 activation and in coupling this activation to septation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 is controlled by a regulatory cascade involving an anti-sigma factor, PROTEIN1 an anti-anti-sigma factor, PROTEIN2 and a membrane-bound phosphatase, PROTEIN3 which converts the inactive, phosphorylated form of PROTEIN4 back to the active form.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 is controlled by a regulatory cascade involving an anti-sigma factor, PROTEIN1 an anti-anti-sigma factor, PROTEIN2 and a membrane-bound phosphatase, PROTEIN3 which converts the inactive, phosphorylated form of PROTEIN4 back to the active form.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 is controlled by a regulatory cascade involving an anti-sigma factor, PROTEIN1 an anti-anti-sigma factor, PROTEIN2 and a membrane-bound phosphatase, PROTEIN3 which converts the inactive, phosphorylated form of PROTEIN4 back to the active form.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 is controlled by a regulatory cascade involving an anti-sigma factor, PROTEIN1 an anti-anti-sigma factor, PROTEIN2 and a membrane-bound phosphatase, PROTEIN3 which converts the inactive, phosphorylated form of PROTEIN4 back to the active form.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 is controlled by a regulatory cascade involving an anti-sigma factor, PROTEIN1 an anti-anti-sigma factor, PROTEIN2 and a membrane-bound phosphatase, PROTEIN3 which converts the inactive, phosphorylated form of PROTEIN4 back to the active form.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 is controlled by a regulatory cascade involving an anti-sigma factor, PROTEIN1 an anti-anti-sigma factor, PROTEIN2 and a membrane-bound phosphatase, PROTEIN3 which converts the inactive, phosphorylated form of PROTEIN4 back to the active form.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 is controlled by a regulatory cascade involving an anti-sigma factor, PROTEIN1 an anti-anti-sigma factor, PROTEIN2 and a membrane-bound phosphatase, PROTEIN3 which converts the inactive, phosphorylated form of PROTEIN4 back to the active form.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 is controlled by a regulatory cascade involving an anti-sigma factor, PROTEIN1 an anti-anti-sigma factor, PROTEIN2 and a membrane-bound phosphatase, PROTEIN3 which converts the inactive, phosphorylated form of PROTEIN4 back to the active form.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 is controlled by a regulatory cascade involving an anti-sigma factor, PROTEIN1 an anti-anti-sigma factor, PROTEIN2 and a membrane-bound phosphatase, PROTEIN3 which converts the inactive, phosphorylated form of PROTEIN4 back to the active form.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 is controlled by a regulatory cascade involving an anti-sigma factor, PROTEIN1 an anti-anti-sigma factor, PROTEIN2 and a membrane-bound phosphatase, PROTEIN3 which converts the inactive, phosphorylated form of PROTEIN4 back to the active form.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 gene of Bacillus subtilis, encoding a catabolic glutamate dehydrogenase, is transcribed by PROTEIN1 RNA polymerase and requires for its expression PROTEIN2 a member of the NtrC/NifA family of proteins that bind to enhancer-like elements, called upstream activating sequences (UAS).\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 gene of Bacillus subtilis, encoding a catabolic glutamate dehydrogenase, is transcribed by PROTEIN1 RNA polymerase and requires for its expression PROTEIN2 a member of the NtrC/NifA family of proteins that bind to enhancer-like elements, called upstream activating sequences (UAS).\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 gene of Bacillus subtilis, encoding a catabolic glutamate dehydrogenase, is transcribed by PROTEIN1 RNA polymerase and requires for its expression PROTEIN2 a member of the NtrC/NifA family of proteins that bind to enhancer-like elements, called upstream activating sequences (UAS).\n",
      "\n",
      "PROTEIN0 PROTEIN1 Unlike the case for other PROTEIN0 genes, PROTEIN1 has no UAS; instead, its expression depends on a sequence located 1.5 kilobases downstream of the rocG promoter, beyond the end of the rocG coding region.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Northern blot and primer extension analyses indicated that PROTEIN0 is transcribed by PROTEIN1 during sporulation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that PROTEIN0 may act as a negative regulator for the transcription of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results suggest that PROTEIN0 may act as a negative regulator for the transcription of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 These results suggest that PROTEIN0 may act as a negative regulator for the transcription of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 These results suggest that PROTEIN0 may act as a negative regulator for the transcription of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results suggest that PROTEIN0 may act as a negative regulator for the transcription of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 These results suggest that PROTEIN0 may act as a negative regulator for the transcription of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 These results suggest that PROTEIN0 may act as a negative regulator for the transcription of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 These results suggest that PROTEIN0 may act as a negative regulator for the transcription of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 These results suggest that PROTEIN0 may act as a negative regulator for the transcription of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 These results suggest that PROTEIN0 may act as a negative regulator for the transcription of PROTEIN1 PROTEIN2 PROTEIN3 and PROTEIN4\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PROTEIN0 PROTEIN1 The first stage, which involves the phosphatase function of PROTEIN0 depends on the cell division protein PROTEIN1 and could correspond to the PROTEIN2 assembly of PROTEIN3 into E-rings.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The first stage, which involves the phosphatase function of PROTEIN0 depends on the cell division protein PROTEIN1 and could correspond to the PROTEIN2 assembly of PROTEIN3 into E-rings.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The first stage, which involves the phosphatase function of PROTEIN0 depends on the cell division protein PROTEIN1 and could correspond to the PROTEIN2 assembly of PROTEIN3 into E-rings.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The first stage, which involves the phosphatase function of PROTEIN0 depends on the cell division protein PROTEIN1 and could correspond to the PROTEIN2 assembly of PROTEIN3 into E-rings.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The first stage, which involves the phosphatase function of PROTEIN0 depends on the cell division protein PROTEIN1 and could correspond to the PROTEIN2 assembly of PROTEIN3 into E-rings.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The first stage, which involves the phosphatase function of PROTEIN0 depends on the cell division protein PROTEIN1 and could correspond to the PROTEIN2 assembly of PROTEIN3 into E-rings.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Evidence based on the use of modified and mutant forms of the phosphatase protein indicates that PROTEIN0 blocks the capacity of unphosphorylated PROTEIN1 to activate PROTEIN2 until formation of the polar septum is completed.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Evidence based on the use of modified and mutant forms of the phosphatase protein indicates that PROTEIN0 blocks the capacity of unphosphorylated PROTEIN1 to activate PROTEIN2 until formation of the polar septum is completed.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Evidence based on the use of modified and mutant forms of the phosphatase protein indicates that PROTEIN0 blocks the capacity of unphosphorylated PROTEIN1 to activate PROTEIN2 until formation of the polar septum is completed.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cell-specific activation of transcription factor PROTEIN0 during sporulation in Bacillus subtilis requires the formation of the polar septum and the activity of a serine phosphatase PROTEIN1 located in the septum.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 phosphatase indirectly activates PROTEIN1 by dephosphorylating a protein PROTEIN2 in the pathway that controls the activity of the transcription factor.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 phosphatase indirectly activates PROTEIN1 by dephosphorylating a protein PROTEIN2 in the pathway that controls the activity of the transcription factor.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 phosphatase indirectly activates PROTEIN1 by dephosphorylating a protein PROTEIN2 in the pathway that controls the activity of the transcription factor.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Dephosphorylation of PROTEIN0 by PROTEIN1 is strictly dependent on the presence of the bivalent metal ions Mn2+ or Mg2+.\n",
      "\n",
      "PROTEIN0 PROTEIN1 (i) The finding that the PROTEIN0 operon is under PROTEIN1 control indicates that the elimination of multidrug compounds might be a new function in multiple stress resistance.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 gene might belong to the increasing number of PROTEIN1 genes whose orthologues are under the control of sigmas in Escherichia coli, indicating that both regulons may fulfil similar functions.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Three new PROTEIN0 genes PROTEIN1 PROTEIN2 and PROTEIN3 encoding proteins with still unknown functions were also described.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Three new PROTEIN0 genes PROTEIN1 PROTEIN2 and PROTEIN3 encoding proteins with still unknown functions were also described.\n",
      "\n",
      "PROTEIN0 PROTEIN3 Three new PROTEIN0 genes PROTEIN1 PROTEIN2 and PROTEIN3 encoding proteins with still unknown functions were also described.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Three new PROTEIN0 genes PROTEIN1 PROTEIN2 and PROTEIN3 encoding proteins with still unknown functions were also described.\n",
      "\n",
      "PROTEIN1 PROTEIN3 Three new PROTEIN0 genes PROTEIN1 PROTEIN2 and PROTEIN3 encoding proteins with still unknown functions were also described.\n",
      "\n",
      "PROTEIN2 PROTEIN3 Three new PROTEIN0 genes PROTEIN1 PROTEIN2 and PROTEIN3 encoding proteins with still unknown functions were also described.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Cell specificity is first established in the forespore under the control of the PROTEIN0 factor, which is itself activated through the action of the PROTEIN1 serine phosphatase, an enzyme targeted to the septum between the two cells.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Most PROTEIN0 genes, and the PROTEIN1 gene, are transcribed by PROTEIN2 RNA polymerase.\n",
      "\n",
      "PROTEIN0 PROTEIN2 Most PROTEIN0 genes, and the PROTEIN1 gene, are transcribed by PROTEIN2 RNA polymerase.\n",
      "\n",
      "PROTEIN1 PROTEIN2 Most PROTEIN0 genes, and the PROTEIN1 gene, are transcribed by PROTEIN2 RNA polymerase.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Previously, it was shown that the PROTEIN0 protein inhibits transcription in vitro of the PROTEIN1 gene encoding sigmaK.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results demonstrate that PROTEIN0 transcription of PROTEIN1 initiates a negative feedback loop in which PROTEIN2 acts as a repressor to limit production of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 These results demonstrate that PROTEIN0 transcription of PROTEIN1 initiates a negative feedback loop in which PROTEIN2 acts as a repressor to limit production of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 These results demonstrate that PROTEIN0 transcription of PROTEIN1 initiates a negative feedback loop in which PROTEIN2 acts as a repressor to limit production of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 These results demonstrate that PROTEIN0 transcription of PROTEIN1 initiates a negative feedback loop in which PROTEIN2 acts as a repressor to limit production of PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 These results demonstrate that PROTEIN0 transcription of PROTEIN1 initiates a negative feedback loop in which PROTEIN2 acts as a repressor to limit production of PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 These results demonstrate that PROTEIN0 transcription of PROTEIN1 initiates a negative feedback loop in which PROTEIN2 acts as a repressor to limit production of PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 A low level of PROTEIN0 activated transcription of PROTEIN1 by PROTEIN2 RNA polymerase in vitro, but a higher level of PROTEIN3 repressed PROTEIN4 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN2 A low level of PROTEIN0 activated transcription of PROTEIN1 by PROTEIN2 RNA polymerase in vitro, but a higher level of PROTEIN3 repressed PROTEIN4 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN3 A low level of PROTEIN0 activated transcription of PROTEIN1 by PROTEIN2 RNA polymerase in vitro, but a higher level of PROTEIN3 repressed PROTEIN4 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN4 A low level of PROTEIN0 activated transcription of PROTEIN1 by PROTEIN2 RNA polymerase in vitro, but a higher level of PROTEIN3 repressed PROTEIN4 transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN2 A low level of PROTEIN0 activated transcription of PROTEIN1 by PROTEIN2 RNA polymerase in vitro, but a higher level of PROTEIN3 repressed PROTEIN4 transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN3 A low level of PROTEIN0 activated transcription of PROTEIN1 by PROTEIN2 RNA polymerase in vitro, but a higher level of PROTEIN3 repressed PROTEIN4 transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN4 A low level of PROTEIN0 activated transcription of PROTEIN1 by PROTEIN2 RNA polymerase in vitro, but a higher level of PROTEIN3 repressed PROTEIN4 transcription.\n",
      "\n",
      "PROTEIN2 PROTEIN3 A low level of PROTEIN0 activated transcription of PROTEIN1 by PROTEIN2 RNA polymerase in vitro, but a higher level of PROTEIN3 repressed PROTEIN4 transcription.\n",
      "\n",
      "PROTEIN2 PROTEIN4 A low level of PROTEIN0 activated transcription of PROTEIN1 by PROTEIN2 RNA polymerase in vitro, but a higher level of PROTEIN3 repressed PROTEIN4 transcription.\n",
      "\n",
      "PROTEIN3 PROTEIN4 A low level of PROTEIN0 activated transcription of PROTEIN1 by PROTEIN2 RNA polymerase in vitro, but a higher level of PROTEIN3 repressed PROTEIN4 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Here, we show that PROTEIN0 binds near the PROTEIN1 transcriptional start site, to act as a repressor.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We show that PROTEIN0 binds to two sites that span the -35 region of the PROTEIN1 promoter.\n",
      "\n",
      "PROTEIN0 PROTEIN1 These results suggest that a rising level of PROTEIN0 in sporulating cells may first activate PROTEIN1 transcription from the upstream site then repress transcription as the downstream site becomes occupied.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 transcription also requires the DNA binding protein PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 In contrast, PROTEIN0 is transcribed in the forespore compartment by RNA polymerase with the forespore-specific PROTEIN1 and appears to give a monocistronic transcript.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN4 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN5 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN1 PROTEIN4 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN1 PROTEIN5 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN2 PROTEIN4 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN2 PROTEIN5 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN3 PROTEIN4 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN3 PROTEIN5 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN4 PROTEIN5 PROTEIN0 negatively controls the transcription of PROTEIN1 by stimulating the transcription of PROTEIN2 PROTEIN3 thereby increasing the production of the PROTEIN4 antisigma factor that inhibits PROTEIN5 activity.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This stress-induced increase in PROTEIN0 expression is absolutely dependent on the alternative sigma factor PROTEIN1 which controls a large stationary-phase and stress regulon.\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 also contributes to the expression of PROTEIN1 a putative membrane-bound glucosyl transferase that is partially controlled by the PROTEIN2 stress response sigma factor.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 also contributes to the expression of PROTEIN1 a putative membrane-bound glucosyl transferase that is partially controlled by the PROTEIN2 stress response sigma factor.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 also contributes to the expression of PROTEIN1 a putative membrane-bound glucosyl transferase that is partially controlled by the PROTEIN2 stress response sigma factor.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Transcriptional studies showed that PROTEIN0 is strongly induced in response to heat, ethanol and salt stress or after starvation for glucose in a PROTEIN1 manner.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Furthermore, PROTEIN0 and PROTEIN1 seemed to be induced after phosphate starvation in the wild type in a PROTEIN2 manner and in the sigB mutant probably via PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 Furthermore, PROTEIN0 and PROTEIN1 seemed to be induced after phosphate starvation in the wild type in a PROTEIN2 manner and in the sigB mutant probably via PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 Furthermore, PROTEIN0 and PROTEIN1 seemed to be induced after phosphate starvation in the wild type in a PROTEIN2 manner and in the sigB mutant probably via PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 Furthermore, PROTEIN0 and PROTEIN1 seemed to be induced after phosphate starvation in the wild type in a PROTEIN2 manner and in the sigB mutant probably via PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 Furthermore, PROTEIN0 and PROTEIN1 seemed to be induced after phosphate starvation in the wild type in a PROTEIN2 manner and in the sigB mutant probably via PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 Furthermore, PROTEIN0 and PROTEIN1 seemed to be induced after phosphate starvation in the wild type in a PROTEIN2 manner and in the sigB mutant probably via PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 The expression of the PROTEIN0 operon was induced by the presence of isoleucine or valine in the growth medium and depended upon the presence of the sigma factor PROTEIN1 a member of the sigma 54 family.\n",
      "\n",
      "PROTEIN0 PROTEIN1 From these results we conclude that PROTEIN0 negatively regulates PROTEIN1 expression by preventing PROTEIN2 transcription of PROTEIN3 possibly through interaction with the control region.\n",
      "\n",
      "PROTEIN0 PROTEIN2 From these results we conclude that PROTEIN0 negatively regulates PROTEIN1 expression by preventing PROTEIN2 transcription of PROTEIN3 possibly through interaction with the control region.\n",
      "\n",
      "PROTEIN0 PROTEIN3 From these results we conclude that PROTEIN0 negatively regulates PROTEIN1 expression by preventing PROTEIN2 transcription of PROTEIN3 possibly through interaction with the control region.\n",
      "\n",
      "PROTEIN1 PROTEIN2 From these results we conclude that PROTEIN0 negatively regulates PROTEIN1 expression by preventing PROTEIN2 transcription of PROTEIN3 possibly through interaction with the control region.\n",
      "\n",
      "PROTEIN1 PROTEIN3 From these results we conclude that PROTEIN0 negatively regulates PROTEIN1 expression by preventing PROTEIN2 transcription of PROTEIN3 possibly through interaction with the control region.\n",
      "\n",
      "PROTEIN2 PROTEIN3 From these results we conclude that PROTEIN0 negatively regulates PROTEIN1 expression by preventing PROTEIN2 transcription of PROTEIN3 possibly through interaction with the control region.\n",
      "\n",
      "PROTEIN0 PROTEIN1 We have shown previously that the transcription of PROTEIN0 is driven by an alternative sigma factor, PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 DNase I footprinting showed that PROTEIN0 binds strongly to two sites in the PROTEIN1 promoter region, binds weakly to one site in the PROTEIN2 promoter, and does not bind specifically to PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 DNase I footprinting showed that PROTEIN0 binds strongly to two sites in the PROTEIN1 promoter region, binds weakly to one site in the PROTEIN2 promoter, and does not bind specifically to PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 DNase I footprinting showed that PROTEIN0 binds strongly to two sites in the PROTEIN1 promoter region, binds weakly to one site in the PROTEIN2 promoter, and does not bind specifically to PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 DNase I footprinting showed that PROTEIN0 binds strongly to two sites in the PROTEIN1 promoter region, binds weakly to one site in the PROTEIN2 promoter, and does not bind specifically to PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 DNase I footprinting showed that PROTEIN0 binds strongly to two sites in the PROTEIN1 promoter region, binds weakly to one site in the PROTEIN2 promoter, and does not bind specifically to PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 DNase I footprinting showed that PROTEIN0 binds strongly to two sites in the PROTEIN1 promoter region, binds weakly to one site in the PROTEIN2 promoter, and does not bind specifically to PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Transcription of the PROTEIN0 PROTEIN1 and PROTEIN2 genes by final PROTEIN3 RNA polymerase is activated by a small, DNA-binding protein called PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 Transcription of the PROTEIN0 PROTEIN1 and PROTEIN2 genes by final PROTEIN3 RNA polymerase is activated by a small, DNA-binding protein called PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 Transcription of the PROTEIN0 PROTEIN1 and PROTEIN2 genes by final PROTEIN3 RNA polymerase is activated by a small, DNA-binding protein called PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 Transcription of the PROTEIN0 PROTEIN1 and PROTEIN2 genes by final PROTEIN3 RNA polymerase is activated by a small, DNA-binding protein called PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 Transcription of the PROTEIN0 PROTEIN1 and PROTEIN2 genes by final PROTEIN3 RNA polymerase is activated by a small, DNA-binding protein called PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 Transcription of the PROTEIN0 PROTEIN1 and PROTEIN2 genes by final PROTEIN3 RNA polymerase is activated by a small, DNA-binding protein called PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 Transcription of the PROTEIN0 PROTEIN1 and PROTEIN2 genes by final PROTEIN3 RNA polymerase is activated by a small, DNA-binding protein called PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 Transcription of the PROTEIN0 PROTEIN1 and PROTEIN2 genes by final PROTEIN3 RNA polymerase is activated by a small, DNA-binding protein called PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 Transcription of the PROTEIN0 PROTEIN1 and PROTEIN2 genes by final PROTEIN3 RNA polymerase is activated by a small, DNA-binding protein called PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 Transcription of the PROTEIN0 PROTEIN1 and PROTEIN2 genes by final PROTEIN3 RNA polymerase is activated by a small, DNA-binding protein called PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 In vitro transcription experiments suggest that the differential pattern of PROTEIN0 gene expression results from the combined action of PROTEIN1 and another transcription factor, PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN2 In vitro transcription experiments suggest that the differential pattern of PROTEIN0 gene expression results from the combined action of PROTEIN1 and another transcription factor, PROTEIN2\n",
      "\n",
      "PROTEIN1 PROTEIN2 In vitro transcription experiments suggest that the differential pattern of PROTEIN0 gene expression results from the combined action of PROTEIN1 and another transcription factor, PROTEIN2\n",
      "\n",
      "PROTEIN0 PROTEIN1 A low concentration of PROTEIN0 activated PROTEIN1 transcription by final PROTEIN2 RNA polymerase, whereas a higher concentration was needed to activate transcription of PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN2 A low concentration of PROTEIN0 activated PROTEIN1 transcription by final PROTEIN2 RNA polymerase, whereas a higher concentration was needed to activate transcription of PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN3 A low concentration of PROTEIN0 activated PROTEIN1 transcription by final PROTEIN2 RNA polymerase, whereas a higher concentration was needed to activate transcription of PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN4 A low concentration of PROTEIN0 activated PROTEIN1 transcription by final PROTEIN2 RNA polymerase, whereas a higher concentration was needed to activate transcription of PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN2 A low concentration of PROTEIN0 activated PROTEIN1 transcription by final PROTEIN2 RNA polymerase, whereas a higher concentration was needed to activate transcription of PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN3 A low concentration of PROTEIN0 activated PROTEIN1 transcription by final PROTEIN2 RNA polymerase, whereas a higher concentration was needed to activate transcription of PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN1 PROTEIN4 A low concentration of PROTEIN0 activated PROTEIN1 transcription by final PROTEIN2 RNA polymerase, whereas a higher concentration was needed to activate transcription of PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN3 A low concentration of PROTEIN0 activated PROTEIN1 transcription by final PROTEIN2 RNA polymerase, whereas a higher concentration was needed to activate transcription of PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN2 PROTEIN4 A low concentration of PROTEIN0 activated PROTEIN1 transcription by final PROTEIN2 RNA polymerase, whereas a higher concentration was needed to activate transcription of PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN3 PROTEIN4 A low concentration of PROTEIN0 activated PROTEIN1 transcription by final PROTEIN2 RNA polymerase, whereas a higher concentration was needed to activate transcription of PROTEIN3 or PROTEIN4\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 at low concentration repressed PROTEIN1 transcription, whereas a higher concentration only partially repressed PROTEIN2 transcription and had little effect on PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN2 PROTEIN0 at low concentration repressed PROTEIN1 transcription, whereas a higher concentration only partially repressed PROTEIN2 transcription and had little effect on PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN3 PROTEIN0 at low concentration repressed PROTEIN1 transcription, whereas a higher concentration only partially repressed PROTEIN2 transcription and had little effect on PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN2 PROTEIN0 at low concentration repressed PROTEIN1 transcription, whereas a higher concentration only partially repressed PROTEIN2 transcription and had little effect on PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN1 PROTEIN3 PROTEIN0 at low concentration repressed PROTEIN1 transcription, whereas a higher concentration only partially repressed PROTEIN2 transcription and had little effect on PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN2 PROTEIN3 PROTEIN0 at low concentration repressed PROTEIN1 transcription, whereas a higher concentration only partially repressed PROTEIN2 transcription and had little effect on PROTEIN3 transcription.\n",
      "\n",
      "PROTEIN0 PROTEIN1 A PROTEIN0 promoter (PW) precedes PROTEIN1 demonstrating that this transcription factor is positively autoregulated.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The results indicate that PROTEIN0 is sufficient to repress the transcription of the PROTEIN1 and PROTEIN2 promoters and also to activate the transcription of the PROTEIN3 promoter.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The results indicate that PROTEIN0 is sufficient to repress the transcription of the PROTEIN1 and PROTEIN2 promoters and also to activate the transcription of the PROTEIN3 promoter.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The results indicate that PROTEIN0 is sufficient to repress the transcription of the PROTEIN1 and PROTEIN2 promoters and also to activate the transcription of the PROTEIN3 promoter.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The results indicate that PROTEIN0 is sufficient to repress the transcription of the PROTEIN1 and PROTEIN2 promoters and also to activate the transcription of the PROTEIN3 promoter.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The results indicate that PROTEIN0 is sufficient to repress the transcription of the PROTEIN1 and PROTEIN2 promoters and also to activate the transcription of the PROTEIN3 promoter.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The results indicate that PROTEIN0 is sufficient to repress the transcription of the PROTEIN1 and PROTEIN2 promoters and also to activate the transcription of the PROTEIN3 promoter.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Transcriptional studies demonstrated that the expression from the PROTEIN0 promoter is induced by L-arabinose, repressed by glucose, and negatively regulated by PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 During endospore formation in Bacillus subtilis, the DNA binding protein PROTEIN0 stimulates transcription from several promoters that are used by RNA polymerase containing PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 PROTEIN0 binds to a site on one of these promoters, PROTEIN1 that overlaps its -35 region.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 promoter drives expression of PROTEIN1 under stress conditions and after glucose starvation, whereas a PROTEIN2 promoter is responsible for PROTEIN3 transcription following amino acid limitation.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 promoter drives expression of PROTEIN1 under stress conditions and after glucose starvation, whereas a PROTEIN2 promoter is responsible for PROTEIN3 transcription following amino acid limitation.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 promoter drives expression of PROTEIN1 under stress conditions and after glucose starvation, whereas a PROTEIN2 promoter is responsible for PROTEIN3 transcription following amino acid limitation.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 promoter drives expression of PROTEIN1 under stress conditions and after glucose starvation, whereas a PROTEIN2 promoter is responsible for PROTEIN3 transcription following amino acid limitation.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 promoter drives expression of PROTEIN1 under stress conditions and after glucose starvation, whereas a PROTEIN2 promoter is responsible for PROTEIN3 transcription following amino acid limitation.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 promoter drives expression of PROTEIN1 under stress conditions and after glucose starvation, whereas a PROTEIN2 promoter is responsible for PROTEIN3 transcription following amino acid limitation.\n",
      "\n",
      "PROTEIN0 PROTEIN1 This indicates that the PROTEIN0 gene product, being a member of both the PROTEIN1 and PROTEIN2 regulons, might negatively regulate the activity of the PROTEIN3 regulon.\n",
      "\n",
      "PROTEIN0 PROTEIN2 This indicates that the PROTEIN0 gene product, being a member of both the PROTEIN1 and PROTEIN2 regulons, might negatively regulate the activity of the PROTEIN3 regulon.\n",
      "\n",
      "PROTEIN0 PROTEIN3 This indicates that the PROTEIN0 gene product, being a member of both the PROTEIN1 and PROTEIN2 regulons, might negatively regulate the activity of the PROTEIN3 regulon.\n",
      "\n",
      "PROTEIN1 PROTEIN2 This indicates that the PROTEIN0 gene product, being a member of both the PROTEIN1 and PROTEIN2 regulons, might negatively regulate the activity of the PROTEIN3 regulon.\n",
      "\n",
      "PROTEIN1 PROTEIN3 This indicates that the PROTEIN0 gene product, being a member of both the PROTEIN1 and PROTEIN2 regulons, might negatively regulate the activity of the PROTEIN3 regulon.\n",
      "\n",
      "PROTEIN2 PROTEIN3 This indicates that the PROTEIN0 gene product, being a member of both the PROTEIN1 and PROTEIN2 regulons, might negatively regulate the activity of the PROTEIN3 regulon.\n",
      "\n",
      "PROTEIN0 PROTEIN1 In addition to the typical PROTEIN0 stress- and starvation-inducible pattern, PROTEIN1 is also induced in response to amino acid depletion.\n",
      "\n",
      "PROTEIN0 PROTEIN1 The addition of PROTEIN0 to in vitro transcription reactions resulted in the stimulation of RNAP holoenzyme activity, but PROTEIN1 was observed to be more sensitive to PROTEIN2 stimulation than PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN2 The addition of PROTEIN0 to in vitro transcription reactions resulted in the stimulation of RNAP holoenzyme activity, but PROTEIN1 was observed to be more sensitive to PROTEIN2 stimulation than PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN3 The addition of PROTEIN0 to in vitro transcription reactions resulted in the stimulation of RNAP holoenzyme activity, but PROTEIN1 was observed to be more sensitive to PROTEIN2 stimulation than PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN2 The addition of PROTEIN0 to in vitro transcription reactions resulted in the stimulation of RNAP holoenzyme activity, but PROTEIN1 was observed to be more sensitive to PROTEIN2 stimulation than PROTEIN3\n",
      "\n",
      "PROTEIN1 PROTEIN3 The addition of PROTEIN0 to in vitro transcription reactions resulted in the stimulation of RNAP holoenzyme activity, but PROTEIN1 was observed to be more sensitive to PROTEIN2 stimulation than PROTEIN3\n",
      "\n",
      "PROTEIN2 PROTEIN3 The addition of PROTEIN0 to in vitro transcription reactions resulted in the stimulation of RNAP holoenzyme activity, but PROTEIN1 was observed to be more sensitive to PROTEIN2 stimulation than PROTEIN3\n",
      "\n",
      "PROTEIN0 PROTEIN1 Although PROTEIN0 is transcriptionally regulated by the key regulatory device that controls sporulation initiation, the PROTEIN1 phosphorelay, there is considerable evidence implicating a mechanism of post-translational control that governs the activity and concentration of sigmaH.\n",
      "\n",
      "PROTEIN0 PROTEIN1 Finally, we show that proper localization of SpoIVA required the expression of one or more genes which, like PROTEIN0 are under the control of the mother cell transcription factor PROTEIN1\n",
      "\n",
      "PROTEIN0 PROTEIN1 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN0 PROTEIN2 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN0 PROTEIN3 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN0 PROTEIN4 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN0 PROTEIN5 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN0 PROTEIN6 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN0 PROTEIN7 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN1 PROTEIN2 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN1 PROTEIN3 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN1 PROTEIN4 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN1 PROTEIN5 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN1 PROTEIN6 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN1 PROTEIN7 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN2 PROTEIN3 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN2 PROTEIN4 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN2 PROTEIN5 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN2 PROTEIN6 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN2 PROTEIN7 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN3 PROTEIN4 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN3 PROTEIN5 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN3 PROTEIN6 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN3 PROTEIN7 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN4 PROTEIN5 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN4 PROTEIN6 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN4 PROTEIN7 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN5 PROTEIN6 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN5 PROTEIN7 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n",
      "PROTEIN6 PROTEIN7 The PROTEIN0 regulon includes a penicillin binding protein PROTEIN1 and a co-transcribed amino acid racemase PROTEIN2 homologues of signal peptide peptidase PROTEIN3 flotillin PROTEIN4 ABC transporters PROTEIN5 non-haem bromoperoxidase PROTEIN6 epoxide hydrolase PROTEIN7 and three small peptides with structural similarities to bacteriocin precursor polypeptides.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for t in trainSet:\n",
    "    print(t)\n",
    "    print()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<H1>Imports</h1>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\Frost\\AppData\\Roaming\\Python\\Python37\\site-packages\\tensorflow\\python\\framework\\dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.\n",
      "  _np_qint8 = np.dtype([(\"qint8\", np.int8, 1)])\n",
      "C:\\Users\\Frost\\AppData\\Roaming\\Python\\Python37\\site-packages\\tensorflow\\python\\framework\\dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.\n",
      "  _np_quint8 = np.dtype([(\"quint8\", np.uint8, 1)])\n",
      "C:\\Users\\Frost\\AppData\\Roaming\\Python\\Python37\\site-packages\\tensorflow\\python\\framework\\dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.\n",
      "  _np_qint16 = np.dtype([(\"qint16\", np.int16, 1)])\n",
      "C:\\Users\\Frost\\AppData\\Roaming\\Python\\Python37\\site-packages\\tensorflow\\python\\framework\\dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.\n",
      "  _np_quint16 = np.dtype([(\"quint16\", np.uint16, 1)])\n",
      "C:\\Users\\Frost\\AppData\\Roaming\\Python\\Python37\\site-packages\\tensorflow\\python\\framework\\dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.\n",
      "  _np_qint32 = np.dtype([(\"qint32\", np.int32, 1)])\n",
      "C:\\Users\\Frost\\AppData\\Roaming\\Python\\Python37\\site-packages\\tensorflow\\python\\framework\\dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.\n",
      "  np_resource = np.dtype([(\"resource\", np.ubyte, 1)])\n",
      "Using TensorFlow backend.\n",
      "[nltk_data] Downloading package punkt to\n",
      "[nltk_data]     C:\\Users\\Frost\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package punkt is already up-to-date!\n",
      "[nltk_data] Downloading package stopwords to\n",
      "[nltk_data]     C:\\Users\\Frost\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package stopwords is already up-to-date!\n",
      "[nltk_data] Downloading package wordnet to\n",
      "[nltk_data]     C:\\Users\\Frost\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package wordnet is already up-to-date!\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "import re\n",
    "import time\n",
    "import numpy as np\n",
    "import tensorflow as tf\n",
    "from numpy import array\n",
    "from numpy import asarray\n",
    "from numpy import zeros\n",
    "\n",
    "from keras.preprocessing.text import Tokenizer\n",
    "from keras.preprocessing.sequence import pad_sequences\n",
    "\n",
    "\n",
    "from keras import regularizers\n",
    "\n",
    "from keras.models import Sequential, Model\n",
    "\n",
    "from keras.layers import Input\n",
    "from keras.layers import Conv1D\n",
    "from keras.layers import Conv2D\n",
    "from keras.layers import Dense\n",
    "from keras.layers import Flatten\n",
    "from keras.layers import Embedding\n",
    "from keras.layers import Reshape\n",
    "from keras.layers import MaxPooling1D\n",
    "from keras.layers import MaxPooling2D\n",
    "from keras.layers import GlobalMaxPooling1D\n",
    "from keras.layers import GlobalMaxPooling2D\n",
    "from keras.layers import BatchNormalization\n",
    "from keras.layers import Dropout\n",
    "from keras.layers import Activation\n",
    "from keras.layers import Lambda\n",
    "from keras.layers.merge import concatenate\n",
    "\n",
    "\n",
    "from keras.layers.advanced_activations import LeakyReLU\n",
    "\n",
    "from keras.callbacks import EarlyStopping, ModelCheckpoint, TensorBoard\n",
    "\n",
    "from keras.optimizers import SGD, Adam, RMSprop\n",
    "\n",
    "\n",
    "\n",
    "import spacy\n",
    "import scispacy\n",
    "import nltk\n",
    "\n",
    "\n",
    "nltk.download('punkt')\n",
    "nltk.download('stopwords')\n",
    "nltk.download('wordnet')\n",
    "\n",
    "nlp = spacy.load(\"en_core_sci_sm\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<h1>Preprocessing Functions</h1>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "def word2vec(x):\n",
    "    return nlp(x).vector"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(50, 96)\n"
     ]
    }
   ],
   "source": [
    "def padWithZeroes(matrix,rows,cols):\n",
    "    result = np.zeros((rows,96))\n",
    "    result[:matrix.shape[0],:matrix.shape[1]] = matrix\n",
    "    return result\n",
    "\n",
    "\n",
    "matrix = np.random.rand(11,96)\n",
    "\n",
    "padded = padWithZeroes(matrix,50,96)\n",
    "\n",
    "print(padded.shape)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'numpy.ndarray'>\n",
      "(50, 96)\n"
     ]
    }
   ],
   "source": [
    "#A GENERAL FUNCTION TO TAKE STRINGS AND CLEAN, SPLIT AND CONVERT TO NUMPY ARRAY\n",
    "def processString(sentence,maxWords):\n",
    "    text=sentence.lower()\n",
    "    tokens = nltk.word_tokenize(text)\n",
    "    #punctuation\n",
    "    #origina: tokens=nltk.tokenize.RegexpTokenizer(r'\\w+').tokenize(text)\n",
    "    tokens=nltk.tokenize.RegexpTokenizer(r'\\w+').tokenize(text)\n",
    "    stopwords=set(nltk.corpus.stopwords.words('english'))\n",
    "    tokens=list(filter(lambda word: not word in stopwords,tokens))\n",
    "    #lemmatize\n",
    "    #tokens = [nltk.stem.WordNetLemmatizer().lemmatize(word) for word in tokens]\n",
    "    #stemming\n",
    "    #tokens = [nltk.stem.SnowballStemmer('english').stem(word) for word in tokens]\n",
    "    #return word2vec( ' '.join(tokens) )\n",
    "    \n",
    "    result = np.array(list(map(word2vec, tokens)))\n",
    "    if len(tokens)<=maxWords:    \n",
    "        result = padWithZeroes(result,maxWords,result.shape[1])\n",
    "    \n",
    "    '''\n",
    "    for token in tokens:\n",
    "        if 'result' in locals():\n",
    "            result = np.concatenate((result,[word2vec(token)]),axis=0)\n",
    "        else:\n",
    "            result = [word2vec(token)]\n",
    "    \n",
    "    result = padWithZeroes(result,maxWords,result.shape[1])\n",
    "    '''\n",
    "    \n",
    "    #result = ' '.join(tokens) \n",
    "    return result\n",
    "    \n",
    "\n",
    "stringToClean=\"being anne:the hero's nests' has - apples/pears,bears, and\\cats. which-way or does she? she's a fiend! (everything i<br />i a)\"\n",
    "\n",
    "resultVectors=processString(stringToClean,50)\n",
    "\n",
    "print(type(resultVectors))\n",
    "#print(resultVectors)\n",
    "print(resultVectors.shape)\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<h1>Embedding</h1>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'\\nt,l = eliminateOverMaxWordExamples(trainSet,trainLabel,50)\\n\\nprint(len(trainSet),len(trainLabel))\\n\\n\\nprint(len(t),len(l))\\n\\n'"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "def eliminateOverMaxWordExamples(dataset,labels,maxWords):\n",
    "\n",
    "    aboveWordLimitIndex = []\n",
    "\n",
    "    for index,value in enumerate(dataset):\n",
    "        if value.shape[0]>50:\n",
    "            print(index)\n",
    "            print(value.shape)\n",
    "            aboveWordLimitIndex +=[index]\n",
    "\n",
    "    dataset = [t for t in dataset if t.shape[0]<=50]\n",
    "\n",
    "    labels = [v for i,v in enumerate(labels) if i not in aboveWordLimitIndex]\n",
    "    \n",
    "    return dataset, labels\n",
    "'''\n",
    "t,l = eliminateOverMaxWordExamples(trainSet,trainLabel,50)\n",
    "\n",
    "print(len(trainSet),len(trainLabel))\n",
    "\n",
    "\n",
    "print(len(t),len(l))\n",
    "\n",
    "'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "if os.path.isfile('padded_training_docs50.npy') and os.path.isfile('trainLabel50.npy'):\n",
    "    padded_training_docs = np.load('padded_training_docs50.npy')\n",
    "    trainLabel = np.load('trainLabel50.npy')\n",
    "else:\n",
    "    trainSet = [processString(t,50) for t in trainSet]\n",
    "    #padded_training_docs = [t for t in trainSet if t.shape[0]<=50]\n",
    "    padded_training_docs, trainLabel = eliminateOverMaxWordExamples(trainSet,trainLabel,50)\n",
    "    np.save('padded_training_docs50.npy', padded_training_docs)\n",
    "    np.save('trainLabel50.npy', trainLabel)\n",
    "    \n",
    "if os.path.isfile('padded_test_docs50.npy') and os.path.isfile('testLabel50.npy'):\n",
    "    padded_test_docs = np.load('padded_test_docs50.npy')\n",
    "    testLabel = np.load('testLabel50.npy')\n",
    "else:\n",
    "    testSet = [processString(t,50) for t in testSet]\n",
    "    #padded_test_docs = [t for t in testSet if t.shape[0]<=50]\n",
    "    padded_test_docs, testLabel = eliminateOverMaxWordExamples(testSet,testLabel,50)\n",
    "    np.save('padded_test_docs50.npy', padded_test_docs)\n",
    "    np.save('testLabel50.npy', testLabel)\n",
    "    \n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "padded_training_docs = np.array(padded_training_docs)\n",
    "trainLabel = np.array(trainLabel)\n",
    "padded_test_docs = np.array(padded_test_docs)\n",
    "testLabel = np.array(testLabel)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "done\n",
      "10235 10235\n",
      "2274 2274\n"
     ]
    }
   ],
   "source": [
    "print(\"done\")\n",
    "print(len(padded_training_docs),len(trainLabel))\n",
    "print(len(padded_test_docs),len(testLabel))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<h1>Functional API</h1>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "epochs_to_run = 300\n",
    "callbackImprovementPercentage = 0.000001\n",
    "patienceEpochs = 5 \n",
    "whatToMonitor = 'val_accuracy'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Model: \"model_1\"\n",
      "_________________________________________________________________\n",
      "Layer (type)                 Output Shape              Param #   \n",
      "=================================================================\n",
      "input_1 (InputLayer)         (None, 50, 96)            0         \n",
      "_________________________________________________________________\n",
      "flatten_1 (Flatten)          (None, 4800)              0         \n",
      "_________________________________________________________________\n",
      "dense_1 (Dense)              (None, 2000)              9602000   \n",
      "_________________________________________________________________\n",
      "dense_2 (Dense)              (None, 1000)              2001000   \n",
      "_________________________________________________________________\n",
      "dense_3 (Dense)              (None, 500)               500500    \n",
      "_________________________________________________________________\n",
      "dense_4 (Dense)              (None, 200)               100200    \n",
      "_________________________________________________________________\n",
      "dense_5 (Dense)              (None, 50)                10050     \n",
      "_________________________________________________________________\n",
      "dense_6 (Dense)              (None, 50)                2550      \n",
      "_________________________________________________________________\n",
      "dense_7 (Dense)              (None, 1)                 51        \n",
      "=================================================================\n",
      "Total params: 12,216,351\n",
      "Trainable params: 12,216,351\n",
      "Non-trainable params: 0\n",
      "_________________________________________________________________\n",
      "WARNING:tensorflow:From C:\\Users\\Frost\\AppData\\Roaming\\Python\\Python37\\site-packages\\tensorflow\\python\\ops\\nn_impl.py:180: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.\n",
      "Instructions for updating:\n",
      "Use tf.where in 2.0, which has the same broadcast rule as np.where\n",
      "\n",
      "Starting training...\n",
      "\n",
      "WARNING:tensorflow:From c:\\users\\frost\\anaconda3\\envs\\tf_gpu\\lib\\site-packages\\keras\\backend\\tensorflow_backend.py:422: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.\n",
      "\n",
      "Epoch 1/300\n",
      "10235/10235 [==============================] - 5s 525us/step - loss: 0.6061 - accuracy: 0.7065\n",
      "Epoch 2/300\n",
      "  288/10235 [..............................] - ETA: 4s - loss: 0.5737 - accuracy: 0.7396"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "c:\\users\\frost\\anaconda3\\envs\\tf_gpu\\lib\\site-packages\\keras\\callbacks\\callbacks.py:846: RuntimeWarning: Early stopping conditioned on metric `val_accuracy` which is not available. Available metrics are: loss,accuracy\n",
      "  (self.monitor, ','.join(list(logs.keys()))), RuntimeWarning\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.6032 - accuracy: 0.7104\n",
      "Epoch 3/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.6027 - accuracy: 0.7104\n",
      "Epoch 4/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.6028 - accuracy: 0.7104\n",
      "Epoch 5/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.6024 - accuracy: 0.7104\n",
      "Epoch 6/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.6025 - accuracy: 0.7104\n",
      "Epoch 7/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.6027 - accuracy: 0.71040s - loss: 0.6032 - accuracy\n",
      "Epoch 8/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.5997 - accuracy: 0.7104\n",
      "Epoch 9/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.5913 - accuracy: 0.7104\n",
      "Epoch 10/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.5816 - accuracy: 0.71040s - loss: 0.5802 - ac\n",
      "Epoch 11/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.5731 - accuracy: 0.7104\n",
      "Epoch 12/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.5695 - accuracy: 0.7083\n",
      "Epoch 13/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.5618 - accuracy: 0.7106\n",
      "Epoch 14/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.5555 - accuracy: 0.7094\n",
      "Epoch 15/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.5499 - accuracy: 0.7092\n",
      "Epoch 16/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.5383 - accuracy: 0.7143\n",
      "Epoch 17/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.5351 - accuracy: 0.7133\n",
      "Epoch 18/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.5286 - accuracy: 0.7195\n",
      "Epoch 19/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.5216 - accuracy: 0.7271\n",
      "Epoch 20/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.5201 - accuracy: 0.7264\n",
      "Epoch 21/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.5203 - accuracy: 0.7325\n",
      "Epoch 22/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.5140 - accuracy: 0.7350\n",
      "Epoch 23/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.5108 - accuracy: 0.7376\n",
      "Epoch 24/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.5023 - accuracy: 0.7408\n",
      "Epoch 25/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.4957 - accuracy: 0.7436\n",
      "Epoch 26/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4909 - accuracy: 0.7460\n",
      "Epoch 27/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4844 - accuracy: 0.7550\n",
      "Epoch 28/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4825 - accuracy: 0.7534\n",
      "Epoch 29/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4838 - accuracy: 0.7586\n",
      "Epoch 30/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.4809 - accuracy: 0.7596\n",
      "Epoch 31/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4786 - accuracy: 0.7576\n",
      "Epoch 32/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4755 - accuracy: 0.76020s - loss: 0.4769 - accuracy: \n",
      "Epoch 33/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.4664 - accuracy: 0.7705\n",
      "Epoch 34/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4723 - accuracy: 0.7674\n",
      "Epoch 35/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4703 - accuracy: 0.7645\n",
      "Epoch 36/300\n",
      "10235/10235 [==============================] - 5s 495us/step - loss: 0.4683 - accuracy: 0.76572s - loss: 0.4 - ETA:  - ETA: 1s\n",
      "Epoch 37/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.4620 - accuracy: 0.7728\n",
      "Epoch 38/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4620 - accuracy: 0.7684\n",
      "Epoch 39/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4612 - accuracy: 0.77660s - loss: 0.462\n",
      "Epoch 40/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4470 - accuracy: 0.7833\n",
      "Epoch 41/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.4491 - accuracy: 0.7828\n",
      "Epoch 42/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4504 - accuracy: 0.7816\n",
      "Epoch 43/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4455 - accuracy: 0.7852\n",
      "Epoch 44/300\n",
      "10235/10235 [==============================] - 5s 495us/step - loss: 0.4545 - accuracy: 0.7811\n",
      "Epoch 45/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4514 - accuracy: 0.7764\n",
      "Epoch 46/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4467 - accuracy: 0.7855\n",
      "Epoch 47/300\n",
      "10235/10235 [==============================] - 5s 499us/step - loss: 0.4402 - accuracy: 0.7932\n",
      "Epoch 48/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.4432 - accuracy: 0.7914\n",
      "Epoch 49/300\n",
      "10235/10235 [==============================] - 5s 496us/step - loss: 0.4356 - accuracy: 0.7929\n",
      "Epoch 50/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4373 - accuracy: 0.7964\n",
      "Epoch 51/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.4344 - accuracy: 0.7958\n",
      "Epoch 52/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4278 - accuracy: 0.7968\n",
      "Epoch 53/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.4362 - accuracy: 0.7936\n",
      "Epoch 54/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4322 - accuracy: 0.7980\n",
      "Epoch 55/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4194 - accuracy: 0.8052\n",
      "Epoch 56/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4186 - accuracy: 0.7991\n",
      "Epoch 57/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4204 - accuracy: 0.8046\n",
      "Epoch 58/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4158 - accuracy: 0.8042\n",
      "Epoch 59/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.4168 - accuracy: 0.8050\n",
      "Epoch 60/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4120 - accuracy: 0.8083\n",
      "Epoch 61/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4101 - accuracy: 0.8094\n",
      "Epoch 62/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.4077 - accuracy: 0.8134\n",
      "Epoch 63/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4062 - accuracy: 0.8119\n",
      "Epoch 64/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.4077 - accuracy: 0.8113\n",
      "Epoch 65/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.3978 - accuracy: 0.8168\n",
      "Epoch 66/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3979 - accuracy: 0.8171\n",
      "Epoch 67/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3882 - accuracy: 0.8211\n",
      "Epoch 68/300\n",
      "10235/10235 [==============================] - 5s 495us/step - loss: 0.3891 - accuracy: 0.8277\n",
      "Epoch 69/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3967 - accuracy: 0.81900s - loss:\n",
      "Epoch 70/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.3971 - accuracy: 0.8199\n",
      "Epoch 71/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3896 - accuracy: 0.8205\n",
      "Epoch 72/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.3817 - accuracy: 0.8298\n",
      "Epoch 73/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3821 - accuracy: 0.8286\n",
      "Epoch 74/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3850 - accuracy: 0.82850s - loss: 0.3851 - accuracy: 0.82\n",
      "Epoch 75/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.3858 - accuracy: 0.8262\n",
      "Epoch 76/300\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.3827 - accuracy: 0.8283\n",
      "Epoch 77/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.3722 - accuracy: 0.8346\n",
      "Epoch 78/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3770 - accuracy: 0.8293\n",
      "Epoch 79/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.3792 - accuracy: 0.8289\n",
      "Epoch 80/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3670 - accuracy: 0.8329\n",
      "Epoch 81/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3773 - accuracy: 0.8257\n",
      "Epoch 82/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3769 - accuracy: 0.8292\n",
      "Epoch 83/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.3688 - accuracy: 0.8306\n",
      "Epoch 84/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.3664 - accuracy: 0.8376\n",
      "Epoch 85/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.3593 - accuracy: 0.8407\n",
      "Epoch 86/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3605 - accuracy: 0.84010s - loss: 0.3617 - \n",
      "Epoch 87/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.3550 - accuracy: 0.8438\n",
      "Epoch 88/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3574 - accuracy: 0.8404\n",
      "Epoch 89/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.3616 - accuracy: 0.8364\n",
      "Epoch 90/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3554 - accuracy: 0.8405\n",
      "Epoch 91/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3582 - accuracy: 0.83870s -\n",
      "Epoch 92/300\n",
      "10235/10235 [==============================] - 5s 495us/step - loss: 0.3480 - accuracy: 0.8460\n",
      "Epoch 93/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.3482 - accuracy: 0.8450\n",
      "Epoch 94/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3437 - accuracy: 0.8512\n",
      "Epoch 95/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.3493 - accuracy: 0.8465\n",
      "Epoch 96/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.3478 - accuracy: 0.8442\n",
      "Epoch 97/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3555 - accuracy: 0.8410\n",
      "Epoch 98/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3437 - accuracy: 0.8469\n",
      "Epoch 99/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.3444 - accuracy: 0.8468\n",
      "Epoch 100/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3315 - accuracy: 0.8517\n",
      "Epoch 101/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.3335 - accuracy: 0.85190s - loss: 0.3314 - accura\n",
      "Epoch 102/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.3391 - accuracy: 0.8497\n",
      "Epoch 103/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3359 - accuracy: 0.8540\n",
      "Epoch 104/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.3312 - accuracy: 0.8532\n",
      "Epoch 105/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.3278 - accuracy: 0.8526\n",
      "Epoch 106/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3237 - accuracy: 0.8559\n",
      "Epoch 107/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3187 - accuracy: 0.8576\n",
      "Epoch 108/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3131 - accuracy: 0.8603\n",
      "Epoch 109/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3051 - accuracy: 0.8656\n",
      "Epoch 110/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3133 - accuracy: 0.8647\n",
      "Epoch 111/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3108 - accuracy: 0.8640\n",
      "Epoch 112/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3092 - accuracy: 0.8666\n",
      "Epoch 113/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3059 - accuracy: 0.8633\n",
      "Epoch 114/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3092 - accuracy: 0.8627\n",
      "Epoch 115/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.3127 - accuracy: 0.86100s - loss: 0.3117 - accura\n",
      "Epoch 116/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.3058 - accuracy: 0.8597\n",
      "Epoch 117/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3120 - accuracy: 0.8605\n",
      "Epoch 118/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3039 - accuracy: 0.8633\n",
      "Epoch 119/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.3180 - accuracy: 0.8579\n",
      "Epoch 120/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.3043 - accuracy: 0.8675\n",
      "Epoch 121/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3021 - accuracy: 0.8641\n",
      "Epoch 122/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3125 - accuracy: 0.8665\n",
      "Epoch 123/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2936 - accuracy: 0.8716\n",
      "Epoch 124/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3000 - accuracy: 0.8689\n",
      "Epoch 125/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.3006 - accuracy: 0.8686\n",
      "Epoch 126/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2905 - accuracy: 0.8703\n",
      "Epoch 127/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2926 - accuracy: 0.8705\n",
      "Epoch 128/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2928 - accuracy: 0.8730\n",
      "Epoch 129/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.2866 - accuracy: 0.8756\n",
      "Epoch 130/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2966 - accuracy: 0.8686\n",
      "Epoch 131/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2905 - accuracy: 0.8728\n",
      "Epoch 132/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2781 - accuracy: 0.8789\n",
      "Epoch 133/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2872 - accuracy: 0.8746\n",
      "Epoch 134/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2798 - accuracy: 0.8779\n",
      "Epoch 135/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2902 - accuracy: 0.8702\n",
      "Epoch 136/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2826 - accuracy: 0.8765\n",
      "Epoch 137/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2844 - accuracy: 0.8734\n",
      "Epoch 138/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2828 - accuracy: 0.8746\n",
      "Epoch 139/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2794 - accuracy: 0.8758\n",
      "Epoch 140/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2700 - accuracy: 0.8792\n",
      "Epoch 141/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.2826 - accuracy: 0.8741\n",
      "Epoch 142/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2661 - accuracy: 0.8822\n",
      "Epoch 143/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2730 - accuracy: 0.8827\n",
      "Epoch 144/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2894 - accuracy: 0.8742\n",
      "Epoch 145/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2814 - accuracy: 0.87930s - loss: 0.2799 - accu\n",
      "Epoch 146/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2816 - accuracy: 0.8756\n",
      "Epoch 147/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2673 - accuracy: 0.8823\n",
      "Epoch 148/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2719 - accuracy: 0.8798\n",
      "Epoch 149/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2628 - accuracy: 0.8877\n",
      "Epoch 150/300\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2636 - accuracy: 0.8859\n",
      "Epoch 151/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2707 - accuracy: 0.8814\n",
      "Epoch 152/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.2757 - accuracy: 0.8791\n",
      "Epoch 153/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2519 - accuracy: 0.8879\n",
      "Epoch 154/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2700 - accuracy: 0.8835\n",
      "Epoch 155/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2671 - accuracy: 0.8808\n",
      "Epoch 156/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2626 - accuracy: 0.8830\n",
      "Epoch 157/300\n",
      "10235/10235 [==============================] - 5s 490us/step - loss: 0.2680 - accuracy: 0.8853\n",
      "Epoch 158/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2687 - accuracy: 0.8819\n",
      "Epoch 159/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2632 - accuracy: 0.8830\n",
      "Epoch 160/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2573 - accuracy: 0.8853\n",
      "Epoch 161/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2615 - accuracy: 0.88611s - loss: 0 - ETA: 0s - los\n",
      "Epoch 162/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2567 - accuracy: 0.8903\n",
      "Epoch 163/300\n",
      "10235/10235 [==============================] - 5s 495us/step - loss: 0.2555 - accuracy: 0.8870\n",
      "Epoch 164/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2621 - accuracy: 0.8857\n",
      "Epoch 165/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2779 - accuracy: 0.8784\n",
      "Epoch 166/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2739 - accuracy: 0.8782\n",
      "Epoch 167/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2687 - accuracy: 0.8838\n",
      "Epoch 168/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2575 - accuracy: 0.8868\n",
      "Epoch 169/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2610 - accuracy: 0.8897\n",
      "Epoch 170/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2580 - accuracy: 0.8895\n",
      "Epoch 171/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2686 - accuracy: 0.8841\n",
      "Epoch 172/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2664 - accuracy: 0.8816\n",
      "Epoch 173/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2636 - accuracy: 0.8821\n",
      "Epoch 174/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2708 - accuracy: 0.8793\n",
      "Epoch 175/300\n",
      "10235/10235 [==============================] - 5s 493us/step - loss: 0.2664 - accuracy: 0.8827\n",
      "Epoch 176/300\n",
      "10235/10235 [==============================] - 5s 498us/step - loss: 0.2579 - accuracy: 0.8899\n",
      "Epoch 177/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2624 - accuracy: 0.8850\n",
      "Epoch 178/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2488 - accuracy: 0.8925\n",
      "Epoch 179/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2572 - accuracy: 0.8870\n",
      "Epoch 180/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2511 - accuracy: 0.8915\n",
      "Epoch 181/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2541 - accuracy: 0.8865\n",
      "Epoch 182/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2575 - accuracy: 0.88940s - loss: 0.2587 - accuracy: \n",
      "Epoch 183/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2453 - accuracy: 0.8940\n",
      "Epoch 184/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2473 - accuracy: 0.8895\n",
      "Epoch 185/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2520 - accuracy: 0.8916\n",
      "Epoch 186/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2533 - accuracy: 0.8911\n",
      "Epoch 187/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2548 - accuracy: 0.8884\n",
      "Epoch 188/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2550 - accuracy: 0.8880\n",
      "Epoch 189/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2497 - accuracy: 0.8873\n",
      "Epoch 190/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2556 - accuracy: 0.8862\n",
      "Epoch 191/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2398 - accuracy: 0.9003\n",
      "Epoch 192/300\n",
      "10235/10235 [==============================] - 5s 488us/step - loss: 0.2387 - accuracy: 0.8966\n",
      "Epoch 193/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2352 - accuracy: 0.8985\n",
      "Epoch 194/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2428 - accuracy: 0.8992\n",
      "Epoch 195/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2419 - accuracy: 0.8952\n",
      "Epoch 196/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2347 - accuracy: 0.8971\n",
      "Epoch 197/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2338 - accuracy: 0.9005\n",
      "Epoch 198/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2409 - accuracy: 0.8957\n",
      "Epoch 199/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2341 - accuracy: 0.8985\n",
      "Epoch 200/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2499 - accuracy: 0.8902\n",
      "Epoch 201/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2276 - accuracy: 0.9001\n",
      "Epoch 202/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2282 - accuracy: 0.9012\n",
      "Epoch 203/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2350 - accuracy: 0.8994\n",
      "Epoch 204/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2362 - accuracy: 0.8965\n",
      "Epoch 205/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2467 - accuracy: 0.8915\n",
      "Epoch 206/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2503 - accuracy: 0.8899\n",
      "Epoch 207/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2430 - accuracy: 0.8960\n",
      "Epoch 208/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2403 - accuracy: 0.8963\n",
      "Epoch 209/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2305 - accuracy: 0.9011\n",
      "Epoch 210/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2371 - accuracy: 0.8975\n",
      "Epoch 211/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2305 - accuracy: 0.9003\n",
      "Epoch 212/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2279 - accuracy: 0.9025\n",
      "Epoch 213/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2433 - accuracy: 0.8968\n",
      "Epoch 214/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2681 - accuracy: 0.8819\n",
      "Epoch 215/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2347 - accuracy: 0.8951\n",
      "Epoch 216/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2328 - accuracy: 0.9005\n",
      "Epoch 217/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2302 - accuracy: 0.8978\n",
      "Epoch 218/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2275 - accuracy: 0.9019\n",
      "Epoch 219/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2236 - accuracy: 0.9014\n",
      "Epoch 220/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2353 - accuracy: 0.8970\n",
      "Epoch 221/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2410 - accuracy: 0.89620s - loss: 0.2352 - \n",
      "Epoch 222/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2315 - accuracy: 0.8974\n",
      "Epoch 223/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2269 - accuracy: 0.9009\n",
      "Epoch 224/300\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2348 - accuracy: 0.8981\n",
      "Epoch 225/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2477 - accuracy: 0.8958\n",
      "Epoch 226/300\n",
      "10235/10235 [==============================] - 5s 492us/step - loss: 0.2328 - accuracy: 0.9010\n",
      "Epoch 227/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2145 - accuracy: 0.9076\n",
      "Epoch 228/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2156 - accuracy: 0.9065\n",
      "Epoch 229/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2276 - accuracy: 0.9028\n",
      "Epoch 230/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2283 - accuracy: 0.9020\n",
      "Epoch 231/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2266 - accuracy: 0.8990\n",
      "Epoch 232/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2235 - accuracy: 0.9018\n",
      "Epoch 233/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2204 - accuracy: 0.9056\n",
      "Epoch 234/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2222 - accuracy: 0.9037\n",
      "Epoch 235/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2244 - accuracy: 0.9023\n",
      "Epoch 236/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2232 - accuracy: 0.9007\n",
      "Epoch 237/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2242 - accuracy: 0.9038\n",
      "Epoch 238/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2391 - accuracy: 0.8960\n",
      "Epoch 239/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2330 - accuracy: 0.8988\n",
      "Epoch 240/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2236 - accuracy: 0.9034\n",
      "Epoch 241/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2203 - accuracy: 0.9024\n",
      "Epoch 242/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2259 - accuracy: 0.8999\n",
      "Epoch 243/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2243 - accuracy: 0.9026\n",
      "Epoch 244/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2177 - accuracy: 0.9045\n",
      "Epoch 245/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2166 - accuracy: 0.9043\n",
      "Epoch 246/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2161 - accuracy: 0.9092\n",
      "Epoch 247/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2184 - accuracy: 0.9069\n",
      "Epoch 248/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2162 - accuracy: 0.9050\n",
      "Epoch 249/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2186 - accuracy: 0.9050\n",
      "Epoch 250/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2202 - accuracy: 0.9032\n",
      "Epoch 251/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2260 - accuracy: 0.8995\n",
      "Epoch 252/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2216 - accuracy: 0.9051\n",
      "Epoch 253/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2181 - accuracy: 0.9043\n",
      "Epoch 254/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2118 - accuracy: 0.9092\n",
      "Epoch 255/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2117 - accuracy: 0.9073\n",
      "Epoch 256/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2177 - accuracy: 0.9056\n",
      "Epoch 257/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2077 - accuracy: 0.9106\n",
      "Epoch 258/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2106 - accuracy: 0.9074\n",
      "Epoch 259/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2123 - accuracy: 0.9097\n",
      "Epoch 260/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2177 - accuracy: 0.9027\n",
      "Epoch 261/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2248 - accuracy: 0.9026\n",
      "Epoch 262/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2233 - accuracy: 0.9016\n",
      "Epoch 263/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2189 - accuracy: 0.9020\n",
      "Epoch 264/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2220 - accuracy: 0.9047\n",
      "Epoch 265/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2252 - accuracy: 0.9032\n",
      "Epoch 266/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2233 - accuracy: 0.9025\n",
      "Epoch 267/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2224 - accuracy: 0.9047\n",
      "Epoch 268/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2180 - accuracy: 0.9060\n",
      "Epoch 269/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2229 - accuracy: 0.9015\n",
      "Epoch 270/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2176 - accuracy: 0.9075\n",
      "Epoch 271/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2074 - accuracy: 0.9081\n",
      "Epoch 272/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2171 - accuracy: 0.9057\n",
      "Epoch 273/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2095 - accuracy: 0.9063\n",
      "Epoch 274/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2159 - accuracy: 0.9064\n",
      "Epoch 275/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2089 - accuracy: 0.9086\n",
      "Epoch 276/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2165 - accuracy: 0.9075\n",
      "Epoch 277/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2205 - accuracy: 0.9050\n",
      "Epoch 278/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2113 - accuracy: 0.9080\n",
      "Epoch 279/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2033 - accuracy: 0.9144\n",
      "Epoch 280/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2182 - accuracy: 0.9058\n",
      "Epoch 281/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2162 - accuracy: 0.9050\n",
      "Epoch 282/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2177 - accuracy: 0.9040\n",
      "Epoch 283/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2115 - accuracy: 0.9058\n",
      "Epoch 284/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2067 - accuracy: 0.9124\n",
      "Epoch 285/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2203 - accuracy: 0.9046\n",
      "Epoch 286/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2047 - accuracy: 0.9129\n",
      "Epoch 287/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2237 - accuracy: 0.9027\n",
      "Epoch 288/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2172 - accuracy: 0.9030\n",
      "Epoch 289/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2107 - accuracy: 0.9096\n",
      "Epoch 290/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2076 - accuracy: 0.9073\n",
      "Epoch 291/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2134 - accuracy: 0.9068\n",
      "Epoch 292/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2136 - accuracy: 0.9078\n",
      "Epoch 293/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2130 - accuracy: 0.9079\n",
      "Epoch 294/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2056 - accuracy: 0.9117\n",
      "Epoch 295/300\n",
      "10235/10235 [==============================] - 5s 487us/step - loss: 0.2080 - accuracy: 0.9096\n",
      "Epoch 296/300\n",
      "10235/10235 [==============================] - 5s 485us/step - loss: 0.2196 - accuracy: 0.9087\n",
      "Epoch 297/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2244 - accuracy: 0.90080s - loss: 0.2246 - ac\n",
      "Epoch 298/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2274 - accuracy: 0.8973\n",
      "Epoch 299/300\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2160 - accuracy: 0.9057\n",
      "Epoch 300/300\n",
      "10235/10235 [==============================] - 5s 489us/step - loss: 0.2150 - accuracy: 0.9093\n",
      "\n",
      "Training done in 25.132305618127187 minutes or 1507.9383370876312 seconds now evaluating...\n",
      "\n",
      "10235/10235 [==============================] - 1s 82us/step\n",
      "Training Accuracy: 92.125058\n",
      "\n",
      "2274/2274 [==============================] - 0s 89us/step\n",
      "Test Accuracy: 69.437116\n"
     ]
    }
   ],
   "source": [
    "#https://keras.io/getting-started/functional-api-guide/\n",
    "\n",
    "sentence_length = 50\n",
    "wordvec_length = 96\n",
    "\n",
    "\n",
    "input_layer = Input(shape=(sentence_length,wordvec_length))\n",
    "\n",
    "#x = Embedding(input_dim=vocab_size, output_dim=wordvec_length, weights=[embedding_matrix],  trainable=True)(input_layer)\n",
    "\n",
    "#x = Conv1D(filters = 1, kernel_size = 64, strides=1, padding='same', use_bias=False)(input_layer)\n",
    "\n",
    "#x = big_block(x)\n",
    "\n",
    "#x = GlobalMaxPooling1D()(x)\n",
    "\n",
    "x = Flatten()(input_layer)\n",
    "\n",
    "x = Dense(2000, activation='sigmoid')(x)\n",
    "\n",
    "x = Dense(1000, activation='sigmoid')(x)\n",
    "\n",
    "x = Dense(500, activation='sigmoid')(x)\n",
    "\n",
    "x = Dense(200, activation='sigmoid')(x)\n",
    "\n",
    "x = Dense(50, activation='sigmoid')(x)\n",
    "\n",
    "x = Dense(50, activation='sigmoid')(x)\n",
    "\n",
    "output_layer = Dense(1, activation='sigmoid')(x)\n",
    "\n",
    "dense_model = Model(inputs = input_layer, outputs = output_layer)\n",
    "\n",
    "dense_model.summary()\n",
    "\n",
    "\n",
    "#compile\n",
    "dense_model.compile(optimizer='adam', loss='binary_crossentropy', metrics=['accuracy'])\n",
    "\n",
    "\n",
    "print()\n",
    "print(\"Starting training...\")\n",
    "print()\n",
    "\n",
    "early_stop = EarlyStopping(monitor=whatToMonitor, \n",
    "                           min_delta=callbackImprovementPercentage, #apparently percentage\n",
    "                           patience=patienceEpochs, \n",
    "                           mode='min', \n",
    "                           verbose=1)\n",
    "\n",
    "\n",
    "\n",
    "start = time.time()\n",
    "\n",
    "# fit the model\n",
    "dense_model.fit(padded_training_docs, trainLabel, epochs=epochs_to_run, verbose=1,callbacks        = [early_stop])\n",
    "\n",
    "durationSecond = time.time()-start\n",
    "\n",
    "print()\n",
    "print(\"Training done in\",durationSecond/60,\"minutes or\",durationSecond,\"seconds now evaluating...\")\n",
    "print()\n",
    "\n",
    "\n",
    "\n",
    "# evaluate the model\n",
    "loss, accuracy = dense_model.evaluate(padded_training_docs, trainLabel, verbose=1)\n",
    "print('Training Accuracy: %f' % (accuracy*100))\n",
    "\n",
    "print()\n",
    "\n",
    "\n",
    "# evaluate the model\n",
    "loss, accuracy = dense_model.evaluate(padded_test_docs, testLabel, verbose=1)\n",
    "print('Test Accuracy: %f' % (accuracy*100))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Model: \"model_2\"\n",
      "_________________________________________________________________\n",
      "Layer (type)                 Output Shape              Param #   \n",
      "=================================================================\n",
      "input_2 (InputLayer)         (None, 50, 96)            0         \n",
      "_________________________________________________________________\n",
      "reshape_1 (Reshape)          (None, 50, 96, 1)         0         \n",
      "_________________________________________________________________\n",
      "conv2d_1 (Conv2D)            (None, 16, 32, 100)       1000      \n",
      "_________________________________________________________________\n",
      "conv2d_2 (Conv2D)            (None, 4, 8, 200)         320200    \n",
      "_________________________________________________________________\n",
      "conv2d_3 (Conv2D)            (None, 1, 1, 400)         2560400   \n",
      "_________________________________________________________________\n",
      "global_max_pooling2d_1 (Glob (None, 400)               0         \n",
      "_________________________________________________________________\n",
      "dense_8 (Dense)              (None, 400)               160400    \n",
      "_________________________________________________________________\n",
      "dense_9 (Dense)              (None, 200)               80200     \n",
      "_________________________________________________________________\n",
      "dense_10 (Dense)             (None, 100)               20100     \n",
      "_________________________________________________________________\n",
      "dense_11 (Dense)             (None, 50)                5050      \n",
      "_________________________________________________________________\n",
      "dense_12 (Dense)             (None, 25)                1275      \n",
      "_________________________________________________________________\n",
      "dense_13 (Dense)             (None, 10)                260       \n",
      "_________________________________________________________________\n",
      "dense_14 (Dense)             (None, 5)                 55        \n",
      "_________________________________________________________________\n",
      "dense_15 (Dense)             (None, 1)                 6         \n",
      "=================================================================\n",
      "Total params: 3,148,946\n",
      "Trainable params: 3,148,946\n",
      "Non-trainable params: 0\n",
      "_________________________________________________________________\n",
      "\n",
      "Starting training...\n",
      "\n",
      "Epoch 1/300\n",
      "10235/10235 [==============================] - 6s 569us/step - loss: 0.5983 - accuracy: 0.7104\n",
      "Epoch 2/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5940 - accuracy: 0.7104\n",
      "Epoch 3/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5932 - accuracy: 0.7104\n",
      "Epoch 4/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5923 - accuracy: 0.7104\n",
      "Epoch 5/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5917 - accuracy: 0.7104\n",
      "Epoch 6/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5917 - accuracy: 0.7104\n",
      "Epoch 7/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5915 - accuracy: 0.7104\n",
      "Epoch 8/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5915 - accuracy: 0.7104\n",
      "Epoch 9/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5914 - accuracy: 0.7095\n",
      "Epoch 10/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5914 - accuracy: 0.7104\n",
      "Epoch 11/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5911 - accuracy: 0.7104\n",
      "Epoch 12/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5915 - accuracy: 0.7097\n",
      "Epoch 13/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5912 - accuracy: 0.7113\n",
      "Epoch 14/300\n",
      "10235/10235 [==============================] - 5s 472us/step - loss: 0.5913 - accuracy: 0.7104\n",
      "Epoch 15/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5910 - accuracy: 0.7095\n",
      "Epoch 16/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5916 - accuracy: 0.7089\n",
      "Epoch 17/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5967 - accuracy: 0.7111\n",
      "Epoch 18/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5914 - accuracy: 0.7090\n",
      "Epoch 19/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5919 - accuracy: 0.7101\n",
      "Epoch 20/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5921 - accuracy: 0.7103\n",
      "Epoch 21/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5876 - accuracy: 0.7104\n",
      "Epoch 22/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5857 - accuracy: 0.7104\n",
      "Epoch 23/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5850 - accuracy: 0.7104\n",
      "Epoch 24/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5845 - accuracy: 0.71040s - loss: 0.5853 - ac\n",
      "Epoch 25/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5843 - accuracy: 0.7104\n",
      "Epoch 26/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5843 - accuracy: 0.71040s - loss: 0.5853 - accura\n",
      "Epoch 27/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5841 - accuracy: 0.7104\n",
      "Epoch 28/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5840 - accuracy: 0.7104\n",
      "Epoch 29/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5841 - accuracy: 0.7104\n",
      "Epoch 30/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5838 - accuracy: 0.71050s - loss:\n",
      "Epoch 31/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5844 - accuracy: 0.7104\n",
      "Epoch 32/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5841 - accuracy: 0.7108\n",
      "Epoch 33/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5841 - accuracy: 0.7102\n",
      "Epoch 34/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5841 - accuracy: 0.7104\n",
      "Epoch 35/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5839 - accuracy: 0.7104\n",
      "Epoch 36/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5841 - accuracy: 0.70980s - loss: 0.5\n",
      "Epoch 37/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5840 - accuracy: 0.7100\n",
      "Epoch 38/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5842 - accuracy: 0.7084\n",
      "Epoch 39/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5841 - accuracy: 0.7103\n",
      "Epoch 40/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5845 - accuracy: 0.7102\n",
      "Epoch 41/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5883 - accuracy: 0.70870s - loss: 0.5869 - accu\n",
      "Epoch 42/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5841 - accuracy: 0.7092\n",
      "Epoch 43/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5840 - accuracy: 0.7104\n",
      "Epoch 44/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5839 - accuracy: 0.70970s - loss: 0.5849 - accuracy: \n",
      "Epoch 45/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5839 - accuracy: 0.7102\n",
      "Epoch 46/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5839 - accuracy: 0.7089\n",
      "Epoch 47/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5840 - accuracy: 0.71002s - l - E\n",
      "Epoch 48/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5839 - accuracy: 0.7092\n",
      "Epoch 49/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5839 - accuracy: 0.7085\n",
      "Epoch 50/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5840 - accuracy: 0.7089\n",
      "Epoch 51/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5838 - accuracy: 0.7102\n",
      "Epoch 52/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5842 - accuracy: 0.7095\n",
      "Epoch 53/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5840 - accuracy: 0.7069\n",
      "Epoch 54/300\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5838 - accuracy: 0.71020s - loss: 0.5837 - accuracy: 0.71\n",
      "Epoch 55/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5839 - accuracy: 0.7110\n",
      "Epoch 56/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5837 - accuracy: 0.7117\n",
      "Epoch 57/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5840 - accuracy: 0.7101\n",
      "Epoch 58/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5837 - accuracy: 0.7096\n",
      "Epoch 59/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5844 - accuracy: 0.7104\n",
      "Epoch 60/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5840 - accuracy: 0.7087\n",
      "Epoch 61/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5839 - accuracy: 0.7076\n",
      "Epoch 62/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5838 - accuracy: 0.7101\n",
      "Epoch 63/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5839 - accuracy: 0.7090\n",
      "Epoch 64/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5837 - accuracy: 0.7104\n",
      "Epoch 65/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5840 - accuracy: 0.7092\n",
      "Epoch 66/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5839 - accuracy: 0.7093\n",
      "Epoch 67/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5840 - accuracy: 0.7093\n",
      "Epoch 68/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5838 - accuracy: 0.7089\n",
      "Epoch 69/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5838 - accuracy: 0.7100\n",
      "Epoch 70/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5837 - accuracy: 0.7107\n",
      "Epoch 71/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5838 - accuracy: 0.7069\n",
      "Epoch 72/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5835 - accuracy: 0.7107\n",
      "Epoch 73/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5837 - accuracy: 0.7075\n",
      "Epoch 74/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5837 - accuracy: 0.7097\n",
      "Epoch 75/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5834 - accuracy: 0.7095\n",
      "Epoch 76/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5835 - accuracy: 0.7093\n",
      "Epoch 77/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5838 - accuracy: 0.7080\n",
      "Epoch 78/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5836 - accuracy: 0.71000s - loss: 0\n",
      "Epoch 79/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5836 - accuracy: 0.7107\n",
      "Epoch 80/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5834 - accuracy: 0.7094\n",
      "Epoch 81/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5833 - accuracy: 0.7077\n",
      "Epoch 82/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5832 - accuracy: 0.70850s - loss: 0.5847 - accura\n",
      "Epoch 83/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5834 - accuracy: 0.7111\n",
      "Epoch 84/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5833 - accuracy: 0.7110\n",
      "Epoch 85/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5831 - accuracy: 0.7109\n",
      "Epoch 86/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5843 - accuracy: 0.7102\n",
      "Epoch 87/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5860 - accuracy: 0.7088\n",
      "Epoch 88/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5841 - accuracy: 0.7098\n",
      "Epoch 89/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5839 - accuracy: 0.7102\n",
      "Epoch 90/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5839 - accuracy: 0.7093\n",
      "Epoch 91/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5838 - accuracy: 0.7092\n",
      "Epoch 92/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5839 - accuracy: 0.7099\n",
      "Epoch 93/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5838 - accuracy: 0.7117\n",
      "Epoch 94/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5838 - accuracy: 0.7105\n",
      "Epoch 95/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5839 - accuracy: 0.7096\n",
      "Epoch 96/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5839 - accuracy: 0.7069\n",
      "Epoch 97/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5838 - accuracy: 0.7095\n",
      "Epoch 98/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5838 - accuracy: 0.7099\n",
      "Epoch 99/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5839 - accuracy: 0.70890s - loss: 0.5836 - accuracy - ETA: 0s - loss: 0.5831 - accuracy: \n",
      "Epoch 100/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5838 - accuracy: 0.7098\n",
      "Epoch 101/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5838 - accuracy: 0.7108\n",
      "Epoch 102/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5839 - accuracy: 0.7097\n",
      "Epoch 103/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5838 - accuracy: 0.7124\n",
      "Epoch 104/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5837 - accuracy: 0.7111\n",
      "Epoch 105/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5839 - accuracy: 0.7083\n",
      "Epoch 106/300\n",
      "10235/10235 [==============================] - 5s 476us/step - loss: 0.5839 - accuracy: 0.7084\n",
      "Epoch 107/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5838 - accuracy: 0.7096\n",
      "Epoch 108/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5837 - accuracy: 0.7086\n",
      "Epoch 109/300\n",
      "10235/10235 [==============================] - 5s 479us/step - loss: 0.5838 - accuracy: 0.7088\n",
      "Epoch 110/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5838 - accuracy: 0.7098\n",
      "Epoch 111/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5836 - accuracy: 0.7064\n",
      "Epoch 112/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5836 - accuracy: 0.7072\n",
      "Epoch 113/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5834 - accuracy: 0.7093\n",
      "Epoch 114/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5834 - accuracy: 0.7089\n",
      "Epoch 115/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5833 - accuracy: 0.7093\n",
      "Epoch 116/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5833 - accuracy: 0.7094\n",
      "Epoch 117/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5833 - accuracy: 0.7111\n",
      "Epoch 118/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5833 - accuracy: 0.7105\n",
      "Epoch 119/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5833 - accuracy: 0.7084\n",
      "Epoch 120/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5832 - accuracy: 0.7115\n",
      "Epoch 121/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5833 - accuracy: 0.7109\n",
      "Epoch 122/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5831 - accuracy: 0.7114\n",
      "Epoch 123/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5832 - accuracy: 0.7085\n",
      "Epoch 124/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5832 - accuracy: 0.7109\n",
      "Epoch 125/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5832 - accuracy: 0.7092\n",
      "Epoch 126/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5831 - accuracy: 0.7104\n",
      "Epoch 127/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5832 - accuracy: 0.7104\n",
      "Epoch 128/300\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5831 - accuracy: 0.7099\n",
      "Epoch 129/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5830 - accuracy: 0.70860s - loss: 0.5831 - accuracy: \n",
      "Epoch 130/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7103\n",
      "Epoch 131/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5831 - accuracy: 0.7085\n",
      "Epoch 132/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5830 - accuracy: 0.7101\n",
      "Epoch 133/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5831 - accuracy: 0.7097\n",
      "Epoch 134/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5831 - accuracy: 0.7099\n",
      "Epoch 135/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5830 - accuracy: 0.7103\n",
      "Epoch 136/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5831 - accuracy: 0.7090\n",
      "Epoch 137/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5830 - accuracy: 0.7109\n",
      "Epoch 138/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7085\n",
      "Epoch 139/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7093\n",
      "Epoch 140/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7091\n",
      "Epoch 141/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5830 - accuracy: 0.7104\n",
      "Epoch 142/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7085\n",
      "Epoch 143/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5830 - accuracy: 0.7095\n",
      "Epoch 144/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7087\n",
      "Epoch 145/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5830 - accuracy: 0.7095\n",
      "Epoch 146/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5830 - accuracy: 0.7093\n",
      "Epoch 147/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5830 - accuracy: 0.7081\n",
      "Epoch 148/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5826 - accuracy: 0.7113\n",
      "Epoch 149/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7095\n",
      "Epoch 150/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7089\n",
      "Epoch 151/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5827 - accuracy: 0.7115\n",
      "Epoch 152/300\n",
      "10235/10235 [==============================] - 5s 476us/step - loss: 0.5828 - accuracy: 0.7104\n",
      "Epoch 153/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5830 - accuracy: 0.7083\n",
      "Epoch 154/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7096\n",
      "Epoch 155/300\n",
      "10235/10235 [==============================] - 5s 476us/step - loss: 0.5830 - accuracy: 0.7073\n",
      "Epoch 156/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7115\n",
      "Epoch 157/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7101\n",
      "Epoch 158/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7106\n",
      "Epoch 159/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7096\n",
      "Epoch 160/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5830 - accuracy: 0.7105\n",
      "Epoch 161/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7112\n",
      "Epoch 162/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7094\n",
      "Epoch 163/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7076\n",
      "Epoch 164/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7108\n",
      "Epoch 165/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7106\n",
      "Epoch 166/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7104\n",
      "Epoch 167/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7120\n",
      "Epoch 168/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7089\n",
      "Epoch 169/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7101\n",
      "Epoch 170/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5830 - accuracy: 0.7092\n",
      "Epoch 171/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7093\n",
      "Epoch 172/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7104\n",
      "Epoch 173/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7096\n",
      "Epoch 174/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7106\n",
      "Epoch 175/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7098\n",
      "Epoch 176/300\n",
      "10235/10235 [==============================] - 5s 476us/step - loss: 0.5828 - accuracy: 0.7078\n",
      "Epoch 177/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7101\n",
      "Epoch 178/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7112\n",
      "Epoch 179/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5830 - accuracy: 0.7105\n",
      "Epoch 180/300\n",
      "10235/10235 [==============================] - 5s 476us/step - loss: 0.5829 - accuracy: 0.7085\n",
      "Epoch 181/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5873 - accuracy: 0.71040s - loss: 0.5869 - accuracy: 0.71\n",
      "Epoch 182/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5833 - accuracy: 0.7095\n",
      "Epoch 183/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5831 - accuracy: 0.7098\n",
      "Epoch 184/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5830 - accuracy: 0.7111\n",
      "Epoch 185/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5830 - accuracy: 0.7096\n",
      "Epoch 186/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5830 - accuracy: 0.7087\n",
      "Epoch 187/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7113\n",
      "Epoch 188/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.70840s - loss: 0.5829 - ac\n",
      "Epoch 189/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7074\n",
      "Epoch 190/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7093\n",
      "Epoch 191/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7101\n",
      "Epoch 192/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7106\n",
      "Epoch 193/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7111\n",
      "Epoch 194/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7107\n",
      "Epoch 195/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7100\n",
      "Epoch 196/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5830 - accuracy: 0.7104\n",
      "Epoch 197/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7079\n",
      "Epoch 198/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7095\n",
      "Epoch 199/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7090\n",
      "Epoch 200/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7086\n",
      "Epoch 201/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7099\n",
      "Epoch 202/300\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7109\n",
      "Epoch 203/300\n",
      "10235/10235 [==============================] - 5s 476us/step - loss: 0.5830 - accuracy: 0.7097\n",
      "Epoch 204/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7105\n",
      "Epoch 205/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.70960s - loss: 0.5838 - accura\n",
      "Epoch 206/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5827 - accuracy: 0.7083\n",
      "Epoch 207/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7085\n",
      "Epoch 208/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7097\n",
      "Epoch 209/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5827 - accuracy: 0.7095\n",
      "Epoch 210/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7112\n",
      "Epoch 211/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7086\n",
      "Epoch 212/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.70930s - loss: 0.5838 - accuracy: 0.\n",
      "Epoch 213/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5830 - accuracy: 0.7114\n",
      "Epoch 214/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7133\n",
      "Epoch 215/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5830 - accuracy: 0.7093\n",
      "Epoch 216/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.70850s - loss: 0\n",
      "Epoch 217/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7091\n",
      "Epoch 218/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7099\n",
      "Epoch 219/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7094\n",
      "Epoch 220/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7083\n",
      "Epoch 221/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5827 - accuracy: 0.7103\n",
      "Epoch 222/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7089\n",
      "Epoch 223/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7104\n",
      "Epoch 224/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7072\n",
      "Epoch 225/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7108\n",
      "Epoch 226/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7083\n",
      "Epoch 227/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7088\n",
      "Epoch 228/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.70960s - loss: 0.5850 - accura\n",
      "Epoch 229/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5827 - accuracy: 0.7085\n",
      "Epoch 230/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7096\n",
      "Epoch 231/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.70890s - loss: 0.582\n",
      "Epoch 232/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5827 - accuracy: 0.7096\n",
      "Epoch 233/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7081\n",
      "Epoch 234/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7093\n",
      "Epoch 235/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7084\n",
      "Epoch 236/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5830 - accuracy: 0.7098\n",
      "Epoch 237/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5827 - accuracy: 0.7113\n",
      "Epoch 238/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5830 - accuracy: 0.70980s - loss: 0\n",
      "Epoch 239/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7088\n",
      "Epoch 240/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7098\n",
      "Epoch 241/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7084\n",
      "Epoch 242/300\n",
      "10235/10235 [==============================] - 5s 472us/step - loss: 0.5828 - accuracy: 0.7104\n",
      "Epoch 243/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5827 - accuracy: 0.7095\n",
      "Epoch 244/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5883 - accuracy: 0.7096\n",
      "Epoch 245/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5909 - accuracy: 0.71040s - loss: 0.5948 - \n",
      "Epoch 246/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5842 - accuracy: 0.7104\n",
      "Epoch 247/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5830 - accuracy: 0.7104\n",
      "Epoch 248/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7104\n",
      "Epoch 249/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7104\n",
      "Epoch 250/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7104\n",
      "Epoch 251/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7104\n",
      "Epoch 252/300\n",
      "10235/10235 [==============================] - 5s 472us/step - loss: 0.5828 - accuracy: 0.7090\n",
      "Epoch 253/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5830 - accuracy: 0.7108\n",
      "Epoch 254/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7086\n",
      "Epoch 255/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7100\n",
      "Epoch 256/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7104\n",
      "Epoch 257/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7101\n",
      "Epoch 258/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7117\n",
      "Epoch 259/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7093\n",
      "Epoch 260/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7115\n",
      "Epoch 261/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7100\n",
      "Epoch 262/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7120\n",
      "Epoch 263/300\n",
      "10235/10235 [==============================] - 5s 476us/step - loss: 0.5829 - accuracy: 0.7095\n",
      "Epoch 264/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7101\n",
      "Epoch 265/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7071\n",
      "Epoch 266/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5826 - accuracy: 0.7099\n",
      "Epoch 267/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5824 - accuracy: 0.7103\n",
      "Epoch 268/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5831 - accuracy: 0.7101\n",
      "Epoch 269/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5827 - accuracy: 0.7070\n",
      "Epoch 270/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7081\n",
      "Epoch 271/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7093\n",
      "Epoch 272/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7091\n",
      "Epoch 273/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7114\n",
      "Epoch 274/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7084\n",
      "Epoch 275/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7100\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Epoch 276/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5827 - accuracy: 0.7113\n",
      "Epoch 277/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7086\n",
      "Epoch 278/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5827 - accuracy: 0.7089\n",
      "Epoch 279/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7076\n",
      "Epoch 280/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7111\n",
      "Epoch 281/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7102\n",
      "Epoch 282/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7085\n",
      "Epoch 283/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5827 - accuracy: 0.7085\n",
      "Epoch 284/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7120\n",
      "Epoch 285/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7129\n",
      "Epoch 286/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7097\n",
      "Epoch 287/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5827 - accuracy: 0.7083\n",
      "Epoch 288/300\n",
      "10235/10235 [==============================] - 5s 476us/step - loss: 0.5828 - accuracy: 0.7095\n",
      "Epoch 289/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7085\n",
      "Epoch 290/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7103\n",
      "Epoch 291/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7099\n",
      "Epoch 292/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7091\n",
      "Epoch 293/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7099\n",
      "Epoch 294/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7085\n",
      "Epoch 295/300\n",
      "10235/10235 [==============================] - 5s 479us/step - loss: 0.5829 - accuracy: 0.7086\n",
      "Epoch 296/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7085\n",
      "Epoch 297/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5828 - accuracy: 0.7096\n",
      "Epoch 298/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5829 - accuracy: 0.7103\n",
      "Epoch 299/300\n",
      "10235/10235 [==============================] - 5s 473us/step - loss: 0.5828 - accuracy: 0.7084\n",
      "Epoch 300/300\n",
      "10235/10235 [==============================] - 5s 475us/step - loss: 0.5829 - accuracy: 0.7082\n",
      "\n",
      "Training done in 24.307468966643015 minutes or 1458.448137998581 seconds now evaluating...\n",
      "\n",
      "10235/10235 [==============================] - 1s 127us/step\n",
      "Training Accuracy: 71.040547\n",
      "\n",
      "2274/2274 [==============================] - 0s 131us/step\n",
      "Test Accuracy: 72.647315\n"
     ]
    }
   ],
   "source": [
    "#https://keras.io/getting-started/functional-api-guide/\n",
    "\n",
    "sentence_length = 50\n",
    "wordvec_length = 96\n",
    "\n",
    "\n",
    "input_layer = Input(shape=(sentence_length,wordvec_length))\n",
    "\n",
    "#x = Embedding(input_dim=vocab_size, output_dim=wordvec_length, weights=[embedding_matrix],  trainable=True)(input_layer)\n",
    "\n",
    "#x = Conv1D(filters = 1, kernel_size = 64, strides=1, padding='same', use_bias=False)(input_layer)\n",
    "\n",
    "#x = big_block(x)\n",
    "\n",
    "#x = GlobalMaxPooling1D()(x)\n",
    "\n",
    "#conv_model.add(Conv2D (kernel_size = (5,5), strides = 5, filters = 20, input_shape=(100,100,1), activation='relu'))\n",
    "#conv_model.add(Conv2D (kernel_size = (5,5), strides = 5, filters = 4,  activation='relu'))\n",
    "\n",
    "x = Reshape((sentence_length, wordvec_length,1), input_shape=(sentence_length,wordvec_length))(input_layer)\n",
    "\n",
    "x = Conv2D (kernel_size = (3,3), strides = 3, filters = 100,  activation='relu')(x)\n",
    "\n",
    "x = Conv2D (kernel_size = (4,4), strides = 4, filters = 200,  activation='relu')(x)\n",
    "\n",
    "x = Conv2D (kernel_size = (4,8), strides = 8, filters = 400,  activation='relu')(x)\n",
    "\n",
    "x = GlobalMaxPooling2D()(x)\n",
    "\n",
    "x = Dense(400, activation='sigmoid')(x)\n",
    "\n",
    "x = Dense(200, activation='sigmoid')(x)\n",
    "\n",
    "x = Dense(100, activation='sigmoid')(x)\n",
    "\n",
    "x = Dense(50, activation='sigmoid')(x)\n",
    "\n",
    "x = Dense(25, activation='sigmoid')(x)\n",
    "\n",
    "x = Dense(10, activation='sigmoid')(x)\n",
    "\n",
    "x = Dense(5, activation='sigmoid')(x)\n",
    "\n",
    "output_layer = Dense(1, activation='sigmoid')(x)\n",
    "\n",
    "cnn_2d_model = Model(inputs = input_layer, outputs = output_layer)\n",
    "\n",
    "cnn_2d_model.summary()\n",
    "\n",
    "\n",
    "#compile\n",
    "cnn_2d_model.compile(optimizer='adam', loss='binary_crossentropy', metrics=['accuracy'])\n",
    "\n",
    "\n",
    "print()\n",
    "print(\"Starting training...\")\n",
    "print()\n",
    "\n",
    "early_stop = EarlyStopping(monitor=whatToMonitor, \n",
    "                           min_delta=callbackImprovementPercentage, #apparently percentage\n",
    "                           patience=patienceEpochs, \n",
    "                           mode='min', \n",
    "                           verbose=1)\n",
    "\n",
    "\n",
    "\n",
    "start = time.time()\n",
    "\n",
    "# fit the model\n",
    "cnn_2d_model.fit(padded_training_docs, trainLabel, epochs=epochs_to_run, verbose=1,callbacks        = [early_stop])\n",
    "\n",
    "durationSecond = time.time()-start\n",
    "\n",
    "print()\n",
    "print(\"Training done in\",durationSecond/60,\"minutes or\",durationSecond,\"seconds now evaluating...\")\n",
    "print()\n",
    "\n",
    "\n",
    "\n",
    "# evaluate the model\n",
    "loss, accuracy = cnn_2d_model.evaluate(padded_training_docs, trainLabel, verbose=1)\n",
    "print('Training Accuracy: %f' % (accuracy*100))\n",
    "\n",
    "print()\n",
    "\n",
    "\n",
    "# evaluate the model\n",
    "loss, accuracy = cnn_2d_model.evaluate(padded_test_docs, testLabel, verbose=1)\n",
    "print('Test Accuracy: %f' % (accuracy*100))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "First Network\n",
      "\n",
      "10235/10235 [==============================] - 1s 82us/step\n",
      "Training Accuracy: 92.125058\n",
      "\n",
      "2274/2274 [==============================] - 0s 82us/step\n",
      "Test Accuracy: 69.437116\n",
      "\n",
      "\n",
      "Second Network\n",
      "\n",
      "10235/10235 [==============================] - 1s 121us/step\n",
      "Training Accuracy: 71.040547\n",
      "\n",
      "2274/2274 [==============================] - 0s 124us/step\n",
      "Test Accuracy: 72.647315\n",
      "\n",
      "\n",
      "Third Network\n",
      "\n"
     ]
    }
   ],
   "source": [
    "print()\n",
    "print()\n",
    "print('First Network')\n",
    "print()\n",
    "\n",
    "\n",
    "# evaluate the model\n",
    "loss, accuracy = dense_model.evaluate(padded_training_docs, trainLabel, verbose=1)\n",
    "print('Training Accuracy: %f' % (accuracy*100))\n",
    "\n",
    "print()\n",
    "\n",
    "\n",
    "# evaluate the model\n",
    "loss, accuracy = dense_model.evaluate(padded_test_docs, testLabel, verbose=1)\n",
    "print('Test Accuracy: %f' % (accuracy*100))\n",
    "\n",
    "\n",
    "print()\n",
    "print()\n",
    "print('Second Network')\n",
    "print()\n",
    "\n",
    "\n",
    "# evaluate the model\n",
    "loss, accuracy = cnn_2d_model.evaluate(padded_training_docs, trainLabel, verbose=1)\n",
    "print('Training Accuracy: %f' % (accuracy*100))\n",
    "\n",
    "print()\n",
    "\n",
    "\n",
    "# evaluate the model\n",
    "loss, accuracy = cnn_2d_model.evaluate(padded_test_docs, testLabel, verbose=1)\n",
    "print('Test Accuracy: %f' % (accuracy*100))\n",
    "\n",
    "\n",
    "\n",
    "print()\n",
    "print()\n",
    "print('Third Network')\n",
    "print()\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
